<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A multiplex, prime editing framework for identifying drug resistance variants at scale
Authors: Chardon, F. M.; Suiter, C.; Daza, R.; Smith, N. T.; Parrish, P.; McDiarmid, T.; Lalanne, J.-B.; Martin, B. K.; Calderon, D.; Ellison, A.; Berger, A.; Shendure, J.; Starita, L. M.
Score: 38.3, Published: 2023-07-30 DOI: 10.1101/2023.07.27.550902
CRISPR-based genome editing has revolutionized functional genomics, enabling screens in which thousands of perturbations of either gene expression or primary genome sequence can be competitively assayed in single experiments.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="A multiplex, prime editing framework for identifying drug resistance variants at scale
Authors: Chardon, F. M.; Suiter, C.; Daza, R.; Smith, N. T.; Parrish, P.; McDiarmid, T.; Lalanne, J.-B.; Martin, B. K.; Calderon, D.; Ellison, A.; Berger, A.; Shendure, J.; Starita, L. M.
Score: 38.3, Published: 2023-07-30 DOI: 10.1101/2023.07.27.550902
CRISPR-based genome editing has revolutionized functional genomics, enabling screens in which thousands of perturbations of either gene expression or primary genome sequence can be competitively assayed in single experiments." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-06T10:36:48+00:00" />
<meta property="article:modified_time" content="2023-08-06T10:36:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="A multiplex, prime editing framework for identifying drug resistance variants at scale
Authors: Chardon, F. M.; Suiter, C.; Daza, R.; Smith, N. T.; Parrish, P.; McDiarmid, T.; Lalanne, J.-B.; Martin, B. K.; Calderon, D.; Ellison, A.; Berger, A.; Shendure, J.; Starita, L. M.
Score: 38.3, Published: 2023-07-30 DOI: 10.1101/2023.07.27.550902
CRISPR-based genome editing has revolutionized functional genomics, enabling screens in which thousands of perturbations of either gene expression or primary genome sequence can be competitively assayed in single experiments."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "A multiplex, prime editing framework for identifying drug resistance variants at scale\nAuthors: Chardon, F. M.; Suiter, C.; Daza, R.; Smith, N. T.; Parrish, P.; McDiarmid, T.; Lalanne, J.-B.; Martin, B. K.; Calderon, D.; Ellison, A.; Berger, A.; Shendure, J.; Starita, L. M.\nScore: 38.3, Published: 2023-07-30 DOI: 10.1101/2023.07.27.550902\nCRISPR-based genome editing has revolutionized functional genomics, enabling screens in which thousands of perturbations of either gene expression or primary genome sequence can be competitively assayed in single experiments.",
  "keywords": [
    
  ],
  "articleBody": " A multiplex, prime editing framework for identifying drug resistance variants at scale\nAuthors: Chardon, F. M.; Suiter, C.; Daza, R.; Smith, N. T.; Parrish, P.; McDiarmid, T.; Lalanne, J.-B.; Martin, B. K.; Calderon, D.; Ellison, A.; Berger, A.; Shendure, J.; Starita, L. M.\nScore: 38.3, Published: 2023-07-30 DOI: 10.1101/2023.07.27.550902\nCRISPR-based genome editing has revolutionized functional genomics, enabling screens in which thousands of perturbations of either gene expression or primary genome sequence can be competitively assayed in single experiments. However, for libraries of specific mutations, a challenge of CRISPR-based screening methods such as saturation genome editing is that only one region (e.g. one exon) can be studied per experiment. Here we describe prime-SGE (\"prime saturation genome editing\"), a new framework based on prime editing, in which libraries of specific mutations can be installed into genes throughout the genome and functionally assessed in a single, multiplex experiment. Prime-SGE is based on quantifying the abundance of prime editing guide RNAs (pegRNAs) in the context of a functional selection, rather than quantifying the mutations themselves. We apply prime-SGE to assay thousands of single nucleotide changes in eight oncogenes for their ability to confer drug resistance to three EGFR tyrosine kinase inhibitors. Although currently restricted to positive selection screens by the limited efficiency of prime editing, our strategy opens the door to the possibility of functionally assaying vast numbers of precise mutations at locations throughout the genome.\nSingle-cell gene expression prediction from DNA sequence at large contexts\nAuthors: Schwessinger, R.; Deasy, J.; Woodruff, R. T.; Young, S.; Branson, K. M.\nScore: 29.7, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550634\nHuman genetic variants impacting traits such as disease susceptibility frequently act through modulation of gene expression in a highly cell-type-specific manner. Computational models capable of predicting gene expression directly from DNA sequence can assist in the interpretation of expression-modulating variants, and machine learning models now operate at the large sequence contexts required for capturing long-range human transcriptional regulation. However, existing predictors have focused on bulk transcriptional measurements where gene expression heterogeneity can be drowned out in broadly defined cell types. Here, we use a transfer learning framework, seq2cells, leveraging a pre-trained epigenome model for gene expression prediction from large sequence contexts at single-cell resolution. We show that seq2cells captures cell-specific gene expression beyond the resolution of pseudo-bulked data. Using seq2cells for variant effect prediction reveals heterogeneity within annotated cell types and enables in silico transfer of variant effects between cell populations. We demonstrate the challenges and value of gene expression and variant effect prediction at single-cell resolution, and offer a path to the interpretation of genomic variation at uncompromising resolution and scale.\nCentury-old chromatin architecture preserved with formaldehyde\nAuthors: Hahn, E. E.; Stiller, J.; Alexander, M. R.; Grealy, A.; Taylor, J. M.; Jackson, N.; Frere, C. H.; Holleley, C. E.\nScore: 21.2, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550239\nCo-ordinated regulation or dysregulation of chromatin architecture underpins fundamental biological processes, such as embryonic development, disease, cellular programming and response to environmental stress. The dynamic and plastic nature of chromatin accessibility is a major driver of phenotypic diversity, but we know shockingly little about the temporal dynamics of chromatin reorganisation and almost nothing prior to the existence of flash-frozen specimens. Linking two disparate fields by their common use and application of the preservative formaldehyde, we present an approach to characterise chromatin architecture in formaldehyde-preserved specimens up to 117 years old. We characterise how over-fixation modulates but does not eliminate genome-wide patterns of differential chromatin accessibility. Our novel analytical approach identifies promoter regions enriched for gene ontology terms matching the tissue of origin, resulting in sex-specific and environment-dependent genome-wide profiles. Contrary to prevailing dogma, we show that over-fixation is essential for the successful recovery of historical chromatin architecture. Our methodological and analytical advances open the door to the first detailed and comprehensive view of the epigenetic past and reveal a new role for museum collections in understanding chromatin architecture dynamics over the last century.\nRegulatory features aid interpretation of 3'UTR variants\nAuthors: Romo, L.; Burge, C. B.; Findlay, S. D.\nScore: 11.6, Published: 2023-08-02 DOI: 10.1101/2023.08.01.551549\nOur ability to determine the clinical impact of variants in 3 untranslated regions (UTRs) of genes remains poor. We provide a thorough analysis of 3UTR variants from several datasets. Variants in putative regulatory elements including RNA-binding protein motifs, eCLIP peaks, and microRNA sites are up to 16 times more likely than other variants to have gene expression and phenotype associations. Heterozygous variants in regulatory motifs result in allele-specific protein binding in cell lines and allele-specific gene expression differences in population studies. In addition, variants in shared regions of alternatively polyadenylated isoforms and those proximal to polyA sites are more likely to affect gene expression and phenotype. Finally, pathogenic 3UTR variants in ClinVar are 20 times more likely than benign variants to fall in a regulatory site. We incorporated these findings into RegVar, a software tool that interprets regulatory elements and annotations for any 3UTR variant, and predicts whether the variant is likely to affect gene expression or phenotype. This tool will help prioritize variants for experimental studies and identify pathogenic variants in patients.\nKoNA: Korean Nucleotide Archive as a New Data Repository for Nucleotide Sequence Data\nAuthors: Ko, G.; Lee, J.; Sim, Y. M.; Song, W.; Yoon, B.-H.; Byeon, I.; Lee, B. H.; Kim, S.-O.; Choi, J.; Jang, I.; Kim, H.; Yang, J. O.; Jang, K.; Kim, S.; Kim, J.-H.; Jeon, J.; Jung, J.; Hwang, S.; Park, J.-H.; Kim, P.-G.; Kim, S.-Y.; Lee, B.\nScore: 11.5, Published: 2023-07-29 DOI: 10.1101/2023.07.27.550450\nDuring the last decade, generation and accumulation of petabase-scale high-throughput sequencing data have resulted in ethical and technical challenges, including access to human data, and transfer, storage, and sharing of enormous amount of data. To promote data-driven research in biology, the Korean government announced that all the biological data generated from government-funded research projects should be deposited in the Korea BioData Station (K-BDS), which consists of multiple databases for individual data types. We introduce the Korean Nucleotide Archive (KoNA), a repository for nucleotide sequence data. As of July 2022, the Korean Read Archive in KoNA has collected over 477 TB of raw next generation sequencing data from several national genome projects. To ensure data quality and prepare for international alignment, a standard operating procedure (SOP) was adopted, which is similar to the International Nucleotide Sequence Database Collaboration. The SOP includes quality control processes for submitted data and metadata using an automated pipeline followed by manual examination. To ensure fast and stable data transfer, a high-speed transmission system called GBox is used in KoNA. Furthermore, the data uploaded to or downloaded from KoNA through GBox can be readily processed in a cloud-computing service for genomic data analysis called Bio-Express. This seamless coupling of KoNA, GBox, and Bio-Express enhances data experience including submission, access, and analysis of raw nucleotide sequences. KoNA not only satisfies the unmet needs for a national sequence repository in Korea, but also provides datasets to researchers globally and contribute to advances in genomics. KoNA is available at https://www.kobic.re.kr/kona/.\nThe Genomic Legacy of Human Management and sex-biased Aurochs hybridization in Iberian Cattle\nAuthors: Günther, T.; Chisausky, J.; Galindo-Pellicena, M. A.; Iriarte, E.; Cortes Gardyn, O.; Eusebi, P. G.; Garcia-Gonzalez, R.; Urena, I.; Moreno, M.; Alday, A.; Rojo, M.; Perez, A.; Tejedor Rodriguez, C.; Garcia Martinez de Lagran, I.; Arsuaga, J. L.; Carretero, J.-M.; Götherström, A.; Smith, C.; Valdiosera, C.\nScore: 12.6, Published: 2023-07-20 DOI: 10.1101/2023.07.19.548914\nCattle have been a valuable economic resource and cultural icon since prehistory. From the initial expansion of domestic cattle into Europe during the Neolithic period, taurine cattle (Bos taurus) and their wild ancestor, the aurochs (B. primigenius), had overlapping ranges leading to ample opportunities for intentional and unintentional hybridization. We performed a bioarchaeological analysis of 24 Bos remains from Iberia dating from the Mesolithic to the Roman period. The archaeogenomic dataset allows us to investigate the extent of domestic-wild hybridization over time, providing insight into the species behavior and human management by aligning changes with cultural and genomic transitions in the archaeological record. Our results show frequent hybridization during the Neolithic and Chalcolithic, likely reflecting a mix of hunting and herding or relatively unmanaged herds, with mostly male aurochs and female domestic cattle involved in hybridization. This is supported by isotopic evidence of ecological niche sharing, with only a few domestic cattle possibly being managed. The proportion of aurochs ancestry in domestic cattle remains relatively constant from about 4000 years ago, probably due to herd management and selection against hybrids, coinciding with other cultural transitions. The constant level of wild ancestry ([~]20%) continues into modern western European breeds including the Spanish Lidia breed which is bred for its aggressiveness and fighting ability, but does not display elevated levels of aurochs ancestry. This study takes a genomic glance at the impact of human actions and wild introgression in the establishment of cattle as one of the most important domestic species today.\nSplam: a deep-learning-based splice site predictor that improves spliced alignments\nAuthors: Chao, K.-H.; Mao, A.; Salzberg, S. L.; Pertea, M.\nScore: 11.6, Published: 2023-07-29 DOI: 10.1101/2023.07.27.550754\nThe process of splicing messenger RNA to remove introns plays a central role in creating genes and gene variants. Here we describe Splam, a novel method for predicting splice junctions in DNA based on deep residual convolutional neural networks. Unlike some previous models, Splam looks at a relatively limited window of 400 base pairs flanking each splice site, motivated by the observation that the biological process of splicing relies primarily on signals within this window. Additionally, Splam introduces the idea of training the network on donor and acceptor pairs together, based on the principal that the splicing machinery recognizes both ends of each intron at once. We compare Splams accuracy to recent state-of-the-art splice site prediction methods, particularly SpliceAI, another method that uses deep neural networks. Our results show that Splam is consistently more accurate than SpliceAI, with an overall accuracy of 96% at predicting human splice junctions. Splam generalizes even to non- human species, including distant ones like the flowering plant Arabidopsis thaliana. Finally, we demonstrate the use of Splam on a novel application: processing the spliced alignments of RNA-seq data in order to identify and eliminate errors. We show that when used in this manner, Splam yields substantial improvements in the accuracy of downstream transcriptome analysis on both poly(A) and ribo-depleted RNA-seq libraries. Overall, Splam offers a faster and more accurate approach to detecting splice junctions, while also providing a reliable and efficient solution for cleaning up erroneous spliced alignments.\nGenetic regulatory effects in response to a high cholesterol, high fat diet in baboons\nAuthors: Lin, W.; Wall, J.; Li, G.; Newman, D.; VandeBerg, J.; Yang, Y.; Abney, M.; Olivier, M.; Gilad, Y.; Cox, L. A.\nScore: 8.6, Published: 2023-08-03 DOI: 10.1101/2023.08.01.551489\nDisease-associated loci discovered through genome-wide association studies (GWAS) are primarily located in non-coding regions with putative regulatory effects on gene expression1. Steady-state, standard expression quantitative trait loci (eQTLs) explain only a limited portion of GWAS signals2-6, while eQTLs involved in gene-by-environment (GxE) interactions have rarely been characterized in humans due to experimental challenges. Here, we characterized gene-by-diet effects in a baboon model system. By analyzing three tissue types obtained from 99 captive baboons, we identify hundreds of diet-responsive eQTLs with high tissue specificity. Diet-responsive eQTLs exhibit genomic localization and genic features that are distinct from steady-state eQTLs. Furthermore, the human orthologs of genes associated with diet-responsive eQTLs are enriched for GWAS genes associated with human metabolic traits, suggesting that context-responsive eQTLs with more complex regulatory effects are likely to explain GWAS hits that do not seem to overlap with standard eQTLs. Our results highlight the dynamic complexity of genetic regulatory effects and the potential of eQTLs with disease-relevant GxE interactions in enhancing the understanding of GWAS signals for human complex disease using the baboon model.\nSystematic assessment of long-read RNA-seq methods for transcript identification and quantification\nAuthors: Pardo-Palacios, F. J.; Wang, D.; Reese, F.; Diekhans, M.; Carbonell-Sala, S.; Williams, B.; Loveland, J. E.; De Maria, M.; Adams, M. S.; Balderrama-Gutierrez, G.; Behera, A. K.; Gonzalez, J. M.; Hunt, T.; Lagarde, J.; Liang, C. E.; Li, H.; Meade, M. J.; Amador, D. A. M.; Prjibelski, A. D.; Birol, I.; Bostan, H.; Brooks, A. M.; Celik, M. H.; Chen, Y.; Du, M. R. M.; Felton, C.; Goke, J.; Hafezqorani, S.; Herwig, R.; Kawaji, H.; Lee, J.; Li, J.-L.; Lienhard, M.; Mikheenko, A.; Mulligan, D.; Nip, K. M.; Pertea, M.; Ritchie, M. E.; Sim, A. D.; Tang, A. D.; Wan, Y. K.; Wang, C.; Wong, B. Y.; Yang, C\nScore: 40.7, Published: 2023-07-27 DOI: 10.1101/2023.07.25.550582\nAbstractThe Long-read RNA-Seq Genome Annotation Assessment Project (LRGASP) Consortium was formed to evaluate the effectiveness of long-read approaches for transcriptome analysis. The consortium generated over 427 million long-read sequences from cDNA and direct RNA datasets, encompassing human, mouse, and manatee species, using different protocols and sequencing platforms. These data were utilized by developers to address challenges in transcript isoform detection and quantification, as well as de novo transcript isoform identification. The study revealed that libraries with longer, more accurate sequences produce more accurate transcripts than those with increased read depth, whereas greater read depth improved quantification accuracy. In well-annotated genomes, tools based on reference sequences demonstrated the best performance. When aiming to detect rare and novel transcripts or when using reference-free approaches, incorporating additional orthogonal data and replicate samples are advised. This collaborative study offers a benchmark for current practices and provides direction for future method development in transcriptome analysis.\nBlueberry and cranberry pangenomes as a resource for future genetic studies and breeding efforts\nAuthors: Yocca, A. E.; Platts, A.; Alger, E.; Teresi, S.; Mengist, M. F.; Benevenuto, J.; Felipe V. Ferrao, L.; Jacobs, M.; Babinski, M.; Magallanes-Lundback, M.; Bayer, P.; Golicz, A.; Humann, J. L.; Main, D.; Espley, R. V.; Chagne, D.; Albert, N. W.; Montanari, S.; Vorsa, N.; Polashock, J.; Diaz-Garcia, L.; Zalapa, J.; Bassil, N. V.; Munoz, P. R.; Iorizzo, M.; Edger, P. P.\nScore: 7.4, Published: 2023-08-02 DOI: 10.1101/2023.07.31.551392\nDomestication of cranberry and blueberry began in the United States in the early 1800s and 1900s, respectively, and in part owing to their flavors and health-promoting benefits are now cultivated and consumed worldwide. The industry continues to face a wide variety of production challenges (e.g. disease pressures) as well as a demand for higher-yielding cultivars with improved fruit quality characteristics. Unfortunately, molecular tools to help guide breeding efforts for these species have been relatively limited compared with those for other high-value crops. Here, we describe the construction and analysis of the first pangenome for both blueberry and cranberry. Our analysis of these pangenomes revealed both crops exhibit great genetic diversity, including the presence-absence variation of 48.4% genes in highbush blueberry and 47.0% genes in cranberry. Auxiliary genes, those not shared by all cultivars, are significantly enriched with molecular functions associated with disease resistance and the biosynthesis of specialized metabolites, including compounds previously associated with improving fruit quality traits. The discovery of thousands of genes, not present in the previous reference genomes for blueberry and cranberry, will serve as the basis of future research and as potential targets for future breeding efforts. The pangenome, as a multiple-sequence alignment, as well as individual annotated genomes, are publicly available for analysis on the Genome Database for Vaccinium - a curated and integrated web-based relational database. Lastly, the core-gene predictions from the pangenomes will serve useful to develop a community genotyping platform to guide future molecular breeding efforts across the family.\n",
  "wordCount" : "2570",
  "inLanguage": "en",
  "datePublished": "2023-08-06T10:36:48Z",
  "dateModified": "2023-08-06T10:36:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta"><span>updated on August 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550902">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550902" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550902">
        <p class="paperTitle">A multiplex, prime editing framework for identifying drug resistance variants at scale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550902" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550902" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chardon, F. M.; Suiter, C.; Daza, R.; Smith, N. T.; Parrish, P.; McDiarmid, T.; Lalanne, J.-B.; Martin, B. K.; Calderon, D.; Ellison, A.; Berger, A.; Shendure, J.; Starita, L. M.</p>
        <p class="info">Score: 38.3, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550902' target='https://doi.org/10.1101/2023.07.27.550902'> 10.1101/2023.07.27.550902</a></p>
        <p class="abstract">CRISPR-based genome editing has revolutionized functional genomics, enabling screens in which thousands of perturbations of either gene expression or primary genome sequence can be competitively assayed in single experiments. However, for libraries of specific mutations, a challenge of CRISPR-based screening methods such as saturation genome editing is that only one region (e.g. one exon) can be studied per experiment. Here we describe prime-SGE (&#34;prime saturation genome editing&#34;), a new framework based on prime editing, in which libraries of specific mutations can be installed into genes throughout the genome and functionally assessed in a single, multiplex experiment. Prime-SGE is based on quantifying the abundance of prime editing guide RNAs (pegRNAs) in the context of a functional selection, rather than quantifying the mutations themselves. We apply prime-SGE to assay thousands of single nucleotide changes in eight oncogenes for their ability to confer drug resistance to three EGFR tyrosine kinase inhibitors. Although currently restricted to positive selection screens by the limited efficiency of prime editing, our strategy opens the door to the possibility of functionally assaying vast numbers of precise mutations at locations throughout the genome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550634">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550634" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550634">
        <p class="paperTitle">Single-cell gene expression prediction from DNA sequence at large contexts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550634" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550634" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schwessinger, R.; Deasy, J.; Woodruff, R. T.; Young, S.; Branson, K. M.</p>
        <p class="info">Score: 29.7, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550634' target='https://doi.org/10.1101/2023.07.26.550634'> 10.1101/2023.07.26.550634</a></p>
        <p class="abstract">Human genetic variants impacting traits such as disease susceptibility frequently act through modulation of gene expression in a highly cell-type-specific manner. Computational models capable of predicting gene expression directly from DNA sequence can assist in the interpretation of expression-modulating variants, and machine learning models now operate at the large sequence contexts required for capturing long-range human transcriptional regulation. However, existing predictors have focused on bulk transcriptional measurements where gene expression heterogeneity can be drowned out in broadly defined cell types. Here, we use a transfer learning framework, seq2cells, leveraging a pre-trained epigenome model for gene expression prediction from large sequence contexts at single-cell resolution. We show that seq2cells captures cell-specific gene expression beyond the resolution of pseudo-bulked data. Using seq2cells for variant effect prediction reveals heterogeneity within annotated cell types and enables in silico transfer of variant effects between cell populations. We demonstrate the challenges and value of gene expression and variant effect prediction at single-cell resolution, and offer a path to the interpretation of genomic variation at uncompromising resolution and scale.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550239">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550239" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550239">
        <p class="paperTitle">Century-old chromatin architecture preserved with formaldehyde</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550239" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550239" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hahn, E. E.; Stiller, J.; Alexander, M. R.; Grealy, A.; Taylor, J. M.; Jackson, N.; Frere, C. H.; Holleley, C. E.</p>
        <p class="info">Score: 21.2, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550239' target='https://doi.org/10.1101/2023.07.26.550239'> 10.1101/2023.07.26.550239</a></p>
        <p class="abstract">Co-ordinated regulation or dysregulation of chromatin architecture underpins fundamental biological processes, such as embryonic development, disease, cellular programming and response to environmental stress. The dynamic and plastic nature of chromatin accessibility is a major driver of phenotypic diversity, but we know shockingly little about the temporal dynamics of chromatin reorganisation and almost nothing prior to the existence of flash-frozen specimens. Linking two disparate fields by their common use and application of the preservative formaldehyde, we present an approach to characterise chromatin architecture in formaldehyde-preserved specimens up to 117 years old. We characterise how over-fixation modulates but does not eliminate genome-wide patterns of differential chromatin accessibility. Our novel analytical approach identifies promoter regions enriched for gene ontology terms matching the tissue of origin, resulting in sex-specific and environment-dependent genome-wide profiles. Contrary to prevailing dogma, we show that over-fixation is essential for the successful recovery of historical chromatin architecture. Our methodological and analytical advances open the door to the first detailed and comprehensive view of the epigenetic past and reveal a new role for museum collections in understanding chromatin architecture dynamics over the last century.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551549">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551549" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551549">
        <p class="paperTitle">Regulatory features aid interpretation of 3&#39;UTR variants</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551549" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551549" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Romo, L.; Burge, C. B.; Findlay, S. D.</p>
        <p class="info">Score: 11.6, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551549' target='https://doi.org/10.1101/2023.08.01.551549'> 10.1101/2023.08.01.551549</a></p>
        <p class="abstract">Our ability to determine the clinical impact of variants in 3 untranslated regions (UTRs) of genes remains poor. We provide a thorough analysis of 3UTR variants from several datasets. Variants in putative regulatory elements including RNA-binding protein motifs, eCLIP peaks, and microRNA sites are up to 16 times more likely than other variants to have gene expression and phenotype associations. Heterozygous variants in regulatory motifs result in allele-specific protein binding in cell lines and allele-specific gene expression differences in population studies. In addition, variants in shared regions of alternatively polyadenylated isoforms and those proximal to polyA sites are more likely to affect gene expression and phenotype. Finally, pathogenic 3UTR variants in ClinVar are 20 times more likely than benign variants to fall in a regulatory site. We incorporated these findings into RegVar, a software tool that interprets regulatory elements and annotations for any 3UTR variant, and predicts whether the variant is likely to affect gene expression or phenotype. This tool will help prioritize variants for experimental studies and identify pathogenic variants in patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550450">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550450" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550450">
        <p class="paperTitle">KoNA: Korean Nucleotide Archive as a New Data Repository for Nucleotide Sequence Data</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550450" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550450" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ko, G.; Lee, J.; Sim, Y. M.; Song, W.; Yoon, B.-H.; Byeon, I.; Lee, B. H.; Kim, S.-O.; Choi, J.; Jang, I.; Kim, H.; Yang, J. O.; Jang, K.; Kim, S.; Kim, J.-H.; Jeon, J.; Jung, J.; Hwang, S.; Park, J.-H.; Kim, P.-G.; Kim, S.-Y.; Lee, B.</p>
        <p class="info">Score: 11.5, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550450' target='https://doi.org/10.1101/2023.07.27.550450'> 10.1101/2023.07.27.550450</a></p>
        <p class="abstract">During the last decade, generation and accumulation of petabase-scale high-throughput sequencing data have resulted in ethical and technical challenges, including access to human data, and transfer, storage, and sharing of enormous amount of data. To promote data-driven research in biology, the Korean government announced that all the biological data generated from government-funded research projects should be deposited in the Korea BioData Station (K-BDS), which consists of multiple databases for individual data types. We introduce the Korean Nucleotide Archive (KoNA), a repository for nucleotide sequence data. As of July 2022, the Korean Read Archive in KoNA has collected over 477 TB of raw next generation sequencing data from several national genome projects. To ensure data quality and prepare for international alignment, a standard operating procedure (SOP) was adopted, which is similar to the International Nucleotide Sequence Database Collaboration. The SOP includes quality control processes for submitted data and metadata using an automated pipeline followed by manual examination. To ensure fast and stable data transfer, a high-speed transmission system called GBox is used in KoNA. Furthermore, the data uploaded to or downloaded from KoNA through GBox can be readily processed in a cloud-computing service for genomic data analysis called Bio-Express. This seamless coupling of KoNA, GBox, and Bio-Express enhances data experience including submission, access, and analysis of raw nucleotide sequences. KoNA not only satisfies the unmet needs for a national sequence repository in Korea, but also provides datasets to researchers globally and contribute to advances in genomics. KoNA is available at https://www.kobic.re.kr/kona/.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.548914">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.548914" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.548914">
        <p class="paperTitle">The Genomic Legacy of Human Management and sex-biased Aurochs hybridization in Iberian Cattle</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.548914" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.548914" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Günther, T.; Chisausky, J.; Galindo-Pellicena, M. A.; Iriarte, E.; Cortes Gardyn, O.; Eusebi, P. G.; Garcia-Gonzalez, R.; Urena, I.; Moreno, M.; Alday, A.; Rojo, M.; Perez, A.; Tejedor Rodriguez, C.; Garcia Martinez de Lagran, I.; Arsuaga, J. L.; Carretero, J.-M.; Götherström, A.; Smith, C.; Valdiosera, C.</p>
        <p class="info">Score: 12.6, Published: 2023-07-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.548914' target='https://doi.org/10.1101/2023.07.19.548914'> 10.1101/2023.07.19.548914</a></p>
        <p class="abstract">Cattle have been a valuable economic resource and cultural icon since prehistory. From the initial expansion of domestic cattle into Europe during the Neolithic period, taurine cattle (Bos taurus) and their wild ancestor, the aurochs (B. primigenius), had overlapping ranges leading to ample opportunities for intentional and unintentional hybridization. We performed a bioarchaeological analysis of 24 Bos remains from Iberia dating from the Mesolithic to the Roman period. The archaeogenomic dataset allows us to investigate the extent of domestic-wild hybridization over time, providing insight into the species behavior and human management by aligning changes with cultural and genomic transitions in the archaeological record. Our results show frequent hybridization during the Neolithic and Chalcolithic, likely reflecting a mix of hunting and herding or relatively unmanaged herds, with mostly male aurochs and female domestic cattle involved in hybridization. This is supported by isotopic evidence of ecological niche sharing, with only a few domestic cattle possibly being managed. The proportion of aurochs ancestry in domestic cattle remains relatively constant from about 4000 years ago, probably due to herd management and selection against hybrids, coinciding with other cultural transitions. The constant level of wild ancestry ([~]20%) continues into modern western European breeds including the Spanish Lidia breed which is bred for its aggressiveness and fighting ability, but does not display elevated levels of aurochs ancestry. This study takes a genomic glance at the impact of human actions and wild introgression in the establishment of cattle as one of the most important domestic species today.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.27.550754">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.27.550754" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.27.550754">
        <p class="paperTitle">Splam: a deep-learning-based splice site predictor that improves spliced alignments</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.27.550754" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.27.550754" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chao, K.-H.; Mao, A.; Salzberg, S. L.; Pertea, M.</p>
        <p class="info">Score: 11.6, Published: 2023-07-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.27.550754' target='https://doi.org/10.1101/2023.07.27.550754'> 10.1101/2023.07.27.550754</a></p>
        <p class="abstract">The process of splicing messenger RNA to remove introns plays a central role in creating genes and gene variants. Here we describe Splam, a novel method for predicting splice junctions in DNA based on deep residual convolutional neural networks. Unlike some previous models, Splam looks at a relatively limited window of 400 base pairs flanking each splice site, motivated by the observation that the biological process of splicing relies primarily on signals within this window. Additionally, Splam introduces the idea of training the network on donor and acceptor pairs together, based on the principal that the splicing machinery recognizes both ends of each intron at once. We compare Splams accuracy to recent state-of-the-art splice site prediction methods, particularly SpliceAI, another method that uses deep neural networks. Our results show that Splam is consistently more accurate than SpliceAI, with an overall accuracy of 96% at predicting human splice junctions. Splam generalizes even to non- human species, including distant ones like the flowering plant Arabidopsis thaliana. Finally, we demonstrate the use of Splam on a novel application: processing the spliced alignments of RNA-seq data in order to identify and eliminate errors. We show that when used in this manner, Splam yields substantial improvements in the accuracy of downstream transcriptome analysis on both poly(A) and ribo-depleted RNA-seq libraries. Overall, Splam offers a faster and more accurate approach to detecting splice junctions, while also providing a reliable and efficient solution for cleaning up erroneous spliced alignments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551489">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551489" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551489">
        <p class="paperTitle">Genetic regulatory effects in response to a high cholesterol, high fat diet in baboons</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551489" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551489" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lin, W.; Wall, J.; Li, G.; Newman, D.; VandeBerg, J.; Yang, Y.; Abney, M.; Olivier, M.; Gilad, Y.; Cox, L. A.</p>
        <p class="info">Score: 8.6, Published: 2023-08-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551489' target='https://doi.org/10.1101/2023.08.01.551489'> 10.1101/2023.08.01.551489</a></p>
        <p class="abstract">Disease-associated loci discovered through genome-wide association studies (GWAS) are primarily located in non-coding regions with putative regulatory effects on gene expression1. Steady-state, standard expression quantitative trait loci (eQTLs) explain only a limited portion of GWAS signals2-6, while eQTLs involved in gene-by-environment (GxE) interactions have rarely been characterized in humans due to experimental challenges. Here, we characterized gene-by-diet effects in a baboon model system. By analyzing three tissue types obtained from 99 captive baboons, we identify hundreds of diet-responsive eQTLs with high tissue specificity. Diet-responsive eQTLs exhibit genomic localization and genic features that are distinct from steady-state eQTLs. Furthermore, the human orthologs of genes associated with diet-responsive eQTLs are enriched for GWAS genes associated with human metabolic traits, suggesting that context-responsive eQTLs with more complex regulatory effects are likely to explain GWAS hits that do not seem to overlap with standard eQTLs. Our results highlight the dynamic complexity of genetic regulatory effects and the potential of eQTLs with disease-relevant GxE interactions in enhancing the understanding of GWAS signals for human complex disease using the baboon model.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550582">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550582" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550582">
        <p class="paperTitle">Systematic assessment of long-read RNA-seq methods for transcript identification and quantification</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550582" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550582" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pardo-Palacios, F. J.; Wang, D.; Reese, F.; Diekhans, M.; Carbonell-Sala, S.; Williams, B.; Loveland, J. E.; De Maria, M.; Adams, M. S.; Balderrama-Gutierrez, G.; Behera, A. K.; Gonzalez, J. M.; Hunt, T.; Lagarde, J.; Liang, C. E.; Li, H.; Meade, M. J.; Amador, D. A. M.; Prjibelski, A. D.; Birol, I.; Bostan, H.; Brooks, A. M.; Celik, M. H.; Chen, Y.; Du, M. R. M.; Felton, C.; Goke, J.; Hafezqorani, S.; Herwig, R.; Kawaji, H.; Lee, J.; Li, J.-L.; Lienhard, M.; Mikheenko, A.; Mulligan, D.; Nip, K. M.; Pertea, M.; Ritchie, M. E.; Sim, A. D.; Tang, A. D.; Wan, Y. K.; Wang, C.; Wong, B. Y.; Yang, C</p>
        <p class="info">Score: 40.7, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550582' target='https://doi.org/10.1101/2023.07.25.550582'> 10.1101/2023.07.25.550582</a></p>
        <p class="abstract">AbstractThe Long-read RNA-Seq Genome Annotation Assessment Project (LRGASP) Consortium was formed to evaluate the effectiveness of long-read approaches for transcriptome analysis. The consortium generated over 427 million long-read sequences from cDNA and direct RNA datasets, encompassing human, mouse, and manatee species, using different protocols and sequencing platforms. These data were utilized by developers to address challenges in transcript isoform detection and quantification, as well as de novo transcript isoform identification. The study revealed that libraries with longer, more accurate sequences produce more accurate transcripts than those with increased read depth, whereas greater read depth improved quantification accuracy. In well-annotated genomes, tools based on reference sequences demonstrated the best performance. When aiming to detect rare and novel transcripts or when using reference-free approaches, incorporating additional orthogonal data and replicate samples are advised. This collaborative study offers a benchmark for current practices and provides direction for future method development in transcriptome analysis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551392">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551392" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551392">
        <p class="paperTitle">Blueberry and cranberry pangenomes as a resource for future genetic studies and breeding efforts</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551392" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551392" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yocca, A. E.; Platts, A.; Alger, E.; Teresi, S.; Mengist, M. F.; Benevenuto, J.; Felipe V. Ferrao, L.; Jacobs, M.; Babinski, M.; Magallanes-Lundback, M.; Bayer, P.; Golicz, A.; Humann, J. L.; Main, D.; Espley, R. V.; Chagne, D.; Albert, N. W.; Montanari, S.; Vorsa, N.; Polashock, J.; Diaz-Garcia, L.; Zalapa, J.; Bassil, N. V.; Munoz, P. R.; Iorizzo, M.; Edger, P. P.</p>
        <p class="info">Score: 7.4, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551392' target='https://doi.org/10.1101/2023.07.31.551392'> 10.1101/2023.07.31.551392</a></p>
        <p class="abstract">Domestication of cranberry and blueberry began in the United States in the early 1800s and 1900s, respectively, and in part owing to their flavors and health-promoting benefits are now cultivated and consumed worldwide. The industry continues to face a wide variety of production challenges (e.g. disease pressures) as well as a demand for higher-yielding cultivars with improved fruit quality characteristics. Unfortunately, molecular tools to help guide breeding efforts for these species have been relatively limited compared with those for other high-value crops. Here, we describe the construction and analysis of the first pangenome for both blueberry and cranberry. Our analysis of these pangenomes revealed both crops exhibit great genetic diversity, including the presence-absence variation of 48.4% genes in highbush blueberry and 47.0% genes in cranberry. Auxiliary genes, those not shared by all cultivars, are significantly enriched with molecular functions associated with disease resistance and the biosynthesis of specialized metabolites, including compounds previously associated with improving fruit quality traits. The discovery of thousands of genes, not present in the previous reference genomes for blueberry and cranberry, will serve as the basis of future research and as potential targets for future breeding efforts. The pangenome, as a multiple-sequence alignment, as well as individual annotated genomes, are publicly available for analysis on the Genome Database for Vaccinium - a curated and integrated web-based relational database. Lastly, the core-gene predictions from the pangenomes will serve useful to develop a community genotyping platform to guide future molecular breeding efforts across the family.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
