<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>genomics</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>genomics | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="An encyclopedia of enhancer-gene regulatory interactions in the human genome
Authors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/genomics/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="genomics" />
<meta property="og:description" content="An encyclopedia of enhancer-gene regulatory interactions in the human genome
Authors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/genomics/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="genomics"/>
<meta name="twitter:description" content="An encyclopedia of enhancer-gene regulatory interactions in the human genome
Authors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "genomics",
      "item": "https://trxiv.yorks0n.com/posts/genomics/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "genomics",
  "name": "genomics",
  "description": "An encyclopedia of enhancer-gene regulatory interactions in the human genome\nAuthors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K.",
  "keywords": [
    
  ],
  "articleBody": " An encyclopedia of enhancer-gene regulatory interactions in the human genome\nAuthors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K.; Perez, E. M.; Nguyen, T. C.; Kelley, D. R.; Finucane, H. K.; Moore, J. E.; Weng, Z.; Kellis, M.; Bassik, M. C.; Price, A. L.; Beer, M. A.; Guigo, R.; Stamatoyannopoulos, J. A.; Aiden, E. L.; Greenleaf, W. J.; Leslie, C. S.; Steinmetz, L. M.; Kundaje, A.; Engreitz, J. M.\nScore: 114.3, Published: 2023-11-13 DOI: 10.1101/2023.11.09.563812\nIdentifying transcriptional enhancers and their target genes is essential for understanding gene regulation and the impact of human genetic variation on disease1-6. Here we create and evaluate a resource of \u003e13 million enhancer-gene regulatory interactions across 352 cell types and tissues, by integrating predictive models, measurements of chromatin state and 3D contacts, and large-scale genetic perturbations generated by the ENCODE Consortium7. We first create a systematic benchmarking pipeline to compare predictive models, assembling a dataset of 10,411 element-gene pairs measured in CRISPR perturbation experiments, \u003e30,000 fine-mapped eQTLs, and 569 fine-mapped GWAS variants linked to a likely causal gene. Using this framework, we develop a new predictive model, ENCODE-rE2G, that achieves state-of-the-art performance across multiple prediction tasks, demonstrating a strategy involving iterative perturbations and supervised machine learning to build increasingly accurate predictive models of enhancer regulation. Using the ENCODE-rE2G model, we build an encyclopedia of enhancer-gene regulatory interactions in the human genome, which reveals global properties of enhancer networks, identifies differences in the functions of genes that have more or less complex regulatory landscapes, and improves analyses to link noncoding variants to target genes and cell types for common, complex diseases. By interpreting the model, we find evidence that, beyond enhancer activity and 3D enhancer-promoter contacts, additional features guide enhancer-promoter communication including promoter class and enhancer-enhancer synergy. Altogether, these genome-wide maps of enhancer-gene regulatory interactions, benchmarking software, predictive models, and insights about enhancer function provide a valuable resource for future studies of gene regulation and human genetics.\nA single-cell atlas of transcribed cis-regulatory elements in the human genome\nAuthors: Moody, J.; Kouno, T.; Kojima, M.; Koya, I.; Leon, J.; Suzuki, A.; Hasegawa, A.; Akiyama, T.; Akiyama, N.; Amagai, M.; Chang, J.-C.; Fukushima-Nomura, A.; Handa, M.; Hino, K.; Hino, M.; Hirata, T.; Imai, Y.; Inoue, K.; Kawasaki, H.; Kimura, T.; Kinoshita, T.; Kubo, K.-i.; Kunii, Y.; Lopez-Redondo, F.; Manabe, R.-i.; Miyai, T.; Morimoto, S.; Nagaoka, A.; Nakajima, J.; Noma, S.; Okazaki, Y.; Ozaki, K.; Saeki, N.; Sakai, H.; Seyama, K.; Shibayama, Y.; Sujino, T.; Tagami, M.; Takahashi, H.; Takao, M.; Takeshita, M.; Takiuchi, T.; Terao, C.; Yip, C. W.; Yoshinaga, S.; Okano, H.; Yamamoto, K.; Kasuka\nScore: 50.1, Published: 2023-11-14 DOI: 10.1101/2023.11.13.566791\nTranscribed cis-regulatory elements (tCREs), such as promoters and enhancers, are fundamental to modulate gene expression and define cell identity. The detailed mapping of tCREs at single-cell resolution is essential for understanding the regulatory mechanisms that govern cellular functions. Prior tCRE catalogs, limited by bulk analysis, have often overlooked cellular heterogeneity. We have constructed a tCRE atlas using single-cell 5-RNA-seq, capturing over 340,000 single-cells from 23 human tissues and annotating more than 175,000 tCREs, substantially enhancing the scope and granularity of existing cis-regulatory element annotations in the human genome. This atlas unveils patterns of gene regulation, revealing connections between broadly expressed promoters and cell type-specific distal tCREs. Assessing trait heritability at single-cell resolution with a novel tCRE module-based approach, we uncovered the nuanced trait-gene regulatory relationships across a continuum of cell populations, offering insights beyond traditional gene-level and bulk-sample analyses. Our study bridges the gap between gene regulation and trait heritability, underscoring the potential of single-cell analysis to elucidate the genetic foundations of complex traits. These insights set the stage for future research to investigate the impact of genetic variations on diseases at the individual level, advancing the understanding of cellular and molecular basis of trait heritability.\nInterpreting cis-regulatory mechanisms from genomic deep neural networks using surrogate models\nAuthors: Seitz, E.; McCandlish, D. M.; Kinney, J. B.; Koo, P. K.\nScore: 24.4, Published: 2023-11-16 DOI: 10.1101/2023.11.14.567120\nDeep neural networks (DNNs) have greatly advanced the ability to predict genome function from sequence. Interpreting genomic DNNs in terms of biological mechanisms, however, remains difficult. Here we introduce SQUID, a genomic DNN interpretability framework based on surrogate modeling. SQUID approximates genomic DNNs in user-specified regions of sequence space using surrogate models, i.e., simpler models that are mechanistically interpretable. Importantly, SQUID removes the confounding effects that nonlinearities and heteroscedastic noise in functional genomics data can have on model interpretation. Benchmarking analysis on multiple genomic DNNs shows that SQUID, when compared to established interpretability methods, identifies motifs that are more consistent across genomic loci and yields improved single-nucleotide variant-effect predictions. SQUID also supports surrogate models that quantify epistatic interactions within and between cis-regulatory elements. SQUID thus advances the ability to mechanistically interpret genomic DNNs.\nAcetylation of H3K115 at the nucleosome dyad is associated with fragile nucleosomes at active regulatory sites.\nAuthors: Sengupta, D.; Friman, E. T.; Illingworth, R. S.; Kumar, Y.; Bickmore, W. A.\nScore: 19.8, Published: 2023-11-13 DOI: 10.1101/2023.11.10.566531\nAcetylation of lysine residues in the tail domain of histone H3 is well characterized. However, acetyl-lysines in the histone globular domain also hold regulatory potential because of their impact on nucleosome dynamics and stability. In this study, we report the genome-wide distribution of acetylated H3 lysine 115 (H3K115ac), a residue on the lateral surface at the nucleosome dyad, in mouse embryonic stem cells. We find that H3K115ac is associated with highly active promoters, particularly those associated with CpG islands, and with enhancers. During differentiation H3K115ac is dynamic, changing in line with gene activation and chromatin accessibility. Most strikingly, unlike other commonly studied histone acetylation marks, H3K115ac is enriched on \"fragile\" nucleosomes within the nucleosome depleted regions of active promoters, and enhancers where it coincides with transcription factor binding. Additionally, we detect H3K115ac-marked fragile nucleosomes at sites most strongly occupied by CTCF, within the CTCF footprint and oriented relative to the CTCF motif. This unusual genomic distribution suggests that H3K115ac could have a role in nucleosome destabilization and that it might be a valuable marker for identifying functionally important regulatory elements in mammalian genomes.\nBacteriocin Distribution Patterns in Enterococcus faecium and Enterococcus lactis: Bioinformatic Analysis Using a Tailored Genomics Framework\nAuthors: Tedim, A. P.; Almeida-Santos, A. C.; Lanza, V. F.; Novais, C.; Coque, T. M.; Freitas, A. R.; Peixe, L.\nScore: 17.2, Published: 2023-11-13 DOI: 10.1101/2023.11.13.566347\nMultidrug-resistant Enterococcus faecium represent a major concern due to their ability to thrive in diverse environments and cause life-threatening infections. While antimicrobial resistance and virulence mechanisms have been extensively studied, the contribution of bacteriocins to E. faeciums adaptability remains poorly explored. E. faecium, within the Bacillota phylum, is a prominent bacteriocin producer. Here, we developed a tailored database of 76 Bacillota bacteriocins (217 sequences, including 40 novel bacteriocins) and applied it to uncover bacteriocin distribution patterns in 997 quality-filtered E. faecium and Enterococcus lactis (former E. faecium clade B) genomes. Curated using computational pipelines and literature mining, our database demonstrates superior precision versus leading public tools in identifying diverse bacteriocins. Distinct bacteriocin profiles emerged between E. faecium and E. lactis, highlighting species-specific adaptations. E. faecium strains from hospitalized patients were significantly enriched for bacteriocins as entA, bac43, bacAS5 and bacAS11. These bacteriocins strongly associated with antibiotic resistance, particularly vancomycin and ampicillin, and Inc18 rep2_pRE25-derivative plasmids, classically associated with vancomycin resistance transposons. Our integrated genomic and epidemiological analysis elucidates meaningful connections between bacteriocin determinants, antimicrobial resistance, mobile genetic elements, and ecological origins in E. faecium. This work significantly expands the knowledge on the understudied bacteriocin diversity in opportunistic enterococci, revealing their contribution to environmental adaptation. Further characterization of strain-level bacteriocin landscapes could inform strategies to combat high-risk clones. Overall, these insights provide a framework for unravelling bacteriocins therapeutic and biotechnological potential.\nRibosome Decision Graphs for the Representation of Eukaryotic RNA Translation Complexity\nAuthors: Tierney, J. A. T.; Swirski, M.; Tjeldnes, H.; Mudge, J. M.; Kufel, J.; Whiffin, N.; Valen, E.; Baranov, P.\nScore: 16.1, Published: 2023-11-12 DOI: 10.1101/2023.11.10.566564\nThe application of ribosome profiling has revealed an unexpected abundance of translation in addition to that responsible for the synthesis of previously annotated protein-coding regions. Multiple short sequences have been found to be translated within single RNA molecules, both within annotated protein-coding and non-coding regions. The biological significance of this translation is a matter of intensive investigation. However, current schematic or annotation-based representations of mRNA translation generally do not account for the apparent multitude of translated regions within the same molecules. They also do not take into account the stochasticity of the process that allows alternative translations of the same RNA molecules by different ribosomes. There is a need for formal representations of mRNA complexity that would enable the analysis of quantitative information on translation and more accurate models for predicting the phenotypic effects of genetic variants affecting translation. To address this, we developed a conceptually novel abstraction that we term Ribosome Decision Graphs (RDGs). RDGs represent translation as multiple ribosome paths through untranslated and translated mRNA segments. We termed the later translons. Non-deterministic events, such as initiation, re-initiation, selenocysteine insertion or ribosomal frameshifting are then represented as branching points. This representation allows for an adequate representation of eukaryotic translation complexity and focuses on locations critical for translation regulation. We show how RDGs can be used for depicting translated regions, analysis of genetic variation and quantitative genome-wide data on translation for characterisation of regulatory modulators of translation.\nTesting Times: Challenges in Disentangling Admixture Histories in Recent and Complex Demographies\nAuthors: Williams, M. P.; Flegontov, P.; Maier, R.; Huber, C. D.\nScore: 15.8, Published: 2023-11-15 DOI: 10.1101/2023.11.13.566841\nPaleogenomics has expanded our knowledge of human evolutionary history. Since the 2020s, the study of ancient DNA has increased its focus on reconstructing the recent past. However, the accuracy of paleogenomic methods in answering questions of historical and archaeological importance amidst the increased demographic complexity and decreased genetic differentiation within the historical period remains an open question. We used two simulation approaches to evaluate the limitations and behavior of commonly used methods, qpAdm and the f3-statistic, on admixture inference. The first is based on branch-length data simulated from four simple demographic models of varying complexities and configurations. The second, an analysis of Eurasian history composed of 59 populations using whole-genome data modified with ancient DNA conditions such as SNP ascertainment, data missingness, and pseudo-haploidization. We show that under conditions resembling historical populations, qpAdm can identify a small candidate set of true sources and populations closely related to them. However, in typical ancient DNA conditions, qpAdm is unable to further distinguish between them, limiting its utility for resolving fine-scaled hypotheses. Notably, we find that complex gene-flow histories generally lead to improvements in the performance of qpAdm and observe no bias in the estimation of admixture weights. We offer a heuristic for admixture inference that incorporates admixture weight estimate and P-values of qpAdm models, and f3-statistics to enhance the power to distinguish between multiple plausible candidates. Finally, we highlight the future potential of qpAdm through whole-genome branch-length f2-statistics, demonstrating the improved demographic inference that could be achieved with advancements in f-statistic estimations.\nImproving Hi-C contact matrices using genome graphs\nAuthors: Shen, Y.; Yu, L.; Qiu, Y.; Zhang, T.; Kingsford, C.\nScore: 13.8, Published: 2023-11-12 DOI: 10.1101/2023.11.08.566275\nThree-dimensional chromosome structure plays an important role in fundamental genomic functions. Hi-C, a high-throughput, sequencing-based technique, has drastically expanded our comprehension of 3D chromosome structures. The first step of Hi-C analysis pipeline involves mapping sequencing reads from Hi-C to linear reference genomes. However, the linear reference genome does not incorporate genetic variation information, which can lead to incorrect read alignments, especially when analyzing samples with substantial genomic differences from the reference such as cancer samples. Using genome graphs as the reference facilitates more accurate mapping of reads, however, new algorithms are required for inferring linear genomes from Hi-C reads mapped on genome graphs and constructing corresponding Hi-C contact matrices, which is a prerequisite for the subsequent steps of the Hi-C analysis such as identifying topologically associated domains and calling chromatin loops. We introduce the problem of genome sequence inference from Hi-C data mediated by genome graphs. We formalize this problem, show the hardness of solving this problem, and introduce a novel heuristic algorithm specifically tailored to this problem. We provide a theoretical analysis to evaluate the efficacy of our algorithm. Finally, our empirical experiments indicate that the linear genomes inferred from our method lead to the creation of improved Hi-C contact matrices. These enhanced matrices show a reduction in erroneous patterns caused by structural variations and are more effective in accurately capturing the structures of topologically associated domains.\nWhat does the fox (genome) say? A chromosome-level assembly for the gray fox (Urocyon cinereoargenteus)\nAuthors: Armstrong, E. E.; Bissell, K. L.; Heikkinen, M. A.; Jessup, A.; Junaid, M. O.; Lee, D. H.; Lieb, E. C.; Liem, J. T.; Martin, E. M.; Moreno, M.; Otgonbayar, K.; Romans, B. W.; Royar, K.; Adler, M. B.; Needle, D. B.; Harkess, A.; Kelley, J. L.; Mooney, J. A.; Mychajliw, A. M.\nScore: 12.5, Published: 2023-11-12 DOI: 10.1101/2023.11.08.566296\nReference genomes have been created from multiple lineages within the Canidae family, however, despite its phylogenetic relevance as a basal genus within the clade, there is currently no reference genome for the gray fox (Urocyon cinereoargenteus). Here, we present a chromosome-level assembly for the gray fox (U. cinereoargenteus), which represents the most contiguous, non-domestic canid reference genome available to date, with 90% of the genome contained in just 34 scaffolds and a contig N50 and scaffold N50 of 59.4 and 72.9 Megabases (Mb), respectively. Repeat analyses identified an increased number of simple repeats relative to other canids. The phylogenetic position of Urocyon allows us to verify the functional loss of PRDM9 in the basal canid lineage, confirming that pseudogenization occurred at least 10 million years ago. Based on mitochondrial DNA, our Vermont sample clusters with other gray fox samples from the northeastern US and contains slightly lower levels of heterozygosity than gray foxes on the west coast of California. This new assembly lays the groundwork for future studies to describe past and present population dynamics, including the delineation of evolutionarily significant units of management relevance.\nModelling the demographic history of human North African genomes points to a recent soft split divergence between populations\nAuthors: Serradell, J. M.; Lorenzo-Salazar, J.; Flores, C.; Lao, O.; Comas, D.\nScore: 24.2, Published: 2023-11-09 DOI: 10.1101/2023.11.07.565966\nBackgroundNorth African human populations present a complex demographic scenario. The presence of an autochthonous genetic component and population substructure, plus extensive gene flow from the Middle East, Europe, and sub-Saharan Africa, have shaped the genetic composition of its people through time. ResultsWe conducted a comprehensive analysis of 364 genomes to construct detailed demographic models for the North African region, encompassing its two primary ethnic groups, the Arab and Amazigh populations. This was achieved through the utilization of the Approximate Bayesian Computation with Deep Learning (ABC-DL) framework and a novel algorithm called Genetic Programming for Population Genetics (GP4PG). This innovative approach enabled us to effectively model intricate demographic scenarios, utilizing a subset of 16 whole-genomes at \u003e30X coverage. The demographic model suggested by GP4PG exhibited a closer alignment with the observed data compared to the ABC-DL model. Both methods point to a back-to-Africa origin of North African individuals and a close relationship of North African with Eurasian populations. Results support different origins for Amazigh and Arab populations, with Amazigh populations originating back in Epipaleolithic times, as early as 22.3 Kya. GP4PG model supports Arabization as the main source of Middle Eastern ancestry in North Africa. The GP4PG model better explaining the observed data includes population substructure in surrounding populations (sub-Saharan Africa and Middle East) with continuous gene flow after the split between populations (migration decay). In contrast to what we observed in the ABC-DL, the best GP4PG model does not require pulses of admixture from surrounding populations into North Africa pointing to soft splits as drivers of divergence in North Africa. ConclusionsWe have built a demographic model on North Africa that points to a back-to-Africa expansion and a differential origin between Arab and Amazigh populations, emphasizing the complex demographic history at a population level.\n",
  "wordCount" : "2729",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/genomics/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      genomics
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.563812">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.563812" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.563812">
        <p class="paperTitle">An encyclopedia of enhancer-gene regulatory interactions in the human genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.563812" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.563812" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gschwind, A. R.; Mualim, K. S.; Karbalayghareh, A.; Sheth, M. U.; Dey, K. K.; Jagoda, E.; Nurtdinov, R. N.; Xi, W.; Tan, A. S.; Jones, H.; Ma, X. R.; Yao, D.; Nasser, J.; Avsec, Z.; James, B. T.; Shamim, M. S.; Durand, N. C.; Rao, S. S. P.; Mahajan, R.; Doughty, B. R.; Andreeva, K.; Ulirsch, J. C.; Fan, K.; Perez, E. M.; Nguyen, T. C.; Kelley, D. R.; Finucane, H. K.; Moore, J. E.; Weng, Z.; Kellis, M.; Bassik, M. C.; Price, A. L.; Beer, M. A.; Guigo, R.; Stamatoyannopoulos, J. A.; Aiden, E. L.; Greenleaf, W. J.; Leslie, C. S.; Steinmetz, L. M.; Kundaje, A.; Engreitz, J. M.</p>
        <p class="info">Score: 114.3, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.563812' target='https://doi.org/10.1101/2023.11.09.563812'> 10.1101/2023.11.09.563812</a></p>
        <p class="abstract">Identifying transcriptional enhancers and their target genes is essential for understanding gene regulation and the impact of human genetic variation on disease1-6. Here we create and evaluate a resource of &gt;13 million enhancer-gene regulatory interactions across 352 cell types and tissues, by integrating predictive models, measurements of chromatin state and 3D contacts, and large-scale genetic perturbations generated by the ENCODE Consortium7. We first create a systematic benchmarking pipeline to compare predictive models, assembling a dataset of 10,411 element-gene pairs measured in CRISPR perturbation experiments, &gt;30,000 fine-mapped eQTLs, and 569 fine-mapped GWAS variants linked to a likely causal gene. Using this framework, we develop a new predictive model, ENCODE-rE2G, that achieves state-of-the-art performance across multiple prediction tasks, demonstrating a strategy involving iterative perturbations and supervised machine learning to build increasingly accurate predictive models of enhancer regulation. Using the ENCODE-rE2G model, we build an encyclopedia of enhancer-gene regulatory interactions in the human genome, which reveals global properties of enhancer networks, identifies differences in the functions of genes that have more or less complex regulatory landscapes, and improves analyses to link noncoding variants to target genes and cell types for common, complex diseases. By interpreting the model, we find evidence that, beyond enhancer activity and 3D enhancer-promoter contacts, additional features guide enhancer-promoter communication including promoter class and enhancer-enhancer synergy. Altogether, these genome-wide maps of enhancer-gene regulatory interactions, benchmarking software, predictive models, and insights about enhancer function provide a valuable resource for future studies of gene regulation and human genetics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566791" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566791">
        <p class="paperTitle">A single-cell atlas of transcribed cis-regulatory elements in the human genome</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moody, J.; Kouno, T.; Kojima, M.; Koya, I.; Leon, J.; Suzuki, A.; Hasegawa, A.; Akiyama, T.; Akiyama, N.; Amagai, M.; Chang, J.-C.; Fukushima-Nomura, A.; Handa, M.; Hino, K.; Hino, M.; Hirata, T.; Imai, Y.; Inoue, K.; Kawasaki, H.; Kimura, T.; Kinoshita, T.; Kubo, K.-i.; Kunii, Y.; Lopez-Redondo, F.; Manabe, R.-i.; Miyai, T.; Morimoto, S.; Nagaoka, A.; Nakajima, J.; Noma, S.; Okazaki, Y.; Ozaki, K.; Saeki, N.; Sakai, H.; Seyama, K.; Shibayama, Y.; Sujino, T.; Tagami, M.; Takahashi, H.; Takao, M.; Takeshita, M.; Takiuchi, T.; Terao, C.; Yip, C. W.; Yoshinaga, S.; Okano, H.; Yamamoto, K.; Kasuka</p>
        <p class="info">Score: 50.1, Published: 2023-11-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566791' target='https://doi.org/10.1101/2023.11.13.566791'> 10.1101/2023.11.13.566791</a></p>
        <p class="abstract">Transcribed cis-regulatory elements (tCREs), such as promoters and enhancers, are fundamental to modulate gene expression and define cell identity. The detailed mapping of tCREs at single-cell resolution is essential for understanding the regulatory mechanisms that govern cellular functions. Prior tCRE catalogs, limited by bulk analysis, have often overlooked cellular heterogeneity. We have constructed a tCRE atlas using single-cell 5-RNA-seq, capturing over 340,000 single-cells from 23 human tissues and annotating more than 175,000 tCREs, substantially enhancing the scope and granularity of existing cis-regulatory element annotations in the human genome. This atlas unveils patterns of gene regulation, revealing connections between broadly expressed promoters and cell type-specific distal tCREs. Assessing trait heritability at single-cell resolution with a novel tCRE module-based approach, we uncovered the nuanced trait-gene regulatory relationships across a continuum of cell populations, offering insights beyond traditional gene-level and bulk-sample analyses. Our study bridges the gap between gene regulation and trait heritability, underscoring the potential of single-cell analysis to elucidate the genetic foundations of complex traits. These insights set the stage for future research to investigate the impact of genetic variations on diseases at the individual level, advancing the understanding of cellular and molecular basis of trait heritability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567120" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567120">
        <p class="paperTitle">Interpreting cis-regulatory mechanisms from genomic deep neural networks using surrogate models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Seitz, E.; McCandlish, D. M.; Kinney, J. B.; Koo, P. K.</p>
        <p class="info">Score: 24.4, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567120' target='https://doi.org/10.1101/2023.11.14.567120'> 10.1101/2023.11.14.567120</a></p>
        <p class="abstract">Deep neural networks (DNNs) have greatly advanced the ability to predict genome function from sequence. Interpreting genomic DNNs in terms of biological mechanisms, however, remains difficult. Here we introduce SQUID, a genomic DNN interpretability framework based on surrogate modeling. SQUID approximates genomic DNNs in user-specified regions of sequence space using surrogate models, i.e., simpler models that are mechanistically interpretable. Importantly, SQUID removes the confounding effects that nonlinearities and heteroscedastic noise in functional genomics data can have on model interpretation. Benchmarking analysis on multiple genomic DNNs shows that SQUID, when compared to established interpretability methods, identifies motifs that are more consistent across genomic loci and yields improved single-nucleotide variant-effect predictions. SQUID also supports surrogate models that quantify epistatic interactions within and between cis-regulatory elements. SQUID thus advances the ability to mechanistically interpret genomic DNNs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566531">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566531" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566531">
        <p class="paperTitle">Acetylation of H3K115 at the nucleosome dyad is associated with fragile nucleosomes at active regulatory sites.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566531" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566531" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sengupta, D.; Friman, E. T.; Illingworth, R. S.; Kumar, Y.; Bickmore, W. A.</p>
        <p class="info">Score: 19.8, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566531' target='https://doi.org/10.1101/2023.11.10.566531'> 10.1101/2023.11.10.566531</a></p>
        <p class="abstract">Acetylation of lysine residues in the tail domain of histone H3 is well characterized. However, acetyl-lysines in the histone globular domain also hold regulatory potential because of their impact on nucleosome dynamics and stability. In this study, we report the genome-wide distribution of acetylated H3 lysine 115 (H3K115ac), a residue on the lateral surface at the nucleosome dyad, in mouse embryonic stem cells. We find that H3K115ac is associated with highly active promoters, particularly those associated with CpG islands, and with enhancers. During differentiation H3K115ac is dynamic, changing in line with gene activation and chromatin accessibility. Most strikingly, unlike other commonly studied histone acetylation marks, H3K115ac is enriched on &#34;fragile&#34; nucleosomes within the nucleosome depleted regions of active promoters, and enhancers where it coincides with transcription factor binding. Additionally, we detect H3K115ac-marked fragile nucleosomes at sites most strongly occupied by CTCF, within the CTCF footprint and oriented relative to the CTCF motif. This unusual genomic distribution suggests that H3K115ac could have a role in nucleosome destabilization and that it might be a valuable marker for identifying functionally important regulatory elements in mammalian genomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566347" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566347">
        <p class="paperTitle">Bacteriocin Distribution Patterns in Enterococcus faecium and Enterococcus lactis: Bioinformatic Analysis Using a Tailored Genomics Framework</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tedim, A. P.; Almeida-Santos, A. C.; Lanza, V. F.; Novais, C.; Coque, T. M.; Freitas, A. R.; Peixe, L.</p>
        <p class="info">Score: 17.2, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566347' target='https://doi.org/10.1101/2023.11.13.566347'> 10.1101/2023.11.13.566347</a></p>
        <p class="abstract">Multidrug-resistant Enterococcus faecium represent a major concern due to their ability to thrive in diverse environments and cause life-threatening infections. While antimicrobial resistance and virulence mechanisms have been extensively studied, the contribution of bacteriocins to E. faeciums adaptability remains poorly explored. E. faecium, within the Bacillota phylum, is a prominent bacteriocin producer. Here, we developed a tailored database of 76 Bacillota bacteriocins (217 sequences, including 40 novel bacteriocins) and applied it to uncover bacteriocin distribution patterns in 997 quality-filtered E. faecium and Enterococcus lactis (former E. faecium clade B) genomes. Curated using computational pipelines and literature mining, our database demonstrates superior precision versus leading public tools in identifying diverse bacteriocins. Distinct bacteriocin profiles emerged between E. faecium and E. lactis, highlighting species-specific adaptations. E. faecium strains from hospitalized patients were significantly enriched for bacteriocins as entA, bac43, bacAS5 and bacAS11. These bacteriocins strongly associated with antibiotic resistance, particularly vancomycin and ampicillin, and Inc18 rep2_pRE25-derivative plasmids, classically associated with vancomycin resistance transposons. Our integrated genomic and epidemiological analysis elucidates meaningful connections between bacteriocin determinants, antimicrobial resistance, mobile genetic elements, and ecological origins in E. faecium. This work significantly expands the knowledge on the understudied bacteriocin diversity in opportunistic enterococci, revealing their contribution to environmental adaptation. Further characterization of strain-level bacteriocin landscapes could inform strategies to combat high-risk clones. Overall, these insights provide a framework for unravelling bacteriocins therapeutic and biotechnological potential.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566564">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566564" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566564">
        <p class="paperTitle">Ribosome Decision Graphs for the Representation of Eukaryotic RNA Translation Complexity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566564" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566564" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tierney, J. A. T.; Swirski, M.; Tjeldnes, H.; Mudge, J. M.; Kufel, J.; Whiffin, N.; Valen, E.; Baranov, P.</p>
        <p class="info">Score: 16.1, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566564' target='https://doi.org/10.1101/2023.11.10.566564'> 10.1101/2023.11.10.566564</a></p>
        <p class="abstract">The application of ribosome profiling has revealed an unexpected abundance of translation in addition to that responsible for the synthesis of previously annotated protein-coding regions. Multiple short sequences have been found to be translated within single RNA molecules, both within annotated protein-coding and non-coding regions. The biological significance of this translation is a matter of intensive investigation. However, current schematic or annotation-based representations of mRNA translation generally do not account for the apparent multitude of translated regions within the same molecules. They also do not take into account the stochasticity of the process that allows alternative translations of the same RNA molecules by different ribosomes. There is a need for formal representations of mRNA complexity that would enable the analysis of quantitative information on translation and more accurate models for predicting the phenotypic effects of genetic variants affecting translation. To address this, we developed a conceptually novel abstraction that we term Ribosome Decision Graphs (RDGs). RDGs represent translation as multiple ribosome paths through untranslated and translated mRNA segments. We termed the later  translons. Non-deterministic events, such as initiation, re-initiation, selenocysteine insertion or ribosomal frameshifting are then represented as branching points. This representation allows for an adequate representation of eukaryotic translation complexity and focuses on locations critical for translation regulation. We show how RDGs can be used for depicting translated regions, analysis of genetic variation and quantitative genome-wide data on translation for characterisation of regulatory modulators of translation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566841">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566841" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566841">
        <p class="paperTitle">Testing Times: Challenges in Disentangling Admixture Histories in Recent and Complex Demographies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566841" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566841" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Williams, M. P.; Flegontov, P.; Maier, R.; Huber, C. D.</p>
        <p class="info">Score: 15.8, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566841' target='https://doi.org/10.1101/2023.11.13.566841'> 10.1101/2023.11.13.566841</a></p>
        <p class="abstract">Paleogenomics has expanded our knowledge of human evolutionary history. Since the 2020s, the study of ancient DNA has increased its focus on reconstructing the recent past. However, the accuracy of paleogenomic methods in answering questions of historical and archaeological importance amidst the increased demographic complexity and decreased genetic differentiation within the historical period remains an open question. We used two simulation approaches to evaluate the limitations and behavior of commonly used methods, qpAdm and the f3-statistic, on admixture inference. The first is based on branch-length data simulated from four simple demographic models of varying complexities and configurations. The second, an analysis of Eurasian history composed of 59 populations using whole-genome data modified with ancient DNA conditions such as SNP ascertainment, data missingness, and pseudo-haploidization. We show that under conditions resembling historical populations, qpAdm can identify a small candidate set of true sources and populations closely related to them. However, in typical ancient DNA conditions, qpAdm is unable to further distinguish between them, limiting its utility for resolving fine-scaled hypotheses. Notably, we find that complex gene-flow histories generally lead to improvements in the performance of qpAdm and observe no bias in the estimation of admixture weights. We offer a heuristic for admixture inference that incorporates admixture weight estimate and P-values of qpAdm models, and f3-statistics to enhance the power to distinguish between multiple plausible candidates. Finally, we highlight the future potential of qpAdm through whole-genome branch-length f2-statistics, demonstrating the improved demographic inference that could be achieved with advancements in f-statistic estimations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566275">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566275" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566275">
        <p class="paperTitle">Improving Hi-C contact matrices using genome graphs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566275" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566275" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shen, Y.; Yu, L.; Qiu, Y.; Zhang, T.; Kingsford, C.</p>
        <p class="info">Score: 13.8, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566275' target='https://doi.org/10.1101/2023.11.08.566275'> 10.1101/2023.11.08.566275</a></p>
        <p class="abstract">Three-dimensional chromosome structure plays an important role in fundamental genomic functions. Hi-C, a high-throughput, sequencing-based technique, has drastically expanded our comprehension of 3D chromosome structures. The first step of Hi-C analysis pipeline involves mapping sequencing reads from Hi-C to linear reference genomes. However, the linear reference genome does not incorporate genetic variation information, which can lead to incorrect read alignments, especially when analyzing samples with substantial genomic differences from the reference such as cancer samples. Using genome graphs as the reference facilitates more accurate mapping of reads, however, new algorithms are required for inferring linear genomes from Hi-C reads mapped on genome graphs and constructing corresponding Hi-C contact matrices, which is a prerequisite for the subsequent steps of the Hi-C analysis such as identifying topologically associated domains and calling chromatin loops. We introduce the problem of genome sequence inference from Hi-C data mediated by genome graphs. We formalize this problem, show the hardness of solving this problem, and introduce a novel heuristic algorithm specifically tailored to this problem. We provide a theoretical analysis to evaluate the efficacy of our algorithm. Finally, our empirical experiments indicate that the linear genomes inferred from our method lead to the creation of improved Hi-C contact matrices. These enhanced matrices show a reduction in erroneous patterns caused by structural variations and are more effective in accurately capturing the structures of topologically associated domains.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566296">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566296" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566296">
        <p class="paperTitle">What does the fox (genome) say? A chromosome-level assembly for the gray fox (Urocyon cinereoargenteus)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566296" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566296" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Armstrong, E. E.; Bissell, K. L.; Heikkinen, M. A.; Jessup, A.; Junaid, M. O.; Lee, D. H.; Lieb, E. C.; Liem, J. T.; Martin, E. M.; Moreno, M.; Otgonbayar, K.; Romans, B. W.; Royar, K.; Adler, M. B.; Needle, D. B.; Harkess, A.; Kelley, J. L.; Mooney, J. A.; Mychajliw, A. M.</p>
        <p class="info">Score: 12.5, Published: 2023-11-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566296' target='https://doi.org/10.1101/2023.11.08.566296'> 10.1101/2023.11.08.566296</a></p>
        <p class="abstract">Reference genomes have been created from multiple lineages within the Canidae family, however, despite its phylogenetic relevance as a basal genus within the clade, there is currently no reference genome for the gray fox (Urocyon cinereoargenteus). Here, we present a chromosome-level assembly for the gray fox (U. cinereoargenteus), which represents the most contiguous, non-domestic canid reference genome available to date, with 90% of the genome contained in just 34 scaffolds and a contig N50 and scaffold N50 of 59.4 and 72.9 Megabases (Mb), respectively. Repeat analyses identified an increased number of simple repeats relative to other canids. The phylogenetic position of Urocyon allows us to verify the functional loss of PRDM9 in the basal canid lineage, confirming that pseudogenization occurred at least 10 million years ago. Based on mitochondrial DNA, our Vermont sample clusters with other gray fox samples from the northeastern US and contains slightly lower levels of heterozygosity than gray foxes on the west coast of California. This new assembly lays the groundwork for future studies to describe past and present population dynamics, including the delineation of evolutionarily significant units of management relevance.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.565966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.565966" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.565966">
        <p class="paperTitle">Modelling the demographic history of human North African genomes points to a recent soft split divergence between populations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.565966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.565966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Serradell, J. M.; Lorenzo-Salazar, J.; Flores, C.; Lao, O.; Comas, D.</p>
        <p class="info">Score: 24.2, Published: 2023-11-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.565966' target='https://doi.org/10.1101/2023.11.07.565966'> 10.1101/2023.11.07.565966</a></p>
        <p class="abstract">BackgroundNorth African human populations present a complex demographic scenario. The presence of an autochthonous genetic component and population substructure, plus extensive gene flow from the Middle East, Europe, and sub-Saharan Africa, have shaped the genetic composition of its people through time.

ResultsWe conducted a comprehensive analysis of 364 genomes to construct detailed demographic models for the North African region, encompassing its two primary ethnic groups, the Arab and Amazigh populations. This was achieved through the utilization of the Approximate Bayesian Computation with Deep Learning (ABC-DL) framework and a novel algorithm called Genetic Programming for Population Genetics (GP4PG). This innovative approach enabled us to effectively model intricate demographic scenarios, utilizing a subset of 16 whole-genomes at &gt;30X coverage. The demographic model suggested by GP4PG exhibited a closer alignment with the observed data compared to the ABC-DL model. Both methods point to a back-to-Africa origin of North African individuals and a close relationship of North African with Eurasian populations. Results support different origins for Amazigh and Arab populations, with Amazigh populations originating back in Epipaleolithic times, as early as 22.3 Kya. GP4PG model supports Arabization as the main source of Middle Eastern ancestry in North Africa. The GP4PG model better explaining the observed data includes population substructure in surrounding populations (sub-Saharan Africa and Middle East) with continuous gene flow after the split between populations (migration decay). In contrast to what we observed in the ABC-DL, the best GP4PG model does not require pulses of admixture from surrounding populations into North Africa pointing to soft splits as drivers of divergence in North Africa.

ConclusionsWe have built a demographic model on North Africa that points to a back-to-Africa expansion and a differential origin between Arab and Amazigh populations, emphasizing the complex demographic history at a population level.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
