<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Three-dimensional assessments are necessary to determine the true spatial tissue composition of diseased tissues
Authors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.
Score: 32.3, Published: 2023-12-06 DOI: 10.1101/2023.12.04.569986
Methods for partially resolved cellular profiling has enabled in-depth quantitative tissue mapping via thinly cut sections to study inter-patient and intra-patient differences in normal human anatomy and disease onset and progression.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Three-dimensional assessments are necessary to determine the true spatial tissue composition of diseased tissues
Authors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.
Score: 32.3, Published: 2023-12-06 DOI: 10.1101/2023.12.04.569986
Methods for partially resolved cellular profiling has enabled in-depth quantitative tissue mapping via thinly cut sections to study inter-patient and intra-patient differences in normal human anatomy and disease onset and progression." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-13T10:39:57+00:00" />
<meta property="article:modified_time" content="2023-12-13T10:39:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Three-dimensional assessments are necessary to determine the true spatial tissue composition of diseased tissues
Authors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.
Score: 32.3, Published: 2023-12-06 DOI: 10.1101/2023.12.04.569986
Methods for partially resolved cellular profiling has enabled in-depth quantitative tissue mapping via thinly cut sections to study inter-patient and intra-patient differences in normal human anatomy and disease onset and progression."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Three-dimensional assessments are necessary to determine the true spatial tissue composition of diseased tissues\nAuthors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.\nScore: 32.3, Published: 2023-12-06 DOI: 10.1101/2023.12.04.569986\nMethods for partially resolved cellular profiling has enabled in-depth quantitative tissue mapping via thinly cut sections to study inter-patient and intra-patient differences in normal human anatomy and disease onset and progression.",
  "keywords": [
    
  ],
  "articleBody": " Three-dimensional assessments are necessary to determine the true spatial tissue composition of diseased tissues\nAuthors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.\nScore: 32.3, Published: 2023-12-06 DOI: 10.1101/2023.12.04.569986\nMethods for partially resolved cellular profiling has enabled in-depth quantitative tissue mapping via thinly cut sections to study inter-patient and intra-patient differences in normal human anatomy and disease onset and progression. These methods often profile extremely limited spatial regions, which may impact the evaluation of heterogeneity due to tissue sub-sampling. Here, we applied CODA, a deep learning-based tissue mapping platform, to reconstruct the 3D microanatomy of surgically resected human pancreas biospecimens obtained from patients diagnosed with pancreatic cancer. To compare differences in the inter- and intra-tumoral heterogeneity, we assessed the bulk and spatially resolved tissue composition of a cohort of two-dimensional (2D) whole slide images (WSIs), and a cohort of 3D serially sectioned and reconstructed tissues of pancreata. Here, we show the strength of using 3D as the gold standard, by measuring the information loss and sampling problems when using WSIs and TMAs. We demonstrate that spatial correlation in microanatomical tissue content decays significantly within a span of just a few microns within tumors. As a corollary, hundreds of TMAs and tens of WSIs are required to estimate spatial bulk tumor composition with \u003c10% error in any given pancreatic tumor. In sum, we demonstrate that 3D assessments are necessary to accurately assess tumor burden and tissue composition. These preliminary results show the importance of rate of sampling necessary to more reliably assess spatially resolved tissue composition.\nSystematic benchmarking of imaging spatial transcriptomics platforms in FFPE tissues\nAuthors: Wang, H.; Huang, R.; Nelson, J.; Gao, C.; Trans, M.; Yeaton, A.; Felt, K.; Pfaff, K. L.; Bowman, T.; Rodig, S. J.; Wei, K.; Goods, B. A.; Farhi, S. L.\nScore: 32.0, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570603\nEmerging imaging spatial transcriptomics (iST) platforms and coupled analytical methods can recover cell-to-cell interactions, groups of spatially covarying genes, and gene signatures associated with pathological features, and are thus particularly well-suited for applications in formalin fixed paraffin embedded (FFPE) tissues. Here, we benchmarked the performance of three commercial iST platforms on serial sections from tissue microarrays (TMAs) containing 23 tumor and normal tissue types for both relative technical and biological performance. On matched genes, we found that 10X Xenium shows higher transcript counts per gene without sacrificing specificity, but that all three platforms concord to orthogonal RNA-seq datasets and can perform spatially resolved cell typing, albeit with different false discovery rates, cell segmentation error frequencies, and with varying degrees of sub-clustering for downstream biological analyses. Taken together, our analyses provide a comprehensive benchmark to guide the choice of iST method as researchers design studies with precious samples in this rapidly evolving field.\nMembrane to cortex attachment determines different mechanical phenotypes in LGR5+ and LGR5- colorectal cancer cells\nAuthors: Conti, S.; Venturini, V.; Canellas-Socias, A.; Cortina, C.; Abenza, J. F.; Attolini, C. S.-O.; Guerra, E. M.; Xu, C. K.; Li, J. H.; Rossetti, L.; Stassi, G.; Roca-Cusachs, P.; Diz-Munoz, A.; Ruprecht, V.; Guck, J.; Batlle, E.; Labernadie, A.; Trepat, X.\nScore: 30.2, Published: 2023-12-05 DOI: 10.1101/2023.12.04.569244\nColorectal cancer tumors are composed of heterogeneous and plastic cell populations, including a pool of cancer stem cells that express LGR5. Whether these distinct cell populations display different mechanical properties, and how these properties might contribute to metastasis is unknown. Using CRC patient derived organoids (PDOs), we found that compared to LGR5- cells, LGR5+ cancer stem cells are stiffer, adhere better to the extracellular matrix (ECM), move slower both as single cells and clusters, display higher nuclear YAP, show a higher survival rate in response to mechanical confinement, and form larger transendothelial gaps. These differences are largely explained by the downregulation of the membrane to cortex attachment proteins Ezrin/Radixin/Moesin (ERMs) in the LGR5+ cells. By analyzing scRNA-seq expression patterns from a patient cohort, we show that this downregulation is a robust signature of colorectal tumors. Our results show that LGR5- cells display a mechanically dynamic phenotype suitable for dissemination from the primary tumor whereas LGR5+ cells display a mechanically stable and resilient phenotype suitable for extravasation and metastatic growth.\nTumor-selective effects of active RAS inhibition in pancreatic ductal adenocarcinoma\nAuthors: Wasko, U. N.; Jiang, J.; Garcia, A. C.; Wang, Y.; Lee, B.; Orlen, M.; Drizyte-Miller, K.; Menard, M.; Dilly, J.; Sastra, S. A.; Palermo, C. F.; Dalton, T.; Hasselluhn, M. C.; Decker-Farrell, A. R.; Chang, S.; Jiang, L.; Wei, X.; Yang, Y.-c.; Helland, C.; Courtney, H.; Gindin, Y.; Zhao, R.; Kemp, S. B.; Clendenin, C.; Sor, R.; Vostrejs, W.; Amparo, A. A.; Hibshman, P. S.; Rees, M. G.; Ronan, M. M.; Roth, J. A.; Bakir, B.; Badgley, M. A.; Chabot, J. A.; Kluger, M. D.; Manji, G. A.; Quintana, E.; Wang, Z.; Smith, J. A. M.; Holderfield, M.; Wildes, D.; Aguirre, A. J.; Der, C. J.; Vonderheide, R. H\nScore: 29.8, Published: 2023-12-04 DOI: 10.1101/2023.12.03.569791\nBroad-spectrum RAS inhibition holds the potential to benefit roughly a quarter of human cancer patients whose tumors are driven by RAS mutations. However, the impact of inhibiting RAS functions in normal tissues is not known. RMC-7977 is a highly selective inhibitor of the active (GTP-bound) forms of KRAS, HRAS, and NRAS, with affinity for both mutant and wild type (WT) variants. As \u003e90% of human pancreatic ductal adenocarcinoma (PDAC) cases are driven by activating mutations in KRAS, we assessed the therapeutic potential of RMC-7977 in a comprehensive range of PDAC models, including human and murine cell lines, human patient-derived organoids, human PDAC explants, subcutaneous and orthotopic cell-line or patient derived xenografts, syngeneic allografts, and genetically engineered mouse models. We observed broad and pronounced anti-tumor activity across these models following direct RAS inhibition at doses and concentrations that were well-tolerated in vivo. Pharmacological analyses revealed divergent responses to RMC-7977 in tumor versus normal tissues. Treated tumors exhibited waves of apoptosis along with sustained proliferative arrest whereas normal tissues underwent only transient decreases in proliferation, with no evidence of apoptosis. Together, these data establish a strong preclinical rationale for the use of broad-spectrum RAS inhibition in the setting of PDAC.\nSingle cell dissection of developmental origins and transcriptional heterogeneity in B-cell acute lymphoblastic leukemia\nAuthors: Iacobucci, I.; Zeng, A. G. X.; Gao, Q.; Garcia-Prat, L.; Baviskar, P.; Shah, S.; Murison, A. J.; Voisin, V.; Chan-Seng-Yue, M.; Cheng, C.; Qu, C.; Bailey, C.; Lear, M.; Witkowski, M.; Zhou, X.; Zaldivar Peraza, A.; Gangwani, K.; Advani, A.; Luger, S. M.; Litzow, M. R.; Rowe, J. M.; Paietta, E. M.; Stock, W.; Dick, J. E.; Mullighan, C. G.\nScore: 13.6, Published: 2023-12-09 DOI: 10.1101/2023.12.04.569954\nSequencing of bulk tumor populations has improved genetic classification and risk assessment of B-ALL, but does not directly examine intratumor heterogeneity or infer leukemia cellular origins. We profiled 89 B-ALL samples by single-cell RNA-seq (scRNA-seq) and compared them to a reference map of normal human B-cell development established using both functional and molecular assays. Intra-sample heterogeneity was driven by cell cycle, metabolism, differentiation, and inflammation transcriptional programs. By inference of B lineage developmental state composition, nearly all samples possessed a high abundance of pro-B cells, with variation between samples mainly driven by sub-populations. However, ZNF384-r and DUX4-r B-ALL showed composition enrichment of hematopoietic stem cells, BCR::ABL1 and KMT2A-r ALL of Early Lymphoid progenitors, MEF2D-r and TCF3::PBX1 of Pre-B cells. Enrichment of Early Lymphoid progenitors correlated with high-risk clinical features. Understanding variation in transcriptional programs and developmental states of B-ALL by scRNA-seq refines existing clinical and genomic classifications and improves prediction of treatment outcome.\nChromothripsis orchestrates leukemic transformation in blast phase MPN through targetable amplification of DYRK1A\nAuthors: Brierley, C. K.; Yip, B. H.; Orlando, G.; Goyal, H.; Wen, S.; Wen, J.; Levine, M. F.; Jakobsdottir, M. G.; Rodriguez-Meira, A.; Adamo, A.; Bashton, M.; Hamblin, A.; Clark, S. A.; O'Sullivan, J.; Murphi, L.; Olijnik-French, A.-A.; Cotton, A.; Narina, S.; Pruett-Miller, S.; Enshaei, A.; Harrison, C.; Drummond, M.; Knapper, S.; Tefferi, A.; Antony-Debre, I.; Thongjuea, S.; Wedge, D. C.; Constantinescu, S.; Papaemmanuil, E.; Psaila, B.; Crispino, J. D.; Mead, A. J.\nScore: 12.3, Published: 2023-12-10 DOI: 10.1101/2023.12.08.570880\nChromothripsis, the process of catastrophic shattering and haphazard repair of chromosomes, is a common event in cancer. Whether chromothripsis might constitute an actionable molecular event amenable to therapeutic targeting remains an open question. We describe recurrent chromothripsis of chromosome 21 in a subset of patients in blast phase of a myeloproliferative neoplasm (BP-MPN), which alongside other structural variants leads to amplification of a region of chromosome 21 in [~]25% of patients ( chr21amp). We report that chr21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. The chr21amp event is highly clonal and present throughout the hematopoietic hierarchy. DYRK1A, a serine threonine kinase and transcription factor, is the only gene in the 2.7Mb minimally amplified region which showed both increased expression and chromatin accessibility compared to non-chr21amp BP-MPN controls. We demonstrate that DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development, including DNA repair, STAT signalling and BCL2 overexpression. DYRK1A is essential for BP-MPN cell proliferation in vitro and in vivo, and DYRK1A inhibition synergises with BCL2 targeting to induce BP-MPN cell apoptosis. Collectively, these findings define the chr21amp event as a prognostic biomarker in BP-MPN and link chromothripsis to a druggable target.\nA Mutation-driven oncofetal regression fuels phenotypic plasticity in colorectal cancer\nAuthors: Mzoughi, S.; Schwarz, M.; Wang, X.; Demircioglu, D.; Ulukaya, G.; Mohammed, K.; Di Tullio, F.; Company, C.; Dramaretska, Y.; Leushacke, M.; Giotti, B.; Lannagan, T.; Lozano-Ojalvo, D.; Hasson, D.; Tsankov, A.; Sansom, O. J.; Marine, J.-C.; Barker, N.; Gargiulo, G.; Guccione, E.\nScore: 11.9, Published: 2023-12-10 DOI: 10.1101/2023.12.10.570854\nTargeting cancer stem cells (CSCs) is crucial for effective cancer treatment1. However, the molecular mechanisms underlying resistance to LGR5+ CSCs depletion in colorectal cancer (CRC)2,3 remain largely elusive. Here, we unveil the existence of a primitive cell state dubbed the oncofetal (OnF) state, which works in tandem with the LGR5+ stem cells (SCs) to fuel tumor evolution in CRC. OnF cells emerge early during intestinal tumorigenesis and exhibit features of lineage plasticity. Normally suppressed by the Retinoid X Receptor (RXR) in mature SCs, the OnF program is triggered by genetic deletion of the gatekeeper APC. We demonstrate that diminished RXR activity unlocks an epigenetic circuity governed by the cooperative action of YAP and AP1, leading to OnF reprogramming. This high-plasticity state is inherently resistant to conventional chemotherapies and its adoption by LGR5+ CSCs enables them to enter a drug-tolerant state. Furthermore, through phenotypic tracing and ablation experiments, we uncover a functional redundancy between the OnF and stem cell (SC) states and show that targeting both cellular states is essential for sustained tumor regression in vivo. Collectively, these findings establish a mechanistic foundation for developing effective combination therapies with enduring impact on CRC treatment.\nDynamic Plasticity Systems Direct Early Adaptation to Treatment in Neuroblastoma\nAuthors: Roux, C.; Hamer, S.; Shea, A.; Chen, E.; Sadr, A. S.; English, C.; Sahoo, S.; Allo Anido, A.; Che, H.; Chesler, L.; Jolly, M. K.; Morgan, M. D.; Bruna, A.\nScore: 10.5, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570359\nIn paediatric cancers like neuroblastoma, limited genetic diversity emphasizes the role of phenotypic heterogeneity in driving malignancy. We investigated this phenomenon using experimental evolution and single-cell techniques in neuroblastoma preclinical models. Our findings reveal that neuroblastoma cells navigate multistable phenotypic landscapes, named plasticity systems. These finely regulate their topology and dynamics enabling tolerance, persistence, and regrowth in response to treatment. While preferential killing of adrenergic cells (ADRN), notably under cisplatin treatment, enriches drug-tolerant persister (DTP) populations with mesenchymal (MES) properties, we also observed phenotypic transitions contributing to DTP entry and exit. Additionally, single-cell-derived clone experiments unveiled a spectrum of heritable plasticity traits linked to functional properties, influencing DTP behaviour. Mathematical modelling supports the critical role of all cell phenotypes in evolutionary adaptation. Collectively, our study depicts plasticity systems as key early cancer drivers of adaptive evolution in neuroblastoma through regulating the multistability and dynamics of phenotypic landscapes. These insights underscore the necessity of decoding plasticity for advancing long-term therapeutic effectiveness.\nAdaptation to ARF6-depletion in KRAS-driven PDAC is abolished by targeting TLR2\nAuthors: Ghose, R.; Das, S.; Urgel-Solas, J.; Pezzano, F.; Datta, D.; Ghose, U.; Ganez-Zapater, A.; Pardo-Lorente, N.; Espinar, L.; Garcia, L.; Cannata, C.; Ruprecht, V.; Janic, A.; Sdelci, S.\nScore: 8.9, Published: 2023-12-02 DOI: 10.1101/2023.11.30.569405\nMetastasis is responsible for nearly 90% of all cancer-related deaths. Despite global efforts to prevent aggressive tumours, cancers such as pancreatic ductal adenocarcinoma (PDAC) are poorly diagnosed in the primary stage, resulting in lethal metastatic disease. RAS mutations are known to promote tumour spread, with mutant KRAS present in up to 90% of cases. Until recently, mutant KRAS remained untargeted and, despite the recent development of inhibitors, results show that tumour cells develop resistance. Another strategy for targeting mutant KRAS-dependent PDAC proliferation and metastasis may come from targeting the downstream effectors of KRAS. One such axis, which controls tumour proliferation, invasiveness and immune evasion, is represented by ARF6-ASAP1. Here we show that targeting ARF6 results in adaptive rewiring that can restore proliferation and invasion potential over time. Using time-series RNA and ATAC sequencing approaches, we identified TLR-dependent NF{kappa}B, TNF and hypoxia signalling as key drivers of adaptation in ARF6-depleted KRAS-dependent PDAC. Using in vitro and in vivo assays, we show that knocking down TLR2 with ARF6 significantly reduces proliferation, migration and invasion. Taken together, our data shed light on a novel co-targeting strategy with the therapeutic potential to counteract PDAC proliferation and metastasis. GRAPHICAL SUMMARY O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=120 SRC=\"FIGDIR/small/569405v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (37K): org.highwire.dtl.DTLVardef@192ac2org.highwire.dtl.DTLVardef@4659e5org.highwire.dtl.DTLVardef@b7e225org.highwire.dtl.DTLVardef@6019aa_HPS_FORMAT_FIGEXP M_FIG C_FIG\nHypoxia alters the effects of hypomethylating agents in acute myeloid leukaemia cells.\nAuthors: Humphries, S.; Burnard, S. M.; Keely, S.; Bond, D. R.; Lee, H. J.\nScore: 6.3, Published: 2023-12-08 DOI: 10.1101/2023.12.07.570313\nBackgroundAcute myeloid leukaemia (AML) is a deadly haematological malignancy that originates from mutated myeloid progenitor cells that lie quiescent in the hypoxic bone marrow. Elderly patients who cannot tolerate standard chemotherapies are administered low-dose hypomethylating agents (HMA) which act in a replication-dependent manner to reprogram the epigenome. Relapse is common following HMA treatment and may arise from quiescent leukaemia cells in the hypoxic bone marrow. Therefore, the effects of hypoxia on HMA efficacy may influence AML progression. ResultsAML cell lines (MOLM-13, MV-4-11, HL-60) were treated with decitabine (100nM) or azacitidine (500-2000nM) in normoxic (21% O2) and hypoxic (1% O2) conditions. Exposure to hypoxia significantly reduced AML cell growth across all cell lines, with no additional effects observed upon HMA treatment. This was associated with distinct effects on DNA methylation. The extent of hypomethylation induced by AZA treatment was reduced in hypoxia, whereas DAC-induced hypomethylation was maintained in low oxygen conditions. Transcriptional response to HMA treatment were also altered in hypoxia, with HMAs failing to up-regulate antigen presentation pathways in hypoxia. In particular, human leukocyte antigens (HLAs) such as HLA-DR were increased upon HMA treatment in normoxia, but not hypoxia. ConclusionOur results suggest that HMA-induced antigen presentation may be impaired in hypoxic tissues such as the bone marrow. This study highlights the need to consider microenvironmental factors when designing co-treatment strategies to improve HMA therapeutic efficacy.\n",
  "wordCount" : "2532",
  "inLanguage": "en",
  "datePublished": "2023-12-13T10:39:57Z",
  "dateModified": "2023-12-13T10:39:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 13, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569986">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569986" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569986">
        <p class="paperTitle">Three-dimensional assessments are necessary to determine the true spatial tissue composition of diseased tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569986" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569986" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.</p>
        <p class="info">Score: 32.3, Published: 2023-12-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569986' target='https://doi.org/10.1101/2023.12.04.569986'> 10.1101/2023.12.04.569986</a></p>
        <p class="abstract">Methods for partially resolved cellular profiling has enabled in-depth quantitative tissue mapping via thinly cut sections to study inter-patient and intra-patient differences in normal human anatomy and disease onset and progression. These methods often profile extremely limited spatial regions, which may impact the evaluation of heterogeneity due to tissue sub-sampling. Here, we applied CODA, a deep learning-based tissue mapping platform, to reconstruct the 3D microanatomy of surgically resected human pancreas biospecimens obtained from patients diagnosed with pancreatic cancer. To compare differences in the inter- and intra-tumoral heterogeneity, we assessed the bulk and spatially resolved tissue composition of a cohort of two-dimensional (2D) whole slide images (WSIs), and a cohort of 3D serially sectioned and reconstructed tissues of pancreata. Here, we show the strength of using 3D as the gold standard, by measuring the information loss and sampling problems when using WSIs and TMAs. We demonstrate that spatial correlation in microanatomical tissue content decays significantly within a span of just a few microns within tumors. As a corollary, hundreds of TMAs and tens of WSIs are required to estimate spatial bulk tumor composition with &lt;10% error in any given pancreatic tumor. In sum, we demonstrate that 3D assessments are necessary to accurately assess tumor burden and tissue composition. These preliminary results show the importance of rate of sampling necessary to more reliably assess spatially resolved tissue composition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570603">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570603" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570603">
        <p class="paperTitle">Systematic benchmarking of imaging spatial transcriptomics platforms in FFPE tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570603" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570603" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, H.; Huang, R.; Nelson, J.; Gao, C.; Trans, M.; Yeaton, A.; Felt, K.; Pfaff, K. L.; Bowman, T.; Rodig, S. J.; Wei, K.; Goods, B. A.; Farhi, S. L.</p>
        <p class="info">Score: 32.0, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570603' target='https://doi.org/10.1101/2023.12.07.570603'> 10.1101/2023.12.07.570603</a></p>
        <p class="abstract">Emerging imaging spatial transcriptomics (iST) platforms and coupled analytical methods can recover cell-to-cell interactions, groups of spatially covarying genes, and gene signatures associated with pathological features, and are thus particularly well-suited for applications in formalin fixed paraffin embedded (FFPE) tissues. Here, we benchmarked the performance of three commercial iST platforms on serial sections from tissue microarrays (TMAs) containing 23 tumor and normal tissue types for both relative technical and biological performance. On matched genes, we found that 10X Xenium shows higher transcript counts per gene without sacrificing specificity, but that all three platforms concord to orthogonal RNA-seq datasets and can perform spatially resolved cell typing, albeit with different false discovery rates, cell segmentation error frequencies, and with varying degrees of sub-clustering for downstream biological analyses. Taken together, our analyses provide a comprehensive benchmark to guide the choice of iST method as researchers design studies with precious samples in this rapidly evolving field.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569244">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569244" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569244">
        <p class="paperTitle">Membrane to cortex attachment determines different mechanical phenotypes in LGR5&#43; and LGR5- colorectal cancer cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569244" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569244" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Conti, S.; Venturini, V.; Canellas-Socias, A.; Cortina, C.; Abenza, J. F.; Attolini, C. S.-O.; Guerra, E. M.; Xu, C. K.; Li, J. H.; Rossetti, L.; Stassi, G.; Roca-Cusachs, P.; Diz-Munoz, A.; Ruprecht, V.; Guck, J.; Batlle, E.; Labernadie, A.; Trepat, X.</p>
        <p class="info">Score: 30.2, Published: 2023-12-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569244' target='https://doi.org/10.1101/2023.12.04.569244'> 10.1101/2023.12.04.569244</a></p>
        <p class="abstract">Colorectal cancer tumors are composed of heterogeneous and plastic cell populations, including a pool of cancer stem cells that express LGR5. Whether these distinct cell populations display different mechanical properties, and how these properties might contribute to metastasis is unknown. Using CRC patient derived organoids (PDOs), we found that compared to LGR5- cells, LGR5&#43; cancer stem cells are stiffer, adhere better to the extracellular matrix (ECM), move slower both as single cells and clusters, display higher nuclear YAP, show a higher survival rate in response to mechanical confinement, and form larger transendothelial gaps. These differences are largely explained by the downregulation of the membrane to cortex attachment proteins Ezrin/Radixin/Moesin (ERMs) in the LGR5&#43; cells. By analyzing scRNA-seq expression patterns from a patient cohort, we show that this downregulation is a robust signature of colorectal tumors. Our results show that LGR5- cells display a mechanically dynamic phenotype suitable for dissemination from the primary tumor whereas LGR5&#43; cells display a mechanically stable and resilient phenotype suitable for extravasation and metastatic growth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.03.569791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.03.569791" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.03.569791">
        <p class="paperTitle">Tumor-selective effects of active RAS inhibition in pancreatic ductal adenocarcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.03.569791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.03.569791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wasko, U. N.; Jiang, J.; Garcia, A. C.; Wang, Y.; Lee, B.; Orlen, M.; Drizyte-Miller, K.; Menard, M.; Dilly, J.; Sastra, S. A.; Palermo, C. F.; Dalton, T.; Hasselluhn, M. C.; Decker-Farrell, A. R.; Chang, S.; Jiang, L.; Wei, X.; Yang, Y.-c.; Helland, C.; Courtney, H.; Gindin, Y.; Zhao, R.; Kemp, S. B.; Clendenin, C.; Sor, R.; Vostrejs, W.; Amparo, A. A.; Hibshman, P. S.; Rees, M. G.; Ronan, M. M.; Roth, J. A.; Bakir, B.; Badgley, M. A.; Chabot, J. A.; Kluger, M. D.; Manji, G. A.; Quintana, E.; Wang, Z.; Smith, J. A. M.; Holderfield, M.; Wildes, D.; Aguirre, A. J.; Der, C. J.; Vonderheide, R. H</p>
        <p class="info">Score: 29.8, Published: 2023-12-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.03.569791' target='https://doi.org/10.1101/2023.12.03.569791'> 10.1101/2023.12.03.569791</a></p>
        <p class="abstract">Broad-spectrum RAS inhibition holds the potential to benefit roughly a quarter of human cancer patients whose tumors are driven by RAS mutations. However, the impact of inhibiting RAS functions in normal tissues is not known. RMC-7977 is a highly selective inhibitor of the active (GTP-bound) forms of KRAS, HRAS, and NRAS, with affinity for both mutant and wild type (WT) variants. As &gt;90% of human pancreatic ductal adenocarcinoma (PDAC) cases are driven by activating mutations in KRAS, we assessed the therapeutic potential of RMC-7977 in a comprehensive range of PDAC models, including human and murine cell lines, human patient-derived organoids, human PDAC explants, subcutaneous and orthotopic cell-line or patient derived xenografts, syngeneic allografts, and genetically engineered mouse models. We observed broad and pronounced anti-tumor activity across these models following direct RAS inhibition at doses and concentrations that were well-tolerated in vivo. Pharmacological analyses revealed divergent responses to RMC-7977 in tumor versus normal tissues. Treated tumors exhibited waves of apoptosis along with sustained proliferative arrest whereas normal tissues underwent only transient decreases in proliferation, with no evidence of apoptosis. Together, these data establish a strong preclinical rationale for the use of broad-spectrum RAS inhibition in the setting of PDAC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569954">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569954" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569954">
        <p class="paperTitle">Single cell dissection of developmental origins and transcriptional heterogeneity in B-cell acute lymphoblastic leukemia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569954" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569954" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Iacobucci, I.; Zeng, A. G. X.; Gao, Q.; Garcia-Prat, L.; Baviskar, P.; Shah, S.; Murison, A. J.; Voisin, V.; Chan-Seng-Yue, M.; Cheng, C.; Qu, C.; Bailey, C.; Lear, M.; Witkowski, M.; Zhou, X.; Zaldivar Peraza, A.; Gangwani, K.; Advani, A.; Luger, S. M.; Litzow, M. R.; Rowe, J. M.; Paietta, E. M.; Stock, W.; Dick, J. E.; Mullighan, C. G.</p>
        <p class="info">Score: 13.6, Published: 2023-12-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569954' target='https://doi.org/10.1101/2023.12.04.569954'> 10.1101/2023.12.04.569954</a></p>
        <p class="abstract">Sequencing of bulk tumor populations has improved genetic classification and risk assessment of B-ALL, but does not directly examine intratumor heterogeneity or infer leukemia cellular origins. We profiled 89 B-ALL samples by single-cell RNA-seq (scRNA-seq) and compared them to a reference map of normal human B-cell development established using both functional and molecular assays. Intra-sample heterogeneity was driven by cell cycle, metabolism, differentiation, and inflammation transcriptional programs. By inference of B lineage developmental state composition, nearly all samples possessed a high abundance of pro-B cells, with variation between samples mainly driven by sub-populations. However, ZNF384-r and DUX4-r B-ALL showed composition enrichment of hematopoietic stem cells, BCR::ABL1 and KMT2A-r ALL of Early Lymphoid progenitors, MEF2D-r and TCF3::PBX1 of Pre-B cells. Enrichment of Early Lymphoid progenitors correlated with high-risk clinical features. Understanding variation in transcriptional programs and developmental states of B-ALL by scRNA-seq refines existing clinical and genomic classifications and improves prediction of treatment outcome.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570880">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570880" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570880">
        <p class="paperTitle">Chromothripsis orchestrates leukemic transformation in blast phase MPN through targetable amplification of DYRK1A</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570880" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570880" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brierley, C. K.; Yip, B. H.; Orlando, G.; Goyal, H.; Wen, S.; Wen, J.; Levine, M. F.; Jakobsdottir, M. G.; Rodriguez-Meira, A.; Adamo, A.; Bashton, M.; Hamblin, A.; Clark, S. A.; O&#39;Sullivan, J.; Murphi, L.; Olijnik-French, A.-A.; Cotton, A.; Narina, S.; Pruett-Miller, S.; Enshaei, A.; Harrison, C.; Drummond, M.; Knapper, S.; Tefferi, A.; Antony-Debre, I.; Thongjuea, S.; Wedge, D. C.; Constantinescu, S.; Papaemmanuil, E.; Psaila, B.; Crispino, J. D.; Mead, A. J.</p>
        <p class="info">Score: 12.3, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570880' target='https://doi.org/10.1101/2023.12.08.570880'> 10.1101/2023.12.08.570880</a></p>
        <p class="abstract">Chromothripsis, the process of catastrophic shattering and haphazard repair of chromosomes, is a common event in cancer. Whether chromothripsis might constitute an actionable molecular event amenable to therapeutic targeting remains an open question. We describe recurrent chromothripsis of chromosome 21 in a subset of patients in blast phase of a myeloproliferative neoplasm (BP-MPN), which alongside other structural variants leads to amplification of a region of chromosome 21 in [~]25% of patients ( chr21amp). We report that chr21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. The chr21amp event is highly clonal and present throughout the hematopoietic hierarchy. DYRK1A, a serine threonine kinase and transcription factor, is the only gene in the 2.7Mb minimally amplified region which showed both increased expression and chromatin accessibility compared to non-chr21amp BP-MPN controls. We demonstrate that DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development, including DNA repair, STAT signalling and BCL2 overexpression. DYRK1A is essential for BP-MPN cell proliferation in vitro and in vivo, and DYRK1A inhibition synergises with BCL2 targeting to induce BP-MPN cell apoptosis. Collectively, these findings define the chr21amp event as a prognostic biomarker in BP-MPN and link chromothripsis to a druggable target.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.10.570854">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.10.570854" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.10.570854">
        <p class="paperTitle">A Mutation-driven oncofetal regression fuels phenotypic plasticity in colorectal cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.10.570854" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.10.570854" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mzoughi, S.; Schwarz, M.; Wang, X.; Demircioglu, D.; Ulukaya, G.; Mohammed, K.; Di Tullio, F.; Company, C.; Dramaretska, Y.; Leushacke, M.; Giotti, B.; Lannagan, T.; Lozano-Ojalvo, D.; Hasson, D.; Tsankov, A.; Sansom, O. J.; Marine, J.-C.; Barker, N.; Gargiulo, G.; Guccione, E.</p>
        <p class="info">Score: 11.9, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.10.570854' target='https://doi.org/10.1101/2023.12.10.570854'> 10.1101/2023.12.10.570854</a></p>
        <p class="abstract">Targeting cancer stem cells (CSCs) is crucial for effective cancer treatment1. However, the molecular mechanisms underlying resistance to LGR5&#43; CSCs depletion in colorectal cancer (CRC)2,3 remain largely elusive. Here, we unveil the existence of a primitive cell state dubbed the oncofetal (OnF) state, which works in tandem with the LGR5&#43; stem cells (SCs) to fuel tumor evolution in CRC. OnF cells emerge early during intestinal tumorigenesis and exhibit features of lineage plasticity. Normally suppressed by the Retinoid X Receptor (RXR) in mature SCs, the OnF program is triggered by genetic deletion of the gatekeeper APC. We demonstrate that diminished RXR activity unlocks an epigenetic circuity governed by the cooperative action of YAP and AP1, leading to OnF reprogramming. This high-plasticity state is inherently resistant to conventional chemotherapies and its adoption by LGR5&#43; CSCs enables them to enter a drug-tolerant state. Furthermore, through phenotypic tracing and ablation experiments, we uncover a functional redundancy between the OnF and stem cell (SC) states and show that targeting both cellular states is essential for sustained tumor regression in vivo. Collectively, these findings establish a mechanistic foundation for developing effective combination therapies with enduring impact on CRC treatment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570359">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570359" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570359">
        <p class="paperTitle">Dynamic Plasticity Systems Direct Early Adaptation to Treatment in Neuroblastoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570359" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570359" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roux, C.; Hamer, S.; Shea, A.; Chen, E.; Sadr, A. S.; English, C.; Sahoo, S.; Allo Anido, A.; Che, H.; Chesler, L.; Jolly, M. K.; Morgan, M. D.; Bruna, A.</p>
        <p class="info">Score: 10.5, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570359' target='https://doi.org/10.1101/2023.12.07.570359'> 10.1101/2023.12.07.570359</a></p>
        <p class="abstract">In paediatric cancers like neuroblastoma, limited genetic diversity emphasizes the role of phenotypic heterogeneity in driving malignancy. We investigated this phenomenon using experimental evolution and single-cell techniques in neuroblastoma preclinical models. Our findings reveal that neuroblastoma cells navigate multistable phenotypic landscapes, named plasticity systems. These finely regulate their topology and dynamics enabling tolerance, persistence, and regrowth in response to treatment. While preferential killing of adrenergic cells (ADRN), notably under cisplatin treatment, enriches drug-tolerant persister (DTP) populations with mesenchymal (MES) properties, we also observed phenotypic transitions contributing to DTP entry and exit. Additionally, single-cell-derived clone experiments unveiled a spectrum of heritable plasticity traits linked to functional properties, influencing DTP behaviour. Mathematical modelling supports the critical role of all cell phenotypes in evolutionary adaptation. Collectively, our study depicts plasticity systems as key early cancer drivers of adaptive evolution in neuroblastoma through regulating the multistability and dynamics of phenotypic landscapes. These insights underscore the necessity of decoding plasticity for advancing long-term therapeutic effectiveness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.30.569405">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.30.569405" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.30.569405">
        <p class="paperTitle">Adaptation to ARF6-depletion in KRAS-driven PDAC is abolished by targeting TLR2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.30.569405" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.30.569405" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ghose, R.; Das, S.; Urgel-Solas, J.; Pezzano, F.; Datta, D.; Ghose, U.; Ganez-Zapater, A.; Pardo-Lorente, N.; Espinar, L.; Garcia, L.; Cannata, C.; Ruprecht, V.; Janic, A.; Sdelci, S.</p>
        <p class="info">Score: 8.9, Published: 2023-12-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.30.569405' target='https://doi.org/10.1101/2023.11.30.569405'> 10.1101/2023.11.30.569405</a></p>
        <p class="abstract">Metastasis is responsible for nearly 90% of all cancer-related deaths. Despite global efforts to prevent aggressive tumours, cancers such as pancreatic ductal adenocarcinoma (PDAC) are poorly diagnosed in the primary stage, resulting in lethal metastatic disease. RAS mutations are known to promote tumour spread, with mutant KRAS present in up to 90% of cases. Until recently, mutant KRAS remained untargeted and, despite the recent development of inhibitors, results show that tumour cells develop resistance. Another strategy for targeting mutant KRAS-dependent PDAC proliferation and metastasis may come from targeting the downstream effectors of KRAS. One such axis, which controls tumour proliferation, invasiveness and immune evasion, is represented by ARF6-ASAP1. Here we show that targeting ARF6 results in adaptive rewiring that can restore proliferation and invasion potential over time. Using time-series RNA and ATAC sequencing approaches, we identified TLR-dependent NF{kappa}B, TNF and hypoxia signalling as key drivers of adaptation in ARF6-depleted KRAS-dependent PDAC. Using in vitro and in vivo assays, we show that knocking down TLR2 with ARF6 significantly reduces proliferation, migration and invasion. Taken together, our data shed light on a novel co-targeting strategy with the therapeutic potential to counteract PDAC proliferation and metastasis.

GRAPHICAL SUMMARY

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=120 SRC=&#34;FIGDIR/small/569405v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (37K):
org.highwire.dtl.DTLVardef@192ac2org.highwire.dtl.DTLVardef@4659e5org.highwire.dtl.DTLVardef@b7e225org.highwire.dtl.DTLVardef@6019aa_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570313">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570313" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570313">
        <p class="paperTitle">Hypoxia alters the effects of hypomethylating agents in acute myeloid leukaemia cells.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570313" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570313" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Humphries, S.; Burnard, S. M.; Keely, S.; Bond, D. R.; Lee, H. J.</p>
        <p class="info">Score: 6.3, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570313' target='https://doi.org/10.1101/2023.12.07.570313'> 10.1101/2023.12.07.570313</a></p>
        <p class="abstract">BackgroundAcute myeloid leukaemia (AML) is a deadly haematological malignancy that originates from mutated myeloid progenitor cells that lie quiescent in the hypoxic bone marrow. Elderly patients who cannot tolerate standard chemotherapies are administered low-dose hypomethylating agents (HMA) which act in a replication-dependent manner to reprogram the epigenome. Relapse is common following HMA treatment and may arise from quiescent leukaemia cells in the hypoxic bone marrow. Therefore, the effects of hypoxia on HMA efficacy may influence AML progression.

ResultsAML cell lines (MOLM-13, MV-4-11, HL-60) were treated with decitabine (100nM) or azacitidine (500-2000nM) in normoxic (21% O2) and hypoxic (1% O2) conditions. Exposure to hypoxia significantly reduced AML cell growth across all cell lines, with no additional effects observed upon HMA treatment. This was associated with distinct effects on DNA methylation. The extent of hypomethylation induced by AZA treatment was reduced in hypoxia, whereas DAC-induced hypomethylation was maintained in low oxygen conditions. Transcriptional response to HMA treatment were also altered in hypoxia, with HMAs failing to up-regulate antigen presentation pathways in hypoxia. In particular, human leukocyte antigens (HLAs) such as HLA-DR were increased upon HMA treatment in normoxia, but not hypoxia.

ConclusionOur results suggest that HMA-induced antigen presentation may be impaired in hypoxic tissues such as the bone marrow. This study highlights the need to consider microenvironmental factors when designing co-treatment strategies to improve HMA therapeutic efficacy.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
