<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues
Authors: Sanghvi, N.; Calvo-Alcaniz, C.; Rajagopal, P.; Scalera, S.; Canu, V.; Sinha, S.; Schischlik, F.; Wang, K.; Madan, S.; Shulman, E.; Papanicolau-Sengos, A.; Blandino, G.; Ruppin, E.; Nair, N.
Score: 31.7, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564810
Metastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to target tissues is challenging.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues
Authors: Sanghvi, N.; Calvo-Alcaniz, C.; Rajagopal, P.; Scalera, S.; Canu, V.; Sinha, S.; Schischlik, F.; Wang, K.; Madan, S.; Shulman, E.; Papanicolau-Sengos, A.; Blandino, G.; Ruppin, E.; Nair, N.
Score: 31.7, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564810
Metastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to target tissues is challenging." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-05T10:37:37+00:00" />
<meta property="article:modified_time" content="2023-11-05T10:37:37+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues
Authors: Sanghvi, N.; Calvo-Alcaniz, C.; Rajagopal, P.; Scalera, S.; Canu, V.; Sinha, S.; Schischlik, F.; Wang, K.; Madan, S.; Shulman, E.; Papanicolau-Sengos, A.; Blandino, G.; Ruppin, E.; Nair, N.
Score: 31.7, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564810
Metastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to target tissues is challenging."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues\nAuthors: Sanghvi, N.; Calvo-Alcaniz, C.; Rajagopal, P.; Scalera, S.; Canu, V.; Sinha, S.; Schischlik, F.; Wang, K.; Madan, S.; Shulman, E.; Papanicolau-Sengos, A.; Blandino, G.; Ruppin, E.; Nair, N.\nScore: 31.7, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564810\nMetastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to target tissues is challenging.",
  "keywords": [
    
  ],
  "articleBody": " Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues\nAuthors: Sanghvi, N.; Calvo-Alcaniz, C.; Rajagopal, P.; Scalera, S.; Canu, V.; Sinha, S.; Schischlik, F.; Wang, K.; Madan, S.; Shulman, E.; Papanicolau-Sengos, A.; Blandino, G.; Ruppin, E.; Nair, N.\nScore: 31.7, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564810\nMetastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to target tissues is challenging. To address this, we assessed whether primary and metastatic tumors resemble their tissue of origin or target tissue in terms of gene expression. We analyzed gene expression profiles from various cancer types, including single-cell and bulk RNA-seq data, in both paired and unpaired primary and metastatic patient cohorts. We quantified the transcriptomic distances between tumor samples and their normal tissues, revealing that primary tumors are more similar to their tissue of origin, while metastases shift towards the target tissue. Pathway-level analysis highlighted critical transcriptomic changes during metastasis. Notably, primary cancers exhibited higher activity in cancer hallmarks, including Activating Invasion and Metastasis, compared to metastatic cancers. This comprehensive landscape analysis provides insight into how cancer tumors adapt to their metastatic environments, providing a transcriptome-wide view of the processes involved.\nOncogenic RAS-Pathway Activation Drives Oncofetal Reprogramming and Creates Therapeutic Vulnerabilities in Juvenile Myelomonocytic Leukemia\nAuthors: Hartmann, M.; Schoenung, M.; Rajak, J.; Maurer, V.; Hai, L.; Bauer, K.; Hakobyan, M.; Staeble, S.; Langstein, J.; Jardine, L.; Roelz, R.; Bohler, S.; Khabirova, E.; Maag, A.-H.; Vonficht, D.; Lebrecht, D.; Bernt, K. M.; Tan, K.; Chen, C.; Alikarami, F.; Boch, T.; Flore, V.; Lutsik, P.; Milsom, M. D.; Raffel, S.; Buske, C.; Haas, S.; Haniffa, M.; Mallm, J.-P.; Behjati, S.; Bonder, M.-J.; Froehling, S.; Niemeyer, C. M.; Hey, J.; Flotho, C.; Plass, C.; Erlacher, M.; Schlesner, M.; Lipka, D. B.\nScore: 22.5, Published: 2023-11-01 DOI: 10.1101/2023.10.27.563754\nAberrant fetal gene expression facilitates tumor-specific cellular plasticity by hijacking molecular programs of embryogenesis. Persistent fetal gene signatures in childhood malignancies are typically explained by their prenatal origins. In contrast, reactivation of fetal gene expression is considered a consequence of oncofetal reprogramming (OFR) in adult malignancies and is associated with aggressive disease. To date, OFR has not been described in the context of childhood malignancies. Here, we performed a comprehensive multi-layered molecular characterization of juvenile myelomonocytic leukemia (JMML) and identified OFR as a hallmark of aggressive JMML. We observed that hematopoietic stem cells (HSCs) aberrantly express mixed developmental programs in JMML. Expression of fetal gene signatures combined with a postnatal epigenetic landscape suggested OFR, which was validated in a JMML mouse model, demonstrating that postnatal activation of RAS signaling is sufficient to induce fetal gene signatures. Integrative analysis identified the fetal HSC maturation marker CD52 as a novel therapeutic target for aggressive JMML. Anti-CD52 treatment depleted human JMML HSCs and disrupted disease propagation in vivo. In summary, this study implicates OFR, defined as postnatal acquisition of fetal transcription signatures, in the pathobiology of a childhood malignancy. We provide evidence for the direct involvement of oncogenic RAS signaling in OFR. Finally, we demonstrate how OFR can be leveraged for the development of novel treatment strategies.\nPrograms, Origins, and Niches of Immunomodulatory Myeloid Cells in Gliomas\nAuthors: Miller, T. E.; El Farran, C. A. K.; Couturier, C. P.; Chen, Z.; D'Antonio, J. P.; Verga, J.; Villanueva, M. A.; Castro, L. N. G.; Tong, Y. E.; Al-Saadi, T.; Chiocca, A. N.; Fischer, D. S.; Heiland, D. H.; Guerriero, J. L.; Petrecca, K.; Suva, M. L.; Shalek, A. K.; Bernstein, B. E.\nScore: 25.2, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563466\nGliomas are incurable malignancies notable for an immunosuppressive microenvironment with abundant myeloid cells whose immunomodulatory properties remain poorly defined. Here, utilizing scRNA-seq data for 183,062 myeloid cells from 85 human tumors, we discover that nearly all glioma-associated myeloid cells express at least one of four immunomodulatory activity programs: Scavenger Immunosuppressive, C1Q Immunosuppressive, CXCR4 Inflammatory, and IL1B Inflammatory. All four programs are present in IDH1 mutant and wild-type gliomas and are expressed in macrophages, monocytes, and microglia whether of blood or resident myeloid cell origins. Integrating our scRNA-seq data with mitochondrial DNA-based lineage tracing, spatial transcriptomics, and organoid explant systems that model peripheral monocyte infiltration, we show that these programs are driven by microenvironmental cues and therapies rather than myeloid cell type, origin, or mutation status. The C1Q Immunosuppressive program is driven by routinely administered dexamethasone. The Scavenger Immunosuppressive program includes ligands with established roles in T-cell suppression, is induced in hypoxic regions, and is associated with immunotherapy resistance. Both immunosuppressive programs are less prevalent in lower-grade gliomas, which are instead enriched for the CXCR4 Inflammatory program. Our study provides a framework to understand immunomodulatory myeloid cells in glioma, and a foundation to develop more effective immunotherapies.\nCo-regulator activity of Mediator of DNA Damage Checkpoint 1 (MDC1) is associated with DNA repair dysfunction and PARP inhibitor sensitivity in lobular carcinoma of the breast\nAuthors: Sottnik, J. L.; Shackleford, M. T.; Richer, A. L.; Fu, R.; Hesselberth, J. R.; Sikora, M. J.\nScore: 7.8, Published: 2023-10-30 DOI: 10.1101/2023.10.29.564555\nInvasive lobular carcinoma of the breast (ILC) are typically estrogen receptor (ER)-positive and present with biomarkers of anti-estrogen sensitive disease, but growing laboratory and clinical data, including poor long-term outcomes faced by patients with ILC, suggest endocrine response and ER function are unique in ILC. We previously identified the DNA repair protein Mediator of DNA Damage Checkpoint 1 (MDC1) as an ILC-specific ER co-regulator necessary for ER genomic activity, and that MDC1 co-regulator activity was associated with dysfunctional canonical DNA repair roles of MDC1. To understand these potentially reciprocal activities of MDC1 in ILC, we profiled the MDC1 interactome and found that MDC1 associated proteins in ILC cells mirror a \"BRCA-mutant\" state lacking MDC1 interaction with key homologous recombination (HR) proteins. Single-cell gene expression and DNA repair activity showed that specific activation of ER:MDC1 target genes was associated with increased PARP-associated DNA repair and decreased HR gene expression. These data suggest that HR is dysfunctional in ILC, which was supported by a lack of DNA damage-induced RAD51 turnover in ILC cells, and an elevated DNA damage response protein signature in a subset of ILC tumors. We tested whether this HR dysfunction could be exploited using PARP inhibition, and found that talazoparib treatment produced a durable growth suppression both in vitro and in ILC cell line xenografts in vivo. The ILC-specific ER:MDC1 association creates a new context for ER and MDC1 function in ILC, at the cost of a DNA repair dysfunction that may be therapeutically targetable. SignificanceILC are rarely associated with biomarkers of overt HR deficiency, as such patients are rarely eligible for treatment with PARP inhibitors. Our work suggests ILC present with a previously unappreciated form of HR dysfunction, linked to ILC-specific genomic activity of ER, that imparts sensitivity to PARP inhibition.\nSpatial tumor immune heterogeneity facilitates subtype co-existence and therapy response via AP1 dichotomy in pancreatic cancer\nAuthors: Klein, L.; Tu, M.; Krebs, N.; Urbach, L.; Grimm, D. G.; Latif, M. U.; Penz, F.; Chan, N.; Aliar, K.; Vyas, F.; Kishore, U.; Hessmann, E.; Trumpp, A.; Espinet, E.; Papantonis, A.; Khokha, R.; Ellenrieder, V.; Gruenwald, B. T.; Singh, S. K.\nScore: 6.3, Published: 2023-11-02 DOI: 10.1101/2023.10.30.563552\nPancreatic ductal adenocarcinoma (PDAC) displays a high degree of spatial subtype heterogeneity. This intratumoral co-existence of classical and basal-like programs is evident in multi-scale transcriptomic and spatial analyses of resected, advanced-stage and chemotherapy-treated specimens and reciprocally linked to a diverse stromal immune microenvironment as well as worse clinical outcome. However, the underlying mechanisms of intratumoral subtype heterogeneity remain largely unclear. Here, by combining preclinical models, multi-center clinical, bulk and compartment-specific transcriptomic, proteomic, and bioimaging data from human specimens, we identified an interplay between neoplastic intrinsic AP1 transcription factor dichotomy and extrinsic CD68+ macrophages as a driver of intratumoral subtype co-existence along with an immunosuppressive tumor microenvironment with T cell exclusion. Our ATAC , ChIP-, and RNA-seq analyses revealed that JUNB/AP1- and HDAC-mediated epigenetic programs repress pro-inflammatory immune signatures in tumor cells, antagonizing cJUN/AP1 signaling to favor a therapy-responsive classical neoplastic identity. Through the tumor microenvironment, this dichotomous regulation was further amplified via regional macrophage populations. Moreover, CD68+/TNF-alpha+ cells associated with a reactive phenotype and reduced CD8+ T cell infiltration in human PDAC tumors. Consequently, combined anti-TNF-alpha immunotherapy and chemotherapy resulted in reduced macrophage counts and promoted CD3+/CD8+ T cell infiltration in basal-like PDAC, leading to improved survival in preclinical murine models. We conclude that tumor cell intrinsic epigenetic programs, together with extrinsic microenvironmental cues, facilitate intratumoral subtype heterogeneity and disease progression.\nDosage amplification dictates oncogenic regulation by the NKX2-1 lineage factor in lung adenocarcinoma\nAuthors: Pulice, J. L.; Meyerson, M.\nScore: 8.3, Published: 2023-10-27 DOI: 10.1101/2023.10.26.563996\nAmplified oncogene expression is a critical and widespread driver event in cancer, yet our understanding of how amplification-mediated elevated dosage mediates oncogenic regulation is limited. Here, we find that the most significant focal amplification event in lung adenocarcinoma (LUAD) targets a lineage super-enhancer near the NKX2-1 lineage transcription factor. The NKX2-1 super-enhancer is targeted by focal and co-amplification with NKX2-1, and activation or repression controls NKX2-1 expression. We find that NKX2-1 is a widespread dependency in LUAD cell lines, where NKX2-1 pioneers enhancer accessibility to drive a lineage addicted state in LUAD, and NKX2-1 confers persistence to EGFR inhibitors. Notably, we find that oncogenic NKX2-1 regulation requires expression above a minimum dosage threshold--NKX2-1 dosage below this threshold is insufficient for cell viability, enhancer remodeling, and TKI persistence. Our data suggest that copy-number amplification can be a gain-of-function alteration, wherein amplification elevates oncogene expression above a critical dosage required for oncogenic regulation and cancer cell survival. O_LSTHighlightsC_LSTO_LIThe most significant amplification event in LUAD targets a lineage super-enhancer that controls expression of the NKX2-1 lineage oncogene. C_LIO_LINKX2-1 is a dosage dependency in most NKX2-1(+) LUAD cell lines C_LIO_LINKX2-1 remodels lineage enhancer accessibility to drive a lineage addicted state and confer persistence to EGFR targeted therapy C_LIO_LINKX2-1 oncogenic regulation requires a minimum oncogenic dosage, which dictates NKX2-1 regulation of enhancer remodeling, TKI persistence, and cancer cell viability C_LI\nMapping ovarian cancer spatial organization uncovers immune evasion drivers at the genetic, cellular, and tissue level\nAuthors: Yeh, C. Y.; Aguirre, K.; Laveroni, O.; Kim, S.; Wang, A.; Liang, B.; Zhang, X.; Han, L. M.; Valbuena, R.; Plevritis, S. K.; Bassik, M. C.; Snyder, M. P.; Howitt, B. E.; Jerby, L.\nScore: 79.3, Published: 2023-10-19 DOI: 10.1101/2023.10.16.562592\nImmune exclusion and evasion are central barriers to the success of immunotherapies and cell therapies in solid tumors. Here we applied single cell spatial and perturbational transcriptomics alongside clinical, histological, and genomic profiling to elucidate immune exclusion and evasion in high-grade serous tubo-ovarian cancer (HGSC). Using high-plex spatial transcriptomics we profiled more than 1.3 million cells from 95 tumors and 60 patients, revealing generalizable principles in HGSC tumor tissue organization. Our data demonstrates that effector T cells resist stroma-mediated trapping and sequestration. However, upon infiltration into the tumor, T cells, as well as Natural Killer (NK) cells, preferentially co-localize only with a subset of malignant cells that manifest a distinct transcriptional cell state. The latter consists of dozens of co-regulated genes and is repressed under various copy number alterations. Performing CRISPR Perturb-seq screens in ovarian cancer cells, we identified functionally diverse genetic perturbations - including knockout of the insulin sensing repressor PTPN1 and the epigenetic regulator ACTR8 - that de-repress the proposed immunogenic malignant cell state identified in patients and indeed sensitize ovarian cancer cells to T cell and NK cell cytotoxicity. Taken together, our study uncovered a profound connection between somatic genetic aberrations, malignant cell transcriptional dysregulation, and immune evasion at the cellular and tissue level, allowing us to identify targets that reprogram malignant cell states as an avenue to unleash anti-tumor immune responses.\nAnalysis of uveal melanoma scRNA sequencing data identifies neoplastic-immune hybrid cells that exhibit metastatic potential\nAuthors: Anderson, A. N.; Conley, P.; Klocke, C. D.; Sengupta, S. K.; Robinson, T. L.; Fan, Y.; Jones, J. A.; Gibbs, S. L.; Skalet, A. H.; Wu, G.; Wong, M. H.\nScore: 4.2, Published: 2023-10-30 DOI: 10.1101/2023.10.24.563815\nUveal melanoma (UM) is the most common non-cutaneous melanoma and is an intraocular malignancy that affects nearly 7,000 individuals per year worldwide. Of these, nearly 50% will progress to metastatic disease for which there are currently no effective therapies. Despite advances in the molecular profiling and metastatic stratification of class 1 and 2 UM tumors, little is known regarding the underlying biology of UM metastasis. Our group has identified a disseminated neoplastic cell population characterized by co-expression of immune and melanoma proteins, circulating hybrid cells (CHCs or hybrids), in patients with UM. Compared to circulating tumor cells, CHCs are detected at an increased prevalence in peripheral blood and can be used as a non-invasive biomarker to predict metastatic progression. To identify mechanisms underlying enhanced hybrid cell dissemination we sought to identify hybrid cells within primary UM tumors, including UM tissue sections and single cell RNA sequencing. Using rigorous doublet discrimination approaches, we identified UM hybrids and evaluated their gene expression and predicted ligand-receptor interactions in relation to other melanoma and immune cells within the primary tumor. We identified several genes and pathways upregulated in hybrid cells, including those involved in enhancing cell motility and cytoskeletal rearrangement, evading immune detection, and altering cellular metabolism. In addition, we identified that hybrid cells express ligand-receptor signaling pathways implicated in promoting cancer metastasis including GAS6-AXL, CXCL12-CXCR4, LGALS9-P4HB and IGF1-IGFR1. These results contribute to our understanding of tumor progression and interactions between tumor cells and immune cells in the UM microenvironment that may promote metastasis.\nImmunosuppressive tumor microenvironment of osteosarcoma\nAuthors: Taylor, A. M.; Ng, P. K. S.; Harder, J. M.; Kumar, P.; Dzis, A. M.; Jillette, N. L.; Goodspeed, A.; Bodlak, A.; Wu, Q.; Isakoff, M. S.; George, J.; Grassman, J. D. S.; Luo, D.; Flynn, W. F.; Courtois, E. T.; Robson, P.; Hayashi, M.; Paulillo, A. T.; de Toledo, S. R. C.; Balarezo, F. S.; Lindsay, A. D.; Hoang, B.; Lau, C. C.\nScore: 2.8, Published: 2023-11-03 DOI: 10.1101/2023.11.01.565008\nBackground - Osteosarcoma (OS) is the most common malignant bone tumor in children. OS is characterized by a high degree of genomic instability, resulting in copy-number alterations and genomic rearrangements with no disease-defining recurrent mutations. Given the diverse genomic landscape of OS and the difficulty of identifying druggable therapeutic targets, use of immunotherapy techniques appears lucrative. However, clinical trials based on molecular characterization have failed to find new effective therapies, and outcomes have not improved over the last 40 years. Materials/Methods - We performed single-cell RNA sequencing (scRNA-seq) using the 10x Genomics Chromium platform on six fresh tumor biopsy samples from pediatric OS patients. Raw data was processed using 10x CellRanger to produce transcript read counts for each cell. After filtering low-quality cells and doublet removal, counts were normalized using Seurat, and cells were integrated across samples with Harmony. Data was combined with a previously-published OS scRNA-seq cohort of six samples (GSE162454). Two additional OS samples were profiled using 10x Genomics Visium spatial transcriptomics for validation of discovered subtypes and to add spatial context. Results - Clustering identified 16 major cell types based on expression of canonical cell markers. Several immunosuppressive cell types were identified via subclustering of major cell types, including neutrophil myeloid-derived suppressor cells (MDSCs), regulatory and exhausted T-cells, and LAMP3+ dendritic cells. Markers for the cell types found in OS were identified for further validation using imaging techniques, including Visium spatial transcriptomics. We performed deconvolution using the scRNA-seq cell identities to examine colocalization of discovered cell types. Overall, the discovered clusters were common between patients, showing consistent cell type proportions. However, we found patient-specific differences in the frequency of some cell types, with one sample showing a higher proportion of T-cells along with increased presence of colocalized IFN-stimulated macrophages, and the other with a greater presence of neutrophils/MDSCs. Conclusions - Using single-cell transcriptomics, we were able to discover the presence of multiple immunosuppressive cell subtypes of neutrophils, T-cells, and dendritic cells. Additionally, spatial transcriptomics revealed multiple similar clusters between samples, and common colocalization of the discovered cell types within those clusters. However, differences in T-cell presence and interferon induction may be indicative of patient-specific immunogenicity in osteosarcoma tumors.\nGenomic hypomethylation in cell-free DNA predicts responses to checkpoint blockade in lung and breast cancer\nAuthors: Kim, K.; Kim, H.; Shin, I.; Noh, S.-J.; Kim, J. Y.; Suh, K. J.; Kim, Y.-N.; Lee, J.-Y.; Cho, D.-Y.; Kim, S. H.; Kim, J. H.; Lee, S.-H.; Choi, J. K.\nScore: 2.5, Published: 2023-11-02 DOI: 10.1101/2023.10.31.565052\nGenomic hypomethylation has recently been identified as a determinant of therapeutic responses to immune checkpoint blockade (ICB). However, tumor tissue is often unattainable, and tissue-based methylation profiling suffers from low tumor purity. In this study, we developed an assay named iMethyl to estimate the genomic hypomethylation status from cell-free DNA (cfDNA) as well as tissue by deep targeted sequencing of young LINE-1 elements with \u003e 400,000 reads per sample. iMethyl was applied to a total of 653 ICB samples encompassing lung cancer (cfDNA n=167; tissue n=137; cfDNA early during treatment n=40), breast cancer (cfDNA n=91; tissue n=50; PBMC n=50; cfDNA at progression n=44), and ovarian cancer (tissue n=74). iMethyl-tissue had better predictive power than tumor mutation burden and PD-L1 expression. Furthermore, iMethyl-liquid predicted ICB responses accurately regardless of the tumor purity of tissue samples. iMethyl-liquid was also able to monitor therapeutic responses early during treatment (3 or 6 weeks after initiation of ICB) and detect progressive hypomethylation accompanying tumor progression. In conclusion, our method allows for reliable noninvasive prediction, early evaluation, and monitoring of clinical responses to ICB therapy.\n",
  "wordCount" : "2918",
  "inLanguage": "en",
  "datePublished": "2023-11-05T10:37:37Z",
  "dateModified": "2023-11-05T10:37:37Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 5, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564810">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564810" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564810">
        <p class="paperTitle">Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564810" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564810" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanghvi, N.; Calvo-Alcaniz, C.; Rajagopal, P.; Scalera, S.; Canu, V.; Sinha, S.; Schischlik, F.; Wang, K.; Madan, S.; Shulman, E.; Papanicolau-Sengos, A.; Blandino, G.; Ruppin, E.; Nair, N.</p>
        <p class="info">Score: 31.7, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564810' target='https://doi.org/10.1101/2023.10.30.564810'> 10.1101/2023.10.30.564810</a></p>
        <p class="abstract">Metastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to target tissues is challenging. To address this, we assessed whether primary and metastatic tumors resemble their tissue of origin or target tissue in terms of gene expression. We analyzed gene expression profiles from various cancer types, including single-cell and bulk RNA-seq data, in both paired and unpaired primary and metastatic patient cohorts. We quantified the transcriptomic distances between tumor samples and their normal tissues, revealing that primary tumors are more similar to their tissue of origin, while metastases shift towards the target tissue. Pathway-level analysis highlighted critical transcriptomic changes during metastasis. Notably, primary cancers exhibited higher activity in cancer hallmarks, including Activating Invasion and Metastasis, compared to metastatic cancers. This comprehensive landscape analysis provides insight into how cancer tumors adapt to their metastatic environments, providing a transcriptome-wide view of the processes involved.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.563754">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.563754" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.563754">
        <p class="paperTitle">Oncogenic RAS-Pathway Activation Drives Oncofetal Reprogramming and Creates Therapeutic Vulnerabilities in Juvenile Myelomonocytic Leukemia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.563754" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.563754" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hartmann, M.; Schoenung, M.; Rajak, J.; Maurer, V.; Hai, L.; Bauer, K.; Hakobyan, M.; Staeble, S.; Langstein, J.; Jardine, L.; Roelz, R.; Bohler, S.; Khabirova, E.; Maag, A.-H.; Vonficht, D.; Lebrecht, D.; Bernt, K. M.; Tan, K.; Chen, C.; Alikarami, F.; Boch, T.; Flore, V.; Lutsik, P.; Milsom, M. D.; Raffel, S.; Buske, C.; Haas, S.; Haniffa, M.; Mallm, J.-P.; Behjati, S.; Bonder, M.-J.; Froehling, S.; Niemeyer, C. M.; Hey, J.; Flotho, C.; Plass, C.; Erlacher, M.; Schlesner, M.; Lipka, D. B.</p>
        <p class="info">Score: 22.5, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.563754' target='https://doi.org/10.1101/2023.10.27.563754'> 10.1101/2023.10.27.563754</a></p>
        <p class="abstract">Aberrant fetal gene expression facilitates tumor-specific cellular plasticity by hijacking molecular programs of embryogenesis. Persistent fetal gene signatures in childhood malignancies are typically explained by their prenatal origins. In contrast, reactivation of fetal gene expression is considered a consequence of oncofetal reprogramming (OFR) in adult malignancies and is associated with aggressive disease. To date, OFR has not been described in the context of childhood malignancies. Here, we performed a comprehensive multi-layered molecular characterization of juvenile myelomonocytic leukemia (JMML) and identified OFR as a hallmark of aggressive JMML. We observed that hematopoietic stem cells (HSCs) aberrantly express mixed developmental programs in JMML. Expression of fetal gene signatures combined with a postnatal epigenetic landscape suggested OFR, which was validated in a JMML mouse model, demonstrating that postnatal activation of RAS signaling is sufficient to induce fetal gene signatures. Integrative analysis identified the fetal HSC maturation marker CD52 as a novel therapeutic target for aggressive JMML. Anti-CD52 treatment depleted human JMML HSCs and disrupted disease propagation in vivo. In summary, this study implicates OFR, defined as postnatal acquisition of fetal transcription signatures, in the pathobiology of a childhood malignancy. We provide evidence for the direct involvement of oncogenic RAS signaling in OFR. Finally, we demonstrate how OFR can be leveraged for the development of novel treatment strategies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563466">
        <p class="paperTitle">Programs, Origins, and Niches of Immunomodulatory Myeloid Cells in Gliomas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miller, T. E.; El Farran, C. A. K.; Couturier, C. P.; Chen, Z.; D&#39;Antonio, J. P.; Verga, J.; Villanueva, M. A.; Castro, L. N. G.; Tong, Y. E.; Al-Saadi, T.; Chiocca, A. N.; Fischer, D. S.; Heiland, D. H.; Guerriero, J. L.; Petrecca, K.; Suva, M. L.; Shalek, A. K.; Bernstein, B. E.</p>
        <p class="info">Score: 25.2, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563466' target='https://doi.org/10.1101/2023.10.24.563466'> 10.1101/2023.10.24.563466</a></p>
        <p class="abstract">Gliomas are incurable malignancies notable for an immunosuppressive microenvironment with abundant myeloid cells whose immunomodulatory properties remain poorly defined. Here, utilizing scRNA-seq data for 183,062 myeloid cells from 85 human tumors, we discover that nearly all glioma-associated myeloid cells express at least one of four immunomodulatory activity programs: Scavenger Immunosuppressive, C1Q Immunosuppressive, CXCR4 Inflammatory, and IL1B Inflammatory. All four programs are present in IDH1 mutant and wild-type gliomas and are expressed in macrophages, monocytes, and microglia whether of blood or resident myeloid cell origins. Integrating our scRNA-seq data with mitochondrial DNA-based lineage tracing, spatial transcriptomics, and organoid explant systems that model peripheral monocyte infiltration, we show that these programs are driven by microenvironmental cues and therapies rather than myeloid cell type, origin, or mutation status. The C1Q Immunosuppressive program is driven by routinely administered dexamethasone. The Scavenger Immunosuppressive program includes ligands with established roles in T-cell suppression, is induced in hypoxic regions, and is associated with immunotherapy resistance. Both immunosuppressive programs are less prevalent in lower-grade gliomas, which are instead enriched for the CXCR4 Inflammatory program. Our study provides a framework to understand immunomodulatory myeloid cells in glioma, and a foundation to develop more effective immunotherapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.29.564555">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.29.564555" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.29.564555">
        <p class="paperTitle">Co-regulator activity of Mediator of DNA Damage Checkpoint 1 (MDC1) is associated with DNA repair dysfunction and PARP inhibitor sensitivity in lobular carcinoma of the breast</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.29.564555" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.29.564555" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sottnik, J. L.; Shackleford, M. T.; Richer, A. L.; Fu, R.; Hesselberth, J. R.; Sikora, M. J.</p>
        <p class="info">Score: 7.8, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.29.564555' target='https://doi.org/10.1101/2023.10.29.564555'> 10.1101/2023.10.29.564555</a></p>
        <p class="abstract">Invasive lobular carcinoma of the breast (ILC) are typically estrogen receptor  (ER)-positive and present with biomarkers of anti-estrogen sensitive disease, but growing laboratory and clinical data, including poor long-term outcomes faced by patients with ILC, suggest endocrine response and ER function are unique in ILC. We previously identified the DNA repair protein Mediator of DNA Damage Checkpoint 1 (MDC1) as an ILC-specific ER co-regulator necessary for ER genomic activity, and that MDC1 co-regulator activity was associated with dysfunctional canonical DNA repair roles of MDC1. To understand these potentially reciprocal activities of MDC1 in ILC, we profiled the MDC1 interactome and found that MDC1 associated proteins in ILC cells mirror a &#34;BRCA-mutant&#34; state lacking MDC1 interaction with key homologous recombination (HR) proteins. Single-cell gene expression and DNA repair activity showed that specific activation of ER:MDC1 target genes was associated with increased PARP-associated DNA repair and decreased HR gene expression. These data suggest that HR is dysfunctional in ILC, which was supported by a lack of DNA damage-induced RAD51 turnover in ILC cells, and an elevated DNA damage response protein signature in a subset of ILC tumors. We tested whether this HR dysfunction could be exploited using PARP inhibition, and found that talazoparib treatment produced a durable growth suppression both in vitro and in ILC cell line xenografts in vivo. The ILC-specific ER:MDC1 association creates a new context for ER and MDC1 function in ILC, at the cost of a DNA repair dysfunction that may be therapeutically targetable.

SignificanceILC are rarely associated with biomarkers of overt HR deficiency, as such patients are rarely eligible for treatment with PARP inhibitors. Our work suggests ILC present with a previously unappreciated form of HR dysfunction, linked to ILC-specific genomic activity of ER, that imparts sensitivity to PARP inhibition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.563552">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.563552" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.563552">
        <p class="paperTitle">Spatial tumor immune heterogeneity facilitates subtype co-existence and therapy response via AP1 dichotomy in pancreatic cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.563552" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.563552" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Klein, L.; Tu, M.; Krebs, N.; Urbach, L.; Grimm, D. G.; Latif, M. U.; Penz, F.; Chan, N.; Aliar, K.; Vyas, F.; Kishore, U.; Hessmann, E.; Trumpp, A.; Espinet, E.; Papantonis, A.; Khokha, R.; Ellenrieder, V.; Gruenwald, B. T.; Singh, S. K.</p>
        <p class="info">Score: 6.3, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.563552' target='https://doi.org/10.1101/2023.10.30.563552'> 10.1101/2023.10.30.563552</a></p>
        <p class="abstract">Pancreatic ductal adenocarcinoma (PDAC) displays a high degree of spatial subtype heterogeneity. This intratumoral co-existence of classical and basal-like programs is evident in multi-scale transcriptomic and spatial analyses of resected, advanced-stage and chemotherapy-treated specimens and reciprocally linked to a diverse stromal immune microenvironment as well as worse clinical outcome. However, the underlying mechanisms of intratumoral subtype heterogeneity remain largely unclear. Here, by combining preclinical models, multi-center clinical, bulk and compartment-specific transcriptomic, proteomic, and bioimaging data from human specimens, we identified an interplay between neoplastic intrinsic AP1 transcription factor dichotomy and extrinsic CD68&#43; macrophages as a driver of intratumoral subtype co-existence along with an immunosuppressive tumor microenvironment with T cell exclusion. Our ATAC , ChIP-, and RNA-seq analyses revealed that JUNB/AP1- and HDAC-mediated epigenetic programs repress pro-inflammatory immune signatures in tumor cells, antagonizing cJUN/AP1 signaling to favor a therapy-responsive classical neoplastic identity. Through the tumor microenvironment, this dichotomous regulation was further amplified via regional macrophage populations. Moreover, CD68&#43;/TNF-alpha&#43; cells associated with a reactive phenotype and reduced CD8&#43; T cell infiltration in human PDAC tumors. Consequently, combined anti-TNF-alpha immunotherapy and chemotherapy resulted in reduced macrophage counts and promoted CD3&#43;/CD8&#43; T cell infiltration in basal-like PDAC, leading to improved survival in preclinical murine models. We conclude that tumor cell intrinsic epigenetic programs, together with extrinsic microenvironmental cues, facilitate intratumoral subtype heterogeneity and disease progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.563996">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.563996" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.563996">
        <p class="paperTitle">Dosage amplification dictates oncogenic regulation by the NKX2-1 lineage factor in lung adenocarcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.563996" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.563996" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pulice, J. L.; Meyerson, M.</p>
        <p class="info">Score: 8.3, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.563996' target='https://doi.org/10.1101/2023.10.26.563996'> 10.1101/2023.10.26.563996</a></p>
        <p class="abstract">Amplified oncogene expression is a critical and widespread driver event in cancer, yet our understanding of how amplification-mediated elevated dosage mediates oncogenic regulation is limited. Here, we find that the most significant focal amplification event in lung adenocarcinoma (LUAD) targets a lineage super-enhancer near the NKX2-1 lineage transcription factor. The NKX2-1 super-enhancer is targeted by focal and co-amplification with NKX2-1, and activation or repression controls NKX2-1 expression. We find that NKX2-1 is a widespread dependency in LUAD cell lines, where NKX2-1 pioneers enhancer accessibility to drive a lineage addicted state in LUAD, and NKX2-1 confers persistence to EGFR inhibitors. Notably, we find that oncogenic NKX2-1 regulation requires expression above a minimum dosage threshold--NKX2-1 dosage below this threshold is insufficient for cell viability, enhancer remodeling, and TKI persistence. Our data suggest that copy-number amplification can be a gain-of-function alteration, wherein amplification elevates oncogene expression above a critical dosage required for oncogenic regulation and cancer cell survival.

O_LSTHighlightsC_LSTO_LIThe most significant amplification event in LUAD targets a lineage super-enhancer that controls expression of the NKX2-1 lineage oncogene.
C_LIO_LINKX2-1 is a dosage dependency in most NKX2-1(&#43;) LUAD cell lines
C_LIO_LINKX2-1 remodels lineage enhancer accessibility to drive a lineage addicted state and confer persistence to EGFR targeted therapy
C_LIO_LINKX2-1 oncogenic regulation requires a minimum oncogenic dosage, which dictates NKX2-1 regulation of enhancer remodeling, TKI persistence, and cancer cell viability
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562592">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562592" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562592">
        <p class="paperTitle">Mapping ovarian cancer spatial organization uncovers immune evasion drivers at the genetic, cellular, and tissue level</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562592" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562592" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yeh, C. Y.; Aguirre, K.; Laveroni, O.; Kim, S.; Wang, A.; Liang, B.; Zhang, X.; Han, L. M.; Valbuena, R.; Plevritis, S. K.; Bassik, M. C.; Snyder, M. P.; Howitt, B. E.; Jerby, L.</p>
        <p class="info">Score: 79.3, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562592' target='https://doi.org/10.1101/2023.10.16.562592'> 10.1101/2023.10.16.562592</a></p>
        <p class="abstract">Immune exclusion and evasion are central barriers to the success of immunotherapies and cell therapies in solid tumors. Here we applied single cell spatial and perturbational transcriptomics alongside clinical, histological, and genomic profiling to elucidate immune exclusion and evasion in high-grade serous tubo-ovarian cancer (HGSC). Using high-plex spatial transcriptomics we profiled more than 1.3 million cells from 95 tumors and 60 patients, revealing generalizable principles in HGSC tumor tissue organization. Our data demonstrates that effector T cells resist stroma-mediated trapping and sequestration. However, upon infiltration into the tumor, T cells, as well as Natural Killer (NK) cells, preferentially co-localize only with a subset of malignant cells that manifest a distinct transcriptional cell state. The latter consists of dozens of co-regulated genes and is repressed under various copy number alterations. Performing CRISPR Perturb-seq screens in ovarian cancer cells, we identified functionally diverse genetic perturbations - including knockout of the insulin sensing repressor PTPN1 and the epigenetic regulator ACTR8 - that de-repress the proposed immunogenic malignant cell state identified in patients and indeed sensitize ovarian cancer cells to T cell and NK cell cytotoxicity. Taken together, our study uncovered a profound connection between somatic genetic aberrations, malignant cell transcriptional dysregulation, and immune evasion at the cellular and tissue level, allowing us to identify targets that reprogram malignant cell states as an avenue to unleash anti-tumor immune responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563815">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563815" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563815">
        <p class="paperTitle">Analysis of uveal melanoma scRNA sequencing data identifies neoplastic-immune hybrid cells that exhibit metastatic potential</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563815" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563815" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Anderson, A. N.; Conley, P.; Klocke, C. D.; Sengupta, S. K.; Robinson, T. L.; Fan, Y.; Jones, J. A.; Gibbs, S. L.; Skalet, A. H.; Wu, G.; Wong, M. H.</p>
        <p class="info">Score: 4.2, Published: 2023-10-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563815' target='https://doi.org/10.1101/2023.10.24.563815'> 10.1101/2023.10.24.563815</a></p>
        <p class="abstract">Uveal melanoma (UM) is the most common non-cutaneous melanoma and is an intraocular malignancy that affects nearly 7,000 individuals per year worldwide. Of these, nearly 50% will progress to metastatic disease for which there are currently no effective therapies. Despite advances in the molecular profiling and metastatic stratification of class 1 and 2 UM tumors, little is known regarding the underlying biology of UM metastasis. Our group has identified a disseminated neoplastic cell population characterized by co-expression of immune and melanoma proteins, circulating hybrid cells (CHCs or hybrids), in patients with UM. Compared to circulating tumor cells, CHCs are detected at an increased prevalence in peripheral blood and can be used as a non-invasive biomarker to predict metastatic progression. To identify mechanisms underlying enhanced hybrid cell dissemination we sought to identify hybrid cells within primary UM tumors, including UM tissue sections and single cell RNA sequencing. Using rigorous doublet discrimination approaches, we identified UM hybrids and evaluated their gene expression and predicted ligand-receptor interactions in relation to other melanoma and immune cells within the primary tumor. We identified several genes and pathways upregulated in hybrid cells, including those involved in enhancing cell motility and cytoskeletal rearrangement, evading immune detection, and altering cellular metabolism. In addition, we identified that hybrid cells express ligand-receptor signaling pathways implicated in promoting cancer metastasis including GAS6-AXL, CXCL12-CXCR4, LGALS9-P4HB and IGF1-IGFR1. These results contribute to our understanding of tumor progression and interactions between tumor cells and immune cells in the UM microenvironment that may promote metastasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565008">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565008" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565008">
        <p class="paperTitle">Immunosuppressive tumor microenvironment of osteosarcoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565008" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565008" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Taylor, A. M.; Ng, P. K. S.; Harder, J. M.; Kumar, P.; Dzis, A. M.; Jillette, N. L.; Goodspeed, A.; Bodlak, A.; Wu, Q.; Isakoff, M. S.; George, J.; Grassman, J. D. S.; Luo, D.; Flynn, W. F.; Courtois, E. T.; Robson, P.; Hayashi, M.; Paulillo, A. T.; de Toledo, S. R. C.; Balarezo, F. S.; Lindsay, A. D.; Hoang, B.; Lau, C. C.</p>
        <p class="info">Score: 2.8, Published: 2023-11-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565008' target='https://doi.org/10.1101/2023.11.01.565008'> 10.1101/2023.11.01.565008</a></p>
        <p class="abstract">Background - Osteosarcoma (OS) is the most common malignant bone tumor in children. OS is characterized by a high degree of genomic instability, resulting in copy-number alterations and genomic rearrangements with no disease-defining recurrent mutations. Given the diverse genomic landscape of OS and the difficulty of identifying druggable therapeutic targets, use of immunotherapy techniques appears lucrative. However, clinical trials based on molecular characterization have failed to find new effective therapies, and outcomes have not improved over the last 40 years. Materials/Methods - We performed single-cell RNA sequencing (scRNA-seq) using the 10x Genomics Chromium platform on six fresh tumor biopsy samples from pediatric OS patients. Raw data was processed using 10x CellRanger to produce transcript read counts for each cell. After filtering low-quality cells and doublet removal, counts were normalized using Seurat, and cells were integrated across samples with Harmony. Data was combined with a previously-published OS scRNA-seq cohort of six samples (GSE162454). Two additional OS samples were profiled using 10x Genomics Visium spatial transcriptomics for validation of discovered subtypes and to add spatial context. Results - Clustering identified 16 major cell types based on expression of canonical cell markers. Several immunosuppressive cell types were identified via subclustering of major cell types, including neutrophil myeloid-derived suppressor cells (MDSCs), regulatory and exhausted T-cells, and LAMP3&#43; dendritic cells. Markers for the cell types found in OS were identified for further validation using imaging techniques, including Visium spatial transcriptomics. We performed deconvolution using the scRNA-seq cell identities to examine colocalization of discovered cell types. Overall, the discovered clusters were common between patients, showing consistent cell type proportions. However, we found patient-specific differences in the frequency of some cell types, with one sample showing a higher proportion of T-cells along with increased presence of colocalized IFN-stimulated macrophages, and the other with a greater presence of neutrophils/MDSCs. Conclusions - Using single-cell transcriptomics, we were able to discover the presence of multiple immunosuppressive cell subtypes of neutrophils, T-cells, and dendritic cells. Additionally, spatial transcriptomics revealed multiple similar clusters between samples, and common colocalization of the discovered cell types within those clusters. However, differences in T-cell presence and interferon induction may be indicative of patient-specific immunogenicity in osteosarcoma tumors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.31.565052">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.31.565052" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.31.565052">
        <p class="paperTitle">Genomic hypomethylation in cell-free DNA predicts responses to checkpoint blockade in lung and breast cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.31.565052" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.31.565052" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, K.; Kim, H.; Shin, I.; Noh, S.-J.; Kim, J. Y.; Suh, K. J.; Kim, Y.-N.; Lee, J.-Y.; Cho, D.-Y.; Kim, S. H.; Kim, J. H.; Lee, S.-H.; Choi, J. K.</p>
        <p class="info">Score: 2.5, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.31.565052' target='https://doi.org/10.1101/2023.10.31.565052'> 10.1101/2023.10.31.565052</a></p>
        <p class="abstract">Genomic hypomethylation has recently been identified as a determinant of therapeutic responses to immune checkpoint blockade (ICB). However, tumor tissue is often unattainable, and tissue-based methylation profiling suffers from low tumor purity. In this study, we developed an assay named iMethyl to estimate the genomic hypomethylation status from cell-free DNA (cfDNA) as well as tissue by deep targeted sequencing of young LINE-1 elements with &gt; 400,000 reads per sample. iMethyl was applied to a total of 653 ICB samples encompassing lung cancer (cfDNA n=167; tissue n=137; cfDNA early during treatment n=40), breast cancer (cfDNA n=91; tissue n=50; PBMC n=50; cfDNA at progression n=44), and ovarian cancer (tissue n=74). iMethyl-tissue had better predictive power than tumor mutation burden and PD-L1 expression. Furthermore, iMethyl-liquid predicted ICB responses accurately regardless of the tumor purity of tissue samples. iMethyl-liquid was also able to monitor therapeutic responses early during treatment (3 or 6 weeks after initiation of ICB) and detect progressive hypomethylation accompanying tumor progression. In conclusion, our method allows for reliable noninvasive prediction, early evaluation, and monitoring of clinical responses to ICB therapy.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
