<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Major data analysis errors invalidate cancer microbiome findings
Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.
Score: 648.7, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993
We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Major data analysis errors invalidate cancer microbiome findings
Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.
Score: 648.7, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993
We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-16T10:37:20+00:00" />
<meta property="article:modified_time" content="2023-08-16T10:37:20+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Major data analysis errors invalidate cancer microbiome findings
Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.
Score: 648.7, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993
We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Major data analysis errors invalidate cancer microbiome findings\nAuthors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.\nScore: 648.7, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993\nWe re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier.",
  "keywords": [
    
  ],
  "articleBody": " Major data analysis errors invalidate cancer microbiome findings\nAuthors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.\nScore: 648.7, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993\nWe re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier. Each of these problems invalidates the results, leading to the conclusion that the microbiome-based classifiers for identifying cancer presented in the study are entirely wrong. These flaws have subsequently affected more than a dozen additional published studies that used the same data and whose results are likely invalid as well.\nBMP2 and BMP7 cooperate with H3.3K27M to promote quiescence and invasiveness in pediatric diffuse midline gliomas\nAuthors: Huchede, P.; Meyer, S.; Berthelot, C.; Hamadou, M.; Bertrand-Chapel, A.; Rakotomalala, A.; MANCEAU, L.; Tomine, J.; Lespinasse, N.; Lewandowski, P.; Cordier-Bussat, M.; Broutier, L.; Dutour, A.; Rochet, I.; Blay, J.-Y.; Degletagne, C.; Attignon, V.; Montero-Carcaboso, A.; Le Grand, M.; Pasquier, E.; Vasiljevic, A.; GILARDI-HEBENSTREIT, P.; Meignan, S.; Leblond, P.; RIBES, V.; Cosset, E.; MARIE, C.\nScore: 15.5, Published: 2023-08-11 DOI: 10.1101/2023.08.09.552628\nPediatric diffuse midline gliomas (pDMG) are an aggressive type of childhood cancer with a fatal outcome. Their major epigenetic determinism has become clear, notably with the identification of K27M mutations in histone H3. However, the synergistic oncogenic mechanisms that induce and maintain tumor cell phenotype have yet to be deciphered. In 20 to 30% of cases, these tumors have an altered BMP signaling pathway with an oncogenic mutation on the BMP type I receptor ALK2, encoded by ACVR1. However, the potential impact of the BMP pathway in tumors non-mutated for ACVR1 is less clear. By integrating bulk, single-cell and spatial transcriptomic data, we show here that the BMP signaling pathway is activated at similar levels between ACVR1 wild type and mutant tumors and identify BMP2 and BMP7 as putative activators of the pathway in a specific subpopulation of cells. By using both pediatric isogenic glioma lines genetically modified to overexpress H3.3K27M and patients-derived DIPG cell lines, we demonstrate that BMP2/7 synergizes with H3.3K27M to induce a transcriptomic rewiring associated with a quiescent but invasive cell state. These data suggest a generic oncogenic role for the BMP pathway in gliomagenesis of pDMG and pave the way for specific targeting of downstream effectors mediating the BMP/K27M crosstalk.\nDNA repair and anti-cancer mechanisms in the longest-living mammal: the bowhead whale\nAuthors: Firsanov, D.; Zacher, M.; Tian, X.; Zhao, Y.; George, J. C.; Sformo, T. L.; Tombline, G.; Biashad, S. A.; Gilman, A.; Hamilton, N.; Patel, A.; Straight, M.; Lee, M.; Lu, J. Y.; Haseljic, E.; Williams, A.; Miller, N.; Gladyshev, V. N.; Zhang, Z.; Vijg, J.; Seluanov, A.; Gorbunova, V.\nScore: 649.8, Published: 2023-08-01 DOI: 10.1101/2023.05.07.539748\nAt over 200 years, the maximum lifespan of the bowhead whale exceeds that of all other mammals. The bowhead is also the second-largest animal on Earth, reaching over 80,000 kg1. In spite of its very large number of cells, the bowhead is not highly cancer-prone, an incongruity termed Petos Paradox2. This has been explained by the evolution of additional tumor suppressor genes in larger animals, which is supported by research on elephants demonstrating expansion of the p53 gene3-5. However, we show here that bowhead whale fibroblasts undergo oncogenic transformation after disruption of fewer tumor suppressors than required for human fibroblasts. Instead, analysis of DNA repair revealed that bowhead cells repair double-strand breaks with uniquely high efficiency and accuracy compared to other mammals. Further, we identified two proteins, CIRBP and RPA2, that are present at high levels in bowhead fibroblasts and increase the efficiency and fidelity of DNA repair in human cells. These results suggest that rather than possessing additional tumor suppressor genes as barriers to oncogenesis, the bowhead whale relies on more accurate and efficient DNA repair to preserve genome integrity. This strategy that does not eliminate cells but repairs them may be critical for the long and cancer-free lifespan of the bowhead whale. Our work demonstrates the value of studying long-lived organisms in identifying novel longevity mechanisms and their potential for translation to humans.\nSCD inhibition eradicates AML displaying high de novo fatty acid desaturation and synergizes with chemotherapy\nAuthors: Dembitz, V.; Lawson, H.; Burt, R.; Philippe, C.; James, S. C.; Atkinson, S.; Durko, J.; Wang, L. M.; Campos, J.; Magee, A. M.; Woodley, K.; Austin, M. J.; Rio-Machin, A.; Casado-Izquierdo, P.; Bewicke-Copley, F.; Rodriguez, G.; Pereira, D.; Oudejans, L.; Boet, E.; von Kriegsheim, A.; Schwaller, J.; Finch, A. J.; Pate, B.; Sarry, J.-E.; Tamburini, J.; Schuringa, J. J.; Hazlehurst, L.; Copland, J. A.; Yuneva, M.; Peck, B.; Cutillas, P.; Fitzgibbon, J.; Rouault-Pierre, K.; Kranc, K.; Gallipoli, P.\nScore: 9.7, Published: 2023-08-05 DOI: 10.1101/2023.08.02.551656\nIdentification of specific and therapeutically actionable vulnerabilities in acute myeloid leukaemia (AML) is needed to improve patients outcome. These features should be ideally present in many patients independently of mutational background. Here we identify de novo fatty acid (FA) desaturation, specifically stearoyl-CoA desaturase (SCD) inhibition, as a therapeutic vulnerability across multiple AML models in vitro and in vivo. SCD inhibition induces AML cell death via pleiotropic effects, and sensitivity is based on their dependency on FA desaturation regardless of mutational profile. Its efficacy is enhanced by driving FA biosynthesis in vitro and is less prominent in vivo due to stromal protection. SCD inhibition increases DNA damage and its combination with standard DNA damage-inducing chemotherapy prolongs survival in aggressive murine AML models. Our work supports developing FA desaturase inhibitors in AML while stressing the importance of identifying predictive biomarkers of response and biologically validated combination therapies to realize their therapeutic potential. KEY POINTSO_LISCD expression is highly prognostic in AML and SCD inhibition is toxic in AML cells displaying high rates of fatty acid desaturation. C_LIO_LISCD inhibition in combination with conventional chemotherapy prolongs survival in murine AML models. C_LI\nSynergistic effects of inhibitors targeting PI3K and Aurora Kinase A in preclinical inflammatory breast cancer models\nAuthors: Al-Ali, N.; Kment, J.; Young, S.; Craig, A.\nScore: 8.7, Published: 2023-08-13 DOI: 10.1101/2023.08.11.552992\nBackground: Inflammatory breast cancer (IBC) is an aggressive clinical subtype of breast cancer often diagnosed in young women. Lymph node and distant metastases are frequently detected at diagnosis of IBC, and improvements in systemic therapies are needed. For IBC that lack hormone or HER2 expression, no targeted therapies are available. Since the phosphatidyl inositol 3' kinase (PI3K) pathway is frequently deregulated in IBC, some studies have tested the pan PI3K inhibitor Buparlisib (BKM120). Although the SUM149 IBC cell line was resistant to Buparlisib, a functional genomic screen showed that silencing of Aurora kinase A (AURKA) sensitized cells to killing by Buparlisib. In this study, we tested whether combination treatments of PI3K and AURKA inhibitors act synergistically to kill IBC cells and tumors. Methods: SUM149 cells were treated with increasing doses of PI3K inhibitor Buparlisib (BKM120) and AURKA inhibitor Alisertib as monotherapies or combination therapies. Effects on target pathways, cytotoxicity, cell cycle, soft agar colony growth and cell migration were analyzed. The individual and combined treatments were also tested in a mammary orthotopic SUM149 tumor xenograft model to measure effects on tumor growth and metastasis Results: The SUM149 IBC cell line treated with Buparlisib showed reduced PI3K/AKT activation but no significant skewing of cell cycle progression. Parallel studies of Alisertib treatment showed that AURKA inhibition led to a significant block in G2/M transition in SUM149 cells. In cytotoxicity assays, Buparlisib and Alisertib combination treatments were highly synergistic compared to monotherapy controls. Evidence of synergy between Buparlisib and Alisertib also extended to soft agar colony growth and wound healing motility in SUM149 cells. The combination of Buparlisib and Alisertib also reduced IBC tumor growth in mammary orthotopic xenograft assays and reduced spontaneous metastases development in lung tissue. Conclusions: Although SUM149 IBC cells were relatively resistant to killing by the PI3K inhibitor Buparlisib, our study showed that co-targeting the mitotic kinase AURKA with Alisertib synergized to limit IBC cell growth and motility, as well as IBC tumor growth and metastasis.\nLoss of Polycomb Repressive Complex 2 function causes asparaginase resistance in T-acute lymphoblastic leukemia through decreased WNT pathway activity\nAuthors: Lefeivre, T.; Tudose, C.; Grosu, T.-I.; Jones, L.; Leon, T.; Wynne, K.; Oliviero, G.; Smith, O.; Trinquand, A.; Mansour, M.; Ryan, C.; Bond, J.\nScore: 13.1, Published: 2023-08-06 DOI: 10.1101/2023.08.04.552014\nLoss-of-function mutations and deletions in core components of the epigenetic Polycomb Repressive Complex 2 (PRC2) are associated with poor prognosis and treatment resistance in T-acute lymphoblastic leukemia (T-ALL). We leveraged clinical mutational and transcriptional data to identify a functional link between PRC2 alterations and changes in WNT signaling pathway activity in leukemia cells. Computational integration of transcriptomic, proteomic and phosphoproteomic data from an isogenic T-ALL cellular model revealed reduced activity of the WNT-dependent stabilization of proteins (WNT/STOP) pathway in cells lacking core PRC2 factor EZH2. We discovered that PRC2 loss significantly reduced sensitivity to key T-ALL treatment asparaginase, and that this was mechanistically linked to increased cellular ubiquitination levels that bolstered leukemia cell asparagine reserves. We further found that asparaginase resistance in PRC2-depleted leukemic blasts could be mitigated by pharmaceutical proteasome inhibition, thereby providing a novel and clinically tractable means to tackle induction treatment failure in high-risk T-ALL.\nPARG inhibition induces nuclear aggregation of PARylated PARP1\nAuthors: Paradkar, S.; Purcell, J.; Cui, A.; Friedman, S.; Sundaram, R.; Jensen, R. B.; Bindra, R. S.\nScore: 6.2, Published: 2023-08-15 DOI: 10.1101/2023.08.12.553088\nPARG inhibitors are currently under clinical development for the treatment of DNA repair-deficient cancers, however, their precise mechanism of action is still unclear. Here we report that PARG inhibition causes increased nuclear PARylated PARP1 that limits PARP1 chromatin binding in response to DNA damage. This PARylated PARP1 accumulates as aggregates at sites distinct from the site of DNA damage, leading to the mis-localization of PARP1. Additionally, these aggregates are formed through PAR chains as abrogating PARP1 catalytic activity prevents their formation. Finally, these PARP1 nuclear aggregates persist long-term and are associated with cleaved cytoplasmic PARP1, a cell death hallmark, which ultimately leads to a non-apoptotic form of cell death. Overall, our data uncovers a novel mechanism of PARG inhibitor cytotoxicity, which will inform ongoing clinical studies.\nGermline mutation rate predicts cancer mortality across 37 vertebrate species\nAuthors: Kapsetaki, S. E.; Compton, Z. T.; Mellon, W.; Vincze, O.; Giraudeau, M. T.; Harrison, T.; Abegglen, L.; Boddy, A.; Maley, C. C.; Schiffman, J.\nScore: 5.9, Published: 2023-08-15 DOI: 10.1101/2023.08.13.553123\nThe explanation for why some species are more susceptible to cancer than others remains an area of intense investigation. Cancer evolves in part through the accumulation of mutations and, therefore, we hypothesized that germline mutation rates would be associated with cancer prevalence and mortality across species. We collected previously published data on germline mutation rate and cancer mortality data for 37 vertebrate species. Germline mutation rate was positively correlated with cancer mortality (P = 0.008). Why animals with increased germline mutation rates die more from cancer remains an open question, however they may benefit from close monitoring for tumors due to hereditary cancer predisposition syndromes. Early diagnoses of cancer in these species may increase their chances of treatment and overall survival.\nEstablishing conditions for the generation and maintenance of estrogen receptor-positive organoid models of breast cancer\nAuthors: Oliphant, M. U.; Akshinthala, D.; Muthuswamy, S. K.\nScore: 5.8, Published: 2023-08-13 DOI: 10.1101/2023.08.09.552657\nPatient-derived organoid models of estrogen receptor-positive (ER+) breast cancer would provide a much-needed tool to better understand drug resistance and disease progression. However, the establishment and long-term maintenance of ER expression, function, and response in vitro remains a significant challenge. Here, we report the development of an ER+ breast tumor organoid medium (BTOM-ER) that conserves ER expression, estrogen responsiveness, and dependence, as well as sensitivity to endocrine therapy of ER+ patient-derived xenograft organoids (PDXO). Our findings demonstrate the utility of subtype-specific culture conditions that better mimic the characteristics of the breast epithelial biology and microenvironment, providing a powerful platform for investigating therapy response and disease progression of ER+ breast cancer.\nA pro-inflammatory stem cell niche drives myelofibrosis through a targetable galectin 1 axis\nAuthors: Li, R.; Colombo, M.; Wang, G.; Rodriguez-Romera, A.; O'Sullivan, J.; Clark, S.-A.; Perez Saez, J.; Meng, Y.; Khan, A.; Wen, S.; Dong, P.; Zhou, W.; Sousos, N.; Murphy, L.; Clarke, M.; Jooss, N.; Olijnik, A.-A.; Wong, Z.; Sirinukunwattana, K.; Ryou, H.; Norfo, R.; Cheng, Q.; Brierley, C.; Carrelha, J.; Ren, Z.; Thongjuea, S.; Rathinam, V.; Krishnan, A. K.; Royston, D.; Rabinovich, G.; Mead, A.; Psaila, B.\nScore: 27.0, Published: 2023-08-07 DOI: 10.1101/2023.08.05.550630\nMyeloproliferative neoplasms are stem cell-driven cancers associated with a large burden of morbidity and mortality. The majority of patients present with early-stage disease, but a substantial proportion progress to myelofibrosis and/or secondary leukemia, advanced cancers with a poor prognosis and high symptom burden. Currently, it remains difficult to predict progression, and we lack therapies that reliably prevent or reverse fibrosis development. A major bottleneck to the discovery of disease-modifying therapies has been an incomplete understanding of the interplay between perturbed cellular and molecular states. Several cell types have individually been implicated, but a comprehensive analysis of myelofibrotic bone marrow is lacking. We therefore mapped the crosstalk between bone marrow cell types in myelofibrotic bone marrow. We found that inflammation and fibrosis are orchestrated by a quartet of immune and stromal cell lineages - with basophils and mast cells creating a TNF signaling hub, communicating with megakaryocytes, mesenchymal stromal cells and pro-inflammatory fibroblasts. We identified the y-galactoside binding protein galectin 1 as a striking biomarker of progression to myelofibrosis and poor survival in multiple patient cohorts, and as a promising therapeutic target, with reduced myeloproliferation and fibrosis in vitro and in vivo and improved survival following galectin 1 inhibition. In human bone marrow organoids, TNF increased galectin 1 expression, suggesting a feedback loop wherein the pro-inflammatory MPN clone creates a self-reinforcing niche, fueling progression to advanced disease. This study provides a valuable resource for studying hematopoietic cell-niche interactions, with broad relevance for cancer-associated inflammation and disorders of tissue fibrosis.\n",
  "wordCount" : "2423",
  "inLanguage": "en",
  "datePublished": "2023-08-16T10:37:20Z",
  "dateModified": "2023-08-16T10:37:20Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on August 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550993">
        <p class="paperTitle">Major data analysis errors invalidate cancer microbiome findings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.</p>
        <p class="info">Score: 648.7, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550993' target='https://doi.org/10.1101/2023.07.28.550993'> 10.1101/2023.07.28.550993</a></p>
        <p class="abstract">We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier. Each of these problems invalidates the results, leading to the conclusion that the microbiome-based classifiers for identifying cancer presented in the study are entirely wrong. These flaws have subsequently affected more than a dozen additional published studies that used the same data and whose results are likely invalid as well.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552628">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552628" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552628">
        <p class="paperTitle">BMP2 and BMP7 cooperate with H3.3K27M to promote quiescence and invasiveness in pediatric diffuse midline gliomas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552628" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552628" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huchede, P.; Meyer, S.; Berthelot, C.; Hamadou, M.; Bertrand-Chapel, A.; Rakotomalala, A.; MANCEAU, L.; Tomine, J.; Lespinasse, N.; Lewandowski, P.; Cordier-Bussat, M.; Broutier, L.; Dutour, A.; Rochet, I.; Blay, J.-Y.; Degletagne, C.; Attignon, V.; Montero-Carcaboso, A.; Le Grand, M.; Pasquier, E.; Vasiljevic, A.; GILARDI-HEBENSTREIT, P.; Meignan, S.; Leblond, P.; RIBES, V.; Cosset, E.; MARIE, C.</p>
        <p class="info">Score: 15.5, Published: 2023-08-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552628' target='https://doi.org/10.1101/2023.08.09.552628'> 10.1101/2023.08.09.552628</a></p>
        <p class="abstract">Pediatric diffuse midline gliomas (pDMG) are an aggressive type of childhood cancer with a fatal outcome. Their major epigenetic determinism has become clear, notably with the identification of K27M mutations in histone H3. However, the synergistic oncogenic mechanisms that induce and maintain tumor cell phenotype have yet to be deciphered.

In 20 to 30% of cases, these tumors have an altered BMP signaling pathway with an oncogenic mutation on the BMP type I receptor ALK2, encoded by ACVR1. However, the potential impact of the BMP pathway in tumors non-mutated for ACVR1 is less clear. By integrating bulk, single-cell and spatial transcriptomic data, we show here that the BMP signaling pathway is activated at similar levels between ACVR1 wild type and mutant tumors and identify BMP2 and BMP7 as putative activators of the pathway in a specific subpopulation of cells. By using both pediatric isogenic glioma lines genetically modified to overexpress H3.3K27M and patients-derived DIPG cell lines, we demonstrate that BMP2/7 synergizes with H3.3K27M to induce a transcriptomic rewiring associated with a quiescent but invasive cell state. These data suggest a generic oncogenic role for the BMP pathway in gliomagenesis of pDMG and pave the way for specific targeting of downstream effectors mediating the BMP/K27M crosstalk.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.05.07.539748">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.05.07.539748" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.05.07.539748">
        <p class="paperTitle">DNA repair and anti-cancer mechanisms in the longest-living mammal: the bowhead whale</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.05.07.539748" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.05.07.539748" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Firsanov, D.; Zacher, M.; Tian, X.; Zhao, Y.; George, J. C.; Sformo, T. L.; Tombline, G.; Biashad, S. A.; Gilman, A.; Hamilton, N.; Patel, A.; Straight, M.; Lee, M.; Lu, J. Y.; Haseljic, E.; Williams, A.; Miller, N.; Gladyshev, V. N.; Zhang, Z.; Vijg, J.; Seluanov, A.; Gorbunova, V.</p>
        <p class="info">Score: 649.8, Published: 2023-08-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.05.07.539748' target='https://doi.org/10.1101/2023.05.07.539748'> 10.1101/2023.05.07.539748</a></p>
        <p class="abstract">At over 200 years, the maximum lifespan of the bowhead whale exceeds that of all other mammals. The bowhead is also the second-largest animal on Earth, reaching over 80,000 kg1. In spite of its very large number of cells, the bowhead is not highly cancer-prone, an incongruity termed Petos Paradox2. This has been explained by the evolution of additional tumor suppressor genes in larger animals, which is supported by research on elephants demonstrating expansion of the p53 gene3-5. However, we show here that bowhead whale fibroblasts undergo oncogenic transformation after disruption of fewer tumor suppressors than required for human fibroblasts. Instead, analysis of DNA repair revealed that bowhead cells repair double-strand breaks with uniquely high efficiency and accuracy compared to other mammals. Further, we identified two proteins, CIRBP and RPA2, that are present at high levels in bowhead fibroblasts and increase the efficiency and fidelity of DNA repair in human cells. These results suggest that rather than possessing additional tumor suppressor genes as barriers to oncogenesis, the bowhead whale relies on more accurate and efficient DNA repair to preserve genome integrity. This strategy that does not eliminate cells but repairs them may be critical for the long and cancer-free lifespan of the bowhead whale. Our work demonstrates the value of studying long-lived organisms in identifying novel longevity mechanisms and their potential for translation to humans.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551656">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551656" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551656">
        <p class="paperTitle">SCD inhibition eradicates AML displaying high de novo fatty acid desaturation and synergizes with chemotherapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551656" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551656" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dembitz, V.; Lawson, H.; Burt, R.; Philippe, C.; James, S. C.; Atkinson, S.; Durko, J.; Wang, L. M.; Campos, J.; Magee, A. M.; Woodley, K.; Austin, M. J.; Rio-Machin, A.; Casado-Izquierdo, P.; Bewicke-Copley, F.; Rodriguez, G.; Pereira, D.; Oudejans, L.; Boet, E.; von Kriegsheim, A.; Schwaller, J.; Finch, A. J.; Pate, B.; Sarry, J.-E.; Tamburini, J.; Schuringa, J. J.; Hazlehurst, L.; Copland, J. A.; Yuneva, M.; Peck, B.; Cutillas, P.; Fitzgibbon, J.; Rouault-Pierre, K.; Kranc, K.; Gallipoli, P.</p>
        <p class="info">Score: 9.7, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551656' target='https://doi.org/10.1101/2023.08.02.551656'> 10.1101/2023.08.02.551656</a></p>
        <p class="abstract">Identification of specific and therapeutically actionable vulnerabilities in acute myeloid leukaemia (AML) is needed to improve patients outcome. These features should be ideally present in many patients independently of mutational background. Here we identify de novo fatty acid (FA) desaturation, specifically stearoyl-CoA desaturase (SCD) inhibition, as a therapeutic vulnerability across multiple AML models in vitro and in vivo. SCD inhibition induces AML cell death via pleiotropic effects, and sensitivity is based on their dependency on FA desaturation regardless of mutational profile. Its efficacy is enhanced by driving FA biosynthesis in vitro and is less prominent in vivo due to stromal protection. SCD inhibition increases DNA damage and its combination with standard DNA damage-inducing chemotherapy prolongs survival in aggressive murine AML models. Our work supports developing FA desaturase inhibitors in AML while stressing the importance of identifying predictive biomarkers of response and biologically validated combination therapies to realize their therapeutic potential.

KEY POINTSO_LISCD expression is highly prognostic in AML and SCD inhibition is toxic in AML cells displaying high rates of fatty acid desaturation.
C_LIO_LISCD inhibition in combination with conventional chemotherapy prolongs survival in murine AML models.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.11.552992">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.11.552992" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.11.552992">
        <p class="paperTitle">Synergistic effects of inhibitors targeting PI3K and Aurora Kinase A in preclinical inflammatory breast cancer models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.11.552992" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.11.552992" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Al-Ali, N.; Kment, J.; Young, S.; Craig, A.</p>
        <p class="info">Score: 8.7, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.11.552992' target='https://doi.org/10.1101/2023.08.11.552992'> 10.1101/2023.08.11.552992</a></p>
        <p class="abstract">Background: Inflammatory breast cancer (IBC) is an aggressive clinical subtype of breast cancer often diagnosed in young women. Lymph node and distant metastases are frequently detected at diagnosis of IBC, and improvements in systemic therapies are needed. For IBC that lack hormone or HER2 expression, no targeted therapies are available. Since the phosphatidyl inositol 3&#39; kinase (PI3K) pathway is frequently deregulated in IBC, some studies have tested the pan PI3K inhibitor Buparlisib (BKM120). Although the SUM149 IBC cell line was resistant to Buparlisib, a functional genomic screen showed that silencing of Aurora kinase A (AURKA) sensitized cells to killing by Buparlisib. In this study, we tested whether combination treatments of PI3K and AURKA inhibitors act synergistically to kill IBC cells and tumors. Methods: SUM149 cells were treated with increasing doses of PI3K inhibitor Buparlisib (BKM120) and AURKA inhibitor Alisertib as monotherapies or combination therapies. Effects on target pathways, cytotoxicity, cell cycle, soft agar colony growth and cell migration were analyzed. The individual and combined treatments were also tested in a mammary orthotopic SUM149 tumor xenograft model to measure effects on tumor growth and metastasis Results: The SUM149 IBC cell line treated with Buparlisib showed reduced PI3K/AKT activation but no significant skewing of cell cycle progression. Parallel studies of Alisertib treatment showed that AURKA inhibition led to a significant block in G2/M transition in SUM149 cells. In cytotoxicity assays, Buparlisib and Alisertib combination treatments were highly synergistic compared to monotherapy controls. Evidence of synergy between Buparlisib and Alisertib also extended to soft agar colony growth and wound healing motility in SUM149 cells. The combination of Buparlisib and Alisertib also reduced IBC tumor growth in mammary orthotopic xenograft assays and reduced spontaneous metastases development in lung tissue. Conclusions: Although SUM149 IBC cells were relatively resistant to killing by the PI3K inhibitor Buparlisib, our study showed that co-targeting the mitotic kinase AURKA with Alisertib synergized to limit IBC cell growth and motility, as well as IBC tumor growth and metastasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552014" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552014">
        <p class="paperTitle">Loss of Polycomb Repressive Complex 2 function causes asparaginase resistance in T-acute lymphoblastic leukemia through decreased WNT pathway activity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lefeivre, T.; Tudose, C.; Grosu, T.-I.; Jones, L.; Leon, T.; Wynne, K.; Oliviero, G.; Smith, O.; Trinquand, A.; Mansour, M.; Ryan, C.; Bond, J.</p>
        <p class="info">Score: 13.1, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552014' target='https://doi.org/10.1101/2023.08.04.552014'> 10.1101/2023.08.04.552014</a></p>
        <p class="abstract">Loss-of-function mutations and deletions in core components of the epigenetic Polycomb Repressive Complex 2 (PRC2) are associated with poor prognosis and treatment resistance in T-acute lymphoblastic leukemia (T-ALL). We leveraged clinical mutational and transcriptional data to identify a functional link between PRC2 alterations and changes in WNT signaling pathway activity in leukemia cells. Computational integration of transcriptomic, proteomic and phosphoproteomic data from an isogenic T-ALL cellular model revealed reduced activity of the WNT-dependent stabilization of proteins (WNT/STOP) pathway in cells lacking core PRC2 factor EZH2. We discovered that PRC2 loss significantly reduced sensitivity to key T-ALL treatment asparaginase, and that this was mechanistically linked to increased cellular ubiquitination levels that bolstered leukemia cell asparagine reserves. We further found that asparaginase resistance in PRC2-depleted leukemic blasts could be mitigated by pharmaceutical proteasome inhibition, thereby providing a novel and clinically tractable means to tackle induction treatment failure in high-risk T-ALL.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.12.553088">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.12.553088" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.12.553088">
        <p class="paperTitle">PARG inhibition induces nuclear aggregation of PARylated PARP1</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.12.553088" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.12.553088" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Paradkar, S.; Purcell, J.; Cui, A.; Friedman, S.; Sundaram, R.; Jensen, R. B.; Bindra, R. S.</p>
        <p class="info">Score: 6.2, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.12.553088' target='https://doi.org/10.1101/2023.08.12.553088'> 10.1101/2023.08.12.553088</a></p>
        <p class="abstract">PARG inhibitors are currently under clinical development for the treatment of DNA repair-deficient cancers, however, their precise mechanism of action is still unclear. Here we report that PARG inhibition causes increased nuclear PARylated PARP1 that limits PARP1 chromatin binding in response to DNA damage. This PARylated PARP1 accumulates as aggregates at sites distinct from the site of DNA damage, leading to the mis-localization of PARP1. Additionally, these aggregates are formed through PAR chains as abrogating PARP1 catalytic activity prevents their formation. Finally, these PARP1 nuclear aggregates persist long-term and are associated with cleaved cytoplasmic PARP1, a cell death hallmark, which ultimately leads to a non-apoptotic form of cell death. Overall, our data uncovers a novel mechanism of PARG inhibitor cytotoxicity, which will inform ongoing clinical studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.13.553123">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.13.553123" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.13.553123">
        <p class="paperTitle">Germline mutation rate predicts cancer mortality across 37 vertebrate species</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.13.553123" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.13.553123" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kapsetaki, S. E.; Compton, Z. T.; Mellon, W.; Vincze, O.; Giraudeau, M. T.; Harrison, T.; Abegglen, L.; Boddy, A.; Maley, C. C.; Schiffman, J.</p>
        <p class="info">Score: 5.9, Published: 2023-08-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.13.553123' target='https://doi.org/10.1101/2023.08.13.553123'> 10.1101/2023.08.13.553123</a></p>
        <p class="abstract">The explanation for why some species are more susceptible to cancer than others remains an area of intense investigation. Cancer evolves in part through the accumulation of mutations and, therefore, we hypothesized that germline mutation rates would be associated with cancer prevalence and mortality across species. We collected previously published data on germline mutation rate and cancer mortality data for 37 vertebrate species. Germline mutation rate was positively correlated with cancer mortality (P = 0.008). Why animals with increased germline mutation rates die more from cancer remains an open question, however they may benefit from close monitoring for tumors due to hereditary cancer predisposition syndromes. Early diagnoses of cancer in these species may increase their chances of treatment and overall survival.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.09.552657">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.09.552657" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.09.552657">
        <p class="paperTitle">Establishing conditions for the generation and maintenance of estrogen receptor-positive organoid models of breast cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.09.552657" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.09.552657" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Oliphant, M. U.; Akshinthala, D.; Muthuswamy, S. K.</p>
        <p class="info">Score: 5.8, Published: 2023-08-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.09.552657' target='https://doi.org/10.1101/2023.08.09.552657'> 10.1101/2023.08.09.552657</a></p>
        <p class="abstract">Patient-derived organoid models of estrogen receptor-positive (ER&#43;) breast cancer would provide a much-needed tool to better understand drug resistance and disease progression. However, the establishment and long-term maintenance of ER expression, function, and response in vitro remains a significant challenge. Here, we report the development of an ER&#43; breast tumor organoid medium (BTOM-ER) that conserves ER expression, estrogen responsiveness, and dependence, as well as sensitivity to endocrine therapy of ER&#43; patient-derived xenograft organoids (PDXO). Our findings demonstrate the utility of subtype-specific culture conditions that better mimic the characteristics of the breast epithelial biology and microenvironment, providing a powerful platform for investigating therapy response and disease progression of ER&#43; breast cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.550630">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.550630" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.550630">
        <p class="paperTitle">A pro-inflammatory stem cell niche drives myelofibrosis through a targetable galectin 1 axis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.550630" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.550630" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, R.; Colombo, M.; Wang, G.; Rodriguez-Romera, A.; O&#39;Sullivan, J.; Clark, S.-A.; Perez Saez, J.; Meng, Y.; Khan, A.; Wen, S.; Dong, P.; Zhou, W.; Sousos, N.; Murphy, L.; Clarke, M.; Jooss, N.; Olijnik, A.-A.; Wong, Z.; Sirinukunwattana, K.; Ryou, H.; Norfo, R.; Cheng, Q.; Brierley, C.; Carrelha, J.; Ren, Z.; Thongjuea, S.; Rathinam, V.; Krishnan, A. K.; Royston, D.; Rabinovich, G.; Mead, A.; Psaila, B.</p>
        <p class="info">Score: 27.0, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.550630' target='https://doi.org/10.1101/2023.08.05.550630'> 10.1101/2023.08.05.550630</a></p>
        <p class="abstract">Myeloproliferative neoplasms are stem cell-driven cancers associated with a large burden of morbidity and mortality. The majority of patients present with early-stage disease, but a substantial proportion progress to myelofibrosis and/or secondary leukemia, advanced cancers with a poor prognosis and high symptom burden. Currently, it remains difficult to predict progression, and we lack therapies that reliably prevent or reverse fibrosis development. A major bottleneck to the discovery of disease-modifying therapies has been an incomplete understanding of the interplay between perturbed cellular and molecular states. Several cell types have individually been implicated, but a comprehensive analysis of myelofibrotic bone marrow is lacking. We therefore mapped the crosstalk between bone marrow cell types in myelofibrotic bone marrow. We found that inflammation and fibrosis are orchestrated by a  quartet of immune and stromal cell lineages - with basophils and mast cells creating a TNF signaling hub, communicating with megakaryocytes, mesenchymal stromal cells and pro-inflammatory fibroblasts. We identified the y-galactoside binding protein galectin 1 as a striking biomarker of progression to myelofibrosis and poor survival in multiple patient cohorts, and as a promising therapeutic target, with reduced myeloproliferation and fibrosis in vitro and in vivo and improved survival following galectin 1 inhibition. In human bone marrow organoids, TNF increased galectin 1 expression, suggesting a feedback loop wherein the pro-inflammatory MPN clone creates a self-reinforcing niche, fueling progression to advanced disease. This study provides a valuable resource for studying hematopoietic cell-niche interactions, with broad relevance for cancer-associated inflammation and disorders of tissue fibrosis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
