<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="CREB activation drives acinar to ductal reprogramming and promote pancreatic cancer progression in animal models of alcoholic chronic pancreatitis
Authors: Srinivasan, S.; Mehra, S.; Bianchi, A.; Singh, S.; Dosch, A. R.; Amirian, H.; Jinka, S.; Krishnamoorthy, V.; Silva, I. D. C.; Box, E. W.; Garrido, V.; Totiger, T. M.; Zhou, Z.; Ban, Y.; Datta, J.; VanSaun, M.; Merchant, N.; Nagathihalli, N.
Score: 11.2, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574376
BACKGROUND AND AIMSIn vivo induction of alcoholic chronic pancreatitis (ACP) causes significant acinar damage, increased fibroinflammatory response, and heightened activation of cyclic response element binding protein 1 (CREB) when compared with alcohol (A) or chronic pancreatitis (CP) mediated pancreatic damage.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="CREB activation drives acinar to ductal reprogramming and promote pancreatic cancer progression in animal models of alcoholic chronic pancreatitis
Authors: Srinivasan, S.; Mehra, S.; Bianchi, A.; Singh, S.; Dosch, A. R.; Amirian, H.; Jinka, S.; Krishnamoorthy, V.; Silva, I. D. C.; Box, E. W.; Garrido, V.; Totiger, T. M.; Zhou, Z.; Ban, Y.; Datta, J.; VanSaun, M.; Merchant, N.; Nagathihalli, N.
Score: 11.2, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574376
BACKGROUND AND AIMSIn vivo induction of alcoholic chronic pancreatitis (ACP) causes significant acinar damage, increased fibroinflammatory response, and heightened activation of cyclic response element binding protein 1 (CREB) when compared with alcohol (A) or chronic pancreatitis (CP) mediated pancreatic damage." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-10T10:40:05+00:00" />
<meta property="article:modified_time" content="2024-01-10T10:40:05+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="CREB activation drives acinar to ductal reprogramming and promote pancreatic cancer progression in animal models of alcoholic chronic pancreatitis
Authors: Srinivasan, S.; Mehra, S.; Bianchi, A.; Singh, S.; Dosch, A. R.; Amirian, H.; Jinka, S.; Krishnamoorthy, V.; Silva, I. D. C.; Box, E. W.; Garrido, V.; Totiger, T. M.; Zhou, Z.; Ban, Y.; Datta, J.; VanSaun, M.; Merchant, N.; Nagathihalli, N.
Score: 11.2, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574376
BACKGROUND AND AIMSIn vivo induction of alcoholic chronic pancreatitis (ACP) causes significant acinar damage, increased fibroinflammatory response, and heightened activation of cyclic response element binding protein 1 (CREB) when compared with alcohol (A) or chronic pancreatitis (CP) mediated pancreatic damage."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "CREB activation drives acinar to ductal reprogramming and promote pancreatic cancer progression in animal models of alcoholic chronic pancreatitis\nAuthors: Srinivasan, S.; Mehra, S.; Bianchi, A.; Singh, S.; Dosch, A. R.; Amirian, H.; Jinka, S.; Krishnamoorthy, V.; Silva, I. D. C.; Box, E. W.; Garrido, V.; Totiger, T. M.; Zhou, Z.; Ban, Y.; Datta, J.; VanSaun, M.; Merchant, N.; Nagathihalli, N.\nScore: 11.2, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574376\nBACKGROUND AND AIMSIn vivo induction of alcoholic chronic pancreatitis (ACP) causes significant acinar damage, increased fibroinflammatory response, and heightened activation of cyclic response element binding protein 1 (CREB) when compared with alcohol (A) or chronic pancreatitis (CP) mediated pancreatic damage.",
  "keywords": [
    
  ],
  "articleBody": " CREB activation drives acinar to ductal reprogramming and promote pancreatic cancer progression in animal models of alcoholic chronic pancreatitis\nAuthors: Srinivasan, S.; Mehra, S.; Bianchi, A.; Singh, S.; Dosch, A. R.; Amirian, H.; Jinka, S.; Krishnamoorthy, V.; Silva, I. D. C.; Box, E. W.; Garrido, V.; Totiger, T. M.; Zhou, Z.; Ban, Y.; Datta, J.; VanSaun, M.; Merchant, N.; Nagathihalli, N.\nScore: 11.2, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574376\nBACKGROUND AND AIMSIn vivo induction of alcoholic chronic pancreatitis (ACP) causes significant acinar damage, increased fibroinflammatory response, and heightened activation of cyclic response element binding protein 1 (CREB) when compared with alcohol (A) or chronic pancreatitis (CP) mediated pancreatic damage. However, the study elucidating the cooperative interaction between CREB and the oncogenic KrasG12D/+(Kras*) in promoting pancreatic cancer progression with ACP remains unexplored. METHODSExperimental ACP induction was established in multiple mouse models, followed by euthanization of the animals at various time intervals during the recovery periods. Tumor latency was determined in these mice cohorts. Here, we established CREB deletion (Crebfl/fl) in Ptf1aCreERTM/+;LSL-KrasG12D+/-(KC) genetic mouse models (KCC-/-). Western blot, phosphokinase array, and qPCR were used to analyze the pancreata of Ptf1aCreERTM+/-, KC and KCC-/- mice. The pancreata of ACP-induced KC mice were subjected to single-cell RNA sequencing (scRNAseq). Further studies involved conducting lineage tracing and acinar cell explant cultures. RESULTSACP induction in KC mice had detrimental effects on the pancreatic damage repair mechanism. The persistent existence of acinar cell-derived ductal lesions demonstrated a prolonged state of hyperactivated CREB. Persistent CREB activation leads to acinar cell reprogramming and increased pro-fibrotic inflammation in KC mice. Acinar-specific Creb ablation reduced advanced PanINs lesions, hindered tumor progression, and restored acinar cell function in ACP-induced mouse models. CONCLUSIONSOur findings demonstrate that CREB cooperates with Kras* to perpetuate an irreversible ADM and PanIN formation. Moreover, CREB sustains oncogenic activity to promote the progression of premalignant lesions toward cancer in the presence of ACP.\nE7-mediated repression of miR-203 promotes LASP1-dependent proliferation in HPV-positive cervical cancer\nAuthors: Patterson, M. R.; Meijers, A. S.; Ryder, E. L.; Scarth, J. A.; Evans, D.; Turner, A. L.; Wasson, C. W.; Daryl, J. E.; James, C. D.; Wang, M.; Ladbury, J. E.; Morgan, I. M.; Samson, A.; Morgan, E. L.; Macdonald, A.\nScore: 5.3, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574687\nHuman papillomaviruses (HPV) are a major cause of malignancy, contributing to [~]5% of all human cancers worldwide, including most cervical cancer cases and a growing number of ano-genital and oral cancers. The major HPV viral oncogenes, E6 and E7, manipulate many host cellular pathways that promote cell proliferation and survival, predisposing infected cells to malignant transformation. Despite the availability of highly effective vaccines, there are still no specific anti-viral therapies targeting HPV or treatments for HPV-associated cancers. As such, a better understanding of viral-host interactions may allow the identification of novel therapeutic targets. Here, we demonstrate that the actin-binding protein LASP1 is upregulated in cervical cancer and significantly correlates with a poorer overall survival. In HPV positive cervical cancer, LASP1 depletion significantly inhibited proliferation in vitro, whilst having minimal effects in HPV negative cervical cancer cells. Furthermore, we show that the LASP1 SH3 domain is essential for LASP1-mediated proliferation in these cells. Mechanistically, we show that HPV E7 regulates LASP1 at the post-transcriptional level by repressing the expression of miR-203, which negatively regulated LASP1 mRNA levels by binding to its 3UTR. Finally, we demonstrated that LASP1 expression is required for the growth of HPV positive cervical cancer cells in an in vivo tumourigenicity model. Together, these data demonstrate that HPV induces LASP1 expression to promote proliferation and survival role in cervical cancer, thus identifying a potential therapeutic target in these cancers.\nBranched-chain keto acids promote an immune-suppressive and neurodegenerative microenvironment in leptomeningeal disease\nAuthors: Khaled, M. L.; Ren, Y.; Kundalia, R.; Alhaddad, H.; Chen, Z.; Wallace, G. C.; Evernden, B.; Ospina, O. E.; Hall, M.; Liu, M.; Darville, L. N. F.; Izumi, V.; Chen, Y. A.; Pilon-Thomas, S.; Stewart, P. A.; Koomen, J. M.; Corallo, S. A.; Jain, M. D.; Robinson, T. J.; Locke, F. L.; Forsyth, P. A.; Smalley, I.\nScore: 6.4, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572239\nLeptomeningeal disease (LMD) occurs when tumors seed into the leptomeningeal space and cerebrospinal fluid (CSF), leading to severe neurological deterioration and poor survival outcomes. We utilized comprehensive multi-omics analyses of CSF from patients with lymphoma LMD to demonstrate an immunosuppressive cellular microenvironment and identified dysregulations in proteins and lipids indicating neurodegenerative processes. Strikingly, we found a significant accumulation of toxic branched-chain keto acids (BCKA) in the CSF of patients with LMD. The BCKA accumulation was found to be a pan-cancer occurrence, evident in lymphoma, breast cancer, and melanoma LMD patients. Functionally, BCKA disrupted the viability and function of endogenous T lymphocytes, chimeric antigen receptor (CAR) T cells, neurons, and meningeal cells. Treatment of LMD mice with BCKA-reducing sodium phenylbutyrate significantly improved neurological function, survival outcomes, and efficacy of anti-CD19 CAR T cell therapy. This is the first report of BCKA accumulation in LMD and provides preclinical evidence that targeting these toxic metabolites improves outcomes.\nPharmacogenomic discovery of genetically targeted cancer therapies optimized against clinical outcomes\nAuthors: Truesdell, P.; Chang, J.; Coto Villa, D.; Zhao, Y.; McIlwain, R.; Young, S.; Hiley, S.; Craig, A. W.; Babak, T.\nScore: 4.5, Published: 2024-01-07 DOI: 10.1101/2024.01.05.574245\nDespite the clinical success of dozens of genetically targeted cancer therapies, the vast majority of patients with tumors caused by loss-of-function (LoF) mutations do not have access to these treatments. This is primarily due to the challenge of developing a drug that treats a disease caused by the absence of a protein target. The success of PARP inhibitors has solidified synthetic lethality (SL) as a means to overcome this obstacle. Recent mapping of SL networks using pooled CRISPR-Cas9 screens is a promising approach for expanding this concept to treating cancers driven by additional LoF drivers. In practice, however, translating signals from cell lines, where these screens are typically conducted, to patient outcomes remains a challenge. We developed a pharmacogemic (PGx) approach called \"Clinically Optimized Driver Associated PGx\" (CODA-PGx) that accurately predicts genetically targeted therapies with clinical-stage efficacy in specific LoF driver contexts. Using approved targeted therapies and cancer drugs with available real-world evidence and molecular data from hundreds of patients, we discovered and optimized the key screening principles predictive of efficacy and overall patient survival. In addition to establishing basic technical conventions, such as drug concentration and screening kinetics, we found that replicating the driver perturbation in the right context, as well as selecting patients where those drivers are genuine founder mutations, were key to accurate translation. We used CODA-PGX to screen a diverse collection of clinical stage drugs and report dozens of novel LoF genetically targeted opportunities; many validated in xenografts and by real-world evidence. Notable examples include treating STAG2-mutant tumors with Carboplatin, SMARCB1-mutant tumors with Oxaliplatin, and TP53BP1-mutant tumors with Etoposide or Bleomycin. One Sentence SummaryWe identified principles of pharmacogenomic screening that predict clinical efficacy in cancer patients with specific driver mutations.\nSpatial transcriptomics analysis identifies a unique tumor-promoting function of the meningeal stroma in melanoma leptomeningeal disease\nAuthors: Alhaddad, H.; Ospina, O. E.; Khaled, M. L.; Ren, Y.; Forsyth, P. A.; Pina, Y.; Macaulay, R.; Law, V.; Tsai, K. Y.; Cress, W. D.; Fridley, B.; Smalley, I.\nScore: 6.1, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572266\nLeptomeningeal disease (LMD) remains a rapidly lethal complication for late-stage melanoma patients. The inaccessible nature of the disease site and lack of understanding of the biology of this unique metastatic site are major barriers to developing efficacious therapies for patients with melanoma LMD. Here, we characterize the tumor microenvironment of the leptomeningeal tissues and patient-matched extra-cranial metastatic sites using spatial transcriptomic analyses with in vitro and in vivo validation. We show the spatial landscape of melanoma LMD to be characterized by a lack of immune infiltration and instead exhibit a higher level of stromal involvement. We show that the tumor-stroma interactions at the leptomeninges activate pathways implicated in tumor-promoting signaling, mediated through upregulation of SERPINA3 at the tumor-stroma interface. Our functional experiments establish that the meningeal stroma is required for melanoma cells to survive in the CSF environment and that these interactions lead to a lack of MAPK inhibitor sensitivity in the tumor. We show that knocking down SERPINA3 or inhibiting the downstream IGR1R/PI3K/AKT axis results in re-sensitization of the tumor to MAPK-targeting therapy and tumor cell death in the leptomeningeal environment. Our data provides a spatial atlas of melanoma LMD, identifies the tumor-promoting role of meningeal stroma, and demonstrates a mechanism for overcoming microenvironment-mediated drug resistance unique to this metastatic site.\nMechanisms underpinning osteosarcoma genome complexity and evolution\nAuthors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.\nScore: 29.4, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573403\nOsteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood. Using multi-region whole-genome sequencing, we find that chromothripsis is an ongoing mutational process, occurring subclonally in 74% of tumours. Chromothripsis drives the acquisition of oncogenic mutations and generates highly unstable derivative chromosomes, the evolution of which drives clonal diversification and intra-tumour heterogeneity. In addition, we report a novel mechanism, loss-translocation-amplification (LTA) chromothripsis, which mediates rapid malignant transformation and punctuated evolution in about half of paediatric and adult high-grade osteosarcomas. Specifically, a single double-strand break triggers concomitant TP53 inactivation and segmental amplifications, often amplifying oncogenes to high copy numbers in extrachromosomal circular DNA elements through breakage-fusion-bridge cycles involving multiple chromosomes. LTA chromothripsis is detected at low frequency in soft-tissue sarcomas, but not in epithelial cancers, including those driven by TP53 mutation. Finally, we identify genome-wide loss of heterozygosity as a strong prognostic indicator for high-grade osteosarcoma.\nHybrid cancer stem cells utilise vascular tracks for collective streaming invasion in a metastasis-on-a-chip device\nAuthors: Scemama, A.; Lunetto, S.; Tailor, A.; Di Cio, S.; Ambler, L.; Coetzee, A.; Cottom, H.; Khurram, S. A.; Gautrot, J.; Biddle, A.\nScore: 4.2, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573897\nCancer stem cells (CSCs) drive cancer metastatic dissemination. They do not do so in a vacuum, and the important influence of the tumour microenvironment (TME) on metastatic dissemination is becoming increasingly recognised. Therapeutic targeting of CSC-TME interactions may be a promising route to suppression of tumour metastasis. However, we must first understand how interactions with the TME influence CSC metastatic dissemination. To achieve this understanding, there is a need for experimental models that enable the analysis of dynamic interactions at single cell resolution within a complex environment. To this end, we utilise a metastasis-on-a-chip device to produce a 3D in vitro model of CSC interaction with a developing microvasculature, that is amenable to precise imaging and real time studies at single cell resolution. We show that the invasive phenotype of oral squamous cell carcinoma (OSCC) cells is markedly altered when in proximity to a microvasculature, with a switch to a hybrid CSC phenotype that undergoes collective streaming invasion. Mechanistically, ECM compression by the developing vasculature creates an environment that is refractory to cancer invasion, whilst leaving abandoned vascular tracks that can be utilised by hybrid CSCs for collective streaming invasion. Human tissue studies identify streaming invasion in association with vascularised regions in OSCC specimens. These findings elucidate the influence of the vasculature on CSC metastatic dissemination in OSCC, and the role of hybrid CSC invasion plasticity in overcoming this TME constraint.\nSingle-cell RNA-seq analysis profiling characterizes differences in cell composition and physiology between normal tissue, treatment-naive, and cisplatin-treated ovarian cancer\nAuthors: Guo, F.; Yang, Z.; Sehouli, J.; Kaufmann, A.\nScore: 3.5, Published: 2024-01-03 DOI: 10.1101/2024.01.02.573967\nBackgroundIntense efforts have focused on identifying heterogeneity of the cellular composition in ovarian cancer. However, tissue composition and physiological conditions of cancer cells in cisplatin-sensitive ovarian cancer remains largely unknown. Moreover, comparisons of different cellular states in normal tissue, in treatment naive ovarian cancer, and in cisplatin-treated tissue after adjuvant therapy of cisplatin-sensitive ovarian cancer at the single-cell level might offer clues for ovarian cancer treatment and prevention of cisplatin-resistance formation. MethodsSingle-cell transcriptome sequencing of a cisplatin-treated ovarian cancer was performed. Data sets of non-tumorous ovarian tissues and treatment-naive ovarian cancer were downloaded from the European Genome-phenome Archive (accession number EGAS00001004987). Quality control, batch effect correction, integration, and clustering analysis of the integrated single-cell transcriptome data was done. Cell subsets were annotated based on surface marker phenotypes. Finally, the proportions of subclusters, the immune cell population, and the potential biological processes among different cellular states were compared. ResultsSixteen distinct cell subsets were identified from the integrated single-cell transcriptome sequencing data of a pool of all tissues. The composition of the three different tissue types was characterized. The proportion of fibroblasts in cisplatin-treated ovarian tumor was remarkably lower than in treatment-naive ovarian tumor (1.33% vs. 13.53%, p \u003c 0.05). Moreover, each subjects sample had differing relative proportions of the identified cell types. In primary untreated ovarian cancer, the prevalent immune cells were B cells and myeloid-related immunosuppressive M2 macrophages. However, there were less B cells and myeloid-related immunosuppressive M2 macrophages after cisplatin-treatment, while significantly more T cells were found. The physiological cellular state in primary untreated ovarian tumors was associated with dysfunctional gene expression and modulation of cellular homeostasis, while cells from cisplatin-treated tumor showed more activation of immune and inflammatory genes as compared to healthy ovarian tissue. ConclusionOur molecular gene expression analysis allowed for the separation and identification of differences in normal ovarian tissues, treatment-naive, and cisplatin-sensitive ovarian cancer cell populations at single-cell resolution. We identified different cell type composition and discriminative marker expression concerning specific cell subsets and identified differences among their physiological cell states. This knowledge may open new possibilities for elucidating important pathogenetic features and therapeutic strategies for treating ovarian cancer.\nReprogramming neuroblastoma by diet-enhanced polyamine depletion\nAuthors: Cherkaoui, S.; Yang, L.; McBride, M. J.; Turn, C.; Lu, W.; Eigenmann, C.; Allen, G. E.; Panasenko, O. O.; Zhang, L.; Vu, A.; Liu, K.; Li, Y.; Gandhi, O. H.; Surrey, L.; Wierer, M.; White, E.; Rabinowitz, J. D.; Hogarty, M. D.; Morscher, R. J.\nScore: 2.8, Published: 2024-01-08 DOI: 10.1101/2024.01.07.573662\nNeuroblastoma is a highly lethal childhood tumor derived from differentiation-arrested neural crest cells1,2. Like all cancers, its growth is fueled by metabolites obtained from either circulation or local biosynthesis3,4. Neuroblastomas depend on local polyamine biosynthesis, with the inhibitor difluoromethylornithine showing clinical activity5. Here we show that such inhibition can be augmented by dietary restriction of upstream amino acid substrates, leading to disruption of oncogenic protein translation, tumor differentiation, and profound survival gains in the TH-MYCN mouse model. Specifically, an arginine/proline-free diet decreases the polyamine precursor ornithine and augments tumor polyamine depletion by difluoromethylornithine. This polyamine depletion causes ribosome stalling, unexpectedly specifically at adenosine-ending codons. Such codons are selectively enriched in cell cycle genes and low in neuronal differentiation genes. Thus, impaired translation of these codons, induced by the diet-drug combination, favors a pro-differentiation proteome. These results suggest that the genes of specific cellular programs have evolved hallmark codon usage preferences that enable coherent translational rewiring in response to metabolic stresses, and that this process can be targeted to activate differentiation of pediatric cancers. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=192 SRC=\"FIGDIR/small/573662v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (64K): org.highwire.dtl.DTLVardef@c3b4c5org.highwire.dtl.DTLVardef@91242borg.highwire.dtl.DTLVardef@dd92baorg.highwire.dtl.DTLVardef@cc17bc_HPS_FORMAT_FIGEXP M_FIG C_FIG Highlights- Extra-tumoral conversion of arginine feeds tumor ornithine via uptake from circulation in MYCN-neuroblastoma. - A proline and arginine free diet enhances pharmacological polyamine depletion via reduced ornithine substrate availability. - Polyamine depletion disrupts oncogenic translation to induce a pro-differentiation proteome causing neuroblast differentiation and prolonged survival in the TH-MYCN mouse model. - Genes of specific cellular programs have evolved codon usage preferences that enable coherent translational rewiring in response to metabolic stress, such as polyamine depletion.\nError-corrected deep targeted sequencing of circulating cell-free DNA from colorectal cancer patients for sensitive detection of circulating tumor DNA\nAuthors: Frydendahl, A.; Rasmussen, M. H.; Jensen, S. O.; Henriksen, T. V.; Demuth, C.; Diekema, M.; Dietzel, H. J.; Wen, S. W. C.; Pedersen, J. S.; Dyrskjoet, L.; Andersen, C. L.\nScore: 3.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573042\nIntroductionCirculating tumor DNA (ctDNA) is a promising biomarker, reflecting the presence of tumor cells. Sequencing-based detection of ctDNA at low tumor fractions is challenging due to the crude error rate of sequencing. To mitigate this challenge, we developed UMIseq, a fixed-panel deep-targeted sequencing approach, which is universally applicable to all colorectal cancer (CRC) patients. MethodsUMIseq features UMI-mediated error correction, exclusion of mutations related to clonal hematopoiesis, a panel of normals for error modeling, and signal integration from single-nucleotide variations, insertions, deletions, and phased mutations. UMIseq was trained and independently validated on pre-operative (pre-OP) plasma from CRC patients (n=364) and healthy individuals (n=61). ResultsUMIseq displayed an area under the curve surpassing 0.95 for allele frequencies (AF) down to 0.05%. In the training cohort, the pre-OP detection rate reached 80% at 95% specificity, while in the validation cohort, it was 70%. UMIseq enabled the detection of AFs down to 0.004%. To assess the potential for detection of residual disease, 26 post-operative plasma samples from stage III CRC patients were analyzed. Detection of ctDNA was associated with recurrence (p =0.08). ConclusionUMIseq demonstrated robust performance with high sensitivity and specificity, enabling the detection of ctDNA at low allele frequencies.\n",
  "wordCount" : "2865",
  "inLanguage": "en",
  "datePublished": "2024-01-10T10:40:05Z",
  "dateModified": "2024-01-10T10:40:05Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 10, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574376" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574376">
        <p class="paperTitle">CREB activation drives acinar to ductal reprogramming and promote pancreatic cancer progression in animal models of alcoholic chronic pancreatitis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Srinivasan, S.; Mehra, S.; Bianchi, A.; Singh, S.; Dosch, A. R.; Amirian, H.; Jinka, S.; Krishnamoorthy, V.; Silva, I. D. C.; Box, E. W.; Garrido, V.; Totiger, T. M.; Zhou, Z.; Ban, Y.; Datta, J.; VanSaun, M.; Merchant, N.; Nagathihalli, N.</p>
        <p class="info">Score: 11.2, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574376' target='https://doi.org/10.1101/2024.01.05.574376'> 10.1101/2024.01.05.574376</a></p>
        <p class="abstract">BACKGROUND AND AIMSIn vivo induction of alcoholic chronic pancreatitis (ACP) causes significant acinar damage, increased fibroinflammatory response, and heightened activation of cyclic response element binding protein 1 (CREB) when compared with alcohol (A) or chronic pancreatitis (CP) mediated pancreatic damage. However, the study elucidating the cooperative interaction between CREB and the oncogenic KrasG12D/&#43;(Kras*) in promoting pancreatic cancer progression with ACP remains unexplored.

METHODSExperimental ACP induction was established in multiple mouse models, followed by euthanization of the animals at various time intervals during the recovery periods. Tumor latency was determined in these mice cohorts. Here, we established CREB deletion (Crebfl/fl) in Ptf1aCreERTM/&#43;;LSL-KrasG12D&#43;/-(KC) genetic mouse models (KCC-/-). Western blot, phosphokinase array, and qPCR were used to analyze the pancreata of Ptf1aCreERTM&#43;/-, KC and KCC-/- mice. The pancreata of ACP-induced KC mice were subjected to single-cell RNA sequencing (scRNAseq). Further studies involved conducting lineage tracing and acinar cell explant cultures.

RESULTSACP induction in KC mice had detrimental effects on the pancreatic damage repair mechanism. The persistent existence of acinar cell-derived ductal lesions demonstrated a prolonged state of hyperactivated CREB. Persistent CREB activation leads to acinar cell reprogramming and increased pro-fibrotic inflammation in KC mice. Acinar-specific Creb ablation reduced advanced PanINs lesions, hindered tumor progression, and restored acinar cell function in ACP-induced mouse models.

CONCLUSIONSOur findings demonstrate that CREB cooperates with Kras* to perpetuate an irreversible ADM and PanIN formation. Moreover, CREB sustains oncogenic activity to promote the progression of premalignant lesions toward cancer in the presence of ACP.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574687">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574687" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574687">
        <p class="paperTitle">E7-mediated repression of miR-203 promotes LASP1-dependent proliferation in HPV-positive cervical cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574687" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574687" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Patterson, M. R.; Meijers, A. S.; Ryder, E. L.; Scarth, J. A.; Evans, D.; Turner, A. L.; Wasson, C. W.; Daryl, J. E.; James, C. D.; Wang, M.; Ladbury, J. E.; Morgan, I. M.; Samson, A.; Morgan, E. L.; Macdonald, A.</p>
        <p class="info">Score: 5.3, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574687' target='https://doi.org/10.1101/2024.01.08.574687'> 10.1101/2024.01.08.574687</a></p>
        <p class="abstract">Human papillomaviruses (HPV) are a major cause of malignancy, contributing to [~]5% of all human cancers worldwide, including most cervical cancer cases and a growing number of ano-genital and oral cancers. The major HPV viral oncogenes, E6 and E7, manipulate many host cellular pathways that promote cell proliferation and survival, predisposing infected cells to malignant transformation. Despite the availability of highly effective vaccines, there are still no specific anti-viral therapies targeting HPV or treatments for HPV-associated cancers. As such, a better understanding of viral-host interactions may allow the identification of novel therapeutic targets. Here, we demonstrate that the actin-binding protein LASP1 is upregulated in cervical cancer and significantly correlates with a poorer overall survival. In HPV positive cervical cancer, LASP1 depletion significantly inhibited proliferation in vitro, whilst having minimal effects in HPV negative cervical cancer cells. Furthermore, we show that the LASP1 SH3 domain is essential for LASP1-mediated proliferation in these cells. Mechanistically, we show that HPV E7 regulates LASP1 at the post-transcriptional level by repressing the expression of miR-203, which negatively regulated LASP1 mRNA levels by binding to its 3UTR. Finally, we demonstrated that LASP1 expression is required for the growth of HPV positive cervical cancer cells in an in vivo tumourigenicity model. Together, these data demonstrate that HPV induces LASP1 expression to promote proliferation and survival role in cervical cancer, thus identifying a potential therapeutic target in these cancers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572239">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572239" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572239">
        <p class="paperTitle">Branched-chain keto acids promote an immune-suppressive and neurodegenerative microenvironment in leptomeningeal disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572239" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572239" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khaled, M. L.; Ren, Y.; Kundalia, R.; Alhaddad, H.; Chen, Z.; Wallace, G. C.; Evernden, B.; Ospina, O. E.; Hall, M.; Liu, M.; Darville, L. N. F.; Izumi, V.; Chen, Y. A.; Pilon-Thomas, S.; Stewart, P. A.; Koomen, J. M.; Corallo, S. A.; Jain, M. D.; Robinson, T. J.; Locke, F. L.; Forsyth, P. A.; Smalley, I.</p>
        <p class="info">Score: 6.4, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572239' target='https://doi.org/10.1101/2023.12.18.572239'> 10.1101/2023.12.18.572239</a></p>
        <p class="abstract">Leptomeningeal disease (LMD) occurs when tumors seed into the leptomeningeal space and cerebrospinal fluid (CSF), leading to severe neurological deterioration and poor survival outcomes. We utilized comprehensive multi-omics analyses of CSF from patients with lymphoma LMD to demonstrate an immunosuppressive cellular microenvironment and identified dysregulations in proteins and lipids indicating neurodegenerative processes. Strikingly, we found a significant accumulation of toxic branched-chain keto acids (BCKA) in the CSF of patients with LMD. The BCKA accumulation was found to be a pan-cancer occurrence, evident in lymphoma, breast cancer, and melanoma LMD patients. Functionally, BCKA disrupted the viability and function of endogenous T lymphocytes, chimeric antigen receptor (CAR) T cells, neurons, and meningeal cells. Treatment of LMD mice with BCKA-reducing sodium phenylbutyrate significantly improved neurological function, survival outcomes, and efficacy of anti-CD19 CAR T cell therapy. This is the first report of BCKA accumulation in LMD and provides preclinical evidence that targeting these toxic metabolites improves outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574245">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574245" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574245">
        <p class="paperTitle">Pharmacogenomic discovery of genetically targeted cancer therapies optimized against clinical outcomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574245" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574245" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Truesdell, P.; Chang, J.; Coto Villa, D.; Zhao, Y.; McIlwain, R.; Young, S.; Hiley, S.; Craig, A. W.; Babak, T.</p>
        <p class="info">Score: 4.5, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574245' target='https://doi.org/10.1101/2024.01.05.574245'> 10.1101/2024.01.05.574245</a></p>
        <p class="abstract">Despite the clinical success of dozens of genetically targeted cancer therapies, the vast majority of patients with tumors caused by loss-of-function (LoF) mutations do not have access to these treatments. This is primarily due to the challenge of developing a drug that treats a disease caused by the absence of a protein target. The success of PARP inhibitors has solidified synthetic lethality (SL) as a means to overcome this obstacle. Recent mapping of SL networks using pooled CRISPR-Cas9 screens is a promising approach for expanding this concept to treating cancers driven by additional LoF drivers. In practice, however, translating signals from cell lines, where these screens are typically conducted, to patient outcomes remains a challenge. We developed a pharmacogemic (PGx) approach called &#34;Clinically Optimized Driver Associated PGx&#34; (CODA-PGx) that accurately predicts genetically targeted therapies with clinical-stage efficacy in specific LoF driver contexts. Using approved targeted therapies and cancer drugs with available real-world evidence and molecular data from hundreds of patients, we discovered and optimized the key screening principles predictive of efficacy and overall patient survival. In addition to establishing basic technical conventions, such as drug concentration and screening kinetics, we found that replicating the driver perturbation in the right context, as well as selecting patients where those drivers are genuine founder mutations, were key to accurate translation. We used CODA-PGX to screen a diverse collection of clinical stage drugs and report dozens of novel LoF genetically targeted opportunities; many validated in xenografts and by real-world evidence. Notable examples include treating STAG2-mutant tumors with Carboplatin, SMARCB1-mutant tumors with Oxaliplatin, and TP53BP1-mutant tumors with Etoposide or Bleomycin.

One Sentence SummaryWe identified principles of pharmacogenomic screening that predict clinical efficacy in cancer patients with specific driver mutations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572266">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572266" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572266">
        <p class="paperTitle">Spatial transcriptomics analysis identifies a unique tumor-promoting function of the meningeal stroma in melanoma leptomeningeal disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572266" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572266" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alhaddad, H.; Ospina, O. E.; Khaled, M. L.; Ren, Y.; Forsyth, P. A.; Pina, Y.; Macaulay, R.; Law, V.; Tsai, K. Y.; Cress, W. D.; Fridley, B.; Smalley, I.</p>
        <p class="info">Score: 6.1, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572266' target='https://doi.org/10.1101/2023.12.18.572266'> 10.1101/2023.12.18.572266</a></p>
        <p class="abstract">Leptomeningeal disease (LMD) remains a rapidly lethal complication for late-stage melanoma patients. The inaccessible nature of the disease site and lack of understanding of the biology of this unique metastatic site are major barriers to developing efficacious therapies for patients with melanoma LMD. Here, we characterize the tumor microenvironment of the leptomeningeal tissues and patient-matched extra-cranial metastatic sites using spatial transcriptomic analyses with in vitro and in vivo validation. We show the spatial landscape of melanoma LMD to be characterized by a lack of immune infiltration and instead exhibit a higher level of stromal involvement. We show that the tumor-stroma interactions at the leptomeninges activate pathways implicated in tumor-promoting signaling, mediated through upregulation of SERPINA3 at the tumor-stroma interface. Our functional experiments establish that the meningeal stroma is required for melanoma cells to survive in the CSF environment and that these interactions lead to a lack of MAPK inhibitor sensitivity in the tumor. We show that knocking down SERPINA3 or inhibiting the downstream IGR1R/PI3K/AKT axis results in re-sensitization of the tumor to MAPK-targeting therapy and tumor cell death in the leptomeningeal environment. Our data provides a spatial atlas of melanoma LMD, identifies the tumor-promoting role of meningeal stroma, and demonstrates a mechanism for overcoming microenvironment-mediated drug resistance unique to this metastatic site.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573403" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573403">
        <p class="paperTitle">Mechanisms underpinning osteosarcoma genome complexity and evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.</p>
        <p class="info">Score: 29.4, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573403' target='https://doi.org/10.1101/2023.12.29.573403'> 10.1101/2023.12.29.573403</a></p>
        <p class="abstract">Osteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood. Using multi-region whole-genome sequencing, we find that chromothripsis is an ongoing mutational process, occurring subclonally in 74% of tumours. Chromothripsis drives the acquisition of oncogenic mutations and generates highly unstable derivative chromosomes, the evolution of which drives clonal diversification and intra-tumour heterogeneity. In addition, we report a novel mechanism, loss-translocation-amplification (LTA) chromothripsis, which mediates rapid malignant transformation and punctuated evolution in about half of paediatric and adult high-grade osteosarcomas. Specifically, a single double-strand break triggers concomitant TP53 inactivation and segmental amplifications, often amplifying oncogenes to high copy numbers in extrachromosomal circular DNA elements through breakage-fusion-bridge cycles involving multiple chromosomes. LTA chromothripsis is detected at low frequency in soft-tissue sarcomas, but not in epithelial cancers, including those driven by TP53 mutation. Finally, we identify genome-wide loss of heterozygosity as a strong prognostic indicator for high-grade osteosarcoma.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573897">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573897" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573897">
        <p class="paperTitle">Hybrid cancer stem cells utilise vascular tracks for collective streaming invasion in a metastasis-on-a-chip device</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573897" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573897" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scemama, A.; Lunetto, S.; Tailor, A.; Di Cio, S.; Ambler, L.; Coetzee, A.; Cottom, H.; Khurram, S. A.; Gautrot, J.; Biddle, A.</p>
        <p class="info">Score: 4.2, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573897' target='https://doi.org/10.1101/2024.01.02.573897'> 10.1101/2024.01.02.573897</a></p>
        <p class="abstract">Cancer stem cells (CSCs) drive cancer metastatic dissemination. They do not do so in a vacuum, and the important influence of the tumour microenvironment (TME) on metastatic dissemination is becoming increasingly recognised. Therapeutic targeting of CSC-TME interactions may be a promising route to suppression of tumour metastasis. However, we must first understand how interactions with the TME influence CSC metastatic dissemination. To achieve this understanding, there is a need for experimental models that enable the analysis of dynamic interactions at single cell resolution within a complex environment. To this end, we utilise a metastasis-on-a-chip device to produce a 3D in vitro model of CSC interaction with a developing microvasculature, that is amenable to precise imaging and real time studies at single cell resolution. We show that the invasive phenotype of oral squamous cell carcinoma (OSCC) cells is markedly altered when in proximity to a microvasculature, with a switch to a hybrid CSC phenotype that undergoes collective streaming invasion. Mechanistically, ECM compression by the developing vasculature creates an environment that is refractory to cancer invasion, whilst leaving abandoned vascular tracks that can be utilised by hybrid CSCs for collective streaming invasion. Human tissue studies identify streaming invasion in association with vascularised regions in OSCC specimens. These findings elucidate the influence of the vasculature on CSC metastatic dissemination in OSCC, and the role of hybrid CSC invasion plasticity in overcoming this TME constraint.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573967">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573967" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573967">
        <p class="paperTitle">Single-cell RNA-seq analysis profiling characterizes differences in cell composition and physiology between normal tissue, treatment-naive, and cisplatin-treated ovarian cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573967" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573967" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, F.; Yang, Z.; Sehouli, J.; Kaufmann, A.</p>
        <p class="info">Score: 3.5, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573967' target='https://doi.org/10.1101/2024.01.02.573967'> 10.1101/2024.01.02.573967</a></p>
        <p class="abstract">BackgroundIntense efforts have focused on identifying heterogeneity of the cellular composition in ovarian cancer. However, tissue composition and physiological conditions of cancer cells in cisplatin-sensitive ovarian cancer remains largely unknown. Moreover, comparisons of different cellular states in normal tissue, in treatment naive ovarian cancer, and in cisplatin-treated tissue after adjuvant therapy of cisplatin-sensitive ovarian cancer at the single-cell level might offer clues for ovarian cancer treatment and prevention of cisplatin-resistance formation.

MethodsSingle-cell transcriptome sequencing of a cisplatin-treated ovarian cancer was performed. Data sets of non-tumorous ovarian tissues and treatment-naive ovarian cancer were downloaded from the European Genome-phenome Archive (accession number EGAS00001004987). Quality control, batch effect correction, integration, and clustering analysis of the integrated single-cell transcriptome data was done. Cell subsets were annotated based on surface marker phenotypes. Finally, the proportions of subclusters, the immune cell population, and the potential biological processes among different cellular states were compared.

ResultsSixteen distinct cell subsets were identified from the integrated single-cell transcriptome sequencing data of a pool of all tissues. The composition of the three different tissue types was characterized. The proportion of fibroblasts in cisplatin-treated ovarian tumor was remarkably lower than in treatment-naive ovarian tumor (1.33% vs. 13.53%, p &lt; 0.05). Moreover, each subjects sample had differing relative proportions of the identified cell types. In primary untreated ovarian cancer, the prevalent immune cells were B cells and myeloid-related immunosuppressive M2 macrophages. However, there were less B cells and myeloid-related immunosuppressive M2 macrophages after cisplatin-treatment, while significantly more T cells were found. The physiological cellular state in primary untreated ovarian tumors was associated with dysfunctional gene expression and modulation of cellular homeostasis, while cells from cisplatin-treated tumor showed more activation of immune and inflammatory genes as compared to healthy ovarian tissue.

ConclusionOur molecular gene expression analysis allowed for the separation and identification of differences in normal ovarian tissues, treatment-naive, and cisplatin-sensitive ovarian cancer cell populations at single-cell resolution. We identified different cell type composition and discriminative marker expression concerning specific cell subsets and identified differences among their physiological cell states. This knowledge may open new possibilities for elucidating important pathogenetic features and therapeutic strategies for treating ovarian cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.573662">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.573662" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.573662">
        <p class="paperTitle">Reprogramming neuroblastoma by diet-enhanced polyamine depletion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.573662" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.573662" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cherkaoui, S.; Yang, L.; McBride, M. J.; Turn, C.; Lu, W.; Eigenmann, C.; Allen, G. E.; Panasenko, O. O.; Zhang, L.; Vu, A.; Liu, K.; Li, Y.; Gandhi, O. H.; Surrey, L.; Wierer, M.; White, E.; Rabinowitz, J. D.; Hogarty, M. D.; Morscher, R. J.</p>
        <p class="info">Score: 2.8, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.573662' target='https://doi.org/10.1101/2024.01.07.573662'> 10.1101/2024.01.07.573662</a></p>
        <p class="abstract">Neuroblastoma is a highly lethal childhood tumor derived from differentiation-arrested neural crest cells1,2. Like all cancers, its growth is fueled by metabolites obtained from either circulation or local biosynthesis3,4. Neuroblastomas depend on local polyamine biosynthesis, with the inhibitor difluoromethylornithine showing clinical activity5. Here we show that such inhibition can be augmented by dietary restriction of upstream amino acid substrates, leading to disruption of oncogenic protein translation, tumor differentiation, and profound survival gains in the TH-MYCN mouse model. Specifically, an arginine/proline-free diet decreases the polyamine precursor ornithine and augments tumor polyamine depletion by difluoromethylornithine. This polyamine depletion causes ribosome stalling, unexpectedly specifically at adenosine-ending codons. Such codons are selectively enriched in cell cycle genes and low in neuronal differentiation genes. Thus, impaired translation of these codons, induced by the diet-drug combination, favors a pro-differentiation proteome. These results suggest that the genes of specific cellular programs have evolved hallmark codon usage preferences that enable coherent translational rewiring in response to metabolic stresses, and that this process can be targeted to activate differentiation of pediatric cancers.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=192 SRC=&#34;FIGDIR/small/573662v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (64K):
org.highwire.dtl.DTLVardef@c3b4c5org.highwire.dtl.DTLVardef@91242borg.highwire.dtl.DTLVardef@dd92baorg.highwire.dtl.DTLVardef@cc17bc_HPS_FORMAT_FIGEXP  M_FIG C_FIG Highlights- Extra-tumoral conversion of arginine feeds tumor ornithine via uptake from circulation in MYCN-neuroblastoma.
- A proline and arginine free diet enhances pharmacological polyamine depletion via reduced ornithine substrate availability.
- Polyamine depletion disrupts oncogenic translation to induce a pro-differentiation proteome causing neuroblast differentiation and prolonged survival in the TH-MYCN mouse model.
- Genes of specific cellular programs have evolved codon usage preferences that enable coherent translational rewiring in response to metabolic stress, such as polyamine depletion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573042">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573042" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573042">
        <p class="paperTitle">Error-corrected deep targeted sequencing of circulating cell-free DNA from colorectal cancer patients for sensitive detection of circulating tumor DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573042" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573042" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frydendahl, A.; Rasmussen, M. H.; Jensen, S. O.; Henriksen, T. V.; Demuth, C.; Diekema, M.; Dietzel, H. J.; Wen, S. W. C.; Pedersen, J. S.; Dyrskjoet, L.; Andersen, C. L.</p>
        <p class="info">Score: 3.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573042' target='https://doi.org/10.1101/2023.12.22.573042'> 10.1101/2023.12.22.573042</a></p>
        <p class="abstract">IntroductionCirculating tumor DNA (ctDNA) is a promising biomarker, reflecting the presence of tumor cells. Sequencing-based detection of ctDNA at low tumor fractions is challenging due to the crude error rate of sequencing. To mitigate this challenge, we developed UMIseq, a fixed-panel deep-targeted sequencing approach, which is universally applicable to all colorectal cancer (CRC) patients.

MethodsUMIseq features UMI-mediated error correction, exclusion of mutations related to clonal hematopoiesis, a panel of normals for error modeling, and signal integration from single-nucleotide variations, insertions, deletions, and phased mutations. UMIseq was trained and independently validated on pre-operative (pre-OP) plasma from CRC patients (n=364) and healthy individuals (n=61).

ResultsUMIseq displayed an area under the curve surpassing 0.95 for allele frequencies (AF) down to 0.05%. In the training cohort, the pre-OP detection rate reached 80% at 95% specificity, while in the validation cohort, it was 70%. UMIseq enabled the detection of AFs down to 0.004%. To assess the potential for detection of residual disease, 26 post-operative plasma samples from stage III CRC patients were analyzed. Detection of ctDNA was associated with recurrence (p =0.08).

ConclusionUMIseq demonstrated robust performance with high sensitivity and specificity, enabling the detection of ctDNA at low allele frequencies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
