<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Mechanisms underpinning osteosarcoma genome complexity and evolution
Authors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.
Score: 26.7, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573403
Osteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Mechanisms underpinning osteosarcoma genome complexity and evolution
Authors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.
Score: 26.7, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573403
Osteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-03T10:39:43+00:00" />
<meta property="article:modified_time" content="2024-01-03T10:39:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Mechanisms underpinning osteosarcoma genome complexity and evolution
Authors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.
Score: 26.7, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573403
Osteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Mechanisms underpinning osteosarcoma genome complexity and evolution\nAuthors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.\nScore: 26.7, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573403\nOsteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood.",
  "keywords": [
    
  ],
  "articleBody": " Mechanisms underpinning osteosarcoma genome complexity and evolution\nAuthors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.\nScore: 26.7, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573403\nOsteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood. Using multi-region whole-genome sequencing, we find that chromothripsis is an ongoing mutational process, occurring subclonally in 74% of tumours. Chromothripsis drives the acquisition of oncogenic mutations and generates highly unstable derivative chromosomes, the evolution of which drives clonal diversification and intra-tumour heterogeneity. In addition, we report a novel mechanism, loss-translocation-amplification (LTA) chromothripsis, which mediates rapid malignant transformation and punctuated evolution in about half of paediatric and adult high-grade osteosarcomas. Specifically, a single double-strand break triggers concomitant TP53 inactivation and segmental amplifications, often amplifying oncogenes to high copy numbers in extrachromosomal circular DNA elements through breakage-fusion-bridge cycles involving multiple chromosomes. LTA chromothripsis is detected at low frequency in soft-tissue sarcomas, but not in epithelial cancers, including those driven by TP53 mutation. Finally, we identify genome-wide loss of heterozygosity as a strong prognostic indicator for high-grade osteosarcoma.\nDecoding mTOR signalling heterogeneity in the tumour microenvironment using multiplexed imaging and graph convolutional networks\nAuthors: Zuhair, R.; Eastwood, M.; Jones, M.; Cross, A.; Hester, J.; Issa, F.; Ginty, F.; Sailem, H.\nScore: 11.9, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573693\nEvaluating the contribution of the tumour microenvironment (TME) in tumour progression has proven a complex challenge due to the intricate interactions within the TME. Multiplexed imaging is an emerging technology that allows concurrent assessment of multiple of these components simultaneously. Here we utilise a highly multiplexed dataset of 61 markers across 746 colorectal tumours to investigate how complex mTOR signalling in different tissue compartments influences patient prognosis. We found that the signalling of mTOR pathway can have heterogeneous activation patterns in tumour and immune compartments which correlate with patient prognosis. Using graph neural networks, we determined the most predictive features of mTOR activity in immune cells and identified relevant cellular subpopulations. We validated our observations using spatial transcriptomics data analysis in an independent patient cohort. Our work provides a framework for studying complex cell signalling and reveals important insights for developing mTOR-based therapies.\nSolid tumor growth depends on an intricate equilibrium of malignant cell states\nAuthors: Torborg, S. R.; Grbovic-Huezo, O.; Singhal, A.; Holm, M.; Wu, K.; Han, X.; Ho, Y.-J.; Haglund, C.; Mitchell, M. J.; Lowe, S. W.; Dow, L. E.; Pitter, K. L.; Sanchez-Rivera, F. J.; Levchenko, A.; Tammela, T.\nScore: 10.5, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573100\nControl of cell identity and number is central to tissue function, yet principles governing organization of malignant cells in tumor tissues remain poorly understood. Using mathematical modeling and candidate-based analysis, we discover primary and metastatic pancreatic ductal adenocarcinoma (PDAC) organize in a stereotypic pattern whereby PDAC cells responding to WNT signals (WNT-R) neighbor WNT-secreting cancer cells (WNT-S). Leveraging lineage-tracing, we reveal the WNT-R state is transient and gives rise to the WNT-S state that is highly stable and committed to organizing malignant tissue. We further show that a subset of WNT-S cells expressing the Notch ligand DLL1 form a functional niche for WNT-R cells. Genetic inactivation of WNT secretion or Notch pathway components, or cytoablation of the WNT-S state disrupts PDAC tissue organization, suppressing tumor growth and metastasis. This work indicates PDAC growth depends on an intricately controlled equilibrium of functionally distinct cancer cell states, uncovering a fundamental principle governing solid tumor growth and revealing new opportunities for therapeutic intervention.\nFractal Dimension and Lacunarity Measures of Glioma Subcomponents Provide a Quantitative Platform Discriminative of IDH Status: A Radiogenomics Approach in Gliomas\nAuthors: Yadav, N.; Mohanty, A.; Aswin, V.; Mishrra, N.; Tiwari, V.\nScore: 7.3, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573519\nBackgroundThe presence of structural and geometric variations within gliomas, even among those with similar histologic grades, reflects the phenotypic heterogeneity unique to a genetic and epigenetic landscape. Whole glioma mass comprises of various subcomponents identified on MR imaging: enhancing, nonenhancing, necrosis, and edema fractions in varied fractions across patients. The geometry of whole tumor mass and the glioma subcomponents is highly irregular. Thereby, traditional Euclidean geometry is not suitable for quantifying the geometric dimensions. Here, we employ non-Euclidean geometric measurements: Fractal Dimension and lacunarity of the glioma subcomponents as a discriminator of IDH and MGMT status of gliomas. MethodsFractality and Lacunarity measurements were obtained using the tumor masks generated for enhancing, nonenhancing, and edema subcomponents from the preoperative T1, T1c, and T2-Flair MRI. Fractality and lacunarity measures of each subcomponent were evaluated between IDH mutant and wildtype gliomas. The fractality and lacunarity measures in IDH mutant and wildtype gliomas were further stratified for MGMT methylated and unmethylated gliomas. The fractality and lacunarities were trained and tested using supervised ML modeling as discriminators of IDH and MGMT status. Further, Cox Hazard estimations and the Kaplan-Meir investigations were performed to evaluate the impact of fractality and lacunarity measures of glioma subcomponents on the overall survival of the patients. ResultsIDH wildtype gliomas had [~]2-fold higher fractality for the enhancing subcomponent compared to IDH mutant enhancing subcomponent, while IDH mutant gliomas showed higher fractality for the nonenhancing subcomponent. Furthermore, the edema subcomponent did not differ for fractality or lacunarity measures between IDH mutant and wildtype gliomas. Fractal or lacunarity measures for either of the three subcomponents do not vary across MGMT methylated and unmethylated status with a given IDH mutant or wildtype gliomas. A combination of fractal measures of the enhancing and nonenhancing subcomponents together provided highly accurate and sensitive discrimination of IDH status using the supervised ML models. Moreover, fractality measure [\u0026ge;] 0.69 for the enhancing subcomponent was associated with shortened patient survival: a fractal dimension value corresponding to that of IDH wild type gliomas. However, fractality and lacunarity estimates were not sensitive for discrimination of MGMT status. ConclusionGlioma structural heterogeneity measured as fractality and lacunarity using routine structural MRI measurements provide a noninvasive quantitative platform definitive of the molecular subtype of gliomas: IDH mutant vs. wildtype. Establishing fractality and/or lacunarity quantities as signatures of prognostic molecular events provides an avenue to bypass the need of biopsy/surgical interventions for decision-making, determining the molecular subtypes and overall clinical management of gliomas. Importance of the StudyThe non-Euclidean geometric measurements such as fractal dimension and lacunarity of enhancing, nonenhancing, and edema subcomponents are potentially unique quantitative metrics, discriminative of IDH status and patient survival. Fractality and Lacunarity estimates using the conventional structural MRI (T1w, T1C, T2, and T2F) provide an easy-to-use quantitative radiogenomics platform for improved clinical decisions, bypassing the need for immediate surgical interventions to ascertain prognostic molecular markers in gliomas, which is likely to improve overall clinical management and outcomes. Key PointsO_LIIncreased fractal dimensions of the enhancing subcomponents in IDH wildtype tumors, suggestive of highly irregular geometry, may potentially serve as a quantitative noninvasive determinant of IDH wildtype tumors. C_LIO_LIA combined fractal estimation of enhancing and nonenhancing subcomponents is the optimal and accurate discriminator of IDH mutant vs. wildtype. C_LIO_LIHigh fractal dimension of enhancing subcomponent and reduced fractality of nonenhancing subcomponent is predictive of shortened patient survival. C_LI\nPrecise single-cell transcriptomic mapping of normal and leukemic cell states reveals unconventional lineage priming in acute myeloid leukemia\nAuthors: Zeng, A. G. X.; Iacobucci, I.; Shah, S.; Mitchell, A.; Wong, G.; Bansal, S.; Gao, Q.; Kim, H.; Kennedy, J. A.; Minden, M. D.; Haferlach, T.; Mullighan, C. G.; Dick, J. E.\nScore: 6.8, Published: 2023-12-27 DOI: 10.1101/2023.12.26.573390\nInitial classification of acute leukemia involves the assignment of blasts to cell states within the hematopoietic hierarchy based on morphological and immunophenotypic features. Yet, these traditional classification approaches lack precision, especially at the level of immature blasts. Single-cell RNA-sequencing (scRNA-seq) enables precise determination of cell state using thousands of markers, thus providing an opportunity to re-examine present-day classification schemes of acute leukemia. Here, we developed a detailed reference map of human bone marrow hematopoiesis from 263,519 single-cell transcriptomes spanning 55 cellular states. Cell state annotations were benchmarked against purified cell populations, and in-depth characterization of gene expression programs underlying hematopoietic differentiation was undertaken. Projection of single-cell transcriptomes from 175 samples spanning acute myeloid leukemia (AML), mixed phenotype acute leukemia (MPAL), and acute erythroid leukemia (AEL) revealed 11 subtypes involving distinct stages of hematopoietic differentiation. These included AML subtypes with notable lymphoid or erythroid lineage priming, challenging traditional diagnostic boundaries between AML, MPAL, and AEL. Quantification of lineage priming in bulk patient cohorts revealed specific genetic alterations associated with this unconventional lineage priming. Integration of transcriptional and genetic information at the single-cell level revealed how genetic subclones can induce lineage restriction, differentiation blocks, or expansion of mature myeloid cells. Furthermore, we demonstrate that distinct cellular hierarchies can co-exist within individual patients, providing insight into AML evolution in response to varying selection pressures. Together, precise mapping of hematopoietic cell states can serve as a foundation for refining disease classification in acute leukemia and understanding response or resistance to emerging therapies.\nKMT2A oncoproteins induce epigenetic resistance to targeted therapies\nAuthors: Janssens, D. H.; Duran, M.; Otto, D. J.; Kirkey, D.; Yi, J. S.; Meshinchi, S.; Sarthy, J. F.; Ahmad, K.; Henikoff, S.\nScore: 3.6, Published: 2023-12-30 DOI: 10.1101/2023.12.29.573681\nChromosomal translocations involving the Lysine-Methyl-Tansferase-2A (KMT2A) locus generate potent oncogenes that cause highly aggressive acute leukemias1. KMT2A and the most frequent translocation partners encode proteins that interact with DNA to regulate developmental gene expression2. KMT2A-oncogenic fusion proteins (oncoproteins) contribute to the epigenetic mechanisms that allow KMT2A-rearranged leukemias to evade targeted therapies. By profiling the oncoprotein-target sites of 34 KMT2A-rearranged leukemia samples, we find that the genomic enrichment of oncoprotein binding is highly variable between samples. At high levels of expression, the oncoproteins preferentially activate either the lymphoid or myeloid lineage program depending on the fusion partner. These fusion-partner-dependent binding sites correspond to the frequencies of each mutation in acute lymphoid leukemia versus acute myeloid leukemia. By profiling a sample that underwent a lymphoid-to-myeloid lineage switching event in response to lymphoid-directed treatment, we find the global oncoprotein levels are reduced and the oncoprotein-target gene network changes. At lower levels of expression, the oncoprotein shifts to a non-canonical regulatory program that favors the myeloid lineage, and in a subset of resistant patients, the Menin inhibitor Revumenib induces a similar response. The dynamic shifts in KMT2A oncoproteins we describe likely contribute to epigenetic resistance of KMT2A-rearranged leukemias to targeted therapies.\nImage-Based Assessment of Natural Killer Cell Activity against Glioblastoma Stem Cells\nAuthors: Du, Y.; Metcalfe, S.; Akunapuram, S.; Richardson, A. M.; Cohen-Gadol, A. A.; Shen, J.\nScore: 3.5, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573629\nGlioblastoma (GBM) poses a significant challenge in oncology and stands as the most aggressive form of brain cancer. A primary contributor to its relentless nature is the stem-like cancer cells, called glioblastoma stem cells (GSCs). GSCs have the capacity for self-renewal and tumorigenesis, leading to frequent GBM recurrences and complicating treatment modalities. While natural killer (NK) cells exhibit potential in targeting and eliminating stem-like cancer cells, their efficacy within the GBM microenvironment is limited due to constrained infiltration and function. To address this limitation, novel investigations focusing on boosting NK cell activity against GSCs are imperative. This study presents two streamlined image-based assays assessing NK cell migration and cytotoxicity towards GSCs. It details protocols and explores the strengths and limitations of these methods. These assays could aid in identifying novel targets to enhance NK cell activity towards GSCs, facilitating the development of NK cell-based immunotherapy for improved GBM treatment.\nThe ribosomal protein L22 binds the MDM4 pre-mRNA and promotes exon skipping to activate p53 upon nucleolar stress\nAuthors: Jansen, J.; Bohnsack, K. E.; Boehlken-Fascher, S.; Bohnsack, M. T.; Dobbelstein, M.\nScore: 3.2, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573614\nThe tumor suppressor p53, along with its antagonists MDM2 and MDM4, represents a central integrator of stress signaling. While DNA damage is the most widely explored trigger of a p53 response, stress can also arise from dysbalanced assembly of ribosomes in nucleoli. Deletions of the gene encoding the ribosomal protein L22 (RPL22; eL22) correlate with the presence of full-length MDM4 mRNA in human cancer, but the mechanistic basis for this phenomenon was hitherto unknown. Here we show that L22, under conditions of ribosomal and nucleolar stress, promotes the skipping of exon 6 within the MDM4 pre-mRNA. Upon L22 depletion, far more full-length MDM4 is maintained despite treatment with nucleolar stressors, leading to diminished p53 activity and enhanced proliferation. Mechanistically, L22 binds to specific RNA elements within intron 6 of MDM4 that correspond to a stem-loop consensus, leading to the skipping of exon 6. This intronic RNA overlaps with the region responsible for splice regulation by ZMAT3. Targeted deletion of these intronic elements largely abolishes L22-mediated exon skipping and re-enables cell proliferation despite nucleolar stressors such as 5-fluorouracil. L22 also governs alternative splicing of the RPL22L1 and UBAP2L mRNAs. Thus, L22 serves as a signaling intermediate that integrates different layers of gene expression. Defects in ribosome synthesis lead to specific alternative splicing, ultimately triggering p53-mediated transcription and arresting cell proliferation.\nTimp2 loss-of-function mutation and TIMP2 treatment in murine model of NSCLC: modulation of immunosuppression and oncogenic signaling\nAuthors: Peeney, D.; Kumar, S.; Singh, T. P.; Liu, Y.; Jensen, S.; Chowdhury, A.; Coates-Park, S.; Rich, J.; Gurung, S.; Fan, Y.; Meerzaman, D.; Stetler-Stevenson, W. G.\nScore: 3.0, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573636\nMounting evidence suggests that the tissue inhibitor of metalloproteinases-2 (TIMP2) can reduce tumor burden and metastasis. However, the demonstration of such anti-tumor activity and associated mechanisms using in vivo tumor models is lacking. The effects of a Timp2 functional mutation and administration of recombinant TIMP2 were examined in both orthotopic and heterotopic murine models of lung cancer using C57Bl/6 syngeneic Lewis Lung 2-luciferase 2 cells (LL2-luc2) cells. Mice harboring a functional mutation of TIMP2 (mT2) display markedly increased primary lung tumor growth, increased mortality, enriched vasculature, and enhanced infiltration of pro-tumorigenic, immunosuppressive myeloid cells. Treatment with recombinant TIMP2 reduced primary tumor growth in both mutant and wild-type (wt) mice. Comparison of transcriptional profiles of lung tissues from tumor-free, wt versus mT2 mice reveals only minor changes. However, lung tumor-bearing mice of both genotypes demonstrate significant genotype-dependent changes in gene expression following treatment with TIMP. In tumor-bearing wt mice, TIMP2 treatment reduced the expression of upstream oncogenic mediators, whereas treatment of mT2 mice resulted in an immunomodulatory phenotype. A heterotopic subcutaneous model generating metastatic pulmonary tumors demonstrated that daily administration of recombinant TIMP2 significantly downregulates the expression of heat shock proteins, suggesting a reduction of cell-stress responses. In summary, we describe how TIMP2 exerts novel, anti-tumor effects in a murine model of lung cancer and that rTIMP2 treatment supports a normalizing effect on the tumor microenvironment. Our findings show that TIMP2 treatment demonstrates significant potential as an adjuvant in the treatment of NSCLC. One Sentence SummaryTIMP2 treatment compensates for TIMP2 mutation to reveal immunomodulatory and oncogene regulation that inhibits tumor growth and metastatic niche formation.\nIdentification of nonsense-mediated decay inhibitors that alter the tumor immune landscape\nAuthors: Cook, A. L.; Sur, S.; Dobbyn, L.; Watson, E.; Cohen, J. D.; Ptak, B.; Lee, B. S.; Paul, S.; Hsiue, E.; Popoli, M.; Vogelstein, B.; Papadopoulos, N.; Bettegowda, C.; Gabrielson, K.; Zhou, S.; Kinzler, K. W.; Wyhs, N.\nScore: 3.0, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573594\nDespite exciting developments in cancer immunotherapy, its broad application is limited by the paucity of targetable antigens on the tumor cell surface. As an intrinsic cellular pathway, nonsense- mediated decay (NMD) conceals neoantigens through the destruction of the RNA products from genes harboring truncating mutations. We developed and conducted a high throughput screen, based on the ratiometric analysis of transcripts, to identify critical mediators of NMD. This screen revealed disruption of kinase SMG1s phosphorylation of UPF1 as a potent disruptor of NMD. This led us to design a novel SMG1 inhibitor, KVS0001, that elevates the expression of transcripts and proteins resulting from truncating mutations in vivo and in vitro. Most importantly, KVS0001 concomitantly increased the presentation of immune-targetable HLA class I-associated peptides from NMD-downregulated proteins on the surface of cancer cells. KVS0001 provides new opportunities for studying NMD and the diseases in which NMD plays a role, including cancer and inherited diseases. One Sentence SummaryDisruption of the nonsense-mediated decay pathway with a newly developed SMG1 inhibitor with in-vivo activity increases the expression of T-cell targetable cancer neoantigens resulting from truncating mutations.\n",
  "wordCount" : "2724",
  "inLanguage": "en",
  "datePublished": "2024-01-03T10:39:43Z",
  "dateModified": "2024-01-03T10:39:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 3, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573403" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573403">
        <p class="paperTitle">Mechanisms underpinning osteosarcoma genome complexity and evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.</p>
        <p class="info">Score: 26.7, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573403' target='https://doi.org/10.1101/2023.12.29.573403'> 10.1101/2023.12.29.573403</a></p>
        <p class="abstract">Osteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood. Using multi-region whole-genome sequencing, we find that chromothripsis is an ongoing mutational process, occurring subclonally in 74% of tumours. Chromothripsis drives the acquisition of oncogenic mutations and generates highly unstable derivative chromosomes, the evolution of which drives clonal diversification and intra-tumour heterogeneity. In addition, we report a novel mechanism, loss-translocation-amplification (LTA) chromothripsis, which mediates rapid malignant transformation and punctuated evolution in about half of paediatric and adult high-grade osteosarcomas. Specifically, a single double-strand break triggers concomitant TP53 inactivation and segmental amplifications, often amplifying oncogenes to high copy numbers in extrachromosomal circular DNA elements through breakage-fusion-bridge cycles involving multiple chromosomes. LTA chromothripsis is detected at low frequency in soft-tissue sarcomas, but not in epithelial cancers, including those driven by TP53 mutation. Finally, we identify genome-wide loss of heterozygosity as a strong prognostic indicator for high-grade osteosarcoma.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573693">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573693" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573693">
        <p class="paperTitle">Decoding mTOR signalling heterogeneity in the tumour microenvironment using multiplexed imaging and graph convolutional networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573693" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573693" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zuhair, R.; Eastwood, M.; Jones, M.; Cross, A.; Hester, J.; Issa, F.; Ginty, F.; Sailem, H.</p>
        <p class="info">Score: 11.9, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573693' target='https://doi.org/10.1101/2023.12.30.573693'> 10.1101/2023.12.30.573693</a></p>
        <p class="abstract">Evaluating the contribution of the tumour microenvironment (TME) in tumour progression has proven a complex challenge due to the intricate interactions within the TME. Multiplexed imaging is an emerging technology that allows concurrent assessment of multiple of these components simultaneously. Here we utilise a highly multiplexed dataset of 61 markers across 746 colorectal tumours to investigate how complex mTOR signalling in different tissue compartments influences patient prognosis. We found that the signalling of mTOR pathway can have heterogeneous activation patterns in tumour and immune compartments which correlate with patient prognosis. Using graph neural networks, we determined the most predictive features of mTOR activity in immune cells and identified relevant cellular subpopulations. We validated our observations using spatial transcriptomics data analysis in an independent patient cohort. Our work provides a framework for studying complex cell signalling and reveals important insights for developing mTOR-based therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573100">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573100" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573100">
        <p class="paperTitle">Solid tumor growth depends on an intricate equilibrium of malignant cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573100" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573100" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Torborg, S. R.; Grbovic-Huezo, O.; Singhal, A.; Holm, M.; Wu, K.; Han, X.; Ho, Y.-J.; Haglund, C.; Mitchell, M. J.; Lowe, S. W.; Dow, L. E.; Pitter, K. L.; Sanchez-Rivera, F. J.; Levchenko, A.; Tammela, T.</p>
        <p class="info">Score: 10.5, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573100' target='https://doi.org/10.1101/2023.12.30.573100'> 10.1101/2023.12.30.573100</a></p>
        <p class="abstract">Control of cell identity and number is central to tissue function, yet principles governing organization of malignant cells in tumor tissues remain poorly understood. Using mathematical modeling and candidate-based analysis, we discover primary and metastatic pancreatic ductal adenocarcinoma (PDAC) organize in a stereotypic pattern whereby PDAC cells responding to WNT signals (WNT-R) neighbor WNT-secreting cancer cells (WNT-S). Leveraging lineage-tracing, we reveal the WNT-R state is transient and gives rise to the WNT-S state that is highly stable and committed to organizing malignant tissue. We further show that a subset of WNT-S cells expressing the Notch ligand DLL1 form a functional niche for WNT-R cells. Genetic inactivation of WNT secretion or Notch pathway components, or cytoablation of the WNT-S state disrupts PDAC tissue organization, suppressing tumor growth and metastasis. This work indicates PDAC growth depends on an intricately controlled equilibrium of functionally distinct cancer cell states, uncovering a fundamental principle governing solid tumor growth and revealing new opportunities for therapeutic intervention.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573519" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573519">
        <p class="paperTitle">Fractal Dimension and Lacunarity Measures of Glioma Subcomponents Provide a Quantitative Platform Discriminative of IDH Status: A Radiogenomics Approach in Gliomas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yadav, N.; Mohanty, A.; Aswin, V.; Mishrra, N.; Tiwari, V.</p>
        <p class="info">Score: 7.3, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573519' target='https://doi.org/10.1101/2023.12.28.573519'> 10.1101/2023.12.28.573519</a></p>
        <p class="abstract">BackgroundThe presence of structural and geometric variations within gliomas, even among those with similar histologic grades, reflects the phenotypic heterogeneity unique to a genetic and epigenetic landscape. Whole glioma mass comprises of various subcomponents identified on MR imaging: enhancing, nonenhancing, necrosis, and edema fractions in varied fractions across patients. The geometry of whole tumor mass and the glioma subcomponents is highly irregular. Thereby, traditional Euclidean geometry is not suitable for quantifying the geometric dimensions. Here, we employ non-Euclidean geometric measurements: Fractal Dimension and lacunarity of the glioma subcomponents as a discriminator of IDH and MGMT status of gliomas.

MethodsFractality and Lacunarity measurements were obtained using the tumor masks generated for enhancing, nonenhancing, and edema subcomponents from the preoperative T1, T1c, and T2-Flair MRI. Fractality and lacunarity measures of each subcomponent were evaluated between IDH mutant and wildtype gliomas. The fractality and lacunarity measures in IDH mutant and wildtype gliomas were further stratified for MGMT methylated and unmethylated gliomas. The fractality and lacunarities were trained and tested using supervised ML modeling as discriminators of IDH and MGMT status. Further, Cox Hazard estimations and the Kaplan-Meir investigations were performed to evaluate the impact of fractality and lacunarity measures of glioma subcomponents on the overall survival of the patients.

ResultsIDH wildtype gliomas had [~]2-fold higher fractality for the enhancing subcomponent compared to IDH mutant enhancing subcomponent, while IDH mutant gliomas showed higher fractality for the nonenhancing subcomponent. Furthermore, the edema subcomponent did not differ for fractality or lacunarity measures between IDH mutant and wildtype gliomas. Fractal or lacunarity measures for either of the three subcomponents do not vary across MGMT methylated and unmethylated status with a given IDH mutant or wildtype gliomas. A combination of fractal measures of the enhancing and nonenhancing subcomponents together provided highly accurate and sensitive discrimination of IDH status using the supervised ML models. Moreover, fractality measure [&amp;ge;] 0.69 for the enhancing subcomponent was associated with shortened patient survival: a fractal dimension value corresponding to that of IDH wild type gliomas. However, fractality and lacunarity estimates were not sensitive for discrimination of MGMT status.

ConclusionGlioma structural heterogeneity measured as fractality and lacunarity using routine structural MRI measurements provide a noninvasive quantitative platform definitive of the molecular subtype of gliomas: IDH mutant vs. wildtype. Establishing fractality and/or lacunarity quantities as signatures of prognostic molecular events provides an avenue to bypass the need of biopsy/surgical interventions for decision-making, determining the molecular subtypes and overall clinical management of gliomas.

Importance of the StudyThe non-Euclidean geometric measurements such as fractal dimension and lacunarity of enhancing, nonenhancing, and edema subcomponents are potentially unique quantitative metrics, discriminative of IDH status and patient survival. Fractality and Lacunarity estimates using the conventional structural MRI (T1w, T1C, T2, and T2F) provide an easy-to-use quantitative radiogenomics platform for improved clinical decisions, bypassing the need for immediate surgical interventions to ascertain prognostic molecular markers in gliomas, which is likely to improve overall clinical management and outcomes.

Key PointsO_LIIncreased fractal dimensions of the enhancing subcomponents in IDH wildtype tumors, suggestive of highly irregular geometry, may potentially serve as a quantitative noninvasive determinant of IDH wildtype tumors.
C_LIO_LIA combined fractal estimation of enhancing and nonenhancing subcomponents is the optimal and accurate discriminator of IDH mutant vs. wildtype.
C_LIO_LIHigh fractal dimension of enhancing subcomponent and reduced fractality of nonenhancing subcomponent is predictive of shortened patient survival.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573390">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573390" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573390">
        <p class="paperTitle">Precise single-cell transcriptomic mapping of normal and leukemic cell states reveals unconventional lineage priming in acute myeloid leukemia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573390" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573390" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zeng, A. G. X.; Iacobucci, I.; Shah, S.; Mitchell, A.; Wong, G.; Bansal, S.; Gao, Q.; Kim, H.; Kennedy, J. A.; Minden, M. D.; Haferlach, T.; Mullighan, C. G.; Dick, J. E.</p>
        <p class="info">Score: 6.8, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573390' target='https://doi.org/10.1101/2023.12.26.573390'> 10.1101/2023.12.26.573390</a></p>
        <p class="abstract">Initial classification of acute leukemia involves the assignment of blasts to cell states within the hematopoietic hierarchy based on morphological and immunophenotypic features. Yet, these traditional classification approaches lack precision, especially at the level of immature blasts. Single-cell RNA-sequencing (scRNA-seq) enables precise determination of cell state using thousands of markers, thus providing an opportunity to re-examine present-day classification schemes of acute leukemia. Here, we developed a detailed reference map of human bone marrow hematopoiesis from 263,519 single-cell transcriptomes spanning 55 cellular states. Cell state annotations were benchmarked against purified cell populations, and in-depth characterization of gene expression programs underlying hematopoietic differentiation was undertaken. Projection of single-cell transcriptomes from 175 samples spanning acute myeloid leukemia (AML), mixed phenotype acute leukemia (MPAL), and acute erythroid leukemia (AEL) revealed 11 subtypes involving distinct stages of hematopoietic differentiation. These included AML subtypes with notable lymphoid or erythroid lineage priming, challenging traditional diagnostic boundaries between AML, MPAL, and AEL. Quantification of lineage priming in bulk patient cohorts revealed specific genetic alterations associated with this unconventional lineage priming. Integration of transcriptional and genetic information at the single-cell level revealed how genetic subclones can induce lineage restriction, differentiation blocks, or expansion of mature myeloid cells. Furthermore, we demonstrate that distinct cellular hierarchies can co-exist within individual patients, providing insight into AML evolution in response to varying selection pressures. Together, precise mapping of hematopoietic cell states can serve as a foundation for refining disease classification in acute leukemia and understanding response or resistance to emerging therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573681">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573681" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573681">
        <p class="paperTitle">KMT2A oncoproteins induce epigenetic resistance to targeted therapies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573681" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573681" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janssens, D. H.; Duran, M.; Otto, D. J.; Kirkey, D.; Yi, J. S.; Meshinchi, S.; Sarthy, J. F.; Ahmad, K.; Henikoff, S.</p>
        <p class="info">Score: 3.6, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573681' target='https://doi.org/10.1101/2023.12.29.573681'> 10.1101/2023.12.29.573681</a></p>
        <p class="abstract">Chromosomal translocations involving the Lysine-Methyl-Tansferase-2A (KMT2A) locus generate potent oncogenes that cause highly aggressive acute leukemias1. KMT2A and the most frequent translocation partners encode proteins that interact with DNA to regulate developmental gene expression2. KMT2A-oncogenic fusion proteins (oncoproteins) contribute to the epigenetic mechanisms that allow KMT2A-rearranged leukemias to evade targeted therapies. By profiling the oncoprotein-target sites of 34 KMT2A-rearranged leukemia samples, we find that the genomic enrichment of oncoprotein binding is highly variable between samples. At high levels of expression, the oncoproteins preferentially activate either the lymphoid or myeloid lineage program depending on the fusion partner. These fusion-partner-dependent binding sites correspond to the frequencies of each mutation in acute lymphoid leukemia versus acute myeloid leukemia. By profiling a sample that underwent a lymphoid-to-myeloid lineage switching event in response to lymphoid-directed treatment, we find the global oncoprotein levels are reduced and the oncoprotein-target gene network changes. At lower levels of expression, the oncoprotein shifts to a non-canonical regulatory program that favors the myeloid lineage, and in a subset of resistant patients, the Menin inhibitor Revumenib induces a similar response. The dynamic shifts in KMT2A oncoproteins we describe likely contribute to epigenetic resistance of KMT2A-rearranged leukemias to targeted therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573629">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573629" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573629">
        <p class="paperTitle">Image-Based Assessment of Natural Killer Cell Activity against Glioblastoma Stem Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573629" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573629" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Du, Y.; Metcalfe, S.; Akunapuram, S.; Richardson, A. M.; Cohen-Gadol, A. A.; Shen, J.</p>
        <p class="info">Score: 3.5, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573629' target='https://doi.org/10.1101/2023.12.29.573629'> 10.1101/2023.12.29.573629</a></p>
        <p class="abstract">Glioblastoma (GBM) poses a significant challenge in oncology and stands as the most aggressive form of brain cancer. A primary contributor to its relentless nature is the stem-like cancer cells, called glioblastoma stem cells (GSCs). GSCs have the capacity for self-renewal and tumorigenesis, leading to frequent GBM recurrences and complicating treatment modalities. While natural killer (NK) cells exhibit potential in targeting and eliminating stem-like cancer cells, their efficacy within the GBM microenvironment is limited due to constrained infiltration and function. To address this limitation, novel investigations focusing on boosting NK cell activity against GSCs are imperative. This study presents two streamlined image-based assays assessing NK cell migration and cytotoxicity towards GSCs. It details protocols and explores the strengths and limitations of these methods. These assays could aid in identifying novel targets to enhance NK cell activity towards GSCs, facilitating the development of NK cell-based immunotherapy for improved GBM treatment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573614">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573614" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573614">
        <p class="paperTitle">The ribosomal protein L22 binds the MDM4 pre-mRNA and promotes exon skipping to activate p53 upon nucleolar stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573614" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573614" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jansen, J.; Bohnsack, K. E.; Boehlken-Fascher, S.; Bohnsack, M. T.; Dobbelstein, M.</p>
        <p class="info">Score: 3.2, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573614' target='https://doi.org/10.1101/2023.12.29.573614'> 10.1101/2023.12.29.573614</a></p>
        <p class="abstract">The tumor suppressor p53, along with its antagonists MDM2 and MDM4, represents a central integrator of stress signaling. While DNA damage is the most widely explored trigger of a p53 response, stress can also arise from dysbalanced assembly of ribosomes in nucleoli. Deletions of the gene encoding the ribosomal protein L22 (RPL22; eL22) correlate with the presence of full-length MDM4 mRNA in human cancer, but the mechanistic basis for this phenomenon was hitherto unknown. Here we show that L22, under conditions of ribosomal and nucleolar stress, promotes the skipping of exon 6 within the MDM4 pre-mRNA. Upon L22 depletion, far more full-length MDM4 is maintained despite treatment with nucleolar stressors, leading to diminished p53 activity and enhanced proliferation. Mechanistically, L22 binds to specific RNA elements within intron 6 of MDM4 that correspond to a stem-loop consensus, leading to the skipping of exon 6. This intronic RNA overlaps with the region responsible for splice regulation by ZMAT3. Targeted deletion of these intronic elements largely abolishes L22-mediated exon skipping and re-enables cell proliferation despite nucleolar stressors such as 5-fluorouracil. L22 also governs alternative splicing of the RPL22L1 and UBAP2L mRNAs. Thus, L22 serves as a signaling intermediate that integrates different layers of gene expression. Defects in ribosome synthesis lead to specific alternative splicing, ultimately triggering p53-mediated transcription and arresting cell proliferation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573636">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573636" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573636">
        <p class="paperTitle">Timp2 loss-of-function mutation and TIMP2 treatment in murine model of NSCLC: modulation of immunosuppression and oncogenic signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573636" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573636" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peeney, D.; Kumar, S.; Singh, T. P.; Liu, Y.; Jensen, S.; Chowdhury, A.; Coates-Park, S.; Rich, J.; Gurung, S.; Fan, Y.; Meerzaman, D.; Stetler-Stevenson, W. G.</p>
        <p class="info">Score: 3.0, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573636' target='https://doi.org/10.1101/2023.12.29.573636'> 10.1101/2023.12.29.573636</a></p>
        <p class="abstract">Mounting evidence suggests that the tissue inhibitor of metalloproteinases-2 (TIMP2) can reduce tumor burden and metastasis. However, the demonstration of such anti-tumor activity and associated mechanisms using in vivo tumor models is lacking. The effects of a Timp2 functional mutation and administration of recombinant TIMP2 were examined in both orthotopic and heterotopic murine models of lung cancer using C57Bl/6 syngeneic Lewis Lung 2-luciferase 2 cells (LL2-luc2) cells. Mice harboring a functional mutation of TIMP2 (mT2) display markedly increased primary lung tumor growth, increased mortality, enriched vasculature, and enhanced infiltration of pro-tumorigenic, immunosuppressive myeloid cells. Treatment with recombinant TIMP2 reduced primary tumor growth in both mutant and wild-type (wt) mice. Comparison of transcriptional profiles of lung tissues from tumor-free, wt versus mT2 mice reveals only minor changes. However, lung tumor-bearing mice of both genotypes demonstrate significant genotype-dependent changes in gene expression following treatment with TIMP. In tumor-bearing wt mice, TIMP2 treatment reduced the expression of upstream oncogenic mediators, whereas treatment of mT2 mice resulted in an immunomodulatory phenotype. A heterotopic subcutaneous model generating metastatic pulmonary tumors demonstrated that daily administration of recombinant TIMP2 significantly downregulates the expression of heat shock proteins, suggesting a reduction of cell-stress responses. In summary, we describe how TIMP2 exerts novel, anti-tumor effects in a murine model of lung cancer and that rTIMP2 treatment supports a normalizing effect on the tumor microenvironment. Our findings show that TIMP2 treatment demonstrates significant potential as an adjuvant in the treatment of NSCLC.

One Sentence SummaryTIMP2 treatment compensates for TIMP2 mutation to reveal immunomodulatory and oncogene regulation that inhibits tumor growth and metastatic niche formation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573594">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573594" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573594">
        <p class="paperTitle">Identification of nonsense-mediated decay inhibitors that alter the tumor immune landscape</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573594" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573594" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cook, A. L.; Sur, S.; Dobbyn, L.; Watson, E.; Cohen, J. D.; Ptak, B.; Lee, B. S.; Paul, S.; Hsiue, E.; Popoli, M.; Vogelstein, B.; Papadopoulos, N.; Bettegowda, C.; Gabrielson, K.; Zhou, S.; Kinzler, K. W.; Wyhs, N.</p>
        <p class="info">Score: 3.0, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573594' target='https://doi.org/10.1101/2023.12.28.573594'> 10.1101/2023.12.28.573594</a></p>
        <p class="abstract">Despite exciting developments in cancer immunotherapy, its broad application is limited by the paucity of targetable antigens on the tumor cell surface. As an intrinsic cellular pathway, nonsense- mediated decay (NMD) conceals neoantigens through the destruction of the RNA products from genes harboring truncating mutations. We developed and conducted a high throughput screen, based on the ratiometric analysis of transcripts, to identify critical mediators of NMD. This screen revealed disruption of kinase SMG1s phosphorylation of UPF1 as a potent disruptor of NMD. This led us to design a novel SMG1 inhibitor, KVS0001, that elevates the expression of transcripts and proteins resulting from truncating mutations in vivo and in vitro. Most importantly, KVS0001 concomitantly increased the presentation of immune-targetable HLA class I-associated peptides from NMD-downregulated proteins on the surface of cancer cells. KVS0001 provides new opportunities for studying NMD and the diseases in which NMD plays a role, including cancer and inherited diseases.

One Sentence SummaryDisruption of the nonsense-mediated decay pathway with a newly developed SMG1 inhibitor with in-vivo activity increases the expression of T-cell targetable cancer neoantigens resulting from truncating mutations.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
