<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Cancer-associated fibroblasts serve as decoys to suppress NK cell anti-cancer cytotoxicity
Authors: Ben-Shmuel, A.; Gruper, Y.; Levi-Galibov, O.; Rosenberg-Fogler, H.; Carradori, G.; Stein, Y.; Dadiani, M.; Naumova, M.; Nevo, R.; Morzaev-Sulzbach, D.; Yagel, G.; Mayer, S.; Gal-Yam, E.; Scherz-Shouval, R.
Score: 17.2, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568355
Cancer associated fibroblasts (CAFs) are among the most abundant components of the breast tumor microenvironment (TME) and major contributors to immune modulation. CAFs are well-known to regulate the activity of diverse types of immune cells including T cells, macrophages and dendritic cells, however little is known about their interaction with Natural killer (NK) cells, which constitute an important arm of anti-tumor immunity.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Cancer-associated fibroblasts serve as decoys to suppress NK cell anti-cancer cytotoxicity
Authors: Ben-Shmuel, A.; Gruper, Y.; Levi-Galibov, O.; Rosenberg-Fogler, H.; Carradori, G.; Stein, Y.; Dadiani, M.; Naumova, M.; Nevo, R.; Morzaev-Sulzbach, D.; Yagel, G.; Mayer, S.; Gal-Yam, E.; Scherz-Shouval, R.
Score: 17.2, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568355
Cancer associated fibroblasts (CAFs) are among the most abundant components of the breast tumor microenvironment (TME) and major contributors to immune modulation. CAFs are well-known to regulate the activity of diverse types of immune cells including T cells, macrophages and dendritic cells, however little is known about their interaction with Natural killer (NK) cells, which constitute an important arm of anti-tumor immunity." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-26T10:37:33+00:00" />
<meta property="article:modified_time" content="2023-11-26T10:37:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Cancer-associated fibroblasts serve as decoys to suppress NK cell anti-cancer cytotoxicity
Authors: Ben-Shmuel, A.; Gruper, Y.; Levi-Galibov, O.; Rosenberg-Fogler, H.; Carradori, G.; Stein, Y.; Dadiani, M.; Naumova, M.; Nevo, R.; Morzaev-Sulzbach, D.; Yagel, G.; Mayer, S.; Gal-Yam, E.; Scherz-Shouval, R.
Score: 17.2, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568355
Cancer associated fibroblasts (CAFs) are among the most abundant components of the breast tumor microenvironment (TME) and major contributors to immune modulation. CAFs are well-known to regulate the activity of diverse types of immune cells including T cells, macrophages and dendritic cells, however little is known about their interaction with Natural killer (NK) cells, which constitute an important arm of anti-tumor immunity."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Cancer-associated fibroblasts serve as decoys to suppress NK cell anti-cancer cytotoxicity\nAuthors: Ben-Shmuel, A.; Gruper, Y.; Levi-Galibov, O.; Rosenberg-Fogler, H.; Carradori, G.; Stein, Y.; Dadiani, M.; Naumova, M.; Nevo, R.; Morzaev-Sulzbach, D.; Yagel, G.; Mayer, S.; Gal-Yam, E.; Scherz-Shouval, R.\nScore: 17.2, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568355\nCancer associated fibroblasts (CAFs) are among the most abundant components of the breast tumor microenvironment (TME) and major contributors to immune modulation. CAFs are well-known to regulate the activity of diverse types of immune cells including T cells, macrophages and dendritic cells, however little is known about their interaction with Natural killer (NK) cells, which constitute an important arm of anti-tumor immunity.",
  "keywords": [
    
  ],
  "articleBody": " Cancer-associated fibroblasts serve as decoys to suppress NK cell anti-cancer cytotoxicity\nAuthors: Ben-Shmuel, A.; Gruper, Y.; Levi-Galibov, O.; Rosenberg-Fogler, H.; Carradori, G.; Stein, Y.; Dadiani, M.; Naumova, M.; Nevo, R.; Morzaev-Sulzbach, D.; Yagel, G.; Mayer, S.; Gal-Yam, E.; Scherz-Shouval, R.\nScore: 17.2, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568355\nCancer associated fibroblasts (CAFs) are among the most abundant components of the breast tumor microenvironment (TME) and major contributors to immune modulation. CAFs are well-known to regulate the activity of diverse types of immune cells including T cells, macrophages and dendritic cells, however little is known about their interaction with Natural killer (NK) cells, which constitute an important arm of anti-tumor immunity. Here we find, using mouse models of cancer and ex-vivo co-cultures, that CAFs inhibit NK cell cytotoxicity towards cancer cells. We unravel the mechanism by which this suppression occurs, through ligand-receptor engagement between NK cells and CAFs leading to CAF cytolysis, which in turn diminishes the expression of activating receptors on NK cells, promoting cancer escape from NK cell surveillance. Analysis of breast cancer patient samples reveals enrichment of NK cells in CAF-rich regions, and upregulation of NK binding ligands on CAFs which is correlated with poor disease outcome. These results reveal a CAF-mediated immunosuppressive decoy mechanism with implications for treatment of solid tumors.\nSingle-molecule systems for detection and monitoring of plasma circulating nucleosomes and oncoproteins in Diffuse Midline Glioma\nAuthors: Erez, N.; Furth, N.; Fedyuk, V.; Wadden, J.; Aittaleb, R.; Schwark, K.; Niculcea, M.; Miclea, M.; Mody, R.; Franson, A.; Eze, A.; Nourmohammadi, N.; Nazarian, J.; Venneti, S.; Koschmann, C.; Shema, E.\nScore: 8.8, Published: 2023-11-21 DOI: 10.1101/2023.11.21.568019\nThe analysis of cell-free tumor DNA (ctDNA) and proteins in the blood of cancer patients potentiates a new generation of non-invasive diagnostics and treatment monitoring approaches. However, confident detection of these tumor-originating markers is challenging, especially in the context of brain tumors, in which extremely low amounts of these analytes circulate in the patients plasma. Here, we applied a sensitive single-molecule technology to profile multiple histone modifications on millions of individual nucleosomes from the plasma of Diffuse Midline Glioma (DMG) patients. The system reveals epigenetic patterns that are unique to DMG, significantly differentiating this group of patients from healthy subjects or individuals diagnosed with other cancer types. We further develop a method to directly capture and quantify the tumor-originating oncoproteins, H3-K27M and mutant p53, from the plasma of children diagnosed with DMG. This single-molecule system allows for accurate molecular classification of patients, utilizing less than 1ml of liquid-biopsy material. Furthermore, we show that our simple and rapid detection strategy correlates with MRI measurements and droplet-digital PCR (ddPCR) measurements of ctDNA, highlighting the utility of this approach for non-invasive treatment monitoring of DMG patients. This work underscores the clinical potential of single-molecule-based, multi-parametric assays for DMG diagnosis and treatment monitoring.\nBiocompatibility characterisation of CMOS-based Lab-on-Chip electrochemical sensors for in vitro cancer cell culture applications\nAuthors: Beykou, M.; Bousgouni, V.; Moser, N.; Georgiou, P.; Bakal, C.\nScore: 8.6, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568427\nLab-on-Chip electrochemical sensors, such as Ion-Sensitive Field-Effect Transistors (ISFETs), are being developed for use in point-of-care diagnostics, such as pH detection of tumour microenvironments, due to their integration with standard Complementary Metal Oxide Semiconductor technology. With this approach, the passivation of the CMOS process is used as a sensing layer to minimise post-processing, and Silicon Nitride (Si3N4) is the most common material at the microchip surface. ISFETs have the potential to be used for cell-based assays however, there is a poor understanding of the biocompatibility of microchip surfaces. Here, we quantitatively evaluated cell adhesion, morphogenesis, proliferation and mechano-responsiveness of both normal and cancer cells cultured on a Si3N4, sensor surface. We demonstrate that both normal and cancer cell adhesion decreased on Si3N4. Activation of the mechano-responsive transcription regulators, YAP/TAZ, are significantly decreased in cancer cells on Si3N4 in comparison to standard cell culture plastic, whilst proliferation marker, Ki67, expression markedly increased. Non-tumorigenic cells on chip showed less sensitivity to culture on Si3N4 than cancer cells. Treatment with extracellular matrix components increased cell adhesion in normal and cancer cell cultures, surpassing the adhesiveness of plastic alone. Moreover, poly-l-ornithine and laminin treatment restored YAP/TAZ levels in both non-tumorigenic and cancer cells to levels comparable to those observed on plastic. Thus, engineering the electrochemical sensor surface with treatments will provide a more physiologically relevant environment for future cell-based assay development on chip.\nEpigenetic heritability of cell plasticity drives cancer drug resistance through one-to-many genotype to phenotype mapping\nAuthors: Fernandez-Mateos, J.; Milite, S.; Oliveira, E.; Vlachogiannis, G.; Chen, B.; Yara, E.; Cresswell, G. D.; James, C.; Patruno, L.; Ascolani, G.; Acar, A.; Heide, T.; Spiteri, I.; Graudenzi, A.; Caravagna, G.; Graham, T.; Magnani, L.; Valeri, N.; Sottoriva, A.\nScore: 43.8, Published: 2023-11-16 DOI: 10.1101/2023.11.15.567140\nDrug resistance is a largely unsolved problem in oncology. Despite the explanatory power of the genetic model of cancer initiation, most treatment resistance is unexplained by genetics alone. Even when known resistance mutations are present, they are often found in a small proportion of the cells in the tumour. So where is the cellular memory that leads to treatment failure? New evidence suggests resistance is multi-factorial, resulting from the contribution of heritable genetic and epigenetic changes, but also non-heritable phenotypic plasticity. However, cell plasticity has proven hard to study as it dynamically changes over time and needs to be distinguished from clonal evolution where cell phenotypes change because of Darwinian selective bottlenecks. Here we dissected the contribution of different evolutionary processes to drug resistance by perturbing patient-derived organoids with multiple drugs in sequence. We combined dense longitudinal tracking, single cell multi- omics, evolutionary modelling, and machine learning archetypal analysis. We found that different drugs select for distinct subclones, an essential requirement for the use of evolutionary therapy with sequential drug treatment. The data supports a model in which the cellular memory is encoded as a heritable configuration of the epigenome, which however produces multiple transcriptional programmes. Those emerge in different proportions depending on the environment, giving rise to cellular plasticity. Epigenetically encoded programmes include reactivation of developmental genes and cell regeneration. A one-to-many (epi)genotype[-\u0026gt;]phenotype map explains how clonal expansions and non- heritable phenotypic plasticity manifest together, including drug tolerant states. This ensures the robustness of drug resistance subclones that can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding their selective advantage.\nAmmonia inhibits antitumor activity of NK cells by decreasing mature perforin\nAuthors: Winiarska, M.; Domagala, J.; Grzywa, T. M.; Baranowska, I.; Kusowska, A.; Fidyt, K.; Marhelava, K.; Pilch, Z.; Graczyk-Jarzynka, A.; Picard, L.; Jastrzebski, K.; Granica, M.; Justyniarska, M.; Urlaub, D.; Bobrowicz, M.; Miaczynska, M.; Watzl, C.\nScore: 4.1, Published: 2023-11-21 DOI: 10.1101/2023.11.20.567708\nImmunotherapy revolutionized cancer treatment in the last decade. Natural killer (NK) cells are one of the key host immunity components against malignant cells. Thus, they are currently extensively investigated in the field of immunotherapy of cancer. Different approaches have been developed to improve the antitumor activity of NK cells. Nonetheless, tumor microenvironment remains an obstacle to effective NK cell-based therapies. Here, we demonstrated that a cancer-conditioned medium suppresses the anti-tumor activity of NK cells. Further, we found that ammonia, a by-product of cancer cell metabolism, accumulates in the cancer-conditioned medium and tumor microenvironment. We identified that ammonia impairs the cytotoxicity of NK cells as well as the effectiveness of antibody-based and chimeric antigen receptor (CAR)-NK-based therapies in vitro. Inhibited activity of NK cells was caused by decreased levels of perforin. This effect was dependent on the lysosomotropic features of ammonia and its ability to increase pH in acidic compartments. In consequence, upon contact with ammonia the mature form of perforin was decreased in NK cells leading to their dysfunction. Our findings demonstrate that in addition to its previously described role of promoting tumor growth as a nitrogen source for tumor biomass ammonia could promote tumor escape as an NK cells immune checkpoint. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=66 SRC=\"FIGDIR/small/567708v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (20K): org.highwire.dtl.DTLVardef@dbe221org.highwire.dtl.DTLVardef@1db1f70org.highwire.dtl.DTLVardef@ed13d7org.highwire.dtl.DTLVardef@b19509_HPS_FORMAT_FIGEXP M_FIG C_FIG HighlightsO_LICancer-conditioned medium suppresses the antitumor activity of NK cells C_LIO_LIAmmonia accumulates in conditioned medium and in the tumor microenvironment C_LIO_LIImpaired cytotoxicity of NK cells is caused by ammonia that decreases perforin levels C_LIO_LIAmmonia causes NK cell dysfunction C_LI\nNext-Gen Profiling of Tumor-resident Stem Cells using Machine learning\nAuthors: Chowdhury, D.; Neekhra, B.; Priyadarshi, S.; Mukherjee, S.; Maity, D.; Gupta, D.; Haldar, S.\nScore: 3.8, Published: 2023-11-25 DOI: 10.1101/2023.11.24.568600\nTumor-resident stem cells, also known as cancer stem cells (CSCs), constitute a subgroup within tumors, play a crucial role in fostering resistance to treatment and the recurrence of tumors, and pose significant challenges for conventional therapeutic methods. Existing approaches for identifying CSCs face notable hurdles related to scalability, reproducibility, and technical consistency across different cancer types due to the adaptable nature of CSCs. In this context, we introduce OSCORP, an innovative machine-learning-driven approach. This methodology quantifies and identifies CSCs, achieving almost 99% accuracy using biopsy bulk RNAseq data. OSCORP leverages genetic similarities between normal and cancer stem cells. By categorizing CSCs into four distinct yet dynamic potency states, this approach provides insights into the differentiation landscape of CSCs, unveiling previously undisclosed facets of tumor heterogeneity. In evaluations conducted on patient samples across 22 cancer types, OSCORP revealed clinical, transcriptomic, and immunological signatures associated with each CSC state. It emerges as a comprehensive tool for understanding and addressing the complexities of cancer stem cells. Ultimately, OSCORP opens up new possibilities for more effective personalized cancer therapies and holds the potential to serve as a clinical tool for monitoring patient-specific CSC changes during treatment or follow-up care.\nRedefining phenotypic intratumor heterogeneity of pancreatic ductal adenocarcinoma: a bottom-up approach\nAuthors: Hilmi, M.; Delecourt, F.; Raffenne, J.; Bourega, T.; Dusetti, N. J.; Iovanna, J.; Blum, Y.; Richard, M.; Neuzillet, C.; Couvelard, A.; De Mestier, L.; Rebours, V.; Nicolle, R.; Cros, J.\nScore: 3.9, Published: 2023-11-18 DOI: 10.1101/2023.11.16.567454\nBackgroundPancreatic ductal adenocarcinoma (PDAC) tumor inter-patient heterogeneity has been well described with two major prognostic subtypes (classical and basal-like). An important intra-patient heterogeneity has been reported but has not yet been extensively studied due to the lack of standardized, reproducible and easily accessible high throughput methods. Material and MethodsWe built an immunohistochemical (IHC) tool capable of differentiating RNA-defined classical and basal-like tumors by selecting relevant antibodies using a multi-step process. The successive stages of i) an in-silico selection from a review literature and a bulk transcriptome analysis of 309 PDACs, ii) a tumor-specific selection from 30 patient-derived xenografts followed by iii) the validation on tissue microarrays in 50 PDAC were conducted. We used our final IHC panel on two independent cohorts of resected PDAC (n=95, whole-slide, n=148, tissue microarrays) for external validation. After digitization and registration of pathology slides, we performed a tile-based-analysis in tumor and pre-neoplastic epithelial areas and a k-means clustering to identify relevant marker combinations. ResultsSequential marker selection led to the following panel: GATA6, CLDN18, TFF1, MUC16, S100A2, KRT17, PanBasal. Four different phenotypes were identified: 1 classical, 1 intermediate (KRT17+) and 2 basal-like (MUC16+ vs S100A2+) with specific biological properties. The presence of a minor basal contingent drastically reduced overall survival, even in classical predominant PDACs (HR=2.36, p=0.01). Analysis of preneoplastic lesions suggested that pancreatic carcinogenesis may follow a progressive evolution from classical toward a basal through an early intermediate phenotype. ConclusionOur IHC panel redefined and easily assessed the high degree of intra- and inter-tumoral heterogeneity of PDAC.\nNK cells shape the clonal evolution of B-ALL cells by IFN-γ production\nAuthors: Buri, M. C.; Shoeb, M. R.; Bykov, A.; Repiscak, P.; Baik, H.; Dupanovic, A.; David, F. O.; Kovacic, B.; Hall-Glenn, F.; Urbanus, J.; Sippl, L.; Stofner, S.; Emminger, D.; Cosgrove, J.; Lehner, M.; Perie, L.; Shumacher, T. N.; Gotthardt, D.; Halbritter, F.; Putz, E. M.\nScore: 4.8, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567430\nThe term cancer immunoediting describes the dual role by which the immune system can both suppress and promote tumour growth and is divided into three phases: elimination, equilibrium and escape. While the importance of CD8+ T cells in all three phases of cancer immunoediting is well-established, the role of NK cells has mainly been attributed to the elimination phase. Here we show that NK cells play a prominent role in all three phases of cancer immunoediting. Extended co-culturing of DNA barcoded mouse BCR/ABLp185+ B acute lymphoblastic leukaemia cells with NK cells allowed for a quantitative measure of NK cell-mediated immunoediting in vitro. Whereas most tumour cell clones were efficiently eliminated by NK cells, a certain fraction of tumour cells harboured an intrinsic resistance. Furthermore, DNA barcoding revealed tumour cell clones with secondary resistance, which stochastically acquiring resistance to NK cells. We found that the production of interferon-{gamma} (IFN-{gamma}) rather than direct cytotoxicity by NK cells, led to the emergence of highly resistant tumour cells. Besides the well-known upregulation of components involved in major histocompatibility complex I surface expression, we also found novel genes that had not previously been linked to NK cell resistance in leukaemic cells, such as Ly6a and Plaat3. Our results demonstrate that tumour cells are actively edited during the equilibrium phase and use different avenues to escape NK cell-mediated eradication. Our findings provide a starting point to develop novel therapeutic approaches to treat tumours that have successfully evaded from NK cell surveillance and to convert them into a controllable disease.\nLess is More: Nek2A Unclusters Extra Centrosomes and Induces Cell Death in Cancer Cells via KIF2C Interaction\nAuthors: Kalkan, B. M.; Ozcan, S. C.; Cicek, E.; Gonen, M.; Acilan, C.\nScore: 2.9, Published: 2023-11-21 DOI: 10.1101/2023.11.21.567992\nUnlike normal cells, cancer cells frequently exhibit extra centrosomes, leading to formation of multipolar spindles that can trigger cell death. Nevertheless, they manage to divide successfully and escape the deadly consequences of unequal segregation of genomic material by coalescing their extra centrosomes into two poles. This unique trait of cancer cells presents a promising target for cancer therapy, focusing on selectively attacking cells with supernumerary centrosomes. Nek2A is a kinase involved in mitotic regulation, including the centrosome cycle, where it phosphorylates linker proteins to separate centrosomes. In this study, we investigated if Nek2A also unclusters extra centrosomes, akin to its separation function. Reduction of Nek2A activity, achieved through knockout, silencing, or inhibition, promotes centrosome clustering, whereas its overexpression results in unclustering. Significantly, this unclustering activity induces cell death, but only in cancer cells with extra centrosomes, both in vitro and in vivo. Notably, none of the known centrosomal (e.g., CNAP1, Rootletin, Gas2L1) or non-centrosomal (e.g., TRF1, HEC1) Nek2A targets were implicated in this unclustering activity. Additionally, Nek2A operated via a mechanism distinct from other unclustering factors like HSET and NuMA. Through TurboID proximity labeling analysis, we identified novel proteins associated with the centrosome or microtubules, expanding the known interaction partners of Nek2A. KIF2C, in particular, emerged as a novel interactor, confirmed through coimmunoprecipitation and localization analysis. The silencing of KIF2C diminished the impact of Nek2A on centrosome unclustering and rescued cell viability. Additionally, elevated Nek2A levels were indicative of better patient outcomes, specifically in those predicted to have excess centrosomes. Therefore, while Nek2A is a proposed target, its use must be specifically adapted to the broader cellular context, especially considering centrosome amplification. Discovering partners such as KIF2C offers fresh insights into cancer biology and new possibilities for targeted treatment.\nA nonenzymatic dependency on inositol-requiring enzyme 1 controls cancer cell cycle progression and tumor growth\nAuthors: Zuazo-Gaztelu, I.; Lawrence, D.; Oikonomidi, I.; Marsters, S.; Pechuan-Jorge, X.; Gaspar, C. J.; Kan, D.; Segal, E.; Clark, K.; Beresini, M.; Braun, M.-G.; Rudolph, J.; Modrusan, Z.; Choi, M.; Sandoval, W.; Reichelt, M.; Kujala, P.; van Dijk, S.; Klumperman, J.; Ashkenazi, A.\nScore: 2.8, Published: 2023-11-23 DOI: 10.1101/2023.11.22.567905\nEndoplasmic-reticulum resident inositol-requiring enzyme 1 (IRE1) supports protein homeostasis via a cytoplasmic kinase-RNase module. Known cancer dependency on IRE1 entails its enzymatic activation of the transcription factor XBP1s and of RNA decay. We discovered that some cancer cells require IRE1 but not its enzymatic activity. IRE1 knockdown, but not enzymatic inhibition or XBP1 disruption, increased DNA damage and chromosome instability while engaging the TP53 pathway and cyclin-dependent kinase inhibitors and attenuating cell cycle progression. IRE1 depletion downregulated factors involved in chromosome replication and segregation and in chromatin remodeling. Immunoelectron microscopy indicated that endogenous IRE1 can localize to the nuclear envelope. Thus, cancer cells can require IRE1 either enzymatically or nonenzymatically, with significant implications for IRE1s biological role and therapeutic targeting.\n",
  "wordCount" : "2712",
  "inLanguage": "en",
  "datePublished": "2023-11-26T10:37:33Z",
  "dateModified": "2023-11-26T10:37:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 26, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568355">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568355" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568355">
        <p class="paperTitle">Cancer-associated fibroblasts serve as decoys to suppress NK cell anti-cancer cytotoxicity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568355" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568355" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ben-Shmuel, A.; Gruper, Y.; Levi-Galibov, O.; Rosenberg-Fogler, H.; Carradori, G.; Stein, Y.; Dadiani, M.; Naumova, M.; Nevo, R.; Morzaev-Sulzbach, D.; Yagel, G.; Mayer, S.; Gal-Yam, E.; Scherz-Shouval, R.</p>
        <p class="info">Score: 17.2, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568355' target='https://doi.org/10.1101/2023.11.23.568355'> 10.1101/2023.11.23.568355</a></p>
        <p class="abstract">Cancer associated fibroblasts (CAFs) are among the most abundant components of the breast tumor microenvironment (TME) and major contributors to immune modulation. CAFs are well-known to regulate the activity of diverse types of immune cells including T cells, macrophages and dendritic cells, however little is known about their interaction with Natural killer (NK) cells, which constitute an important arm of anti-tumor immunity. Here we find, using mouse models of cancer and ex-vivo co-cultures, that CAFs inhibit NK cell cytotoxicity towards cancer cells. We unravel the mechanism by which this suppression occurs, through ligand-receptor engagement between NK cells and CAFs leading to CAF cytolysis, which in turn diminishes the expression of activating receptors on NK cells, promoting cancer escape from NK cell surveillance. Analysis of breast cancer patient samples reveals enrichment of NK cells in CAF-rich regions, and upregulation of NK binding ligands on CAFs which is correlated with poor disease outcome. These results reveal a CAF-mediated immunosuppressive decoy mechanism with implications for treatment of solid tumors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.568019">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.568019" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.568019">
        <p class="paperTitle">Single-molecule systems for detection and monitoring of plasma circulating nucleosomes and oncoproteins in Diffuse Midline Glioma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.568019" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.568019" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Erez, N.; Furth, N.; Fedyuk, V.; Wadden, J.; Aittaleb, R.; Schwark, K.; Niculcea, M.; Miclea, M.; Mody, R.; Franson, A.; Eze, A.; Nourmohammadi, N.; Nazarian, J.; Venneti, S.; Koschmann, C.; Shema, E.</p>
        <p class="info">Score: 8.8, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.568019' target='https://doi.org/10.1101/2023.11.21.568019'> 10.1101/2023.11.21.568019</a></p>
        <p class="abstract">The analysis of cell-free tumor DNA (ctDNA) and proteins in the blood of cancer patients potentiates a new generation of non-invasive diagnostics and treatment monitoring approaches. However, confident detection of these tumor-originating markers is challenging, especially in the context of brain tumors, in which extremely low amounts of these analytes circulate in the patients plasma. Here, we applied a sensitive single-molecule technology to profile multiple histone modifications on millions of individual nucleosomes from the plasma of Diffuse Midline Glioma (DMG) patients. The system reveals epigenetic patterns that are unique to DMG, significantly differentiating this group of patients from healthy subjects or individuals diagnosed with other cancer types. We further develop a method to directly capture and quantify the tumor-originating oncoproteins, H3-K27M and mutant p53, from the plasma of children diagnosed with DMG. This single-molecule system allows for accurate molecular classification of patients, utilizing less than 1ml of liquid-biopsy material. Furthermore, we show that our simple and rapid detection strategy correlates with MRI measurements and droplet-digital PCR (ddPCR) measurements of ctDNA, highlighting the utility of this approach for non-invasive treatment monitoring of DMG patients. This work underscores the clinical potential of single-molecule-based, multi-parametric assays for DMG diagnosis and treatment monitoring.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568427">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568427" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568427">
        <p class="paperTitle">Biocompatibility characterisation of CMOS-based Lab-on-Chip electrochemical sensors for in vitro cancer cell culture applications</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568427" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568427" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beykou, M.; Bousgouni, V.; Moser, N.; Georgiou, P.; Bakal, C.</p>
        <p class="info">Score: 8.6, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568427' target='https://doi.org/10.1101/2023.11.23.568427'> 10.1101/2023.11.23.568427</a></p>
        <p class="abstract">Lab-on-Chip electrochemical sensors, such as Ion-Sensitive Field-Effect Transistors (ISFETs), are being developed for use in point-of-care diagnostics, such as pH detection of tumour microenvironments, due to their integration with standard Complementary Metal Oxide Semiconductor technology. With this approach, the passivation of the CMOS process is used as a sensing layer to minimise post-processing, and Silicon Nitride (Si3N4) is the most common material at the microchip surface. ISFETs have the potential to be used for cell-based assays however, there is a poor understanding of the biocompatibility of microchip surfaces. Here, we quantitatively evaluated cell adhesion, morphogenesis, proliferation and mechano-responsiveness of both normal and cancer cells cultured on a Si3N4, sensor surface. We demonstrate that both normal and cancer cell adhesion decreased on Si3N4. Activation of the mechano-responsive transcription regulators, YAP/TAZ, are significantly decreased in cancer cells on Si3N4 in comparison to standard cell culture plastic, whilst proliferation marker, Ki67, expression markedly increased. Non-tumorigenic cells on chip showed less sensitivity to culture on Si3N4 than cancer cells. Treatment with extracellular matrix components increased cell adhesion in normal and cancer cell cultures, surpassing the adhesiveness of plastic alone. Moreover, poly-l-ornithine and laminin treatment restored YAP/TAZ levels in both non-tumorigenic and cancer cells to levels comparable to those observed on plastic. Thus, engineering the electrochemical sensor surface with treatments will provide a more physiologically relevant environment for future cell-based assay development on chip.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567140">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567140" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567140">
        <p class="paperTitle">Epigenetic heritability of cell plasticity drives cancer drug resistance through one-to-many genotype to phenotype mapping</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567140" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567140" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fernandez-Mateos, J.; Milite, S.; Oliveira, E.; Vlachogiannis, G.; Chen, B.; Yara, E.; Cresswell, G. D.; James, C.; Patruno, L.; Ascolani, G.; Acar, A.; Heide, T.; Spiteri, I.; Graudenzi, A.; Caravagna, G.; Graham, T.; Magnani, L.; Valeri, N.; Sottoriva, A.</p>
        <p class="info">Score: 43.8, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567140' target='https://doi.org/10.1101/2023.11.15.567140'> 10.1101/2023.11.15.567140</a></p>
        <p class="abstract">Drug resistance is a largely unsolved problem in oncology. Despite the explanatory power of the genetic model of cancer initiation, most treatment resistance is unexplained by genetics alone. Even when known resistance mutations are present, they are often found in a small proportion of the cells in the tumour. So where is the cellular memory that leads to treatment failure? New evidence suggests resistance is multi-factorial, resulting from the contribution of heritable genetic and epigenetic changes, but also non-heritable phenotypic plasticity. However, cell plasticity has proven hard to study as it dynamically changes over time and needs to be distinguished from clonal evolution where cell phenotypes change because of Darwinian selective bottlenecks. Here we dissected the contribution of different evolutionary processes to drug resistance by perturbing patient-derived organoids with multiple drugs in sequence. We combined dense longitudinal tracking, single cell multi- omics, evolutionary modelling, and machine learning archetypal analysis. We found that different drugs select for distinct subclones, an essential requirement for the use of evolutionary therapy with sequential drug treatment. The data supports a model in which the cellular memory is encoded as a heritable configuration of the epigenome, which however produces multiple transcriptional programmes. Those emerge in different proportions depending on the environment, giving rise to cellular plasticity. Epigenetically encoded programmes include reactivation of developmental genes and cell regeneration. A one-to-many (epi)genotype[-&amp;gt;]phenotype map explains how clonal expansions and non- heritable phenotypic plasticity manifest together, including drug tolerant states. This ensures the robustness of drug resistance subclones that can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding their selective advantage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.20.567708">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.20.567708" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.20.567708">
        <p class="paperTitle">Ammonia inhibits antitumor activity of NK cells by decreasing mature perforin</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.20.567708" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.20.567708" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Winiarska, M.; Domagala, J.; Grzywa, T. M.; Baranowska, I.; Kusowska, A.; Fidyt, K.; Marhelava, K.; Pilch, Z.; Graczyk-Jarzynka, A.; Picard, L.; Jastrzebski, K.; Granica, M.; Justyniarska, M.; Urlaub, D.; Bobrowicz, M.; Miaczynska, M.; Watzl, C.</p>
        <p class="info">Score: 4.1, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.20.567708' target='https://doi.org/10.1101/2023.11.20.567708'> 10.1101/2023.11.20.567708</a></p>
        <p class="abstract">Immunotherapy revolutionized cancer treatment in the last decade. Natural killer (NK) cells are one of the key host immunity components against malignant cells. Thus, they are currently extensively investigated in the field of immunotherapy of cancer. Different approaches have been developed to improve the antitumor activity of NK cells. Nonetheless, tumor microenvironment remains an obstacle to effective NK cell-based therapies. Here, we demonstrated that a cancer-conditioned medium suppresses the anti-tumor activity of NK cells. Further, we found that ammonia, a by-product of cancer cell metabolism, accumulates in the cancer-conditioned medium and tumor microenvironment. We identified that ammonia impairs the cytotoxicity of NK cells as well as the effectiveness of antibody-based and chimeric antigen receptor (CAR)-NK-based therapies in vitro. Inhibited activity of NK cells was caused by decreased levels of perforin. This effect was dependent on the lysosomotropic features of ammonia and its ability to increase pH in acidic compartments. In consequence, upon contact with ammonia the mature form of perforin was decreased in NK cells leading to their dysfunction. Our findings demonstrate that in addition to its previously described role of promoting tumor growth as a nitrogen source for tumor biomass ammonia could promote tumor escape as an NK cells immune checkpoint.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=66 SRC=&#34;FIGDIR/small/567708v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (20K):
org.highwire.dtl.DTLVardef@dbe221org.highwire.dtl.DTLVardef@1db1f70org.highwire.dtl.DTLVardef@ed13d7org.highwire.dtl.DTLVardef@b19509_HPS_FORMAT_FIGEXP  M_FIG C_FIG HighlightsO_LICancer-conditioned medium suppresses the antitumor activity of NK cells
C_LIO_LIAmmonia accumulates in conditioned medium and in the tumor microenvironment
C_LIO_LIImpaired cytotoxicity of NK cells is caused by ammonia that decreases perforin levels
C_LIO_LIAmmonia causes NK cell dysfunction
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568600">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568600" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568600">
        <p class="paperTitle">Next-Gen Profiling of Tumor-resident Stem Cells using Machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568600" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568600" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chowdhury, D.; Neekhra, B.; Priyadarshi, S.; Mukherjee, S.; Maity, D.; Gupta, D.; Haldar, S.</p>
        <p class="info">Score: 3.8, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568600' target='https://doi.org/10.1101/2023.11.24.568600'> 10.1101/2023.11.24.568600</a></p>
        <p class="abstract">Tumor-resident stem cells, also known as cancer stem cells (CSCs), constitute a subgroup within tumors, play a crucial role in fostering resistance to treatment and the recurrence of tumors, and pose significant challenges for conventional therapeutic methods. Existing approaches for identifying CSCs face notable hurdles related to scalability, reproducibility, and technical consistency across different cancer types due to the adaptable nature of CSCs. In this context, we introduce OSCORP, an innovative machine-learning-driven approach. This methodology quantifies and identifies CSCs, achieving almost 99% accuracy using biopsy bulk RNAseq data. OSCORP leverages genetic similarities between normal and cancer stem cells. By categorizing CSCs into four distinct yet dynamic potency states, this approach provides insights into the differentiation landscape of CSCs, unveiling previously undisclosed facets of tumor heterogeneity. In evaluations conducted on patient samples across 22 cancer types, OSCORP revealed clinical, transcriptomic, and immunological signatures associated with each CSC state. It emerges as a comprehensive tool for understanding and addressing the complexities of cancer stem cells. Ultimately, OSCORP opens up new possibilities for more effective personalized cancer therapies and holds the potential to serve as a clinical tool for monitoring patient-specific CSC changes during treatment or follow-up care.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567454">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567454" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567454">
        <p class="paperTitle">Redefining phenotypic intratumor heterogeneity of pancreatic ductal adenocarcinoma: a bottom-up approach</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567454" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567454" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hilmi, M.; Delecourt, F.; Raffenne, J.; Bourega, T.; Dusetti, N. J.; Iovanna, J.; Blum, Y.; Richard, M.; Neuzillet, C.; Couvelard, A.; De Mestier, L.; Rebours, V.; Nicolle, R.; Cros, J.</p>
        <p class="info">Score: 3.9, Published: 2023-11-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567454' target='https://doi.org/10.1101/2023.11.16.567454'> 10.1101/2023.11.16.567454</a></p>
        <p class="abstract">BackgroundPancreatic ductal adenocarcinoma (PDAC) tumor inter-patient heterogeneity has been well described with two major prognostic subtypes (classical and basal-like). An important intra-patient heterogeneity has been reported but has not yet been extensively studied due to the lack of standardized, reproducible and easily accessible high throughput methods.

Material and MethodsWe built an immunohistochemical (IHC) tool capable of differentiating RNA-defined classical and basal-like tumors by selecting relevant antibodies using a multi-step process. The successive stages of i) an in-silico selection from a review literature and a bulk transcriptome analysis of 309 PDACs, ii) a tumor-specific selection from 30 patient-derived xenografts followed by iii) the validation on tissue microarrays in 50 PDAC were conducted. We used our final IHC panel on two independent cohorts of resected PDAC (n=95, whole-slide, n=148, tissue microarrays) for external validation. After digitization and registration of pathology slides, we performed a tile-based-analysis in tumor and pre-neoplastic epithelial areas and a k-means clustering to identify relevant marker combinations.

ResultsSequential marker selection led to the following panel: GATA6, CLDN18, TFF1, MUC16, S100A2, KRT17, PanBasal. Four different phenotypes were identified: 1 classical, 1 intermediate (KRT17&#43;) and 2 basal-like (MUC16&#43; vs S100A2&#43;) with specific biological properties. The presence of a minor basal contingent drastically reduced overall survival, even in classical predominant PDACs (HR=2.36, p=0.01). Analysis of preneoplastic lesions suggested that pancreatic carcinogenesis may follow a progressive evolution from classical toward a basal through an early intermediate phenotype.

ConclusionOur IHC panel redefined and easily assessed the high degree of intra- and inter-tumoral heterogeneity of PDAC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567430">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567430" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567430">
        <p class="paperTitle">NK cells shape the clonal evolution of B-ALL cells by IFN-γ production</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567430" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567430" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buri, M. C.; Shoeb, M. R.; Bykov, A.; Repiscak, P.; Baik, H.; Dupanovic, A.; David, F. O.; Kovacic, B.; Hall-Glenn, F.; Urbanus, J.; Sippl, L.; Stofner, S.; Emminger, D.; Cosgrove, J.; Lehner, M.; Perie, L.; Shumacher, T. N.; Gotthardt, D.; Halbritter, F.; Putz, E. M.</p>
        <p class="info">Score: 4.8, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567430' target='https://doi.org/10.1101/2023.11.16.567430'> 10.1101/2023.11.16.567430</a></p>
        <p class="abstract">The term cancer immunoediting describes the dual role by which the immune system can both suppress and promote tumour growth and is divided into three phases: elimination, equilibrium and escape. While the importance of CD8&#43; T cells in all three phases of cancer immunoediting is well-established, the role of NK cells has mainly been attributed to the elimination phase. Here we show that NK cells play a prominent role in all three phases of cancer immunoediting. Extended co-culturing of DNA barcoded mouse BCR/ABLp185&#43; B acute lymphoblastic leukaemia cells with NK cells allowed for a quantitative measure of NK cell-mediated immunoediting in vitro. Whereas most tumour cell clones were efficiently eliminated by NK cells, a certain fraction of tumour cells harboured an intrinsic resistance. Furthermore, DNA barcoding revealed tumour cell clones with secondary resistance, which stochastically acquiring resistance to NK cells. We found that the production of interferon-{gamma} (IFN-{gamma}) rather than direct cytotoxicity by NK cells, led to the emergence of highly resistant tumour cells. Besides the well-known upregulation of components involved in major histocompatibility complex I surface expression, we also found novel genes that had not previously been linked to NK cell resistance in leukaemic cells, such as Ly6a and Plaat3. Our results demonstrate that tumour cells are actively edited during the equilibrium phase and use different avenues to escape NK cell-mediated eradication. Our findings provide a starting point to develop novel therapeutic approaches to treat tumours that have successfully evaded from NK cell surveillance and to convert them into a controllable disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.21.567992">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.21.567992" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.21.567992">
        <p class="paperTitle">Less is More: Nek2A Unclusters Extra Centrosomes and Induces Cell Death in Cancer Cells via KIF2C Interaction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.21.567992" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.21.567992" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kalkan, B. M.; Ozcan, S. C.; Cicek, E.; Gonen, M.; Acilan, C.</p>
        <p class="info">Score: 2.9, Published: 2023-11-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.21.567992' target='https://doi.org/10.1101/2023.11.21.567992'> 10.1101/2023.11.21.567992</a></p>
        <p class="abstract">Unlike normal cells, cancer cells frequently exhibit extra centrosomes, leading to formation of multipolar spindles that can trigger cell death. Nevertheless, they manage to divide successfully and escape the deadly consequences of unequal segregation of genomic material by coalescing their extra centrosomes into two poles. This unique trait of cancer cells presents a promising target for cancer therapy, focusing on selectively attacking cells with supernumerary centrosomes.

Nek2A is a kinase involved in mitotic regulation, including the centrosome cycle, where it phosphorylates linker proteins to separate centrosomes. In this study, we investigated if Nek2A also unclusters extra centrosomes, akin to its separation function. Reduction of Nek2A activity, achieved through knockout, silencing, or inhibition, promotes centrosome clustering, whereas its overexpression results in unclustering. Significantly, this unclustering activity induces cell death, but only in cancer cells with extra centrosomes, both in vitro and in vivo. Notably, none of the known centrosomal (e.g., CNAP1, Rootletin, Gas2L1) or non-centrosomal (e.g., TRF1, HEC1) Nek2A targets were implicated in this unclustering activity. Additionally, Nek2A operated via a mechanism distinct from other unclustering factors like HSET and NuMA.

Through TurboID proximity labeling analysis, we identified novel proteins associated with the centrosome or microtubules, expanding the known interaction partners of Nek2A. KIF2C, in particular, emerged as a novel interactor, confirmed through coimmunoprecipitation and localization analysis. The silencing of KIF2C diminished the impact of Nek2A on centrosome unclustering and rescued cell viability. Additionally, elevated Nek2A levels were indicative of better patient outcomes, specifically in those predicted to have excess centrosomes. Therefore, while Nek2A is a proposed target, its use must be specifically adapted to the broader cellular context, especially considering centrosome amplification. Discovering partners such as KIF2C offers fresh insights into cancer biology and new possibilities for targeted treatment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.22.567905">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.22.567905" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.22.567905">
        <p class="paperTitle">A nonenzymatic dependency on inositol-requiring enzyme 1 controls cancer cell cycle progression and tumor growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.22.567905" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.22.567905" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zuazo-Gaztelu, I.; Lawrence, D.; Oikonomidi, I.; Marsters, S.; Pechuan-Jorge, X.; Gaspar, C. J.; Kan, D.; Segal, E.; Clark, K.; Beresini, M.; Braun, M.-G.; Rudolph, J.; Modrusan, Z.; Choi, M.; Sandoval, W.; Reichelt, M.; Kujala, P.; van Dijk, S.; Klumperman, J.; Ashkenazi, A.</p>
        <p class="info">Score: 2.8, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.22.567905' target='https://doi.org/10.1101/2023.11.22.567905'> 10.1101/2023.11.22.567905</a></p>
        <p class="abstract">Endoplasmic-reticulum resident inositol-requiring enzyme 1 (IRE1) supports protein homeostasis via a cytoplasmic kinase-RNase module. Known cancer dependency on IRE1 entails its enzymatic activation of the transcription factor XBP1s and of RNA decay. We discovered that some cancer cells require IRE1 but not its enzymatic activity. IRE1 knockdown, but not enzymatic inhibition or XBP1 disruption, increased DNA damage and chromosome instability while engaging the TP53 pathway and cyclin-dependent kinase inhibitors and attenuating cell cycle progression. IRE1 depletion downregulated factors involved in chromosome replication and segregation and in chromatin remodeling. Immunoelectron microscopy indicated that endogenous IRE1 can localize to the nuclear envelope. Thus, cancer cells can require IRE1 either enzymatically or nonenzymatically, with significant implications for IRE1s biological role and therapeutic targeting.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
