<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep intravital brain tumor imaging enabled by tailored three-photon microscopy and analysis
Authors: Schubert, M. C.; Soyka, S. J.; Tamimi, A.; Maus, E.; Denninger, R.; Wissmann, N.; Reyhan, E.; Tetzlaff, S. K.; Beretta, C.; Drumm, M.; Schroers, J.; Steffens, A.; Walshon, J.; McCortney, K.; Heiland, S.; Golebiewska, A.; Kurz, F. T.; Wick, W.; Winkler, F.; Kreshuk, A.; Kuner, T.; Horbinski, C.; Prevedel, R.; Venkataramani, V.
Score: 48.2, Published: 2023-06-21 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Deep intravital brain tumor imaging enabled by tailored three-photon microscopy and analysis
Authors: Schubert, M. C.; Soyka, S. J.; Tamimi, A.; Maus, E.; Denninger, R.; Wissmann, N.; Reyhan, E.; Tetzlaff, S. K.; Beretta, C.; Drumm, M.; Schroers, J.; Steffens, A.; Walshon, J.; McCortney, K.; Heiland, S.; Golebiewska, A.; Kurz, F. T.; Wick, W.; Winkler, F.; Kreshuk, A.; Kuner, T.; Horbinski, C.; Prevedel, R.; Venkataramani, V.
Score: 48.2, Published: 2023-06-21 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-02T10:39:32+00:00" />
<meta property="article:modified_time" content="2023-07-02T10:39:32+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Deep intravital brain tumor imaging enabled by tailored three-photon microscopy and analysis
Authors: Schubert, M. C.; Soyka, S. J.; Tamimi, A.; Maus, E.; Denninger, R.; Wissmann, N.; Reyhan, E.; Tetzlaff, S. K.; Beretta, C.; Drumm, M.; Schroers, J.; Steffens, A.; Walshon, J.; McCortney, K.; Heiland, S.; Golebiewska, A.; Kurz, F. T.; Wick, W.; Winkler, F.; Kreshuk, A.; Kuner, T.; Horbinski, C.; Prevedel, R.; Venkataramani, V.
Score: 48.2, Published: 2023-06-21 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Deep intravital brain tumor imaging enabled by tailored three-photon microscopy and analysis\nAuthors: Schubert, M. C.; Soyka, S. J.; Tamimi, A.; Maus, E.; Denninger, R.; Wissmann, N.; Reyhan, E.; Tetzlaff, S. K.; Beretta, C.; Drumm, M.; Schroers, J.; Steffens, A.; Walshon, J.; McCortney, K.; Heiland, S.; Golebiewska, A.; Kurz, F. T.; Wick, W.; Winkler, F.; Kreshuk, A.; Kuner, T.; Horbinski, C.; Prevedel, R.; Venkataramani, V.\nScore: 48.2, Published: 2023-06-21 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Deep intravital brain tumor imaging enabled by tailored three-photon microscopy and analysis\nAuthors: Schubert, M. C.; Soyka, S. J.; Tamimi, A.; Maus, E.; Denninger, R.; Wissmann, N.; Reyhan, E.; Tetzlaff, S. K.; Beretta, C.; Drumm, M.; Schroers, J.; Steffens, A.; Walshon, J.; McCortney, K.; Heiland, S.; Golebiewska, A.; Kurz, F. T.; Wick, W.; Winkler, F.; Kreshuk, A.; Kuner, T.; Horbinski, C.; Prevedel, R.; Venkataramani, V.\nScore: 48.2, Published: 2023-06-21 DOI: 10.1101/2023.06.17.545350\nIntravital two-photon microscopy has emerged as a powerful technology to study brain tumor biology and its temporal dynamics, including invasion, proliferation and therapeutic resistance in the superficial layers of the mouse cortex. However, intravital microscopy of deeper cortical layers and especially the subcortical white matter, an important route of glioblastoma invasion and recurrence, has not yet been feasible due to low signal-to-noise ratios, missing spatiotemporal resolution and the inability to delineate myelinated axonal tracts. Here, we present a tailored intravital microscopy and artificial intelligence-based analysis methodology and workflow that enables routine deep imaging of glioblastoma over extended time periods, named Deep3P. We show that three-photon microscopy, adaptive optics, as well as customized deep learning-based denoising and machine learning segmentation together allow for deep brain intravital investigation of tumor biology up to 1.2 mm depth. Leveraging this approach, we find that perivascular invasion is a preferred invasion route into the corpus callosum as compared to intracortical glioblastoma invasion and uncover two vascular mechanisms of glioblastoma migration in the white matter. Furthermore, we can define an imaging biomarker of white matter disruption during early glioblastoma colonization. Taken together, Deep3P allows for an efficient and non-invasive investigation of brain tumor biology and its tumor microenvironment in unprecedented deep white and gray matter of the living mouse, opening up novel opportunities for studying the neuroscience of brain tumors and other model systems.\nImmortalization and transformation of primary cells mediated by engineered ecDNAs\nAuthors: Pradella, D.; Zhang, M.; Gao, R.; Melissa, Y. A.; Gluchowska, K. M.; Cendon Florez, Y.; Mishra, T.; La Rocca, G.; Weigl, M.; Jiao, Z.; Nguyen, H. H. M.; Grimm, F.; Lisi, M.; Mastroleo, C.; Chen, K.; Luebeck, J.; Bafna, V.; Antonescu, C. R.; Ventura, A.\nScore: 41.2, Published: 2023-06-26 DOI: 10.1101/2023.06.25.546239\nFocal gene amplifications are among the most common cancer-associated mutations, but their evolution and contribution to tumorigenesis have proven challenging to recapitulate in primary cells and model organisms. Here we describe a general approach to engineer large (\u003e1 Mbp) focal amplifications mediated by extrachromosomal circular DNAs (ecDNAs, also known as \"double minutes\") in a spatiotemporally controlled manner in cancer cell lines and in primary cells derived from genetically engineered mice. With this strategy, ecDNA formation can be coupled with expression of fluorescent reporters or other selectable markers to enable the identification and tracking of ecDNA-containing cells. We demonstrate the feasibility of this approach by engineering MDM2-containing ecDNAs in near-diploid human cells, showing that GFP expression can be used to track ecDNA dynamics under physiological conditions or in the presence of specific selective pressures. We also apply this approach to generate mice harboring inducible Myc- and Mdm2-containing ecDNAs analogous to those spontaneously occurring in human cancers. We show that the engineered ecDNAs rapidly accumulate in primary cells derived from these animals, promoting proliferation, immortalization, and transformation.\nMAPS: Pathologist-level cell type annotation from tissue images through machine learning\nAuthors: Shaban, M.; Bai, Y.; Qiu, H.; Mao, S.; Yeung, J.; Yeo, Y. Y.; Shanmugam, V.; Shanmugam, V.; Chen, H.; Zhu, B.; Nolan, G. P.; Shipp, M. A.; Rodig, S. J.; Jiang, S.; Mahmood, F.\nScore: 24.7, Published: 2023-06-27 DOI: 10.1101/2023.06.25.546474\nHighly multiplexed protein imaging is emerging as a potent technique for analyzing protein distribution within cells and tissues in their native context. However, existing cell annotation methods utilizing high-plex spatial proteomics data are resource intensive and necessitate iterative expert input, thereby constraining their scalability and practicality for extensive datasets. We introduce MAPS (Machine learning for Analysis of Proteomics in Spatial biology), a machine learning approach facilitating rapid and precise cell type identification with human-level accuracy from spatial proteomics data. Validated on multiple in-house and publicly available MIBI and CODEX datasets, MAPS outperforms current annotation techniques in terms of speed and accuracy, achieving pathologist-level precision even for challenging cell types, including tumor cells of immune origin. By democratizing rapidly deployable and scalable machine learning annotation, MAPS holds significant potential to expedite advances in tissue biology and disease comprehension.\nEpigenomic analysis of Formalin-Fixed Paraffin-Embedded samples by CUT\u0026Tag\nAuthors: Henikoff, S.; Henikoff, J. G.; Ahmad, K.; Paranal, R. M.; Janssens, D. H.; Russell, Z. R.; Szulzewsky, F.; Kugel, S.; Holland, E. C.\nScore: 26.0, Published: 2023-06-22 DOI: 10.1101/2023.06.20.545743\nFor more than a century, Formalin Fixed Paraffin Embedded (FFPE) sample preparation has been the preferred method for long-term preservation of biological material. However, the use of FFPE samples for epigenomic studies has been difficult because of chromatin damage from long exposure to high concentrations of formaldehyde. Previously, we introduced Cleavage Under Targeted Accessible Chromatin (CUTAC), an antibody-targeted chromatin accessibility mapping protocol based on CUT\u0026Tag. Here we show that simple modifications of our single-tube CUTAC protocol are sufficient to produce high-resolution maps of paused RNA Polymerase II (RNAPII) at enhancers and promoters using FFPE samples. We find that transcriptional regulatory element differences produced by FFPE-CUTAC distinguish between mouse brain tumor specimens and identify regulatory element markers with high confidence and precision. Our simple work-flow is suitable for automation, making possible affordable epigenomic profiling of archived biological samples for biomarker identification, clinical applications and retrospective studies.\nTherapy-associated remodeling of pancreatic cancer revealed by single-cell spatial transcriptomics and optimal transport analysis\nAuthors: Shiau, C.; Cao, J.; Gregory, M. T.; Gong, D.; Yin, X.; Cho, J.-W.; Wang, P. L.; Su, J.; Wang, S.; Reeves, J. W.; Kim, T. K.; Kim, Y.; Guo, J. A.; Lester, N. A.; Schurman, N.; Barth, J. L.; Weissleder, R.; Jacks, T.; Qadan, M.; Hong, T. S.; Wo, J. Y.; Roberts, H.; Beechem, J. M.; Fernandez-del Castillo, C.; Mino-Kenudson, M.; Ting, D. T.; Hemberg, M.; Hwang, W. L.\nScore: 13.3, Published: 2023-06-29 DOI: 10.1101/2023.06.28.546848\nIn combination with cell intrinsic properties, interactions in the tumor microenvironment modulate therapeutic response. We leveraged high-plex single-cell spatial transcriptomics to dissect the remodeling of multicellular neighborhoods and cell-cell interactions in human pancreatic cancer associated with specific malignant subtypes and neoadjuvant chemotherapy/radiotherapy. We developed Spatially Constrained Optimal Transport Interaction Analysis (SCOTIA), an optimal transport model with a cost function that includes both spatial distance and ligand-receptor gene expression. Our results uncovered a marked change in ligand-receptor interactions between cancer-associated fibroblasts and malignant cells in response to treatment, which was supported by orthogonal datasets, including an ex vivo tumoroid co-culture system. Overall, this study demonstrates that characterization of the tumor microenvironment using high-plex single-cell spatial transcriptomics allows for identification of molecular interactions that may play a role in the emergence of chemoresistance and establishes a translational spatial biology paradigm that can be broadly applied to other malignancies, diseases, and treatments.\nKetogenic diet as a metabolic vehicle for enhancing the therapeutic efficacy of mebendazole and devimistat in preclinical pediatric glioma\nAuthors: Mukherjee, P.; Greenwood, B.; Henao, J.; Kiebish, M. A.; Seyfried, T. N.\nScore: 26.5, Published: 2023-06-11 DOI: 10.1101/2023.06.09.544252\nInvasion of high-grade glioma (HGG) cells through the brain and spinal cord is a leading cause of cancer death in children. Despite advances in treatment, survivors often suffer from life-long adverse effects of the toxic therapies. This study investigated the influence of nutritional ketosis on the therapeutic action of mebendazole (MBZ) and devimistat (CPI-613) against the highly invasive VM-M3 glioblastoma cells in juvenile syngeneic p20-p25 mice; a preclinical model of pediatric HGG. Cerebral implantation of the VM-M3 glioblastoma cells invaded throughout the brain and the spinal column similar to that seen commonly in children with malignant glioma. The maximum therapeutic benefit of MBZ and CPI-613 on tumour invasion and mouse survival occurred only when the drugs were administered together with a ketogenic diet (KD). MBZ reduced VM-M3 tumour cell growth and invasion when evaluated under in-vitro and in-vivo conditions through inhibition of both the glutaminolysis and the glycolysis pathways. Moreover, administration of the drugs with the KD allowed a low dosing for the juvenile mice, which minimized toxicity while improving overall survival. This preclinical study in juvenile mice highlights the potential importance of a diet/drug therapeutic strategy for managing childhood brain cancer.\nPancreatic Tumor-Derived Extracellular Vesicles Stimulate Schwann Cell Phenotype Indicative of Perineural Invasion via IL-8 Signaling\nAuthors: Gregory, E.; Powers, I.; Jamshidi-Parsian, A.; Griffin, R. J.; Song, Y.\nScore: 3.9, Published: 2023-06-28 DOI: 10.1101/2023.06.26.546629\nPancreatic cancer remains a pre-eminent cause of cancer-related deaths with late-stage diagnoses leading to an 11% five-year survival rate. Moreover, perineural invasion (PNI), in which cancer cells migrate into adjacent nerves, occurs in an overwhelming majority of patients, further enhancing tumor metastasis. PNI has only recently been recognized as a key contributor to cancer progression; thus, there are insufficient treatment options for the disease. Attention has been focused on glial Schwann cells (SC) for their mediation of pancreatic PNI. Under stress, SCs dedifferentiate from their mature state to facilitate the repair of peripheral nerves; however, this signaling can also re-direct cancer cells to accelerate PNI. Limited research has explored the mechanism that causes this shift in SC phenotype in cancer. Tumor-derived extracellular vesicles (TEV) have been implicated in other avenues of cancer development, such as pre-metastatic niche formation in secondary locations, yet how TEVs contribute to PNI has not been fully explored. In this study, we highlight TEVs as initiators of SC activation into a PNI-associated phenotype. Proteomic and pathway assessments of TEVs revealed an elevation in interleukin-8 (IL-8) signaling and nuclear factor kappa B (NF{kappa}B) over healthy cell-derived EVs. TEV-treated SCs exhibited higher levels of activation markers, which were successfully neutralized with IL-8 inhibition. Additionally, TEVs increased NF{kappa}B subunit p65 nuclear translocation, which may lead to increased secretion of cytokines and proteases indicative of SC activation and PNI. These findings present a novel mechanism that may be targeted for the treatment of pancreatic cancer PNI. Statement of SignificanceIdentifying pancreatic tumor extracellular vesicles as key players in Schwann cell activation and perineural invasion by way of IL-8 will educate for more specialized and effective targets for an under-valued disease.\nSynergistic effects of PARP inhibitors by Schlafen 11 and BRCA2-deficiency through accumulation of single-strand DNA gaps behind a fork\nAuthors: Onji, H.; Tate, S.; Sakaue, T.; Onishi, N.; Matsumoto, T.; Sugiyama, T.; Higashiyama, S.; Murai, J.\nScore: 2.8, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546820\nPoly (ADP-ribose) polymerase (PARP) inhibitors (PARPis) induce synthetic lethality in breast cancer gene (BRCA)-deficient tumors. Besides the original model proposed by accumulation of double-strand DNA breaks due to the impaired homologous recombination, accumulation of single-strand DNA (ssDNA) gaps due to impaired BRCA-mediated Okazaki fragment processing has emerged as an alternative mechanism of synthetic lethality. Accordingly, PARPis induce ssDNA gaps behind a replication fork in BRCA-deficient cells. Schlafen 11 (SLFN11), a member of the SLFN family, binds replication protein A (RPA)-coated ssDNA gaps and sensitizes cancer cells to DNA-damaging anticancer agents. These facts motivated us to examine the combinational effects of SLFN11 and BRCA-deficiency on PARPis sensitivity. Here, we show that SLFN11 and BRCA2-deficiency synergistically increased sensitivity to PARPis (talazoparib, niraparib, olaparib, and veliparib) at specific concentrations, where SLFN11 alone showed marginal effects. Using chromatin-bound proteins and alkaline BrdU comet assays in human cancer cells, we revealed that ssDNA gaps induced by PARPis were increased by SLFN11 or BRCA2-deficiency and that the combination of the two had the greatest effect. SLFN11 was recruited to and colocalized with chromatin-bound RPA2 under PARPis. SLFN11 recruited around a fork under DNA damage blocked replication, whereas SLFN11 recruited behind a fork under PARPis did not. Chromatin recruitment of SLFN11 and RPA2 was attenuated by the MRE11 inhibitor mirin. Hence, our studies showed that BRCA2-deficiency increased ssDNA gaps behind a fork under PARPis treatment, where SLFN11 bound and further increased the gaps. Our findings provide a mechanistic understanding of favorable responses to PARPis in SLFN11-proficient and BRCA-deficient tumors.\nEx vivo quantification of anti-tumor T-cell activity upon anti-PD-1 treatment in patient-derived lung tumor-on-chip\nAuthors: veith, i.; mencattini, a.; nurmik, m.; damei, i.; lansche, c.; brosseau, s.; gropplero, g.; corgnac, s.; filippi, j.; pote, n.; mordant, p.; tosello, j.; sedlik, c.; piaggio, e.; girard, n.; camonis, j.; shirvani, h.; mami-chouaib, f.; mechta-grigoriou, f.; descroix, s.; martinelli, e.; zalcman, g.; PARRINI, M. C.\nScore: 3.1, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545960\nThere is a compelling need for new approaches to predict efficacy of immunotherapy drugs. Tumor-on-chip technology exploits microfluidics to generate 3D cell co-cultures embedded in hydrogels that recapitulate immune and stromal characteristics of a simplified tumor ecosystem. Here, we present the development and validation of lung-tumor-on-chip platforms to quickly and precisely measure ex vivo the effects of immune check-point inhibitors on T-cell-mediated cancer cell death, by exploiting the power of live imaging and advanced image analysis algorithms. These tumor-on-chips were generated with patient-derived autologous primary cells isolated from fresh lung cancer samples, opening the path for applications in personalized medicine. Moreover, cancer-associated fibroblasts were shown to impair the response to anti-PD-1, indicating that tumor-on-chips are capable of recapitulating stroma-dependent mechanisms of immunotherapy resistance. This interdisciplinary combination of microfluidic devices, clinically-relevant cell models, and advanced computational methods, can innovatively improve both the fundamental understanding and clinical efficacy of immuno-oncology drugs.\nHepatoblastoma exhibits a predominantly myeloid immune landscape and reveals opportunities for macrophage targeted immunotherapy\nAuthors: Krijgsman, D.; Kraaier, L.; Verdonschot, M.; Schubert, S.; Leusen, J.; Duiker, E.; de Kleine, R.; de Meijer, V.; de Krijger, R.; Zsiros, J.; Peng, W. C.; Vercoulen, Y.\nScore: 2.0, Published: 2023-06-28 DOI: 10.1101/2023.06.28.546852\nBackground \u0026 Aims: Hepatoblastoma (HB) is a rare form of pediatric liver cancer which is currently treated with chemotherapy and surgery. The side effects of chemotherapy pose a major problem in HB and underline the need for an alternative treatment option. We aimed to characterize the immune landscape of HB to improve our understanding of the immunologic contribution to this disease and explore immunotherapeutic options. Methods: An imaging mass cytometry panel of 36 antibodies was used on tissue of treatment-naive HB (n=5), and chemotherapy-treated HB (n=3), with paired distal normal liver tissue. Immunofluorescence was used to stain HB and normal liver tissue for Kupffer cell marker MARCO. A public single-cell RNA-sequencing (scRNA-seq) dataset was analyzed consisting of 9 chemotherapy-treated HB and paired normal liver tissue. Results: HB showed a heterogeneous immune landscape predominantly comprising macrophages and monocytes with high expression of immune checkpoints CD47, SIRPa, and VISTA, whereas T cells were limited. Chemotherapy increased influx of macrophages and CD8+ T cells in HB. Transcriptome profiling demonstrated an early activated phenotype of CD8+ T cells in chemotherapy-treated HB and absence of an exhaustion signature and immune checkpoint expression. Furthermore, tumor-associated macrophages had low MARCO expression, upregulated inflammatory markers and a high liver tissue residency score while expressing other Kupffer cell markers, such as CD5L, to a variable degree. Conclusions: The absence of immune checkpoints and exhaustion markers in CD8+ T cells prohibits T cell-targeting by immune checkpoint blockade in HB patients. Instead, HB tumors contain a large myeloid compartment which provide opportunities for macrophage targeting, thereby paving the way for the development of improved treatment strategies for HB patients.\n",
  "wordCount" : "2517",
  "inLanguage": "en",
  "datePublished": "2023-07-02T10:39:32Z",
  "dateModified": "2023-07-02T10:39:32Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on July 2, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.17.545350">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.17.545350" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.17.545350">
        <p class="paperTitle">Deep intravital brain tumor imaging enabled by tailored three-photon microscopy and analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.17.545350" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.17.545350" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schubert, M. C.; Soyka, S. J.; Tamimi, A.; Maus, E.; Denninger, R.; Wissmann, N.; Reyhan, E.; Tetzlaff, S. K.; Beretta, C.; Drumm, M.; Schroers, J.; Steffens, A.; Walshon, J.; McCortney, K.; Heiland, S.; Golebiewska, A.; Kurz, F. T.; Wick, W.; Winkler, F.; Kreshuk, A.; Kuner, T.; Horbinski, C.; Prevedel, R.; Venkataramani, V.</p>
        <p class="info">Score: 48.2, Published: 2023-06-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.17.545350' target='https://doi.org/10.1101/2023.06.17.545350'> 10.1101/2023.06.17.545350</a></p>
        <p class="abstract">Intravital two-photon microscopy has emerged as a powerful technology to study brain tumor biology and its temporal dynamics, including invasion, proliferation and therapeutic resistance in the superficial layers of the mouse cortex. However, intravital microscopy of deeper cortical layers and especially the subcortical white matter, an important route of glioblastoma invasion and recurrence, has not yet been feasible due to low signal-to-noise ratios, missing spatiotemporal resolution and the inability to delineate myelinated axonal tracts. Here, we present a tailored intravital microscopy and artificial intelligence-based analysis methodology and workflow that enables routine deep imaging of glioblastoma over extended time periods, named Deep3P. We show that three-photon microscopy, adaptive optics, as well as customized deep learning-based denoising and machine learning segmentation together allow for deep brain intravital investigation of tumor biology up to 1.2 mm depth. Leveraging this approach, we find that perivascular invasion is a preferred invasion route into the corpus callosum as compared to intracortical glioblastoma invasion and uncover two vascular mechanisms of glioblastoma migration in the white matter. Furthermore, we can define an imaging biomarker of white matter disruption during early glioblastoma colonization. Taken together, Deep3P allows for an efficient and non-invasive investigation of brain tumor biology and its tumor microenvironment in unprecedented deep white and gray matter of the living mouse, opening up novel opportunities for studying the neuroscience of brain tumors and other model systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.25.546239">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.25.546239" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.25.546239">
        <p class="paperTitle">Immortalization and transformation of primary cells mediated by engineered ecDNAs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.25.546239" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.25.546239" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pradella, D.; Zhang, M.; Gao, R.; Melissa, Y. A.; Gluchowska, K. M.; Cendon Florez, Y.; Mishra, T.; La Rocca, G.; Weigl, M.; Jiao, Z.; Nguyen, H. H. M.; Grimm, F.; Lisi, M.; Mastroleo, C.; Chen, K.; Luebeck, J.; Bafna, V.; Antonescu, C. R.; Ventura, A.</p>
        <p class="info">Score: 41.2, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.25.546239' target='https://doi.org/10.1101/2023.06.25.546239'> 10.1101/2023.06.25.546239</a></p>
        <p class="abstract">Focal gene amplifications are among the most common cancer-associated mutations, but their evolution and contribution to tumorigenesis have proven challenging to recapitulate in primary cells and model organisms. Here we describe a general approach to engineer large (&gt;1 Mbp) focal amplifications mediated by extrachromosomal circular DNAs (ecDNAs, also known as &#34;double minutes&#34;) in a spatiotemporally controlled manner in cancer cell lines and in primary cells derived from genetically engineered mice. With this strategy, ecDNA formation can be coupled with expression of fluorescent reporters or other selectable markers to enable the identification and tracking of ecDNA-containing cells. We demonstrate the feasibility of this approach by engineering MDM2-containing ecDNAs in near-diploid human cells, showing that GFP expression can be used to track ecDNA dynamics under physiological conditions or in the presence of specific selective pressures. We also apply this approach to generate mice harboring inducible Myc- and Mdm2-containing ecDNAs analogous to those spontaneously occurring in human cancers. We show that the engineered ecDNAs rapidly accumulate in primary cells derived from these animals, promoting proliferation, immortalization, and transformation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.25.546474">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.25.546474" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.25.546474">
        <p class="paperTitle">MAPS: Pathologist-level cell type annotation from tissue images through machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.25.546474" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.25.546474" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shaban, M.; Bai, Y.; Qiu, H.; Mao, S.; Yeung, J.; Yeo, Y. Y.; Shanmugam, V.; Shanmugam, V.; Chen, H.; Zhu, B.; Nolan, G. P.; Shipp, M. A.; Rodig, S. J.; Jiang, S.; Mahmood, F.</p>
        <p class="info">Score: 24.7, Published: 2023-06-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.25.546474' target='https://doi.org/10.1101/2023.06.25.546474'> 10.1101/2023.06.25.546474</a></p>
        <p class="abstract">Highly multiplexed protein imaging is emerging as a potent technique for analyzing protein distribution within cells and tissues in their native context. However, existing cell annotation methods utilizing high-plex spatial proteomics data are resource intensive and necessitate iterative expert input, thereby constraining their scalability and practicality for extensive datasets. We introduce MAPS (Machine learning for Analysis of Proteomics in Spatial biology), a machine learning approach facilitating rapid and precise cell type identification with human-level accuracy from spatial proteomics data. Validated on multiple in-house and publicly available MIBI and CODEX datasets, MAPS outperforms current annotation techniques in terms of speed and accuracy, achieving pathologist-level precision even for challenging cell types, including tumor cells of immune origin. By democratizing rapidly deployable and scalable machine learning annotation, MAPS holds significant potential to expedite advances in tissue biology and disease comprehension.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545743">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545743" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545743">
        <p class="paperTitle">Epigenomic analysis of Formalin-Fixed Paraffin-Embedded samples by CUT&amp;Tag</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545743" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545743" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Henikoff, S.; Henikoff, J. G.; Ahmad, K.; Paranal, R. M.; Janssens, D. H.; Russell, Z. R.; Szulzewsky, F.; Kugel, S.; Holland, E. C.</p>
        <p class="info">Score: 26.0, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545743' target='https://doi.org/10.1101/2023.06.20.545743'> 10.1101/2023.06.20.545743</a></p>
        <p class="abstract">For more than a century, Formalin Fixed Paraffin Embedded (FFPE) sample preparation has been the preferred method for long-term preservation of biological material. However, the use of FFPE samples for epigenomic studies has been difficult because of chromatin damage from long exposure to high concentrations of formaldehyde. Previously, we introduced Cleavage Under Targeted Accessible Chromatin (CUTAC), an antibody-targeted chromatin accessibility mapping protocol based on CUT&amp;Tag. Here we show that simple modifications of our single-tube CUTAC protocol are sufficient to produce high-resolution maps of paused RNA Polymerase II (RNAPII) at enhancers and promoters using FFPE samples. We find that transcriptional regulatory element differences produced by FFPE-CUTAC distinguish between mouse brain tumor specimens and identify regulatory element markers with high confidence and precision. Our simple work-flow is suitable for automation, making possible affordable epigenomic profiling of archived biological samples for biomarker identification, clinical applications and retrospective studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546848">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546848" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546848">
        <p class="paperTitle">Therapy-associated remodeling of pancreatic cancer revealed by single-cell spatial transcriptomics and optimal transport analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546848" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546848" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shiau, C.; Cao, J.; Gregory, M. T.; Gong, D.; Yin, X.; Cho, J.-W.; Wang, P. L.; Su, J.; Wang, S.; Reeves, J. W.; Kim, T. K.; Kim, Y.; Guo, J. A.; Lester, N. A.; Schurman, N.; Barth, J. L.; Weissleder, R.; Jacks, T.; Qadan, M.; Hong, T. S.; Wo, J. Y.; Roberts, H.; Beechem, J. M.; Fernandez-del Castillo, C.; Mino-Kenudson, M.; Ting, D. T.; Hemberg, M.; Hwang, W. L.</p>
        <p class="info">Score: 13.3, Published: 2023-06-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546848' target='https://doi.org/10.1101/2023.06.28.546848'> 10.1101/2023.06.28.546848</a></p>
        <p class="abstract">In combination with cell intrinsic properties, interactions in the tumor microenvironment modulate therapeutic response. We leveraged high-plex single-cell spatial transcriptomics to dissect the remodeling of multicellular neighborhoods and cell-cell interactions in human pancreatic cancer associated with specific malignant subtypes and neoadjuvant chemotherapy/radiotherapy. We developed Spatially Constrained Optimal Transport Interaction Analysis (SCOTIA), an optimal transport model with a cost function that includes both spatial distance and ligand-receptor gene expression. Our results uncovered a marked change in ligand-receptor interactions between cancer-associated fibroblasts and malignant cells in response to treatment, which was supported by orthogonal datasets, including an ex vivo tumoroid co-culture system. Overall, this study demonstrates that characterization of the tumor microenvironment using high-plex single-cell spatial transcriptomics allows for identification of molecular interactions that may play a role in the emergence of chemoresistance and establishes a translational spatial biology paradigm that can be broadly applied to other malignancies, diseases, and treatments.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.09.544252">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.09.544252" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.09.544252">
        <p class="paperTitle">Ketogenic diet as a metabolic vehicle for enhancing the therapeutic efficacy of mebendazole and devimistat in preclinical pediatric glioma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.09.544252" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.09.544252" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mukherjee, P.; Greenwood, B.; Henao, J.; Kiebish, M. A.; Seyfried, T. N.</p>
        <p class="info">Score: 26.5, Published: 2023-06-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.09.544252' target='https://doi.org/10.1101/2023.06.09.544252'> 10.1101/2023.06.09.544252</a></p>
        <p class="abstract">Invasion of high-grade glioma (HGG) cells through the brain and spinal cord is a leading cause of cancer death in children. Despite advances in treatment, survivors often suffer from life-long adverse effects of the toxic therapies. This study investigated the influence of nutritional ketosis on the therapeutic action of mebendazole (MBZ) and devimistat (CPI-613) against the highly invasive VM-M3 glioblastoma cells in juvenile syngeneic p20-p25 mice; a preclinical model of pediatric HGG. Cerebral implantation of the VM-M3 glioblastoma cells invaded throughout the brain and the spinal column similar to that seen commonly in children with malignant glioma. The maximum therapeutic benefit of MBZ and CPI-613 on tumour invasion and mouse survival occurred only when the drugs were administered together with a ketogenic diet (KD). MBZ reduced VM-M3 tumour cell growth and invasion when evaluated under in-vitro and in-vivo conditions through inhibition of both the glutaminolysis and the glycolysis pathways.

Moreover, administration of the drugs with the KD allowed a low dosing for the juvenile mice, which minimized toxicity while improving overall survival. This preclinical study in juvenile mice highlights the potential importance of a diet/drug therapeutic strategy for managing childhood brain cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.26.546629">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.26.546629" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.26.546629">
        <p class="paperTitle">Pancreatic Tumor-Derived Extracellular Vesicles Stimulate Schwann Cell Phenotype Indicative of Perineural Invasion via IL-8 Signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.26.546629" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.26.546629" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gregory, E.; Powers, I.; Jamshidi-Parsian, A.; Griffin, R. J.; Song, Y.</p>
        <p class="info">Score: 3.9, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.26.546629' target='https://doi.org/10.1101/2023.06.26.546629'> 10.1101/2023.06.26.546629</a></p>
        <p class="abstract">Pancreatic cancer remains a pre-eminent cause of cancer-related deaths with late-stage diagnoses leading to an 11% five-year survival rate. Moreover, perineural invasion (PNI), in which cancer cells migrate into adjacent nerves, occurs in an overwhelming majority of patients, further enhancing tumor metastasis. PNI has only recently been recognized as a key contributor to cancer progression; thus, there are insufficient treatment options for the disease. Attention has been focused on glial Schwann cells (SC) for their mediation of pancreatic PNI. Under stress, SCs dedifferentiate from their mature state to facilitate the repair of peripheral nerves; however, this signaling can also re-direct cancer cells to accelerate PNI. Limited research has explored the mechanism that causes this shift in SC phenotype in cancer. Tumor-derived extracellular vesicles (TEV) have been implicated in other avenues of cancer development, such as pre-metastatic niche formation in secondary locations, yet how TEVs contribute to PNI has not been fully explored. In this study, we highlight TEVs as initiators of SC activation into a PNI-associated phenotype. Proteomic and pathway assessments of TEVs revealed an elevation in interleukin-8 (IL-8) signaling and nuclear factor kappa B (NF{kappa}B) over healthy cell-derived EVs. TEV-treated SCs exhibited higher levels of activation markers, which were successfully neutralized with IL-8 inhibition. Additionally, TEVs increased NF{kappa}B subunit p65 nuclear translocation, which may lead to increased secretion of cytokines and proteases indicative of SC activation and PNI. These findings present a novel mechanism that may be targeted for the treatment of pancreatic cancer PNI.

Statement of SignificanceIdentifying pancreatic tumor extracellular vesicles as key players in Schwann cell activation and perineural invasion by way of IL-8 will educate for more specialized and effective targets for an under-valued disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546820">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546820" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546820">
        <p class="paperTitle">Synergistic effects of PARP inhibitors by Schlafen 11 and BRCA2-deficiency through accumulation of single-strand DNA gaps behind a fork</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546820" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546820" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Onji, H.; Tate, S.; Sakaue, T.; Onishi, N.; Matsumoto, T.; Sugiyama, T.; Higashiyama, S.; Murai, J.</p>
        <p class="info">Score: 2.8, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546820' target='https://doi.org/10.1101/2023.06.28.546820'> 10.1101/2023.06.28.546820</a></p>
        <p class="abstract">Poly (ADP-ribose) polymerase (PARP) inhibitors (PARPis) induce synthetic lethality in breast cancer gene (BRCA)-deficient tumors. Besides the original model proposed by accumulation of double-strand DNA breaks due to the impaired homologous recombination, accumulation of single-strand DNA (ssDNA) gaps due to impaired BRCA-mediated Okazaki fragment processing has emerged as an alternative mechanism of synthetic lethality. Accordingly, PARPis induce ssDNA gaps behind a replication fork in BRCA-deficient cells. Schlafen 11 (SLFN11), a member of the SLFN family, binds replication protein A (RPA)-coated ssDNA gaps and sensitizes cancer cells to DNA-damaging anticancer agents. These facts motivated us to examine the combinational effects of SLFN11 and BRCA-deficiency on PARPis sensitivity. Here, we show that SLFN11 and BRCA2-deficiency synergistically increased sensitivity to PARPis (talazoparib, niraparib, olaparib, and veliparib) at specific concentrations, where SLFN11 alone showed marginal effects. Using chromatin-bound proteins and alkaline BrdU comet assays in human cancer cells, we revealed that ssDNA gaps induced by PARPis were increased by SLFN11 or BRCA2-deficiency and that the combination of the two had the greatest effect. SLFN11 was recruited to and colocalized with chromatin-bound RPA2 under PARPis. SLFN11 recruited around a fork under DNA damage blocked replication, whereas SLFN11 recruited behind a fork under PARPis did not. Chromatin recruitment of SLFN11 and RPA2 was attenuated by the MRE11 inhibitor mirin. Hence, our studies showed that BRCA2-deficiency increased ssDNA gaps behind a fork under PARPis treatment, where SLFN11 bound and further increased the gaps. Our findings provide a mechanistic understanding of favorable responses to PARPis in SLFN11-proficient and BRCA-deficient tumors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545960">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545960" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545960">
        <p class="paperTitle">Ex vivo quantification of anti-tumor T-cell activity upon anti-PD-1 treatment in patient-derived lung tumor-on-chip</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545960" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545960" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: veith, i.; mencattini, a.; nurmik, m.; damei, i.; lansche, c.; brosseau, s.; gropplero, g.; corgnac, s.; filippi, j.; pote, n.; mordant, p.; tosello, j.; sedlik, c.; piaggio, e.; girard, n.; camonis, j.; shirvani, h.; mami-chouaib, f.; mechta-grigoriou, f.; descroix, s.; martinelli, e.; zalcman, g.; PARRINI, M. C.</p>
        <p class="info">Score: 3.1, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545960' target='https://doi.org/10.1101/2023.06.21.545960'> 10.1101/2023.06.21.545960</a></p>
        <p class="abstract">There is a compelling need for new approaches to predict efficacy of immunotherapy drugs. Tumor-on-chip technology exploits microfluidics to generate 3D cell co-cultures embedded in hydrogels that recapitulate immune and stromal characteristics of a simplified tumor ecosystem. Here, we present the development and validation of lung-tumor-on-chip platforms to quickly and precisely measure ex vivo the effects of immune check-point inhibitors on T-cell-mediated cancer cell death, by exploiting the power of live imaging and advanced image analysis algorithms. These tumor-on-chips were generated with patient-derived autologous primary cells isolated from fresh lung cancer samples, opening the path for applications in personalized medicine. Moreover, cancer-associated fibroblasts were shown to impair the response to anti-PD-1, indicating that tumor-on-chips are capable of recapitulating stroma-dependent mechanisms of immunotherapy resistance. This interdisciplinary combination of microfluidic devices, clinically-relevant cell models, and advanced computational methods, can innovatively improve both the fundamental understanding and clinical efficacy of immuno-oncology drugs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.28.546852">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.28.546852" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.28.546852">
        <p class="paperTitle">Hepatoblastoma exhibits a predominantly myeloid immune landscape and reveals opportunities for macrophage targeted immunotherapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.28.546852" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.28.546852" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krijgsman, D.; Kraaier, L.; Verdonschot, M.; Schubert, S.; Leusen, J.; Duiker, E.; de Kleine, R.; de Meijer, V.; de Krijger, R.; Zsiros, J.; Peng, W. C.; Vercoulen, Y.</p>
        <p class="info">Score: 2.0, Published: 2023-06-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.28.546852' target='https://doi.org/10.1101/2023.06.28.546852'> 10.1101/2023.06.28.546852</a></p>
        <p class="abstract">Background &amp; Aims: Hepatoblastoma (HB) is a rare form of pediatric liver cancer which is currently treated with chemotherapy and surgery. The side effects of chemotherapy pose a major problem in HB and underline the need for an alternative treatment option. We aimed to characterize the immune landscape of HB to improve our understanding of the immunologic contribution to this disease and explore immunotherapeutic options. Methods: An imaging mass cytometry panel of 36 antibodies was used on tissue of treatment-naive HB (n=5), and chemotherapy-treated HB (n=3), with paired distal normal liver tissue. Immunofluorescence was used to stain HB and normal liver tissue for Kupffer cell marker MARCO. A public single-cell RNA-sequencing (scRNA-seq) dataset was analyzed consisting of 9 chemotherapy-treated HB and paired normal liver tissue. Results: HB showed a heterogeneous immune landscape predominantly comprising macrophages and monocytes with high expression of immune checkpoints CD47, SIRPa, and VISTA, whereas T cells were limited. Chemotherapy increased influx of macrophages and CD8&#43; T cells in HB. Transcriptome profiling demonstrated an early activated phenotype of CD8&#43; T cells in chemotherapy-treated HB and absence of an exhaustion signature and immune checkpoint expression. Furthermore, tumor-associated macrophages had low MARCO expression, upregulated inflammatory markers and a high liver tissue residency score while expressing other Kupffer cell markers, such as CD5L, to a variable degree. Conclusions: The absence of immune checkpoints and exhaustion markers in CD8&#43; T cells prohibits T cell-targeting by immune checkpoint blockade in HB patients. Instead, HB tumors contain a large myeloid compartment which provide opportunities for macrophage targeting, thereby paving the way for the development of improved treatment strategies for HB patients.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
