<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Mechanisms underpinning osteosarcoma genome complexity and evolution
Authors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.
Score: 28.2, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573403
Osteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Mechanisms underpinning osteosarcoma genome complexity and evolution
Authors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.
Score: 28.2, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573403
Osteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-07T10:38:59+00:00" />
<meta property="article:modified_time" content="2024-01-07T10:38:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Mechanisms underpinning osteosarcoma genome complexity and evolution
Authors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.
Score: 28.2, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573403
Osteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Mechanisms underpinning osteosarcoma genome complexity and evolution\nAuthors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.\nScore: 28.2, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573403\nOsteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood.",
  "keywords": [
    
  ],
  "articleBody": " Mechanisms underpinning osteosarcoma genome complexity and evolution\nAuthors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.\nScore: 28.2, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573403\nOsteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood. Using multi-region whole-genome sequencing, we find that chromothripsis is an ongoing mutational process, occurring subclonally in 74% of tumours. Chromothripsis drives the acquisition of oncogenic mutations and generates highly unstable derivative chromosomes, the evolution of which drives clonal diversification and intra-tumour heterogeneity. In addition, we report a novel mechanism, loss-translocation-amplification (LTA) chromothripsis, which mediates rapid malignant transformation and punctuated evolution in about half of paediatric and adult high-grade osteosarcomas. Specifically, a single double-strand break triggers concomitant TP53 inactivation and segmental amplifications, often amplifying oncogenes to high copy numbers in extrachromosomal circular DNA elements through breakage-fusion-bridge cycles involving multiple chromosomes. LTA chromothripsis is detected at low frequency in soft-tissue sarcomas, but not in epithelial cancers, including those driven by TP53 mutation. Finally, we identify genome-wide loss of heterozygosity as a strong prognostic indicator for high-grade osteosarcoma.\nDecoding mTOR signalling heterogeneity in the tumour microenvironment using multiplexed imaging and graph convolutional networks\nAuthors: Zuhair, R.; Eastwood, M.; Jones, M.; Cross, A.; Hester, J.; Issa, F.; Ginty, F.; Sailem, H.\nScore: 13.4, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573693\nEvaluating the contribution of the tumour microenvironment (TME) in tumour progression has proven a complex challenge due to the intricate interactions within the TME. Multiplexed imaging is an emerging technology that allows concurrent assessment of multiple of these components simultaneously. Here we utilise a highly multiplexed dataset of 61 markers across 746 colorectal tumours to investigate how complex mTOR signalling in different tissue compartments influences patient prognosis. We found that the signalling of mTOR pathway can have heterogeneous activation patterns in tumour and immune compartments which correlate with patient prognosis. Using graph neural networks, we determined the most predictive features of mTOR activity in immune cells and identified relevant cellular subpopulations. We validated our observations using spatial transcriptomics data analysis in an independent patient cohort. Our work provides a framework for studying complex cell signalling and reveals important insights for developing mTOR-based therapies.\nSolid tumor growth depends on an intricate equilibrium of malignant cell states\nAuthors: Torborg, S. R.; Grbovic-Huezo, O.; Singhal, A.; Holm, M.; Wu, K.; Han, X.; Ho, Y.-J.; Haglund, C.; Mitchell, M. J.; Lowe, S. W.; Dow, L. E.; Pitter, K. L.; Sanchez-Rivera, F. J.; Levchenko, A.; Tammela, T.\nScore: 10.5, Published: 2023-12-30 DOI: 10.1101/2023.12.30.573100\nControl of cell identity and number is central to tissue function, yet principles governing organization of malignant cells in tumor tissues remain poorly understood. Using mathematical modeling and candidate-based analysis, we discover primary and metastatic pancreatic ductal adenocarcinoma (PDAC) organize in a stereotypic pattern whereby PDAC cells responding to WNT signals (WNT-R) neighbor WNT-secreting cancer cells (WNT-S). Leveraging lineage-tracing, we reveal the WNT-R state is transient and gives rise to the WNT-S state that is highly stable and committed to organizing malignant tissue. We further show that a subset of WNT-S cells expressing the Notch ligand DLL1 form a functional niche for WNT-R cells. Genetic inactivation of WNT secretion or Notch pathway components, or cytoablation of the WNT-S state disrupts PDAC tissue organization, suppressing tumor growth and metastasis. This work indicates PDAC growth depends on an intricately controlled equilibrium of functionally distinct cancer cell states, uncovering a fundamental principle governing solid tumor growth and revealing new opportunities for therapeutic intervention.\nCREB activation drives acinar to ductal reprogramming and promote pancreatic cancer progression in animal models of alcoholic chronic pancreatitis\nAuthors: Srinivasan, S.; Mehra, S.; Bianchi, A.; Singh, S.; Dosch, A. R.; Amirian, H.; Jinka, S.; Krishnamoorthy, V.; Silva, I. D. C.; Box, E. W.; Garrido, V.; Totiger, T. M.; Zhou, Z.; Ban, Y.; Datta, J.; VanSaun, M.; Merchant, N.; Nagathihalli, N.\nScore: 6.4, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574376\nBACKGROUND AND AIMS: In vivo induction of alcoholic chronic pancreatitis (ACP) causes significant acinar damage, increased fibroinflammatory response, and heightened activation of cyclic response element binding protein 1 (CREB) when compared with alcohol (A) or chronic pancreatitis (CP) mediated pancreatic damage. However, the study elucidating the cooperative interaction between CREB and the oncogenic KrasG12D/+(Kras*) in promoting pancreatic cancer progression with ACP remains unexplored. METHODS: Experimental ACP induction was established in multiple mouse models, followed by euthanization of the animals at various time intervals during the recovery periods. Tumor latency was determined in these mice cohorts. Here, we established CREB deletion (Crebfl/fl) in Ptf1aCreERTM/+;LSL-KrasG12D+/-(KC) genetic mouse models (KCC-/-). Western blot, phosphokinase array, and qPCR were used to analyze the pancreata of Ptf1aCreERTM+/-, KC and KCC-/- mice. The pancreata of ACP-induced KC mice were subjected to single-cell RNA sequencing (scRNAseq). Further studies involved conducting lineage tracing and acinar cell explant cultures. RESULTS: ACP induction in KC mice had detrimental effects on the pancreatic damage repair mechanism. The persistent existence of acinar cell-derived ductal lesions demonstrated a prolonged state of hyperactivated CREB. Persistent CREB activation leads to acinar cell reprogramming and increased pro-fibrotic inflammation in KC mice. Acinar-specific Creb ablation reduced advanced PanINs lesions, hindered tumor progression, and restored acinar cell function in ACP-induced mouse models. CONCLUSIONS: Our findings demonstrate that CREB cooperates with Kras* to perpetuate an irreversible ADM and PanIN formation. Moreover, CREB sustains oncogenic activity to promote the progression of premalignant lesions toward cancer in the presence of ACP.\nBranched-chain keto acids promote an immune-suppressive and neurodegenerative microenvironment in leptomeningeal disease\nAuthors: Khaled, M. L.; Ren, Y.; Kundalia, R.; Alhaddad, H.; Chen, Z.; Wallace, G. C.; Evernden, B.; Ospina, O. E.; Hall, M.; Liu, M.; Darville, L. N. F.; Izumi, V.; Chen, Y. A.; Pilon-Thomas, S.; Stewart, P. A.; Koomen, J. M.; Corallo, S. A.; Jain, M. D.; Robinson, T. J.; Locke, F. L.; Forsyth, P. A.; Smalley, I.\nScore: 6.4, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572239\nLeptomeningeal disease (LMD) occurs when tumors seed into the leptomeningeal space and cerebrospinal fluid (CSF), leading to severe neurological deterioration and poor survival outcomes. We utilized comprehensive multi-omics analyses of CSF from patients with lymphoma LMD to demonstrate an immunosuppressive cellular microenvironment and identified dysregulations in proteins and lipids indicating neurodegenerative processes. Strikingly, we found a significant accumulation of toxic branched-chain keto acids (BCKA) in the CSF of patients with LMD. The BCKA accumulation was found to be a pan-cancer occurrence, evident in lymphoma, breast cancer, and melanoma LMD patients. Functionally, BCKA disrupted the viability and function of endogenous T lymphocytes, chimeric antigen receptor (CAR) T cells, neurons, and meningeal cells. Treatment of LMD mice with BCKA-reducing sodium phenylbutyrate significantly improved neurological function, survival outcomes, and efficacy of anti-CD19 CAR T cell therapy. This is the first report of BCKA accumulation in LMD and provides preclinical evidence that targeting these toxic metabolites improves outcomes.\nSpatial transcriptomics analysis identifies a unique tumor-promoting function of the meningeal stroma in melanoma leptomeningeal disease\nAuthors: Alhaddad, H.; Ospina, O. E.; Khaled, M. L.; Ren, Y.; Forsyth, P. A.; Pina, Y.; Macaulay, R.; Law, V.; Tsai, K. Y.; Cress, W. D.; Fridley, B.; Smalley, I.\nScore: 6.1, Published: 2023-12-19 DOI: 10.1101/2023.12.18.572266\nLeptomeningeal disease (LMD) remains a rapidly lethal complication for late-stage melanoma patients. The inaccessible nature of the disease site and lack of understanding of the biology of this unique metastatic site are major barriers to developing efficacious therapies for patients with melanoma LMD. Here, we characterize the tumor microenvironment of the leptomeningeal tissues and patient-matched extra-cranial metastatic sites using spatial transcriptomic analyses with in vitro and in vivo validation. We show the spatial landscape of melanoma LMD to be characterized by a lack of immune infiltration and instead exhibit a higher level of stromal involvement. We show that the tumor-stroma interactions at the leptomeninges activate pathways implicated in tumor-promoting signaling, mediated through upregulation of SERPINA3 at the tumor-stroma interface. Our functional experiments establish that the meningeal stroma is required for melanoma cells to survive in the CSF environment and that these interactions lead to a lack of MAPK inhibitor sensitivity in the tumor. We show that knocking down SERPINA3 or inhibiting the downstream IGR1R/PI3K/AKT axis results in re-sensitization of the tumor to MAPK-targeting therapy and tumor cell death in the leptomeningeal environment. Our data provides a spatial atlas of melanoma LMD, identifies the tumor-promoting role of meningeal stroma, and demonstrates a mechanism for overcoming microenvironment-mediated drug resistance unique to this metastatic site.\nHybrid cancer stem cells utilise vascular tracks for collective streaming invasion in a metastasis-on-a-chip device\nAuthors: Scemama, A.; Lunetto, S.; Tailor, A.; Di Cio, S.; Ambler, L.; Coetzee, A.; Cottom, H.; Khurram, S. A.; Gautrot, J.; Biddle, A.\nScore: 3.7, Published: 2024-01-02 DOI: 10.1101/2024.01.02.573897\nCancer stem cells (CSCs) drive cancer metastatic dissemination. They do not do so in a vacuum, and the important influence of the tumour microenvironment (TME) on metastatic dissemination is becoming increasingly recognised. Therapeutic targeting of CSC-TME interactions may be a promising route to suppression of tumour metastasis. However, we must first understand how interactions with the TME influence CSC metastatic dissemination. To achieve this understanding, there is a need for experimental models that enable the analysis of dynamic interactions at single cell resolution within a complex environment. To this end, we utilise a metastasis-on-a-chip device to produce a 3D in vitro model of CSC interaction with a developing microvasculature, that is amenable to precise imaging and real time studies at single cell resolution. We show that the invasive phenotype of oral squamous cell carcinoma (OSCC) cells is markedly altered when in proximity to a microvasculature, with a switch to a hybrid CSC phenotype that undergoes collective streaming invasion. Mechanistically, ECM compression by the developing vasculature creates an environment that is refractory to cancer invasion, whilst leaving abandoned vascular tracks that can be utilised by hybrid CSCs for collective streaming invasion. Human tissue studies identify streaming invasion in association with vascularised regions in OSCC specimens. These findings elucidate the influence of the vasculature on CSC metastatic dissemination in OSCC, and the role of hybrid CSC invasion plasticity in overcoming this TME constraint.\nSingle-cell RNA-seq analysis profiling characterizes differences in cell composition and physiology between normal tissue, treatment-naive, and cisplatin-treated ovarian cancer\nAuthors: Guo, F.; Yang, Z.; Sehouli, J.; Kaufmann, A.\nScore: 3.5, Published: 2024-01-03 DOI: 10.1101/2024.01.02.573967\nBackgroundIntense efforts have focused on identifying heterogeneity of the cellular composition in ovarian cancer. However, tissue composition and physiological conditions of cancer cells in cisplatin-sensitive ovarian cancer remains largely unknown. Moreover, comparisons of different cellular states in normal tissue, in treatment naive ovarian cancer, and in cisplatin-treated tissue after adjuvant therapy of cisplatin-sensitive ovarian cancer at the single-cell level might offer clues for ovarian cancer treatment and prevention of cisplatin-resistance formation. MethodsSingle-cell transcriptome sequencing of a cisplatin-treated ovarian cancer was performed. Data sets of non-tumorous ovarian tissues and treatment-naive ovarian cancer were downloaded from the European Genome-phenome Archive (accession number EGAS00001004987). Quality control, batch effect correction, integration, and clustering analysis of the integrated single-cell transcriptome data was done. Cell subsets were annotated based on surface marker phenotypes. Finally, the proportions of subclusters, the immune cell population, and the potential biological processes among different cellular states were compared. ResultsSixteen distinct cell subsets were identified from the integrated single-cell transcriptome sequencing data of a pool of all tissues. The composition of the three different tissue types was characterized. The proportion of fibroblasts in cisplatin-treated ovarian tumor was remarkably lower than in treatment-naive ovarian tumor (1.33% vs. 13.53%, p \u003c 0.05). Moreover, each subjects sample had differing relative proportions of the identified cell types. In primary untreated ovarian cancer, the prevalent immune cells were B cells and myeloid-related immunosuppressive M2 macrophages. However, there were less B cells and myeloid-related immunosuppressive M2 macrophages after cisplatin-treatment, while significantly more T cells were found. The physiological cellular state in primary untreated ovarian tumors was associated with dysfunctional gene expression and modulation of cellular homeostasis, while cells from cisplatin-treated tumor showed more activation of immune and inflammatory genes as compared to healthy ovarian tissue. ConclusionOur molecular gene expression analysis allowed for the separation and identification of differences in normal ovarian tissues, treatment-naive, and cisplatin-sensitive ovarian cancer cell populations at single-cell resolution. We identified different cell type composition and discriminative marker expression concerning specific cell subsets and identified differences among their physiological cell states. This knowledge may open new possibilities for elucidating important pathogenetic features and therapeutic strategies for treating ovarian cancer.\nBreakage fusion bridge cycles drive high oncogene copy number, but not intratumoral genetic heterogeneity or rapid cancer genome change\nAuthors: Dehkordi, S. R.; Wong, I. T.-L.; Ni, J.; Luebeck, J.; Zhu, K.; Prasad, G.; Krockenberger, L.; Xu, G.; Chowdhury, B.; Rajkumar, U.; Caplin, A.; Muliaditan, D.; Coruh, C.; Jin, Q.; Turner, K.; Teo, S. X.; Pang, A. W. C.; Alexandrov, L.; Chua, C. E. L.; Furnari, F.; Paulson, T.; Law, J.; Chang, H.; Yue, F.; DasGupta, R.; Zhao, J.; Mischel, P. S.; Bafna, V.\nScore: 4.0, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571349\nOncogene amplification is a major driver of cancer pathogenesis. Breakage fusion bridge (BFB) cycles, like extrachromosomal DNA (ecDNA), can lead to high copy numbers of oncogenes, but their impact on intratumoral heterogeneity, treatment response, and patient survival are not well understood due to difficulty in detecting them by DNA sequencing. We describe a novel algorithm that detects and reconstructs BFB amplifications using optical genome maps (OGMs), called OM2BFB. OM2BFB showed high precision (\u003e93%) and recall (92%) in detecting BFB amplifications in cancer cell lines, PDX models and primary tumors. OM-based comparisons demonstrated that short-read BFB detection using our AmpliconSuite (AS) toolkit also achieved high precision, albeit with reduced sensitivity. We detected 371 BFB events using whole genome sequences from 2,557 primary tumors and cancer lines. BFB amplifications were preferentially found in cervical, head and neck, lung, and esophageal cancers, but rarely in brain cancers. BFB amplified genes show lower variance of gene expression, with fewer options for regulatory rewiring relative to ecDNA amplified genes. BFB positive (BFB (+)) tumors showed reduced heterogeneity of amplicon structures, and delayed onset of resistance, relative to ecDNA(+) tumors. EcDNA and BFB amplifications represent contrasting mechanisms to increase the copy numbers of oncogene with markedly different characteristics that suggest different routes for intervention.\nGene expression landscape of cutaneous squamous cell carcinoma progression\nAuthors: Bencomo, T.; Lee, C. S.\nScore: 4.3, Published: 2023-12-18 DOI: 10.1101/2023.12.11.570862\nBackgroundCutaneous squamous cell carcinomas (SCC) are the second most common human cancer and have been characterized by RNA sequencing (RNA-Seq); however, the transferability of findings from individual studies may be limited by small sample sizes and diverse analysis protocols. ObjectivesTo define the transcriptome landscape at different stages in the progression of normal skin to SCC through a meta-analysis of publicly available RNA-Seq samples MethodsWhole-transcriptome data from 73 normal skin samples, 46 actinic keratoses (AK), 16 in situ SCC, 13 keratoacanthomas (KA), and 147 SCC (including 30 SCC from immunocompromised patients and 8 SCC from individuals with recessive dystrophic epidermolysis bullosa [RDEB]) was uniformly processed to harmonize gene expression. Differential expression, fusion detection, and cell-type deconvolution analyses were performed. ResultsIndividual RNA-Seq studies of SCC demonstrated study-specific clustering and varied widely in their differential gene expression detection. Following batch correction, we defined a consensus set of differentially expressed genes (DEGs), including those altered in the preinvasive stages of SCC development, and used single-cell RNA-Seq data to demonstrate that DEGs are often, but not always, expressed by tumor-specific keratinocytes (TSKs). Analysis of the cellular composition of SCC, KA, and RDEB-SCC identified an increase in differentiated keratinocytes in KA, while RDEB-SCC contained the most TSKs. Compared to SCC arising in immunocompetent patients, SCC from immunosuppressed individuals demonstrated fewer memory B cells and CD8 T cells. A comprehensive and unbiased search for fusion transcripts in SCC and intermediate disease stages identified few candidates that recur in \u003e1% of all specimens, suggesting most SCC are not driven by oncogenic gene fusions. Finally, using GTEx data, we distilled a novel 300-gene signature of chronic sun exposure that affirms greater cumulative ultraviolet (UV) exposure in later stages of SCC development. ConclusionsOur results define the gene expression landscape of SCC progression, characterize cell subpopulation heterogeneity in SCC subtypes that contribute to their distinct clinical phenotypes, demonstrate that gene fusions are not a common cause of SCC, and identify UV-responsive genes associated with SCC development. What is already known about this topic?O_LICutaneous squamous cell carcinoma (SCC) is one of the most common cancers worldwide. C_LIO_LISeveral studies have examined gene expression changes in SCC using RNA sequencing (RNA-Seq) but comparison of their results is difficult due to inter-study variation and diverse bioinformatic pipelines and protocols. C_LIO_LIA few gene fusions have been described in SCC, but a comprehensive characterization of fusion transcripts in patient samples has not been performed. C_LI What does this study add?O_LIWe re-analyzed RNA-Seq data from 11 studies of SCC and its preinvasive stages to create a list of consensus differentially expressed genes and identify those that are UV-responsive. C_LIO_LIClinically aggressive SCC displayed more tumor-specific keratinocytes, while keratoacanthomas contained more differentiated keratinocytes. SCC in immunocompetent persons had more memory B cells and CD8 T cells than those arising in immunosuppressed individuals. C_LIO_LIPreviously reported gene fusions were not detected and most fusion candidates did not demonstrate pathogenic features. C_LI What is the translational message?O_LIOur analysis harmonizes differing results from previous studies to provide a robust list of genes implicated in SCC development. C_LIO_LIOur findings suggest gene fusions are not a common driver event in SCC. C_LI\n",
  "wordCount" : "2913",
  "inLanguage": "en",
  "datePublished": "2024-01-07T10:38:59Z",
  "dateModified": "2024-01-07T10:38:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573403" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573403">
        <p class="paperTitle">Mechanisms underpinning osteosarcoma genome complexity and evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.</p>
        <p class="info">Score: 28.2, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573403' target='https://doi.org/10.1101/2023.12.29.573403'> 10.1101/2023.12.29.573403</a></p>
        <p class="abstract">Osteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood. Using multi-region whole-genome sequencing, we find that chromothripsis is an ongoing mutational process, occurring subclonally in 74% of tumours. Chromothripsis drives the acquisition of oncogenic mutations and generates highly unstable derivative chromosomes, the evolution of which drives clonal diversification and intra-tumour heterogeneity. In addition, we report a novel mechanism, loss-translocation-amplification (LTA) chromothripsis, which mediates rapid malignant transformation and punctuated evolution in about half of paediatric and adult high-grade osteosarcomas. Specifically, a single double-strand break triggers concomitant TP53 inactivation and segmental amplifications, often amplifying oncogenes to high copy numbers in extrachromosomal circular DNA elements through breakage-fusion-bridge cycles involving multiple chromosomes. LTA chromothripsis is detected at low frequency in soft-tissue sarcomas, but not in epithelial cancers, including those driven by TP53 mutation. Finally, we identify genome-wide loss of heterozygosity as a strong prognostic indicator for high-grade osteosarcoma.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573693">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573693" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573693">
        <p class="paperTitle">Decoding mTOR signalling heterogeneity in the tumour microenvironment using multiplexed imaging and graph convolutional networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573693" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573693" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zuhair, R.; Eastwood, M.; Jones, M.; Cross, A.; Hester, J.; Issa, F.; Ginty, F.; Sailem, H.</p>
        <p class="info">Score: 13.4, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573693' target='https://doi.org/10.1101/2023.12.30.573693'> 10.1101/2023.12.30.573693</a></p>
        <p class="abstract">Evaluating the contribution of the tumour microenvironment (TME) in tumour progression has proven a complex challenge due to the intricate interactions within the TME. Multiplexed imaging is an emerging technology that allows concurrent assessment of multiple of these components simultaneously. Here we utilise a highly multiplexed dataset of 61 markers across 746 colorectal tumours to investigate how complex mTOR signalling in different tissue compartments influences patient prognosis. We found that the signalling of mTOR pathway can have heterogeneous activation patterns in tumour and immune compartments which correlate with patient prognosis. Using graph neural networks, we determined the most predictive features of mTOR activity in immune cells and identified relevant cellular subpopulations. We validated our observations using spatial transcriptomics data analysis in an independent patient cohort. Our work provides a framework for studying complex cell signalling and reveals important insights for developing mTOR-based therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.30.573100">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.30.573100" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.30.573100">
        <p class="paperTitle">Solid tumor growth depends on an intricate equilibrium of malignant cell states</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.30.573100" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.30.573100" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Torborg, S. R.; Grbovic-Huezo, O.; Singhal, A.; Holm, M.; Wu, K.; Han, X.; Ho, Y.-J.; Haglund, C.; Mitchell, M. J.; Lowe, S. W.; Dow, L. E.; Pitter, K. L.; Sanchez-Rivera, F. J.; Levchenko, A.; Tammela, T.</p>
        <p class="info">Score: 10.5, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.30.573100' target='https://doi.org/10.1101/2023.12.30.573100'> 10.1101/2023.12.30.573100</a></p>
        <p class="abstract">Control of cell identity and number is central to tissue function, yet principles governing organization of malignant cells in tumor tissues remain poorly understood. Using mathematical modeling and candidate-based analysis, we discover primary and metastatic pancreatic ductal adenocarcinoma (PDAC) organize in a stereotypic pattern whereby PDAC cells responding to WNT signals (WNT-R) neighbor WNT-secreting cancer cells (WNT-S). Leveraging lineage-tracing, we reveal the WNT-R state is transient and gives rise to the WNT-S state that is highly stable and committed to organizing malignant tissue. We further show that a subset of WNT-S cells expressing the Notch ligand DLL1 form a functional niche for WNT-R cells. Genetic inactivation of WNT secretion or Notch pathway components, or cytoablation of the WNT-S state disrupts PDAC tissue organization, suppressing tumor growth and metastasis. This work indicates PDAC growth depends on an intricately controlled equilibrium of functionally distinct cancer cell states, uncovering a fundamental principle governing solid tumor growth and revealing new opportunities for therapeutic intervention.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574376" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574376">
        <p class="paperTitle">CREB activation drives acinar to ductal reprogramming and promote pancreatic cancer progression in animal models of alcoholic chronic pancreatitis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Srinivasan, S.; Mehra, S.; Bianchi, A.; Singh, S.; Dosch, A. R.; Amirian, H.; Jinka, S.; Krishnamoorthy, V.; Silva, I. D. C.; Box, E. W.; Garrido, V.; Totiger, T. M.; Zhou, Z.; Ban, Y.; Datta, J.; VanSaun, M.; Merchant, N.; Nagathihalli, N.</p>
        <p class="info">Score: 6.4, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574376' target='https://doi.org/10.1101/2024.01.05.574376'> 10.1101/2024.01.05.574376</a></p>
        <p class="abstract">BACKGROUND AND AIMS: In vivo induction of alcoholic chronic pancreatitis (ACP) causes significant acinar damage, increased fibroinflammatory response, and heightened activation of cyclic response element binding protein 1 (CREB) when compared with alcohol (A) or chronic pancreatitis (CP) mediated pancreatic damage. However, the study elucidating the cooperative interaction between CREB and the oncogenic KrasG12D/&#43;(Kras*) in promoting pancreatic cancer progression with ACP remains unexplored. METHODS: Experimental ACP induction was established in multiple mouse models, followed by euthanization of the animals at various time intervals during the recovery periods. Tumor latency was determined in these mice cohorts. Here, we established CREB deletion (Crebfl/fl) in Ptf1aCreERTM/&#43;;LSL-KrasG12D&#43;/-(KC) genetic mouse models (KCC-/-). Western blot, phosphokinase array, and qPCR were used to analyze the pancreata of Ptf1aCreERTM&#43;/-, KC and KCC-/-  mice. The pancreata of ACP-induced KC mice were subjected to single-cell RNA sequencing (scRNAseq). Further studies involved conducting lineage tracing and acinar cell explant cultures. RESULTS: ACP induction in KC mice had detrimental effects on the pancreatic damage repair mechanism. The persistent existence of acinar cell-derived ductal lesions demonstrated a prolonged state of hyperactivated CREB. Persistent CREB activation leads to acinar cell reprogramming and increased pro-fibrotic inflammation in KC mice. Acinar-specific Creb ablation reduced advanced PanINs lesions, hindered tumor progression, and restored acinar cell function in ACP-induced mouse models. CONCLUSIONS: Our findings demonstrate that CREB cooperates with Kras* to perpetuate an irreversible ADM and PanIN formation. Moreover, CREB sustains oncogenic activity to promote the progression of premalignant lesions toward cancer in the presence of ACP.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572239">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572239" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572239">
        <p class="paperTitle">Branched-chain keto acids promote an immune-suppressive and neurodegenerative microenvironment in leptomeningeal disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572239" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572239" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Khaled, M. L.; Ren, Y.; Kundalia, R.; Alhaddad, H.; Chen, Z.; Wallace, G. C.; Evernden, B.; Ospina, O. E.; Hall, M.; Liu, M.; Darville, L. N. F.; Izumi, V.; Chen, Y. A.; Pilon-Thomas, S.; Stewart, P. A.; Koomen, J. M.; Corallo, S. A.; Jain, M. D.; Robinson, T. J.; Locke, F. L.; Forsyth, P. A.; Smalley, I.</p>
        <p class="info">Score: 6.4, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572239' target='https://doi.org/10.1101/2023.12.18.572239'> 10.1101/2023.12.18.572239</a></p>
        <p class="abstract">Leptomeningeal disease (LMD) occurs when tumors seed into the leptomeningeal space and cerebrospinal fluid (CSF), leading to severe neurological deterioration and poor survival outcomes. We utilized comprehensive multi-omics analyses of CSF from patients with lymphoma LMD to demonstrate an immunosuppressive cellular microenvironment and identified dysregulations in proteins and lipids indicating neurodegenerative processes. Strikingly, we found a significant accumulation of toxic branched-chain keto acids (BCKA) in the CSF of patients with LMD. The BCKA accumulation was found to be a pan-cancer occurrence, evident in lymphoma, breast cancer, and melanoma LMD patients. Functionally, BCKA disrupted the viability and function of endogenous T lymphocytes, chimeric antigen receptor (CAR) T cells, neurons, and meningeal cells. Treatment of LMD mice with BCKA-reducing sodium phenylbutyrate significantly improved neurological function, survival outcomes, and efficacy of anti-CD19 CAR T cell therapy. This is the first report of BCKA accumulation in LMD and provides preclinical evidence that targeting these toxic metabolites improves outcomes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572266">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572266" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572266">
        <p class="paperTitle">Spatial transcriptomics analysis identifies a unique tumor-promoting function of the meningeal stroma in melanoma leptomeningeal disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572266" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572266" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alhaddad, H.; Ospina, O. E.; Khaled, M. L.; Ren, Y.; Forsyth, P. A.; Pina, Y.; Macaulay, R.; Law, V.; Tsai, K. Y.; Cress, W. D.; Fridley, B.; Smalley, I.</p>
        <p class="info">Score: 6.1, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572266' target='https://doi.org/10.1101/2023.12.18.572266'> 10.1101/2023.12.18.572266</a></p>
        <p class="abstract">Leptomeningeal disease (LMD) remains a rapidly lethal complication for late-stage melanoma patients. The inaccessible nature of the disease site and lack of understanding of the biology of this unique metastatic site are major barriers to developing efficacious therapies for patients with melanoma LMD. Here, we characterize the tumor microenvironment of the leptomeningeal tissues and patient-matched extra-cranial metastatic sites using spatial transcriptomic analyses with in vitro and in vivo validation. We show the spatial landscape of melanoma LMD to be characterized by a lack of immune infiltration and instead exhibit a higher level of stromal involvement. We show that the tumor-stroma interactions at the leptomeninges activate pathways implicated in tumor-promoting signaling, mediated through upregulation of SERPINA3 at the tumor-stroma interface. Our functional experiments establish that the meningeal stroma is required for melanoma cells to survive in the CSF environment and that these interactions lead to a lack of MAPK inhibitor sensitivity in the tumor. We show that knocking down SERPINA3 or inhibiting the downstream IGR1R/PI3K/AKT axis results in re-sensitization of the tumor to MAPK-targeting therapy and tumor cell death in the leptomeningeal environment. Our data provides a spatial atlas of melanoma LMD, identifies the tumor-promoting role of meningeal stroma, and demonstrates a mechanism for overcoming microenvironment-mediated drug resistance unique to this metastatic site.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573897">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573897" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573897">
        <p class="paperTitle">Hybrid cancer stem cells utilise vascular tracks for collective streaming invasion in a metastasis-on-a-chip device</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573897" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573897" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Scemama, A.; Lunetto, S.; Tailor, A.; Di Cio, S.; Ambler, L.; Coetzee, A.; Cottom, H.; Khurram, S. A.; Gautrot, J.; Biddle, A.</p>
        <p class="info">Score: 3.7, Published: 2024-01-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573897' target='https://doi.org/10.1101/2024.01.02.573897'> 10.1101/2024.01.02.573897</a></p>
        <p class="abstract">Cancer stem cells (CSCs) drive cancer metastatic dissemination. They do not do so in a vacuum, and the important influence of the tumour microenvironment (TME) on metastatic dissemination is becoming increasingly recognised. Therapeutic targeting of CSC-TME interactions may be a promising route to suppression of tumour metastasis. However, we must first understand how interactions with the TME influence CSC metastatic dissemination. To achieve this understanding, there is a need for experimental models that enable the analysis of dynamic interactions at single cell resolution within a complex environment. To this end, we utilise a metastasis-on-a-chip device to produce a 3D in vitro model of CSC interaction with a developing microvasculature, that is amenable to precise imaging and real time studies at single cell resolution. We show that the invasive phenotype of oral squamous cell carcinoma (OSCC) cells is markedly altered when in proximity to a microvasculature, with a switch to a hybrid CSC phenotype that undergoes collective streaming invasion. Mechanistically, ECM compression by the developing vasculature creates an environment that is refractory to cancer invasion, whilst leaving abandoned vascular tracks that can be utilised by hybrid CSCs for collective streaming invasion. Human tissue studies identify streaming invasion in association with vascularised regions in OSCC specimens. These findings elucidate the influence of the vasculature on CSC metastatic dissemination in OSCC, and the role of hybrid CSC invasion plasticity in overcoming this TME constraint.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.02.573967">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.02.573967" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.02.573967">
        <p class="paperTitle">Single-cell RNA-seq analysis profiling characterizes differences in cell composition and physiology between normal tissue, treatment-naive, and cisplatin-treated ovarian cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.02.573967" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.02.573967" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Guo, F.; Yang, Z.; Sehouli, J.; Kaufmann, A.</p>
        <p class="info">Score: 3.5, Published: 2024-01-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.02.573967' target='https://doi.org/10.1101/2024.01.02.573967'> 10.1101/2024.01.02.573967</a></p>
        <p class="abstract">BackgroundIntense efforts have focused on identifying heterogeneity of the cellular composition in ovarian cancer. However, tissue composition and physiological conditions of cancer cells in cisplatin-sensitive ovarian cancer remains largely unknown. Moreover, comparisons of different cellular states in normal tissue, in treatment naive ovarian cancer, and in cisplatin-treated tissue after adjuvant therapy of cisplatin-sensitive ovarian cancer at the single-cell level might offer clues for ovarian cancer treatment and prevention of cisplatin-resistance formation.

MethodsSingle-cell transcriptome sequencing of a cisplatin-treated ovarian cancer was performed. Data sets of non-tumorous ovarian tissues and treatment-naive ovarian cancer were downloaded from the European Genome-phenome Archive (accession number EGAS00001004987). Quality control, batch effect correction, integration, and clustering analysis of the integrated single-cell transcriptome data was done. Cell subsets were annotated based on surface marker phenotypes. Finally, the proportions of subclusters, the immune cell population, and the potential biological processes among different cellular states were compared.

ResultsSixteen distinct cell subsets were identified from the integrated single-cell transcriptome sequencing data of a pool of all tissues. The composition of the three different tissue types was characterized. The proportion of fibroblasts in cisplatin-treated ovarian tumor was remarkably lower than in treatment-naive ovarian tumor (1.33% vs. 13.53%, p &lt; 0.05). Moreover, each subjects sample had differing relative proportions of the identified cell types. In primary untreated ovarian cancer, the prevalent immune cells were B cells and myeloid-related immunosuppressive M2 macrophages. However, there were less B cells and myeloid-related immunosuppressive M2 macrophages after cisplatin-treatment, while significantly more T cells were found. The physiological cellular state in primary untreated ovarian tumors was associated with dysfunctional gene expression and modulation of cellular homeostasis, while cells from cisplatin-treated tumor showed more activation of immune and inflammatory genes as compared to healthy ovarian tissue.

ConclusionOur molecular gene expression analysis allowed for the separation and identification of differences in normal ovarian tissues, treatment-naive, and cisplatin-sensitive ovarian cancer cell populations at single-cell resolution. We identified different cell type composition and discriminative marker expression concerning specific cell subsets and identified differences among their physiological cell states. This knowledge may open new possibilities for elucidating important pathogenetic features and therapeutic strategies for treating ovarian cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571349">
        <p class="paperTitle">Breakage fusion bridge cycles drive high oncogene copy number, but not intratumoral genetic heterogeneity or rapid cancer genome change</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dehkordi, S. R.; Wong, I. T.-L.; Ni, J.; Luebeck, J.; Zhu, K.; Prasad, G.; Krockenberger, L.; Xu, G.; Chowdhury, B.; Rajkumar, U.; Caplin, A.; Muliaditan, D.; Coruh, C.; Jin, Q.; Turner, K.; Teo, S. X.; Pang, A. W. C.; Alexandrov, L.; Chua, C. E. L.; Furnari, F.; Paulson, T.; Law, J.; Chang, H.; Yue, F.; DasGupta, R.; Zhao, J.; Mischel, P. S.; Bafna, V.</p>
        <p class="info">Score: 4.0, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571349' target='https://doi.org/10.1101/2023.12.12.571349'> 10.1101/2023.12.12.571349</a></p>
        <p class="abstract">Oncogene amplification is a major driver of cancer pathogenesis. Breakage fusion bridge (BFB) cycles, like extrachromosomal DNA (ecDNA), can lead to high copy numbers of oncogenes, but their impact on intratumoral heterogeneity, treatment response, and patient survival are not well understood due to difficulty in detecting them by DNA sequencing. We describe a novel algorithm that detects and reconstructs BFB amplifications using optical genome maps (OGMs), called OM2BFB. OM2BFB showed high precision (&gt;93%) and recall (92%) in detecting BFB amplifications in cancer cell lines, PDX models and primary tumors. OM-based comparisons demonstrated that short-read BFB detection using our AmpliconSuite (AS) toolkit also achieved high precision, albeit with reduced sensitivity. We detected 371 BFB events using whole genome sequences from 2,557 primary tumors and cancer lines. BFB amplifications were preferentially found in cervical, head and neck, lung, and esophageal cancers, but rarely in brain cancers. BFB amplified genes show lower variance of gene expression, with fewer options for regulatory rewiring relative to ecDNA amplified genes. BFB positive (BFB (&#43;)) tumors showed reduced heterogeneity of amplicon structures, and delayed onset of resistance, relative to ecDNA(&#43;) tumors. EcDNA and BFB amplifications represent contrasting mechanisms to increase the copy numbers of oncogene with markedly different characteristics that suggest different routes for intervention.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.11.570862">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.11.570862" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.11.570862">
        <p class="paperTitle">Gene expression landscape of cutaneous squamous cell carcinoma progression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.11.570862" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.11.570862" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bencomo, T.; Lee, C. S.</p>
        <p class="info">Score: 4.3, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.11.570862' target='https://doi.org/10.1101/2023.12.11.570862'> 10.1101/2023.12.11.570862</a></p>
        <p class="abstract">BackgroundCutaneous squamous cell carcinomas (SCC) are the second most common human cancer and have been characterized by RNA sequencing (RNA-Seq); however, the transferability of findings from individual studies may be limited by small sample sizes and diverse analysis protocols.

ObjectivesTo define the transcriptome landscape at different stages in the progression of normal skin to SCC through a meta-analysis of publicly available RNA-Seq samples

MethodsWhole-transcriptome data from 73 normal skin samples, 46 actinic keratoses (AK), 16 in situ SCC, 13 keratoacanthomas (KA), and 147 SCC (including 30 SCC from immunocompromised patients and 8 SCC from individuals with recessive dystrophic epidermolysis bullosa [RDEB]) was uniformly processed to harmonize gene expression. Differential expression, fusion detection, and cell-type deconvolution analyses were performed.

ResultsIndividual RNA-Seq studies of SCC demonstrated study-specific clustering and varied widely in their differential gene expression detection. Following batch correction, we defined a consensus set of differentially expressed genes (DEGs), including those altered in the preinvasive stages of SCC development, and used single-cell RNA-Seq data to demonstrate that DEGs are often, but not always, expressed by tumor-specific keratinocytes (TSKs). Analysis of the cellular composition of SCC, KA, and RDEB-SCC identified an increase in differentiated keratinocytes in KA, while RDEB-SCC contained the most TSKs. Compared to SCC arising in immunocompetent patients, SCC from immunosuppressed individuals demonstrated fewer memory B cells and CD8 T cells. A comprehensive and unbiased search for fusion transcripts in SCC and intermediate disease stages identified few candidates that recur in &gt;1% of all specimens, suggesting most SCC are not driven by oncogenic gene fusions. Finally, using GTEx data, we distilled a novel 300-gene signature of chronic sun exposure that affirms greater cumulative ultraviolet (UV) exposure in later stages of SCC development.

ConclusionsOur results define the gene expression landscape of SCC progression, characterize cell subpopulation heterogeneity in SCC subtypes that contribute to their distinct clinical phenotypes, demonstrate that gene fusions are not a common cause of SCC, and identify UV-responsive genes associated with SCC development.

What is already known about this topic?O_LICutaneous squamous cell carcinoma (SCC) is one of the most common cancers worldwide.
C_LIO_LISeveral studies have examined gene expression changes in SCC using RNA sequencing (RNA-Seq) but comparison of their results is difficult due to inter-study variation and diverse bioinformatic pipelines and protocols.
C_LIO_LIA few gene fusions have been described in SCC, but a comprehensive characterization of fusion transcripts in patient samples has not been performed.
C_LI

What does this study add?O_LIWe re-analyzed RNA-Seq data from 11 studies of SCC and its preinvasive stages to create a list of consensus differentially expressed genes and identify those that are UV-responsive.
C_LIO_LIClinically aggressive SCC displayed more tumor-specific keratinocytes, while keratoacanthomas contained more differentiated keratinocytes. SCC in immunocompetent persons had more memory B cells and CD8 T cells than those arising in immunosuppressed individuals.
C_LIO_LIPreviously reported gene fusions were not detected and most fusion candidates did not demonstrate pathogenic features.
C_LI

What is the translational message?O_LIOur analysis harmonizes differing results from previous studies to provide a robust list of genes implicated in SCC development.
C_LIO_LIOur findings suggest gene fusions are not a common driver event in SCC.
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
