<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deciphering the diversity and sequence of extracellular matrix and cellular spatial patterns in lung adenocarcinoma using topological data analysis
Authors: Yoon, I. H. R.; Jenkins, R.; Colliver, E.; Zhang, H.; Novo, D.; Moore, D.; Ramsden, Z.; Rullan, A.; Fu, X.; Yuan, Y.; Harrington, H. A.; Swanton, C.; Byrne, H. M.; Sahai, E.
Score: 31.1, Published: 2024-01-17 DOI: 10.1101/2024.01.05.574362
Extracellular matrix (ECM) organization influences cancer development and progression. It modulates the invasion of cancer cells and can hinder the access of immune cells to cancer cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Deciphering the diversity and sequence of extracellular matrix and cellular spatial patterns in lung adenocarcinoma using topological data analysis
Authors: Yoon, I. H. R.; Jenkins, R.; Colliver, E.; Zhang, H.; Novo, D.; Moore, D.; Ramsden, Z.; Rullan, A.; Fu, X.; Yuan, Y.; Harrington, H. A.; Swanton, C.; Byrne, H. M.; Sahai, E.
Score: 31.1, Published: 2024-01-17 DOI: 10.1101/2024.01.05.574362
Extracellular matrix (ECM) organization influences cancer development and progression. It modulates the invasion of cancer cells and can hinder the access of immune cells to cancer cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-24T10:40:04+00:00" />
<meta property="article:modified_time" content="2024-01-24T10:40:04+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Deciphering the diversity and sequence of extracellular matrix and cellular spatial patterns in lung adenocarcinoma using topological data analysis
Authors: Yoon, I. H. R.; Jenkins, R.; Colliver, E.; Zhang, H.; Novo, D.; Moore, D.; Ramsden, Z.; Rullan, A.; Fu, X.; Yuan, Y.; Harrington, H. A.; Swanton, C.; Byrne, H. M.; Sahai, E.
Score: 31.1, Published: 2024-01-17 DOI: 10.1101/2024.01.05.574362
Extracellular matrix (ECM) organization influences cancer development and progression. It modulates the invasion of cancer cells and can hinder the access of immune cells to cancer cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Deciphering the diversity and sequence of extracellular matrix and cellular spatial patterns in lung adenocarcinoma using topological data analysis\nAuthors: Yoon, I. H. R.; Jenkins, R.; Colliver, E.; Zhang, H.; Novo, D.; Moore, D.; Ramsden, Z.; Rullan, A.; Fu, X.; Yuan, Y.; Harrington, H. A.; Swanton, C.; Byrne, H. M.; Sahai, E.\nScore: 31.1, Published: 2024-01-17 DOI: 10.1101/2024.01.05.574362\nExtracellular matrix (ECM) organization influences cancer development and progression. It modulates the invasion of cancer cells and can hinder the access of immune cells to cancer cells.",
  "keywords": [
    
  ],
  "articleBody": " Deciphering the diversity and sequence of extracellular matrix and cellular spatial patterns in lung adenocarcinoma using topological data analysis\nAuthors: Yoon, I. H. R.; Jenkins, R.; Colliver, E.; Zhang, H.; Novo, D.; Moore, D.; Ramsden, Z.; Rullan, A.; Fu, X.; Yuan, Y.; Harrington, H. A.; Swanton, C.; Byrne, H. M.; Sahai, E.\nScore: 31.1, Published: 2024-01-17 DOI: 10.1101/2024.01.05.574362\nExtracellular matrix (ECM) organization influences cancer development and progression. It modulates the invasion of cancer cells and can hinder the access of immune cells to cancer cells. Effective quantification of ECM architecture and its relationship to the position of different cell types is, therefore, important when investigating the role of ECM in cancer development. Using topological data analysis (TDA), particularly persistent homology and Dowker persistent homology, we develop a novel analysis pipeline for quantifying ECM architecture, spatial patterns of cell positions, and the spatial relationships between distinct constituents of the tumour microenvironment. We apply the pipeline to 44 surgical specimens of lung adenocarcinoma from the lung TRACERx study stained with picrosirius red and haematoxylin. We show that persistent homology effectively encodes the architectural features of the tumour microenvironment. Inference using pseudo-time analysis and spatial mapping to centimetre scale tissues suggests a gradual and progressive route of change in ECM architecture, with two different end states. Dowker persistent homology enables the analysis of spatial relationship between any pair of constituents of the tumour microenvironment, such as ECM, cancer cells, and leukocytes. We use Dowker persistent homology to quantify the spatial segregation of cancer and immune cells over different length scales. A combined analysis of both topological and non-topological features of the tumour microenvironment indicates that progressive changes in the ECM are linked to increased immune exclusion and reduced oxidative metabolism.\nChromosome 7 to the rescue: overcoming chromosome 10 loss in gliomas\nAuthors: Nair, N. U.; Schaffer, A. A.; Gertz, M. E.; Cheng, K.; Zerbib, J.; Sahu, A. D.; Leor, G.; Shulman, E. D.; Aldape, K. D.; Ben-David, U.; Ruppin, E.\nScore: 23.7, Published: 2024-01-22 DOI: 10.1101/2024.01.17.576103\nThe co-occurrence of chromosome 10 loss and chromosome 7 gain in gliomas is the most frequent loss-gain co-aneuploidy pair in human cancers, a phenomenon that has been investigated without resolution since the late 1980s. Expanding beyond previous gene-centric studies, we investigate the co-occurrence in a genome-wide manner taking an evolutionary perspective. First, by mining large tumor aneuploidy data, we predict that the more likely order is 10 loss followed by 7 gain. Second, by analyzing extensive genomic and transcriptomic data from both patients and cell lines, we find that this co-occurrence can be explained by functional rescue interactions that are highly enriched on 7, which can possibly compensate for any detrimental consequences arising from the loss of 10. Finally, by analyzing transcriptomic data from normal, non-cancerous, human brain tissues, we provide a plausible reason why this co-occurrence happens preferentially in cancers originating in certain regions of the brain.\nCombining RASG12C(ON) inhibitor with SHP2 inhibition sensitises immune excluded lung tumours to immune checkpoint blockade: a strategy for turning cold tumours hot\nAuthors: Anastasiou, P.; Moore, C.; Rana, S.; de Castro, A.; Tomaschko, M.; Boumelha, J.; Mugarza, E.; Blaj, C.; de Carne Trecesson, S.; Goldstone, R.; Smith, J.; Quintana, E.; Molina-Arcas, M.; Downward, J.\nScore: 20.1, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575765\nMutant selective drugs targeting the inactive, GDP-bound form of KRASG12C have been approved for use in lung cancer, but responses are short-lived due to rapid development of resistance. In this study we use a novel covalent tri-complex inhibitor, RMC-4998, that targets RASG12C in its active, GTP-bound form to investigate treatment of KRAS mutant lung cancer in various immune competent mouse models. While this RASG12C(ON) inhibitor was more potent than the KRASG12C(OFF) inhibitor adagrasib, rapid pathway reactivation was still observed. This could be delayed using combined treatment with a SHP2 inhibitor, RMC-4550, which not only impacted RAS pathway signalling within the tumour cells but also remodelled the tumour microenvironment (TME) to be less immunosuppressive and promoted interferon responses. In an inflamed, \"hot\", mouse model of lung cancer, RASG12C(ON) and SHP2 inhibitors in combination drive durable responses by suppressing tumour relapse and inducing development of immune memory, which can also be induced by combination of RASG12C(ON) and PD-1 inhibitors. In contrast, in an immune excluded, \"cold\", mouse model of lung cancer, combined RASG12C(ON) and SHP2 inhibition does not cause durable responses, but does sensitise tumours to immune checkpoint blockade, enabling efficient tumour rejection, accompanied by significant TME reorganization, including depletion of immunosuppressive innate immune cells and recruitment and activation of T and NK cells. These preclinical results demonstrate the potential of the combination of RASG12C(ON) inhibitors with SHP2 inhibitors to sensitize anti-PD-1 refractory tumours to immune checkpoint blockade by stimulating anti-tumour immunity as well as by targeting KRAS-driven proliferation in tumour cells.\nDiscovery of Highly Active Kynureninases for Cancer Immunotherapy through Protein Language Model\nAuthors: Eom, H.; Cho, K. S.; Lee, J.; Kim, S.; Park, S.; Kim, H.; Yang, J.; Han, Y.-H.; Lee, J.; Seok, C.; Lee, M. S.; Song, W. J.; Steinegger, M.\nScore: 6.9, Published: 2024-01-20 DOI: 10.1101/2024.01.16.575968\nOvercoming the immunosuppressive tumor microenvironment is a promising strategy in anticancer therapy. L-kynurenine, a strong immunosuppressive metabolite can be degraded through kynureninases. Through homology searches and protein language models, we identified and then experimentally determined the efficacy of four top-ranked kynureninases. The catalytically most active one nearly doubles turnover number over the prior best, reducing tumor weight by 3.42 times in mouse model comparisons, and thus, presenting substantial therapeutic potential.\nDivergent trajectories to structural diversity impact patient survival in high grade serous ovarian cancer\nAuthors: Ewing, A.; Meynert, A.; Silk, R.; Aitken, S.; Bendixsen, D. P.; Churchman, M.; Brown, S. L.; Hamdan, A.; Mattocks, J.; Grimes, G. R.; Ballinger, T.; Hollis, R. L.; Herrington, C. S.; Thomson, J. P.; Sherwood, K.; Parry, T.; Esiri-Bloom, E.; Bartos, C.; Croy, I.; Ferguson, M.; Lennie, M.; McGoldrick, T.; McPhail, N.; Siddiqui, N.; Glasspool, R.; Mackean, M.; Nussey, F.; McDade, B.; Ennis, D.; The Scottish Genomes Partnership, ; McMahon, L.; Matakidou, A.; Dougherty, B.; March, R.; Barrett, J. C.; McNeish, I. A.; Biankin, A. V.; Roxburgh, P.; Gourley, C.; Semple, C. A.\nScore: 14.7, Published: 2024-01-15 DOI: 10.1101/2024.01.12.575376\nDeciphering the structural variation across tumour genomes is crucial to determine the events driving tumour progression and better understand tumour adaptation and evolution. High grade serous ovarian cancer (HGSOC) is an exemplar tumour type showing extreme, but poorly characterised structural diversity. We comprehensively describe the mutational landscape driving HGSOC, exploiting a large (N=324), deeply whole genome sequenced dataset. We reveal two divergent evolutionary trajectories, affecting patient survival and involving differing genomic environments. One involves homologous recombination repair deficiency (HRD) while the other is dominated by whole genome duplication (WGD) with frequent chromothripsis, breakage-fusion-bridges and extra-chromosomal DNA. These trajectories contribute to structural variation hotspots, containing novel candidate driver genes with significantly altered expression. While structural variation predominantly drives tumorigenesis, we also find high mtDNA mutation loads associated with shorter patient survival, and acting in combination with alterations in the nuclear genome to impact prognosis and suggesting new strategies for patient stratification.\nIntegrated cancer cell-specific single-cell RNA-seq datasets of immune checkpoint blockade-treated patients\nAuthors: Gondal, M. N.; Cieslik, M.; Chinnaiyan, A. M.\nScore: 6.0, Published: 2024-01-22 DOI: 10.1101/2024.01.17.576110\nImmune checkpoint blockade (ICB) therapies have emerged as a promising avenue for the treatment of various cancers. Despite their success, the efficacy of these treatments is variable across patients and cancer types. Numerous single-cell RNA-sequencing (scRNA-seq) studies have been conducted to unravel cell-specific responses to ICB treatment. However, these studies are limited in their sample sizes and require advanced coding skills for exploration. Here, we have compiled eight scRNA-seq datasets from nine cancer types, encompassing 174 patients, and 90,270 cancer cells. This compilation forms a unique resource tailored for investigating how cancer cells respond to ICB treatment across cancer types. We meticulously curated, quality-checked, pre-processed, and analyzed the data, ensuring easy access for researchers. Moreover, we designed a user-friendly interface for seamless exploration. By sharing the code and data for creating these interfaces, we aim to assist fellow researchers. These resources offer valuable support to those interested in leveraging and exploring single-cell datasets across diverse cancer types, facilitating a comprehensive understanding of ICB responses.\nDirect excitatory synapses between neurons and tumor cells drive brain metastatic seeding of breast cancer and melanoma\nAuthors: Venkataramani, V.; Karreman, M. A.; Nguyen, L. C.; Tehranian, C.; Hebach, N.; Mayer, C.; Meyer, L.; Mughal, S.; Reifenberger, G.; Felsberg, J.; Koehrer, K.; Schubert, M. C.; Westphal, D.; Breckwolt, M.; Brors, B.; Wick, W.; Kuner, T.; Winkler, F.\nScore: 148.1, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574608\nInteractions between neurons and cancer cells are found in many malignancies, but their relevance for metastatic organ colonization remain largely unknown. It is also unclear whether any direct synaptic communication between neurons and cancer cells of non-neural tumor types exists, and if so, whether this can support metastasis and thus cancer progression. Here we show that excitatory synapses are formed between neurons and brain-metastatic melanoma and breast cancer cells. This starts at an early microscopic stage after extravasation into the brain parenchyma, during residence of cancer cells in the perivascular niche, a critical step for their survival. These neuron-cancer synapses showed a bona fide synaptic ultrastructure, and generated excitatory postsynaptic currents mediated by glutamate receptors of the AMPA subtype in cancer cells. In accordance, AMPA receptor signatures were consistently detected in preclinical and patient samples of melanoma and breast cancer brain metastases. Genetic perturbation and pharmacological inhibition of AMPA receptors with the approved antiepileptic drug perampanel in models of breast and melanoma cancer reduced the number of brain metastases and overall brain metastatic burden. These findings demonstrate for the first time that neurons can form biologically relevant direct synapses with non-neural cancer cells. In brain metastasis, a particularly challenging complication of many common malignancies, this non-canonical stimulatory synaptic interaction offers novel therapeutic opportunities.\nUtilizing a dual endogenous reporter system to identify functional regulators of aberrant stem cell and differentiation activity in colorectal cancer\nAuthors: Spisak, S.; Chen, D.; Likasitwatanakul, P.; Doan, P.; Li, Z.; Bala, P.; Vizkeleti, L.; De Silva, P.; Giannakis, M.; Wolpin, B.; Qi, J.; Sethi, N.\nScore: 4.2, Published: 2024-01-17 DOI: 10.1101/2023.06.21.545895\nAberrant stem cell-like activity and impaired differentiation are central to the development of colorectal cancer (CRC). To identify functional mediators that regulate these key cellular programs in CRC, we developed an endogenous reporter system by genome-editing human CRC cell lines with knock-in fluorescent reporters at the SOX9 and KRT20 locus to report aberrant stem cell-like activity and differentiation, respectively, and then performed pooled genetic perturbation screens. Constructing a dual reporter system that simultaneously monitored aberrant stem cell-like and differentiation activity in the same CRC cell line improved our signal to noise discrimination. Using a focused-library CRISPR screen targeting 78 epigenetic regulators with 542 sgRNAs, we identified factors that contribute to stem cell-like activity and differentiation in CRC. Perturbation single cell RNA sequencing (Perturb-seq) of validated hits nominated SMARCB1 of the BAF complex (also known as SWI/SNF) as a negative regulator of differentiation across an array of neoplastic colon models. SMARCB1 is a dependency in CRC and required for in vivo growth of human CRC models. These studies highlight the utility of a biologically designed endogenous reporter system to uncover novel therapeutic targets for drug development.\nCharacterization of a novel humanized heavy chain antibody targeting endogenous retroviruses with anti-lymphoma activity\nAuthors: Spriano, F.; Cascione, L.; Sgrignani, J.; Bendik, N.; Napoli, S.; Sartori, G.; Cannas, E.; Gong, T.; Arribas, A. J.; Pizzi, M.; Rossi, D.; Robbiani, D. F.; Cavalli, A.; Bertoni, F.\nScore: 3.2, Published: 2024-01-18 DOI: 10.1101/2024.01.17.576027\nLymphomas continue to pose therapeutic challenges, with a considerable portion of patients facing refractory disease. This study focuses on Diffuse Large B-cell Lymphoma (DLBCL), the most prevalent lymphoma type. Within the human genome, transposable elements (TEs), particularly Human Endogenous Retroviruses (HERVs), constitute a significant yet understudied portion. Among HERVs, the HERV-K family, specifically HERV-K113 and HERV-K115, has intact open reading frames. Epigenetic regulation tightly controls HERV expression, and aberrant expression has been observed in various cancers, including lymphomas. This research investigates the potential of HERV-K as a therapeutic target in DLBCL. The study encompasses comprehensive methods, including RNA extraction, PCR detection, flow cytometry, immunoblotting, peptide prediction, phage display, surface plasmon resonance, ELISA, antibody-dependent cell-mediated cytotoxicity, internalization assays, and bioinformatic analysis. Results reveal the presence and expression of HERVs in lymphoma patients and cell lines, with the HERV-K envelope protein identified as a crucial contributor to lymphoma cell growth. Moreover, the study identifies immunogenic regions of HERV-K, leading to the development of a humanized camelid nanobody (FF-01) with potential therapeutic applications. Furthermore, bioinformatic analysis differentiates DLBCL subgroups based on TE expression, providing insights into prognostic variations. Patients with high HERV-K113 expression show activation of pathways related to antiviral responses, suggesting a viral mimicry state. In conclusion, the study highlights the clinical relevance of HERVs in lymphomas, proposing them as novel therapeutic targets. The newly developed nanobody FF-01 demonstrates anti-lymphoma activity through antibody-dependent cellular cytotoxicity and internalization. This research opens avenues for exploring endogenous retroviruses as targets for immunotherapy in lymphomas, showcasing the potential of FF-01 as a promising candidate for further investigation.\nCell polarity proteins promote macropinocytosis in response to metabolic stress\nAuthors: Lambies, G.; Lee, S.-W.; Duong-Polk, K.; Aza-Blanc, P.; Maganti, S.; Dawson, D. W.; Commisso, C.\nScore: 2.9, Published: 2024-01-20 DOI: 10.1101/2024.01.16.575943\nMacropinocytosis has emerged as a nutrient-scavenging pathway that cancer cells exploit to survive the nutrient-deprived conditions of the tumor microenvironment. Cancer cells are especially reliant on glutamine for their survival, and in pancreatic ductal adenocarcinoma (PDAC) cells, glutamine deficiency can enhance the stimulation of macropinocytosis, allowing the cells to escape metabolic stress through the production of extracellular-protein-derived amino acids. Here, we identify the atypical protein kinase C (aPKC) enzymes, PKC{zeta} and PKC{iota}, as novel regulators of macropinocytosis. In normal epithelial cells, aPKCs are known to regulate cell polarity in association with the scaffold proteins Par3 and Par6, controlling the function of several targets, including the Par1 kinases. In PDAC cells, we identify that each of these cell polarity proteins are required for glutamine stress-induced macropinocytosis. Mechanistically, we find that the aPKCs are regulated by EGFR signaling or by the transcription factor CREM to promote the relocation of Par3 to microtubules, facilitating macropinocytosis in a dynein-dependent manner. Importantly, we determine that cell fitness impairment caused by aPKC depletion is rescued by the restoration of macropinocytosis and that aPKCs support PDAC growth in vivo. These results identify a previously unappreciated role for cell polarity proteins in the regulation of macropinocytosis and provide a better understanding of the mechanistic underpinnings that control macropinocytic uptake in the context of metabolic stress.\n",
  "wordCount" : "2437",
  "inLanguage": "en",
  "datePublished": "2024-01-24T10:40:04Z",
  "dateModified": "2024-01-24T10:40:04Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 24, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574362">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574362" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574362">
        <p class="paperTitle">Deciphering the diversity and sequence of extracellular matrix and cellular spatial patterns in lung adenocarcinoma using topological data analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574362" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574362" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yoon, I. H. R.; Jenkins, R.; Colliver, E.; Zhang, H.; Novo, D.; Moore, D.; Ramsden, Z.; Rullan, A.; Fu, X.; Yuan, Y.; Harrington, H. A.; Swanton, C.; Byrne, H. M.; Sahai, E.</p>
        <p class="info">Score: 31.1, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574362' target='https://doi.org/10.1101/2024.01.05.574362'> 10.1101/2024.01.05.574362</a></p>
        <p class="abstract">Extracellular matrix (ECM) organization influences cancer development and progression. It modulates the invasion of cancer cells and can hinder the access of immune cells to cancer cells. Effective quantification of ECM architecture and its relationship to the position of different cell types is, therefore, important when investigating the role of ECM in cancer development. Using topological data analysis (TDA), particularly persistent homology and Dowker persistent homology, we develop a novel analysis pipeline for quantifying ECM architecture, spatial patterns of cell positions, and the spatial relationships between distinct constituents of the tumour microenvironment. We apply the pipeline to 44 surgical specimens of lung adenocarcinoma from the lung TRACERx study stained with picrosirius red and haematoxylin. We show that persistent homology effectively encodes the architectural features of the tumour microenvironment. Inference using pseudo-time analysis and spatial mapping to centimetre scale tissues suggests a gradual and progressive route of change in ECM architecture, with two different end states. Dowker persistent homology enables the analysis of spatial relationship between any pair of constituents of the tumour microenvironment, such as ECM, cancer cells, and leukocytes. We use Dowker persistent homology to quantify the spatial segregation of cancer and immune cells over different length scales. A combined analysis of both topological and non-topological features of the tumour microenvironment indicates that progressive changes in the ECM are linked to increased immune exclusion and reduced oxidative metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576103">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576103" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576103">
        <p class="paperTitle">Chromosome 7 to the rescue: overcoming chromosome 10 loss in gliomas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576103" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576103" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nair, N. U.; Schaffer, A. A.; Gertz, M. E.; Cheng, K.; Zerbib, J.; Sahu, A. D.; Leor, G.; Shulman, E. D.; Aldape, K. D.; Ben-David, U.; Ruppin, E.</p>
        <p class="info">Score: 23.7, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576103' target='https://doi.org/10.1101/2024.01.17.576103'> 10.1101/2024.01.17.576103</a></p>
        <p class="abstract">The co-occurrence of chromosome 10 loss and chromosome 7 gain in gliomas is the most frequent loss-gain co-aneuploidy pair in human cancers, a phenomenon that has been investigated without resolution since the late 1980s. Expanding beyond previous gene-centric studies, we investigate the co-occurrence in a genome-wide manner taking an evolutionary perspective. First, by mining large tumor aneuploidy data, we predict that the more likely order is 10 loss followed by 7 gain. Second, by analyzing extensive genomic and transcriptomic data from both patients and cell lines, we find that this co-occurrence can be explained by functional rescue interactions that are highly enriched on 7, which can possibly compensate for any detrimental consequences arising from the loss of 10. Finally, by analyzing transcriptomic data from normal, non-cancerous, human brain tissues, we provide a plausible reason why this co-occurrence happens preferentially in cancers originating in certain regions of the brain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575765" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575765">
        <p class="paperTitle">Combining RASG12C(ON) inhibitor with SHP2 inhibition sensitises immune excluded lung tumours to immune checkpoint blockade: a strategy for turning cold tumours hot</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Anastasiou, P.; Moore, C.; Rana, S.; de Castro, A.; Tomaschko, M.; Boumelha, J.; Mugarza, E.; Blaj, C.; de Carne Trecesson, S.; Goldstone, R.; Smith, J.; Quintana, E.; Molina-Arcas, M.; Downward, J.</p>
        <p class="info">Score: 20.1, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575765' target='https://doi.org/10.1101/2024.01.15.575765'> 10.1101/2024.01.15.575765</a></p>
        <p class="abstract">Mutant selective drugs targeting the inactive, GDP-bound form of KRASG12C have been approved for use in lung cancer, but responses are short-lived due to rapid development of resistance. In this study we use a novel covalent tri-complex inhibitor, RMC-4998, that targets RASG12C in its active, GTP-bound form to investigate treatment of KRAS mutant lung cancer in various immune competent mouse models. While this RASG12C(ON) inhibitor was more potent than the KRASG12C(OFF) inhibitor adagrasib, rapid pathway reactivation was still observed. This could be delayed using combined treatment with a SHP2 inhibitor, RMC-4550, which not only impacted RAS pathway signalling within the tumour cells but also remodelled the tumour microenvironment (TME) to be less immunosuppressive and promoted interferon responses. In an inflamed, &#34;hot&#34;, mouse model of lung cancer, RASG12C(ON) and SHP2 inhibitors in combination drive durable responses by suppressing tumour relapse and inducing development of immune memory, which can also be induced by combination of RASG12C(ON) and PD-1 inhibitors. In contrast, in an immune excluded, &#34;cold&#34;, mouse model of lung cancer, combined RASG12C(ON) and SHP2 inhibition does not cause durable responses, but does sensitise tumours to immune checkpoint blockade, enabling efficient tumour rejection, accompanied by significant TME reorganization, including depletion of immunosuppressive innate immune cells and recruitment and activation of T and NK cells. These preclinical results demonstrate the potential of the combination of RASG12C(ON) inhibitors with SHP2 inhibitors to sensitize anti-PD-1 refractory tumours to immune checkpoint blockade by stimulating anti-tumour immunity as well as by targeting KRAS-driven proliferation in tumour cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575968">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575968" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575968">
        <p class="paperTitle">Discovery of Highly Active Kynureninases for Cancer Immunotherapy through Protein Language Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575968" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575968" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eom, H.; Cho, K. S.; Lee, J.; Kim, S.; Park, S.; Kim, H.; Yang, J.; Han, Y.-H.; Lee, J.; Seok, C.; Lee, M. S.; Song, W. J.; Steinegger, M.</p>
        <p class="info">Score: 6.9, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575968' target='https://doi.org/10.1101/2024.01.16.575968'> 10.1101/2024.01.16.575968</a></p>
        <p class="abstract">Overcoming the immunosuppressive tumor microenvironment is a promising strategy in anticancer therapy. L-kynurenine, a strong immunosuppressive metabolite can be degraded through kynureninases. Through homology searches and protein language models, we identified and then experimentally determined the efficacy of four top-ranked kynureninases. The catalytically most active one nearly doubles turnover number over the prior best, reducing tumor weight by 3.42 times in mouse model comparisons, and thus, presenting substantial therapeutic potential.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.12.575376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.12.575376" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.12.575376">
        <p class="paperTitle">Divergent trajectories to structural diversity impact patient survival in high grade serous ovarian cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.12.575376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.12.575376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ewing, A.; Meynert, A.; Silk, R.; Aitken, S.; Bendixsen, D. P.; Churchman, M.; Brown, S. L.; Hamdan, A.; Mattocks, J.; Grimes, G. R.; Ballinger, T.; Hollis, R. L.; Herrington, C. S.; Thomson, J. P.; Sherwood, K.; Parry, T.; Esiri-Bloom, E.; Bartos, C.; Croy, I.; Ferguson, M.; Lennie, M.; McGoldrick, T.; McPhail, N.; Siddiqui, N.; Glasspool, R.; Mackean, M.; Nussey, F.; McDade, B.; Ennis, D.; The Scottish Genomes Partnership,  ; McMahon, L.; Matakidou, A.; Dougherty, B.; March, R.; Barrett, J. C.; McNeish, I. A.; Biankin, A. V.; Roxburgh, P.; Gourley, C.; Semple, C. A.</p>
        <p class="info">Score: 14.7, Published: 2024-01-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.12.575376' target='https://doi.org/10.1101/2024.01.12.575376'> 10.1101/2024.01.12.575376</a></p>
        <p class="abstract">Deciphering the structural variation across tumour genomes is crucial to determine the events driving tumour progression and better understand tumour adaptation and evolution. High grade serous ovarian cancer (HGSOC) is an exemplar tumour type showing extreme, but poorly characterised structural diversity. We comprehensively describe the mutational landscape driving HGSOC, exploiting a large (N=324), deeply whole genome sequenced dataset. We reveal two divergent evolutionary trajectories, affecting patient survival and involving differing genomic environments. One involves homologous recombination repair deficiency (HRD) while the other is dominated by whole genome duplication (WGD) with frequent chromothripsis, breakage-fusion-bridges and extra-chromosomal DNA. These trajectories contribute to structural variation hotspots, containing novel candidate driver genes with significantly altered expression. While structural variation predominantly drives tumorigenesis, we also find high mtDNA mutation loads associated with shorter patient survival, and acting in combination with alterations in the nuclear genome to impact prognosis and suggesting new strategies for patient stratification.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576110">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576110" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576110">
        <p class="paperTitle">Integrated cancer cell-specific single-cell RNA-seq datasets of immune checkpoint blockade-treated patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576110" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576110" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gondal, M. N.; Cieslik, M.; Chinnaiyan, A. M.</p>
        <p class="info">Score: 6.0, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576110' target='https://doi.org/10.1101/2024.01.17.576110'> 10.1101/2024.01.17.576110</a></p>
        <p class="abstract">Immune checkpoint blockade (ICB) therapies have emerged as a promising avenue for the treatment of various cancers. Despite their success, the efficacy of these treatments is variable across patients and cancer types. Numerous single-cell RNA-sequencing (scRNA-seq) studies have been conducted to unravel cell-specific responses to ICB treatment. However, these studies are limited in their sample sizes and require advanced coding skills for exploration. Here, we have compiled eight scRNA-seq datasets from nine cancer types, encompassing 174 patients, and 90,270 cancer cells. This compilation forms a unique resource tailored for investigating how cancer cells respond to ICB treatment across cancer types. We meticulously curated, quality-checked, pre-processed, and analyzed the data, ensuring easy access for researchers. Moreover, we designed a user-friendly interface for seamless exploration. By sharing the code and data for creating these interfaces, we aim to assist fellow researchers. These resources offer valuable support to those interested in leveraging and exploring single-cell datasets across diverse cancer types, facilitating a comprehensive understanding of ICB responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574608">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574608" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574608">
        <p class="paperTitle">Direct excitatory synapses between neurons and tumor cells drive brain metastatic seeding of breast cancer and melanoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574608" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574608" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venkataramani, V.; Karreman, M. A.; Nguyen, L. C.; Tehranian, C.; Hebach, N.; Mayer, C.; Meyer, L.; Mughal, S.; Reifenberger, G.; Felsberg, J.; Koehrer, K.; Schubert, M. C.; Westphal, D.; Breckwolt, M.; Brors, B.; Wick, W.; Kuner, T.; Winkler, F.</p>
        <p class="info">Score: 148.1, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574608' target='https://doi.org/10.1101/2024.01.08.574608'> 10.1101/2024.01.08.574608</a></p>
        <p class="abstract">Interactions between neurons and cancer cells are found in many malignancies, but their relevance for metastatic organ colonization remain largely unknown. It is also unclear whether any direct synaptic communication between neurons and cancer cells of non-neural tumor types exists, and if so, whether this can support metastasis and thus cancer progression. Here we show that excitatory synapses are formed between neurons and brain-metastatic melanoma and breast cancer cells. This starts at an early microscopic stage after extravasation into the brain parenchyma, during residence of cancer cells in the perivascular niche, a critical step for their survival. These neuron-cancer synapses showed a bona fide synaptic ultrastructure, and generated excitatory postsynaptic currents mediated by glutamate receptors of the AMPA subtype in cancer cells. In accordance, AMPA receptor signatures were consistently detected in preclinical and patient samples of melanoma and breast cancer brain metastases. Genetic perturbation and pharmacological inhibition of AMPA receptors with the approved antiepileptic drug perampanel in models of breast and melanoma cancer reduced the number of brain metastases and overall brain metastatic burden. These findings demonstrate for the first time that neurons can form biologically relevant direct synapses with non-neural cancer cells. In brain metastasis, a particularly challenging complication of many common malignancies, this non-canonical stimulatory synaptic interaction offers novel therapeutic opportunities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545895">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545895" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545895">
        <p class="paperTitle">Utilizing a dual endogenous reporter system to identify functional regulators of aberrant stem cell and differentiation activity in colorectal cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545895" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545895" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Spisak, S.; Chen, D.; Likasitwatanakul, P.; Doan, P.; Li, Z.; Bala, P.; Vizkeleti, L.; De Silva, P.; Giannakis, M.; Wolpin, B.; Qi, J.; Sethi, N.</p>
        <p class="info">Score: 4.2, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545895' target='https://doi.org/10.1101/2023.06.21.545895'> 10.1101/2023.06.21.545895</a></p>
        <p class="abstract">Aberrant stem cell-like activity and impaired differentiation are central to the development of colorectal cancer (CRC). To identify functional mediators that regulate these key cellular programs in CRC, we developed an endogenous reporter system by genome-editing human CRC cell lines with knock-in fluorescent reporters at the SOX9 and KRT20 locus to report aberrant stem cell-like activity and differentiation, respectively, and then performed pooled genetic perturbation screens. Constructing a dual reporter system that simultaneously monitored aberrant stem cell-like and differentiation activity in the same CRC cell line improved our signal to noise discrimination. Using a focused-library CRISPR screen targeting 78 epigenetic regulators with 542 sgRNAs, we identified factors that contribute to stem cell-like activity and differentiation in CRC. Perturbation single cell RNA sequencing (Perturb-seq) of validated hits nominated SMARCB1 of the BAF complex (also known as SWI/SNF) as a negative regulator of differentiation across an array of neoplastic colon models. SMARCB1 is a dependency in CRC and required for in vivo growth of human CRC models. These studies highlight the utility of a biologically designed endogenous reporter system to uncover novel therapeutic targets for drug development.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576027">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576027" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576027">
        <p class="paperTitle">Characterization of a novel humanized heavy chain antibody targeting endogenous retroviruses with anti-lymphoma activity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576027" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576027" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Spriano, F.; Cascione, L.; Sgrignani, J.; Bendik, N.; Napoli, S.; Sartori, G.; Cannas, E.; Gong, T.; Arribas, A. J.; Pizzi, M.; Rossi, D.; Robbiani, D. F.; Cavalli, A.; Bertoni, F.</p>
        <p class="info">Score: 3.2, Published: 2024-01-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576027' target='https://doi.org/10.1101/2024.01.17.576027'> 10.1101/2024.01.17.576027</a></p>
        <p class="abstract">Lymphomas continue to pose therapeutic challenges, with a considerable portion of patients facing refractory disease. This study focuses on Diffuse Large B-cell Lymphoma (DLBCL), the most prevalent lymphoma type. Within the human genome, transposable elements (TEs), particularly Human Endogenous Retroviruses (HERVs), constitute a significant yet understudied portion. Among HERVs, the HERV-K family, specifically HERV-K113 and HERV-K115, has intact open reading frames. Epigenetic regulation tightly controls HERV expression, and aberrant expression has been observed in various cancers, including lymphomas.

This research investigates the potential of HERV-K as a therapeutic target in DLBCL. The study encompasses comprehensive methods, including RNA extraction, PCR detection, flow cytometry, immunoblotting, peptide prediction, phage display, surface plasmon resonance, ELISA, antibody-dependent cell-mediated cytotoxicity, internalization assays, and bioinformatic analysis. Results reveal the presence and expression of HERVs in lymphoma patients and cell lines, with the HERV-K envelope protein identified as a crucial contributor to lymphoma cell growth. Moreover, the study identifies immunogenic regions of HERV-K, leading to the development of a humanized camelid nanobody (FF-01) with potential therapeutic applications. Furthermore, bioinformatic analysis differentiates DLBCL subgroups based on TE expression, providing insights into prognostic variations. Patients with high HERV-K113 expression show activation of pathways related to antiviral responses, suggesting a viral mimicry state.

In conclusion, the study highlights the clinical relevance of HERVs in lymphomas, proposing them as novel therapeutic targets. The newly developed nanobody FF-01 demonstrates anti-lymphoma activity through antibody-dependent cellular cytotoxicity and internalization. This research opens avenues for exploring endogenous retroviruses as targets for immunotherapy in lymphomas, showcasing the potential of FF-01 as a promising candidate for further investigation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575943">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575943" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575943">
        <p class="paperTitle">Cell polarity proteins promote macropinocytosis in response to metabolic stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575943" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575943" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lambies, G.; Lee, S.-W.; Duong-Polk, K.; Aza-Blanc, P.; Maganti, S.; Dawson, D. W.; Commisso, C.</p>
        <p class="info">Score: 2.9, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575943' target='https://doi.org/10.1101/2024.01.16.575943'> 10.1101/2024.01.16.575943</a></p>
        <p class="abstract">Macropinocytosis has emerged as a nutrient-scavenging pathway that cancer cells exploit to survive the nutrient-deprived conditions of the tumor microenvironment. Cancer cells are especially reliant on glutamine for their survival, and in pancreatic ductal adenocarcinoma (PDAC) cells, glutamine deficiency can enhance the stimulation of macropinocytosis, allowing the cells to escape metabolic stress through the production of extracellular-protein-derived amino acids. Here, we identify the atypical protein kinase C (aPKC) enzymes, PKC{zeta} and PKC{iota}, as novel regulators of macropinocytosis. In normal epithelial cells, aPKCs are known to regulate cell polarity in association with the scaffold proteins Par3 and Par6, controlling the function of several targets, including the Par1 kinases. In PDAC cells, we identify that each of these cell polarity proteins are required for glutamine stress-induced macropinocytosis. Mechanistically, we find that the aPKCs are regulated by EGFR signaling or by the transcription factor CREM to promote the relocation of Par3 to microtubules, facilitating macropinocytosis in a dynein-dependent manner. Importantly, we determine that cell fitness impairment caused by aPKC depletion is rescued by the restoration of macropinocytosis and that aPKCs support PDAC growth in vivo. These results identify a previously unappreciated role for cell polarity proteins in the regulation of macropinocytosis and provide a better understanding of the mechanistic underpinnings that control macropinocytic uptake in the context of metabolic stress.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
