<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Direct excitatory synapses between neurons and tumor cells drive brain metastatic seeding of breast cancer and melanoma
Authors: Venkataramani, V.; Karreman, M. A.; Nguyen, L. C.; Tehranian, C.; Hebach, N.; Mayer, C.; Meyer, L.; Mughal, S.; Reifenberger, G.; Felsberg, J.; Koehrer, K.; Schubert, M. C.; Westphal, D.; Breckwolt, M.; Brors, B.; Wick, W.; Kuner, T.; Winkler, F.
Score: 135.0, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574608
Interactions between neurons and cancer cells are found in many malignancies, but their relevance for metastatic organ colonization remain largely unknown.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Direct excitatory synapses between neurons and tumor cells drive brain metastatic seeding of breast cancer and melanoma
Authors: Venkataramani, V.; Karreman, M. A.; Nguyen, L. C.; Tehranian, C.; Hebach, N.; Mayer, C.; Meyer, L.; Mughal, S.; Reifenberger, G.; Felsberg, J.; Koehrer, K.; Schubert, M. C.; Westphal, D.; Breckwolt, M.; Brors, B.; Wick, W.; Kuner, T.; Winkler, F.
Score: 135.0, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574608
Interactions between neurons and cancer cells are found in many malignancies, but their relevance for metastatic organ colonization remain largely unknown." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-14T10:38:40+00:00" />
<meta property="article:modified_time" content="2024-01-14T10:38:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Direct excitatory synapses between neurons and tumor cells drive brain metastatic seeding of breast cancer and melanoma
Authors: Venkataramani, V.; Karreman, M. A.; Nguyen, L. C.; Tehranian, C.; Hebach, N.; Mayer, C.; Meyer, L.; Mughal, S.; Reifenberger, G.; Felsberg, J.; Koehrer, K.; Schubert, M. C.; Westphal, D.; Breckwolt, M.; Brors, B.; Wick, W.; Kuner, T.; Winkler, F.
Score: 135.0, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574608
Interactions between neurons and cancer cells are found in many malignancies, but their relevance for metastatic organ colonization remain largely unknown."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Direct excitatory synapses between neurons and tumor cells drive brain metastatic seeding of breast cancer and melanoma\nAuthors: Venkataramani, V.; Karreman, M. A.; Nguyen, L. C.; Tehranian, C.; Hebach, N.; Mayer, C.; Meyer, L.; Mughal, S.; Reifenberger, G.; Felsberg, J.; Koehrer, K.; Schubert, M. C.; Westphal, D.; Breckwolt, M.; Brors, B.; Wick, W.; Kuner, T.; Winkler, F.\nScore: 135.0, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574608\nInteractions between neurons and cancer cells are found in many malignancies, but their relevance for metastatic organ colonization remain largely unknown.",
  "keywords": [
    
  ],
  "articleBody": " Direct excitatory synapses between neurons and tumor cells drive brain metastatic seeding of breast cancer and melanoma\nAuthors: Venkataramani, V.; Karreman, M. A.; Nguyen, L. C.; Tehranian, C.; Hebach, N.; Mayer, C.; Meyer, L.; Mughal, S.; Reifenberger, G.; Felsberg, J.; Koehrer, K.; Schubert, M. C.; Westphal, D.; Breckwolt, M.; Brors, B.; Wick, W.; Kuner, T.; Winkler, F.\nScore: 135.0, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574608\nInteractions between neurons and cancer cells are found in many malignancies, but their relevance for metastatic organ colonization remain largely unknown. It is also unclear whether any direct synaptic communication between neurons and cancer cells of non-neural tumor types exists, and if so, whether this can support metastasis and thus cancer progression. Here we show that excitatory synapses are formed between neurons and brain-metastatic melanoma and breast cancer cells. This starts at an early microscopic stage after extravasation into the brain parenchyma, during residence of cancer cells in the perivascular niche, a critical step for their survival. These neuron-cancer synapses showed a bona fide synaptic ultrastructure, and generated excitatory postsynaptic currents mediated by glutamate receptors of the AMPA subtype in cancer cells. In accordance, AMPA receptor signatures were consistently detected in preclinical and patient samples of melanoma and breast cancer brain metastases. Genetic perturbation and pharmacological inhibition of AMPA receptors with the approved antiepileptic drug perampanel in models of breast and melanoma cancer reduced the number of brain metastases and overall brain metastatic burden. These findings demonstrate for the first time that neurons can form biologically relevant direct synapses with non-neural cancer cells. In brain metastasis, a particularly challenging complication of many common malignancies, this non-canonical stimulatory synaptic interaction offers novel therapeutic opportunities.\nCREB activation drives acinar to ductal reprogramming and promote pancreatic cancer progression in animal models of alcoholic chronic pancreatitis\nAuthors: Srinivasan, S.; Mehra, S.; Bianchi, A.; Singh, S.; Dosch, A. R.; Amirian, H.; Jinka, S.; Krishnamoorthy, V.; Silva, I. D. C.; Box, E. W.; Garrido, V.; Totiger, T. M.; Zhou, Z.; Ban, Y.; Datta, J.; VanSaun, M.; Merchant, N.; Nagathihalli, N.\nScore: 11.0, Published: 2024-01-05 DOI: 10.1101/2024.01.05.574376\nBACKGROUND AND AIMSIn vivo induction of alcoholic chronic pancreatitis (ACP) causes significant acinar damage, increased fibroinflammatory response, and heightened activation of cyclic response element binding protein 1 (CREB) when compared with alcohol (A) or chronic pancreatitis (CP) mediated pancreatic damage. However, the study elucidating the cooperative interaction between CREB and the oncogenic KrasG12D/+(Kras*) in promoting pancreatic cancer progression with ACP remains unexplored. METHODSExperimental ACP induction was established in multiple mouse models, followed by euthanization of the animals at various time intervals during the recovery periods. Tumor latency was determined in these mice cohorts. Here, we established CREB deletion (Crebfl/fl) in Ptf1aCreERTM/+;LSL-KrasG12D+/-(KC) genetic mouse models (KCC-/-). Western blot, phosphokinase array, and qPCR were used to analyze the pancreata of Ptf1aCreERTM+/-, KC and KCC-/- mice. The pancreata of ACP-induced KC mice were subjected to single-cell RNA sequencing (scRNAseq). Further studies involved conducting lineage tracing and acinar cell explant cultures. RESULTSACP induction in KC mice had detrimental effects on the pancreatic damage repair mechanism. The persistent existence of acinar cell-derived ductal lesions demonstrated a prolonged state of hyperactivated CREB. Persistent CREB activation leads to acinar cell reprogramming and increased pro-fibrotic inflammation in KC mice. Acinar-specific Creb ablation reduced advanced PanINs lesions, hindered tumor progression, and restored acinar cell function in ACP-induced mouse models. CONCLUSIONSOur findings demonstrate that CREB cooperates with Kras* to perpetuate an irreversible ADM and PanIN formation. Moreover, CREB sustains oncogenic activity to promote the progression of premalignant lesions toward cancer in the presence of ACP.\nE7-mediated repression of miR-203 promotes LASP1-dependent proliferation in HPV-positive cervical cancer\nAuthors: Patterson, M. R.; Meijers, A. S.; Ryder, E. L.; Scarth, J. A.; Evans, D.; Turner, A. L.; Wasson, C. W.; Daryl, J. E.; James, C. D.; Wang, M.; Ladbury, J. E.; Morgan, I. M.; Samson, A.; Morgan, E. L.; Macdonald, A.\nScore: 5.9, Published: 2024-01-08 DOI: 10.1101/2024.01.08.574687\nHuman papillomaviruses (HPV) are a major cause of malignancy, contributing to [~]5% of all human cancers worldwide, including most cervical cancer cases and a growing number of ano-genital and oral cancers. The major HPV viral oncogenes, E6 and E7, manipulate many host cellular pathways that promote cell proliferation and survival, predisposing infected cells to malignant transformation. Despite the availability of highly effective vaccines, there are still no specific anti-viral therapies targeting HPV or treatments for HPV-associated cancers. As such, a better understanding of viral-host interactions may allow the identification of novel therapeutic targets. Here, we demonstrate that the actin-binding protein LASP1 is upregulated in cervical cancer and significantly correlates with a poorer overall survival. In HPV positive cervical cancer, LASP1 depletion significantly inhibited proliferation in vitro, whilst having minimal effects in HPV negative cervical cancer cells. Furthermore, we show that the LASP1 SH3 domain is essential for LASP1-mediated proliferation in these cells. Mechanistically, we show that HPV E7 regulates LASP1 at the post-transcriptional level by repressing the expression of miR-203, which negatively regulated LASP1 mRNA levels by binding to its 3UTR. Finally, we demonstrated that LASP1 expression is required for the growth of HPV positive cervical cancer cells in an in vivo tumourigenicity model. Together, these data demonstrate that HPV induces LASP1 expression to promote proliferation and survival role in cervical cancer, thus identifying a potential therapeutic target in these cancers.\nSignaling events at TMEM doorways provide potential targets for inhibiting breast cancer dissemination\nAuthors: Surve, C. R.; Duran, C. L.; Ye, X.; Chen, X.; Lin, Y.; Harney, A. S.; Wang, Y.; Sharma, V. P.; Stanley, E. R.; McAuliffe, J. C.; Entenberg, D.; Oktay, M. H.; Condeelis, J. S.\nScore: 5.2, Published: 2024-01-09 DOI: 10.1101/2024.01.08.574676\nTumor cell intravasation is essential for metastatic dissemination, but its exact mechanism is incompletely understood. We have previously shown that in breast cancer, the direct and stable association of a tumor cell expressing Mena, a Tie2hi/Vegfhi macrophage, and a vascular endothelial cell, creates an intravasation portal, called a \"tumor microenvironment of metastasis\" (TMEM) doorway, for tumor cell intravasation, leading to dissemination to distant sites. The density of TMEM doorways, also called TMEM doorway score, is a clinically validated prognostic marker of distant metastasis in breast cancer patients. Although we know that tumor cells utilize TMEM doorway-associated transient vascular openings to intravasate, the precise signaling mechanisms involved in TMEM doorway function are only partially understood. Using two mouse models of breast cancer and an in vitro assay of intravasation, we report that CSF-1 secreted by the TMEM doorway tumor cell stimulates local secretion of VEGF-A from the Tie2hi TMEM doorway macrophage, leading to the dissociation of endothelial junctions between TMEM doorway associated endothelial cells, supporting tumor cell intravasation. Acute blockage of CSF-1R signaling decreases macrophage VEGF secretion as well as TMEM doorway-associated vascular opening, tumor cell trans-endothelial migration, and dissemination. These new insights into signaling events regulating TMEM doorway function should be explored further as treatment strategies for metastatic disease.\nA stochastic population model for the impact of cancer cell dormancy on therapy success\nAuthors: Blath, J.; Kraut, A.; Paul, T.; Tobias, A.\nScore: 7.6, Published: 2023-12-15 DOI: 10.1101/2023.12.15.571717\nAO_SCPLOWBSTRACTC_SCPLOWTherapy evasion - and subsequent disease progression - is a major challenge in current oncology. An important role in this context seems to be played by various forms of cancer cell dormancy. For example, therapy-induced dormancy, over short timescales, can create serious obstacles to aggressive treatment approaches such as chemotherapy, and long-term dormancy may lead to relapses and metastases even many years after an initially successful treatment. The underlying dormancy-related mechanisms are complex and highly diverse, so that the analysis even of basic patterns of the population-level consequences of dormancy requires abstraction and idealization, as well as the identification of the relevant specific scenarios. In this paper, we focus on a situation in which individual cancer cells may switch into and out of a dormant state both spontaneously as well as in response to treatment, and over relatively short time-spans. We introduce a mathematical toy model, based on stochastic agent-based interactions, for the dynamics of cancer cell populations involving individual short-term dormancy, and allow for a range of (multi-drug) therapy protocols. Our analysis shows that in our idealized model, even a small initial population of dormant cells can lead to therapy failure under classical (and in the absence of dormancy successful) single-drug treatments. We further investigate the effectiveness of several multidrug regimes (manipulating dormant cancer cells in specific ways) and provide some basic rules for the design of (multi-)drug treatment protocols depending on the types and parameters of dormancy mechanisms present in the population.\nPharmacogenomic discovery of genetically targeted cancer therapies optimized against clinical outcomes\nAuthors: Truesdell, P.; Chang, J.; Coto Villa, D.; Zhao, Y.; McIlwain, R.; Young, S.; Hiley, S.; Craig, A. W.; Babak, T.\nScore: 4.8, Published: 2024-01-07 DOI: 10.1101/2024.01.05.574245\nDespite the clinical success of dozens of genetically targeted cancer therapies, the vast majority of patients with tumors caused by loss-of-function (LoF) mutations do not have access to these treatments. This is primarily due to the challenge of developing a drug that treats a disease caused by the absence of a protein target. The success of PARP inhibitors has solidified synthetic lethality (SL) as a means to overcome this obstacle. Recent mapping of SL networks using pooled CRISPR-Cas9 screens is a promising approach for expanding this concept to treating cancers driven by additional LoF drivers. In practice, however, translating signals from cell lines, where these screens are typically conducted, to patient outcomes remains a challenge. We developed a pharmacogemic (PGx) approach called \"Clinically Optimized Driver Associated PGx\" (CODA-PGx) that accurately predicts genetically targeted therapies with clinical-stage efficacy in specific LoF driver contexts. Using approved targeted therapies and cancer drugs with available real-world evidence and molecular data from hundreds of patients, we discovered and optimized the key screening principles predictive of efficacy and overall patient survival. In addition to establishing basic technical conventions, such as drug concentration and screening kinetics, we found that replicating the driver perturbation in the right context, as well as selecting patients where those drivers are genuine founder mutations, were key to accurate translation. We used CODA-PGX to screen a diverse collection of clinical stage drugs and report dozens of novel LoF genetically targeted opportunities; many validated in xenografts and by real-world evidence. Notable examples include treating STAG2-mutant tumors with Carboplatin, SMARCB1-mutant tumors with Oxaliplatin, and TP53BP1-mutant tumors with Etoposide or Bleomycin. One Sentence SummaryWe identified principles of pharmacogenomic screening that predict clinical efficacy in cancer patients with specific driver mutations.\nReprogramming neuroblastoma by diet-enhanced polyamine depletion\nAuthors: Cherkaoui, S.; Yang, L.; McBride, M. J.; Turn, C.; Lu, W.; Eigenmann, C.; Allen, G. E.; Panasenko, O. O.; Zhang, L.; Vu, A.; Liu, K.; Li, Y.; Gandhi, O. H.; Surrey, L.; Wierer, M.; White, E.; Rabinowitz, J. D.; Hogarty, M. D.; Morscher, R. J.\nScore: 3.8, Published: 2024-01-08 DOI: 10.1101/2024.01.07.573662\nNeuroblastoma is a highly lethal childhood tumor derived from differentiation-arrested neural crest cells1,2. Like all cancers, its growth is fueled by metabolites obtained from either circulation or local biosynthesis3,4. Neuroblastomas depend on local polyamine biosynthesis, with the inhibitor difluoromethylornithine showing clinical activity5. Here we show that such inhibition can be augmented by dietary restriction of upstream amino acid substrates, leading to disruption of oncogenic protein translation, tumor differentiation, and profound survival gains in the TH-MYCN mouse model. Specifically, an arginine/proline-free diet decreases the polyamine precursor ornithine and augments tumor polyamine depletion by difluoromethylornithine. This polyamine depletion causes ribosome stalling, unexpectedly specifically at adenosine-ending codons. Such codons are selectively enriched in cell cycle genes and low in neuronal differentiation genes. Thus, impaired translation of these codons, induced by the diet-drug combination, favors a pro-differentiation proteome. These results suggest that the genes of specific cellular programs have evolved hallmark codon usage preferences that enable coherent translational rewiring in response to metabolic stresses, and that this process can be targeted to activate differentiation of pediatric cancers. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=192 SRC=\"FIGDIR/small/573662v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (64K): org.highwire.dtl.DTLVardef@12fdaccorg.highwire.dtl.DTLVardef@c59f47org.highwire.dtl.DTLVardef@5c2721org.highwire.dtl.DTLVardef@d2128_HPS_FORMAT_FIGEXP M_FIG C_FIG Highlights- Extra-tumoral conversion of arginine feeds tumor ornithine via uptake from circulation in MYCN-neuroblastoma. - A proline and arginine free diet enhances pharmacological polyamine depletion via reduced ornithine substrate availability. - Polyamine depletion disrupts oncogenic translation to induce a pro-differentiation proteome causing neuroblast differentiation and prolonged survival in the TH-MYCN mouse model. - Genes of specific cellular programs have evolved codon usage preferences that enable coherent translational rewiring in response to metabolic stress, such as polyamine depletion.\nError-corrected deep targeted sequencing of circulating cell-free DNA from colorectal cancer patients for sensitive detection of circulating tumor DNA\nAuthors: Frydendahl, A.; Rasmussen, M. H.; Jensen, S. O.; Henriksen, T. V.; Demuth, C.; Diekema, M.; Dietzel, H. J.; Wen, S. W. C.; Pedersen, J. S.; Dyrskjoet, L.; Andersen, C. L.\nScore: 3.2, Published: 2023-12-23 DOI: 10.1101/2023.12.22.573042\nIntroductionCirculating tumor DNA (ctDNA) is a promising biomarker, reflecting the presence of tumor cells. Sequencing-based detection of ctDNA at low tumor fractions is challenging due to the crude error rate of sequencing. To mitigate this challenge, we developed UMIseq, a fixed-panel deep-targeted sequencing approach, which is universally applicable to all colorectal cancer (CRC) patients. MethodsUMIseq features UMI-mediated error correction, exclusion of mutations related to clonal hematopoiesis, a panel of normals for error modeling, and signal integration from single-nucleotide variations, insertions, deletions, and phased mutations. UMIseq was trained and independently validated on pre-operative (pre-OP) plasma from CRC patients (n=364) and healthy individuals (n=61). ResultsUMIseq displayed an area under the curve surpassing 0.95 for allele frequencies (AF) down to 0.05%. In the training cohort, the pre-OP detection rate reached 80% at 95% specificity, while in the validation cohort, it was 70%. UMIseq enabled the detection of AFs down to 0.004%. To assess the potential for detection of residual disease, 26 post-operative plasma samples from stage III CRC patients were analyzed. Detection of ctDNA was associated with recurrence (p =0.08). ConclusionUMIseq demonstrated robust performance with high sensitivity and specificity, enabling the detection of ctDNA at low allele frequencies.\nClustering of RNA co-expression network identifies novel long non-coding RNA biomarkers in squamous cell carcinoma\nAuthors: Nissinen, L.; Haalisto, J.; Riihila, P.; Piipponen, M.; Kahari, V.-M.\nScore: 3.4, Published: 2023-12-21 DOI: 10.1101/2023.12.20.571624\nLong non-coding RNAs (lncRNAs) have been shown to play an important role in cancer progression. Cutaneous squamous cell carcinoma is the most common metastatic skin cancer with increasing incidence worldwide. The prognosis of the metastatic cSCC is poor, and currently there are no established biomarkers to predict metastatic risk nor specific therapeutic targets for advanced or metastatic cSCC. To elucidate the role of lncRNAs in cSCC, RNA sequencing of patient derived cSCC cell lines and normal human epidermal keratinocytes was performed. The correlation analysis of differentially expressed lncRNA and protein-coding genes revealed six distinct clusters. One of the upregulated clusters involved genes related to cell motility. Upregulation of the expression of lncRNAs involved in cSCC cell motility in cSCC and head and neck SCC (HNSCC) cells was confirmed by qRT-PCR. Upregulation of HOTTIP and LINC00543 was also noted in SCC tumors in vivo and was associated with worse prognosis in HNSCC and lung SCC cohorts in the TCGA data, respectively. Altogether, these results reveal a novel set of lncRNAs involved in cSCC cell locomotion. These lncRNAs may serve as potential novel biomarkers or a biomarker panel and as putative therapeutic targets in locally advanced and metastatic cSCC.\nEndoplasmic reticulum stress signaling actively contributes to therapy resistance in colorectal cancer\nAuthors: Sasaki, D.; Sato, N.; Wilhelm, D.; Fischer, J.; Gissibl, J.; Nakatsuji, M.; Haller, D.; Ishihara, H.; Janssen, K.-P.\nScore: 2.4, Published: 2024-01-08 DOI: 10.1101/2024.01.07.574523\nPurposeWe investigated the involvement of endoplasmic reticulum (ER) stress signaling in cancer cell responses to chemo- and radiotherapy, focusing on three main ER stress mediators, the transcription factors ATF4, XBP1 and ATF6. MethodsPublic cancer genome datasets were assessed for alterations in ER stress mediators. Surgically resected colorectal cancer tissues were tested by flow cytometry and used to generate patient-derived organoids. Human cell lines and organoids were characterized under oxaliplatin treatment, alone or combined with pharmacological inhibitors of the three ER stress branches, or X-ray irradiation, for cytotoxicity, activation of ER stress and proteome changes. To monitor ER stress in real time, stable HEK293 kidney epithelial cell lines were established expressing ATF4, XBP1, or ATF6, fused with a fluorophore. ResultsGenomic amplification of ATF6, but not ATF4 or XBP1, was frequent in solid tumor entities like breast, lung and colorectal cancer and significantly associated with reduced disease-free survival. In colorectal cancer, increased ATF6 was associated with genetic instability. Basal ER stress mediator expression was correlated to chemoresistance in colorectal cancer cell lines, and generally high in cancer cells compared to HEK293 cells. With proteomics and live HEK293-based reporter lines, we noted that oxaliplatin treatment induced ER stress in a remarkably different way from the canonical ER stress inducer thapsigargin. Moreover, modulation of ER stress signaling by exogenous expression of the stress mediators positively affects chemoresistance, and pharmacological inhibition of ATF6 sensitizes ER-stressed HCT116 colorectal cancer cells to chemotherapy. Of note, cellular stress responses was strongly dependent on the individual transcription factor: XBP1-driven response appeared multi-functional, involved in ribosome biogenesis stress and associated with oxaliplatin resistance. ATF6-dependent stress signaling was involved in DNA damage repair, and was essential for radioresistance. Moreover, chemoresistance in HCT116 cancer cells was impaired by pharmacological ATF6 inhibition. ConclusionActivation of the ER stress signaling may be critically involved in acquired chemo- and radioresistance. Due to their apparent cytoprotective roles, ATF6 and XBP1 could be attractive predictive biomarkers and putative therapeutic targets. SUMMARYTo address their roles in the clinical context, genomic alterations of ATF4, XBP1 and/or ATF6 in human solid tumors were assessed with respect to prognosis and genomic instability. Moreover, surgically resected CRC patient tissues were tested for expression of ER stress markers by flow cytometry and associated with clinical characteristics. In addition, a panel of human cell lines and patient-derived colon organoids were characterized under therapeutic conditions for expression and activation of ER stress proteins, and resulting cytotoxicity was determined. To monitor and modulate ER stress activation in live cells with subcellular resolution, stable reporter cell lines expressing ATF4, sXBP1 or ATF6 proteins fused with a fluorophore were established. These lines were tested for gene or protein expression and cytotoxicity assays to analyze how activation or inhibition of ER stress proteins affects the cellular responses to oxaliplatin treatment or X-ray irradiation. Finally, mass spectrometric proteome analysis was performed to obtain an unbiased readout on the cellular responses to chemotherapy driven by the activation of the ER stress proteins.\n",
  "wordCount" : "3067",
  "inLanguage": "en",
  "datePublished": "2024-01-14T10:38:40Z",
  "dateModified": "2024-01-14T10:38:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 14, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574608">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574608" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574608">
        <p class="paperTitle">Direct excitatory synapses between neurons and tumor cells drive brain metastatic seeding of breast cancer and melanoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574608" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574608" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venkataramani, V.; Karreman, M. A.; Nguyen, L. C.; Tehranian, C.; Hebach, N.; Mayer, C.; Meyer, L.; Mughal, S.; Reifenberger, G.; Felsberg, J.; Koehrer, K.; Schubert, M. C.; Westphal, D.; Breckwolt, M.; Brors, B.; Wick, W.; Kuner, T.; Winkler, F.</p>
        <p class="info">Score: 135.0, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574608' target='https://doi.org/10.1101/2024.01.08.574608'> 10.1101/2024.01.08.574608</a></p>
        <p class="abstract">Interactions between neurons and cancer cells are found in many malignancies, but their relevance for metastatic organ colonization remain largely unknown. It is also unclear whether any direct synaptic communication between neurons and cancer cells of non-neural tumor types exists, and if so, whether this can support metastasis and thus cancer progression. Here we show that excitatory synapses are formed between neurons and brain-metastatic melanoma and breast cancer cells. This starts at an early microscopic stage after extravasation into the brain parenchyma, during residence of cancer cells in the perivascular niche, a critical step for their survival. These neuron-cancer synapses showed a bona fide synaptic ultrastructure, and generated excitatory postsynaptic currents mediated by glutamate receptors of the AMPA subtype in cancer cells. In accordance, AMPA receptor signatures were consistently detected in preclinical and patient samples of melanoma and breast cancer brain metastases. Genetic perturbation and pharmacological inhibition of AMPA receptors with the approved antiepileptic drug perampanel in models of breast and melanoma cancer reduced the number of brain metastases and overall brain metastatic burden. These findings demonstrate for the first time that neurons can form biologically relevant direct synapses with non-neural cancer cells. In brain metastasis, a particularly challenging complication of many common malignancies, this non-canonical stimulatory synaptic interaction offers novel therapeutic opportunities.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574376">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574376" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574376">
        <p class="paperTitle">CREB activation drives acinar to ductal reprogramming and promote pancreatic cancer progression in animal models of alcoholic chronic pancreatitis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574376" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574376" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Srinivasan, S.; Mehra, S.; Bianchi, A.; Singh, S.; Dosch, A. R.; Amirian, H.; Jinka, S.; Krishnamoorthy, V.; Silva, I. D. C.; Box, E. W.; Garrido, V.; Totiger, T. M.; Zhou, Z.; Ban, Y.; Datta, J.; VanSaun, M.; Merchant, N.; Nagathihalli, N.</p>
        <p class="info">Score: 11.0, Published: 2024-01-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574376' target='https://doi.org/10.1101/2024.01.05.574376'> 10.1101/2024.01.05.574376</a></p>
        <p class="abstract">BACKGROUND AND AIMSIn vivo induction of alcoholic chronic pancreatitis (ACP) causes significant acinar damage, increased fibroinflammatory response, and heightened activation of cyclic response element binding protein 1 (CREB) when compared with alcohol (A) or chronic pancreatitis (CP) mediated pancreatic damage. However, the study elucidating the cooperative interaction between CREB and the oncogenic KrasG12D/&#43;(Kras*) in promoting pancreatic cancer progression with ACP remains unexplored.

METHODSExperimental ACP induction was established in multiple mouse models, followed by euthanization of the animals at various time intervals during the recovery periods. Tumor latency was determined in these mice cohorts. Here, we established CREB deletion (Crebfl/fl) in Ptf1aCreERTM/&#43;;LSL-KrasG12D&#43;/-(KC) genetic mouse models (KCC-/-). Western blot, phosphokinase array, and qPCR were used to analyze the pancreata of Ptf1aCreERTM&#43;/-, KC and KCC-/- mice. The pancreata of ACP-induced KC mice were subjected to single-cell RNA sequencing (scRNAseq). Further studies involved conducting lineage tracing and acinar cell explant cultures.

RESULTSACP induction in KC mice had detrimental effects on the pancreatic damage repair mechanism. The persistent existence of acinar cell-derived ductal lesions demonstrated a prolonged state of hyperactivated CREB. Persistent CREB activation leads to acinar cell reprogramming and increased pro-fibrotic inflammation in KC mice. Acinar-specific Creb ablation reduced advanced PanINs lesions, hindered tumor progression, and restored acinar cell function in ACP-induced mouse models.

CONCLUSIONSOur findings demonstrate that CREB cooperates with Kras* to perpetuate an irreversible ADM and PanIN formation. Moreover, CREB sustains oncogenic activity to promote the progression of premalignant lesions toward cancer in the presence of ACP.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574687">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574687" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574687">
        <p class="paperTitle">E7-mediated repression of miR-203 promotes LASP1-dependent proliferation in HPV-positive cervical cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574687" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574687" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Patterson, M. R.; Meijers, A. S.; Ryder, E. L.; Scarth, J. A.; Evans, D.; Turner, A. L.; Wasson, C. W.; Daryl, J. E.; James, C. D.; Wang, M.; Ladbury, J. E.; Morgan, I. M.; Samson, A.; Morgan, E. L.; Macdonald, A.</p>
        <p class="info">Score: 5.9, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574687' target='https://doi.org/10.1101/2024.01.08.574687'> 10.1101/2024.01.08.574687</a></p>
        <p class="abstract">Human papillomaviruses (HPV) are a major cause of malignancy, contributing to [~]5% of all human cancers worldwide, including most cervical cancer cases and a growing number of ano-genital and oral cancers. The major HPV viral oncogenes, E6 and E7, manipulate many host cellular pathways that promote cell proliferation and survival, predisposing infected cells to malignant transformation. Despite the availability of highly effective vaccines, there are still no specific anti-viral therapies targeting HPV or treatments for HPV-associated cancers. As such, a better understanding of viral-host interactions may allow the identification of novel therapeutic targets. Here, we demonstrate that the actin-binding protein LASP1 is upregulated in cervical cancer and significantly correlates with a poorer overall survival. In HPV positive cervical cancer, LASP1 depletion significantly inhibited proliferation in vitro, whilst having minimal effects in HPV negative cervical cancer cells. Furthermore, we show that the LASP1 SH3 domain is essential for LASP1-mediated proliferation in these cells. Mechanistically, we show that HPV E7 regulates LASP1 at the post-transcriptional level by repressing the expression of miR-203, which negatively regulated LASP1 mRNA levels by binding to its 3UTR. Finally, we demonstrated that LASP1 expression is required for the growth of HPV positive cervical cancer cells in an in vivo tumourigenicity model. Together, these data demonstrate that HPV induces LASP1 expression to promote proliferation and survival role in cervical cancer, thus identifying a potential therapeutic target in these cancers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.08.574676">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.08.574676" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.08.574676">
        <p class="paperTitle">Signaling events at TMEM doorways provide potential targets for inhibiting breast cancer dissemination</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.08.574676" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.08.574676" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Surve, C. R.; Duran, C. L.; Ye, X.; Chen, X.; Lin, Y.; Harney, A. S.; Wang, Y.; Sharma, V. P.; Stanley, E. R.; McAuliffe, J. C.; Entenberg, D.; Oktay, M. H.; Condeelis, J. S.</p>
        <p class="info">Score: 5.2, Published: 2024-01-09 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.08.574676' target='https://doi.org/10.1101/2024.01.08.574676'> 10.1101/2024.01.08.574676</a></p>
        <p class="abstract">Tumor cell intravasation is essential for metastatic dissemination, but its exact mechanism is incompletely understood. We have previously shown that in breast cancer, the direct and stable association of a tumor cell expressing Mena, a Tie2hi/Vegfhi macrophage, and a vascular endothelial cell, creates an intravasation portal, called a &#34;tumor microenvironment of metastasis&#34; (TMEM) doorway, for tumor cell intravasation, leading to dissemination to distant sites. The density of TMEM doorways, also called TMEM doorway score, is a clinically validated prognostic marker of distant metastasis in breast cancer patients. Although we know that tumor cells utilize TMEM doorway-associated transient vascular openings to intravasate, the precise signaling mechanisms involved in TMEM doorway function are only partially understood. Using two mouse models of breast cancer and an in vitro assay of intravasation, we report that CSF-1 secreted by the TMEM doorway tumor cell stimulates local secretion of VEGF-A from the Tie2hi TMEM doorway macrophage, leading to the dissociation of endothelial junctions between TMEM doorway associated endothelial cells, supporting tumor cell intravasation. Acute blockage of CSF-1R signaling decreases macrophage VEGF secretion as well as TMEM doorway-associated vascular opening, tumor cell trans-endothelial migration, and dissemination. These new insights into signaling events regulating TMEM doorway function should be explored further as treatment strategies for metastatic disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571717">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571717" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571717">
        <p class="paperTitle">A stochastic population model for the impact of cancer cell dormancy on therapy success</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571717" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571717" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blath, J.; Kraut, A.; Paul, T.; Tobias, A.</p>
        <p class="info">Score: 7.6, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571717' target='https://doi.org/10.1101/2023.12.15.571717'> 10.1101/2023.12.15.571717</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWTherapy evasion - and subsequent disease progression - is a major challenge in current oncology. An important role in this context seems to be played by various forms of cancer cell dormancy. For example, therapy-induced dormancy, over short timescales, can create serious obstacles to aggressive treatment approaches such as chemotherapy, and long-term dormancy may lead to relapses and metastases even many years after an initially successful treatment. The underlying dormancy-related mechanisms are complex and highly diverse, so that the analysis even of basic patterns of the population-level consequences of dormancy requires abstraction and idealization, as well as the identification of the relevant specific scenarios.

In this paper, we focus on a situation in which individual cancer cells may switch into and out of a dormant state both spontaneously as well as in response to treatment, and over relatively short time-spans. We introduce a mathematical  toy model, based on stochastic agent-based interactions, for the dynamics of cancer cell populations involving individual short-term dormancy, and allow for a range of (multi-drug) therapy protocols. Our analysis shows that in our idealized model, even a small initial population of dormant cells can lead to therapy failure under classical (and in the absence of dormancy successful) single-drug treatments. We further investigate the effectiveness of several multidrug regimes (manipulating dormant cancer cells in specific ways) and provide some basic rules for the design of (multi-)drug treatment protocols depending on the types and parameters of dormancy mechanisms present in the population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574245">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574245" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574245">
        <p class="paperTitle">Pharmacogenomic discovery of genetically targeted cancer therapies optimized against clinical outcomes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574245" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574245" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Truesdell, P.; Chang, J.; Coto Villa, D.; Zhao, Y.; McIlwain, R.; Young, S.; Hiley, S.; Craig, A. W.; Babak, T.</p>
        <p class="info">Score: 4.8, Published: 2024-01-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574245' target='https://doi.org/10.1101/2024.01.05.574245'> 10.1101/2024.01.05.574245</a></p>
        <p class="abstract">Despite the clinical success of dozens of genetically targeted cancer therapies, the vast majority of patients with tumors caused by loss-of-function (LoF) mutations do not have access to these treatments. This is primarily due to the challenge of developing a drug that treats a disease caused by the absence of a protein target. The success of PARP inhibitors has solidified synthetic lethality (SL) as a means to overcome this obstacle. Recent mapping of SL networks using pooled CRISPR-Cas9 screens is a promising approach for expanding this concept to treating cancers driven by additional LoF drivers. In practice, however, translating signals from cell lines, where these screens are typically conducted, to patient outcomes remains a challenge. We developed a pharmacogemic (PGx) approach called &#34;Clinically Optimized Driver Associated PGx&#34; (CODA-PGx) that accurately predicts genetically targeted therapies with clinical-stage efficacy in specific LoF driver contexts. Using approved targeted therapies and cancer drugs with available real-world evidence and molecular data from hundreds of patients, we discovered and optimized the key screening principles predictive of efficacy and overall patient survival. In addition to establishing basic technical conventions, such as drug concentration and screening kinetics, we found that replicating the driver perturbation in the right context, as well as selecting patients where those drivers are genuine founder mutations, were key to accurate translation. We used CODA-PGX to screen a diverse collection of clinical stage drugs and report dozens of novel LoF genetically targeted opportunities; many validated in xenografts and by real-world evidence. Notable examples include treating STAG2-mutant tumors with Carboplatin, SMARCB1-mutant tumors with Oxaliplatin, and TP53BP1-mutant tumors with Etoposide or Bleomycin.

One Sentence SummaryWe identified principles of pharmacogenomic screening that predict clinical efficacy in cancer patients with specific driver mutations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.573662">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.573662" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.573662">
        <p class="paperTitle">Reprogramming neuroblastoma by diet-enhanced polyamine depletion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.573662" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.573662" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cherkaoui, S.; Yang, L.; McBride, M. J.; Turn, C.; Lu, W.; Eigenmann, C.; Allen, G. E.; Panasenko, O. O.; Zhang, L.; Vu, A.; Liu, K.; Li, Y.; Gandhi, O. H.; Surrey, L.; Wierer, M.; White, E.; Rabinowitz, J. D.; Hogarty, M. D.; Morscher, R. J.</p>
        <p class="info">Score: 3.8, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.573662' target='https://doi.org/10.1101/2024.01.07.573662'> 10.1101/2024.01.07.573662</a></p>
        <p class="abstract">Neuroblastoma is a highly lethal childhood tumor derived from differentiation-arrested neural crest cells1,2. Like all cancers, its growth is fueled by metabolites obtained from either circulation or local biosynthesis3,4. Neuroblastomas depend on local polyamine biosynthesis, with the inhibitor difluoromethylornithine showing clinical activity5. Here we show that such inhibition can be augmented by dietary restriction of upstream amino acid substrates, leading to disruption of oncogenic protein translation, tumor differentiation, and profound survival gains in the TH-MYCN mouse model. Specifically, an arginine/proline-free diet decreases the polyamine precursor ornithine and augments tumor polyamine depletion by difluoromethylornithine. This polyamine depletion causes ribosome stalling, unexpectedly specifically at adenosine-ending codons. Such codons are selectively enriched in cell cycle genes and low in neuronal differentiation genes. Thus, impaired translation of these codons, induced by the diet-drug combination, favors a pro-differentiation proteome. These results suggest that the genes of specific cellular programs have evolved hallmark codon usage preferences that enable coherent translational rewiring in response to metabolic stresses, and that this process can be targeted to activate differentiation of pediatric cancers.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=192 SRC=&#34;FIGDIR/small/573662v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (64K):
org.highwire.dtl.DTLVardef@12fdaccorg.highwire.dtl.DTLVardef@c59f47org.highwire.dtl.DTLVardef@5c2721org.highwire.dtl.DTLVardef@d2128_HPS_FORMAT_FIGEXP  M_FIG C_FIG Highlights- Extra-tumoral conversion of arginine feeds tumor ornithine via uptake from circulation in MYCN-neuroblastoma.
- A proline and arginine free diet enhances pharmacological polyamine depletion via reduced ornithine substrate availability.
- Polyamine depletion disrupts oncogenic translation to induce a pro-differentiation proteome causing neuroblast differentiation and prolonged survival in the TH-MYCN mouse model.
- Genes of specific cellular programs have evolved codon usage preferences that enable coherent translational rewiring in response to metabolic stress, such as polyamine depletion.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.22.573042">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.22.573042" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.22.573042">
        <p class="paperTitle">Error-corrected deep targeted sequencing of circulating cell-free DNA from colorectal cancer patients for sensitive detection of circulating tumor DNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.22.573042" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.22.573042" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frydendahl, A.; Rasmussen, M. H.; Jensen, S. O.; Henriksen, T. V.; Demuth, C.; Diekema, M.; Dietzel, H. J.; Wen, S. W. C.; Pedersen, J. S.; Dyrskjoet, L.; Andersen, C. L.</p>
        <p class="info">Score: 3.2, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.22.573042' target='https://doi.org/10.1101/2023.12.22.573042'> 10.1101/2023.12.22.573042</a></p>
        <p class="abstract">IntroductionCirculating tumor DNA (ctDNA) is a promising biomarker, reflecting the presence of tumor cells. Sequencing-based detection of ctDNA at low tumor fractions is challenging due to the crude error rate of sequencing. To mitigate this challenge, we developed UMIseq, a fixed-panel deep-targeted sequencing approach, which is universally applicable to all colorectal cancer (CRC) patients.

MethodsUMIseq features UMI-mediated error correction, exclusion of mutations related to clonal hematopoiesis, a panel of normals for error modeling, and signal integration from single-nucleotide variations, insertions, deletions, and phased mutations. UMIseq was trained and independently validated on pre-operative (pre-OP) plasma from CRC patients (n=364) and healthy individuals (n=61).

ResultsUMIseq displayed an area under the curve surpassing 0.95 for allele frequencies (AF) down to 0.05%. In the training cohort, the pre-OP detection rate reached 80% at 95% specificity, while in the validation cohort, it was 70%. UMIseq enabled the detection of AFs down to 0.004%. To assess the potential for detection of residual disease, 26 post-operative plasma samples from stage III CRC patients were analyzed. Detection of ctDNA was associated with recurrence (p =0.08).

ConclusionUMIseq demonstrated robust performance with high sensitivity and specificity, enabling the detection of ctDNA at low allele frequencies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.571624">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.571624" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.571624">
        <p class="paperTitle">Clustering of RNA co-expression network identifies novel long non-coding RNA biomarkers in squamous cell carcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.571624" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.571624" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nissinen, L.; Haalisto, J.; Riihila, P.; Piipponen, M.; Kahari, V.-M.</p>
        <p class="info">Score: 3.4, Published: 2023-12-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.571624' target='https://doi.org/10.1101/2023.12.20.571624'> 10.1101/2023.12.20.571624</a></p>
        <p class="abstract">Long non-coding RNAs (lncRNAs) have been shown to play an important role in cancer progression. Cutaneous squamous cell carcinoma is the most common metastatic skin cancer with increasing incidence worldwide. The prognosis of the metastatic cSCC is poor, and currently there are no established biomarkers to predict metastatic risk nor specific therapeutic targets for advanced or metastatic cSCC. To elucidate the role of lncRNAs in cSCC, RNA sequencing of patient derived cSCC cell lines and normal human epidermal keratinocytes was performed. The correlation analysis of differentially expressed lncRNA and protein-coding genes revealed six distinct clusters. One of the upregulated clusters involved genes related to cell motility. Upregulation of the expression of lncRNAs involved in cSCC cell motility in cSCC and head and neck SCC (HNSCC) cells was confirmed by qRT-PCR. Upregulation of HOTTIP and LINC00543 was also noted in SCC tumors in vivo and was associated with worse prognosis in HNSCC and lung SCC cohorts in the TCGA data, respectively. Altogether, these results reveal a novel set of lncRNAs involved in cSCC cell locomotion. These lncRNAs may serve as potential novel biomarkers or a biomarker panel and as putative therapeutic targets in locally advanced and metastatic cSCC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.07.574523">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.07.574523" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.07.574523">
        <p class="paperTitle">Endoplasmic reticulum stress signaling actively contributes to therapy resistance in colorectal cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.07.574523" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.07.574523" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sasaki, D.; Sato, N.; Wilhelm, D.; Fischer, J.; Gissibl, J.; Nakatsuji, M.; Haller, D.; Ishihara, H.; Janssen, K.-P.</p>
        <p class="info">Score: 2.4, Published: 2024-01-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.07.574523' target='https://doi.org/10.1101/2024.01.07.574523'> 10.1101/2024.01.07.574523</a></p>
        <p class="abstract">PurposeWe investigated the involvement of endoplasmic reticulum (ER) stress signaling in cancer cell responses to chemo- and radiotherapy, focusing on three main ER stress mediators, the transcription factors ATF4, XBP1 and ATF6.

MethodsPublic cancer genome datasets were assessed for alterations in ER stress mediators. Surgically resected colorectal cancer tissues were tested by flow cytometry and used to generate patient-derived organoids. Human cell lines and organoids were characterized under oxaliplatin treatment, alone or combined with pharmacological inhibitors of the three ER stress branches, or X-ray irradiation, for cytotoxicity, activation of ER stress and proteome changes. To monitor ER stress in real time, stable HEK293 kidney epithelial cell lines were established expressing ATF4, XBP1, or ATF6, fused with a fluorophore.

ResultsGenomic amplification of ATF6, but not ATF4 or XBP1, was frequent in solid tumor entities like breast, lung and colorectal cancer and significantly associated with reduced disease-free survival. In colorectal cancer, increased ATF6 was associated with genetic instability. Basal ER stress mediator expression was correlated to chemoresistance in colorectal cancer cell lines, and generally high in cancer cells compared to HEK293 cells. With proteomics and live HEK293-based reporter lines, we noted that oxaliplatin treatment induced ER stress in a remarkably different way from the canonical ER stress inducer thapsigargin. Moreover, modulation of ER stress signaling by exogenous expression of the stress mediators positively affects chemoresistance, and pharmacological inhibition of ATF6 sensitizes ER-stressed HCT116 colorectal cancer cells to chemotherapy. Of note, cellular stress responses was strongly dependent on the individual transcription factor: XBP1-driven response appeared multi-functional, involved in ribosome biogenesis stress and associated with oxaliplatin resistance. ATF6-dependent stress signaling was involved in DNA damage repair, and was essential for radioresistance. Moreover, chemoresistance in HCT116 cancer cells was impaired by pharmacological ATF6 inhibition.

ConclusionActivation of the ER stress signaling may be critically involved in acquired chemo- and radioresistance. Due to their apparent cytoprotective roles, ATF6 and XBP1 could be attractive predictive biomarkers and putative therapeutic targets.

SUMMARYTo address their roles in the clinical context, genomic alterations of ATF4, XBP1 and/or ATF6 in human solid tumors were assessed with respect to prognosis and genomic instability. Moreover, surgically resected CRC patient tissues were tested for expression of ER stress markers by flow cytometry and associated with clinical characteristics. In addition, a panel of human cell lines and patient-derived colon organoids were characterized under therapeutic conditions for expression and activation of ER stress proteins, and resulting cytotoxicity was determined. To monitor and modulate ER stress activation in live cells with subcellular resolution, stable reporter cell lines expressing ATF4, sXBP1 or ATF6 proteins fused with a fluorophore were established. These lines were tested for gene or protein expression and cytotoxicity assays to analyze how activation or inhibition of ER stress proteins affects the cellular responses to oxaliplatin treatment or X-ray irradiation. Finally, mass spectrometric proteome analysis was performed to obtain an unbiased readout on the cellular responses to chemotherapy driven by the activation of the ER stress proteins.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
