<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Harnessing extracellular vesicles for efficient siRNA delivery in vitro and in vivo
Authors: Qiu, T.; Zhai, Y.; Yi, Y.; Wang, X.; Hu, R.; Niu, S.; Wang, C.; Cui, C.; Xu, K.; He, X.
Score: 441.5, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554924
To fulfill the potential of small interference RNA (siRNA) therapeutics, diverse and efficient delivery platforms for siRNA are urgently needed. Extracellular vesicles (EVs) are endogenous, cell secreted nano-vehicles which have been explored for functional siRNA delivery, yet impurities in EVs preparation and lack of efficient siRNA loading method have been limiting its further development into clinical applications.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Harnessing extracellular vesicles for efficient siRNA delivery in vitro and in vivo
Authors: Qiu, T.; Zhai, Y.; Yi, Y.; Wang, X.; Hu, R.; Niu, S.; Wang, C.; Cui, C.; Xu, K.; He, X.
Score: 441.5, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554924
To fulfill the potential of small interference RNA (siRNA) therapeutics, diverse and efficient delivery platforms for siRNA are urgently needed. Extracellular vesicles (EVs) are endogenous, cell secreted nano-vehicles which have been explored for functional siRNA delivery, yet impurities in EVs preparation and lack of efficient siRNA loading method have been limiting its further development into clinical applications." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-09-06T10:39:07+00:00" />
<meta property="article:modified_time" content="2023-09-06T10:39:07+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Harnessing extracellular vesicles for efficient siRNA delivery in vitro and in vivo
Authors: Qiu, T.; Zhai, Y.; Yi, Y.; Wang, X.; Hu, R.; Niu, S.; Wang, C.; Cui, C.; Xu, K.; He, X.
Score: 441.5, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554924
To fulfill the potential of small interference RNA (siRNA) therapeutics, diverse and efficient delivery platforms for siRNA are urgently needed. Extracellular vesicles (EVs) are endogenous, cell secreted nano-vehicles which have been explored for functional siRNA delivery, yet impurities in EVs preparation and lack of efficient siRNA loading method have been limiting its further development into clinical applications."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Harnessing extracellular vesicles for efficient siRNA delivery in vitro and in vivo\nAuthors: Qiu, T.; Zhai, Y.; Yi, Y.; Wang, X.; Hu, R.; Niu, S.; Wang, C.; Cui, C.; Xu, K.; He, X.\nScore: 441.5, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554924\nTo fulfill the potential of small interference RNA (siRNA) therapeutics, diverse and efficient delivery platforms for siRNA are urgently needed. Extracellular vesicles (EVs) are endogenous, cell secreted nano-vehicles which have been explored for functional siRNA delivery, yet impurities in EVs preparation and lack of efficient siRNA loading method have been limiting its further development into clinical applications.",
  "keywords": [
    
  ],
  "articleBody": " Harnessing extracellular vesicles for efficient siRNA delivery in vitro and in vivo\nAuthors: Qiu, T.; Zhai, Y.; Yi, Y.; Wang, X.; Hu, R.; Niu, S.; Wang, C.; Cui, C.; Xu, K.; He, X.\nScore: 441.5, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554924\nTo fulfill the potential of small interference RNA (siRNA) therapeutics, diverse and efficient delivery platforms for siRNA are urgently needed. Extracellular vesicles (EVs) are endogenous, cell secreted nano-vehicles which have been explored for functional siRNA delivery, yet impurities in EVs preparation and lack of efficient siRNA loading method have been limiting its further development into clinical applications. In this work, we have achieved high quality EVs production and an siRNA loading system via EVs. The prepared EV-siRNA system was able to deliver siRNA cargo to model cell lines to silence the expression of targeted genes with high potency. Furthermore, EV-siRNA administration in a tumor xenograft model significantly inhibited tumor growth. Therefore, this study establishes an innovative and efficient EV-siRNA platform for siRNA delivery in vitro and in vivo.\nGlioblastoma is spatially organized by neurodevelopmental programs and a glial-like wound healing response\nAuthors: Mossi Albiach, A.; Janusauskas, J.; Kapustova, I.; Kvedaraite, E.; Codeluppi, S.; Munting, J. B.; Borm, L. E.; Kjaer Jacobsen, J.; Shamikh, A.; Persson, O.; Linnarsson, S.\nScore: 36.8, Published: 2023-09-04 DOI: 10.1101/2023.09.01.555882\nGlioblastoma is the deadliest brain cancer, characterized by great cellular diversity and unique histology. To understand the spatial organization of transcriptional cell states, we mapped the expression of 888 genes in centimeter-scale tissue sections from a large patient cohort. We found a hierarchy of cellular states akin to normal brain development, including proliferating and differentiated cells interacting with the stroma. We discovered that mesenchymal-like glioblastoma cells comprised a major glial-like wound-response component and a distinct gliosarcoma-specific malignant fibroblast type. Our analysis highlighted hypoxia, tissue damage, and wound healing as major factors in glioblastoma spatial organization. Tumor microenvironment varied along the hypoxia gradient, inducing the recruitment of monocytes and pro-tumorigenic macrophages, and propagating wound response program activation in malignant glial cells. Our study reveals the dynamic progression of glioblastoma organization independent of its mutational profile, in response to tumor-induced injury.\nKRAS allelic imbalance drives tumour initiation yet suppresses metastasis in colorectal cancer in vivo\nAuthors: Najumudeen, A. K.; Fey, S. K.; Millet, L. M.; Ford, C. A.; Gilroy, K.; Gunduz, N.; Ridgway, R. A.; Anderson, E.; Strathdee, D.; Clark, W.; Nixon, C.; Morton, J. P.; Campbell, A. D.; Sansom, O. J.\nScore: 18.1, Published: 2023-08-30 DOI: 10.1101/2023.08.29.555396\nOncogenic KRAS mutations are well-described functionally and are known to drive tumorigenesis. Recent reports describe a significant prevalence of KRAS allelic imbalances or gene dosage changes in human cancers, including loss of the wild-type allele in KRAS mutant cancers. However, there is still much debate over the function of wild-type KRAS in tumour initiation, progression and therapeutic response. We have developed a genetically engineered mouse model which allows deletion of the wild-type copy of Kras in the context of an intact oncogenic Kras in colorectal cancer. We observe that in the presence of oncogenic Kras, wild-type Kras acts to restrain tumour growth. Mechanistically, deletion of wild-type Kras exacerbates oncogenic KRAS signalling through MAPK and thus drives tumour initiation. Absence of wild-type Kras potentiates the oncogenic effect of KRASG12D, while presence of wild-type Kras is associated with resistance to inhibition of MEK1/2 in KRASG12D driven tumours. Importantly, loss of wild-type Kras in oncogenic KRAS-driven aggressive tumours significantly alters tumour progression, metastasis while impacting tumour immune cell infiltration. This study demonstrates a suppressive role for wild-type Kras during colon tumour initiation and highlights the critical impact of wild-type Kras upon therapeutic response to MAPK and tumour progression in Kras mutant cancers. HighlightsO_LIWild-type KRAS suppresses mutant KRASG12D mediated proliferation and signalling in colorectal cancer models in vivo C_LIO_LIConcomitant loss of wild-type KRAS and activation of WNT signalling promotes mutant KRAS-driven tumour initiation. C_LIO_LIWild-type KRAS promotes resistance to MAPK inhibition in KRAS mutant tumours C_LIO_LILoss of wild-type KRAS inhibits metastasis of late-stage mutant KRAS colorectal cancer models. C_LI\nKras loss of heterozygosity promotes MAPK dependent pancreatic ductal adenocarcinoma and induces therapeutic sensitivity\nAuthors: Fey, S. K.; Najumudeen, A. K.; Ford, C. A.; Gilroy, K.; Upstill-Goddard, R.; Clark, W.; Nixon, C.; Chang, D. K.; Barry, S. T.; Morton, J. P.; Campbell, A. D.; Sansom, O. J.\nScore: 12.8, Published: 2023-08-29 DOI: 10.1101/2023.08.29.552598\nPancreatic cancer is characterised by the prevalence of oncogenic mutations in KRAS. Previous studies have reported that altered Kras gene dosage drives progression and metastatic incidence in pancreatic cancer. While the role of oncogenic KRAS mutation is well characterised, the relevance of the partnering wild-type KRAS allele in pancreatic cancer is less well understood and controversial. Using in vivo mouse modelling of pancreatic cancer, we demonstrate that wild-type Kras restrains the oncogenic impact of mutant Kras, and drastically impacts both Kras-mediated tumourigenesis and therapeutic response. Mechanistically, deletion of wild-type Kras increases oncogenic Kras signalling through the downstream MAPK effector pathway, driving pancreatic intraepithelial neoplasia (PanIN) initiation. In addition, in the KPC mouse model, a more aggressive model of pancreatic cancer, loss of wild-type KRAS leads to accelerated initiation but delayed tumour progression. These tumours had altered stroma, downregulated Myc levels and an enrichment for immunogenic gene signatures. Importantly, loss of wild-type Kras sensitises Kras mutant tumours to MEK1/2 inhibition though tumours eventually become resistant and then rapidly progress. This study demonstrates the repressive role of wild-type Kras during pancreatic tumourigenesis and highlights the critical impact of the presence of wild-type KRAS on tumourigenesis and therapeutic response in pancreatic cancer.\nEvolvability of cancer-associated genes under APOBEC3A/B selection\nAuthors: Song, J.-H.; Davalos, L.; MacCarthy, T.; Damaghi, M.\nScore: 8.3, Published: 2023-08-27 DOI: 10.1101/2023.08.27.554991\nEvolvability is an emergent hallmark of cancer that depends on intra-tumor heterogeneity and, ultimately, genetic variation. Mutations generated by APOBEC3 cytidine deaminases can contribute to genetic variation and the consequences of APOBEC activation differ depending on the stage of cancer, with the most significant impact observed during the early stages. However, how APOBEC activity shapes evolutionary patterns of genes in the host genome and differential impacts on cancer-associated and non-cancer genes remain unclear. Analyzing over 40,000 human protein-coding transcripts, we identified distinct distribution patterns of APOBEC3A/B TC motifs between cancer-related genes and controls, suggesting unique associations with cancer. Studying a bat species with many more APOBEC3 genes, we found diverse motif patterns in orthologs of cancer genes compared to controls, similar to humans and suggesting APOBEC evolution to reduce impacts on the genome rather than the converse. Simulations confirmed that APOBEC-induced heterogeneity enhances cancer evolution, shaping clonal dynamics through bimodal introduction of mutations in certain classes of genes. Our results suggest that a major consequence of the bimodal distribution of APOBEC affects greater cancer heterogeneity. HighlightsO_LIUsing a measure of the extent which genes are affected by APOBEC activity, we found that many genes are maximally robust in the human genome. Interestingly, the distribution in the genome of a bat that has many APOBEC genes was similar. C_LIO_LIIn contrast, when analyzing the subset of cancer-associated genes the distribution is bimodal with many genes appearing to susceptible to APOBEC activity. C_LIO_LIAnalysis of orthologs of human genes and other species showed a wide range in the extent to which APOBEC affects genes having the same function. C_LIO_LIThe bimodal distribution of susceptibility to APOBEC drives greater heterogeneity in simulated clonal evolution of cancer. C_LI\nIntegration of Multiple Spatial-Omics Modalities Reveals Unique Insights into Molecular Heterogeneity of Prostate Cancer\nAuthors: Zhang, W.; Spotbeen, X.; Vanuytven, S.; Kint, S.; Sarretto, T.; Socciarelli, F.; Vandereyken, K.; Dehairs, J.; Idkowiak, J.; Wouters, D.; Alvira Larizgoitia, J. I.; Partel, G.; Ly, A.; de Laat, V.; Q. Mantas, M. J.; Gevaert, T.; Devlies, W.; Mah, C. Y.; Butler, L. M.; Loda, M.; Joniau, S.; De Moor, B.; Sifrim, A.; Ellis, S. R.; Voet, T.; Claesen, M.; Verbeeck, N.; Swinnen, J. V.\nScore: 14.9, Published: 2023-08-28 DOI: 10.1101/2023.08.28.555056\nRecent advances in spatial omics methods are revolutionising biomedical research by enabling detailed molecular analyses of cells and their interactions in their native state. As most technologies capture only a specific type of molecules, there is an unmet need to enable integration of multiple spatial-omics datasets. This, however, presents several challenges as these analyses typically operate on separate tissue sections at disparate spatial resolutions. Here, we established a spatial multi-omics integration pipeline enabling co-registration and granularity matching, and applied it to integrate spatial transcriptomics, mass spectrometry-based lipidomics, single nucleus RNA-seq and histomorphological information from human prostate cancer patient samples. This approach revealed unique correlations between lipids and gene expression profiles that are linked to distinct cell populations and histopathological disease states and uncovered molecularly different subregions not discernible by morphology alone. By its ability to correlate datasets that span across the biomolecular and spatial scale, the application of this novel spatial multi-omics integration pipeline provides unprecedented insight into the intricate interplay between different classes of molecules in a tissue context. In addition, it has unique hypothesis-generating potential, and holds promise for applications in molecular pathology, biomarker and target discovery and other tissue-based research fields.\nCellular taxonomy of the preleukemic bone marrow niche of acute myeloid leukemia\nAuthors: Goda, C.; Kulkarni, R.; Rudich, A.; Karunasiri, M.; Balcioglu, O.; Al-Marrawi, Y.; Korn, E.; Kanna, S.; Garfinkle, E. A. R.; Mundy-Bosse, B.; Zhang, B.; Marcucci, G.; Mardis, E. R.; Garzon, R.; Miller, K. E.; Dorrance, A. M.\nScore: 5.7, Published: 2023-09-04 DOI: 10.1101/2023.09.01.553259\nMutations in hematopoietic stem/progenitor cells (HSPCs) can remain dormant within the bone marrow (BM) for decades before leukemia onset. Understanding the mechanisms by which these mutant clones eventually slead to full blown leukemia is of critical importance to develop strategies to eliminate these clones before they achieve their full leukemogenic potential. Recent data suggest that leukemic stem cells (LSCs) induce alterations within BM microenvironment (BMM) favoring LSC growth over normal HSCs. However, the cross talk between preleukemic stem cells (pLSC) and BMM is not completely understood. We hypothesize that pLSC induces critical changes within the BMM that are critical for leukemogenesis. To address this question, we are using our previously developed murine model of AML that highly recapitulates the human disease, develops AML sporadically with a preleukemic phase in which mice display normal white blood counts (WBCs) and absence of blasts in the BM. Thus, this is an excellent model to evaluate changes in the BMM that occurs during progression into AML. Using this model we performed single cell RNA-sequencing on cells from the BMM compared to wild-type (WT) controls. Overall, we defined the transcriptional profiles of pre-leukemic BMM cells and observed decreased percentages of normal BMM cells such as LepR+ mesenchymal stem cells (MSCs) and endothelial cells (ECs), known to regulate normal HSC function. Concomitantly, we found increases in CD55+ fibroblasts and NG2+ pericytes, that might play a more important role in regulation of pre-LSCs. Preleukemic CD55+ fibroblasts had a higher proliferation rate and showed significant down-regulation of several collagen genes known for regulating extra cellular matrix (ECM) including: Col1a1, Col1a2, Col3a1, Col4a1, and Col6a1, suggesting that ECM remodeling occurs in the early stages of leukemogenesis. Importantly, co-culture assays found that pre-leukemic CD55+ BM fibroblasts expanded pre-LSCs significantly over normal HSCs. In conclusion, we have identified distinct changes in the preleukemic BMM and identified a novel CD55+ fibroblast population that is expanded in preleukemic BMM that promote the fitness of pre-LSCs over normal HSCs.\nImpact of resistance on therapeutic design: a Moran model of cancer growth\nAuthors: Lacy, M. S.; Jenner, A. L.\nScore: 7.6, Published: 2023-08-28 DOI: 10.1101/2023.08.28.555214\nResistance of cancers to treatments, such as chemotherapy, largely arise due to cell mutations. These mutations allow cells to resist apoptosis and inevitably lead to recurrence and often progression to more aggressive cancer forms. Sustained-low dose therapies are being considered as an alternative over maximum tolerated dose treatments, whereby a smaller drug dosage is given over a longer period of time. However, understanding the impact that the presence of treatment-resistant clones may have on these new treatment modalities is crucial to validating them as a therapeutic avenue. In this study, a Moran process is used to capture stochastic mutations arising in cancer cells, inferring treatment resistance. The model is used to predict the probability of cancer recurrence given varying treatment modalities. The simulations predict that sustained-low dose therapies would be virtually ineffective for a cancer with a non-negligible probability of developing a sub-clone with resistance tendencies. Furthermore, calibrating the model to in vivo measurements for breast cancer treatment with Herceptin, the model suggests that standard treatment regimens are ineffective in this mouse model. Using a simple Moran model, it is possible to explore the likelihood of treatment success given a non-negligible probability of treatment resistant mutations and suggest more robust therapeutic schedules.\nIntegration of Mass Cytometry and Mass Spectrometry Imaging for Spatially Resolved Single Cell Metabolic Profiling\nAuthors: Nunes, J. B.; Ijsselsteijn, M. E.; Abdelaal, T.; Ursem, R.; van der Ploeg, M.; Everts, B.; Mahfouz, A.; Heijs, B.; de Miranda, N.\nScore: 4.5, Published: 2023-08-30 DOI: 10.1101/2023.08.29.555282\nIntegration of spatial omics technologies can provide important insights into the biology of tissues. We combined mass spectrometry imaging-based metabolomics and imaging mass cytometry-based immunophenotyping on the same single tissue section to reveal metabolic heterogeneity within tissues and its association with specific cell populations like cancer cells or immune cells. This approach has the potential to greatly increase our understanding of tissue-level interplay between metabolic processes and their cellular components.\nNovel YAP1/TAZ pathway inhibitors identified through phenotypic screening with potent anti-tumor activity via blockade of GGTase-I / Rho-GTPase signaling\nAuthors: Graham, K.; Lienau, P.; Bader, B.; Prechtl, S.; Naujoks, J.; Lesche, R.; Kuehnlenz, J.; Brzezinka, K.; Potze, L.; Zanconato, F.; Nicke, B.; Montebaur, A.; Bone, W.; Golfier, S.; Kaulfuss, S.; Kopitz, C.; Pilari, S.; Steuber, H.; Hayat, S.; Kamburov, A.; Steffen, A.; Schlicker, A.; Buchgraber, P.; Braeuer, N.; Font, N. A.; Heinrich, T.; Kuhnke, L.; Nowak-Reppel, K.; Stresemann, C.; Steigemann, P.; Walter, A. O.; Blotta, S.; Ocker, M.; Lakner, A.; Mumberg, D.; Eis, K.; Piccolo, S.; Lange, M.\nScore: 4.0, Published: 2023-08-31 DOI: 10.1101/2023.08.30.555331\nThis study describes the identification and target deconvolution of novel small molecule inhibitors of oncogenic YAP1/TAZ activity with potent anti-tumor activity in vivo. A high-throughput screen (HTS) of 3.8 million compounds was conducted using a cellular YAP1/TAZ reporter assay. Target deconvolution studies identified the geranylgeranyltransferase-I (GGTase-I) complex, as the direct target of YAP1/TAZ pathway inhibitors. The novel small molecule inhibitors block the activation of Rho-GTPases, leading to subsequent inactivation of YAP1/TAZ and inhibition of cancer cell proliferation in vitro. Multi-parameter optimization resulted in BAY-593, an in vivo probe with favorable PK properties, which demonstrated anti-tumor activity and blockade of YAP1/TAZ signaling in vivo. SIGNIFICANCEYAP1/TAZ have been shown to be aberrantly activated oncogenes in several human solid tumors, resulting in enhanced cell proliferation, metastasis and provision of a pro-tumorigenic microenvironment, making YAP1/TAZ targets for novel cancer therapies. Yet, the development of effective inhibitors of these potent oncogenes has been challenging. In this work, we break new ground in this direction through the identification of novel inhibitors of YAP1/TAZ activity. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=192 HEIGHT=200 SRC=\"FIGDIR/small/555331v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (33K): org.highwire.dtl.DTLVardef@1fcdd9forg.highwire.dtl.DTLVardef@3a063borg.highwire.dtl.DTLVardef@16f318forg.highwire.dtl.DTLVardef@11eb782_HPS_FORMAT_FIGEXP M_FIG C_FIG HIGHLIGHTSO_LINovel YAP1/TAZ pathway inhibitors identified by phenotypic high-throughput screen C_LIO_LITarget deconvolution identifies GGTase-I as the direct target of the novel YAP1/TAZ pathway inhibitors C_LIO_LIGGTase-I inhibitors block Rho-GTPase signaling and downstream YAP1/TAZ C_LIO_LIGGTase-I inhibitor BAY-593 demonstrates potent anti-tumor activity in vivo C_LI\n",
  "wordCount" : "2550",
  "inLanguage": "en",
  "datePublished": "2023-09-06T10:39:07Z",
  "dateModified": "2023-09-06T10:39:07Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on September 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.26.554924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.26.554924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.26.554924">
        <p class="paperTitle">Harnessing extracellular vesicles for efficient siRNA delivery in vitro and in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.26.554924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.26.554924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Qiu, T.; Zhai, Y.; Yi, Y.; Wang, X.; Hu, R.; Niu, S.; Wang, C.; Cui, C.; Xu, K.; He, X.</p>
        <p class="info">Score: 441.5, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.26.554924' target='https://doi.org/10.1101/2023.08.26.554924'> 10.1101/2023.08.26.554924</a></p>
        <p class="abstract">To fulfill the potential of small interference RNA (siRNA) therapeutics, diverse and efficient delivery platforms for siRNA are urgently needed. Extracellular vesicles (EVs) are endogenous, cell secreted nano-vehicles which have been explored for functional siRNA delivery, yet impurities in EVs preparation and lack of efficient siRNA loading method have been limiting its further development into clinical applications. In this work, we have achieved high quality EVs production and an siRNA loading system via EVs. The prepared EV-siRNA system was able to deliver siRNA cargo to model cell lines to silence the expression of targeted genes with high potency. Furthermore, EV-siRNA administration in a tumor xenograft model significantly inhibited tumor growth. Therefore, this study establishes an innovative and efficient EV-siRNA platform for siRNA delivery in vitro and in vivo.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.555882">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.555882" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.555882">
        <p class="paperTitle">Glioblastoma is spatially organized by neurodevelopmental programs and a glial-like wound healing response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.555882" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.555882" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mossi Albiach, A.; Janusauskas, J.; Kapustova, I.; Kvedaraite, E.; Codeluppi, S.; Munting, J. B.; Borm, L. E.; Kjaer Jacobsen, J.; Shamikh, A.; Persson, O.; Linnarsson, S.</p>
        <p class="info">Score: 36.8, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.555882' target='https://doi.org/10.1101/2023.09.01.555882'> 10.1101/2023.09.01.555882</a></p>
        <p class="abstract">Glioblastoma is the deadliest brain cancer, characterized by great cellular diversity and unique histology. To understand the spatial organization of transcriptional cell states, we mapped the expression of 888 genes in centimeter-scale tissue sections from a large patient cohort. We found a hierarchy of cellular states akin to normal brain development, including proliferating and differentiated cells interacting with the stroma. We discovered that mesenchymal-like glioblastoma cells comprised a major glial-like wound-response component and a distinct gliosarcoma-specific malignant fibroblast type. Our analysis highlighted hypoxia, tissue damage, and wound healing as major factors in glioblastoma spatial organization. Tumor microenvironment varied along the hypoxia gradient, inducing the recruitment of monocytes and pro-tumorigenic macrophages, and propagating wound response program activation in malignant glial cells. Our study reveals the dynamic progression of glioblastoma organization independent of its mutational profile, in response to tumor-induced injury.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555396">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555396" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555396">
        <p class="paperTitle">KRAS allelic imbalance drives tumour initiation yet suppresses metastasis in colorectal cancer in vivo</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555396" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555396" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Najumudeen, A. K.; Fey, S. K.; Millet, L. M.; Ford, C. A.; Gilroy, K.; Gunduz, N.; Ridgway, R. A.; Anderson, E.; Strathdee, D.; Clark, W.; Nixon, C.; Morton, J. P.; Campbell, A. D.; Sansom, O. J.</p>
        <p class="info">Score: 18.1, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555396' target='https://doi.org/10.1101/2023.08.29.555396'> 10.1101/2023.08.29.555396</a></p>
        <p class="abstract">Oncogenic KRAS mutations are well-described functionally and are known to drive tumorigenesis. Recent reports describe a significant prevalence of KRAS allelic imbalances or gene dosage changes in human cancers, including loss of the wild-type allele in KRAS mutant cancers. However, there is still much debate over the function of wild-type KRAS in tumour initiation, progression and therapeutic response. We have developed a genetically engineered mouse model which allows deletion of the wild-type copy of Kras in the context of an intact oncogenic Kras in colorectal cancer. We observe that in the presence of oncogenic Kras, wild-type Kras acts to restrain tumour growth. Mechanistically, deletion of wild-type Kras exacerbates oncogenic KRAS signalling through MAPK and thus drives tumour initiation. Absence of wild-type Kras potentiates the oncogenic effect of KRASG12D, while presence of wild-type Kras is associated with resistance to inhibition of MEK1/2 in KRASG12D driven tumours. Importantly, loss of wild-type Kras in oncogenic KRAS-driven aggressive tumours significantly alters tumour progression, metastasis while impacting tumour immune cell infiltration. This study demonstrates a suppressive role for wild-type Kras during colon tumour initiation and highlights the critical impact of wild-type Kras upon therapeutic response to MAPK and tumour progression in Kras mutant cancers.

HighlightsO_LIWild-type KRAS suppresses mutant KRASG12D mediated proliferation and signalling in colorectal cancer models in vivo
C_LIO_LIConcomitant loss of wild-type KRAS and activation of WNT signalling promotes mutant KRAS-driven tumour initiation.
C_LIO_LIWild-type KRAS promotes resistance to MAPK inhibition in KRAS mutant tumours
C_LIO_LILoss of wild-type KRAS inhibits metastasis of late-stage mutant KRAS colorectal cancer models.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.552598">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.552598" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.552598">
        <p class="paperTitle">Kras loss of heterozygosity promotes MAPK dependent pancreatic ductal adenocarcinoma and induces therapeutic sensitivity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.552598" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.552598" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fey, S. K.; Najumudeen, A. K.; Ford, C. A.; Gilroy, K.; Upstill-Goddard, R.; Clark, W.; Nixon, C.; Chang, D. K.; Barry, S. T.; Morton, J. P.; Campbell, A. D.; Sansom, O. J.</p>
        <p class="info">Score: 12.8, Published: 2023-08-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.552598' target='https://doi.org/10.1101/2023.08.29.552598'> 10.1101/2023.08.29.552598</a></p>
        <p class="abstract">Pancreatic cancer is characterised by the prevalence of oncogenic mutations in KRAS. Previous studies have reported that altered Kras gene dosage drives progression and metastatic incidence in pancreatic cancer. While the role of oncogenic KRAS mutation is well characterised, the relevance of the partnering wild-type KRAS allele in pancreatic cancer is less well understood and controversial. Using in vivo mouse modelling of pancreatic cancer, we demonstrate that wild-type Kras restrains the oncogenic impact of mutant Kras, and drastically impacts both Kras-mediated tumourigenesis and therapeutic response. Mechanistically, deletion of wild-type Kras increases oncogenic Kras signalling through the downstream MAPK effector pathway, driving pancreatic intraepithelial neoplasia (PanIN) initiation. In addition, in the KPC mouse model, a more aggressive model of pancreatic cancer, loss of wild-type KRAS leads to accelerated initiation but delayed tumour progression. These tumours had altered stroma, downregulated Myc levels and an enrichment for immunogenic gene signatures. Importantly, loss of wild-type Kras sensitises Kras mutant tumours to MEK1/2 inhibition though tumours eventually become resistant and then rapidly progress. This study demonstrates the repressive role of wild-type Kras during pancreatic tumourigenesis and highlights the critical impact of the presence of wild-type KRAS on tumourigenesis and therapeutic response in pancreatic cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.27.554991">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.27.554991" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.27.554991">
        <p class="paperTitle">Evolvability of cancer-associated genes under APOBEC3A/B selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.27.554991" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.27.554991" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Song, J.-H.; Davalos, L.; MacCarthy, T.; Damaghi, M.</p>
        <p class="info">Score: 8.3, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.27.554991' target='https://doi.org/10.1101/2023.08.27.554991'> 10.1101/2023.08.27.554991</a></p>
        <p class="abstract">Evolvability is an emergent hallmark of cancer that depends on intra-tumor heterogeneity and, ultimately, genetic variation. Mutations generated by APOBEC3 cytidine deaminases can contribute to genetic variation and the consequences of APOBEC activation differ depending on the stage of cancer, with the most significant impact observed during the early stages. However, how APOBEC activity shapes evolutionary patterns of genes in the host genome and differential impacts on cancer-associated and non-cancer genes remain unclear. Analyzing over 40,000 human protein-coding transcripts, we identified distinct distribution patterns of APOBEC3A/B TC motifs between cancer-related genes and controls, suggesting unique associations with cancer. Studying a bat species with many more APOBEC3 genes, we found diverse motif patterns in orthologs of cancer genes compared to controls, similar to humans and suggesting APOBEC evolution to reduce impacts on the genome rather than the converse. Simulations confirmed that APOBEC-induced heterogeneity enhances cancer evolution, shaping clonal dynamics through bimodal introduction of mutations in certain classes of genes. Our results suggest that a major consequence of the bimodal distribution of APOBEC affects greater cancer heterogeneity.

HighlightsO_LIUsing a measure of the extent which genes are affected by APOBEC activity, we found that many genes are maximally robust in the human genome. Interestingly, the distribution in the genome of a bat that has many APOBEC genes was similar.
C_LIO_LIIn contrast, when analyzing the subset of cancer-associated genes the distribution is bimodal with many genes appearing to susceptible to APOBEC activity.
C_LIO_LIAnalysis of orthologs of human genes and other species showed a wide range in the extent to which APOBEC affects genes having the same function.
C_LIO_LIThe bimodal distribution of susceptibility to APOBEC drives greater heterogeneity in simulated clonal evolution of cancer.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555056">
        <p class="paperTitle">Integration of Multiple Spatial-Omics Modalities Reveals Unique Insights into Molecular Heterogeneity of Prostate Cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, W.; Spotbeen, X.; Vanuytven, S.; Kint, S.; Sarretto, T.; Socciarelli, F.; Vandereyken, K.; Dehairs, J.; Idkowiak, J.; Wouters, D.; Alvira Larizgoitia, J. I.; Partel, G.; Ly, A.; de Laat, V.; Q. Mantas, M. J.; Gevaert, T.; Devlies, W.; Mah, C. Y.; Butler, L. M.; Loda, M.; Joniau, S.; De Moor, B.; Sifrim, A.; Ellis, S. R.; Voet, T.; Claesen, M.; Verbeeck, N.; Swinnen, J. V.</p>
        <p class="info">Score: 14.9, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555056' target='https://doi.org/10.1101/2023.08.28.555056'> 10.1101/2023.08.28.555056</a></p>
        <p class="abstract">Recent advances in spatial omics methods are revolutionising biomedical research by enabling detailed molecular analyses of cells and their interactions in their native state. As most technologies capture only a specific type of molecules, there is an unmet need to enable integration of multiple spatial-omics datasets. This, however, presents several challenges as these analyses typically operate on separate tissue sections at disparate spatial resolutions. Here, we established a spatial multi-omics integration pipeline enabling co-registration and granularity matching, and applied it to integrate spatial transcriptomics, mass spectrometry-based lipidomics, single nucleus RNA-seq and histomorphological information from human prostate cancer patient samples. This approach revealed unique correlations between lipids and gene expression profiles that are linked to distinct cell populations and histopathological disease states and uncovered molecularly different subregions not discernible by morphology alone. By its ability to correlate datasets that span across the biomolecular and spatial scale, the application of this novel spatial multi-omics integration pipeline provides unprecedented insight into the intricate interplay between different classes of molecules in a tissue context. In addition, it has unique hypothesis-generating potential, and holds promise for applications in molecular pathology, biomarker and target discovery and other tissue-based research fields.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.01.553259">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.01.553259" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.01.553259">
        <p class="paperTitle">Cellular taxonomy of the preleukemic bone marrow niche of acute myeloid leukemia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.01.553259" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.01.553259" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Goda, C.; Kulkarni, R.; Rudich, A.; Karunasiri, M.; Balcioglu, O.; Al-Marrawi, Y.; Korn, E.; Kanna, S.; Garfinkle, E. A. R.; Mundy-Bosse, B.; Zhang, B.; Marcucci, G.; Mardis, E. R.; Garzon, R.; Miller, K. E.; Dorrance, A. M.</p>
        <p class="info">Score: 5.7, Published: 2023-09-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.01.553259' target='https://doi.org/10.1101/2023.09.01.553259'> 10.1101/2023.09.01.553259</a></p>
        <p class="abstract">Mutations in hematopoietic stem/progenitor cells (HSPCs) can remain dormant within the bone marrow (BM) for decades before leukemia onset. Understanding the mechanisms by which these mutant clones eventually slead to full blown leukemia is of critical importance to develop strategies to eliminate these clones before they achieve their full leukemogenic potential. Recent data suggest that leukemic stem cells (LSCs) induce alterations within BM microenvironment (BMM) favoring LSC growth over normal HSCs. However, the cross talk between preleukemic stem cells (pLSC) and BMM is not completely understood. We hypothesize that pLSC induces critical changes within the BMM that are critical for leukemogenesis. To address this question, we are using our previously developed murine model of AML that highly recapitulates the human disease, develops AML sporadically with a preleukemic phase in which mice display normal white blood counts (WBCs) and absence of blasts in the BM. Thus, this is an excellent model to evaluate changes in the BMM that occurs during progression into AML. Using this model we performed single cell RNA-sequencing on cells from the BMM compared to wild-type (WT) controls. Overall, we defined the transcriptional profiles of pre-leukemic BMM cells and observed decreased percentages of normal BMM cells such as LepR&#43; mesenchymal stem cells (MSCs) and endothelial cells (ECs), known to regulate normal HSC function. Concomitantly, we found increases in CD55&#43; fibroblasts and NG2&#43; pericytes, that might play a more important role in regulation of pre-LSCs. Preleukemic CD55&#43; fibroblasts had a higher proliferation rate and showed significant down-regulation of several collagen genes known for regulating extra cellular matrix (ECM) including: Col1a1, Col1a2, Col3a1, Col4a1, and Col6a1, suggesting that ECM remodeling occurs in the early stages of leukemogenesis. Importantly, co-culture assays found that pre-leukemic CD55&#43; BM fibroblasts expanded pre-LSCs significantly over normal HSCs. In conclusion, we have identified distinct changes in the preleukemic BMM and identified a novel CD55&#43; fibroblast population that is expanded in preleukemic BMM that promote the fitness of pre-LSCs over normal HSCs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555214">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555214" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555214">
        <p class="paperTitle">Impact of resistance on therapeutic design: a Moran model of cancer growth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555214" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555214" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lacy, M. S.; Jenner, A. L.</p>
        <p class="info">Score: 7.6, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555214' target='https://doi.org/10.1101/2023.08.28.555214'> 10.1101/2023.08.28.555214</a></p>
        <p class="abstract">Resistance of cancers to treatments, such as chemotherapy, largely arise due to cell mutations. These mutations allow cells to resist apoptosis and inevitably lead to recurrence and often progression to more aggressive cancer forms. Sustained-low dose therapies are being considered as an alternative over maximum tolerated dose treatments, whereby a smaller drug dosage is given over a longer period of time. However, understanding the impact that the presence of treatment-resistant clones may have on these new treatment modalities is crucial to validating them as a therapeutic avenue. In this study, a Moran process is used to capture stochastic mutations arising in cancer cells, inferring treatment resistance. The model is used to predict the probability of cancer recurrence given varying treatment modalities. The simulations predict that sustained-low dose therapies would be virtually ineffective for a cancer with a non-negligible probability of developing a sub-clone with resistance tendencies. Furthermore, calibrating the model to in vivo measurements for breast cancer treatment with Herceptin, the model suggests that standard treatment regimens are ineffective in this mouse model. Using a simple Moran model, it is possible to explore the likelihood of treatment success given a non-negligible probability of treatment resistant mutations and suggest more robust therapeutic schedules.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.29.555282">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.29.555282" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.29.555282">
        <p class="paperTitle">Integration of Mass Cytometry and Mass Spectrometry Imaging for Spatially Resolved Single Cell Metabolic Profiling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.29.555282" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.29.555282" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nunes, J. B.; Ijsselsteijn, M. E.; Abdelaal, T.; Ursem, R.; van der Ploeg, M.; Everts, B.; Mahfouz, A.; Heijs, B.; de Miranda, N.</p>
        <p class="info">Score: 4.5, Published: 2023-08-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.29.555282' target='https://doi.org/10.1101/2023.08.29.555282'> 10.1101/2023.08.29.555282</a></p>
        <p class="abstract">Integration of spatial omics technologies can provide important insights into the biology of tissues. We combined mass spectrometry imaging-based metabolomics and imaging mass cytometry-based immunophenotyping on the same single tissue section to reveal metabolic heterogeneity within tissues and its association with specific cell populations like cancer cells or immune cells. This approach has the potential to greatly increase our understanding of tissue-level interplay between metabolic processes and their cellular components.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.30.555331">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.30.555331" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.30.555331">
        <p class="paperTitle">Novel YAP1/TAZ pathway inhibitors identified through phenotypic screening with potent anti-tumor activity via blockade of GGTase-I / Rho-GTPase signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.30.555331" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.30.555331" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Graham, K.; Lienau, P.; Bader, B.; Prechtl, S.; Naujoks, J.; Lesche, R.; Kuehnlenz, J.; Brzezinka, K.; Potze, L.; Zanconato, F.; Nicke, B.; Montebaur, A.; Bone, W.; Golfier, S.; Kaulfuss, S.; Kopitz, C.; Pilari, S.; Steuber, H.; Hayat, S.; Kamburov, A.; Steffen, A.; Schlicker, A.; Buchgraber, P.; Braeuer, N.; Font, N. A.; Heinrich, T.; Kuhnke, L.; Nowak-Reppel, K.; Stresemann, C.; Steigemann, P.; Walter, A. O.; Blotta, S.; Ocker, M.; Lakner, A.; Mumberg, D.; Eis, K.; Piccolo, S.; Lange, M.</p>
        <p class="info">Score: 4.0, Published: 2023-08-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.30.555331' target='https://doi.org/10.1101/2023.08.30.555331'> 10.1101/2023.08.30.555331</a></p>
        <p class="abstract">This study describes the identification and target deconvolution of novel small molecule inhibitors of oncogenic YAP1/TAZ activity with potent anti-tumor activity in vivo. A high-throughput screen (HTS) of 3.8 million compounds was conducted using a cellular YAP1/TAZ reporter assay. Target deconvolution studies identified the geranylgeranyltransferase-I (GGTase-I) complex, as the direct target of YAP1/TAZ pathway inhibitors. The novel small molecule inhibitors block the activation of Rho-GTPases, leading to subsequent inactivation of YAP1/TAZ and inhibition of cancer cell proliferation in vitro. Multi-parameter optimization resulted in BAY-593, an in vivo probe with favorable PK properties, which demonstrated anti-tumor activity and blockade of YAP1/TAZ signaling in vivo.

SIGNIFICANCEYAP1/TAZ have been shown to be aberrantly activated oncogenes in several human solid tumors, resulting in enhanced cell proliferation, metastasis and provision of a pro-tumorigenic microenvironment, making YAP1/TAZ targets for novel cancer therapies. Yet, the development of effective inhibitors of these potent oncogenes has been challenging. In this work, we break new ground in this direction through the identification of novel inhibitors of YAP1/TAZ activity.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=192 HEIGHT=200 SRC=&#34;FIGDIR/small/555331v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (33K):
org.highwire.dtl.DTLVardef@1fcdd9forg.highwire.dtl.DTLVardef@3a063borg.highwire.dtl.DTLVardef@16f318forg.highwire.dtl.DTLVardef@11eb782_HPS_FORMAT_FIGEXP  M_FIG C_FIG HIGHLIGHTSO_LINovel YAP1/TAZ pathway inhibitors identified by phenotypic high-throughput screen
C_LIO_LITarget deconvolution identifies GGTase-I as the direct target of the novel YAP1/TAZ pathway inhibitors
C_LIO_LIGGTase-I inhibitors block Rho-GTPase signaling and downstream YAP1/TAZ
C_LIO_LIGGTase-I inhibitor BAY-593 demonstrates potent anti-tumor activity in vivo
C_LI</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
