<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Therapeutic targeting of KRAS mutation-driven tumorigenesis by extracellular vesicles loaded with small interfering RNA
Authors: Zhai, Y.; Niu, S.; Qiu, T.; Yi, Y.; Yan, Y.; Hu, R.; He, X.; Xu, K.
Score: 51.5, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576015
RAS is a well-known oncogene contributing to significant proportion of cancer incidences, yet it remains as an undruggable target to majority of therapeutic modalities. Here, we explored the potential of extracellular vesicles (EVs) for therapeutic targeting of KRAS mutation-driven tumorigenesis.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Therapeutic targeting of KRAS mutation-driven tumorigenesis by extracellular vesicles loaded with small interfering RNA
Authors: Zhai, Y.; Niu, S.; Qiu, T.; Yi, Y.; Yan, Y.; Hu, R.; He, X.; Xu, K.
Score: 51.5, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576015
RAS is a well-known oncogene contributing to significant proportion of cancer incidences, yet it remains as an undruggable target to majority of therapeutic modalities. Here, we explored the potential of extracellular vesicles (EVs) for therapeutic targeting of KRAS mutation-driven tumorigenesis." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-31T10:38:28+00:00" />
<meta property="article:modified_time" content="2024-01-31T10:38:28+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Therapeutic targeting of KRAS mutation-driven tumorigenesis by extracellular vesicles loaded with small interfering RNA
Authors: Zhai, Y.; Niu, S.; Qiu, T.; Yi, Y.; Yan, Y.; Hu, R.; He, X.; Xu, K.
Score: 51.5, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576015
RAS is a well-known oncogene contributing to significant proportion of cancer incidences, yet it remains as an undruggable target to majority of therapeutic modalities. Here, we explored the potential of extracellular vesicles (EVs) for therapeutic targeting of KRAS mutation-driven tumorigenesis."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Therapeutic targeting of KRAS mutation-driven tumorigenesis by extracellular vesicles loaded with small interfering RNA\nAuthors: Zhai, Y.; Niu, S.; Qiu, T.; Yi, Y.; Yan, Y.; Hu, R.; He, X.; Xu, K.\nScore: 51.5, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576015\nRAS is a well-known oncogene contributing to significant proportion of cancer incidences, yet it remains as an undruggable target to majority of therapeutic modalities. Here, we explored the potential of extracellular vesicles (EVs) for therapeutic targeting of KRAS mutation-driven tumorigenesis.",
  "keywords": [
    
  ],
  "articleBody": " Therapeutic targeting of KRAS mutation-driven tumorigenesis by extracellular vesicles loaded with small interfering RNA\nAuthors: Zhai, Y.; Niu, S.; Qiu, T.; Yi, Y.; Yan, Y.; Hu, R.; He, X.; Xu, K.\nScore: 51.5, Published: 2024-01-20 DOI: 10.1101/2024.01.17.576015\nRAS is a well-known oncogene contributing to significant proportion of cancer incidences, yet it remains as an undruggable target to majority of therapeutic modalities. Here, we explored the potential of extracellular vesicles (EVs) for therapeutic targeting of KRAS mutation-driven tumorigenesis. EVs loaded with small interfering RNA (siRNA) against KRAS successfully inhibited tumor xenograft growth when injected intratumorally in mice models. Intriguingly, when injected intravenously, EVs were still able to accumulate in tumors and deliver KRAS targeting siRNA payload in sufficient amount to show tumor growth inhibition. Therefore, EV-siRNA platform show great promises for therapeutic targeting of KRAS mutations and other undruggable targets.\nDiscovery of Highly Active Kynureninases for Cancer Immunotherapy through Protein Language Model\nAuthors: Eom, H.; Cho, K. S.; Lee, J.; Kim, S.; Park, S.; Kim, H.; Yang, J.; Han, Y.-H.; Lee, J.; Seok, C.; Lee, M. S.; Song, W. J.; Steinegger, M.\nScore: 27.1, Published: 2024-01-20 DOI: 10.1101/2024.01.16.575968\nOvercoming the immunosuppressive tumor microenvironment is a promising strategy in anticancer therapy. L-kynurenine, a strong immunosuppressive metabolite can be degraded through kynureninases. Through homology searches and protein language models, we identified and then experimentally determined the efficacy of four top-ranked kynureninases. The catalytically most active one nearly doubles turnover number over the prior best, reducing tumor weight by 3.42 times in mouse model comparisons, and thus, presenting substantial therapeutic potential.\nThe temporal evolution of cancer hallmarks\nAuthors: Gourmet, L.; Ramazzotti, D.; Mallick, P.; Walker-Samuel, S.; Zapata, L.\nScore: 14.9, Published: 2024-01-23 DOI: 10.1101/2024.01.21.576566\nCancer hallmarks describe key physiological characteristics that distinguish cancers from normal tissues. The temporal order in which these hallmarks appear during cancer pathogenesis is of interest from both evolutionary and clinical perspectives but has not been investigated before. Here, we order hallmarks based on the allele frequency and selective advantage of mutations in cancer hallmark genes across \u003e10k untreated primary tumors and \u003e8K healthy tissues. Using this novel approach, we identified a common evolutionary trajectory for 27 of 32 cancer types with genomic instability as the first and immune evasion as the last hallmark. We demonstrated widespread positive selection in cancer and strong negative selection in normal tissues for all hallmarks. Notable exceptions to the hallmark ordering in tumours were melanomas (uveal and skin) suggesting that strong environmental factors could disrupt common evolutionary paths. Clustering of hallmark trajectories across patients revealed 2 clusters defined by early or late genomic instability, with differential prognosis. Our study is the first to identify the temporal order of cancer hallmarks during tumorigenesis and demonstrate a prognostic value that could be exploited for early detection and risk stratification across multiple cancer types.\nPersister cancer cells are characterized by H4K20me3 heterochromatin that defines a low inflammatory profile\nAuthors: Ramponi, V.; Richart, L.; Kovatcheva, M.; Stephan-Otto Attolini, C.; Capellades, J.; Lord, A. E.; Yanes, O.; Ficz, G.; Serrano, M.\nScore: 14.2, Published: 2024-01-29 DOI: 10.1101/2024.01.26.577389\nAnti-cancer therapies may induce proliferative arrest in cancer cells in the form of senescence or drug-tolerant persistency, the latter being a reversible arrest with similarities to embryonic diapause. Here, we use mTOR/PI3K inhibition to develop and characterize a model of persistency/diapause-like arrest in human cancer cells of various origins. We show that persister and senescent cancer cells share an expanded lysosomal compartment and hypersensitivity to BCL-XL inhibition. However, persister cells do not exhibit other features of senescence, such as the loss of Lamin B1, senescence-associated {beta}-galactosidase activity, and an inflammatory phenotype. Compared to senescent cells, persister cells have a profoundly diminished senescence-associated secretory phenotype (SASP), low activation of interferon signaling pathways and lack upregulation of MHC-I presentation. Based on a genome-wide CRISPR/Cas9 screen performed in diapause mouse embryonic stem cells (mESC), we discover that persister human cancer cells are hypersensitive to the inhibition of one-carbon metabolism. This finding led us to uncover that the repressive heterochromatic mark H4K20me3 is enriched at promoters of SASP and interferon response genes in persister cells, but not in senescent cells. Collectively, we define novel features and vulnerabilities of persister cancer cells and we provide insight into the epigenetic mechanisms underlying their low inflammatory and immunogenic activity.\nMammalian SWI/SNF complex activity regulates POU2F3 and constitutes a targetable dependency in small cell lung cancer\nAuthors: Duplaquet, L.; So, K.; Ying, A. W.; Li, X.; Li, Y.; Qiu, X.; Li, R.; Singh, S.; Wu, X. S.; Liu, Q.; Qi, J.; Somerville, T. D. D.; Heiling, H.; Mazzola, E.; Lee, Y.; Zoller, T.; Vakoc, C. R.; Doench, J. G.; Forrester, W. C.; Abrams, T.; Long, H. W.; Niederst, M. J.; Kadoch, C.; Oser, M. G.\nScore: 13.5, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576304\nSmall cell lung cancers (SCLC) are comprised of heterogeneous subtypes marked by lineage-specific transcription factors, including ASCL1, NEUROD1, and POU2F3. POU2F3-positive SCLC, [~]12% of all cases, are uniquely dependent on POU2F3 itself; as such, approaches to attenuate POU2F3 expression may represent new therapeutic opportunities. Here using genome-scale screens for regulators of POU2F3 expression and SCLC proliferation, we define mSWI/SNF complexes, including non-canonical BAF (ncBAF) complexes, as top dependencies specific to POU2F3-positive SCLC. Notably, clinical-grade pharmacologic mSWI/SNF inhibition attenuates proliferation of all POU2F3-positive SCLCs, while disruption of ncBAF via BRD9 degradation is uniquely effective in pure non-neuroendocrine POU2F3-SCLCs. mSWI/SNF maintains accessibility over gene loci central to POU2F3-mediated gene regulatory networks. Finally, chemical targeting of SMARCA4/2 mSWI/SNF ATPases and BRD9 decrease POU2F3-SCLC tumor growth and increase survival in vivo. Taken together, these results characterize mSWI/SNF-mediated global governance of the POU2F3 oncogenic program and suggest mSWI/SNF inhibition as a therapeutic strategy for SCLC.\nHypoxia-induced Complement Component 3 Promotes Aggressive Tumor Growth in the Glioblastoma Microenvironment\nAuthors: Rosberg, R.; Smolag, K. I.; Sjolund, J.; Johansson, E.; Bergelin, C.; Wahlden, J.; Pantazopoulou, V.; Ceberg, C.; Pietras, K.; Blom, A. M.; Pietras, A.\nScore: 11.6, Published: 2024-01-29 DOI: 10.1101/2024.01.28.577617\nGlioblastoma (GBM) is the most aggressive form of glioma with a high rate of relapse despite intensive treatment. Tumor recurrence is tightly linked to radio-resistance, which in turn is associated with hypoxia. Here, we discovered a strong link between hypoxia and local complement signaling using publicly available bulk, single cell, and spatially resolved transcriptomic data from human GBM patients. Complement component 3 (C3) and the receptor C3AR1 were both associated with aggressive disease and shorter survival in human glioma. In a genetically engineered mouse model of GBM, we found C3 specifically in hypoxic tumor areas. In vitro, we found an oxygen level-dependent increase in C3 and C3AR1 expression in response to hypoxia in several GBM and stromal cell types. Presence of C3 increased proliferation of GBM cells under hypoxic conditions, as well as clonal survival of GBM cells following radiation. Targeting C3aR using the antagonist SB290157 decreased GBM cell self-renewal in vitro, and prolonged survival of glioma bearing mice both alone and in combination with radiotherapy while reducing the number of M2-polarized macrophages. Our findings establish a strong link between hypoxia and complement pathways in GBM, and support a role of hypoxia-induced C3a-C3aR signaling as a contributor to glioma aggressiveness.\nIncreased interaction between connexin43 and microtubules is critical for glioblastoma stem-like cell maintenance and tumorigenicity.\nAuthors: Smyth, J. W.; Guo, S.; O'Rourke, L.; Deaver, S.; Dahlka, J.; Nurmemmedov, E.; Sheng, Z.; Gourdie, R. G.; Lamouille, S.\nScore: 5.6, Published: 2024-01-27 DOI: 10.1101/2024.01.26.576347\nGlioblastoma (GBM) is the most common primary tumor of the central nervous system. One major challenge in GBM treatment is the resistance to chemotherapy and radiotherapy observed in subpopulations of cancer cells, including GBM stem-like cells (GSCs). These cells hold the ability to self-renew or differentiate following treatment, participating in tumor recurrence. The gap junction protein connexin43 (Cx43) has complex roles in oncogenesis and we have previously demonstrated an association between Cx43 and GBM chemotherapy resistance. Here, we report, for the first time, increased direct interaction between non-junctional Cx43 with microtubules in the cytoplasm of GSCs. We hypothesize that non-junctional Cx43/microtubule complexing is critical for GSC maintenance and survival and sought to specifically disrupt this interaction while maintaining other Cx43 functions, such as gap junction formation. Using a Cx43 mimetic peptide of the carboxyl terminal tubulin-binding domain of Cx43 (JM2), we successfully ablated Cx43 interaction with microtubules in GSCs. Importantly, administration of JM2 significantly decreased GSC survival in vitro, and limited GSC-derived tumor growth in vivo. Together, these results identify JM2 as a novel peptide drug to ablate GSCs in GBM treatment.\nSMAC mimetics overcome apoptotic resistance in ovarian cancer through MSLN-TNF alpha axis\nAuthors: Coelho, R.; Seashore-Ludlow, B.; Schutz, S.; Christopher Lombardo, F.; Moussaud-Lamodiere, E.; Casanova, R.; Ficek-Pascual, J.; Brunhilde Labrosse, K.; Lopez-Nunez, M.; Rimmer, N.; Fedier, A.; Lima, R.; Montavon Sartorius, C.; Kurzeder, C.; Singer, F.; Bertolini, A.; The TumorProfiler Consortium, ; Ratsch, G.; Bodenmiller, B.; Kallioniemi, O.; Ostling, P.; David, L.; Heinzelmann-Schwarz, V.; Jacob, F.\nScore: 3.7, Published: 2024-01-26 DOI: 10.1101/2024.01.24.576987\nResistance to chemotherapy and PARPi inhibitors remains a critical challenge in the treatment of epithelial ovarian cancer, mainly due to disabled apoptotic responses in tumor cells. Given mesothelins pivotal role in ovarian cancer and its restricted expression in healthy tissues, we conducted a drug-screening discovery analysis across a range of genetically modified cancer cells to unveil mesothelins therapeutic impact. We observed enhanced cell death in cancer cells with low mesothelin expressing when exposed to a second mitochondria-derived activator of caspases (SMAC) mimetics, and demonstrated a compelling synergy when combined with chemotherapy in ex vivo patient-derived cultures and zebrafish tumor xenografts. Mechanistically, the addition of the SMAC mimetics drug birinapant to either carboplatin or paclitaxel triggered the activation of the Caspase 8-dependent apoptotic program facilitated by TNF signaling. Multimodal analysis of neoadjuvant-treated patient samples further revealed an association between tumor-associated macrophages and the activation of TNF-related pathways. Our proposed bimodal treatment shows promise in enhancing the clinical management of patients by harnessing the potential of SMAC mimetics alongside conventional chemotherapy.\nContext-dependent effects of CDKN2A and other 9p21 gene losses during the evolution of oesophageal cancer\nAuthors: Ganguli, P.; Colomina-Basanta, C.; Mendez, A.; Armero-Pineiro, M.; Zahra, A.; Acha-Sagrado, A.; Misetic, H.; Devonshire, G.; Kelly, G.; Freeman, A.; Green, M.; Nye, E.; Oesophageal Cancer Clinical and Molecular Stratification (OCCAMS) Consortium, ; Rodriguez-Justo, M.; Spencer, J.; Fitzgerald, R. C.; Ciccarelli, F. D.\nScore: 3.5, Published: 2024-01-26 DOI: 10.1101/2024.01.24.576991\nCDKN2A is a tumour suppressor gene located in chromosome 9p21 and frequently lost in Barretts oesophagus (BO) and oesophageal adenocarcinoma (OAC). How CDKN2A and other 9p21 gene co-deletions affect OAC evolution remains understudied. We explored the effects of 9p21 loss in OACs and cancer progressor and non-progressor BOs with matched genomic, transcriptomic, and clinical data. Despite its cancer driver role, CDKN2A loss in BO prevents OAC initiation by counter-selecting acquisition of TP53 alterations. 9p21 gene co-deletions predict poor patient survival in OAC but not BO through context-dependent effects on cell cycle, oxidative phosphorylation, and interferon response. Immune quantifications using bulk transcriptome, RNAscope and high-dimensional tissue imaging showed that IFNE loss reduces immune infiltration in BO but not OAC. Mechanistically, CDKN2A loss suppresses the maintenance of squamous epithelium, contributing to a more aggressive phenotype. Our study demonstrates context-dependent roles of cancer genes during disease evolution, with consequences for cancer detection and patient management.\nA CANCER PERSISTENT DNA REPAIR CIRCUIT DRIVEN BY MDM2, MDM4 (MDMX), AND MUTANT P53 FOR RECRUITMENT OF MDC1 AND 53BP1 TO CHROMATIN\nAuthors: Ellison, V.; Polotskaia, A.; Leybengrub, P.; Qiu, W.; Lee, R.; Hendrickson, R. C.; Hu, W.; Bargonetti, J.\nScore: 3.4, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576487\nThe influence of the metastasis promoting proteins mutant p53 (mtp53) and MDM2 on Cancer Persistent Repair (CPR) to promote cancer cell survival is understudied. Interactions between the DNA repair choice protein 53BP1 and wild type tumor suppressor protein p53 (wtp53) regulates cell cycle control. Cancer cells often express elevated levels of transcriptionally inactive missense mutant p53 (mtp53) that interacts with MDM2 and MDM4/MDMX (herein called MDMX). The ability of mtp53 to maintain a 53BP1 interaction while in the context of interactions with MDM2 and MDMX has not been described. We asked if MDM2 regulates chromatin-based phosphorylation events in the context of mtp53 by comparing the chromatin of T47D breast cancer cells with and without MDM2 in a phospho-peptide stable isotope labeling in cell culture (SILAC) screen. We found reduced phospho-53BP1 chromatin association, which we confirmed by chromatin fractionation and immunofluorescence in multiple breast cancer cell lines. We used the Proximity Ligation Assay (PLA) in breast cancer cell lines and detected 53BP1 in close proximity to mtp53, MDM2, and the DNA repair protein MDC1. Through disruption of the mtp53-MDM2 interaction, by either Nutlin 3a or a mtp53 R273H C-terminal deletion, we uncovered that mtp53 was required for MDM2-53BP1 interaction foci. Our data suggests that mtp53 works with MDM2 and 53BP1 to promote CPR and cell survival.\n",
  "wordCount" : "2127",
  "inLanguage": "en",
  "datePublished": "2024-01-31T10:38:28Z",
  "dateModified": "2024-01-31T10:38:28Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 31, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576015">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576015" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576015">
        <p class="paperTitle">Therapeutic targeting of KRAS mutation-driven tumorigenesis by extracellular vesicles loaded with small interfering RNA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576015" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576015" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhai, Y.; Niu, S.; Qiu, T.; Yi, Y.; Yan, Y.; Hu, R.; He, X.; Xu, K.</p>
        <p class="info">Score: 51.5, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576015' target='https://doi.org/10.1101/2024.01.17.576015'> 10.1101/2024.01.17.576015</a></p>
        <p class="abstract">RAS is a well-known oncogene contributing to significant proportion of cancer incidences, yet it remains as an undruggable target to majority of therapeutic modalities. Here, we explored the potential of extracellular vesicles (EVs) for therapeutic targeting of KRAS mutation-driven tumorigenesis. EVs loaded with small interfering RNA (siRNA) against KRAS successfully inhibited tumor xenograft growth when injected intratumorally in mice models. Intriguingly, when injected intravenously, EVs were still able to accumulate in tumors and deliver KRAS targeting siRNA payload in sufficient amount to show tumor growth inhibition. Therefore, EV-siRNA platform show great promises for therapeutic targeting of KRAS mutations and other undruggable targets.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575968">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575968" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575968">
        <p class="paperTitle">Discovery of Highly Active Kynureninases for Cancer Immunotherapy through Protein Language Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575968" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575968" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eom, H.; Cho, K. S.; Lee, J.; Kim, S.; Park, S.; Kim, H.; Yang, J.; Han, Y.-H.; Lee, J.; Seok, C.; Lee, M. S.; Song, W. J.; Steinegger, M.</p>
        <p class="info">Score: 27.1, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575968' target='https://doi.org/10.1101/2024.01.16.575968'> 10.1101/2024.01.16.575968</a></p>
        <p class="abstract">Overcoming the immunosuppressive tumor microenvironment is a promising strategy in anticancer therapy. L-kynurenine, a strong immunosuppressive metabolite can be degraded through kynureninases. Through homology searches and protein language models, we identified and then experimentally determined the efficacy of four top-ranked kynureninases. The catalytically most active one nearly doubles turnover number over the prior best, reducing tumor weight by 3.42 times in mouse model comparisons, and thus, presenting substantial therapeutic potential.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.576566">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.576566" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.576566">
        <p class="paperTitle">The temporal evolution of cancer hallmarks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.576566" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.576566" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gourmet, L.; Ramazzotti, D.; Mallick, P.; Walker-Samuel, S.; Zapata, L.</p>
        <p class="info">Score: 14.9, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.576566' target='https://doi.org/10.1101/2024.01.21.576566'> 10.1101/2024.01.21.576566</a></p>
        <p class="abstract">Cancer hallmarks describe key physiological characteristics that distinguish cancers from normal tissues. The temporal order in which these hallmarks appear during cancer pathogenesis is of interest from both evolutionary and clinical perspectives but has not been investigated before. Here, we order hallmarks based on the allele frequency and selective advantage of mutations in cancer hallmark genes across &gt;10k untreated primary tumors and &gt;8K healthy tissues. Using this novel approach, we identified a common evolutionary trajectory for 27 of 32 cancer types with genomic instability as the first and immune evasion as the last hallmark. We demonstrated widespread positive selection in cancer and strong negative selection in normal tissues for all hallmarks. Notable exceptions to the hallmark ordering in tumours were melanomas (uveal and skin) suggesting that strong environmental factors could disrupt common evolutionary paths. Clustering of hallmark trajectories across patients revealed 2 clusters defined by early or late genomic instability, with differential prognosis. Our study is the first to identify the temporal order of cancer hallmarks during tumorigenesis and demonstrate a prognostic value that could be exploited for early detection and risk stratification across multiple cancer types.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577389" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577389">
        <p class="paperTitle">Persister cancer cells are characterized by H4K20me3 heterochromatin that defines a low inflammatory profile</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramponi, V.; Richart, L.; Kovatcheva, M.; Stephan-Otto Attolini, C.; Capellades, J.; Lord, A. E.; Yanes, O.; Ficz, G.; Serrano, M.</p>
        <p class="info">Score: 14.2, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577389' target='https://doi.org/10.1101/2024.01.26.577389'> 10.1101/2024.01.26.577389</a></p>
        <p class="abstract">Anti-cancer therapies may induce proliferative arrest in cancer cells in the form of senescence or drug-tolerant persistency, the latter being a reversible arrest with similarities to embryonic diapause. Here, we use mTOR/PI3K inhibition to develop and characterize a model of persistency/diapause-like arrest in human cancer cells of various origins. We show that persister and senescent cancer cells share an expanded lysosomal compartment and hypersensitivity to BCL-XL inhibition. However, persister cells do not exhibit other features of senescence, such as the loss of Lamin B1, senescence-associated {beta}-galactosidase activity, and an inflammatory phenotype. Compared to senescent cells, persister cells have a profoundly diminished senescence-associated secretory phenotype (SASP), low activation of interferon signaling pathways and lack upregulation of MHC-I presentation. Based on a genome-wide CRISPR/Cas9 screen performed in diapause mouse embryonic stem cells (mESC), we discover that persister human cancer cells are hypersensitive to the inhibition of one-carbon metabolism. This finding led us to uncover that the repressive heterochromatic mark H4K20me3 is enriched at promoters of SASP and interferon response genes in persister cells, but not in senescent cells. Collectively, we define novel features and vulnerabilities of persister cancer cells and we provide insight into the epigenetic mechanisms underlying their low inflammatory and immunogenic activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.576304">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.576304" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.576304">
        <p class="paperTitle">Mammalian SWI/SNF complex activity regulates POU2F3 and constitutes a targetable dependency in small cell lung cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.576304" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.576304" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duplaquet, L.; So, K.; Ying, A. W.; Li, X.; Li, Y.; Qiu, X.; Li, R.; Singh, S.; Wu, X. S.; Liu, Q.; Qi, J.; Somerville, T. D. D.; Heiling, H.; Mazzola, E.; Lee, Y.; Zoller, T.; Vakoc, C. R.; Doench, J. G.; Forrester, W. C.; Abrams, T.; Long, H. W.; Niederst, M. J.; Kadoch, C.; Oser, M. G.</p>
        <p class="info">Score: 13.5, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.576304' target='https://doi.org/10.1101/2024.01.21.576304'> 10.1101/2024.01.21.576304</a></p>
        <p class="abstract">Small cell lung cancers (SCLC) are comprised of heterogeneous subtypes marked by lineage-specific transcription factors, including ASCL1, NEUROD1, and POU2F3. POU2F3-positive SCLC, [~]12% of all cases, are uniquely dependent on POU2F3 itself; as such, approaches to attenuate POU2F3 expression may represent new therapeutic opportunities. Here using genome-scale screens for regulators of POU2F3 expression and SCLC proliferation, we define mSWI/SNF complexes, including non-canonical BAF (ncBAF) complexes, as top dependencies specific to POU2F3-positive SCLC. Notably, clinical-grade pharmacologic mSWI/SNF inhibition attenuates proliferation of all POU2F3-positive SCLCs, while disruption of ncBAF via BRD9 degradation is uniquely effective in pure non-neuroendocrine POU2F3-SCLCs. mSWI/SNF maintains accessibility over gene loci central to POU2F3-mediated gene regulatory networks. Finally, chemical targeting of SMARCA4/2 mSWI/SNF ATPases and BRD9 decrease POU2F3-SCLC tumor growth and increase survival in vivo. Taken together, these results characterize mSWI/SNF-mediated global governance of the POU2F3 oncogenic program and suggest mSWI/SNF inhibition as a therapeutic strategy for SCLC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577617">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577617" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577617">
        <p class="paperTitle">Hypoxia-induced Complement Component 3 Promotes Aggressive Tumor Growth in the Glioblastoma Microenvironment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577617" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577617" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rosberg, R.; Smolag, K. I.; Sjolund, J.; Johansson, E.; Bergelin, C.; Wahlden, J.; Pantazopoulou, V.; Ceberg, C.; Pietras, K.; Blom, A. M.; Pietras, A.</p>
        <p class="info">Score: 11.6, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577617' target='https://doi.org/10.1101/2024.01.28.577617'> 10.1101/2024.01.28.577617</a></p>
        <p class="abstract">Glioblastoma (GBM) is the most aggressive form of glioma with a high rate of relapse despite intensive treatment. Tumor recurrence is tightly linked to radio-resistance, which in turn is associated with hypoxia. Here, we discovered a strong link between hypoxia and local complement signaling using publicly available bulk, single cell, and spatially resolved transcriptomic data from human GBM patients. Complement component 3 (C3) and the receptor C3AR1 were both associated with aggressive disease and shorter survival in human glioma. In a genetically engineered mouse model of GBM, we found C3 specifically in hypoxic tumor areas. In vitro, we found an oxygen level-dependent increase in C3 and C3AR1 expression in response to hypoxia in several GBM and stromal cell types. Presence of C3 increased proliferation of GBM cells under hypoxic conditions, as well as clonal survival of GBM cells following radiation. Targeting C3aR using the antagonist SB290157 decreased GBM cell self-renewal in vitro, and prolonged survival of glioma bearing mice both alone and in combination with radiotherapy while reducing the number of M2-polarized macrophages. Our findings establish a strong link between hypoxia and complement pathways in GBM, and support a role of hypoxia-induced C3a-C3aR signaling as a contributor to glioma aggressiveness.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.576347">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.576347" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.576347">
        <p class="paperTitle">Increased interaction between connexin43 and microtubules is critical for glioblastoma stem-like cell maintenance and tumorigenicity.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.576347" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.576347" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Smyth, J. W.; Guo, S.; O&#39;Rourke, L.; Deaver, S.; Dahlka, J.; Nurmemmedov, E.; Sheng, Z.; Gourdie, R. G.; Lamouille, S.</p>
        <p class="info">Score: 5.6, Published: 2024-01-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.576347' target='https://doi.org/10.1101/2024.01.26.576347'> 10.1101/2024.01.26.576347</a></p>
        <p class="abstract">Glioblastoma (GBM) is the most common primary tumor of the central nervous system. One major challenge in GBM treatment is the resistance to chemotherapy and radiotherapy observed in subpopulations of cancer cells, including GBM stem-like cells (GSCs). These cells hold the ability to self-renew or differentiate following treatment, participating in tumor recurrence. The gap junction protein connexin43 (Cx43) has complex roles in oncogenesis and we have previously demonstrated an association between Cx43 and GBM chemotherapy resistance. Here, we report, for the first time, increased direct interaction between non-junctional Cx43 with microtubules in the cytoplasm of GSCs. We hypothesize that non-junctional Cx43/microtubule complexing is critical for GSC maintenance and survival and sought to specifically disrupt this interaction while maintaining other Cx43 functions, such as gap junction formation. Using a Cx43 mimetic peptide of the carboxyl terminal tubulin-binding domain of Cx43 (JM2), we successfully ablated Cx43 interaction with microtubules in GSCs. Importantly, administration of JM2 significantly decreased GSC survival in vitro, and limited GSC-derived tumor growth in vivo. Together, these results identify JM2 as a novel peptide drug to ablate GSCs in GBM treatment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.576987">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.576987" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.576987">
        <p class="paperTitle">SMAC mimetics overcome apoptotic resistance in ovarian cancer through MSLN-TNF alpha axis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.576987" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.576987" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coelho, R.; Seashore-Ludlow, B.; Schutz, S.; Christopher Lombardo, F.; Moussaud-Lamodiere, E.; Casanova, R.; Ficek-Pascual, J.; Brunhilde Labrosse, K.; Lopez-Nunez, M.; Rimmer, N.; Fedier, A.; Lima, R.; Montavon Sartorius, C.; Kurzeder, C.; Singer, F.; Bertolini, A.; The TumorProfiler Consortium,  ; Ratsch, G.; Bodenmiller, B.; Kallioniemi, O.; Ostling, P.; David, L.; Heinzelmann-Schwarz, V.; Jacob, F.</p>
        <p class="info">Score: 3.7, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.576987' target='https://doi.org/10.1101/2024.01.24.576987'> 10.1101/2024.01.24.576987</a></p>
        <p class="abstract">Resistance to chemotherapy and PARPi inhibitors remains a critical challenge in the treatment of epithelial ovarian cancer, mainly due to disabled apoptotic responses in tumor cells. Given mesothelins pivotal role in ovarian cancer and its restricted expression in healthy tissues, we conducted a drug-screening discovery analysis across a range of genetically modified cancer cells to unveil mesothelins therapeutic impact. We observed enhanced cell death in cancer cells with low mesothelin expressing when exposed to a second mitochondria-derived activator of caspases (SMAC) mimetics, and demonstrated a compelling synergy when combined with chemotherapy in ex vivo patient-derived cultures and zebrafish tumor xenografts. Mechanistically, the addition of the SMAC mimetics drug birinapant to either carboplatin or paclitaxel triggered the activation of the Caspase 8-dependent apoptotic program facilitated by TNF signaling. Multimodal analysis of neoadjuvant-treated patient samples further revealed an association between tumor-associated macrophages and the activation of TNF-related pathways. Our proposed bimodal treatment shows promise in enhancing the clinical management of patients by harnessing the potential of SMAC mimetics alongside conventional chemotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.576991">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.576991" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.576991">
        <p class="paperTitle">Context-dependent effects of CDKN2A and other 9p21 gene losses during the evolution of oesophageal cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.576991" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.576991" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ganguli, P.; Colomina-Basanta, C.; Mendez, A.; Armero-Pineiro, M.; Zahra, A.; Acha-Sagrado, A.; Misetic, H.; Devonshire, G.; Kelly, G.; Freeman, A.; Green, M.; Nye, E.; Oesophageal Cancer Clinical and Molecular Stratification (OCCAMS) Consortium,  ; Rodriguez-Justo, M.; Spencer, J.; Fitzgerald, R. C.; Ciccarelli, F. D.</p>
        <p class="info">Score: 3.5, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.576991' target='https://doi.org/10.1101/2024.01.24.576991'> 10.1101/2024.01.24.576991</a></p>
        <p class="abstract">CDKN2A is a tumour suppressor gene located in chromosome 9p21 and frequently lost in Barretts oesophagus (BO) and oesophageal adenocarcinoma (OAC). How CDKN2A and other 9p21 gene co-deletions affect OAC evolution remains understudied. We explored the effects of 9p21 loss in OACs and cancer progressor and non-progressor BOs with matched genomic, transcriptomic, and clinical data. Despite its cancer driver role, CDKN2A loss in BO prevents OAC initiation by counter-selecting acquisition of TP53 alterations. 9p21 gene co-deletions predict poor patient survival in OAC but not BO through context-dependent effects on cell cycle, oxidative phosphorylation, and interferon response. Immune quantifications using bulk transcriptome, RNAscope and high-dimensional tissue imaging showed that IFNE loss reduces immune infiltration in BO but not OAC. Mechanistically, CDKN2A loss suppresses the maintenance of squamous epithelium, contributing to a more aggressive phenotype. Our study demonstrates context-dependent roles of cancer genes during disease evolution, with consequences for cancer detection and patient management.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576487">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576487" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576487">
        <p class="paperTitle">A CANCER PERSISTENT DNA REPAIR CIRCUIT DRIVEN BY MDM2, MDM4 (MDMX), AND MUTANT P53 FOR RECRUITMENT OF MDC1 AND 53BP1 TO CHROMATIN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576487" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576487" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ellison, V.; Polotskaia, A.; Leybengrub, P.; Qiu, W.; Lee, R.; Hendrickson, R. C.; Hu, W.; Bargonetti, J.</p>
        <p class="info">Score: 3.4, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576487' target='https://doi.org/10.1101/2024.01.20.576487'> 10.1101/2024.01.20.576487</a></p>
        <p class="abstract">The influence of the metastasis promoting proteins mutant p53 (mtp53) and MDM2 on Cancer Persistent Repair (CPR) to promote cancer cell survival is understudied. Interactions between the DNA repair choice protein 53BP1 and wild type tumor suppressor protein p53 (wtp53) regulates cell cycle control. Cancer cells often express elevated levels of transcriptionally inactive missense mutant p53 (mtp53) that interacts with MDM2 and MDM4/MDMX (herein called MDMX). The ability of mtp53 to maintain a 53BP1 interaction while in the context of interactions with MDM2 and MDMX has not been described. We asked if MDM2 regulates chromatin-based phosphorylation events in the context of mtp53 by comparing the chromatin of T47D breast cancer cells with and without MDM2 in a phospho-peptide stable isotope labeling in cell culture (SILAC) screen. We found reduced phospho-53BP1 chromatin association, which we confirmed by chromatin fractionation and immunofluorescence in multiple breast cancer cell lines. We used the Proximity Ligation Assay (PLA) in breast cancer cell lines and detected 53BP1 in close proximity to mtp53, MDM2, and the DNA repair protein MDC1. Through disruption of the mtp53-MDM2 interaction, by either Nutlin 3a or a mtp53 R273H C-terminal deletion, we uncovered that mtp53 was required for MDM2-53BP1 interaction foci. Our data suggests that mtp53 works with MDM2 and 53BP1 to promote CPR and cell survival.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
