<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Alveolar differentiation drives resistance to KRAS inhibition in lung adenocarcinoma
Authors: Li, Z.; Zhuang, X.; Pan, C.-H.; Yan, Y.; Thummalapalli, R.; Hallin, J.; Torborg, S.; Singhal, A.; Chang, J. C.; Manchado, E.; Dow, L. E.; Yaeger, R.; Christensen, J. G.; Lowe, S. W.; Rudin, C. M.; Joost, S.; Tammela, T.
Score: 34.3, Published: 2023-10-01 DOI: 10.1101/2023.09.29.560194
Lung adenocarcinoma (LUAD), commonly driven by KRAS mutations, is responsible for 7% of all cancer mortality.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Alveolar differentiation drives resistance to KRAS inhibition in lung adenocarcinoma
Authors: Li, Z.; Zhuang, X.; Pan, C.-H.; Yan, Y.; Thummalapalli, R.; Hallin, J.; Torborg, S.; Singhal, A.; Chang, J. C.; Manchado, E.; Dow, L. E.; Yaeger, R.; Christensen, J. G.; Lowe, S. W.; Rudin, C. M.; Joost, S.; Tammela, T.
Score: 34.3, Published: 2023-10-01 DOI: 10.1101/2023.09.29.560194
Lung adenocarcinoma (LUAD), commonly driven by KRAS mutations, is responsible for 7% of all cancer mortality." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-04T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-10-04T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Alveolar differentiation drives resistance to KRAS inhibition in lung adenocarcinoma
Authors: Li, Z.; Zhuang, X.; Pan, C.-H.; Yan, Y.; Thummalapalli, R.; Hallin, J.; Torborg, S.; Singhal, A.; Chang, J. C.; Manchado, E.; Dow, L. E.; Yaeger, R.; Christensen, J. G.; Lowe, S. W.; Rudin, C. M.; Joost, S.; Tammela, T.
Score: 34.3, Published: 2023-10-01 DOI: 10.1101/2023.09.29.560194
Lung adenocarcinoma (LUAD), commonly driven by KRAS mutations, is responsible for 7% of all cancer mortality."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Alveolar differentiation drives resistance to KRAS inhibition in lung adenocarcinoma\nAuthors: Li, Z.; Zhuang, X.; Pan, C.-H.; Yan, Y.; Thummalapalli, R.; Hallin, J.; Torborg, S.; Singhal, A.; Chang, J. C.; Manchado, E.; Dow, L. E.; Yaeger, R.; Christensen, J. G.; Lowe, S. W.; Rudin, C. M.; Joost, S.; Tammela, T.\nScore: 34.3, Published: 2023-10-01 DOI: 10.1101/2023.09.29.560194\nLung adenocarcinoma (LUAD), commonly driven by KRAS mutations, is responsible for 7% of all cancer mortality.",
  "keywords": [
    
  ],
  "articleBody": " Alveolar differentiation drives resistance to KRAS inhibition in lung adenocarcinoma\nAuthors: Li, Z.; Zhuang, X.; Pan, C.-H.; Yan, Y.; Thummalapalli, R.; Hallin, J.; Torborg, S.; Singhal, A.; Chang, J. C.; Manchado, E.; Dow, L. E.; Yaeger, R.; Christensen, J. G.; Lowe, S. W.; Rudin, C. M.; Joost, S.; Tammela, T.\nScore: 34.3, Published: 2023-10-01 DOI: 10.1101/2023.09.29.560194\nLung adenocarcinoma (LUAD), commonly driven by KRAS mutations, is responsible for 7% of all cancer mortality. The first allele-specific KRAS inhibitors were recently approved in LUAD, but clinical benefit is limited by intrinsic and acquired resistance. LUAD predominantly arises from alveolar type 2 (AT2) cells, which function as facultative alveolar stem cells by self-renewing and replacing alveolar type 1 (AT1) cells. Using genetically engineered mouse models, patient-derived xenografts, and patient samples we found inhibition of KRAS promotes transition to a quiescent AT1-like cancer cell state in LUAD tumors. Similarly, suppressing Kras induced AT1 differentiation of wild-type AT2 cells upon lung injury. The AT1-like LUAD cells exhibited high growth and differentiation potential upon treatment cessation, whereas ablation of the AT1-like cells robustly improved treatment response to KRAS inhibitors. Our results uncover an unexpected role for KRAS in promoting intra-tumoral heterogeneity and suggest targeting alveolar differentiation may augment KRAS-targeted therapies in LUAD.\necDNA amplification of MYC drives intratumor copy-number heterogeneity and adaptation to stress in PDAC\nAuthors: Malinova, A.; Schreyer, D.; Fiorini, E.; Pasini, D.; Bevere, M.; D'Agosto, S.; Andreani, S.; Lupo, F.; Veghini, L.; Grimaldi, S.; Pedron, S.; Nourse, C.; Salvia, R.; Malleo, G.; Ruzzenente, A.; Guglielmi, A.; Milella, M.; Lawlor, R. T.; Luchini, C.; Pilarsky, C.; Scarpa, A.; Tuveson, D.; Bailey, P.; Corbo, V.\nScore: 20.7, Published: 2023-09-29 DOI: 10.1101/2023.09.27.559717\nIntratumor heterogeneity and phenotypic plasticity drive tumour progression and therapy resistance. Oncogene dosage variation contributes to cell state transitions and phenotypic heterogeneity, thereby providing a substrate for somatic evolution. Nonetheless, the genetic mechanisms underlying phenotypic heterogeneity are still poorly understood. Here, we show that extrachromosomal DNA (ecDNA) is a major source of high-level focal amplification in key oncogenes and a major contributor of MYC heterogeneity in pancreatic ductal adenocarcinoma (PDAC). We demonstrate that ecDNA can drive exceptionally high dosage of MYC and afford cancer cells rapid adaptation to microenvironmental changes. The continued maintenance of extrachromosomal MYC is uniquely ensured by the presence of the selective pressure. We also show that MYC dosage affects cell morphology and dependence of cancer cells on stromal niche factors, with the highest MYC levels correlating with squamous-like phenotypes. Our work provides the first detailed analysis of ecDNAs in PDAC and describes a new genetic mechanism driving MYC heterogeneity in PDAC.\nMapping the tumor stress network reveals dynamic shifts in the stromal oxidative stress response\nAuthors: Lior, C.; Barki, D.; Kelsen, D.; Iacobuzio-Donahue, C.; Scherz-Shouval, R.\nScore: 10.3, Published: 2023-10-01 DOI: 10.1101/2023.09.29.560126\nThe tumor microenvironment (TME) is a challenging environment where cells must cope with stressful conditions such as fluctuating pH levels, hypoxia, and free radicals. In response, stress pathways are activated, which can both promote and inhibit tumorigenesis. In this study, we set out to characterize the stress response landscape across four carcinomas: breast, pancreas, ovary, and prostate tumors, focusing on five pathways: Heat shock response, oxidative stress response, unfolded protein response, hypoxia stress response, and DNA damage response. Using a combination of experimental and computational methods, we create an atlas of the stress response landscape across various types of carcinomas. We find that stress responses are heterogeneously activated in the TME, and highly activated near cancer cells. Focusing on the non-immune stroma we find, across tumor types, that NRF2 and the oxidative stress response are distinctly activated in immune-regulatory cancer-associated fibroblasts and in a unique subset of cancer associated pericytes. Our study thus provides an interactome of stress responses in cancer, offering new ways to intersect survival pathways within the tumor, and advance cancer therapy.\nSelective Mitochondrial Respiratory Complex I Subunit Deficiency Causes Tumor Immunogenicity\nAuthors: Liang, J.; Vitale, T.; Zhang, X.; Jackson, T. D.; Yu, D.; Jedrychowsk, M.; Gygi, S. P.; Widlund, H. R.; Wucherpfennig, K.; Puigserver, P.\nScore: 7.3, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560316\nTargeting of specific metabolic pathways in tumor cells has the potential to sensitize them to immune-mediated attack. Here we provide evidence for a specific means of mitochondrial respiratory Complex I (CI) inhibition that improves tumor immunogenicity and sensitivity to immune checkpoint blockade (ICB). Targeted genetic deletion of the CI subunits Ndufs4 and Ndufs6, but not other subunits, induces an immune-dependent tumor growth attenuation in mouse melanoma models. We show that deletion of Ndufs4 induces expression of the transcription factor Nlrc5 and genes in the MHC class I antigen presentation and processing pathway. This induction of MHC-related genes is driven by an accumulation of pyruvate dehydrogenase-dependent mitochondrial acetyl-CoA downstream of CI subunit deletion. This work provides a novel functional modality by which selective CI inhibition restricts tumor growth, suggesting that specific targeting of Ndufs4, or related CI subunits, increases T-cell mediated immunity and sensitivity to ICB.\nExpression of the CIC-DUX4 fusion oncoprotein mimics human CIC-rearranged sarcoma in genetically engineered mouse models\nAuthors: Hendrickson, P. G.; Oristian, K. M.; Browne, M. R.; Lou, L.; Ma, Y.; Cardona, D. M.; Linardic, C. M.; Kirsch, D. G.\nScore: 7.0, Published: 2023-09-28 DOI: 10.1101/2023.09.26.559519\nCIC-DUX4 sarcoma (CDS) is a rare but highly aggressive undifferentiated small round cell sarcoma driven by a fusion between the tumor suppressor Capicua (CIC) and DUX4. Currently, there are no effective treatments and efforts to identify and translate better therapies are limited by the scarcity of tissues and patients. To address this limitation, we generated three genetically engineered mouse models of CDS (Ch7CDS, Ai9CDS, and TOPCDS). Remarkably, chimeric mice from all three conditional models developed spontaneous tumors and widespread metastasis in the absence of Cre-recombinase. The penetrance of spontaneous (Cre-independent) tumor formation was complete irrespective of bi-allelic CIC function and loxP site proximity. Characterization of primary and metastatic mouse tumors showed that they consistently expressed the CIC-DUX4 fusion protein as well as other downstream markers of the disease credentialing these models as CDS. In addition, tumor-derived cell lines were generated and ChIP-seq was preformed to map fusion-gene specific binding using an N-terminal HA epitope tag. These datasets, along with paired H3K27ac ChIP-seq maps, validate CIC-DUX4 as a neomorphic transcriptional activator. Moreover, they are consistent with a model where ETS family transcription factors are cooperative and redundant drivers of the core regulatory circuitry in CDS.\nClonal fitness decline in somatic differentiation hierarchies\nAuthors: Ahmed, I.; Dingli, D.; Huang, W.; Werner, B.\nScore: 6.9, Published: 2023-09-29 DOI: 10.1101/2023.09.27.559763\nThe concept of clonal fitness is fundamental to describe the evolutionary dynamics in somatic tissues. It is now well established that otherwise healthy somatic tissues become increasingly populated by expanding clones with age. However, the dynamic properties and respective fitnesses of these clones are less well understood. Here we show, that in somatic tissues organised as a differentiation hierarchy, theory predicts a natural decline of effective clonal fitness over time in the absence of additional driver events. This decline is intrinsic to the tissue organisation and can be captured quantitatively by a simple heuristic equation that is proportional to 1/time. We also show that the expected fitness decline is directly observable in human haematopoiesis. The predicted short and long term dynamics agree with in vivo observations using data of Neutrophil recovery after bone marrow transplants and naturally progressing Chronic Lymphocyte Leukemia (CLL). We further show that theory predicts the existence of a long term equilibrium fitness. All CLL patients transition into a stable equilibrium fitness eventually. We find significant inter-patient variation of long term fitness and a strong correlation with disease aggressiveness. Interestingly, CLL long term fitness can be forecast based on the early stages of disease progression, suggesting a Big Bang like model for CLL evolution.\nThe target gene regulatory network of miR-497 in angiosarcoma\nAuthors: Benton, A.; Terwilliger, E.; Moriarty, N. M.; Liu, B.; Murphy, A.; Maluvac, H.; Shu, M.; Gartenhaus, L. E.; Janson, N. D.; Pfeffer, C. M.; Utturkar, S. M.; Parkinson, E. I.; Lanman, N. A.; Hanna, J. A.\nScore: 7.2, Published: 2023-09-25 DOI: 10.1101/2023.09.24.559218\nAngiosarcoma (AS) is a vascular sarcoma that is highly aggressive and metastatic. Due to its rarity, treatment options for patients are limited, therefore more research is needed to identify possible therapeutic vulnerabilities. We previously found that conditional deletion of Dicer1 drives AS development in mice. Given the role of DICER1 in canonical microRNA (miRNA) biogenesis, this suggests that miRNA loss is important in AS development. After testing miRNAs previously suggested to have a tumor-suppressive role in AS, microRNA-497-5p (miR-497) suppressed cell viability most significantly. We also found that miR-497 overexpression led to significantly reduced cell migration and tumor formation. To understand the mechanism of miR-497 in tumor suppression, we identified clinically relevant target genes using a combination of RNA-sequencing data in an AS cell line, expression data from AS patients, and target prediction algorithms. We validated miR-497 direct regulation of CCND2, CDK6, and VAT1. One of these genes, VAT1, is an understudied protein that has been suggested to promote cell migration and metastasis in other cancers. Indeed, we find that pharmacologic inhibition of VAT1 with the natural product Neocarzilin A reduces AS migration. This work provides insight into the mechanisms of miR-497 and its target genes in AS pathogenesis.\nEvolutionary fingerprints of EMT in pancreatic cancers\nAuthors: Perelli, L.; Zhang, L.; Mangiameli, S.; Russell, A. J. C.; Giannese, F.; Peng, F.; Carbone, F.; Le, C.; Khan, H.; Citron, F.; Soeung, M.; Lam, T.; Lundgren, S.; Zhu, C.; Catania, D.; Feng, N.; Gurreri, E.; Sgambato, A.; Tortora, G.; Draetta, G.; Tonon, G.; Futreal, A.; Giuliani, V.; Carugo, A.; Viale, A.; Heffernan, T.; Wang, L.; Cittaro, D.; Chen, F.; Genovese, G.\nScore: 11.3, Published: 2023-09-19 DOI: 10.1101/2023.09.18.558231\nMesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in pancreatic cancer. We leveraged ad hoc somatic mosaic genome engineering, lineage tracing and ablation technologies and dynamic genetic reporters to trace and ablate tumor-specific lineages along the phenotypic spectrum of epithelial to mesenchymal plasticity. The experimental evidences clarify the essential contribution of mesenchymal lineages to pancreatic cancer evolution and metastatic dissemination. Spatial genomic analysis combined with single cell transcriptomic and epigenomic profiling of epithelial and mesenchymal lineages reveals that EMT promotes with the emergence of chromosomal instability (CIN). Specifically tumor lineages with mesenchymal features display highly conserved patterns of genomic evolution including complex structural genomic rearrangements and chromotriptic events. Genetic ablation of mesenchymal lineages robustly abolished these mutational processes and evolutionary patterns, as confirmed by cross species analysis of pancreatic and other human epithelial cancers. Mechanistically, we discovered that malignant cells with mesenchymal features display increased chromatin accessibility, particularly in the pericentromeric and centromeric regions, which in turn results in delayed mitosis and catastrophic cell division. Therefore, EMT favors the emergence of high-fitness tumor cells, strongly supporting the concept of a cell-state, lineage-restricted patterns of evolution, where cancer cell sub-clonal speciation is propagated to progenies only through restricted functional compartments. Restraining those evolutionary routes through genetic ablation of clones capable of mesenchymal plasticity and extinction of the derived lineages completely abrogates the malignant potential of one of the most aggressive form of human cancer.\nNuclear speckles regulate HIF-2α programs and correlate with patient survival in kidney cancer\nAuthors: Alexander, K. A.; Yu, R.; Skuli, N.; Coffey, N. J.; Nguyen, S.; Faunce, C.; Huang, H.; Dardani, I.; Good, A. L.; Lim, J.; Li, C.; Biddle, N.; Joyce, E. F.; Raj, A.; Lee, D.; Keith, B.; Simon, M. C.; Berger, S. L.\nScore: 5.5, Published: 2023-09-16 DOI: 10.1101/2023.09.14.557228\nNuclear speckles are membrane-less bodies within the cell nucleus enriched in RNA biogenesis, processing, and export factors. In this study we investigated speckle phenotype variation in human cancer, finding a reproducible speckle signature, based on RNA expression of speckle-resident proteins, across \u003e20 cancer types. Of these, clear cell renal cell carcinoma (ccRCC) exhibited a clear correlation between the presence of this speckle expression signature, imaging-based speckle phenotype, and clinical outcomes. ccRCC is typified by hyperactivation of the HIF-2 transcription factor, and we demonstrate here that HIF-2 drives physical association of a select subset of its target genes with nuclear speckles. Disruption of HIF-2-driven speckle association via deletion of its speckle targeting motifs (STMs)--defined in this study--led to defective induction of speckle-associating HIF-2 target genes without impacting non-speckle-associating HIF-2 target genes. We further identify the RNA export complex, TREX, as being specifically altered in speckle signature, and knockdown of key TREX component, ALYREF, also compromises speckle-associated gene expression. By integrating tissue culture functional studies with tumor genomic and imaging analysis, we show that HIF-2 gene regulatory programs are impacted by specific manipulation of speckle phenotype and by abrogation of speckle targeting abilities of HIF-2. These findings suggest that, in ccRCC, a key biological function of nuclear speckles is to modulate expression of a specific subset of HIF-2-regulated target genes that, in turn, influence patient outcomes. We also identify STMs in other transcription factors, suggesting that DNA-speckle targeting may be a general mechanism of gene regulation. HIGHLIGHTS- Nuclear speckles shown to reproducibly vary in cancer, predicting patient survival in ccRCC - HIF-2 drives DNA/gene-speckle contacts dependent on identified speckle targeting motifs within HIF-2 - Putative speckle targeting motifs are highly enriched among regulators of gene expression - Partitioning of transcription factor functional programs may be a major biological function of nuclear speckles\nTargeting Acute Myeloid Leukemia Stem Cells Through Perturbation of Mitochondrial Calcium\nAuthors: Sheth, A. I.; Engel, K.; Tolison, H.; Althoff, M. J.; Amaya, M. L.; Krug, A.; Young, T.; Pei, S.; Patel, S. B.; Minhajuddin, M.; Winters, A.; Miller, R.; Shelton, I.; St-Germain, J.; Ling, T.; Jones, C.; Raught, B.; Gillen, A. E.; Ransom, M. R.; Staggs, S.; Smith, C.; Pollyea, D. A.; Stevens, B. M.; Jordan, C. T.\nScore: 4.8, Published: 2023-10-03 DOI: 10.1101/2023.10.02.560330\nWe previously reported that acute myeloid leukemia stem cells (LSCs) are uniquely reliant on oxidative phosphorylation (OXPHOS) for survival. Moreover, maintenance of OXPHOS is dependent on BCL2, creating a therapeutic opportunity to target LSCs using the BCL2 inhibitor drug venetoclax. While venetoclax-based regimens have indeed shown promising clinical activity, the emergence of drug resistance is prevalent. Thus, in the present study, we investigated how mitochondrial properties may influence mechanisms that dictate venetoclax responsiveness. Our data show that utilization of mitochondrial calcium is fundamentally different between drug responsive and non-responsive LSCs. By comparison, venetoclax-resistant LSCs demonstrate a more active metabolic (i.e., OXPHOS) status with relatively high steady-state levels of calcium. Consequently, we tested genetic and pharmacological approaches to target the mitochondrial calcium uniporter, MCU. We demonstrate that inhibition of calcium uptake sharply reduces OXPHOS and leads to eradication of venetoclax-resistant LSCs. These findings demonstrate a central role for calcium signaling in the biology of LSCs and provide a therapeutic avenue for clinical management of venetoclax resistance.\n",
  "wordCount" : "2505",
  "inLanguage": "en",
  "datePublished": "2023-10-04T10:39:33Z",
  "dateModified": "2023-10-04T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on October 4, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560194">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560194" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560194">
        <p class="paperTitle">Alveolar differentiation drives resistance to KRAS inhibition in lung adenocarcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560194" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560194" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, Z.; Zhuang, X.; Pan, C.-H.; Yan, Y.; Thummalapalli, R.; Hallin, J.; Torborg, S.; Singhal, A.; Chang, J. C.; Manchado, E.; Dow, L. E.; Yaeger, R.; Christensen, J. G.; Lowe, S. W.; Rudin, C. M.; Joost, S.; Tammela, T.</p>
        <p class="info">Score: 34.3, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560194' target='https://doi.org/10.1101/2023.09.29.560194'> 10.1101/2023.09.29.560194</a></p>
        <p class="abstract">Lung adenocarcinoma (LUAD), commonly driven by KRAS mutations, is responsible for 7% of all cancer mortality. The first allele-specific KRAS inhibitors were recently approved in LUAD, but clinical benefit is limited by intrinsic and acquired resistance. LUAD predominantly arises from alveolar type 2 (AT2) cells, which function as facultative alveolar stem cells by self-renewing and replacing alveolar type 1 (AT1) cells. Using genetically engineered mouse models, patient-derived xenografts, and patient samples we found inhibition of KRAS promotes transition to a quiescent AT1-like cancer cell state in LUAD tumors. Similarly, suppressing Kras induced AT1 differentiation of wild-type AT2 cells upon lung injury. The AT1-like LUAD cells exhibited high growth and differentiation potential upon treatment cessation, whereas ablation of the AT1-like cells robustly improved treatment response to KRAS inhibitors. Our results uncover an unexpected role for KRAS in promoting intra-tumoral heterogeneity and suggest targeting alveolar differentiation may augment KRAS-targeted therapies in LUAD.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559717">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559717" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559717">
        <p class="paperTitle">ecDNA amplification of MYC drives intratumor copy-number heterogeneity and adaptation to stress in PDAC</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559717" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559717" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Malinova, A.; Schreyer, D.; Fiorini, E.; Pasini, D.; Bevere, M.; D&#39;Agosto, S.; Andreani, S.; Lupo, F.; Veghini, L.; Grimaldi, S.; Pedron, S.; Nourse, C.; Salvia, R.; Malleo, G.; Ruzzenente, A.; Guglielmi, A.; Milella, M.; Lawlor, R. T.; Luchini, C.; Pilarsky, C.; Scarpa, A.; Tuveson, D.; Bailey, P.; Corbo, V.</p>
        <p class="info">Score: 20.7, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559717' target='https://doi.org/10.1101/2023.09.27.559717'> 10.1101/2023.09.27.559717</a></p>
        <p class="abstract">Intratumor heterogeneity and phenotypic plasticity drive tumour progression and therapy resistance. Oncogene dosage variation contributes to cell state transitions and phenotypic heterogeneity, thereby providing a substrate for somatic evolution. Nonetheless, the genetic mechanisms underlying phenotypic heterogeneity are still poorly understood. Here, we show that extrachromosomal DNA (ecDNA) is a major source of high-level focal amplification in key oncogenes and a major contributor of MYC heterogeneity in pancreatic ductal adenocarcinoma (PDAC). We demonstrate that ecDNA can drive exceptionally high dosage of MYC and afford cancer cells rapid adaptation to microenvironmental changes. The continued maintenance of extrachromosomal MYC is uniquely ensured by the presence of the selective pressure. We also show that MYC dosage affects cell morphology and dependence of cancer cells on stromal niche factors, with the highest MYC levels correlating with squamous-like phenotypes. Our work provides the first detailed analysis of ecDNAs in PDAC and describes a new genetic mechanism driving MYC heterogeneity in PDAC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.29.560126">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.29.560126" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.29.560126">
        <p class="paperTitle">Mapping the tumor stress network reveals dynamic shifts in the stromal oxidative stress response</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.29.560126" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.29.560126" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lior, C.; Barki, D.; Kelsen, D.; Iacobuzio-Donahue, C.; Scherz-Shouval, R.</p>
        <p class="info">Score: 10.3, Published: 2023-10-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.29.560126' target='https://doi.org/10.1101/2023.09.29.560126'> 10.1101/2023.09.29.560126</a></p>
        <p class="abstract">The tumor microenvironment (TME) is a challenging environment where cells must cope with stressful conditions such as fluctuating pH levels, hypoxia, and free radicals. In response, stress pathways are activated, which can both promote and inhibit tumorigenesis. In this study, we set out to characterize the stress response landscape across four carcinomas: breast, pancreas, ovary, and prostate tumors, focusing on five pathways: Heat shock response, oxidative stress response, unfolded protein response, hypoxia stress response, and DNA damage response. Using a combination of experimental and computational methods, we create an atlas of the stress response landscape across various types of carcinomas. We find that stress responses are heterogeneously activated in the TME, and highly activated near cancer cells. Focusing on the non-immune stroma we find, across tumor types, that NRF2 and the oxidative stress response are distinctly activated in immune-regulatory cancer-associated fibroblasts and in a unique subset of cancer associated pericytes. Our study thus provides an interactome of stress responses in cancer, offering new ways to intersect survival pathways within the tumor, and advance cancer therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560316">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560316" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560316">
        <p class="paperTitle">Selective Mitochondrial Respiratory Complex I Subunit Deficiency Causes Tumor Immunogenicity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560316" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560316" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Liang, J.; Vitale, T.; Zhang, X.; Jackson, T. D.; Yu, D.; Jedrychowsk, M.; Gygi, S. P.; Widlund, H. R.; Wucherpfennig, K.; Puigserver, P.</p>
        <p class="info">Score: 7.3, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560316' target='https://doi.org/10.1101/2023.10.02.560316'> 10.1101/2023.10.02.560316</a></p>
        <p class="abstract">Targeting of specific metabolic pathways in tumor cells has the potential to sensitize them to immune-mediated attack. Here we provide evidence for a specific means of mitochondrial respiratory Complex I (CI) inhibition that improves tumor immunogenicity and sensitivity to immune checkpoint blockade (ICB). Targeted genetic deletion of the CI subunits Ndufs4 and Ndufs6, but not other subunits, induces an immune-dependent tumor growth attenuation in mouse melanoma models. We show that deletion of Ndufs4 induces expression of the transcription factor Nlrc5 and genes in the MHC class I antigen presentation and processing pathway. This induction of MHC-related genes is driven by an accumulation of pyruvate dehydrogenase-dependent mitochondrial acetyl-CoA downstream of CI subunit deletion. This work provides a novel functional modality by which selective CI inhibition restricts tumor growth, suggesting that specific targeting of Ndufs4, or related CI subunits, increases T-cell mediated immunity and sensitivity to ICB.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.26.559519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.26.559519" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.26.559519">
        <p class="paperTitle">Expression of the CIC-DUX4 fusion oncoprotein mimics human CIC-rearranged sarcoma in genetically engineered mouse models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.26.559519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.26.559519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hendrickson, P. G.; Oristian, K. M.; Browne, M. R.; Lou, L.; Ma, Y.; Cardona, D. M.; Linardic, C. M.; Kirsch, D. G.</p>
        <p class="info">Score: 7.0, Published: 2023-09-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.26.559519' target='https://doi.org/10.1101/2023.09.26.559519'> 10.1101/2023.09.26.559519</a></p>
        <p class="abstract">CIC-DUX4 sarcoma (CDS) is a rare but highly aggressive undifferentiated small round cell sarcoma driven by a fusion between the tumor suppressor Capicua (CIC) and DUX4. Currently, there are no effective treatments and efforts to identify and translate better therapies are limited by the scarcity of tissues and patients. To address this limitation, we generated three genetically engineered mouse models of CDS (Ch7CDS, Ai9CDS, and TOPCDS). Remarkably, chimeric mice from all three conditional models developed spontaneous tumors and widespread metastasis in the absence of Cre-recombinase. The penetrance of spontaneous (Cre-independent) tumor formation was complete irrespective of bi-allelic CIC function and loxP site proximity. Characterization of primary and metastatic mouse tumors showed that they consistently expressed the CIC-DUX4 fusion protein as well as other downstream markers of the disease credentialing these models as CDS. In addition, tumor-derived cell lines were generated and ChIP-seq was preformed to map fusion-gene specific binding using an N-terminal HA epitope tag. These datasets, along with paired H3K27ac ChIP-seq maps, validate CIC-DUX4 as a neomorphic transcriptional activator. Moreover, they are consistent with a model where ETS family transcription factors are cooperative and redundant drivers of the core regulatory circuitry in CDS.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.27.559763">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.27.559763" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.27.559763">
        <p class="paperTitle">Clonal fitness decline in somatic differentiation hierarchies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.27.559763" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.27.559763" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ahmed, I.; Dingli, D.; Huang, W.; Werner, B.</p>
        <p class="info">Score: 6.9, Published: 2023-09-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.27.559763' target='https://doi.org/10.1101/2023.09.27.559763'> 10.1101/2023.09.27.559763</a></p>
        <p class="abstract">The concept of clonal fitness is fundamental to describe the evolutionary dynamics in somatic tissues. It is now well established that otherwise healthy somatic tissues become increasingly populated by expanding clones with age. However, the dynamic properties and respective fitnesses of these clones are less well understood. Here we show, that in somatic tissues organised as a differentiation hierarchy, theory predicts a natural decline of effective clonal fitness over time in the absence of additional driver events. This decline is intrinsic to the tissue organisation and can be captured quantitatively by a simple heuristic equation that is proportional to 1/time. We also show that the expected fitness decline is directly observable in human haematopoiesis. The predicted short and long term dynamics agree with in vivo observations using data of Neutrophil recovery after bone marrow transplants and naturally progressing Chronic Lymphocyte Leukemia (CLL). We further show that theory predicts the existence of a long term equilibrium fitness. All CLL patients transition into a stable equilibrium fitness eventually. We find significant inter-patient variation of long term fitness and a strong correlation with disease aggressiveness. Interestingly, CLL long term fitness can be forecast based on the early stages of disease progression, suggesting a Big Bang like model for CLL evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.24.559218">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.24.559218" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.24.559218">
        <p class="paperTitle">The target gene regulatory network of miR-497 in angiosarcoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.24.559218" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.24.559218" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benton, A.; Terwilliger, E.; Moriarty, N. M.; Liu, B.; Murphy, A.; Maluvac, H.; Shu, M.; Gartenhaus, L. E.; Janson, N. D.; Pfeffer, C. M.; Utturkar, S. M.; Parkinson, E. I.; Lanman, N. A.; Hanna, J. A.</p>
        <p class="info">Score: 7.2, Published: 2023-09-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.24.559218' target='https://doi.org/10.1101/2023.09.24.559218'> 10.1101/2023.09.24.559218</a></p>
        <p class="abstract">Angiosarcoma (AS) is a vascular sarcoma that is highly aggressive and metastatic. Due to its rarity, treatment options for patients are limited, therefore more research is needed to identify possible therapeutic vulnerabilities. We previously found that conditional deletion of Dicer1 drives AS development in mice. Given the role of DICER1 in canonical microRNA (miRNA) biogenesis, this suggests that miRNA loss is important in AS development. After testing miRNAs previously suggested to have a tumor-suppressive role in AS, microRNA-497-5p (miR-497) suppressed cell viability most significantly. We also found that miR-497 overexpression led to significantly reduced cell migration and tumor formation. To understand the mechanism of miR-497 in tumor suppression, we identified clinically relevant target genes using a combination of RNA-sequencing data in an AS cell line, expression data from AS patients, and target prediction algorithms. We validated miR-497 direct regulation of CCND2, CDK6, and VAT1. One of these genes, VAT1, is an understudied protein that has been suggested to promote cell migration and metastasis in other cancers. Indeed, we find that pharmacologic inhibition of VAT1 with the natural product Neocarzilin A reduces AS migration. This work provides insight into the mechanisms of miR-497 and its target genes in AS pathogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558231">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558231" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558231">
        <p class="paperTitle">Evolutionary fingerprints of EMT in pancreatic cancers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558231" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558231" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perelli, L.; Zhang, L.; Mangiameli, S.; Russell, A. J. C.; Giannese, F.; Peng, F.; Carbone, F.; Le, C.; Khan, H.; Citron, F.; Soeung, M.; Lam, T.; Lundgren, S.; Zhu, C.; Catania, D.; Feng, N.; Gurreri, E.; Sgambato, A.; Tortora, G.; Draetta, G.; Tonon, G.; Futreal, A.; Giuliani, V.; Carugo, A.; Viale, A.; Heffernan, T.; Wang, L.; Cittaro, D.; Chen, F.; Genovese, G.</p>
        <p class="info">Score: 11.3, Published: 2023-09-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558231' target='https://doi.org/10.1101/2023.09.18.558231'> 10.1101/2023.09.18.558231</a></p>
        <p class="abstract">Mesenchymal plasticity has been extensively described in advanced and metastatic epithelial cancers; however, its functional role in malignant progression, metastatic dissemination and therapy response is controversial. More importantly, the role of epithelial mesenchymal transition (EMT) and cell plasticity in tumor heterogeneity, clonal selection and clonal evolution is poorly understood. Functionally, our work clarifies the contribution of EMT to malignant progression and metastasis in pancreatic cancer. We leveraged ad hoc somatic mosaic genome engineering, lineage tracing and ablation technologies and dynamic genetic reporters to trace and ablate tumor-specific lineages along the phenotypic spectrum of epithelial to mesenchymal plasticity. The experimental evidences clarify the essential contribution of mesenchymal lineages to pancreatic cancer evolution and metastatic dissemination. Spatial genomic analysis combined with single cell transcriptomic and epigenomic profiling of epithelial and mesenchymal lineages reveals that EMT promotes with the emergence of chromosomal instability (CIN). Specifically tumor lineages with mesenchymal features display highly conserved patterns of genomic evolution including complex structural genomic rearrangements and chromotriptic events. Genetic ablation of mesenchymal lineages robustly abolished these mutational processes and evolutionary patterns, as confirmed by cross species analysis of pancreatic and other human epithelial cancers. Mechanistically, we discovered that malignant cells with mesenchymal features display increased chromatin accessibility, particularly in the pericentromeric and centromeric regions, which in turn results in delayed mitosis and catastrophic cell division. Therefore, EMT favors the emergence of high-fitness tumor cells, strongly supporting the concept of a cell-state, lineage-restricted patterns of evolution, where cancer cell sub-clonal speciation is propagated to progenies only through restricted functional compartments. Restraining those evolutionary routes through genetic ablation of clones capable of mesenchymal plasticity and extinction of the derived lineages completely abrogates the malignant potential of one of the most aggressive form of human cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557228">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557228" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557228">
        <p class="paperTitle">Nuclear speckles regulate HIF-2α programs and correlate with patient survival in kidney cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557228" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557228" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Alexander, K. A.; Yu, R.; Skuli, N.; Coffey, N. J.; Nguyen, S.; Faunce, C.; Huang, H.; Dardani, I.; Good, A. L.; Lim, J.; Li, C.; Biddle, N.; Joyce, E. F.; Raj, A.; Lee, D.; Keith, B.; Simon, M. C.; Berger, S. L.</p>
        <p class="info">Score: 5.5, Published: 2023-09-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557228' target='https://doi.org/10.1101/2023.09.14.557228'> 10.1101/2023.09.14.557228</a></p>
        <p class="abstract">Nuclear speckles are membrane-less bodies within the cell nucleus enriched in RNA biogenesis, processing, and export factors. In this study we investigated speckle phenotype variation in human cancer, finding a reproducible speckle signature, based on RNA expression of speckle-resident proteins, across &gt;20 cancer types. Of these, clear cell renal cell carcinoma (ccRCC) exhibited a clear correlation between the presence of this speckle expression signature, imaging-based speckle phenotype, and clinical outcomes. ccRCC is typified by hyperactivation of the HIF-2 transcription factor, and we demonstrate here that HIF-2 drives physical association of a select subset of its target genes with nuclear speckles. Disruption of HIF-2-driven speckle association via deletion of its speckle targeting motifs (STMs)--defined in this study--led to defective induction of speckle-associating HIF-2 target genes without impacting non-speckle-associating HIF-2 target genes. We further identify the RNA export complex, TREX, as being specifically altered in speckle signature, and knockdown of key TREX component, ALYREF, also compromises speckle-associated gene expression. By integrating tissue culture functional studies with tumor genomic and imaging analysis, we show that HIF-2 gene regulatory programs are impacted by specific manipulation of speckle phenotype and by abrogation of speckle targeting abilities of HIF-2. These findings suggest that, in ccRCC, a key biological function of nuclear speckles is to modulate expression of a specific subset of HIF-2-regulated target genes that, in turn, influence patient outcomes. We also identify STMs in other transcription factors, suggesting that DNA-speckle targeting may be a general mechanism of gene regulation.

HIGHLIGHTS- Nuclear speckles shown to reproducibly vary in cancer, predicting patient survival in ccRCC
- HIF-2 drives DNA/gene-speckle contacts dependent on identified speckle targeting motifs within HIF-2
- Putative speckle targeting motifs are highly enriched among regulators of gene expression
- Partitioning of transcription factor functional programs may be a major biological function of nuclear speckles</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.02.560330">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.02.560330" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.02.560330">
        <p class="paperTitle">Targeting Acute Myeloid Leukemia Stem Cells Through Perturbation of Mitochondrial Calcium</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.02.560330" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.02.560330" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sheth, A. I.; Engel, K.; Tolison, H.; Althoff, M. J.; Amaya, M. L.; Krug, A.; Young, T.; Pei, S.; Patel, S. B.; Minhajuddin, M.; Winters, A.; Miller, R.; Shelton, I.; St-Germain, J.; Ling, T.; Jones, C.; Raught, B.; Gillen, A. E.; Ransom, M. R.; Staggs, S.; Smith, C.; Pollyea, D. A.; Stevens, B. M.; Jordan, C. T.</p>
        <p class="info">Score: 4.8, Published: 2023-10-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.02.560330' target='https://doi.org/10.1101/2023.10.02.560330'> 10.1101/2023.10.02.560330</a></p>
        <p class="abstract">We previously reported that acute myeloid leukemia stem cells (LSCs) are uniquely reliant on oxidative phosphorylation (OXPHOS) for survival. Moreover, maintenance of OXPHOS is dependent on BCL2, creating a therapeutic opportunity to target LSCs using the BCL2 inhibitor drug venetoclax. While venetoclax-based regimens have indeed shown promising clinical activity, the emergence of drug resistance is prevalent. Thus, in the present study, we investigated how mitochondrial properties may influence mechanisms that dictate venetoclax responsiveness. Our data show that utilization of mitochondrial calcium is fundamentally different between drug responsive and non-responsive LSCs. By comparison, venetoclax-resistant LSCs demonstrate a more active metabolic (i.e., OXPHOS) status with relatively high steady-state levels of calcium. Consequently, we tested genetic and pharmacological approaches to target the mitochondrial calcium uniporter, MCU. We demonstrate that inhibition of calcium uptake sharply reduces OXPHOS and leads to eradication of venetoclax-resistant LSCs. These findings demonstrate a central role for calcium signaling in the biology of LSCs and provide a therapeutic avenue for clinical management of venetoclax resistance.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
