<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Systematic benchmarking of imaging spatial transcriptomics platforms in FFPE tissues
Authors: Wang, H.; Huang, R.; Nelson, J.; Gao, C.; Tran, M.; Yeaton, A.; Felt, K.; Pfaff, K. L.; Bowman, T.; Rodig, S. J.; Wei, K.; Goods, B. A.; Farhi, S. L.
Score: 569.4, Published: 2023-12-19 DOI: 10.1101/2023.12.07.570603
Emerging imaging spatial transcriptomics (iST) platforms and coupled analytical methods can recover cell-to-cell interactions, groups of spatially covarying genes, and gene signatures associated with pathological features, and are thus particularly well-suited for applications in formalin fixed paraffin embedded (FFPE) tissues.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Systematic benchmarking of imaging spatial transcriptomics platforms in FFPE tissues
Authors: Wang, H.; Huang, R.; Nelson, J.; Gao, C.; Tran, M.; Yeaton, A.; Felt, K.; Pfaff, K. L.; Bowman, T.; Rodig, S. J.; Wei, K.; Goods, B. A.; Farhi, S. L.
Score: 569.4, Published: 2023-12-19 DOI: 10.1101/2023.12.07.570603
Emerging imaging spatial transcriptomics (iST) platforms and coupled analytical methods can recover cell-to-cell interactions, groups of spatially covarying genes, and gene signatures associated with pathological features, and are thus particularly well-suited for applications in formalin fixed paraffin embedded (FFPE) tissues." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-20T10:36:27+00:00" />
<meta property="article:modified_time" content="2023-12-20T10:36:27+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Systematic benchmarking of imaging spatial transcriptomics platforms in FFPE tissues
Authors: Wang, H.; Huang, R.; Nelson, J.; Gao, C.; Tran, M.; Yeaton, A.; Felt, K.; Pfaff, K. L.; Bowman, T.; Rodig, S. J.; Wei, K.; Goods, B. A.; Farhi, S. L.
Score: 569.4, Published: 2023-12-19 DOI: 10.1101/2023.12.07.570603
Emerging imaging spatial transcriptomics (iST) platforms and coupled analytical methods can recover cell-to-cell interactions, groups of spatially covarying genes, and gene signatures associated with pathological features, and are thus particularly well-suited for applications in formalin fixed paraffin embedded (FFPE) tissues."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Systematic benchmarking of imaging spatial transcriptomics platforms in FFPE tissues\nAuthors: Wang, H.; Huang, R.; Nelson, J.; Gao, C.; Tran, M.; Yeaton, A.; Felt, K.; Pfaff, K. L.; Bowman, T.; Rodig, S. J.; Wei, K.; Goods, B. A.; Farhi, S. L.\nScore: 569.4, Published: 2023-12-19 DOI: 10.1101/2023.12.07.570603\nEmerging imaging spatial transcriptomics (iST) platforms and coupled analytical methods can recover cell-to-cell interactions, groups of spatially covarying genes, and gene signatures associated with pathological features, and are thus particularly well-suited for applications in formalin fixed paraffin embedded (FFPE) tissues.",
  "keywords": [
    
  ],
  "articleBody": " Systematic benchmarking of imaging spatial transcriptomics platforms in FFPE tissues\nAuthors: Wang, H.; Huang, R.; Nelson, J.; Gao, C.; Tran, M.; Yeaton, A.; Felt, K.; Pfaff, K. L.; Bowman, T.; Rodig, S. J.; Wei, K.; Goods, B. A.; Farhi, S. L.\nScore: 569.4, Published: 2023-12-19 DOI: 10.1101/2023.12.07.570603\nEmerging imaging spatial transcriptomics (iST) platforms and coupled analytical methods can recover cell-to-cell interactions, groups of spatially covarying genes, and gene signatures associated with pathological features, and are thus particularly well-suited for applications in formalin fixed paraffin embedded (FFPE) tissues. Here, we benchmarked the performance of three commercial iST platforms on serial sections from tissue microarrays (TMAs) containing 23 tumor and normal tissue types for both relative technical and biological performance. On matched genes, we found that 10X Xenium shows higher transcript counts per gene without sacrificing specificity, but that all three platforms concord to orthogonal RNA-seq datasets and can perform spatially resolved cell typing, albeit with different false discovery rates, cell segmentation error frequencies, and with varying degrees of sub-clustering for downstream biological analyses. Taken together, our analyses provide a comprehensive benchmark to guide the choice of iST method as researchers design studies with precious samples in this rapidly evolving field.\nROR2 regulates cellular plasticity in pancreatic neoplasia and adenocarcinoma\nAuthors: Benitz, S.; Steep, A.; Nasser, M.; Preall, J.; Mahajan, U. M.; McQuithey, H.; Loveless, I.; Davis, E. T.; Wen, H.-J.; Long, D. W.; Metzler, T.; Zwernik, S.; Louw, M.; Rempinski, D.; Salas-Escabillas, D.; Brender, S.; Song, L.; Huang, L.; Zhang, Z.; Steele, N.; Regel, I.; Bednar, F.; Crawford, H. C.\nScore: 21.6, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571566\nCellular plasticity is a hallmark of pancreatic ductal adenocarcinoma (PDAC) starting from the conversion of normal cells to precancerous lesions to the progression of carcinoma subtypes associated with aggressiveness and therapeutic response. We found that normal acinar cell differentiation, maintained by the transcription factor Pdx1, suppresses a broad gastric cell identity that is maintained in metaplasia, neoplasia, and the classical subtype of PDAC. We have identified the receptor tyrosine kinase Ror2 as marker of a gastric metaplasia (SPEM)-like identity. Ablation of Ror2 in a mouse model of pancreatic tumorigenesis promoted a switch to a gastric pit cell identity that was largely persistent upon progression to carcinoma. In both human and mouse pancreatic cancer, ROR2 activity continued to antagonize the gastric pit cell identity, strongly promoting an epithelial mesenchymal transition, conferring resistance to KRAS inhibition, and vulnerability to AKT inhibition.\nChromothripsis orchestrates leukemic transformation in blast phase MPN through targetable amplification of DYRK1A\nAuthors: Brierley, C. K.; Yip, B. H.; Orlando, G.; Goyal, H.; Wen, S.; Wen, J.; Levine, M. F.; Jakobsdottir, M. G.; Rodriguez-Meira, A.; Adamo, A.; Bashton, M.; Hamblin, A.; Clark, S. A.; O'Sullivan, J.; Murphi, L.; Olijnik-French, A.-A.; Cotton, A.; Narina, S.; Pruett-Miller, S.; Enshaei, A.; Harrison, C.; Drummond, M.; Knapper, S.; Tefferi, A.; Antony-Debre, I.; Thongjuea, S.; Wedge, D. C.; Constantinescu, S.; Papaemmanuil, E.; Psaila, B.; Crispino, J. D.; Mead, A. J.\nScore: 25.3, Published: 2023-12-10 DOI: 10.1101/2023.12.08.570880\nChromothripsis, the process of catastrophic shattering and haphazard repair of chromosomes, is a common event in cancer. Whether chromothripsis might constitute an actionable molecular event amenable to therapeutic targeting remains an open question. We describe recurrent chromothripsis of chromosome 21 in a subset of patients in blast phase of a myeloproliferative neoplasm (BP-MPN), which alongside other structural variants leads to amplification of a region of chromosome 21 in [~]25% of patients ( chr21amp). We report that chr21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. The chr21amp event is highly clonal and present throughout the hematopoietic hierarchy. DYRK1A, a serine threonine kinase and transcription factor, is the only gene in the 2.7Mb minimally amplified region which showed both increased expression and chromatin accessibility compared to non-chr21amp BP-MPN controls. We demonstrate that DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development, including DNA repair, STAT signalling and BCL2 overexpression. DYRK1A is essential for BP-MPN cell proliferation in vitro and in vivo, and DYRK1A inhibition synergises with BCL2 targeting to induce BP-MPN cell apoptosis. Collectively, these findings define the chr21amp event as a prognostic biomarker in BP-MPN and link chromothripsis to a druggable target.\nPreleukemic clonal evolution revealed using single-cell DNA sequencing and computational modelling\nAuthors: Blundell, J. R.; Poon, G. Y. P.; Vedi, A.; Laurenti, E.; Sanders, M. A.; Valk, P.\nScore: 11.5, Published: 2023-12-16 DOI: 10.1101/2023.12.15.571872\nThe acquisition of cancer driver mutations can predate diagnosis by decades. However, our understanding of the evolution that occurs in these early stages of cancer remains limited because most studies focus only on sequencing the malignant clone that \"wins\" the evolutionary race. The representation of driver mutations in the stem cells which do not transform into the malignant clone provides a window into this precancerous evolution. Here, we isolate preleukemic haematopoietic stem cells (pHSCs) from the bone marrow of 16 patients with Acute Myeloid Leukaemia (AML) and perform single-cell DNA sequencing on thousands of pHSCs to reconstruct phylogenetic trees for the preleukemic evolution in each patient. In the majority of cases AML required the acquisition of 3 or 4 driver mutations in the same cell lineage. However the reconstructed phylogenetic trees are highly variable across the 16 patients. The structure of the trees and the sizes of intermediate preleukemic clones encode important information about the roles of selection, mutation and genetic drift operating during precancerous evolution. Using simulations and machine learning approaches, we show that the highly variable patterns observed in pHSC trees emerge naturally from a model of somatic evolution in which there is strong positive selection acting throughout the entire multi-hit trajectory towards AML. We infer that the acquisition of each driver mutation provides an additional selective advantage of [~] 15% per year to pHSC clones relative to their immediate ancestors. Using these inferred levels of positive selection we show that individuals destined to develop AML late in life are likely to harbour multiple-mutant clones that have arisen anomalously early in life. Our findings suggest that the identification of a single HSC carrying 2 driver mutations in early adulthood may provide a rational strategy to identify individuals at high risk of AML progression.\nEnhanced Complement Expression in the Tumor Microenvironment Following Neoadjuvant Therapy: Implications for Immunomodulation and Survival in Pancreatic Ductal Adenocarcinoma\nAuthors: Zhang, X.; Liu, Y.-Z.; Lan, R.; Liu, Y.; Pillarisetty, V. G.; Li, D.; Zhao, C. L.; Sarkar, S.; Liu, W.; Hanna, I.; Gupta, M.; Hajdu, C.; Melamed, J.; Widmer, J.; Allendorf, J.\nScore: 8.8, Published: 2023-12-15 DOI: 10.1101/2023.10.26.564099\nPurposeNeoadjuvant therapy (NAT) is increasingly being used for pancreatic ductal adenocarcinoma (PDAC) treatment. However, its distinct effects on carcinoma cells and the tumor microenvironment (TME) are not fully understood. This study employs spatial transcriptomics and single-cell RNA sequencing to investigate how NAT differentially remodels PDACs carcinoma cells and TME. Experimental DesignWe used spatial transcriptomics to compare gene expression profiles in carcinoma cells and the TME between NAT-treated and NAT-naive PDAC patients and correlated with their clinicopathologic features. Complementary single-nucleus RNA sequencing (snRNA-seq) analysis was conducted to validate our findings and identify cell types driving NAT-induced gene expression alterations. ResultsWe found NAT not only induces apoptosis and inhibits proliferation in carcinoma cells but also significantly remodels the TME. Notably, NAT induces a coordinated upregulation of multiple key complement genes (C3, C1S, C1R, C4B and C7) in the TME, making the complement pathway one of the most significantly affected pathways by NAT. Patients with higher TME complement expression following NAT exhibit improved overall survival; more immunomodulatory and neurotrophic cancer-associated fibroblasts (CAFs); more CD4+ T cells, monocytes and mast cells; and lower immune exhaustion gene expression. snRNA-seq analysis demonstrates C3 complement upregulation specifically in CAFs but not in other stroma cell types. ConclusionsOur findings indicate that NAT may reduce immunosuppression in PDAC by enhancing complement production and signaling within the TME. These findings suggest that local complement dynamics could serve as a novel biomarker for prognosis, evaluating treatment response and resistance, and guiding therapeutic strategies in NAT-treated PDAC patients. Translational RelevanceAs neoadjuvant therapy (NAT) increasingly becomes the preferred approach in treating resectable and borderline resectable pancreatic ductal adenocarcinoma (PDAC), there is a growing demand for novel biomarkers specifically tailored for post-NAT PDAC patients. Our study focused on how NAT differentially remodels the tumor cells and the tumor microenvironment (TME). We demonstrate that NAT can enhance local complement production and signaling in PDACs TME, which is associated with reduced immune exhaustion and improved overall survival. Our results highlight the importance of local complement dynamics in influencing treatment response, resistance, and overall clinical outcomes in PDAC. This new mechanism provides new opportunities in biomarker development which could facilitate more accurate prognostication and precise treatment stratification, particularly for patients undergoing NAT in PDAC.\nDistinct host preconditioning regimens differentially impact the antitumor potency of adoptively transferred Th17 cells.\nAuthors: Wittling, M. C.; Knochelmann, H. M.; Wyatt, M. M.; Rangel Rivera, G. O.; Cole, A. C.; Lesinski, G. B.; Paulos, C. M.\nScore: 7.5, Published: 2023-12-18 DOI: 10.1101/2023.12.18.572179\nBackground: Mechanisms by which distinct methods of host preconditioning impact the efficacy of adoptively transferred antitumor T helper cells is unknown. Methods: CD4+ T cells with a transgenic TCR that recognize TRP-1 melanoma antigen were polarized to the T helper 17 (Th17) phenotype and then transferred into melanoma-bearing mice preconditioned with either total body irradiation or chemotherapy. Results: We found that preconditioning mice with a non-myeloablative dose of total body irradiation (TBI of 5 Gy) was more effective than using an equivalently dosed non-myeloablative chemotherapy (CTX at 200 mg/kg) at augmenting therapeutic activity of anti-tumor TRP-1 Th17 cells. Anti-tumor Th17 cells engrafted better following preconditioning with TBI and regressed large established melanoma in all animals. Conversely, only half of mice survived long-term when preconditioned with CTX and infused with anti-melanoma Th17 cells. IL-17 and IFN-{gamma} produced by the infused Th17 cells, were detected in animals given either TBI or CTX preconditioning. Interestingly, inflammatory cytokines (G-CSF, IL-6, MCP-1, IL-5, and KC) were significantly elevated in the serum of mice preconditioned with TBI versus CTX after Th17 therapy. Conclusions: Our results indicate, for the first time, that the antitumor response, persistence, and cytokine profiles resulting from Th17 therapy are impacted by the specific regimen of host preconditioning. This work is important for understanding mechanisms that promote long-lived responses by ACT, particularly as CD4+ based T cell therapies are now emerging in the clinic.\nSingle-nucleotide-resolution genomic maps of O6-methylguanine from the glioblastoma drug temozolomide\nAuthors: Buechel, J.; Mingard, C.; Takhaveev, V.; Reinert, P. B.; Keller, G.; Kloter, T.; Huber, S.; McKeague, M.; Sturla, S. J.\nScore: 6.7, Published: 2023-12-13 DOI: 10.1101/2023.12.12.571283\nTemozolomide kills cancer cells by forming O6-methylguanine (O6-MeG), which leads to apoptosis by mismatch-repair overload, as an important basis for glioblastoma therapy. However, O6-MeG repair by O6-methylguanine-DNA methyltransferase (MGMT) strongly contributes to drug resistance. Genomic profiles of O6-MeG may be useful for understanding how O6-MeG accumulation can be influenced by repair mechanisms, but there are no methods to map genomic locations of O6-MeG. Here, we devised an immunoprecipitation- and polymerase-stalling-based strategy to locate O6-MeG across the whole genome at single-nucleotide resolution. We applied this strategy, termed O6-MeG-seq, to analyze O6-MeG formation and repair with regards to sequence contexts and functional genomic regions in glioblastoma-derived cell lines deficient in MGMT and evaluated the impact of MGMT transfection on these profiles. O6-MeG signatures from the sequencing data were highly similar to mutational signatures extracted from cancer genomes of patients previously treated with temozolomide, consistent with O6-MeG initiating mutations. Furthermore, it appears that MGMT does not preferentially repair O6-MeG with respect to sequence context, chromatin state or gene expression level, however, may protect oncogenes from mutations. Finally, an MGMT-independent strand bias in O6-MeG accumulation in highly expressed genes suggests alternative repair influences. These data provide high resolution insight on how O6-MeG formation and repair is impacted by genome structure and regulation. O6-MeG-seq also provides a strategy for future studies of DNA modification signatures as diagnostic markers for addressing drug resistance and preventing secondary cancers.\nMelanoma innervation is associated with cancer progression in a zebrafish xenograft model\nAuthors: LORENZINI, F.; MARINES, J.; Le FRIEC, J.; DO KHOA, N.; NIETO, M. A.; SANCHEZ-LAORDEN, B.; MIONE, M.; FONTENILLE, L.; KISSA, K.\nScore: 5.8, Published: 2023-12-14 DOI: 10.1101/2023.12.13.571512\nThe peripheral nervous system has a key role in regulating tumour biology in different types of cancer. Here, by modelling aggressive melanoma in larval zebrafish xenografts, we highlight the dynamics of tumour innervation in the tumour microenvironment (TME). Axonogenesis and dendritogenesis are detected in the motoneurons surrounding the melanoma niche and neurogenesis is observed in the nearby population of the enteric nervous system. We also demonstrate the crucial role of catecholamines in promoting melanoma progression, supporting in vivo cancer cell dissemination and invasion. This zebrafish model will allow to uncover neural markers associated with melanoma progression to help in the design of innovative anti-neurogenic therapies targeting specifically the neuronal signals that regulate melanoma progression. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=171 SRC=\"FIGDIR/small/571512v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (41K): org.highwire.dtl.DTLVardef@13dd11org.highwire.dtl.DTLVardef@16905corg.highwire.dtl.DTLVardef@15fb42borg.highwire.dtl.DTLVardef@2da605_HPS_FORMAT_FIGEXP M_FIG C_FIG HighlightsO_LITransplantation of human melanoma cells in 3 dpf zebrafish swim bladder allows the development of aggressive melanoma, which cells invade the surrounding organs and migrate over distant locations. C_LIO_LIThe presence of melanoma cells in the larval zebrafish induces morphological changes in the motoneurons inside the tumour niche, including increased axon length and dendritic arborization. C_LIO_LIThe invasion of melanoma cells in the larval intestine promotes neurogenesis of enteric neurons. C_LIO_LITransplanted melanoma cells display direct contact with enteric neurons in the intestinal region and migrate along axons to escape from the primary cancer mass, as a mechanism similar to vessel co-option during metastatic dissemination. C_LIO_LICatecholamines promote melanoma cell migration and invasion in the zebrafish, modelling melanoma progression. C_LI\nA high-throughput screening approach to discover potential colorectal cancer chemotherapeutics: Repurposing drugs to disrupt 14-3-3 protein-BAD interactions\nAuthors: He, S.; Delgadillo Silva, L.; Rutter, G. A.; Lim, G. E.\nScore: 5.2, Published: 2023-12-15 DOI: 10.1101/2023.12.14.571727\nInducing apoptosis in different types of cancer cells is an effective therapeutic strategy. However, the success of existing chemotherapeutics can be compromised by tumor cell resistance and systemic off-target effects. Therefore, the discovery of pro-apoptotic compounds with minimal systemic side-effects is crucial. 14-3-3 proteins are molecular scaffolds that serve as important regulators of cell survival. Our previous study demonstrated that 14-3-3{zeta} can sequester BAD, a pro-apoptotic member of the BCL-2 protein family, in the cytoplasm and prevent its translocation to mitochondria to inhibit the induction of apoptosis. Despite being a critical mechanism of cell survival, it is unclear whether disrupting 14-3-3 protein:BAD interactions could be harnessed as a chemotherapeutic approach. Herein, we established a BRET-based high-throughput drug screening approach (Z-score= 0.52) capable of identifying molecules that can disrupt 14-3-3{zeta}BAD interactions. An FDA-approved drug library containing 1971 compounds was used for screening, and the capacity of identified hits to induce cell death was examined in NIH3T3-fibroblasts and colorectal cancer cell lines, HT-29 and Caco-2. Our in vitro results suggest that terfenadine, penfluridol, and lomitapide could be potentially repurposed for treating colorectal cancer. Moreover, our screening method demonstrates the feasibility of identifying pro-apoptotic agents that can be applied towards conditions where aberrant cell growth or function are key determinants of disease pathogenesis.\nModelling glioblastoma resistance to temozolomide. Combination of spheroid and mathematical models to simulate cellular adaptation in vitro.\nAuthors: Perez-Aliacar, M.; Ayensa-Jimenez, J.; Randelovic, T.; Ochoa, I.; Doblare, M.\nScore: 7.8, Published: 2023-11-24 DOI: 10.1101/2023.11.24.568421\nAO_SCPLOWBSTRACTC_SCPLOWDrug resistance is one of the biggest challenges in the fight against cancer. In particular, in the case of glioblastoma, the most lethal brain tumour, resistance to temozolomide (the standard of care drug for chemotherapy in this tumour), is one of the main reasons behind treatment failure and hence responsible for the poor prognosis of patients diagnosed with this disease. In this paper, we combine the power of three-dimensional in vitro experiments of treated glioblastoma spheroids with mathematical models of tumour evolution and adaptation. We use a novel approach based on internal variables for modelling the acquisition of resistance to temozolomide that is observed in a group of treated spheroids in the experiments. These internal variables describe the cells phenotypic state, which depends on the history of drug exposure and affects cell behaviour. We use model selection to determine the most parsimonious model and calibrate it to reproduce the experimental data, obtaining a high level of agreement between the in vitro and in silico outcomes. A sensitivity analysis is carried out to investigate the impact of each model parameter in the predictions. More importantly, we show the utility of our model for answering biological questions, such as what is the intrinsic adaptation mechanism, or for separating the sensitive and resistant populations. We conclude that the proposed in silico framework, in combination with experiments, can be useful to improve our understanding of the mechanisms behind drug resistance in glioblastoma and to eventually set some guidelines for the design of new treatment schemes.\n",
  "wordCount" : "2807",
  "inLanguage": "en",
  "datePublished": "2023-12-20T10:36:27Z",
  "dateModified": "2023-12-20T10:36:27Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 20, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.07.570603">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.07.570603" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.07.570603">
        <p class="paperTitle">Systematic benchmarking of imaging spatial transcriptomics platforms in FFPE tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.07.570603" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.07.570603" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, H.; Huang, R.; Nelson, J.; Gao, C.; Tran, M.; Yeaton, A.; Felt, K.; Pfaff, K. L.; Bowman, T.; Rodig, S. J.; Wei, K.; Goods, B. A.; Farhi, S. L.</p>
        <p class="info">Score: 569.4, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.07.570603' target='https://doi.org/10.1101/2023.12.07.570603'> 10.1101/2023.12.07.570603</a></p>
        <p class="abstract">Emerging imaging spatial transcriptomics (iST) platforms and coupled analytical methods can recover cell-to-cell interactions, groups of spatially covarying genes, and gene signatures associated with pathological features, and are thus particularly well-suited for applications in formalin fixed paraffin embedded (FFPE) tissues. Here, we benchmarked the performance of three commercial iST platforms on serial sections from tissue microarrays (TMAs) containing 23 tumor and normal tissue types for both relative technical and biological performance. On matched genes, we found that 10X Xenium shows higher transcript counts per gene without sacrificing specificity, but that all three platforms concord to orthogonal RNA-seq datasets and can perform spatially resolved cell typing, albeit with different false discovery rates, cell segmentation error frequencies, and with varying degrees of sub-clustering for downstream biological analyses. Taken together, our analyses provide a comprehensive benchmark to guide the choice of iST method as researchers design studies with precious samples in this rapidly evolving field.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571566">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571566" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571566">
        <p class="paperTitle">ROR2 regulates cellular plasticity in pancreatic neoplasia and adenocarcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571566" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571566" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Benitz, S.; Steep, A.; Nasser, M.; Preall, J.; Mahajan, U. M.; McQuithey, H.; Loveless, I.; Davis, E. T.; Wen, H.-J.; Long, D. W.; Metzler, T.; Zwernik, S.; Louw, M.; Rempinski, D.; Salas-Escabillas, D.; Brender, S.; Song, L.; Huang, L.; Zhang, Z.; Steele, N.; Regel, I.; Bednar, F.; Crawford, H. C.</p>
        <p class="info">Score: 21.6, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571566' target='https://doi.org/10.1101/2023.12.13.571566'> 10.1101/2023.12.13.571566</a></p>
        <p class="abstract">Cellular plasticity is a hallmark of pancreatic ductal adenocarcinoma (PDAC) starting from the conversion of normal cells to precancerous lesions to the progression of carcinoma subtypes associated with aggressiveness and therapeutic response. We found that normal acinar cell differentiation, maintained by the transcription factor Pdx1, suppresses a broad gastric cell identity that is maintained in metaplasia, neoplasia, and the classical subtype of PDAC. We have identified the receptor tyrosine kinase Ror2 as marker of a gastric metaplasia (SPEM)-like identity. Ablation of Ror2 in a mouse model of pancreatic tumorigenesis promoted a switch to a gastric pit cell identity that was largely persistent upon progression to carcinoma. In both human and mouse pancreatic cancer, ROR2 activity continued to antagonize the gastric pit cell identity, strongly promoting an epithelial mesenchymal transition, conferring resistance to KRAS inhibition, and vulnerability to AKT inhibition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.08.570880">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.08.570880" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.08.570880">
        <p class="paperTitle">Chromothripsis orchestrates leukemic transformation in blast phase MPN through targetable amplification of DYRK1A</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.08.570880" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.08.570880" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Brierley, C. K.; Yip, B. H.; Orlando, G.; Goyal, H.; Wen, S.; Wen, J.; Levine, M. F.; Jakobsdottir, M. G.; Rodriguez-Meira, A.; Adamo, A.; Bashton, M.; Hamblin, A.; Clark, S. A.; O&#39;Sullivan, J.; Murphi, L.; Olijnik-French, A.-A.; Cotton, A.; Narina, S.; Pruett-Miller, S.; Enshaei, A.; Harrison, C.; Drummond, M.; Knapper, S.; Tefferi, A.; Antony-Debre, I.; Thongjuea, S.; Wedge, D. C.; Constantinescu, S.; Papaemmanuil, E.; Psaila, B.; Crispino, J. D.; Mead, A. J.</p>
        <p class="info">Score: 25.3, Published: 2023-12-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.08.570880' target='https://doi.org/10.1101/2023.12.08.570880'> 10.1101/2023.12.08.570880</a></p>
        <p class="abstract">Chromothripsis, the process of catastrophic shattering and haphazard repair of chromosomes, is a common event in cancer. Whether chromothripsis might constitute an actionable molecular event amenable to therapeutic targeting remains an open question. We describe recurrent chromothripsis of chromosome 21 in a subset of patients in blast phase of a myeloproliferative neoplasm (BP-MPN), which alongside other structural variants leads to amplification of a region of chromosome 21 in [~]25% of patients ( chr21amp). We report that chr21amp BP-MPN has a particularly aggressive and treatment-resistant phenotype. The chr21amp event is highly clonal and present throughout the hematopoietic hierarchy. DYRK1A, a serine threonine kinase and transcription factor, is the only gene in the 2.7Mb minimally amplified region which showed both increased expression and chromatin accessibility compared to non-chr21amp BP-MPN controls. We demonstrate that DYRK1A is a central node at the nexus of multiple cellular functions critical for BP-MPN development, including DNA repair, STAT signalling and BCL2 overexpression. DYRK1A is essential for BP-MPN cell proliferation in vitro and in vivo, and DYRK1A inhibition synergises with BCL2 targeting to induce BP-MPN cell apoptosis. Collectively, these findings define the chr21amp event as a prognostic biomarker in BP-MPN and link chromothripsis to a druggable target.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.15.571872">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.15.571872" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.15.571872">
        <p class="paperTitle">Preleukemic clonal evolution revealed using single-cell DNA sequencing and computational modelling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.15.571872" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.15.571872" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Blundell, J. R.; Poon, G. Y. P.; Vedi, A.; Laurenti, E.; Sanders, M. A.; Valk, P.</p>
        <p class="info">Score: 11.5, Published: 2023-12-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.15.571872' target='https://doi.org/10.1101/2023.12.15.571872'> 10.1101/2023.12.15.571872</a></p>
        <p class="abstract">The acquisition of cancer driver mutations can predate diagnosis by decades. However, our understanding of the evolution that occurs in these early stages of cancer remains limited because most studies focus only on sequencing the malignant clone that &#34;wins&#34; the evolutionary race. The representation of driver mutations in the stem cells which do not transform into the malignant clone provides a window into this precancerous evolution. Here, we isolate preleukemic haematopoietic stem cells (pHSCs) from the bone marrow of 16 patients with Acute Myeloid Leukaemia (AML) and perform single-cell DNA sequencing on thousands of pHSCs to reconstruct phylogenetic trees for the preleukemic evolution in each patient. In the majority of cases AML required the acquisition of 3 or 4 driver mutations in the same cell lineage. However the reconstructed phylogenetic trees are highly variable across the 16 patients. The structure of the trees and the sizes of intermediate preleukemic clones encode important information about the roles of selection, mutation and genetic drift operating during precancerous evolution. Using simulations and machine learning approaches, we show that the highly variable patterns observed in pHSC trees emerge naturally from a model of somatic evolution in which there is strong positive selection acting throughout the entire multi-hit trajectory towards AML. We infer that the acquisition of each driver mutation provides an additional selective advantage of [~] 15% per year to pHSC clones relative to their immediate ancestors. Using these inferred levels of positive selection we show that individuals destined to develop AML late in life are likely to harbour multiple-mutant clones that have arisen anomalously early in life. Our findings suggest that the identification of a single HSC carrying 2 driver mutations in early adulthood may provide a rational strategy to identify individuals at high risk of AML progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.564099">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.564099" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.564099">
        <p class="paperTitle">Enhanced Complement Expression in the Tumor Microenvironment Following Neoadjuvant Therapy: Implications for Immunomodulation and Survival in Pancreatic Ductal Adenocarcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.564099" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.564099" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, X.; Liu, Y.-Z.; Lan, R.; Liu, Y.; Pillarisetty, V. G.; Li, D.; Zhao, C. L.; Sarkar, S.; Liu, W.; Hanna, I.; Gupta, M.; Hajdu, C.; Melamed, J.; Widmer, J.; Allendorf, J.</p>
        <p class="info">Score: 8.8, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.564099' target='https://doi.org/10.1101/2023.10.26.564099'> 10.1101/2023.10.26.564099</a></p>
        <p class="abstract">PurposeNeoadjuvant therapy (NAT) is increasingly being used for pancreatic ductal adenocarcinoma (PDAC) treatment. However, its distinct effects on carcinoma cells and the tumor microenvironment (TME) are not fully understood. This study employs spatial transcriptomics and single-cell RNA sequencing to investigate how NAT differentially remodels PDACs carcinoma cells and TME.

Experimental DesignWe used spatial transcriptomics to compare gene expression profiles in carcinoma cells and the TME between NAT-treated and NAT-naive PDAC patients and correlated with their clinicopathologic features. Complementary single-nucleus RNA sequencing (snRNA-seq) analysis was conducted to validate our findings and identify cell types driving NAT-induced gene expression alterations.

ResultsWe found NAT not only induces apoptosis and inhibits proliferation in carcinoma cells but also significantly remodels the TME. Notably, NAT induces a coordinated upregulation of multiple key complement genes (C3, C1S, C1R, C4B and C7) in the TME, making the complement pathway one of the most significantly affected pathways by NAT. Patients with higher TME complement expression following NAT exhibit improved overall survival; more immunomodulatory and neurotrophic cancer-associated fibroblasts (CAFs); more CD4&#43; T cells, monocytes and mast cells; and lower immune exhaustion gene expression. snRNA-seq analysis demonstrates C3 complement upregulation specifically in CAFs but not in other stroma cell types.

ConclusionsOur findings indicate that NAT may reduce immunosuppression in PDAC by enhancing complement production and signaling within the TME. These findings suggest that local complement dynamics could serve as a novel biomarker for prognosis, evaluating treatment response and resistance, and guiding therapeutic strategies in NAT-treated PDAC patients.

Translational RelevanceAs neoadjuvant therapy (NAT) increasingly becomes the preferred approach in treating resectable and borderline resectable pancreatic ductal adenocarcinoma (PDAC), there is a growing demand for novel biomarkers specifically tailored for post-NAT PDAC patients. Our study focused on how NAT differentially remodels the tumor cells and the tumor microenvironment (TME). We demonstrate that NAT can enhance local complement production and signaling in PDACs TME, which is associated with reduced immune exhaustion and improved overall survival. Our results highlight the importance of local complement dynamics in influencing treatment response, resistance, and overall clinical outcomes in PDAC. This new mechanism provides new opportunities in biomarker development which could facilitate more accurate prognostication and precise treatment stratification, particularly for patients undergoing NAT in PDAC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.18.572179">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.18.572179" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.18.572179">
        <p class="paperTitle">Distinct host preconditioning regimens differentially impact the antitumor potency of adoptively transferred Th17 cells.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.18.572179" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.18.572179" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wittling, M. C.; Knochelmann, H. M.; Wyatt, M. M.; Rangel Rivera, G. O.; Cole, A. C.; Lesinski, G. B.; Paulos, C. M.</p>
        <p class="info">Score: 7.5, Published: 2023-12-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.18.572179' target='https://doi.org/10.1101/2023.12.18.572179'> 10.1101/2023.12.18.572179</a></p>
        <p class="abstract">Background: Mechanisms by which distinct methods of host preconditioning impact the efficacy of adoptively transferred antitumor T helper cells is unknown. Methods: CD4&#43; T cells with a transgenic TCR that recognize TRP-1 melanoma antigen were polarized to the T helper 17 (Th17) phenotype and then transferred into melanoma-bearing mice preconditioned with either total body irradiation or chemotherapy. Results: We found that preconditioning mice with a non-myeloablative dose of total body irradiation (TBI of 5 Gy) was more effective than using an equivalently dosed non-myeloablative chemotherapy (CTX at 200 mg/kg) at augmenting therapeutic activity of anti-tumor TRP-1 Th17 cells. Anti-tumor Th17 cells engrafted better following preconditioning with TBI and regressed large established melanoma in all animals. Conversely, only half of mice survived long-term when preconditioned with CTX and infused with anti-melanoma Th17 cells. IL-17 and IFN-{gamma} produced by the infused Th17 cells, were detected in animals given either TBI or CTX preconditioning. Interestingly, inflammatory cytokines (G-CSF, IL-6, MCP-1, IL-5, and KC) were significantly elevated in the serum of mice preconditioned with TBI versus CTX after Th17 therapy. Conclusions: Our results indicate, for the first time, that the antitumor response, persistence, and cytokine profiles resulting from Th17 therapy are impacted by the specific regimen of host preconditioning. This work is important for understanding mechanisms that promote long-lived responses by ACT, particularly as CD4&#43; based T cell therapies are now emerging in the clinic.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.12.571283">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.12.571283" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.12.571283">
        <p class="paperTitle">Single-nucleotide-resolution genomic maps of O6-methylguanine from the glioblastoma drug temozolomide</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.12.571283" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.12.571283" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Buechel, J.; Mingard, C.; Takhaveev, V.; Reinert, P. B.; Keller, G.; Kloter, T.; Huber, S.; McKeague, M.; Sturla, S. J.</p>
        <p class="info">Score: 6.7, Published: 2023-12-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.12.571283' target='https://doi.org/10.1101/2023.12.12.571283'> 10.1101/2023.12.12.571283</a></p>
        <p class="abstract">Temozolomide kills cancer cells by forming O6-methylguanine (O6-MeG), which leads to apoptosis by mismatch-repair overload, as an important basis for glioblastoma therapy. However, O6-MeG repair by O6-methylguanine-DNA methyltransferase (MGMT) strongly contributes to drug resistance. Genomic profiles of O6-MeG may be useful for understanding how O6-MeG accumulation can be influenced by repair mechanisms, but there are no methods to map genomic locations of O6-MeG. Here, we devised an immunoprecipitation- and polymerase-stalling-based strategy to locate O6-MeG across the whole genome at single-nucleotide resolution. We applied this strategy, termed O6-MeG-seq, to analyze O6-MeG formation and repair with regards to sequence contexts and functional genomic regions in glioblastoma-derived cell lines deficient in MGMT and evaluated the impact of MGMT transfection on these profiles. O6-MeG signatures from the sequencing data were highly similar to mutational signatures extracted from cancer genomes of patients previously treated with temozolomide, consistent with O6-MeG initiating mutations. Furthermore, it appears that MGMT does not preferentially repair O6-MeG with respect to sequence context, chromatin state or gene expression level, however, may protect oncogenes from mutations. Finally, an MGMT-independent strand bias in O6-MeG accumulation in highly expressed genes suggests alternative repair influences. These data provide high resolution insight on how O6-MeG formation and repair is impacted by genome structure and regulation. O6-MeG-seq also provides a strategy for future studies of DNA modification signatures as diagnostic markers for addressing drug resistance and preventing secondary cancers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.13.571512">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.13.571512" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.13.571512">
        <p class="paperTitle">Melanoma innervation is associated with cancer progression in a zebrafish xenograft model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.13.571512" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.13.571512" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: LORENZINI, F.; MARINES, J.; Le FRIEC, J.; DO KHOA, N.; NIETO, M. A.; SANCHEZ-LAORDEN, B.; MIONE, M.; FONTENILLE, L.; KISSA, K.</p>
        <p class="info">Score: 5.8, Published: 2023-12-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.13.571512' target='https://doi.org/10.1101/2023.12.13.571512'> 10.1101/2023.12.13.571512</a></p>
        <p class="abstract">The peripheral nervous system has a key role in regulating tumour biology in different types of cancer. Here, by modelling aggressive melanoma in larval zebrafish xenografts, we highlight the dynamics of tumour innervation in the tumour microenvironment (TME). Axonogenesis and dendritogenesis are detected in the motoneurons surrounding the melanoma niche and neurogenesis is observed in the nearby population of the enteric nervous system. We also demonstrate the crucial role of catecholamines in promoting melanoma progression, supporting in vivo cancer cell dissemination and invasion. This zebrafish model will allow to uncover neural markers associated with melanoma progression to help in the design of innovative anti-neurogenic therapies targeting specifically the neuronal signals that regulate melanoma progression.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=171 SRC=&#34;FIGDIR/small/571512v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (41K):
org.highwire.dtl.DTLVardef@13dd11org.highwire.dtl.DTLVardef@16905corg.highwire.dtl.DTLVardef@15fb42borg.highwire.dtl.DTLVardef@2da605_HPS_FORMAT_FIGEXP  M_FIG C_FIG HighlightsO_LITransplantation of human melanoma cells in 3 dpf zebrafish swim bladder allows the development of aggressive melanoma, which cells invade the surrounding organs and migrate over distant locations.
C_LIO_LIThe presence of melanoma cells in the larval zebrafish induces morphological changes in the motoneurons inside the tumour niche, including increased axon length and dendritic arborization.
C_LIO_LIThe invasion of melanoma cells in the larval intestine promotes neurogenesis of enteric neurons.
C_LIO_LITransplanted melanoma cells display direct contact with enteric neurons in the intestinal region and migrate along axons to escape from the primary cancer mass, as a mechanism similar to vessel co-option during metastatic dissemination.
C_LIO_LICatecholamines promote melanoma cell migration and invasion in the zebrafish, modelling melanoma progression.
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.14.571727">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.14.571727" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.14.571727">
        <p class="paperTitle">A high-throughput screening approach to discover potential colorectal cancer chemotherapeutics: Repurposing drugs to disrupt 14-3-3 protein-BAD interactions</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.14.571727" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.14.571727" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: He, S.; Delgadillo Silva, L.; Rutter, G. A.; Lim, G. E.</p>
        <p class="info">Score: 5.2, Published: 2023-12-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.14.571727' target='https://doi.org/10.1101/2023.12.14.571727'> 10.1101/2023.12.14.571727</a></p>
        <p class="abstract">Inducing apoptosis in different types of cancer cells is an effective therapeutic strategy. However, the success of existing chemotherapeutics can be compromised by tumor cell resistance and systemic off-target effects. Therefore, the discovery of pro-apoptotic compounds with minimal systemic side-effects is crucial. 14-3-3 proteins are molecular scaffolds that serve as important regulators of cell survival. Our previous study demonstrated that 14-3-3{zeta} can sequester BAD, a pro-apoptotic member of the BCL-2 protein family, in the cytoplasm and prevent its translocation to mitochondria to inhibit the induction of apoptosis. Despite being a critical mechanism of cell survival, it is unclear whether disrupting 14-3-3 protein:BAD interactions could be harnessed as a chemotherapeutic approach. Herein, we established a BRET-based high-throughput drug screening approach (Z-score= 0.52) capable of identifying molecules that can disrupt 14-3-3{zeta}BAD interactions. An FDA-approved drug library containing 1971 compounds was used for screening, and the capacity of identified hits to induce cell death was examined in NIH3T3-fibroblasts and colorectal cancer cell lines, HT-29 and Caco-2. Our in vitro results suggest that terfenadine, penfluridol, and lomitapide could be potentially repurposed for treating colorectal cancer. Moreover, our screening method demonstrates the feasibility of identifying pro-apoptotic agents that can be applied towards conditions where aberrant cell growth or function are key determinants of disease pathogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568421">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568421" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568421">
        <p class="paperTitle">Modelling glioblastoma resistance to temozolomide. Combination of spheroid and mathematical models to simulate cellular adaptation in vitro.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568421" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568421" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Perez-Aliacar, M.; Ayensa-Jimenez, J.; Randelovic, T.; Ochoa, I.; Doblare, M.</p>
        <p class="info">Score: 7.8, Published: 2023-11-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568421' target='https://doi.org/10.1101/2023.11.24.568421'> 10.1101/2023.11.24.568421</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWDrug resistance is one of the biggest challenges in the fight against cancer. In particular, in the case of glioblastoma, the most lethal brain tumour, resistance to temozolomide (the standard of care drug for chemotherapy in this tumour), is one of the main reasons behind treatment failure and hence responsible for the poor prognosis of patients diagnosed with this disease.

In this paper, we combine the power of three-dimensional in vitro experiments of treated glioblastoma spheroids with mathematical models of tumour evolution and adaptation. We use a novel approach based on internal variables for modelling the acquisition of resistance to temozolomide that is observed in a group of treated spheroids in the experiments. These internal variables describe the cells phenotypic state, which depends on the history of drug exposure and affects cell behaviour. We use model selection to determine the most parsimonious model and calibrate it to reproduce the experimental data, obtaining a high level of agreement between the in vitro and in silico outcomes. A sensitivity analysis is carried out to investigate the impact of each model parameter in the predictions. More importantly, we show the utility of our model for answering biological questions, such as what is the intrinsic adaptation mechanism, or for separating the sensitive and resistant populations. We conclude that the proposed in silico framework, in combination with experiments, can be useful to improve our understanding of the mechanisms behind drug resistance in glioblastoma and to eventually set some guidelines for the design of new treatment schemes.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
