<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Drug screening in human physiologic medium identifies uric acid as an inhibitor of rigosertib efficacy
Authors: Rawat, V.; DeLear, P.; Prashanth, P.; Ozgurses, M. E.; Tebeje, A.; Burns, P. A.; Conger, K. O.; Solis, C.; Hasnain, Y.; Novikova, A.; Endress, J. E.; Gonzalez-Sanchez, P.; Dong, W.; Stephanopoulos, G.; DeNicola, G. M.; Harris, I. S.; Sept, D. S.; Mason, F. M.; Coloff, J. L.
Score: 31.7, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550731">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Drug screening in human physiologic medium identifies uric acid as an inhibitor of rigosertib efficacy
Authors: Rawat, V.; DeLear, P.; Prashanth, P.; Ozgurses, M. E.; Tebeje, A.; Burns, P. A.; Conger, K. O.; Solis, C.; Hasnain, Y.; Novikova, A.; Endress, J. E.; Gonzalez-Sanchez, P.; Dong, W.; Stephanopoulos, G.; DeNicola, G. M.; Harris, I. S.; Sept, D. S.; Mason, F. M.; Coloff, J. L.
Score: 31.7, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550731" />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-30T10:36:57+00:00" />
<meta property="article:modified_time" content="2023-07-30T10:36:57+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Drug screening in human physiologic medium identifies uric acid as an inhibitor of rigosertib efficacy
Authors: Rawat, V.; DeLear, P.; Prashanth, P.; Ozgurses, M. E.; Tebeje, A.; Burns, P. A.; Conger, K. O.; Solis, C.; Hasnain, Y.; Novikova, A.; Endress, J. E.; Gonzalez-Sanchez, P.; Dong, W.; Stephanopoulos, G.; DeNicola, G. M.; Harris, I. S.; Sept, D. S.; Mason, F. M.; Coloff, J. L.
Score: 31.7, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550731"/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Drug screening in human physiologic medium identifies uric acid as an inhibitor of rigosertib efficacy\nAuthors: Rawat, V.; DeLear, P.; Prashanth, P.; Ozgurses, M. E.; Tebeje, A.; Burns, P. A.; Conger, K. O.; Solis, C.; Hasnain, Y.; Novikova, A.; Endress, J. E.; Gonzalez-Sanchez, P.; Dong, W.; Stephanopoulos, G.; DeNicola, G. M.; Harris, I. S.; Sept, D. S.; Mason, F. M.; Coloff, J. L.\nScore: 31.7, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550731",
  "keywords": [
    
  ],
  "articleBody": " Drug screening in human physiologic medium identifies uric acid as an inhibitor of rigosertib efficacy\nAuthors: Rawat, V.; DeLear, P.; Prashanth, P.; Ozgurses, M. E.; Tebeje, A.; Burns, P. A.; Conger, K. O.; Solis, C.; Hasnain, Y.; Novikova, A.; Endress, J. E.; Gonzalez-Sanchez, P.; Dong, W.; Stephanopoulos, G.; DeNicola, G. M.; Harris, I. S.; Sept, D. S.; Mason, F. M.; Coloff, J. L.\nScore: 31.7, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550731\nThe non-physiological nutrient levels found in traditional culture media have been shown to affect numerous aspects of cancer cell physiology, including how cells respond to certain therapeutic agents. Here, we comprehensively evaluated how physiological nutrient levels impact therapeutic response by performing drug screening in human plasma-like medium (HPLM). We observed dramatic nutrient-dependent changes in sensitivity to a variety of FDA-approved and clinically trialed compounds, including rigosertib, an experimental cancer therapeutic that has recently failed in phase 3 clinical trials. Mechanistically, we found that the ability of rigosertib to destabilize microtubules is strongly inhibited by the purine metabolism waste product uric acid, which is uniquely abundant in humans relative to traditional in vitro and in vivo cancer models. Structural modelling studies suggest that uric acid interacts with the tubulin-rigosertib complex and may act as an uncompetitive inhibitor of rigosertib. These results offer a possible explanation for the failure of rigosertib in clinical trials and demonstrate the utility of physiological media to achieve in vitro results that better represent human therapeutic responses.\nERBB signalling contributes to immune evasion in KRAS-driven lung adenocarcinoma\nAuthors: Laing, S.; Kruspig, B.; Shaw, R.; Officer-Jones, L.; Edwards, S.; McKinven, D.; Hsieh, Y.-C.; Powley, I.; Brady, N.; Pennie, R.; Kwan, R.; Lima, A.; Myrta, S.; Periyasamy, M.; Dye, I. C.; Nixon, C.; Clark, G.; Junttila, M. R.; Maddalo, D.; Miller, C.; Ali, S.; Fuchter, M. J.; Nickles, D.; Kirschner, K.; Brown, R. B.; LeQuesne, J.; Strathdee, D.; Coffelt, S. B.; Roberts, E.; Murphy, D. J.\nScore: 19.3, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550274\nImmunotherapy is increasingly viewed as treatment of choice for lung cancer, however, clinical responses to immune checkpoint blockade remain highly unpredictable and are largely transient. A deeper mechanistic understanding of the dynamics of tumour:immune interactions is needed to drive rational development of improved treatment strategies. Progress is hampered by a paucity of autochthonous model systems in which to interrogate the 2-way interactions of immune responses to evolving tumours and vice-versa. Specifically, commonly used genetically engineered mouse models typically lack the genetic diversity needed to drive an adaptive immune response. APOBEC mutagenesis signatures are prominent in lung cancer and APOBEC activity is predicted to drive immune visibility through Cytidine deaminase activity, coupled with inaccurate DNA-repair responses. We therefore generated a CRE-inducible APOBEC3B allele, interbred with multiple oncogenic drivers of lung adenocarcinoma, and used the resulting mice to investigate the response to PD1 blockade at single cell resolution. SIGNIFICANCEUsing our novel immune-visible model of KRas-driven autochthonous lung adenocarcinoma, we uncovered a surprising increase in tumour-cell expression of EGFR/ERBB ligands following treatment with -PD1 and present evidence that transient ERBB blockade can restore immune surveillance in KRas mutant LuAd and combine effectively with immune checkpoint blockade.\nPredicting response to cytotoxic chemotherapy\nAuthors: Thompson, J.; Madrid, L.; Hernando, B.; Sauer, C.; Vias, M.; Escobar-Rey, M.; Leung, W.-K.; Huckstep, J.; Sekowska, M.; Hosking, K.; Jimenez-Linan, M.; Reinius, M.; Dobson, H.; De Silva, D.; Fernandez-Sanroman, A.; Sanders, D.; Correia Martins, F.; Quintela-Fandino, M.; Markowetz, F.; Yip, J.; Brenton, J. D.; Piskorz, A.; Macintyre, G.\nScore: 16.6, Published: 2023-07-26 DOI: 10.1101/2023.01.28.525988\nCytotoxic chemotherapies have been a crucial part of cancer treatment for over 40 years. While their primary target is cancer cells, they can also harm normal cells, resulting in dose-limiting toxicity. Most chemotherapies were approved before the advent of precision biomarkers, as such, many patients experience severe toxic side effects without any benefit. To address this challenge, we have developed three precision biomarkers to predict response to platins, taxanes, and anthracyclines. Based on chromosomal instability (CIN) signatures, these biomarkers can be computed from a single genomic test. For platins and taxanes, we used CIN signatures related to impaired homologous recombination, while for anthracyclines, we discovered a CIN signature representing micronuclei induction which predicts resistance. In a clinical study involving 41 high-grade serous ovarian cancers, patients predicted to be sensitive by these biomarkers showed significantly prolonged progression-free survival. To further validate the effectiveness of the taxane and anthracycline predictors, we conducted a retrospective randomised control study involving 182 ovarian and 219 breast cancer patients. Patients predicted as resistant showed increased risk of time to treatment failure compared to standard of care, hazard ratios of 1.73 (95%CI=0.98-3.07) for taxane in ovarian, 3.67 (95%CI=2.12-6.34) for taxane in breast, and 1.93 (95%CI=1.22-3.04) for doxorubicin in ovarian. We also found that liquid biopsies can be used to make these predictions in up to 30% of ovarian cancer patients. Our findings highlight the clinical value of CIN signatures in predicting treatment response to various chemotherapies across multiple different types of cancer. The ability to quantify multiple CIN signature biomarkers using a single genomic test offers a unified approach to guide treatment decisions for cytotoxic chemotherapies. Ultimately, this has the potential to transform the current one-size-fits-all chemotherapy approach into a more precise and tailored form of medicine.\nAdaptive therapy achieves long-term control of chemotherapy resistance in high grade ovarian cancer\nAuthors: Hockings, H.; Lakatos, E.; Huang, W.; Mossner, M.; Khan, M. A.; Metcalf, S.; Nicolini, F.; Smith, K.; Baker, A.-M.; Graham, T.; Lockley, M.\nScore: 13.5, Published: 2023-07-25 DOI: 10.1101/2023.07.21.549688\nDrug resistance results in poor outcomes for most patients with metastatic cancer. Adaptive Therapy (AT) proposes to address this by exploiting presumed fitness costs incurred by drug-resistant cells when drug is absent, and prescribing dose reductions to allow fitter, sensitive cells to re-grow and re- sensitise the tumour. However, empirical evidence for treatment-induced fitness change is lacking. We show that fitness costs in chemotherapy-resistant ovarian cancer cause selective decline and apoptosis of resistant populations in low-resource conditions. Moreover, carboplatin AT caused fluctuations in sensitive/resistant tumour population size in vitro and significantly extended survival of tumour-bearing mice. In sequential blood-derived cell-free DNA and tumour samples obtained longitudinally from ovarian cancer patients during treatment, we inferred resistant cancer cell population size through therapy and observed it correlated strongly with disease burden. These data have enabled us to launch a multicentre, phase 2 randomised controlled trial (ACTOv) to evaluate AT in ovarian cancer.\nA heterozygous CEBPA mutation disrupting the bZIP domain causes MDS disease progression\nAuthors: Almaghrabi, R.; Alyayhawi, Y.; Keane, P.; Ward, C. R. J.; Bayley, R.; Sargas, C.; Menendez, P.; Murphy, G. J.; Sobahy, T.; Baghdadi, M. A.; Flemban, A. F.; Kabrah, S.; Akerman, I.; Raghavan, M.; Barragan, E.; Bonifer, C.; Garcia, P.\nScore: 10.8, Published: 2023-07-26 DOI: 10.1101/2023.07.24.550349\nMyelodysplastic syndrome disease (MDS) has a variable risk for progression to AML. Mutations in CEBPA are associated with a high risk of disease progression, but whether this mutation is causative for AML development is unclear. To answer this question, we generated patient-derived, MDS-specific iPSCs recapitulating the patient disease phenotype upon differentiation to blood, with hematopoietic progenitor cells showing erythroid and myeloid-dysplasia. Introduction of a frameshift mutation affecting the C/EBP bZIP domain led to disease progression, with a reduction in clonogenic potential, block in granulocyte development and increased self-renewal capacity of erythroid progenitors. ATAC-seq revealed that the acquisition of this mutation reshaped the chromatin landscape at distal cis-regulatory regions, promoting changes in clonal composition as observed by single cell RNAseq. Our work identifies mutant CEBPA as causative for MDS disease progression, providing a new isogenic MDS experimental model for drug screening to improve diagnostic and therapeutic strategies. HighlightsO_LIDevelopment of isogenic iPSC model of clonal evolution of MDS C_LIO_LIMonoallelic disruption of CEBPA bZIP domain is causative for MDS disease progression C_LIO_LIMonoallelic disruption of CEBPA bZIP reshapes chromatin landscape C_LIO_LIPatient derived iPSCs recapitulate drug responsiveness C_LI\nCRISPR/Cas9 screenings unearth protein arginine methyltransferase 7 as a novel driver of metastasis in prostate cancer\nAuthors: Rodrigo-Faus, M.; Vincelle-Nieto, A.; Vidal, N.; Puente, J.; Saiz-Pardo, M.; Lopez-Garcia, A.; Mendiburu-Elicabe, M.; Palao, N.; Baquero, C.; Cuesta, A. M.; Qu, H.-Q.; Hakonarson, H.; Musteanu, M.; Reyes-Palomares, A.; Porras, A.; Bragado, P.; Gutierrez-Uzquiza, A.\nScore: 9.0, Published: 2023-07-22 DOI: 10.1101/2023.07.20.549704\nOwing to the inefficacy of available treatments, the survival rate of patients with metastatic prostate cancer (mPCa) is severely decreased. Therefore, it is crucial to identify new therapeutic targets to increase their survival. This study aim was to identify the most relevant regulators of mPCa onset by performing two high-throughput CRISPR/Cas9 screenings. Furthermore, some of the top hits were validated using small interfering RNA (siRNA) technology, with protein arginine methyltransferase 7 (PRMT7) being the best candidate. Its inhibition or depletion via CRISPR significantly reduced mPCa cell capacities in vitro. Moreover, PRMT7 ablation reduced mPCa appearance in chicken chorioallantoic membrane and mouse xenograft assays. Molecularly, PRMT7 reprograms the expression of several adhesion molecules through methylation of several transcription factors, such as FoxK1 or NR1H2, which results in primary tumor PCa cell adhesion loss and motility gain. Importantly, PRMT7 is upregulated in advanced stages of Spanish PCa tumor samples and PRMT7 pharmacological inhibition reduces the dissemination of mPCa cells. Thus, here is shown that PRMT7 is a potential therapeutic target and biomarker of mPCa.\nAn integrative proteomics approach identifies tyrosine kinase KIT as a novel therapeutic target for SPINK1-positive prostate cancer.\nAuthors: Manzar, N.; Khan, U. K.; Goel, A.; Carskadon, S.; Gupta, N.; Palanisamy, N.; Ateeq, B.\nScore: 5.7, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550265\nElevated Serine Peptidase Inhibitor, Kazal type 1 (SPINK1) levels in [~]10-25% of prostate cancer (PCa) patients associate with aggressive phenotype, for which there are limited treatment choices and dismal clinical outcomes. Using an integrative proteomics approach involving label-free phosphoproteome and proteome profiling, we delineated the downstream signaling pathways involved in SPINK1-mediated tumorigenesis in PCa, and identified tyrosine kinase KIT as a highly enriched kinase. Furthermore, high to moderate levels of KIT expression was detected in [~]85% of SPINK1-positive PCa specimens. KIT signaling regulates SPINK1-associated oncogenesis, and treatment with KIT inhibitor reduces tumor growth and distant metastases in preclinical mice models. Mechanistically, KIT signaling modulates WNT/{beta}-catenin pathway and confers stemness-related features in PCa. Notably, inhibiting KIT signaling restores AR/REST levels, forming a feedback loop enabling SPINK1 repression. Overall, we uncover the role of KIT signaling downstream of SPINK1 in maintaining lineage plasticity and provide new treatment modalities for advanced-stage SPINK1-positive subtype.\nLocal Adaptive Mapping of Karyotype Fitness Landscapes\nAuthors: Beck, R. J.; Andor, N.\nScore: 5.9, Published: 2023-07-24 DOI: 10.1101/2023.07.14.549079\nAneuploidy occurs in most solid tumors and has the potential to dramatically modify cellular phenotype and fitness. Despite the importance of aneuploidy in tumor evolution, quantitative understanding of the evolutionary landscape of aneuploidy is lacking. To address this question, we developed a method to infer the fitness landscape of either arm-level or whole-chromosome level karyotypes. Our method takes longitudinal single cell sequencing data from an evolving cell population as input, then estimates the fitness of thousands of karyotypes located near to the input data in karyotype space. The predictive ability of the method was validated using artificial data generated from an agent based model, as well as data from a selection of in vitro and in vivo passaged cell lines. We applied our pipeline to an in vitro dataset of serially passaged cells and - based on topological analysis of the fitness landscape around diploid and tetraploid karyotypes - found support for the hypothesis that whole genome doubling benefits tumour cells by conferring robustness against missegregations.\nEnhancer plasticity in endometrial tumorigenesis demarcates non-coding driver mutations and alterations in 3D genome organization to stimulate oncogene expression\nAuthors: Gregoricchio, S.; Kojic, A.; Hoogstraat, M.; Schuurman, K.; Stelloo, S.; Severson, T. M.; O'Mara, T. A.; Droog, M.; Singh, A. A.; Glubb, D. M.; Wessels, L. F. A.; Vermeulen, M.; van Leeuwen, F.; Zwart, W.\nScore: 5.3, Published: 2023-07-27 DOI: 10.1101/2023.07.25.550368\nThe incidence and mortality of Endometrial Cancer (EC) is on the rise. 85% of ECs depend on Estrogen Receptor alpha (ER) for proliferation, but little is known about its transcriptional regulation in these tumors. We generated epigenomics and Hi-C data streams in healthy and tumor endometrial tissues, identifying robust ER reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with WGS data from metastatic samples revealed a striking enrichment of non-coding somatic mutations at tumor-enriched ER sites. Through machine learning-based predictions and interaction proteomics analyses, we identified an enhancer mutation which alters 3D genome organization, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. In summary, we identified a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ER.\nComprehensive multi-platform tyrosine kinase profiling reveals novel actionable FGFR aberrations across pediatric and AYA sarcomas\nAuthors: Fordham, A. M.; Brown, L. M.; Mayoh, C.; Salib, A.; Barger, Z. A.; Wong, M.; Sian, T. C. C. L. K.; Xie, J.; Gunther, K.; Trebilcock, P.; Terry, R. L.; Barahona, P.; Ajuyah, P.; Sherstyuk, A.; Avila, A.; Cadiz, R.; Perkins, C. M.; Gifford, A. J.; Mao, J.; Zhao, A.; O'Regan, L. P.; Gorgels, D.; Lau, L. M.; Ziegler, D. S.; Haber, M.; Tyrrell, V.; Lock, R. B.; Cowley, M.; Nicholls, W.; Daly, R. J.; Ekert, P. G.; Fleuren, E. D.\nScore: 5.4, Published: 2023-07-21 DOI: 10.1101/2023.07.19.548825\nNo targeted agents are approved for pediatric sarcomas. Tyrosine kinase (TK) inhibitors represent attractive therapeutic candidates, however, beyond rare TK-activating fusions or mutations, predictive biomarkers are lacking. RNA overexpression of TKs is more commonly observed in pediatric sarcomas, however, an unresolved question is when upregulated TK expression is associated with kinase activation and signaling dependence. We explored the TK molecular landscape of 107 sarcoma patients from the ZERO Childhood Cancer precision medicine program using whole genomic and transcriptomic sequencing. Phosphoproteomic analyses of tyrosine phosphorylation (pY) and functional in vitro and in vivo assays were also performed in cell lines and patient-derived xenografts (PDXs). Our integrated analysis shows that although novel genomic driver lesions are rare, they are present and therapeutically actionable in selected patients as exemplified by a novel LSM1-FGFR1 fusion identified in an osteosarcoma patient. We further show that in certain contexts, TK expression data can be used to indicate TK pathway activity and predict TK-inhibitor sensitivity. We exemplify the utility of FGFR-inhibitors in PAX3-FOXO1 fusion-positive rhabdomyosarcomas (FP-RMS) mediated by high FGFR4 and FGF8 RNA expression levels, and overt activation of FGFR4 (FGFR4_pY). We demonstrate marked tumor growth inhibition in all FP-RMS PDXs treated with single agent FGF401 (FGFR4-specific inhibitor) and single agent lenvatinib (multi-kinase FGFR-inhibitor). Clinical benefit of lenvatinib in a relapsed metastatic FP-RMS patient further exemplifies that FGFR-inhibitors deserve additional investigation in FP-RMS patients. Statement of significanceOur multi-omic interrogation of sarcomas in the ZERO Childhood Cancer program illustrates how an RNA-expression biomarker signature (FGFR4+/FGF8+) in association with FGFR4 activation identifies that PAX3-FOXO1-positive rhabdomyosarcoma patients could benefit from FGFR-inhibitors.\n",
  "wordCount" : "2476",
  "inLanguage": "en",
  "datePublished": "2023-07-30T10:36:57Z",
  "dateModified": "2023-07-30T10:36:57Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on July 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550731">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550731" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550731">
        <p class="paperTitle">Drug screening in human physiologic medium identifies uric acid as an inhibitor of rigosertib efficacy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550731" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550731" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rawat, V.; DeLear, P.; Prashanth, P.; Ozgurses, M. E.; Tebeje, A.; Burns, P. A.; Conger, K. O.; Solis, C.; Hasnain, Y.; Novikova, A.; Endress, J. E.; Gonzalez-Sanchez, P.; Dong, W.; Stephanopoulos, G.; DeNicola, G. M.; Harris, I. S.; Sept, D. S.; Mason, F. M.; Coloff, J. L.</p>
        <p class="info">Score: 31.7, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550731' target='https://doi.org/10.1101/2023.07.26.550731'> 10.1101/2023.07.26.550731</a></p>
        <p class="abstract">The non-physiological nutrient levels found in traditional culture media have been shown to affect numerous aspects of cancer cell physiology, including how cells respond to certain therapeutic agents. Here, we comprehensively evaluated how physiological nutrient levels impact therapeutic response by performing drug screening in human plasma-like medium (HPLM). We observed dramatic nutrient-dependent changes in sensitivity to a variety of FDA-approved and clinically trialed compounds, including rigosertib, an experimental cancer therapeutic that has recently failed in phase 3 clinical trials. Mechanistically, we found that the ability of rigosertib to destabilize microtubules is strongly inhibited by the purine metabolism waste product uric acid, which is uniquely abundant in humans relative to traditional in vitro and in vivo cancer models. Structural modelling studies suggest that uric acid interacts with the tubulin-rigosertib complex and may act as an uncompetitive inhibitor of rigosertib. These results offer a possible explanation for the failure of rigosertib in clinical trials and demonstrate the utility of physiological media to achieve in vitro results that better represent human therapeutic responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550274">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550274" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550274">
        <p class="paperTitle">ERBB signalling contributes to immune evasion in KRAS-driven lung adenocarcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550274" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550274" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Laing, S.; Kruspig, B.; Shaw, R.; Officer-Jones, L.; Edwards, S.; McKinven, D.; Hsieh, Y.-C.; Powley, I.; Brady, N.; Pennie, R.; Kwan, R.; Lima, A.; Myrta, S.; Periyasamy, M.; Dye, I. C.; Nixon, C.; Clark, G.; Junttila, M. R.; Maddalo, D.; Miller, C.; Ali, S.; Fuchter, M. J.; Nickles, D.; Kirschner, K.; Brown, R. B.; LeQuesne, J.; Strathdee, D.; Coffelt, S. B.; Roberts, E.; Murphy, D. J.</p>
        <p class="info">Score: 19.3, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550274' target='https://doi.org/10.1101/2023.07.24.550274'> 10.1101/2023.07.24.550274</a></p>
        <p class="abstract">Immunotherapy is increasingly viewed as treatment of choice for lung cancer, however, clinical responses to immune checkpoint blockade remain highly unpredictable and are largely transient. A deeper mechanistic understanding of the dynamics of tumour:immune interactions is needed to drive rational development of improved treatment strategies. Progress is hampered by a paucity of autochthonous model systems in which to interrogate the 2-way interactions of immune responses to evolving tumours and vice-versa. Specifically, commonly used genetically engineered mouse models typically lack the genetic diversity needed to drive an adaptive immune response. APOBEC mutagenesis signatures are prominent in lung cancer and APOBEC activity is predicted to drive immune visibility through Cytidine deaminase activity, coupled with inaccurate DNA-repair responses. We therefore generated a CRE-inducible APOBEC3B allele, interbred with multiple oncogenic drivers of lung adenocarcinoma, and used the resulting mice to investigate the response to PD1 blockade at single cell resolution.

SIGNIFICANCEUsing our novel immune-visible model of KRas-driven autochthonous lung adenocarcinoma, we uncovered a surprising increase in tumour-cell expression of EGFR/ERBB ligands following treatment with -PD1 and present evidence that transient ERBB blockade can restore immune surveillance in KRas mutant LuAd and combine effectively with immune checkpoint blockade.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.28.525988">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.28.525988" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.28.525988">
        <p class="paperTitle">Predicting response to cytotoxic chemotherapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.28.525988" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.28.525988" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thompson, J.; Madrid, L.; Hernando, B.; Sauer, C.; Vias, M.; Escobar-Rey, M.; Leung, W.-K.; Huckstep, J.; Sekowska, M.; Hosking, K.; Jimenez-Linan, M.; Reinius, M.; Dobson, H.; De Silva, D.; Fernandez-Sanroman, A.; Sanders, D.; Correia Martins, F.; Quintela-Fandino, M.; Markowetz, F.; Yip, J.; Brenton, J. D.; Piskorz, A.; Macintyre, G.</p>
        <p class="info">Score: 16.6, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.28.525988' target='https://doi.org/10.1101/2023.01.28.525988'> 10.1101/2023.01.28.525988</a></p>
        <p class="abstract">Cytotoxic chemotherapies have been a crucial part of cancer treatment for over 40 years. While their primary target is cancer cells, they can also harm normal cells, resulting in dose-limiting toxicity. Most chemotherapies were approved before the advent of precision biomarkers, as such, many patients experience severe toxic side effects without any benefit. To address this challenge, we have developed three precision biomarkers to predict response to platins, taxanes, and anthracyclines. Based on chromosomal instability (CIN) signatures, these biomarkers can be computed from a single genomic test. For platins and taxanes, we used CIN signatures related to impaired homologous recombination, while for anthracyclines, we discovered a CIN signature representing micronuclei induction which predicts resistance. In a clinical study involving 41 high-grade serous ovarian cancers, patients predicted to be sensitive by these biomarkers showed significantly prolonged progression-free survival. To further validate the effectiveness of the taxane and anthracycline predictors, we conducted a retrospective randomised control study involving 182 ovarian and 219 breast cancer patients. Patients predicted as resistant showed increased risk of time to treatment failure compared to standard of care, hazard ratios of 1.73 (95%CI=0.98-3.07) for taxane in ovarian, 3.67 (95%CI=2.12-6.34) for taxane in breast, and 1.93 (95%CI=1.22-3.04) for doxorubicin in ovarian. We also found that liquid biopsies can be used to make these predictions in up to 30% of ovarian cancer patients. Our findings highlight the clinical value of CIN signatures in predicting treatment response to various chemotherapies across multiple different types of cancer. The ability to quantify multiple CIN signature biomarkers using a single genomic test offers a unified approach to guide treatment decisions for cytotoxic chemotherapies. Ultimately, this has the potential to transform the current one-size-fits-all chemotherapy approach into a more precise and tailored form of medicine.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.549688">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.549688" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.549688">
        <p class="paperTitle">Adaptive therapy achieves long-term control of chemotherapy resistance in high grade ovarian cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.549688" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.549688" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hockings, H.; Lakatos, E.; Huang, W.; Mossner, M.; Khan, M. A.; Metcalf, S.; Nicolini, F.; Smith, K.; Baker, A.-M.; Graham, T.; Lockley, M.</p>
        <p class="info">Score: 13.5, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.549688' target='https://doi.org/10.1101/2023.07.21.549688'> 10.1101/2023.07.21.549688</a></p>
        <p class="abstract">Drug resistance results in poor outcomes for most patients with metastatic cancer. Adaptive Therapy (AT) proposes to address this by exploiting presumed fitness costs incurred by drug-resistant cells when drug is absent, and prescribing dose reductions to allow fitter, sensitive cells to re-grow and re- sensitise the tumour. However, empirical evidence for treatment-induced fitness change is lacking. We show that fitness costs in chemotherapy-resistant ovarian cancer cause selective decline and apoptosis of resistant populations in low-resource conditions. Moreover, carboplatin AT caused fluctuations in sensitive/resistant tumour population size in vitro and significantly extended survival of tumour-bearing mice. In sequential blood-derived cell-free DNA and tumour samples obtained longitudinally from ovarian cancer patients during treatment, we inferred resistant cancer cell population size through therapy and observed it correlated strongly with disease burden. These data have enabled us to launch a multicentre, phase 2 randomised controlled trial (ACTOv) to evaluate AT in ovarian cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550349">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550349" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550349">
        <p class="paperTitle">A heterozygous CEBPA mutation disrupting the bZIP domain causes MDS disease progression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550349" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550349" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Almaghrabi, R.; Alyayhawi, Y.; Keane, P.; Ward, C. R. J.; Bayley, R.; Sargas, C.; Menendez, P.; Murphy, G. J.; Sobahy, T.; Baghdadi, M. A.; Flemban, A. F.; Kabrah, S.; Akerman, I.; Raghavan, M.; Barragan, E.; Bonifer, C.; Garcia, P.</p>
        <p class="info">Score: 10.8, Published: 2023-07-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550349' target='https://doi.org/10.1101/2023.07.24.550349'> 10.1101/2023.07.24.550349</a></p>
        <p class="abstract">Myelodysplastic syndrome disease (MDS) has a variable risk for progression to AML. Mutations in CEBPA are associated with a high risk of disease progression, but whether this mutation is causative for AML development is unclear. To answer this question, we generated patient-derived, MDS-specific iPSCs recapitulating the patient disease phenotype upon differentiation to blood, with hematopoietic progenitor cells showing erythroid and myeloid-dysplasia. Introduction of a frameshift mutation affecting the C/EBP bZIP domain led to disease progression, with a reduction in clonogenic potential, block in granulocyte development and increased self-renewal capacity of erythroid progenitors. ATAC-seq revealed that the acquisition of this mutation reshaped the chromatin landscape at distal cis-regulatory regions, promoting changes in clonal composition as observed by single cell RNAseq. Our work identifies mutant CEBPA as causative for MDS disease progression, providing a new isogenic MDS experimental model for drug screening to improve diagnostic and therapeutic strategies.

HighlightsO_LIDevelopment of isogenic iPSC model of clonal evolution of MDS
C_LIO_LIMonoallelic disruption of CEBPA bZIP domain is causative for MDS disease progression
C_LIO_LIMonoallelic disruption of CEBPA bZIP reshapes chromatin landscape
C_LIO_LIPatient derived iPSCs recapitulate drug responsiveness
C_LI</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549704">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549704" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549704">
        <p class="paperTitle">CRISPR/Cas9 screenings unearth protein arginine methyltransferase 7 as a novel driver of metastasis in prostate cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549704" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549704" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rodrigo-Faus, M.; Vincelle-Nieto, A.; Vidal, N.; Puente, J.; Saiz-Pardo, M.; Lopez-Garcia, A.; Mendiburu-Elicabe, M.; Palao, N.; Baquero, C.; Cuesta, A. M.; Qu, H.-Q.; Hakonarson, H.; Musteanu, M.; Reyes-Palomares, A.; Porras, A.; Bragado, P.; Gutierrez-Uzquiza, A.</p>
        <p class="info">Score: 9.0, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549704' target='https://doi.org/10.1101/2023.07.20.549704'> 10.1101/2023.07.20.549704</a></p>
        <p class="abstract">Owing to the inefficacy of available treatments, the survival rate of patients with metastatic prostate cancer (mPCa) is severely decreased. Therefore, it is crucial to identify new therapeutic targets to increase their survival. This study aim was to identify the most relevant regulators of mPCa onset by performing two high-throughput CRISPR/Cas9 screenings. Furthermore, some of the top hits were validated using small interfering RNA (siRNA) technology, with protein arginine methyltransferase 7 (PRMT7) being the best candidate. Its inhibition or depletion via CRISPR significantly reduced mPCa cell capacities in vitro. Moreover, PRMT7 ablation reduced mPCa appearance in chicken chorioallantoic membrane and mouse xenograft assays. Molecularly, PRMT7 reprograms the expression of several adhesion molecules through methylation of several transcription factors, such as FoxK1 or NR1H2, which results in primary tumor PCa cell adhesion loss and motility gain. Importantly, PRMT7 is upregulated in advanced stages of Spanish PCa tumor samples and PRMT7 pharmacological inhibition reduces the dissemination of mPCa cells. Thus, here is shown that PRMT7 is a potential therapeutic target and biomarker of mPCa.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550265" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550265">
        <p class="paperTitle">An integrative proteomics approach identifies tyrosine kinase KIT as a novel therapeutic target for SPINK1-positive prostate cancer.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manzar, N.; Khan, U. K.; Goel, A.; Carskadon, S.; Gupta, N.; Palanisamy, N.; Ateeq, B.</p>
        <p class="info">Score: 5.7, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550265' target='https://doi.org/10.1101/2023.07.24.550265'> 10.1101/2023.07.24.550265</a></p>
        <p class="abstract">Elevated Serine Peptidase Inhibitor, Kazal type 1 (SPINK1) levels in [~]10-25% of prostate cancer (PCa) patients associate with aggressive phenotype, for which there are limited treatment choices and dismal clinical outcomes. Using an integrative proteomics approach involving label-free phosphoproteome and proteome profiling, we delineated the downstream signaling pathways involved in SPINK1-mediated tumorigenesis in PCa, and identified tyrosine kinase KIT as a highly enriched kinase. Furthermore, high to moderate levels of KIT expression was detected in [~]85% of SPINK1-positive PCa specimens. KIT signaling regulates SPINK1-associated oncogenesis, and treatment with KIT inhibitor reduces tumor growth and distant metastases in preclinical mice models. Mechanistically, KIT signaling modulates WNT/{beta}-catenin pathway and confers stemness-related features in PCa. Notably, inhibiting KIT signaling restores AR/REST levels, forming a feedback loop enabling SPINK1 repression. Overall, we uncover the role of KIT signaling downstream of SPINK1 in maintaining lineage plasticity and provide new treatment modalities for advanced-stage SPINK1-positive subtype.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549079">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549079" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549079">
        <p class="paperTitle">Local Adaptive Mapping of Karyotype Fitness Landscapes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549079" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549079" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beck, R. J.; Andor, N.</p>
        <p class="info">Score: 5.9, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549079' target='https://doi.org/10.1101/2023.07.14.549079'> 10.1101/2023.07.14.549079</a></p>
        <p class="abstract">Aneuploidy occurs in most solid tumors and has the potential to dramatically modify cellular phenotype and fitness. Despite the importance of aneuploidy in tumor evolution, quantitative understanding of the evolutionary landscape of aneuploidy is lacking. To address this question, we developed a method to infer the fitness landscape of either arm-level or whole-chromosome level karyotypes. Our method takes longitudinal single cell sequencing data from an evolving cell population as input, then estimates the fitness of thousands of karyotypes located near to the input data in karyotype space. The predictive ability of the method was validated using artificial data generated from an agent based model, as well as data from a selection of in vitro and in vivo passaged cell lines. We applied our pipeline to an in vitro dataset of serially passaged cells and - based on topological analysis of the fitness landscape around diploid and tetraploid karyotypes - found support for the hypothesis that whole genome doubling benefits tumour cells by conferring robustness against missegregations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550368">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550368" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550368">
        <p class="paperTitle">Enhancer plasticity in endometrial tumorigenesis demarcates non-coding driver mutations and alterations in 3D genome organization to stimulate oncogene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550368" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550368" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gregoricchio, S.; Kojic, A.; Hoogstraat, M.; Schuurman, K.; Stelloo, S.; Severson, T. M.; O&#39;Mara, T. A.; Droog, M.; Singh, A. A.; Glubb, D. M.; Wessels, L. F. A.; Vermeulen, M.; van Leeuwen, F.; Zwart, W.</p>
        <p class="info">Score: 5.3, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550368' target='https://doi.org/10.1101/2023.07.25.550368'> 10.1101/2023.07.25.550368</a></p>
        <p class="abstract">The incidence and mortality of Endometrial Cancer (EC) is on the rise. 85% of ECs depend on Estrogen Receptor alpha (ER) for proliferation, but little is known about its transcriptional regulation in these tumors. We generated epigenomics and Hi-C data streams in healthy and tumor endometrial tissues, identifying robust ER reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with WGS data from metastatic samples revealed a striking enrichment of non-coding somatic mutations at tumor-enriched ER sites. Through machine learning-based predictions and interaction proteomics analyses, we identified an enhancer mutation which alters 3D genome organization, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. In summary, we identified a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ER.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.548825">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.548825" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.548825">
        <p class="paperTitle">Comprehensive multi-platform tyrosine kinase profiling reveals novel actionable FGFR aberrations across pediatric and AYA sarcomas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.548825" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.548825" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fordham, A. M.; Brown, L. M.; Mayoh, C.; Salib, A.; Barger, Z. A.; Wong, M.; Sian, T. C. C. L. K.; Xie, J.; Gunther, K.; Trebilcock, P.; Terry, R. L.; Barahona, P.; Ajuyah, P.; Sherstyuk, A.; Avila, A.; Cadiz, R.; Perkins, C. M.; Gifford, A. J.; Mao, J.; Zhao, A.; O&#39;Regan, L. P.; Gorgels, D.; Lau, L. M.; Ziegler, D. S.; Haber, M.; Tyrrell, V.; Lock, R. B.; Cowley, M.; Nicholls, W.; Daly, R. J.; Ekert, P. G.; Fleuren, E. D.</p>
        <p class="info">Score: 5.4, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.548825' target='https://doi.org/10.1101/2023.07.19.548825'> 10.1101/2023.07.19.548825</a></p>
        <p class="abstract">No targeted agents are approved for pediatric sarcomas. Tyrosine kinase (TK) inhibitors represent attractive therapeutic candidates, however, beyond rare TK-activating fusions or mutations, predictive biomarkers are lacking. RNA overexpression of TKs is more commonly observed in pediatric sarcomas, however, an unresolved question is when upregulated TK expression is associated with kinase activation and signaling dependence. We explored the TK molecular landscape of 107 sarcoma patients from the ZERO Childhood Cancer precision medicine program using whole genomic and transcriptomic sequencing. Phosphoproteomic analyses of tyrosine phosphorylation (pY) and functional in vitro and in vivo assays were also performed in cell lines and patient-derived xenografts (PDXs). Our integrated analysis shows that although novel genomic driver lesions are rare, they are present and therapeutically actionable in selected patients as exemplified by a novel LSM1-FGFR1 fusion identified in an osteosarcoma patient. We further show that in certain contexts, TK expression data can be used to indicate TK pathway activity and predict TK-inhibitor sensitivity. We exemplify the utility of FGFR-inhibitors in PAX3-FOXO1 fusion-positive rhabdomyosarcomas (FP-RMS) mediated by high FGFR4 and FGF8 RNA expression levels, and overt activation of FGFR4 (FGFR4_pY). We demonstrate marked tumor growth inhibition in all FP-RMS PDXs treated with single agent FGF401 (FGFR4-specific inhibitor) and single agent lenvatinib (multi-kinase FGFR-inhibitor). Clinical benefit of lenvatinib in a relapsed metastatic FP-RMS patient further exemplifies that FGFR-inhibitors deserve additional investigation in FP-RMS patients.

Statement of significanceOur multi-omic interrogation of sarcomas in the ZERO Childhood Cancer program illustrates how an RNA-expression biomarker signature (FGFR4&#43;/FGF8&#43;) in association with FGFR4 activation identifies that PAX3-FOXO1-positive rhabdomyosarcoma patients could benefit from FGFR-inhibitors.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
