<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Coordinated inheritance of extrachromosomal DNA species in human cancer cells
Authors: Hung, K. L.; Jones, M. G.; Wong, I. T.-L.; Lange, J. T.; Luebeck, J.; Scanu, E.; He, B. J.; Brückner, L.; Li, R.; Gonzalez, R. C.; Schmargon, R.; Dörr, J. R.; Belk, J. A.; Bafna, V.; Werner, B.; Huang, W.; Henssen, A. G.; Mischel, P. S.; Chang, H. Y.
Score: 32.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549597
The chromosomal theory of inheritance has dominated human genetics, including cancer genetics.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Coordinated inheritance of extrachromosomal DNA species in human cancer cells
Authors: Hung, K. L.; Jones, M. G.; Wong, I. T.-L.; Lange, J. T.; Luebeck, J.; Scanu, E.; He, B. J.; Brückner, L.; Li, R.; Gonzalez, R. C.; Schmargon, R.; Dörr, J. R.; Belk, J. A.; Bafna, V.; Werner, B.; Huang, W.; Henssen, A. G.; Mischel, P. S.; Chang, H. Y.
Score: 32.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549597
The chromosomal theory of inheritance has dominated human genetics, including cancer genetics." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-26T10:37:30+00:00" />
<meta property="article:modified_time" content="2023-07-26T10:37:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Coordinated inheritance of extrachromosomal DNA species in human cancer cells
Authors: Hung, K. L.; Jones, M. G.; Wong, I. T.-L.; Lange, J. T.; Luebeck, J.; Scanu, E.; He, B. J.; Brückner, L.; Li, R.; Gonzalez, R. C.; Schmargon, R.; Dörr, J. R.; Belk, J. A.; Bafna, V.; Werner, B.; Huang, W.; Henssen, A. G.; Mischel, P. S.; Chang, H. Y.
Score: 32.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549597
The chromosomal theory of inheritance has dominated human genetics, including cancer genetics."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Coordinated inheritance of extrachromosomal DNA species in human cancer cells\nAuthors: Hung, K. L.; Jones, M. G.; Wong, I. T.-L.; Lange, J. T.; Luebeck, J.; Scanu, E.; He, B. J.; Brückner, L.; Li, R.; Gonzalez, R. C.; Schmargon, R.; Dörr, J. R.; Belk, J. A.; Bafna, V.; Werner, B.; Huang, W.; Henssen, A. G.; Mischel, P. S.; Chang, H. Y.\nScore: 32.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549597\nThe chromosomal theory of inheritance has dominated human genetics, including cancer genetics.",
  "keywords": [
    
  ],
  "articleBody": " Coordinated inheritance of extrachromosomal DNA species in human cancer cells\nAuthors: Hung, K. L.; Jones, M. G.; Wong, I. T.-L.; Lange, J. T.; Luebeck, J.; Scanu, E.; He, B. J.; Brückner, L.; Li, R.; Gonzalez, R. C.; Schmargon, R.; Dörr, J. R.; Belk, J. A.; Bafna, V.; Werner, B.; Huang, W.; Henssen, A. G.; Mischel, P. S.; Chang, H. Y.\nScore: 32.0, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549597\nThe chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences, herein termed ecDNA species, can co-exist to facilitate intermolecular cooperation in cancer cells. However, how multiple ecDNA species within a tumor cell are assorted and maintained across somatic cell generations to drive cancer cell evolution is not known. Here we show that cooperative ecDNA species can be coordinately inherited through mitotic co-segregation. Imaging and single-cell analyses show that multiple ecDNAs encoding distinct oncogenes co-occur and are correlated in copy number in human cancer cells. EcDNA species are coordinately segregated asymmetrically during mitosis, resulting in daughter cells with simultaneous copy number gains in multiple ecDNA species prior to any selection. Computational modeling reveals the quantitative principles of ecDNA co-segregation and co-selection, predicting their observed distributions in cancer cells. Finally, we show that coordinated inheritance of ecDNAs enables co-amplification of specialized ecDNAs containing only enhancer elements and guides therapeutic strategies to jointly deplete cooperating ecDNA oncogenes. Coordinated inheritance of ecDNAs confers stability to oncogene cooperation and novel gene regulatory circuits, allowing winning combinations of epigenetic states to be transmitted across cell generations.\nA Cell Cycle-Dependent Ferroptosis Sensitivity Switch Governed by EMP2\nAuthors: Rodencal, J.; Kim, N.; Li, V.; He, A.; Lange, M.; He, J.; Tarangelo, A.; Schafer, Z. T.; Olzmann, J.; Sage, J.; Long, J. Z.; Dixon, S. J.\nScore: 9.5, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549715\nFerroptosis is a non-apoptotic form of cell death characterized by iron-dependent lipid peroxidation. Ferroptosis can be induced by system xc- cystine/glutamate antiporter inhibition or by direct inhibition of the phospholipid hydroperoxidase glutathione peroxidase 4 (GPX4). The regulation of ferroptosis in response to system xc- inhibition versus direct GPX4 inhibition may be distinct. Here, we show that cell cycle arrest enhances sensitivity to ferroptosis triggered by GPX4 inhibition but not system xc- inhibition. Arrested cells have increased levels of oxidizable polyunsaturated fatty acid-containing phospholipids, which drives sensitivity to GPX4 inhibition. Epithelial membrane protein 2 (EMP2) expression is reduced upon cell cycle arrest and is sufficient to enhance ferroptosis in response to direct GPX4 inhibition. An orally bioavailable GPX4 inhibitor increased markers of ferroptotic lipid peroxidation in vivo in combination with a cell cycle arresting agent. Thus, responses to different ferroptosis-inducing stimuli can be regulated by cell cycle state.\nCRISPR/Cas9 screenings unearth protein arginine methyltransferase 7 as a novel driver of metastasis in prostate cancer\nAuthors: Rodrigo-Faus, M.; Vincelle-Nieto, A.; Vidal, N.; Puente, J.; Saiz-Pardo, M.; Lopez-Garcia, A.; Mendiburu-Elicabe, M.; Palao, N.; Baquero, C.; Cuesta, A. M.; Qu, H.-Q.; Hakonarson, H.; Musteanu, M.; Reyes-Palomares, A.; Porras, A.; Bragado, P.; Gutierrez-Uzquiza, A.\nScore: 7.3, Published: 2023-07-22 DOI: 10.1101/2023.07.20.549704\nOwing to the inefficacy of available treatments, the survival rate of patients with metastatic prostate cancer (mPCa) is severely decreased. Therefore, it is crucial to identify new therapeutic targets to increase their survival. This study aim was to identify the most relevant regulators of mPCa onset by performing two high-throughput CRISPR/Cas9 screenings. Furthermore, some of the top hits were validated using small interfering RNA (siRNA) technology, with protein arginine methyltransferase 7 (PRMT7) being the best candidate. Its inhibition or depletion via CRISPR significantly reduced mPCa cell capacities in vitro. Moreover, PRMT7 ablation reduced mPCa appearance in chicken chorioallantoic membrane and mouse xenograft assays. Molecularly, PRMT7 reprograms the expression of several adhesion molecules through methylation of several transcription factors, such as FoxK1 or NR1H2, which results in primary tumor PCa cell adhesion loss and motility gain. Importantly, PRMT7 is upregulated in advanced stages of Spanish PCa tumor samples and PRMT7 pharmacological inhibition reduces the dissemination of mPCa cells. Thus, here is shown that PRMT7 is a potential therapeutic target and biomarker of mPCa.\nEGFR-to-Src family tyrosine kinase switching in proliferating-DTP TNBC cells creates a hyperphosphorylation-dependent vulnerability to EGFR TKI\nAuthors: Chaudhary, N.; Choudhary, B. S.; Shivashankar, A.; Manna, S.; Ved, K.; Shaikh, S. E.; Khanna, S.; Barr, J.; Dani, J.; Verma, N.\nScore: 6.5, Published: 2023-07-19 DOI: 10.1101/2023.07.17.549374\nTriple-Negative Breast Cancer (TNBC) is the most aggressive type of breast malignancy, with chemotherapy as the only mainstay treatment. TNBC patients have the worst prognoses as a large fraction of them do not achieve complete pathological response post-treatment and develop drug-resistant residual disease. Molecular mechanisms that trigger proliferation in drug-resistant chemo-residual TNBC cells are poorly understood due to the lack of investigations using clinically relevant cellular models. In this study, we have established TNBC subtype-specific cellular models of proliferating drug-tolerant persister (PDTP) cells using different classes of chemotherapeutic agents that recapitulate clinical residual disease with molecular heterogeneity. Analysis of total phospho-tyrosine signals in TNBC PDTPs showed an enhanced phospho-tyrosine content compared to the parental cells (PC). Interestingly, using mass-spectrometry analysis, we identified a dramatic decrease in epidermal growth factor receptor (EGFR) expression in the PDTPs, while the presence of hyper-activated tyrosine phosphorylation of EGFR compared to PC. Further, we show that EGFR has enhanced lysosomal trafficking in PDTPs with a concomitant increase in N-Myc Downstream Regulated-1 expression that co-localizes with EGFR to mediate receptor degradation. More surprisingly, we found that reduced protein levels of EGFR are coupled with a robust increase in Src family kinases, including Lyn and Fyn kinases, that creates a hyper-phosphorylation state of EGFR-Src tyrosine kinases axis in PDTPs and mediates downstream over-activation of STAT3, AKT and MAP kinases. Moreover, paclitaxel-derived PDTPs show increased sensitivity to EGFR TKI Gefitinib and its combination with paclitaxel selectively induced cell death in PDTP-P TNBC cells and 3D spheroids by strongly downregulating phosphorylation of EGFR-Src with concomitant downregulation of Lyn and Fyn tyrosine kinases. Collectively, this study identifies a unique hyper-phosphorylation cellular state of TNBC PDTPs established by switching of EGFR-Src family tyrosine kinases creating a vulnerability to EGFR TKI.\nIn vivo single-cell CRISPR uncovers distinct TNF-alphaprograms in clonal expansion and tumorigenesis\nAuthors: Renz, P. F.; Ghoshdastider, U.; Baghai Sain, S.; Valdivia-Francia, F.; Khandekar, A.; Ormiston, M.; Bernasconi, M.; Kretz, J. A.; Lee, M.; Hyams, K.; Forny, M.; Pohly, M.; Ficht, X.; Ellis, S. J.; Moor, A. E.; Sendoel, A.\nScore: 16.6, Published: 2023-07-14 DOI: 10.1101/2023.07.13.548697\nThe tumor evolution model posits that malignant transformation is preceded by randomly distributed driver mutations in cancer genes, which cause clonal expansions in phenotypically normal tissues. Although clonal expansions occur frequently in human epithelia and can remodel almost entire tissues, the mechanisms behind why only a small number of clones transform into malignant tumors remain enigmatic. Here, we develop an in vivo single-cell CRISPR strategy to systematically investigate tissue-wide clonal dynamics of the 150 most frequently mutated squamous cell carcinoma genes. We couple ultrasound-guided in utero lentiviral microinjections, single-cell RNA sequencing, guide capture and spatial transcriptomics to longitudinally monitor cell type-specific clonal expansions, document their underlying gene programs and contrast clonal expansions from tumor initiation. We uncover a TNF- signaling module that acts as a generalizable driver of clonal expansions in epithelial tissues. Conversely, during tumorigenesis, the TNF- signaling module is downregulated, and instead, we identify a subpopulation of invasive cancer cells that switch to an autocrine TNF- gene program. By analyzing clonally expanded perturbations and their frequency in tumors, we demonstrate that the autocrine TNF- gene program is associated with epithelial-mesenchymal transition (EMT) and is preexistent in a subpopulation of expanded epidermal stem cells, contributing to the predisposition for tumor initiation. Finally, we provide in vivo evidence that the epithelial TNF- gene program is sufficient to mediate invasive properties of epidermal stem cells and show that the TNF- signature correlates with shorter overall survival in human squamous cell carcinoma patients. Collectively, our study demonstrates the power of applying in vivo single-cell CRISPR screening to mammalian tissues and unveils distinct TNF- programs in tumor evolution. Understanding the biology of clonal expansions in phenotypically normal epithelia and the mechanisms governing their transformation will guide the development of novel strategies for early cancer detection and therapy.\nA novel ILK/STAT3 pathway controls plasticity in a neural stem cell model of glioblastoma.\nAuthors: Loftus, A. E. P.; Romano, M. S.; Nguyen Phuong, A.; Muir, M. T.; Dawson, J. C.; Avalle, L.; Douglas, A. T.; Mort, R. L.; Byron, A.; Carragher, N. O.; Pollard, S. M.; Brunton, V. G.; Frame, M. C.\nScore: 5.4, Published: 2023-07-19 DOI: 10.1101/2023.07.19.549653\nGlioblastomas (GBM) are driven by malignant neural stem-like cells that display extensive heterogeneity and phenotypic plasticity, which drives tumour progression and therapeutic resistance. Here we show that the nodal extracellular matrix-cell adhesion protein integrin-linked kinase (ILK; a pseudokinase), is a key determinant of phenotypic plasticity and the mesenchymal-like, invasive cell state in mouse GBM stem cells. We found that a novel ILK-STAT3 signalling pathway is required for plasticity that enables the transition of GBM stem cells to an astrocyte-like state both in vitro and in vivo. GBM cells genetically depleted of ILK become predominantly stabilised in a transcriptionally-defined progenitor-like state that is characterised by lack of response to differentiation cues and constitutive proliferation. Loss of ILK or interference with STAT3 impairs differentiation potential, reducing phenotypic plasticity of tumour cell populations; additionally, ILK loss causes a mesenchymal- to epithelial-like morphological transition and suppression of malignancy-associated features. Our work defines ILK as a central regulator of multiple GBM phenotypes including phenotypic plasticity and mesenchymal state.\nTranscriptional regulation of Dyskerin via canonical WNT signaling modulates sphingolipid biosynthesis and drives colorectal cancer.\nAuthors: Nigam, S.; Khan, U. K.; Praveen, A.; Shendre, A.; Carskadon, S.; Kapoor, A.; Tewari, A.; Chandra, A.; Palanisamy, N.; Ateeq, B.\nScore: 4.7, Published: 2023-07-19 DOI: 10.1101/2023.07.18.549480\nTargeting EGFR has been effective in RAS/RAF wild-type colorectal cancer (CRC) patients. However, residual tumor relapses, necessitating the importance of biomarker-guided novel therapeutics. We show elevated DKC1 in [~]88% of CRC patients with poor recurrence-free survival. Clinically, DKC1-positive patients exhibit similarity with CMS2 class, the canonical subtype with active WNT signaling. We show functional significance of DKC1 in cell proliferation, stemness, DNA repair, and survival. Further, mice bearing DKC1 knockdown xenografts show [~]81% reduction in tumor burden. Mechanistically, WNT/{beta}-catenin signaling orchestrates DKC1 expression, then, DKC1/SOX2 complex regulates SGPP2, modulating sphingolipids metabolism. Downregulation of DKC1 in CRC lead to reduced SGPP2 levels leading to dysregulation of sphingolipid biosynthesis. Of note, DKC1-high CRC patients show accumulation of ceramides, namely C23 and C24, signifying their utility in diagnosis. Collectively, we delineate the mechanistic circuitry involved in DKC1-mediated CRC progression, propose ceramides as biomarker, and underscore WNT-based therapeutics for DKC1-positive patients.\nLocal Adaptive Mapping of Karyotype Fitness Landscapes\nAuthors: Beck, R. J.; Andor, N.\nScore: 5.1, Published: 2023-07-24 DOI: 10.1101/2023.07.14.549079\nAneuploidy occurs in most solid tumors and has the potential to dramatically modify cellular phenotype and fitness. Despite the importance of aneuploidy in tumor evolution, quantitative understanding of the evolutionary landscape of aneuploidy is lacking. To address this question, we developed a method to infer the fitness landscape of either arm-level or whole-chromosome level karyotypes. Our method takes longitudinal single cell sequencing data from an evolving cell population as input, then estimates the fitness of thousands of karyotypes located near to the input data in karyotype space. The predictive ability of the method was validated using artificial data generated from an agent based model, as well as data from a selection of in vitro and in vivo passaged cell lines. We applied our pipeline to an in vitro dataset of serially passaged cells and - based on topological analysis of the fitness landscape around diploid and tetraploid karyotypes - found support for the hypothesis that whole genome doubling benefits tumour cells by conferring robustness against missegregations.\nAn integrative proteomics approach identifies tyrosine kinase KIT as a novel therapeutic target for SPINK1-positive prostate cancer.\nAuthors: Manzar, N.; Khan, U. K.; Goel, A.; Carskadon, S.; Gupta, N.; Palanisamy, N.; Ateeq, B.\nScore: 4.5, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550265\nElevated Serine Peptidase Inhibitor, Kazal type 1 (SPINK1) levels in ~10-25% of prostate cancer (PCa) patients associate with aggressive phenotype, for which there are limited treatment choices and dismal clinical outcomes. Using an integrative proteomics approach involving label-free phosphoproteome and proteome profiling, we delineated the downstream signaling pathways involved in SPINK1-mediated tumorigenesis in PCa, and identified tyrosine kinase KIT as a highly enriched kinase. Furthermore, high to moderate levels of KIT expression was detected in ~85% of SPINK1-positive PCa specimens. KIT signaling regulates SPINK1-associated oncogenesis, and treatment with KIT inhibitor reduces tumor growth and distant metastases in preclinical mice models. Mechanistically, KIT signaling modulates WNT/beta-catenin pathway and confers stemness-related features in PCa. Notably, inhibiting KIT signaling restores AR/REST levels, forming a feedback loop enabling SPINK1 repression. Overall, we uncover the role of KIT signaling downstream of SPINK1 in maintaining lineage plasticity and provide new treatment modalities for advanced-stage SPINK1-positive subtype.\nComprehensive multi-platform tyrosine kinase profiling reveals novel actionable FGFR aberrations across pediatric and AYA sarcomas\nAuthors: Fordham, A. M.; Brown, L. M.; Mayoh, C.; Salib, A.; Barger, Z. A.; Wong, M.; Sian, T. C. C. L. K.; Xie, J.; Gunther, K.; Trebilcock, P.; Terry, R. L.; Barahona, P.; Ajuyah, P.; Sherstyuk, A.; Avila, A.; Cadiz, R.; Perkins, C. M.; Gifford, A. J.; Mao, J.; Zhao, A.; O'Regan, L. P.; Gorgels, D.; Lau, L. M.; Ziegler, D. S.; Haber, M.; Tyrrell, V.; Lock, R. B.; Cowley, M.; Nicholls, W.; Daly, R. J.; Ekert, P. G.; Fleuren, E. D.\nScore: 3.7, Published: 2023-07-21 DOI: 10.1101/2023.07.19.548825\nNo targeted agents are approved for pediatric sarcomas. Tyrosine kinase (TK) inhibitors represent attractive therapeutic candidates, however, beyond rare TK-activating fusions or mutations, predictive biomarkers are lacking. RNA overexpression of TKs is more commonly observed in pediatric sarcomas, however, an unresolved question is when upregulated TK expression is associated with kinase activation and signaling dependence. We explored the TK molecular landscape of 107 sarcoma patients from the ZERO Childhood Cancer precision medicine program using whole genomic and transcriptomic sequencing. Phosphoproteomic analyses of tyrosine phosphorylation (pY) and functional in vitro and in vivo assays were also performed in cell lines and patient-derived xenografts (PDXs). Our integrated analysis shows that although novel genomic driver lesions are rare, they are present and therapeutically actionable in selected patients as exemplified by a novel LSM1-FGFR1 fusion identified in an osteosarcoma patient. We further show that in certain contexts, TK expression data can be used to indicate TK pathway activity and predict TK-inhibitor sensitivity. We exemplify the utility of FGFR-inhibitors in PAX3-FOXO1 fusion-positive rhabdomyosarcomas (FP-RMS) mediated by high FGFR4 and FGF8 RNA expression levels, and overt activation of FGFR4 (FGFR4_pY). We demonstrate marked tumor growth inhibition in all FP-RMS PDXs treated with single agent FGF401 (FGFR4-specific inhibitor) and single agent lenvatinib (multi-kinase FGFR-inhibitor). Clinical benefit of lenvatinib in a relapsed metastatic FP-RMS patient further exemplifies that FGFR-inhibitors deserve additional investigation in FP-RMS patients. Statement of significanceOur multi-omic interrogation of sarcomas in the ZERO Childhood Cancer program illustrates how an RNA-expression biomarker signature (FGFR4+/FGF8+) in association with FGFR4 activation identifies that PAX3-FOXO1-positive rhabdomyosarcoma patients could benefit from FGFR-inhibitors.\n",
  "wordCount" : "2552",
  "inLanguage": "en",
  "datePublished": "2023-07-26T10:37:30Z",
  "dateModified": "2023-07-26T10:37:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on July 26, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549597">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549597" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549597">
        <p class="paperTitle">Coordinated inheritance of extrachromosomal DNA species in human cancer cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549597" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549597" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hung, K. L.; Jones, M. G.; Wong, I. T.-L.; Lange, J. T.; Luebeck, J.; Scanu, E.; He, B. J.; Brückner, L.; Li, R.; Gonzalez, R. C.; Schmargon, R.; Dörr, J. R.; Belk, J. A.; Bafna, V.; Werner, B.; Huang, W.; Henssen, A. G.; Mischel, P. S.; Chang, H. Y.</p>
        <p class="info">Score: 32.0, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549597' target='https://doi.org/10.1101/2023.07.18.549597'> 10.1101/2023.07.18.549597</a></p>
        <p class="abstract">The chromosomal theory of inheritance has dominated human genetics, including cancer genetics. Genes on the same chromosome segregate together while genes on different chromosomes assort independently, providing a fundamental tenet of Mendelian inheritance. Extrachromosomal DNA (ecDNA) is a frequent event in cancer that drives oncogene amplification, dysregulated gene expression and intratumoral heterogeneity, including through random segregation during cell division. Distinct ecDNA sequences, herein termed ecDNA species, can co-exist to facilitate intermolecular cooperation in cancer cells. However, how multiple ecDNA species within a tumor cell are assorted and maintained across somatic cell generations to drive cancer cell evolution is not known. Here we show that cooperative ecDNA species can be coordinately inherited through mitotic co-segregation. Imaging and single-cell analyses show that multiple ecDNAs encoding distinct oncogenes co-occur and are correlated in copy number in human cancer cells. EcDNA species are coordinately segregated asymmetrically during mitosis, resulting in daughter cells with simultaneous copy number gains in multiple ecDNA species prior to any selection. Computational modeling reveals the quantitative principles of ecDNA co-segregation and co-selection, predicting their observed distributions in cancer cells. Finally, we show that coordinated inheritance of ecDNAs enables co-amplification of specialized ecDNAs containing only enhancer elements and guides therapeutic strategies to jointly deplete cooperating ecDNA oncogenes. Coordinated inheritance of ecDNAs confers stability to oncogene cooperation and novel gene regulatory circuits, allowing winning combinations of epigenetic states to be transmitted across cell generations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549715">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549715" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549715">
        <p class="paperTitle">A Cell Cycle-Dependent Ferroptosis Sensitivity Switch Governed by EMP2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549715" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549715" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rodencal, J.; Kim, N.; Li, V.; He, A.; Lange, M.; He, J.; Tarangelo, A.; Schafer, Z. T.; Olzmann, J.; Sage, J.; Long, J. Z.; Dixon, S. J.</p>
        <p class="info">Score: 9.5, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549715' target='https://doi.org/10.1101/2023.07.19.549715'> 10.1101/2023.07.19.549715</a></p>
        <p class="abstract">Ferroptosis is a non-apoptotic form of cell death characterized by iron-dependent lipid peroxidation. Ferroptosis can be induced by system xc- cystine/glutamate antiporter inhibition or by direct inhibition of the phospholipid hydroperoxidase glutathione peroxidase 4 (GPX4). The regulation of ferroptosis in response to system xc- inhibition versus direct GPX4 inhibition may be distinct. Here, we show that cell cycle arrest enhances sensitivity to ferroptosis triggered by GPX4 inhibition but not system xc- inhibition. Arrested cells have increased levels of oxidizable polyunsaturated fatty acid-containing phospholipids, which drives sensitivity to GPX4 inhibition. Epithelial membrane protein 2 (EMP2) expression is reduced upon cell cycle arrest and is sufficient to enhance ferroptosis in response to direct GPX4 inhibition. An orally bioavailable GPX4 inhibitor increased markers of ferroptotic lipid peroxidation in vivo in combination with a cell cycle arresting agent. Thus, responses to different ferroptosis-inducing stimuli can be regulated by cell cycle state.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.20.549704">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.20.549704" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.20.549704">
        <p class="paperTitle">CRISPR/Cas9 screenings unearth protein arginine methyltransferase 7 as a novel driver of metastasis in prostate cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.20.549704" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.20.549704" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rodrigo-Faus, M.; Vincelle-Nieto, A.; Vidal, N.; Puente, J.; Saiz-Pardo, M.; Lopez-Garcia, A.; Mendiburu-Elicabe, M.; Palao, N.; Baquero, C.; Cuesta, A. M.; Qu, H.-Q.; Hakonarson, H.; Musteanu, M.; Reyes-Palomares, A.; Porras, A.; Bragado, P.; Gutierrez-Uzquiza, A.</p>
        <p class="info">Score: 7.3, Published: 2023-07-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.20.549704' target='https://doi.org/10.1101/2023.07.20.549704'> 10.1101/2023.07.20.549704</a></p>
        <p class="abstract">Owing to the inefficacy of available treatments, the survival rate of patients with metastatic prostate cancer (mPCa) is severely decreased. Therefore, it is crucial to identify new therapeutic targets to increase their survival. This study aim was to identify the most relevant regulators of mPCa onset by performing two high-throughput CRISPR/Cas9 screenings. Furthermore, some of the top hits were validated using small interfering RNA (siRNA) technology, with protein arginine methyltransferase 7 (PRMT7) being the best candidate. Its inhibition or depletion via CRISPR significantly reduced mPCa cell capacities in vitro. Moreover, PRMT7 ablation reduced mPCa appearance in chicken chorioallantoic membrane and mouse xenograft assays. Molecularly, PRMT7 reprograms the expression of several adhesion molecules through methylation of several transcription factors, such as FoxK1 or NR1H2, which results in primary tumor PCa cell adhesion loss and motility gain. Importantly, PRMT7 is upregulated in advanced stages of Spanish PCa tumor samples and PRMT7 pharmacological inhibition reduces the dissemination of mPCa cells. Thus, here is shown that PRMT7 is a potential therapeutic target and biomarker of mPCa.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.17.549374">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.17.549374" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.17.549374">
        <p class="paperTitle">EGFR-to-Src family tyrosine kinase switching in proliferating-DTP TNBC cells creates a hyperphosphorylation-dependent vulnerability to EGFR TKI</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.17.549374" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.17.549374" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chaudhary, N.; Choudhary, B. S.; Shivashankar, A.; Manna, S.; Ved, K.; Shaikh, S. E.; Khanna, S.; Barr, J.; Dani, J.; Verma, N.</p>
        <p class="info">Score: 6.5, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.17.549374' target='https://doi.org/10.1101/2023.07.17.549374'> 10.1101/2023.07.17.549374</a></p>
        <p class="abstract">Triple-Negative Breast Cancer (TNBC) is the most aggressive type of breast malignancy, with chemotherapy as the only mainstay treatment. TNBC patients have the worst prognoses as a large fraction of them do not achieve complete pathological response post-treatment and develop drug-resistant residual disease. Molecular mechanisms that trigger proliferation in drug-resistant chemo-residual TNBC cells are poorly understood due to the lack of investigations using clinically relevant cellular models. In this study, we have established TNBC subtype-specific cellular models of proliferating drug-tolerant persister (PDTP) cells using different classes of chemotherapeutic agents that recapitulate clinical residual disease with molecular heterogeneity. Analysis of total phospho-tyrosine signals in TNBC PDTPs showed an enhanced phospho-tyrosine content compared to the parental cells (PC). Interestingly, using mass-spectrometry analysis, we identified a dramatic decrease in epidermal growth factor receptor (EGFR) expression in the PDTPs, while the presence of hyper-activated tyrosine phosphorylation of EGFR compared to PC. Further, we show that EGFR has enhanced lysosomal trafficking in PDTPs with a concomitant increase in N-Myc Downstream Regulated-1 expression that co-localizes with EGFR to mediate receptor degradation. More surprisingly, we found that reduced protein levels of EGFR are coupled with a robust increase in Src family kinases, including Lyn and Fyn kinases, that creates a hyper-phosphorylation state of EGFR-Src tyrosine kinases axis in PDTPs and mediates downstream over-activation of STAT3, AKT and MAP kinases. Moreover, paclitaxel-derived PDTPs show increased sensitivity to EGFR TKI Gefitinib and its combination with paclitaxel selectively induced cell death in PDTP-P TNBC cells and 3D spheroids by strongly downregulating phosphorylation of EGFR-Src with concomitant downregulation of Lyn and Fyn tyrosine kinases. Collectively, this study identifies a unique hyper-phosphorylation cellular state of TNBC PDTPs established by switching of EGFR-Src family tyrosine kinases creating a vulnerability to EGFR TKI.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548697">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548697" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548697">
        <p class="paperTitle">In vivo single-cell CRISPR uncovers distinct TNF-alphaprograms in clonal expansion and tumorigenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548697" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548697" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Renz, P. F.; Ghoshdastider, U.; Baghai Sain, S.; Valdivia-Francia, F.; Khandekar, A.; Ormiston, M.; Bernasconi, M.; Kretz, J. A.; Lee, M.; Hyams, K.; Forny, M.; Pohly, M.; Ficht, X.; Ellis, S. J.; Moor, A. E.; Sendoel, A.</p>
        <p class="info">Score: 16.6, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548697' target='https://doi.org/10.1101/2023.07.13.548697'> 10.1101/2023.07.13.548697</a></p>
        <p class="abstract">The tumor evolution model posits that malignant transformation is preceded by randomly distributed driver mutations in cancer genes, which cause clonal expansions in phenotypically normal tissues. Although clonal expansions occur frequently in human epithelia and can remodel almost entire tissues, the mechanisms behind why only a small number of clones transform into malignant tumors remain enigmatic. Here, we develop an in vivo single-cell CRISPR strategy to systematically investigate tissue-wide clonal dynamics of the 150 most frequently mutated squamous cell carcinoma genes. We couple ultrasound-guided in utero lentiviral microinjections, single-cell RNA sequencing, guide capture and spatial transcriptomics to longitudinally monitor cell type-specific clonal expansions, document their underlying gene programs and contrast clonal expansions from tumor initiation. We uncover a TNF- signaling module that acts as a generalizable driver of clonal expansions in epithelial tissues. Conversely, during tumorigenesis, the TNF- signaling module is downregulated, and instead, we identify a subpopulation of invasive cancer cells that switch to an autocrine TNF- gene program. By analyzing clonally expanded perturbations and their frequency in tumors, we demonstrate that the autocrine TNF- gene program is associated with epithelial-mesenchymal transition (EMT) and is preexistent in a subpopulation of expanded epidermal stem cells, contributing to the predisposition for tumor initiation. Finally, we provide in vivo evidence that the epithelial TNF- gene program is sufficient to mediate invasive properties of epidermal stem cells and show that the TNF- signature correlates with shorter overall survival in human squamous cell carcinoma patients. Collectively, our study demonstrates the power of applying in vivo single-cell CRISPR screening to mammalian tissues and unveils distinct TNF- programs in tumor evolution. Understanding the biology of clonal expansions in phenotypically normal epithelia and the mechanisms governing their transformation will guide the development of novel strategies for early cancer detection and therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.549653">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.549653" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.549653">
        <p class="paperTitle">A novel ILK/STAT3 pathway controls plasticity in a neural stem cell model of glioblastoma.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.549653" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.549653" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Loftus, A. E. P.; Romano, M. S.; Nguyen Phuong, A.; Muir, M. T.; Dawson, J. C.; Avalle, L.; Douglas, A. T.; Mort, R. L.; Byron, A.; Carragher, N. O.; Pollard, S. M.; Brunton, V. G.; Frame, M. C.</p>
        <p class="info">Score: 5.4, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.549653' target='https://doi.org/10.1101/2023.07.19.549653'> 10.1101/2023.07.19.549653</a></p>
        <p class="abstract">Glioblastomas (GBM) are driven by malignant neural stem-like cells that display extensive heterogeneity and phenotypic plasticity, which drives tumour progression and therapeutic resistance. Here we show that the nodal extracellular matrix-cell adhesion protein integrin-linked kinase (ILK; a pseudokinase), is a key determinant of phenotypic plasticity and the mesenchymal-like, invasive cell state in mouse GBM stem cells. We found that a novel ILK-STAT3 signalling pathway is required for plasticity that enables the transition of GBM stem cells to an astrocyte-like state both in vitro and in vivo. GBM cells genetically depleted of ILK become predominantly stabilised in a transcriptionally-defined progenitor-like state that is characterised by lack of response to differentiation cues and constitutive proliferation. Loss of ILK or interference with STAT3 impairs differentiation potential, reducing phenotypic plasticity of tumour cell populations; additionally, ILK loss causes a mesenchymal- to epithelial-like morphological transition and suppression of malignancy-associated features. Our work defines ILK as a central regulator of multiple GBM phenotypes including phenotypic plasticity and mesenchymal state.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.18.549480">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.18.549480" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.18.549480">
        <p class="paperTitle">Transcriptional regulation of Dyskerin via canonical WNT signaling modulates sphingolipid biosynthesis and drives colorectal cancer.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.18.549480" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.18.549480" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nigam, S.; Khan, U. K.; Praveen, A.; Shendre, A.; Carskadon, S.; Kapoor, A.; Tewari, A.; Chandra, A.; Palanisamy, N.; Ateeq, B.</p>
        <p class="info">Score: 4.7, Published: 2023-07-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.18.549480' target='https://doi.org/10.1101/2023.07.18.549480'> 10.1101/2023.07.18.549480</a></p>
        <p class="abstract">Targeting EGFR has been effective in RAS/RAF wild-type colorectal cancer (CRC) patients. However, residual tumor relapses, necessitating the importance of biomarker-guided novel therapeutics. We show elevated DKC1 in [~]88% of CRC patients with poor recurrence-free survival. Clinically, DKC1-positive patients exhibit similarity with CMS2 class, the canonical subtype with active WNT signaling. We show functional significance of DKC1 in cell proliferation, stemness, DNA repair, and survival. Further, mice bearing DKC1 knockdown xenografts show [~]81% reduction in tumor burden. Mechanistically, WNT/{beta}-catenin signaling orchestrates DKC1 expression, then, DKC1/SOX2 complex regulates SGPP2, modulating sphingolipids metabolism. Downregulation of DKC1 in CRC lead to reduced SGPP2 levels leading to dysregulation of sphingolipid biosynthesis. Of note, DKC1-high CRC patients show accumulation of ceramides, namely C23 and C24, signifying their utility in diagnosis. Collectively, we delineate the mechanistic circuitry involved in DKC1-mediated CRC progression, propose ceramides as biomarker, and underscore WNT-based therapeutics for DKC1-positive patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.14.549079">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.14.549079" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.14.549079">
        <p class="paperTitle">Local Adaptive Mapping of Karyotype Fitness Landscapes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.14.549079" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.14.549079" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Beck, R. J.; Andor, N.</p>
        <p class="info">Score: 5.1, Published: 2023-07-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.14.549079' target='https://doi.org/10.1101/2023.07.14.549079'> 10.1101/2023.07.14.549079</a></p>
        <p class="abstract">Aneuploidy occurs in most solid tumors and has the potential to dramatically modify cellular phenotype and fitness. Despite the importance of aneuploidy in tumor evolution, quantitative understanding of the evolutionary landscape of aneuploidy is lacking. To address this question, we developed a method to infer the fitness landscape of either arm-level or whole-chromosome level karyotypes. Our method takes longitudinal single cell sequencing data from an evolving cell population as input, then estimates the fitness of thousands of karyotypes located near to the input data in karyotype space. The predictive ability of the method was validated using artificial data generated from an agent based model, as well as data from a selection of in vitro and in vivo passaged cell lines. We applied our pipeline to an in vitro dataset of serially passaged cells and - based on topological analysis of the fitness landscape around diploid and tetraploid karyotypes - found support for the hypothesis that whole genome doubling benefits tumour cells by conferring robustness against missegregations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550265" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550265">
        <p class="paperTitle">An integrative proteomics approach identifies tyrosine kinase KIT as a novel therapeutic target for SPINK1-positive prostate cancer.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manzar, N.; Khan, U. K.; Goel, A.; Carskadon, S.; Gupta, N.; Palanisamy, N.; Ateeq, B.</p>
        <p class="info">Score: 4.5, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550265' target='https://doi.org/10.1101/2023.07.24.550265'> 10.1101/2023.07.24.550265</a></p>
        <p class="abstract">Elevated Serine Peptidase Inhibitor, Kazal type 1 (SPINK1) levels in ~10-25% of prostate cancer (PCa) patients associate with aggressive phenotype, for which there are limited treatment choices and dismal clinical outcomes. Using an integrative proteomics approach involving label-free phosphoproteome and proteome profiling, we delineated the downstream signaling pathways involved in SPINK1-mediated tumorigenesis in PCa, and identified tyrosine kinase KIT as a highly enriched kinase. Furthermore, high to moderate levels of KIT expression was detected in ~85% of SPINK1-positive PCa specimens. KIT signaling regulates SPINK1-associated oncogenesis, and treatment with KIT inhibitor reduces tumor growth and distant metastases in preclinical mice models. Mechanistically, KIT signaling modulates WNT/beta-catenin pathway and confers stemness-related features in PCa. Notably, inhibiting KIT signaling restores AR/REST levels, forming a feedback loop enabling SPINK1 repression. Overall, we uncover the role of KIT signaling downstream of SPINK1 in maintaining lineage plasticity and provide new treatment modalities for advanced-stage SPINK1-positive subtype.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.19.548825">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.19.548825" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.19.548825">
        <p class="paperTitle">Comprehensive multi-platform tyrosine kinase profiling reveals novel actionable FGFR aberrations across pediatric and AYA sarcomas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.19.548825" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.19.548825" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fordham, A. M.; Brown, L. M.; Mayoh, C.; Salib, A.; Barger, Z. A.; Wong, M.; Sian, T. C. C. L. K.; Xie, J.; Gunther, K.; Trebilcock, P.; Terry, R. L.; Barahona, P.; Ajuyah, P.; Sherstyuk, A.; Avila, A.; Cadiz, R.; Perkins, C. M.; Gifford, A. J.; Mao, J.; Zhao, A.; O&#39;Regan, L. P.; Gorgels, D.; Lau, L. M.; Ziegler, D. S.; Haber, M.; Tyrrell, V.; Lock, R. B.; Cowley, M.; Nicholls, W.; Daly, R. J.; Ekert, P. G.; Fleuren, E. D.</p>
        <p class="info">Score: 3.7, Published: 2023-07-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.19.548825' target='https://doi.org/10.1101/2023.07.19.548825'> 10.1101/2023.07.19.548825</a></p>
        <p class="abstract">No targeted agents are approved for pediatric sarcomas. Tyrosine kinase (TK) inhibitors represent attractive therapeutic candidates, however, beyond rare TK-activating fusions or mutations, predictive biomarkers are lacking. RNA overexpression of TKs is more commonly observed in pediatric sarcomas, however, an unresolved question is when upregulated TK expression is associated with kinase activation and signaling dependence. We explored the TK molecular landscape of 107 sarcoma patients from the ZERO Childhood Cancer precision medicine program using whole genomic and transcriptomic sequencing. Phosphoproteomic analyses of tyrosine phosphorylation (pY) and functional in vitro and in vivo assays were also performed in cell lines and patient-derived xenografts (PDXs). Our integrated analysis shows that although novel genomic driver lesions are rare, they are present and therapeutically actionable in selected patients as exemplified by a novel LSM1-FGFR1 fusion identified in an osteosarcoma patient. We further show that in certain contexts, TK expression data can be used to indicate TK pathway activity and predict TK-inhibitor sensitivity. We exemplify the utility of FGFR-inhibitors in PAX3-FOXO1 fusion-positive rhabdomyosarcomas (FP-RMS) mediated by high FGFR4 and FGF8 RNA expression levels, and overt activation of FGFR4 (FGFR4_pY). We demonstrate marked tumor growth inhibition in all FP-RMS PDXs treated with single agent FGF401 (FGFR4-specific inhibitor) and single agent lenvatinib (multi-kinase FGFR-inhibitor). Clinical benefit of lenvatinib in a relapsed metastatic FP-RMS patient further exemplifies that FGFR-inhibitors deserve additional investigation in FP-RMS patients.

Statement of significanceOur multi-omic interrogation of sarcomas in the ZERO Childhood Cancer program illustrates how an RNA-expression biomarker signature (FGFR4&#43;/FGF8&#43;) in association with FGFR4 activation identifies that PAX3-FOXO1-positive rhabdomyosarcoma patients could benefit from FGFR-inhibitors.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
