<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Integrative spatial analysis reveals a multi-layered organization of glioblastoma
Authors: Greenwald, A. C.; Darnell, N. G.; Hoefflin, R.; Simkin, D.; Gonzalez-Castro, L. N.; Mount, C.; Hirsch, D.; Nomura, M.; Talpir, T.; Kedmi, M.; Goliand, I.; Medici, G.; Li, B.; Keren-Shaul, H.; Weller, M.; Addadi, Y.; Neidert, M. C.; Suva, M. L.; Tirosh, I.
Score: 78.3, Published: 2023-07-08 DOI: 10.1101/2023.07.06.547924
Glioma contains malignant cells in diverse states. Hypoxic regions are associated with a unique histology of pseudopalisading cells, while other regions appear to have limited histological organization, reflecting the diffuse nature of glioma cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Integrative spatial analysis reveals a multi-layered organization of glioblastoma
Authors: Greenwald, A. C.; Darnell, N. G.; Hoefflin, R.; Simkin, D.; Gonzalez-Castro, L. N.; Mount, C.; Hirsch, D.; Nomura, M.; Talpir, T.; Kedmi, M.; Goliand, I.; Medici, G.; Li, B.; Keren-Shaul, H.; Weller, M.; Addadi, Y.; Neidert, M. C.; Suva, M. L.; Tirosh, I.
Score: 78.3, Published: 2023-07-08 DOI: 10.1101/2023.07.06.547924
Glioma contains malignant cells in diverse states. Hypoxic regions are associated with a unique histology of pseudopalisading cells, while other regions appear to have limited histological organization, reflecting the diffuse nature of glioma cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-16T10:39:40+00:00" />
<meta property="article:modified_time" content="2023-07-16T10:39:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Integrative spatial analysis reveals a multi-layered organization of glioblastoma
Authors: Greenwald, A. C.; Darnell, N. G.; Hoefflin, R.; Simkin, D.; Gonzalez-Castro, L. N.; Mount, C.; Hirsch, D.; Nomura, M.; Talpir, T.; Kedmi, M.; Goliand, I.; Medici, G.; Li, B.; Keren-Shaul, H.; Weller, M.; Addadi, Y.; Neidert, M. C.; Suva, M. L.; Tirosh, I.
Score: 78.3, Published: 2023-07-08 DOI: 10.1101/2023.07.06.547924
Glioma contains malignant cells in diverse states. Hypoxic regions are associated with a unique histology of pseudopalisading cells, while other regions appear to have limited histological organization, reflecting the diffuse nature of glioma cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Integrative spatial analysis reveals a multi-layered organization of glioblastoma\nAuthors: Greenwald, A. C.; Darnell, N. G.; Hoefflin, R.; Simkin, D.; Gonzalez-Castro, L. N.; Mount, C.; Hirsch, D.; Nomura, M.; Talpir, T.; Kedmi, M.; Goliand, I.; Medici, G.; Li, B.; Keren-Shaul, H.; Weller, M.; Addadi, Y.; Neidert, M. C.; Suva, M. L.; Tirosh, I.\nScore: 78.3, Published: 2023-07-08 DOI: 10.1101/2023.07.06.547924\nGlioma contains malignant cells in diverse states. Hypoxic regions are associated with a unique histology of pseudopalisading cells, while other regions appear to have limited histological organization, reflecting the diffuse nature of glioma cells.",
  "keywords": [
    
  ],
  "articleBody": " Integrative spatial analysis reveals a multi-layered organization of glioblastoma\nAuthors: Greenwald, A. C.; Darnell, N. G.; Hoefflin, R.; Simkin, D.; Gonzalez-Castro, L. N.; Mount, C.; Hirsch, D.; Nomura, M.; Talpir, T.; Kedmi, M.; Goliand, I.; Medici, G.; Li, B.; Keren-Shaul, H.; Weller, M.; Addadi, Y.; Neidert, M. C.; Suva, M. L.; Tirosh, I.\nScore: 78.3, Published: 2023-07-08 DOI: 10.1101/2023.07.06.547924\nGlioma contains malignant cells in diverse states. Hypoxic regions are associated with a unique histology of pseudopalisading cells, while other regions appear to have limited histological organization, reflecting the diffuse nature of glioma cells. Here, we combine spatial transcriptomics with spatial proteomics and novel computational approaches to define glioma cellular states at high granularity and uncover their organization. We find three prominent modes of cellular organization. First, cells in any given state tend to be spatially clustered, such that tumors are composed of small local environments that are each typically enriched with one major cellular state. Second, specific pairs of states preferentially reside in proximity across multiple scales. Despite the unique composition of each tumor, this pairing of states remains largely consistent across tumors. Third, the pairwise interactions that we detect collectively define a global architecture composed of five layers. Hypoxia appears to drive this 5-layered organization, as it is both associated with unique states of surrounding cells and with a long-range organization that extends from the hypoxic core to the infiltrative edge of the tumor. Accordingly, tumor regions distant from any hypoxic foci and tumors that lack hypoxia such as IDH-mutant glioma are less organized. In summary, we provide a conceptual framework for the organization of gliomas at the resolution of cellular states and highlight the role of hypoxia as a long-range tissue organizer.\nOncogene EVI1 Drives Acute Myeloid Leukemia Via a Targetable Interaction with CTBP2\nAuthors: Pastoors, D.; Havermans, M.; Mulet-Lazaro, R.; Brian, D.; Noort, W.; Grasel, J.; Hoogenboezem, R.; Smeenk, L.; Demmers, J. A.; Enver, T.; Groen, R. W.; Bindels, E.; Delwel, R.\nScore: 23.1, Published: 2023-07-12 DOI: 10.1101/2023.07.11.548358\nAcute myeloid leukemia (AML) driven by the activation of EVI1 due to chromosome 3q26/MECOM rearrangements is incurable. Since transcription factors like EVI1 are notoriously hard to target, insight into the mechanism by which EVI1 drives myeloid transformation could provide alternative avenues for therapy. Applying protein folding predictions combined with proteomics technologies we demonstrate interaction with CTBP1 and CTBP2 via a single PLDLS motif in EVI1 is indispensable for leukemic transformation. A 4x PLDLS repeat construct outcompetes binding of EVI1 to CTBP1 and CTBP2 and inhibits proliferation of 3q26/MECOM rearranged AML in vitro and in xenotransplant models. This proof-of-concept study opens the possibility to target one of the most incurable forms of AML with specific EVI1-CTBP inhibitors. Our findings may also have important implications for other tumour types with aberrant expression of EVI1 or for cancers transformed by other CTBP1/2 dependent oncogenic transcription factors.\nChemical alkylation of Asp12 enables mutant selective targeting of K-Ras(G12D)\nAuthors: Zhang, Z.; Zheng, Q.; Guiley, K. Z.; Shokat, K.\nScore: 18.2, Published: 2023-07-02 DOI: 10.1101/2023.07.01.547359\nK-Ras is the most commonly mutated oncogene in human cancer, yet direct small-molecule targeting of K-Ras mutants has been mostly unsuccessful until recently. The discovery of an allosteric pocket under Switch-II with covalent cysteine-crosslinking molecules has allowed for the development of targeted therapies that selectively engage the highly reactive acquired cysteine in the K-Ras(G12C) mutation without affecting the wild-type protein. Sotorasib and adagrasib, two advanced Switch-II Pocket inhibitors, have received FDA approval to treat K-Ras(G12C)-driven non-small cell lung cancer. However, the most frequent K-Ras mutation G12D particularly prevalent in pancreatic ductal adenocarcinoma has remained untargetable with covalent drugs due to the poor nucleophilicity of the somatic aspartate residue. Here we present a set of {beta}-lactone-based electrophiles which exploit ring strain to crosslink K-Ras(G12D) at the mutant aspartate to form stable covalent complexes in cells, effectively blocking Ras-effector interaction and downstream signaling. Structural insights from x-ray crystallography and exploitation of the differential chemoselectivity and stereoelectronic requirements for attack of the {beta}-lactone electrophile allowed development of a substituted {beta}-lactone which resisted attack by aqueous buffer but enabled rapid attack by aspartic acid-12 in K-Ras. Our results demonstrate the rational design of covalent inhibitors to target a non-catalytic carboxylic acid side chain in K-Ras(G12D) which has resisted traditional drug discovery efforts.\nIn Vivo Monitoring of Cellular Senescence by Photoacoustic and Fluorescence Imaging Utilizing a Nanostructured Organic Probe\nAuthors: Baker, A. G.; Ou, H.-L.; Hartono, M.; Popov, A. B.; Brown, E. L.; Joseph, J.; Golinska, M.; Sanghera, C.; Gonzalez-Gualda, E.; Macias, D.; Else, T. R.; Greer, H. F.; Vernet, A.; Bohndiek, S. E.; Fruk, L.; Munoz-Espin, D.\nScore: 14.6, Published: 2023-07-13 DOI: 10.1101/2023.07.12.548691\nSenescent cells accumulate in multiple age-related disorders, including cancer, exacerbating the pathological manifestations, and the eradication of these cells has emerged as a promising therapeutic strategy. Despite the impact of senescence in diseases, the development of tools to monitor the senescent burden in vivo remains a challenge due to their suboptimal specificity, translatability, and tissue penetrance. Here, we have designed a nanostructured organic probe (NanoJaggs) based on biocompatible indocyanine green dye (ICG) building blocks forming J-aggregates, which possess distinct spectral properties allowing both fluorescence and photoacoustic tomography (PAT) detection. We show that NanoJaggs are taken up by an active process of endocytosis and exhibit selective accumulation at the lysosomal compartment in several in vitro models for senescence. Finally, NanoJagg probe is validated in two in vivo studies including live PAT imaging and shows remarkable specificity to tumours with chemotherapy-induced senescence compared to untreated proliferative tumors. In vitro, ex vivo and in vivo all indicate that NanoJaggs are a clinically translatable tool for detection of senescence and their robust PAT signal makes them suitable for longitudinal monitoring of the senescent burden in solid tumors after chemo or radiotherapy.\nIntegrative analyses uncover mechanisms by which aging drives B cell lymphoma\nAuthors: Castro, J. P.; Shindyapina, A. V.; Barbieri, A.; Ying, K.; Strelkova, O. S.; Paulo, J. A.; Tyshkovskiy, A.; Meinl, R.; Kerepesi, C.; Petrashen, A. P.; Mariotti, M.; Meer, M.; Hu, Y.; Karamyshev, A.; Losyev, G.; Indzhykulian, A. A.; Gygi, S. P.; Sedivy, J. M.; Manis, J. P.; Gladyshev, V. N.\nScore: 28.3, Published: 2023-07-11 DOI: 10.1101/2021.02.23.432500\nWhile cancer is an age-related disease, many cancer studies utilize younger animal models. Here, we uncover how a cancer, B-cell lymphoma, develops as a consequence of a naturally aged system. We show that this malignancy is associated with increased cell size, splenomegaly, and a newly discovered age-associated clonal B-cell (ACBC) population. Driven by exogenous c-Myc activation, hypermethylated promoters and somatic mutations, ACBC cells clonally expand independent of germinal centers (IgM+) and show increased biological age and hypomethylation in partially methylated domains related to mitotic solo-CpGs. Epigenetic changes in transformed mouse B cells are enriched for changes observed in human B-cell lymphomas. Mechanistically, the data suggest that cancerous ACBC cells originate from age-associated B cells, in part involving CD22 protein signaling fostered by the aging microenvironment. Transplantation assays demonstrate that ACBC evolve to become self-sufficient and support malignancy when transferred into young recipients. Inhibition of mTOR or c-Myc in old mice attenuates premalignant changes in B cells during aging and emerges as a therapeutic strategy to delay the onset of age-related lymphoma. Together, we show how aging contributes to B-cell lymphoma through a previously unrecognized mechanism involving cell-intrinsic changes and the aged microenvironment, characterize a model that captures the origin and progression of spontaneous cancer during aging and identify candidate interventions against age-associated lymphoma.\nOn minimising tumoural growth under treatment resistance\nAuthors: Fischer, M. M.; Bluethgen, N.\nScore: 10.3, Published: 2023-07-04 DOI: 10.1101/2023.07.04.547696\nAO_SCPLOWBSTRACTC_SCPLOWDrug resistance is a major challenge for curative cancer treatment, representing the main reason of death in patients. Evolutionary biology suggests pauses between treatment rounds as a way to delay or even avoid resistance emergence. Indeed, this approach has already shown promising preclinical and early clinical results, and stimulated the development of mathematical models for finding optimal treatment protocols. Due to their complexity, however, these models do not lend themself to a rigorous mathematical analysis, hence so far clinical recommendations generally relied on numerical simulations and ad-hoc heuristics. Here, we derive two mathematical models describing tumour growth under genetic and epigenetic treatment resistance, respectively, which are simple enough for a complete analytical investigation. First, we find key differences in response to treatment protocols between the two modes of resistance. Second, we identify the optimal treatment protocol which leads to the largest possible tumour shrinkage rate. Third, we fit the \"epigenetic model\" to previously published xenograft experiment data, finding excellent agreement, underscoring the biological validity of our approach. Finally, we use the fitted model to calculate the optimal treatment protocol for this specific experiment, which we demonstrate to cause curative treatment, making it superior to previous approaches which generally aimed at stabilising tumour burden. Overall, our approach underscores the usefulness of simple mathematical models and their analytical examination, and we anticipate our findings to guide future preclinical and, ultimately, clinical research in optimising treatment regimes.\nFully Automated Sequential Immunofluorescence (seqIF) for Hyperplex Spatial Proteomics\nAuthors: Rivest, F.; Eroglu, D.; Pelz, B.; Kowal, J.; Kehren, A.; Procopio, M. G.; Bordignon, P.; Peres, E.; Ammann, M.; Dorel, E.; Scalmazzi, S.; Bruno, L.; Ruegg, M.; Campargue, G.; Casqueiro, G.; Arn, L.; Fischer, J.; Brajkovic, S.; Joris, P.; Cassano, M.; Dupouy, D.\nScore: 6.8, Published: 2023-07-10 DOI: 10.1101/2023.07.07.548135\nTissues are complex environments where different cell types are in constant interaction with each other and with non-cellular components. Preserving the spatial context during proteomics analyses of tissue samples has become an important objective for different applications, one of the most important being the investigation of the tumor microenvironment. Here, we describe a multiplexed protein biomarker detection method on the COMET instrument, coined sequential ImmunoFluorescence (seqIF). The fully automated method uses successive applications of antibody incubation and elution, and in-situ imaging enabled by an integrated microscope and a microfluidic chip that provides optimized optical access to the sample. We show seqIF data on different sample types such as tumor and healthy tissue, including 40-plex on a single tissue section that is obtained in less than 24 hours, using off-the-shelf antibodies. We also present extensive characterization of the developed method, including elution efficiency, epitope stability, repeatability and reproducibility, signal uniformity, and dynamic range, in addition to marker and panel optimization strategies. The streamlined workflow using off-the-shelf antibodies, data quality enabling quantitative downstream analysis, and ease of reaching hyperplex levels make seqIF suitable for immune-oncology research and other disciplines requiring spatial analysis, paving the way for its adoption in clinical settings.\nMethNet: a robust approach to identify regulatory hubs and their distal targets in cancer\nAuthors: Sakellaropoulos, T.; Do, C.; Jiang, G.; Cova, G.; Meyn, P.; Dimartino, D.; Ramaswami, S.; Heguy, A.; Tsirigos, A.; Skok, J. A.\nScore: 6.7, Published: 2023-07-10 DOI: 10.1101/2023.07.07.548142\nAberrations in the capacity of DNA/chromatin modifiers and transcription factors to bind non-coding regions can lead to changes in gene regulation and impact disease phenotypes. However, identifying distal regulatory elements and connecting them with their target genes remains challenging. Here, we present MethNet, a pipeline that integrates large-scale DNA methylation and gene expression data across multiple cancers, to uncover novel cis regulatory elements (CREs) in a 1Mb region around every promoter in the genome. MethNet identifies clusters of highly ranked CREs, referred to as hubs, which contribute to the regulation of multiple genes and significantly affect patient survival. Promoter-capture Hi-C confirmed that highly ranked associations involve physical interactions between CREs and their gene targets, and CRISPRi based scRNA Perturb-seq validated the functional impact of CREs. Thus, MethNet-identified CREs represent a valuable resource for unraveling complex mechanisms underlying gene expression, and for prioritizing the verification of predicted non-coding disease hotspots.\nIn vivo single-cell CRISPR uncovers distinct TNF-alphaprograms in clonal expansion and tumorigenesis\nAuthors: Renz, P. F.; Ghoshdastider, U.; Baghai Sain, S.; Valdivia-Francia, F.; Khandekar, A.; Ormiston, M.; Bernasconi, M.; Kretz, J. A.; Lee, M.; Hyams, K.; Forny, M.; Pohly, M.; Ficht, X.; Ellis, S. J.; Moor, A. E.; Sendoel, A.\nScore: 6.1, Published: 2023-07-14 DOI: 10.1101/2023.07.13.548697\nThe tumor evolution model posits that malignant transformation is preceded by randomly distributed driver mutations in cancer genes, which cause clonal expansions in phenotypically normal tissues. Although clonal expansions occur frequently in human epithelia and can remodel almost entire tissues, the mechanisms behind why only a small number of clones transform into malignant tumors remain enigmatic. Here, we develop an in vivo single-cell CRISPR strategy to systematically investigate tissue-wide clonal dynamics of the 150 most frequently mutated squamous cell carcinoma genes. We couple ultrasound-guided in utero lentiviral microinjections, single-cell RNA sequencing, guide capture and spatial transcriptomics to longitudinally monitor cell type-specific clonal expansions, document their underlying gene programs and contrast clonal expansions from tumor initiation. We uncover a TNF-alpha signaling module that acts as a generalizable driver of clonal expansions in epithelial tissues. Conversely, during tumorigenesis, the TNF-alpha signaling module is downregulated, and instead, we identify a subpopulation of invasive cancer cells that switch to an autocrine TNF-alpha gene program. By analyzing clonally expanded perturbations and their frequency in tumors, we demonstrate that the autocrine TNF- gene program is associated with epithelial-mesenchymal transition (EMT) and is preexistent in a subpopulation of expanded epidermal stem cells, contributing to the predisposition for tumor initiation. Finally, we provide in vivo evidence that the epithelial TNF-alpha gene program is sufficient to mediate invasive properties of epidermal stem cells and show that the TNF-alpha signature correlates with shorter overall survival in human squamous cell carcinoma patients. Collectively, our study demonstrates the power of applying in vivo single-cell CRISPR screening to mammalian tissues and unveils distinct TNF-alpha programs in tumor evolution. Understanding the biology of clonal expansions in phenotypically normal epithelia and the mechanisms governing their transformation will guide the development of novel strategies for early cancer detection and therapy.\nCytokine storm mitigation for exogenous immune agonists\nAuthors: Kareva, I.; Gevertz, J. L.\nScore: 7.6, Published: 2023-07-07 DOI: 10.1101/2023.07.07.548130\nCytokine storm is a life-threatening inflammatory response characterized by hyperactivation of the immune system. It can be caused by various therapies, auto-immune conditions, or pathogens, such as respiratory syndrome coronavirus 2 (SARS-CoV-2) which causes coronavirus disease COVID-19. Here we propose a conceptual mathematical model describing the phenomenology of cytokine-immune interactions when a tumor is treated by an exogenous immune cell agonist which has the potential to cause a cytokine storm, such as CAR T cell therapy. Numerical simulations reveal that as a function of just two model parameters, the same drug dose and regimen could result in one of four outcomes: treatment success without a storm, treatment success with a storm, treatment failure without a storm, and treatment failure with a storm. We then explore a scenario in which tumor control is accompanied by a storm and ask if it is possible to modulate the duration and frequency of drug administration (without changing the cumulative dose) in order to preserve efficacy while preventing the storm. Simulations reveal existence of a \"sweet spot\" in protocol space (number versus spacing of doses) for which tumor control is achieved without inducing a cytokine storm. This theoretical model, which contains a number of parameters that can be estimated experimentally, contributes to our understanding of what triggers a cytokine storm, and how the likelihood of its occurrence can be mitigated.\n",
  "wordCount" : "2533",
  "inLanguage": "en",
  "datePublished": "2023-07-16T10:39:40Z",
  "dateModified": "2023-07-16T10:39:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on July 16, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547924">
        <p class="paperTitle">Integrative spatial analysis reveals a multi-layered organization of glioblastoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Greenwald, A. C.; Darnell, N. G.; Hoefflin, R.; Simkin, D.; Gonzalez-Castro, L. N.; Mount, C.; Hirsch, D.; Nomura, M.; Talpir, T.; Kedmi, M.; Goliand, I.; Medici, G.; Li, B.; Keren-Shaul, H.; Weller, M.; Addadi, Y.; Neidert, M. C.; Suva, M. L.; Tirosh, I.</p>
        <p class="info">Score: 78.3, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547924' target='https://doi.org/10.1101/2023.07.06.547924'> 10.1101/2023.07.06.547924</a></p>
        <p class="abstract">Glioma contains malignant cells in diverse states. Hypoxic regions are associated with a unique histology of pseudopalisading cells, while other regions appear to have limited histological organization, reflecting the diffuse nature of glioma cells. Here, we combine spatial transcriptomics with spatial proteomics and novel computational approaches to define glioma cellular states at high granularity and uncover their organization. We find three prominent modes of cellular organization. First, cells in any given state tend to be spatially clustered, such that tumors are composed of small local environments that are each typically enriched with one major cellular state. Second, specific pairs of states preferentially reside in proximity across multiple scales. Despite the unique composition of each tumor, this pairing of states remains largely consistent across tumors. Third, the pairwise interactions that we detect collectively define a global architecture composed of five layers. Hypoxia appears to drive this 5-layered organization, as it is both associated with unique states of surrounding cells and with a long-range organization that extends from the hypoxic core to the infiltrative edge of the tumor. Accordingly, tumor regions distant from any hypoxic foci and tumors that lack hypoxia such as IDH-mutant glioma are less organized. In summary, we provide a conceptual framework for the organization of gliomas at the resolution of cellular states and highlight the role of hypoxia as a long-range tissue organizer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.11.548358">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.11.548358" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.11.548358">
        <p class="paperTitle">Oncogene EVI1 Drives Acute Myeloid Leukemia Via a Targetable Interaction with CTBP2</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.11.548358" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.11.548358" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pastoors, D.; Havermans, M.; Mulet-Lazaro, R.; Brian, D.; Noort, W.; Grasel, J.; Hoogenboezem, R.; Smeenk, L.; Demmers, J. A.; Enver, T.; Groen, R. W.; Bindels, E.; Delwel, R.</p>
        <p class="info">Score: 23.1, Published: 2023-07-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.11.548358' target='https://doi.org/10.1101/2023.07.11.548358'> 10.1101/2023.07.11.548358</a></p>
        <p class="abstract">Acute myeloid leukemia (AML) driven by the activation of EVI1 due to chromosome 3q26/MECOM rearrangements is incurable. Since transcription factors like EVI1 are notoriously hard to target, insight into the mechanism by which EVI1 drives myeloid transformation could provide alternative avenues for therapy. Applying protein folding predictions combined with proteomics technologies we demonstrate interaction with CTBP1 and CTBP2 via a single PLDLS motif in EVI1 is indispensable for leukemic transformation. A 4x PLDLS repeat construct outcompetes binding of EVI1 to CTBP1 and CTBP2 and inhibits proliferation of 3q26/MECOM rearranged AML in vitro and in xenotransplant models. This proof-of-concept study opens the possibility to target one of the most incurable forms of AML with specific EVI1-CTBP inhibitors. Our findings may also have important implications for other tumour types with aberrant expression of EVI1 or for cancers transformed by other CTBP1/2 dependent oncogenic transcription factors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.01.547359">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.01.547359" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.01.547359">
        <p class="paperTitle">Chemical alkylation of Asp12 enables mutant selective targeting of K-Ras(G12D)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.01.547359" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.01.547359" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Z.; Zheng, Q.; Guiley, K. Z.; Shokat, K.</p>
        <p class="info">Score: 18.2, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.01.547359' target='https://doi.org/10.1101/2023.07.01.547359'> 10.1101/2023.07.01.547359</a></p>
        <p class="abstract">K-Ras is the most commonly mutated oncogene in human cancer, yet direct small-molecule targeting of K-Ras mutants has been mostly unsuccessful until recently. The discovery of an allosteric pocket under Switch-II with covalent cysteine-crosslinking molecules has allowed for the development of targeted therapies that selectively engage the highly reactive acquired cysteine in the K-Ras(G12C) mutation without affecting the wild-type protein. Sotorasib and adagrasib, two advanced Switch-II Pocket inhibitors, have received FDA approval to treat K-Ras(G12C)-driven non-small cell lung cancer. However, the most frequent K-Ras mutation G12D particularly prevalent in pancreatic ductal adenocarcinoma has remained untargetable with covalent drugs due to the poor nucleophilicity of the somatic aspartate residue. Here we present a set of {beta}-lactone-based electrophiles which exploit ring strain to crosslink K-Ras(G12D) at the mutant aspartate to form stable covalent complexes in cells, effectively blocking Ras-effector interaction and downstream signaling. Structural insights from x-ray crystallography and exploitation of the differential chemoselectivity and stereoelectronic requirements for attack of the {beta}-lactone electrophile allowed development of a substituted {beta}-lactone which resisted attack by aqueous buffer but enabled rapid attack by aspartic acid-12 in K-Ras. Our results demonstrate the rational design of covalent inhibitors to target a non-catalytic carboxylic acid side chain in K-Ras(G12D) which has resisted traditional drug discovery efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.12.548691">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.12.548691" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.12.548691">
        <p class="paperTitle">In Vivo Monitoring of Cellular Senescence by Photoacoustic and Fluorescence Imaging Utilizing a Nanostructured Organic Probe</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.12.548691" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.12.548691" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Baker, A. G.; Ou, H.-L.; Hartono, M.; Popov, A. B.; Brown, E. L.; Joseph, J.; Golinska, M.; Sanghera, C.; Gonzalez-Gualda, E.; Macias, D.; Else, T. R.; Greer, H. F.; Vernet, A.; Bohndiek, S. E.; Fruk, L.; Munoz-Espin, D.</p>
        <p class="info">Score: 14.6, Published: 2023-07-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.12.548691' target='https://doi.org/10.1101/2023.07.12.548691'> 10.1101/2023.07.12.548691</a></p>
        <p class="abstract">Senescent cells accumulate in multiple age-related disorders, including cancer, exacerbating the pathological manifestations, and the eradication of these cells has emerged as a promising therapeutic strategy. Despite the impact of senescence in diseases, the development of tools to monitor the senescent burden in vivo remains a challenge due to their suboptimal specificity, translatability, and tissue penetrance. Here, we have designed a nanostructured organic probe (NanoJaggs) based on biocompatible indocyanine green dye (ICG) building blocks forming J-aggregates, which possess distinct spectral properties allowing both fluorescence and photoacoustic tomography (PAT) detection. We show that NanoJaggs are taken up by an active process of endocytosis and exhibit selective accumulation at the lysosomal compartment in several in vitro models for senescence. Finally, NanoJagg probe is validated in two in vivo studies including live PAT imaging and shows remarkable specificity to tumours with chemotherapy-induced senescence compared to untreated proliferative tumors. In vitro, ex vivo and in vivo all indicate that NanoJaggs are a clinically translatable tool for detection of senescence and their robust PAT signal makes them suitable for longitudinal monitoring of the senescent burden in solid tumors after chemo or radiotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2021.02.23.432500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2021.02.23.432500" aria-expanded="false" aria-controls="flush-collapse10.1101/2021.02.23.432500">
        <p class="paperTitle">Integrative analyses uncover mechanisms by which aging drives B cell lymphoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2021.02.23.432500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2021.02.23.432500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Castro, J. P.; Shindyapina, A. V.; Barbieri, A.; Ying, K.; Strelkova, O. S.; Paulo, J. A.; Tyshkovskiy, A.; Meinl, R.; Kerepesi, C.; Petrashen, A. P.; Mariotti, M.; Meer, M.; Hu, Y.; Karamyshev, A.; Losyev, G.; Indzhykulian, A. A.; Gygi, S. P.; Sedivy, J. M.; Manis, J. P.; Gladyshev, V. N.</p>
        <p class="info">Score: 28.3, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2021.02.23.432500' target='https://doi.org/10.1101/2021.02.23.432500'> 10.1101/2021.02.23.432500</a></p>
        <p class="abstract">While cancer is an age-related disease, many cancer studies utilize younger animal models. Here, we uncover how a cancer, B-cell lymphoma, develops as a consequence of a naturally aged system. We show that this malignancy is associated with increased cell size, splenomegaly, and a newly discovered age-associated clonal B-cell (ACBC) population. Driven by exogenous c-Myc activation, hypermethylated promoters and somatic mutations, ACBC cells clonally expand independent of germinal centers (IgM&#43;) and show increased biological age and hypomethylation in partially methylated domains related to mitotic solo-CpGs. Epigenetic changes in transformed mouse B cells are enriched for changes observed in human B-cell lymphomas. Mechanistically, the data suggest that cancerous ACBC cells originate from age-associated B cells, in part involving CD22 protein signaling fostered by the aging microenvironment. Transplantation assays demonstrate that ACBC evolve to become self-sufficient and support malignancy when transferred into young recipients. Inhibition of mTOR or c-Myc in old mice attenuates premalignant changes in B cells during aging and emerges as a therapeutic strategy to delay the onset of age-related lymphoma. Together, we show how aging contributes to B-cell lymphoma through a previously unrecognized mechanism involving cell-intrinsic changes and the aged microenvironment, characterize a model that captures the origin and progression of spontaneous cancer during aging and identify candidate interventions against age-associated lymphoma.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547696">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547696" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547696">
        <p class="paperTitle">On minimising tumoural growth under treatment resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547696" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547696" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fischer, M. M.; Bluethgen, N.</p>
        <p class="info">Score: 10.3, Published: 2023-07-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547696' target='https://doi.org/10.1101/2023.07.04.547696'> 10.1101/2023.07.04.547696</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWDrug resistance is a major challenge for curative cancer treatment, representing the main reason of death in patients. Evolutionary biology suggests pauses between treatment rounds as a way to delay or even avoid resistance emergence. Indeed, this approach has already shown promising preclinical and early clinical results, and stimulated the development of mathematical models for finding optimal treatment protocols. Due to their complexity, however, these models do not lend themself to a rigorous mathematical analysis, hence so far clinical recommendations generally relied on numerical simulations and ad-hoc heuristics. Here, we derive two mathematical models describing tumour growth under genetic and epigenetic treatment resistance, respectively, which are simple enough for a complete analytical investigation. First, we find key differences in response to treatment protocols between the two modes of resistance. Second, we identify the optimal treatment protocol which leads to the largest possible tumour shrinkage rate. Third, we fit the &#34;epigenetic model&#34; to previously published xenograft experiment data, finding excellent agreement, underscoring the biological validity of our approach. Finally, we use the fitted model to calculate the optimal treatment protocol for this specific experiment, which we demonstrate to cause curative treatment, making it superior to previous approaches which generally aimed at stabilising tumour burden. Overall, our approach underscores the usefulness of simple mathematical models and their analytical examination, and we anticipate our findings to guide future preclinical and, ultimately, clinical research in optimising treatment regimes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548135" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548135">
        <p class="paperTitle">Fully Automated Sequential Immunofluorescence (seqIF) for Hyperplex Spatial Proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rivest, F.; Eroglu, D.; Pelz, B.; Kowal, J.; Kehren, A.; Procopio, M. G.; Bordignon, P.; Peres, E.; Ammann, M.; Dorel, E.; Scalmazzi, S.; Bruno, L.; Ruegg, M.; Campargue, G.; Casqueiro, G.; Arn, L.; Fischer, J.; Brajkovic, S.; Joris, P.; Cassano, M.; Dupouy, D.</p>
        <p class="info">Score: 6.8, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548135' target='https://doi.org/10.1101/2023.07.07.548135'> 10.1101/2023.07.07.548135</a></p>
        <p class="abstract">Tissues are complex environments where different cell types are in constant interaction with each other and with non-cellular components. Preserving the spatial context during proteomics analyses of tissue samples has become an important objective for different applications, one of the most important being the investigation of the tumor microenvironment. Here, we describe a multiplexed protein biomarker detection method on the COMET instrument, coined sequential ImmunoFluorescence (seqIF). The fully automated method uses successive applications of antibody incubation and elution, and in-situ imaging enabled by an integrated microscope and a microfluidic chip that provides optimized optical access to the sample. We show seqIF data on different sample types such as tumor and healthy tissue, including 40-plex on a single tissue section that is obtained in less than 24 hours, using off-the-shelf antibodies. We also present extensive characterization of the developed method, including elution efficiency, epitope stability, repeatability and reproducibility, signal uniformity, and dynamic range, in addition to marker and panel optimization strategies. The streamlined workflow using off-the-shelf antibodies, data quality enabling quantitative downstream analysis, and ease of reaching hyperplex levels make seqIF suitable for immune-oncology research and other disciplines requiring spatial analysis, paving the way for its adoption in clinical settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548142">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548142" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548142">
        <p class="paperTitle">MethNet: a robust approach to identify regulatory hubs and their distal targets in cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548142" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548142" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sakellaropoulos, T.; Do, C.; Jiang, G.; Cova, G.; Meyn, P.; Dimartino, D.; Ramaswami, S.; Heguy, A.; Tsirigos, A.; Skok, J. A.</p>
        <p class="info">Score: 6.7, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548142' target='https://doi.org/10.1101/2023.07.07.548142'> 10.1101/2023.07.07.548142</a></p>
        <p class="abstract">Aberrations in the capacity of DNA/chromatin modifiers and transcription factors to bind non-coding regions can lead to changes in gene regulation and impact disease phenotypes. However, identifying distal regulatory elements and connecting them with their target genes remains challenging. Here, we present MethNet, a pipeline that integrates large-scale DNA methylation and gene expression data across multiple cancers, to uncover novel cis regulatory elements (CREs) in a 1Mb region around every promoter in the genome. MethNet identifies clusters of highly ranked CREs, referred to as  hubs, which contribute to the regulation of multiple genes and significantly affect patient survival. Promoter-capture Hi-C confirmed that highly ranked associations involve physical interactions between CREs and their gene targets, and CRISPRi based scRNA Perturb-seq validated the functional impact of CREs. Thus, MethNet-identified CREs represent a valuable resource for unraveling complex mechanisms underlying gene expression, and for prioritizing the verification of predicted non-coding disease hotspots.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.13.548697">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.13.548697" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.13.548697">
        <p class="paperTitle">In vivo single-cell CRISPR uncovers distinct TNF-alphaprograms in clonal expansion and tumorigenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.13.548697" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.13.548697" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Renz, P. F.; Ghoshdastider, U.; Baghai Sain, S.; Valdivia-Francia, F.; Khandekar, A.; Ormiston, M.; Bernasconi, M.; Kretz, J. A.; Lee, M.; Hyams, K.; Forny, M.; Pohly, M.; Ficht, X.; Ellis, S. J.; Moor, A. E.; Sendoel, A.</p>
        <p class="info">Score: 6.1, Published: 2023-07-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.13.548697' target='https://doi.org/10.1101/2023.07.13.548697'> 10.1101/2023.07.13.548697</a></p>
        <p class="abstract">The tumor evolution model posits that malignant transformation is preceded by randomly distributed driver mutations in cancer genes, which cause clonal expansions in phenotypically normal tissues. Although clonal expansions occur frequently in human epithelia and can remodel almost entire tissues, the mechanisms behind why only a small number of clones transform into malignant tumors remain enigmatic. Here, we develop an in vivo single-cell CRISPR strategy to systematically investigate tissue-wide clonal dynamics of the 150 most frequently mutated squamous cell carcinoma genes. We couple ultrasound-guided in utero lentiviral microinjections, single-cell RNA sequencing, guide capture and spatial transcriptomics to longitudinally monitor cell type-specific clonal expansions, document their underlying gene programs and contrast clonal expansions from tumor initiation. We uncover a TNF-alpha signaling module that acts as a generalizable driver of clonal expansions in epithelial tissues. Conversely, during tumorigenesis, the TNF-alpha signaling module is downregulated, and instead, we identify a subpopulation of invasive cancer cells that switch to an autocrine TNF-alpha gene program. By analyzing clonally expanded perturbations and their frequency in tumors, we demonstrate that the autocrine TNF- gene program is associated with epithelial-mesenchymal transition (EMT) and is preexistent in a subpopulation of expanded epidermal stem cells, contributing to the predisposition for tumor initiation. Finally, we provide in vivo evidence that the epithelial TNF-alpha gene program is sufficient to mediate invasive properties of epidermal stem cells and show that the TNF-alpha signature correlates with shorter overall survival in human squamous cell carcinoma patients. Collectively, our study demonstrates the power of applying in vivo single-cell CRISPR screening to mammalian tissues and unveils distinct TNF-alpha programs in tumor evolution. Understanding the biology of clonal expansions in phenotypically normal epithelia and the mechanisms governing their transformation will guide the development of novel strategies for early cancer detection and therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548130">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548130" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548130">
        <p class="paperTitle">Cytokine storm mitigation for exogenous immune agonists</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548130" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548130" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kareva, I.; Gevertz, J. L.</p>
        <p class="info">Score: 7.6, Published: 2023-07-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548130' target='https://doi.org/10.1101/2023.07.07.548130'> 10.1101/2023.07.07.548130</a></p>
        <p class="abstract">Cytokine storm is a life-threatening inflammatory response characterized by hyperactivation of the immune system. It can be caused by various therapies, auto-immune conditions, or pathogens, such as respiratory syndrome coronavirus 2 (SARS-CoV-2) which causes coronavirus disease COVID-19. Here we propose a conceptual mathematical model describing the phenomenology of cytokine-immune interactions when a tumor is treated by an exogenous immune cell agonist which has the potential to cause a cytokine storm, such as CAR T cell therapy. Numerical simulations reveal that as a function of just two model parameters, the same drug dose and regimen could result in one of four outcomes: treatment success without a storm, treatment success with a storm, treatment failure without a storm, and treatment failure with a storm. We then explore a scenario in which tumor control is accompanied by a storm and ask if it is possible to modulate the duration and frequency of drug administration (without changing the cumulative dose) in order to preserve efficacy while preventing the storm. Simulations reveal existence of a &#34;sweet spot&#34; in protocol space (number versus spacing of doses) for which tumor control is achieved without inducing a cytokine storm. This theoretical model, which contains a number of parameters that can be estimated experimentally, contributes to our understanding of what triggers a cytokine storm, and how the likelihood of its occurrence can be mitigated.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
