<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Three-dimensional assessments are necessary to determine the true, spatially-resolved composition of tissues
Authors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.
Score: 52.8, Published: 2024-02-03 DOI: 10.1101/2023.12.04.569986
Methods for spatially resolved cellular profiling using thinly cut sections have enabled in-depth quantitative tissue mapping to study inter-sample and intra-sample differences in normal human anatomy and disease onset and progression.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Three-dimensional assessments are necessary to determine the true, spatially-resolved composition of tissues
Authors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.
Score: 52.8, Published: 2024-02-03 DOI: 10.1101/2023.12.04.569986
Methods for spatially resolved cellular profiling using thinly cut sections have enabled in-depth quantitative tissue mapping to study inter-sample and intra-sample differences in normal human anatomy and disease onset and progression." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-07T10:40:15+00:00" />
<meta property="article:modified_time" content="2024-02-07T10:40:15+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Three-dimensional assessments are necessary to determine the true, spatially-resolved composition of tissues
Authors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.
Score: 52.8, Published: 2024-02-03 DOI: 10.1101/2023.12.04.569986
Methods for spatially resolved cellular profiling using thinly cut sections have enabled in-depth quantitative tissue mapping to study inter-sample and intra-sample differences in normal human anatomy and disease onset and progression."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Three-dimensional assessments are necessary to determine the true, spatially-resolved composition of tissues\nAuthors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.\nScore: 52.8, Published: 2024-02-03 DOI: 10.1101/2023.12.04.569986\nMethods for spatially resolved cellular profiling using thinly cut sections have enabled in-depth quantitative tissue mapping to study inter-sample and intra-sample differences in normal human anatomy and disease onset and progression.",
  "keywords": [
    
  ],
  "articleBody": " Three-dimensional assessments are necessary to determine the true, spatially-resolved composition of tissues\nAuthors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.\nScore: 52.8, Published: 2024-02-03 DOI: 10.1101/2023.12.04.569986\nMethods for spatially resolved cellular profiling using thinly cut sections have enabled in-depth quantitative tissue mapping to study inter-sample and intra-sample differences in normal human anatomy and disease onset and progression. These methods often profile extremely limited regions, which may impact the evaluation of heterogeneity due to tissue sub-sampling. Here, we applied CODA, a deep learning-based tissue mapping platform, to reconstruct the three-dimensional (3D) microanatomy of grossly normal and cancer-containing human pancreas biospecimens obtained from individuals who underwent pancreatic resection. To compare inter-and intra-sample heterogeneity, we assessed bulk and spatially resolved tissue composition in a cohort of two-dimensional (2D) whole slide images (WSIs) and a cohort of thick slabs of pancreas tissue that were digitally reconstructed in 3D from serial sections. To demonstrate the marked under sampling of 2D assessments, we simulated the number of WSIs and tissue microarrays (TMAs) necessary to represent the compositional heterogeneity of 3D data within 10% error to reveal that tens of WSIs and hundreds of TMA cores are sometimes needed. We show that spatial correlation of different pancreatic structures decay significantly within a span of microns, demonstrating that 2D histological sections may not be representative of their neighboring tissues. In sum, we demonstrate that 3D assessments are necessary to accurately assess tissue composition in normal and abnormal specimens and in order to accurately determine neoplastic content. These results emphasize the importance of intra-sample heterogeneity in tissue mapping efforts.\nMechanical confinement governs phenotypic plasticity in melanoma\nAuthors: Hunter, M. V.; Montal, E.; Ma, Y.; Moncada, R.; Yanai, I.; Koche, R. P.; White, R. M.\nScore: 16.8, Published: 2024-02-02 DOI: 10.1101/2024.01.30.577120\nPhenotype switching is a form of cellular plasticity in which cancer cells reversibly move between two opposite extremes - proliferative versus invasive states. While it has long been hypothesised that such switching is triggered by external cues, the identity of these cues has remained elusive. Here, we demonstrate that mechanical confinement mediates phenotype switching through chromatin remodelling. Using a zebrafish model of melanoma coupled with human samples, we profiled tumor cells at the interface between the tumor and surrounding microenvironment. Morphological analysis of these rare cells showed flattened, elliptical nuclei suggestive of mechanical confinement by adjacent tissue. Spatial and single-cell transcriptomics demonstrated that the interface cells adopted a gene program of neuronal invasion, including acquisition of an acetylated tubulin cage that protects the nucleus during migration. We identified the DNA-bending protein HMGB2 as a confinement-induced mediator of the neuronal state. HMGB2 is upregulated in confined cells, and quantitative modelling revealed that confinement prolongs contact time between HMGB2 and chromatin, leading to changes in chromatin configuration that favor the neuronal phenotype. Genetic disruption of HMGB2 showed that it regulates the trade-off between proliferative and invasive states, in which confined HMGB2high tumor cells are less proliferative but more drug resistant. Our results implicate the mechanical microenvironment as a mechanism driving phenotype switching in melanoma.\nPersister cancer cells are characterized by H4K20me3 heterochromatin that defines a low inflammatory profile\nAuthors: Ramponi, V.; Richart, L.; Kovatcheva, M.; Stephan-Otto Attolini, C.; Capellades, J.; Lord, A. E.; Yanes, O.; Ficz, G.; Serrano, M.\nScore: 22.5, Published: 2024-01-29 DOI: 10.1101/2024.01.26.577389\nAnti-cancer therapies may induce proliferative arrest in cancer cells in the form of senescence or drug-tolerant persistency, the latter being a reversible arrest with similarities to embryonic diapause. Here, we use mTOR/PI3K inhibition to develop and characterize a model of persistency/diapause-like arrest in human cancer cells of various origins. We show that persister and senescent cancer cells share an expanded lysosomal compartment and hypersensitivity to BCL-XL inhibition. However, persister cells do not exhibit other features of senescence, such as the loss of Lamin B1, senescence-associated b-galactosidase activity, and an inflammatory phenotype. Compared to senescent cells, persister cells have a profoundly diminished senescence-associated secretory phenotype (SASP), low activation of interferon signaling pathways and lack upregulation of MHC-I presentation. Based on a genome-wide CRISPR/Cas9 screen performed in diapause mouse embryonic stem cells (mESC), we discover that persister human cancer cells are hypersensitive to the inhibition of one-carbon metabolism. This finding led us to uncover that the repressive heterochromatic mark H4K20me3 is enriched at promoters of SASP and interferon response genes in persister cells, but not in senescent cells. Collectively, we define novel features and vulnerabilities of persister cancer cells and we provide insight into the epigenetic mechanisms underlying their low inflammatory and immunogenic activity.\nUpregulated cholesterol biosynthesis facilitates the survival of methylation-retaining AML cells following decitabine treatment.\nAuthors: Bond, D. R.; Burnard, S. M.; Uddipto, K.; Hunt, K. V.; Harvey, B. M.; Steffens Reinhardt, L.; Lawlor-O'Neill, C.; Roper, E. A.; Humphries, S.; Murray, H. C.; Bowden, N. A.; Enjeti, A. K.; Verrills, N. M.; Riveros, C.; Le Cao, K.-A.; Lee, H. J.\nScore: 10.5, Published: 2024-02-01 DOI: 10.1101/2024.01.30.577864\nDNA hypomethylating agents (HMAs) are used to treat acute myeloid leukaemia (AML) and myelodysplasia patients who are unsuitable for intensive chemotherapy, but low response rates and therapy-resistant relapse remain significant challenges. To optimise HMA efficacy, we must understand how resistance and relapse arise from cells that survive treatment. Here we combine single-cell multi-omic analysis with parallel colony-forming assays to link HMA-induced molecular heterogeneity with functional consequences in AML cells. HMAs, azacytidine (AZA) and decitabine (DAC), induced global epigenetic heterogeneity associated with upregulation of inflammatory responses and cell death pathways in a subset of hypomethylated cells. Some AML cells maintained high DNA methylation during treatment, and these methylation-retaining cells had increased self-renewal capacity following DAC, but not AZA. Molecular profiling of individual colonies revealed upregulated cholesterol biosynthesis as an adaptation to HMA treatment, and inhibition by rosuvastatin enhanced DAC effects in vitro and in vivo. Thus, HMA-induced heterogeneity has important implications for AML cell growth and statins are a candidate co-treatment strategy to delay or prevent HMA-resistant relapse.\nHuman IL-6 fosters long-term engraftment of patient derived disease-driving myeloma cells in immunodeficient mice\nAuthors: Hasanali, Z. S.; Garfall, A.; Burzenski, L.; Shultz, L.; Tang, Y.; Kadu, S.; Sheppard, N.; Dopkin, D.; Vogl, D. T.; Cohen, A. D.; Waxman, A. J.; Susanibar-Adaniya, S.; Carroll, M.; Stadtmauer, E.; Allman, D. M.\nScore: 11.2, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576547\nMultiple myeloma is a largely incurable and life-threatening malignancy of antibody-secreting plasma cells. An effective and widely available animal model that recapitulates human myeloma and related plasma cell disorders is lacking. We show that busulfan-conditioned hIL-6 transgenic NSG mice (NSG+hIL6) reliably support the engraftment of malignant and pre-malignant human plasma cells including from patients diagnosed with monoclonal gammopathy of undetermined significance, pre- and post-relapse myeloma, plasma cell leukemia, and AL amyloidosis. Consistent with human disease, NSG+hIL6 mice engrafted with patient-derived myeloma cells, developed serum M spikes, and a majority developed anemia, hypercalcemia, and/or bone lesions. Single cell RNA sequencing showed non-malignant and malignant cell engraftment, the latter expressing a wide array of mRNAs associated with myeloma cell survival and proliferation. Myeloma engrafted mice given CAR T-cells targeting plasma cells or bortezomib experienced reduced tumor burden. Our results establish NSG+hIL6 mice as an effective patient derived xenograft model for study and preclinical drug development of multiple myeloma and related plasma cell disorders.\nSomatic gene delivery for flexible in vivo modeling of high-risk sarcoma\nAuthors: Imle, R.; Blosel, D.; Kommoss, F.; Zhao, E.; Autry, R.; Blume, C.; Lupar, D.; Schmitt, L.; Winter, C.; Wagner, L.; Placke, S.; von Eicke, M.; Hertwig, M.; Peterziel, H.; Oehme, I.; Scheuerman, S.; Seitz, C.; Geyer, F.; Cidre-Aranaz, F.; Grunewald, T.; Vokuhl, C.; Chudasama, P.; Scholl, C.; Schmidt, C.; Gunther, P.; Sill, M.; Jones, K.; Pfister, S.; Banito, A.\nScore: 7.6, Published: 2024-02-01 DOI: 10.1101/2024.01.30.577924\nA particular challenge hampering therapeutic advancements for high-risk sarcoma patients is the broad spectrum of molecularly distinct sarcoma entities and the corresponding lack of suitable model systems to recapitulate and study these diseases. To overcome this predicament, we developed a novel genetically-controlled, yet versatile mouse modeling platform allowing delivery of different genetic lesions by electroporation (EPO) of the thigh muscle wildtype mice. This optimized sarcoma EPO-GEMM (EPO-based genetically engineered mouse model) platform allowed the generation of ten biologically distinct sarcoma entities, including Synovial Sarcoma (SS), fusion-positive and fusion-negative Rhabdomyosarcoma (RMS), Alveolar Soft Part Sarcoma (ASPS), Undifferentiated Pleomorphic Sarcoma (UPS) and Infantile Fibrosarcoma (IFS). Comprehensive molecular profiling and cross-species analyses confirmed faithful recapitulation of the human disease, including the expression of relevant immunotherapy targets. Syngeneic allografting enabled reliable preservation and scalability of Sarcoma-EPO-GEMMs for treatment trials, such as B7-H3-directed CAR-T cell therapy in an immunocompetent background.\nDeciphering the diversity and sequence of extracellular matrix and cellular spatial patterns in lung adenocarcinoma using topological data analysis\nAuthors: Yoon, I. H. R.; Jenkins, R.; Colliver, E.; Zhang, H.; Novo, D.; Moore, D.; Ramsden, Z.; Rullan, A.; Fu, X.; Yuan, Y.; Harrington, H. A.; Swanton, C.; Byrne, H. M.; Sahai, E.\nScore: 38.4, Published: 2024-01-17 DOI: 10.1101/2024.01.05.574362\nExtracellular matrix (ECM) organization influences cancer development and progression. It modulates the invasion of cancer cells and can hinder the access of immune cells to cancer cells. Effective quantification of ECM architecture and its relationship to the position of different cell types is, therefore, important when investigating the role of ECM in cancer development. Using topological data analysis (TDA), particularly persistent homology and Dowker persistent homology, we develop a novel analysis pipeline for quantifying ECM architecture, spatial patterns of cell positions, and the spatial relationships between distinct constituents of the tumour microenvironment. We apply the pipeline to 44 surgical specimens of lung adenocarcinoma from the lung TRACERx study stained with picrosirius red and haematoxylin. We show that persistent homology effectively encodes the architectural features of the tumour microenvironment. Inference using pseudo-time analysis and spatial mapping to centimetre scale tissues suggests a gradual and progressive route of change in ECM architecture, with two different end states. Dowker persistent homology enables the analysis of spatial relationship between any pair of constituents of the tumour microenvironment, such as ECM, cancer cells, and leukocytes. We use Dowker persistent homology to quantify the spatial segregation of cancer and immune cells over different length scales. A combined analysis of both topological and non-topological features of the tumour microenvironment indicates that progressive changes in the ECM are linked to increased immune exclusion and reduced oxidative metabolism.\nMYBL2 drives prostate cancer plasticity and identifies CDK2 as a therapeutic vulnerability in RB1-loss and neuroendocrine prostate cancer\nAuthors: Ellis, L.; German, B.; Singh, J. N.; Burkhart, D. L.; Sheahan, A.; Bergom, H.; Morel, K. L.; Beltran, H.; Hwang, J.; Lawrenson, K.\nScore: 5.2, Published: 2024-02-02 DOI: 10.1101/2024.01.31.578216\nPhenotypic plasticity is a recognized mechanism driving therapeutic resistance in prostate cancer (PCa) patients. While underlying molecular causations driving phenotypic plasticity have been identified, therapeutic success is yet to be achieved. To identify putative master regulator transcription factors (MR-TF) driving phenotypic plasticity in PCa, this work utilized a multiomic approach using genetically engineered mouse models of prostate cancer combined with patient data to identify MYBL2 as a significantly enriched transcription factor in PCa exhibiting phenotypic plasticity. Genetic inhibition of Mybl2 using independent murine PCa cell lines representing phenotypic plasticity demonstrated Mybl2 loss significantly decreased in vivo growth as well as cell fitness and repressed gene expression signatures involved in pluripotency and stemness. Because MYBL2 is currently not druggable, a MYBL2 gene signature was employed to identify cyclin-dependent kinase-2 (CDK2) as a potential therapeutic target. CDK2 inhibition phenocopied genetic loss of Mybl2 and significantly decreased in vivo tumor growth associated with enrichment of DNA damage. Together, this work demonstrates MYBL2 as an important MR-TF driving phenotypic plasticity in PCa. Further, high MYBL2 activity identifies PCa that would be responsive to CDK2 inhibition. SignificancePCa that escapes therapy targeting the androgen receptor signaling pathways via phenotypic plasticity are currently untreatable. Our study identifies MYBL2 as a MR-TF in phenotypic plastic PCa and implicates CDK2 inhibition as novel therapeutic target for this most lethal subtype of PCa.\nAccelerated Hematopoietic Stem Cell Aging in Space\nAuthors: Pham, J.; Isquith, J.; Balaian, L.; Ladel, L.; Nandi, S. P.; Mack, K.; van der Werf, I.; Klacking, E.; Ruiz, A.; Mays, D.; Gamble, P.; Giza, S.; Janowitz, J.; Nienaber, T.; Mishra, T.; Kulidjian, A.; Stoudemire, J.; Snyder, M. P.; Clements, T.; Muotri, A. R.; Morris, S. R.; Whisenant, T.; Alexandrov, L. B.; Jamieson, C. H. M.\nScore: 4.7, Published: 2024-02-01 DOI: 10.1101/2024.01.28.577076\nStem cell aging is accelerated by macroenvironmental and microenvironmental stressors, including inflammation. Previously, the NASA Twins study revealed inflammatory cytokine upregulation, chromosomal alterations, and telomere changes suggestive of accelerated aging in low-Earth orbit (LEO). To investigate the effects of spaceflight on human hematopoietic stem and progenitor cell (HSPC) aging, the NASA-supported Integrated Space Stem Cell Orbital Research team performed four independent 30- to 45-day NASA missions with matched flight and ground HSPC nanobioreactors in automated CubeLabs. These experiments revealed loss of HSPC dormancy, reduced self-renewal capacity, mitochondrial DNA amplification, APOBEC3-induced C-to-T mutagenesis, reduced ADAR1p150 expression, and alterations in the expression of repetitive elements. These molecular changes are indicative of accelerated HSPC aging and pre-leukemia stem cell generation in space and may be predictable and preventable.\nDeciphering the phenotypic heterogeneity and drug response in cancer cells using genome-wide activity and interaction of chromatin domains.\nAuthors: Pandey, N.; Sharma, M.; Mathur, A.; Nzelu, G. A.; Hakimullah, M.; Jha, I. P.; Chandra, O.; Mishra, S.; Bhattacharya, J.; Sharma, A.; Foo, R.; Sandhu, K. S.; Mandoli, A.; DasGupta, R.; KUMAR, V.\nScore: 11.9, Published: 2024-01-26 DOI: 10.1101/2023.01.15.524115\nThe effect of colocalization of genes in the topologically associated domains (TADs) and their activity orchestration in cancer samples and cells, hand-in-hand with drug-response have not been comprehensively studied. Here, we analyzed patterns in the activity of TADs in cancer-cells along with chromatin interaction profiles to understand their modes of drug-response. Analysis of transcriptomes of 819 cancer cell lines revealed that their response to multiple drugs were more correlated with the activity of TADs than individual genes. Applying our approach to 9014 cancer patients data of 20 different cancer-types further strengthened the association between drug response and the activity of TADs, instead of individual genes, highlighting similarity despite cancer heterogeneity. CRISPR-mediated knock-out of regulatory sites inside a TAD associated with cisplatin-response of oral cancer cells and discovery of primate-specific gain of synteny of genes within a TAD containing EGFR gene and its consequence, also demonstrate greater utility of TAD-activity based analysis.\n",
  "wordCount" : "2406",
  "inLanguage": "en",
  "datePublished": "2024-02-07T10:40:15Z",
  "dateModified": "2024-02-07T10:40:15Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 7, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569986">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569986" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569986">
        <p class="paperTitle">Three-dimensional assessments are necessary to determine the true, spatially-resolved composition of tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569986" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569986" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.</p>
        <p class="info">Score: 52.8, Published: 2024-02-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569986' target='https://doi.org/10.1101/2023.12.04.569986'> 10.1101/2023.12.04.569986</a></p>
        <p class="abstract">Methods for spatially resolved cellular profiling using thinly cut sections have enabled in-depth quantitative tissue mapping to study inter-sample and intra-sample differences in normal human anatomy and disease onset and progression. These methods often profile extremely limited regions, which may impact the evaluation of heterogeneity due to tissue sub-sampling. Here, we applied CODA, a deep learning-based tissue mapping platform, to reconstruct the three-dimensional (3D) microanatomy of grossly normal and cancer-containing human pancreas biospecimens obtained from individuals who underwent pancreatic resection. To compare inter-and intra-sample heterogeneity, we assessed bulk and spatially resolved tissue composition in a cohort of two-dimensional (2D) whole slide images (WSIs) and a cohort of thick slabs of pancreas tissue that were digitally reconstructed in 3D from serial sections. To demonstrate the marked under sampling of 2D assessments, we simulated the number of WSIs and tissue microarrays (TMAs) necessary to represent the compositional heterogeneity of 3D data within 10% error to reveal that tens of WSIs and hundreds of TMA cores are sometimes needed. We show that spatial correlation of different pancreatic structures decay significantly within a span of microns, demonstrating that 2D histological sections may not be representative of their neighboring tissues. In sum, we demonstrate that 3D assessments are necessary to accurately assess tissue composition in normal and abnormal specimens and in order to accurately determine neoplastic content. These results emphasize the importance of intra-sample heterogeneity in tissue mapping efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577120" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577120">
        <p class="paperTitle">Mechanical confinement governs phenotypic plasticity in melanoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hunter, M. V.; Montal, E.; Ma, Y.; Moncada, R.; Yanai, I.; Koche, R. P.; White, R. M.</p>
        <p class="info">Score: 16.8, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577120' target='https://doi.org/10.1101/2024.01.30.577120'> 10.1101/2024.01.30.577120</a></p>
        <p class="abstract">Phenotype switching is a form of cellular plasticity in which cancer cells reversibly move between two opposite extremes - proliferative versus invasive states. While it has long been hypothesised that such switching is triggered by external cues, the identity of these cues has remained elusive. Here, we demonstrate that mechanical confinement mediates phenotype switching through chromatin remodelling. Using a zebrafish model of melanoma coupled with human samples, we profiled tumor cells at the interface between the tumor and surrounding microenvironment. Morphological analysis of these rare cells showed flattened, elliptical nuclei suggestive of mechanical confinement by adjacent tissue. Spatial and single-cell transcriptomics demonstrated that the interface cells adopted a gene program of neuronal invasion, including acquisition of an acetylated tubulin cage that protects the nucleus during migration. We identified the DNA-bending protein HMGB2 as a confinement-induced mediator of the neuronal state. HMGB2 is upregulated in confined cells, and quantitative modelling revealed that confinement prolongs contact time between HMGB2 and chromatin, leading to changes in chromatin configuration that favor the neuronal phenotype. Genetic disruption of HMGB2 showed that it regulates the trade-off between proliferative and invasive states, in which confined HMGB2high tumor cells are less proliferative but more drug resistant. Our results implicate the mechanical microenvironment as a mechanism driving phenotype switching in melanoma.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.26.577389">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.26.577389" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.26.577389">
        <p class="paperTitle">Persister cancer cells are characterized by H4K20me3 heterochromatin that defines a low inflammatory profile</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.26.577389" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.26.577389" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ramponi, V.; Richart, L.; Kovatcheva, M.; Stephan-Otto Attolini, C.; Capellades, J.; Lord, A. E.; Yanes, O.; Ficz, G.; Serrano, M.</p>
        <p class="info">Score: 22.5, Published: 2024-01-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.26.577389' target='https://doi.org/10.1101/2024.01.26.577389'> 10.1101/2024.01.26.577389</a></p>
        <p class="abstract">Anti-cancer therapies may induce proliferative arrest in cancer cells in the form of senescence or drug-tolerant persistency, the latter being a reversible arrest with similarities to embryonic diapause. Here, we use mTOR/PI3K inhibition to develop and characterize a model of persistency/diapause-like arrest in human cancer cells of various origins. We show that persister and senescent cancer cells share an expanded lysosomal compartment and hypersensitivity to BCL-XL inhibition. However, persister cells do not exhibit other features of senescence, such as the loss of Lamin B1, senescence-associated b-galactosidase activity, and an inflammatory phenotype. Compared to senescent cells, persister cells have a profoundly diminished senescence-associated secretory phenotype (SASP), low activation of interferon signaling pathways and lack upregulation of MHC-I presentation. Based on a genome-wide CRISPR/Cas9 screen performed in diapause mouse embryonic stem cells (mESC), we discover that persister human cancer cells are hypersensitive to the inhibition of one-carbon metabolism. This finding led us to uncover that the repressive heterochromatic mark H4K20me3 is enriched at promoters of SASP and interferon response genes in persister cells, but not in senescent cells. Collectively, we define novel features and vulnerabilities of persister cancer cells and we provide insight into the epigenetic mechanisms underlying their low inflammatory and immunogenic activity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577864">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577864" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577864">
        <p class="paperTitle">Upregulated cholesterol biosynthesis facilitates the survival of methylation-retaining AML cells following decitabine treatment.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577864" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577864" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bond, D. R.; Burnard, S. M.; Uddipto, K.; Hunt, K. V.; Harvey, B. M.; Steffens Reinhardt, L.; Lawlor-O&#39;Neill, C.; Roper, E. A.; Humphries, S.; Murray, H. C.; Bowden, N. A.; Enjeti, A. K.; Verrills, N. M.; Riveros, C.; Le Cao, K.-A.; Lee, H. J.</p>
        <p class="info">Score: 10.5, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577864' target='https://doi.org/10.1101/2024.01.30.577864'> 10.1101/2024.01.30.577864</a></p>
        <p class="abstract">DNA hypomethylating agents (HMAs) are used to treat acute myeloid leukaemia (AML) and myelodysplasia patients who are unsuitable for intensive chemotherapy, but low response rates and therapy-resistant relapse remain significant challenges. To optimise HMA efficacy, we must understand how resistance and relapse arise from cells that survive treatment. Here we combine single-cell multi-omic analysis with parallel colony-forming assays to link HMA-induced molecular heterogeneity with functional consequences in AML cells. HMAs, azacytidine (AZA) and decitabine (DAC), induced global epigenetic heterogeneity associated with upregulation of inflammatory responses and cell death pathways in a subset of hypomethylated cells. Some AML cells maintained high DNA methylation during treatment, and these methylation-retaining cells had increased self-renewal capacity following DAC, but not AZA. Molecular profiling of individual colonies revealed upregulated cholesterol biosynthesis as an adaptation to HMA treatment, and inhibition by rosuvastatin enhanced DAC effects in vitro and in vivo. Thus, HMA-induced heterogeneity has important implications for AML cell growth and statins are a candidate co-treatment strategy to delay or prevent HMA-resistant relapse.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.576547">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.576547" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.576547">
        <p class="paperTitle">Human IL-6 fosters long-term engraftment of patient derived disease-driving myeloma cells in immunodeficient mice</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.576547" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.576547" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hasanali, Z. S.; Garfall, A.; Burzenski, L.; Shultz, L.; Tang, Y.; Kadu, S.; Sheppard, N.; Dopkin, D.; Vogl, D. T.; Cohen, A. D.; Waxman, A. J.; Susanibar-Adaniya, S.; Carroll, M.; Stadtmauer, E.; Allman, D. M.</p>
        <p class="info">Score: 11.2, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.576547' target='https://doi.org/10.1101/2024.01.21.576547'> 10.1101/2024.01.21.576547</a></p>
        <p class="abstract">Multiple myeloma is a largely incurable and life-threatening malignancy of antibody-secreting plasma cells. An effective and widely available animal model that recapitulates human myeloma and related plasma cell disorders is lacking. We show that busulfan-conditioned hIL-6 transgenic NSG mice (NSG&#43;hIL6) reliably support the engraftment of malignant and pre-malignant human plasma cells including from patients diagnosed with monoclonal gammopathy of undetermined significance, pre- and post-relapse myeloma, plasma cell leukemia, and AL amyloidosis. Consistent with human disease, NSG&#43;hIL6 mice engrafted with patient-derived myeloma cells, developed serum M spikes, and a majority developed anemia, hypercalcemia, and/or bone lesions. Single cell RNA sequencing showed non-malignant and malignant cell engraftment, the latter expressing a wide array of mRNAs associated with myeloma cell survival and proliferation. Myeloma engrafted mice given CAR T-cells targeting plasma cells or bortezomib experienced reduced tumor burden. Our results establish NSG&#43;hIL6 mice as an effective patient derived xenograft model for study and preclinical drug development of multiple myeloma and related plasma cell disorders.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577924" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577924">
        <p class="paperTitle">Somatic gene delivery for flexible in vivo modeling of high-risk sarcoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Imle, R.; Blosel, D.; Kommoss, F.; Zhao, E.; Autry, R.; Blume, C.; Lupar, D.; Schmitt, L.; Winter, C.; Wagner, L.; Placke, S.; von Eicke, M.; Hertwig, M.; Peterziel, H.; Oehme, I.; Scheuerman, S.; Seitz, C.; Geyer, F.; Cidre-Aranaz, F.; Grunewald, T.; Vokuhl, C.; Chudasama, P.; Scholl, C.; Schmidt, C.; Gunther, P.; Sill, M.; Jones, K.; Pfister, S.; Banito, A.</p>
        <p class="info">Score: 7.6, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577924' target='https://doi.org/10.1101/2024.01.30.577924'> 10.1101/2024.01.30.577924</a></p>
        <p class="abstract">A particular challenge hampering therapeutic advancements for high-risk sarcoma patients is the broad spectrum of molecularly distinct sarcoma entities and the corresponding lack of suitable model systems to recapitulate and study these diseases. To overcome this predicament, we developed a novel genetically-controlled, yet versatile mouse modeling platform allowing delivery of different genetic lesions by electroporation (EPO) of the thigh muscle wildtype mice. This optimized sarcoma EPO-GEMM (EPO-based genetically engineered mouse model) platform allowed the generation of ten biologically distinct sarcoma entities, including Synovial Sarcoma (SS), fusion-positive and fusion-negative Rhabdomyosarcoma (RMS), Alveolar Soft Part Sarcoma (ASPS), Undifferentiated Pleomorphic Sarcoma (UPS) and Infantile Fibrosarcoma (IFS). Comprehensive molecular profiling and cross-species analyses confirmed faithful recapitulation of the human disease, including the expression of relevant immunotherapy targets. Syngeneic allografting enabled reliable preservation and scalability of Sarcoma-EPO-GEMMs for treatment trials, such as B7-H3-directed CAR-T cell therapy in an immunocompetent background.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.05.574362">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.05.574362" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.05.574362">
        <p class="paperTitle">Deciphering the diversity and sequence of extracellular matrix and cellular spatial patterns in lung adenocarcinoma using topological data analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.05.574362" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.05.574362" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yoon, I. H. R.; Jenkins, R.; Colliver, E.; Zhang, H.; Novo, D.; Moore, D.; Ramsden, Z.; Rullan, A.; Fu, X.; Yuan, Y.; Harrington, H. A.; Swanton, C.; Byrne, H. M.; Sahai, E.</p>
        <p class="info">Score: 38.4, Published: 2024-01-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.05.574362' target='https://doi.org/10.1101/2024.01.05.574362'> 10.1101/2024.01.05.574362</a></p>
        <p class="abstract">Extracellular matrix (ECM) organization influences cancer development and progression. It modulates the invasion of cancer cells and can hinder the access of immune cells to cancer cells. Effective quantification of ECM architecture and its relationship to the position of different cell types is, therefore, important when investigating the role of ECM in cancer development. Using topological data analysis (TDA), particularly persistent homology and Dowker persistent homology, we develop a novel analysis pipeline for quantifying ECM architecture, spatial patterns of cell positions, and the spatial relationships between distinct constituents of the tumour microenvironment. We apply the pipeline to 44 surgical specimens of lung adenocarcinoma from the lung TRACERx study stained with picrosirius red and haematoxylin. We show that persistent homology effectively encodes the architectural features of the tumour microenvironment. Inference using pseudo-time analysis and spatial mapping to centimetre scale tissues suggests a gradual and progressive route of change in ECM architecture, with two different end states. Dowker persistent homology enables the analysis of spatial relationship between any pair of constituents of the tumour microenvironment, such as ECM, cancer cells, and leukocytes. We use Dowker persistent homology to quantify the spatial segregation of cancer and immune cells over different length scales. A combined analysis of both topological and non-topological features of the tumour microenvironment indicates that progressive changes in the ECM are linked to increased immune exclusion and reduced oxidative metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578216">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578216" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578216">
        <p class="paperTitle">MYBL2 drives prostate cancer plasticity and identifies CDK2 as a therapeutic vulnerability in RB1-loss and neuroendocrine prostate cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578216" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578216" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ellis, L.; German, B.; Singh, J. N.; Burkhart, D. L.; Sheahan, A.; Bergom, H.; Morel, K. L.; Beltran, H.; Hwang, J.; Lawrenson, K.</p>
        <p class="info">Score: 5.2, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578216' target='https://doi.org/10.1101/2024.01.31.578216'> 10.1101/2024.01.31.578216</a></p>
        <p class="abstract">Phenotypic plasticity is a recognized mechanism driving therapeutic resistance in prostate cancer (PCa) patients. While underlying molecular causations driving phenotypic plasticity have been identified, therapeutic success is yet to be achieved. To identify putative master regulator transcription factors (MR-TF) driving phenotypic plasticity in PCa, this work utilized a multiomic approach using genetically engineered mouse models of prostate cancer combined with patient data to identify MYBL2 as a significantly enriched transcription factor in PCa exhibiting phenotypic plasticity. Genetic inhibition of Mybl2 using independent murine PCa cell lines representing phenotypic plasticity demonstrated Mybl2 loss significantly decreased in vivo growth as well as cell fitness and repressed gene expression signatures involved in pluripotency and stemness. Because MYBL2 is currently not druggable, a MYBL2 gene signature was employed to identify cyclin-dependent kinase-2 (CDK2) as a potential therapeutic target. CDK2 inhibition phenocopied genetic loss of Mybl2 and significantly decreased in vivo tumor growth associated with enrichment of DNA damage. Together, this work demonstrates MYBL2 as an important MR-TF driving phenotypic plasticity in PCa. Further, high MYBL2 activity identifies PCa that would be responsive to CDK2 inhibition.

SignificancePCa that escapes therapy targeting the androgen receptor signaling pathways via phenotypic plasticity are currently untreatable. Our study identifies MYBL2 as a MR-TF in phenotypic plastic PCa and implicates CDK2 inhibition as novel therapeutic target for this most lethal subtype of PCa.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.28.577076">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.28.577076" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.28.577076">
        <p class="paperTitle">Accelerated Hematopoietic Stem Cell Aging in Space</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.28.577076" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.28.577076" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pham, J.; Isquith, J.; Balaian, L.; Ladel, L.; Nandi, S. P.; Mack, K.; van der Werf, I.; Klacking, E.; Ruiz, A.; Mays, D.; Gamble, P.; Giza, S.; Janowitz, J.; Nienaber, T.; Mishra, T.; Kulidjian, A.; Stoudemire, J.; Snyder, M. P.; Clements, T.; Muotri, A. R.; Morris, S. R.; Whisenant, T.; Alexandrov, L. B.; Jamieson, C. H. M.</p>
        <p class="info">Score: 4.7, Published: 2024-02-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.28.577076' target='https://doi.org/10.1101/2024.01.28.577076'> 10.1101/2024.01.28.577076</a></p>
        <p class="abstract">Stem cell aging is accelerated by macroenvironmental and microenvironmental stressors, including inflammation. Previously, the NASA Twins study revealed inflammatory cytokine upregulation, chromosomal alterations, and telomere changes suggestive of accelerated aging in low-Earth orbit (LEO). To investigate the effects of spaceflight on human hematopoietic stem and progenitor cell (HSPC) aging, the NASA-supported Integrated Space Stem Cell Orbital Research team performed four independent 30- to 45-day NASA missions with matched flight and ground HSPC nanobioreactors in automated CubeLabs. These experiments revealed loss of HSPC dormancy, reduced self-renewal capacity, mitochondrial DNA amplification, APOBEC3-induced C-to-T mutagenesis, reduced ADAR1p150 expression, and alterations in the expression of repetitive elements. These molecular changes are indicative of accelerated HSPC aging and pre-leukemia stem cell generation in space and may be predictable and preventable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.15.524115">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.15.524115" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.15.524115">
        <p class="paperTitle">Deciphering the phenotypic heterogeneity and drug response in cancer cells using genome-wide activity and interaction of chromatin domains.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.15.524115" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.15.524115" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pandey, N.; Sharma, M.; Mathur, A.; Nzelu, G. A.; Hakimullah, M.; Jha, I. P.; Chandra, O.; Mishra, S.; Bhattacharya, J.; Sharma, A.; Foo, R.; Sandhu, K. S.; Mandoli, A.; DasGupta, R.; KUMAR, V.</p>
        <p class="info">Score: 11.9, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.15.524115' target='https://doi.org/10.1101/2023.01.15.524115'> 10.1101/2023.01.15.524115</a></p>
        <p class="abstract">The effect of colocalization of genes in the topologically associated domains (TADs) and their activity orchestration in cancer samples and cells, hand-in-hand with drug-response have not been comprehensively studied. Here, we analyzed patterns in the activity of TADs in cancer-cells along with chromatin interaction profiles to understand their modes of drug-response. Analysis of transcriptomes of 819 cancer cell lines revealed that their response to multiple drugs were more correlated with the activity of TADs than individual genes. Applying our approach to 9014 cancer patients data of 20 different cancer-types further strengthened the association between drug response and the activity of TADs, instead of individual genes, highlighting similarity despite cancer heterogeneity. CRISPR-mediated knock-out of regulatory sites inside a TAD associated with cisplatin-response of oral cancer cells and discovery of primate-specific gain of synteny of genes within a TAD containing EGFR gene and its consequence, also demonstrate greater utility of TAD-activity based analysis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
