<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Discovery of Highly Active Kynureninases for Cancer Immunotherapy through Protein Language Model
Authors: Eom, H.; Cho, K. S.; Lee, J.; Kim, S.; Park, S.; Kim, H.; Yang, J.; Han, Y.-H.; Lee, J.; Seok, C.; Lee, M. S.; Song, W. J.; Steinegger, M.
Score: 27.1, Published: 2024-01-20 DOI: 10.1101/2024.01.16.575968
Overcoming the immunosuppressive tumor microenvironment is a promising strategy in anticancer therapy. L-kynurenine, a strong immunosuppressive metabolite can be degraded through kynureninases.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Discovery of Highly Active Kynureninases for Cancer Immunotherapy through Protein Language Model
Authors: Eom, H.; Cho, K. S.; Lee, J.; Kim, S.; Park, S.; Kim, H.; Yang, J.; Han, Y.-H.; Lee, J.; Seok, C.; Lee, M. S.; Song, W. J.; Steinegger, M.
Score: 27.1, Published: 2024-01-20 DOI: 10.1101/2024.01.16.575968
Overcoming the immunosuppressive tumor microenvironment is a promising strategy in anticancer therapy. L-kynurenine, a strong immunosuppressive metabolite can be degraded through kynureninases." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-01-28T10:37:59+00:00" />
<meta property="article:modified_time" content="2024-01-28T10:37:59+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Discovery of Highly Active Kynureninases for Cancer Immunotherapy through Protein Language Model
Authors: Eom, H.; Cho, K. S.; Lee, J.; Kim, S.; Park, S.; Kim, H.; Yang, J.; Han, Y.-H.; Lee, J.; Seok, C.; Lee, M. S.; Song, W. J.; Steinegger, M.
Score: 27.1, Published: 2024-01-20 DOI: 10.1101/2024.01.16.575968
Overcoming the immunosuppressive tumor microenvironment is a promising strategy in anticancer therapy. L-kynurenine, a strong immunosuppressive metabolite can be degraded through kynureninases."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Discovery of Highly Active Kynureninases for Cancer Immunotherapy through Protein Language Model\nAuthors: Eom, H.; Cho, K. S.; Lee, J.; Kim, S.; Park, S.; Kim, H.; Yang, J.; Han, Y.-H.; Lee, J.; Seok, C.; Lee, M. S.; Song, W. J.; Steinegger, M.\nScore: 27.1, Published: 2024-01-20 DOI: 10.1101/2024.01.16.575968\nOvercoming the immunosuppressive tumor microenvironment is a promising strategy in anticancer therapy. L-kynurenine, a strong immunosuppressive metabolite can be degraded through kynureninases.",
  "keywords": [
    
  ],
  "articleBody": " Discovery of Highly Active Kynureninases for Cancer Immunotherapy through Protein Language Model\nAuthors: Eom, H.; Cho, K. S.; Lee, J.; Kim, S.; Park, S.; Kim, H.; Yang, J.; Han, Y.-H.; Lee, J.; Seok, C.; Lee, M. S.; Song, W. J.; Steinegger, M.\nScore: 27.1, Published: 2024-01-20 DOI: 10.1101/2024.01.16.575968\nOvercoming the immunosuppressive tumor microenvironment is a promising strategy in anticancer therapy. L-kynurenine, a strong immunosuppressive metabolite can be degraded through kynureninases. Through homology searches and protein language models, we identified and then experimentally determined the efficacy of four top-ranked kynureninases. The catalytically most active one nearly doubles turnover number over the prior best, reducing tumor weight by 3.42 times in mouse model comparisons, and thus, presenting substantial therapeutic potential.\nChromosome 7 to the rescue: overcoming chromosome 10 loss in gliomas\nAuthors: Nair, N. U.; Schaffer, A. A.; Gertz, M. E.; Cheng, K.; Zerbib, J.; Sahu, A. D.; Leor, G.; Shulman, E. D.; Aldape, K. D.; Ben-David, U.; Ruppin, E.\nScore: 24.5, Published: 2024-01-22 DOI: 10.1101/2024.01.17.576103\nThe co-occurrence of chromosome 10 loss and chromosome 7 gain in gliomas is the most frequent loss-gain co-aneuploidy pair in human cancers, a phenomenon that has been investigated without resolution since the late 1980s. Expanding beyond previous gene-centric studies, we investigate the co-occurrence in a genome-wide manner taking an evolutionary perspective. First, by mining large tumor aneuploidy data, we predict that the more likely order is 10 loss followed by 7 gain. Second, by analyzing extensive genomic and transcriptomic data from both patients and cell lines, we find that this co-occurrence can be explained by functional rescue interactions that are highly enriched on 7, which can possibly compensate for any detrimental consequences arising from the loss of 10. Finally, by analyzing transcriptomic data from normal, non-cancerous, human brain tissues, we provide a plausible reason why this co-occurrence happens preferentially in cancers originating in certain regions of the brain.\nThe temporal evolution of cancer hallmarks\nAuthors: Gourmet, L.; Ramazzotti, D.; Mallick, P.; Walker-Samuel, S.; Zapata, L.\nScore: 14.9, Published: 2024-01-23 DOI: 10.1101/2024.01.21.576566\nCancer hallmarks describe key physiological characteristics that distinguish cancers from normal tissues. The temporal order in which these hallmarks appear during cancer pathogenesis is of interest from both evolutionary and clinical perspectives but has not been investigated before. Here, we order hallmarks based on the allele frequency and selective advantage of mutations in cancer hallmark genes across \u003e10k untreated primary tumors and \u003e8K healthy tissues. Using this novel approach, we identified a common evolutionary trajectory for 27 of 32 cancer types with genomic instability as the first and immune evasion as the last hallmark. We demonstrated widespread positive selection in cancer and strong negative selection in normal tissues for all hallmarks. Notable exceptions to the hallmark ordering in tumours were melanomas (uveal and skin) suggesting that strong environmental factors could disrupt common evolutionary paths. Clustering of hallmark trajectories across patients revealed 2 clusters defined by early or late genomic instability, with differential prognosis. Our study is the first to identify the temporal order of cancer hallmarks during tumorigenesis and demonstrate a prognostic value that could be exploited for early detection and risk stratification across multiple cancer types.\nMammalian SWI/SNF complex activity regulates POU2F3 and constitutes a targetable dependency in small cell lung cancer\nAuthors: Duplaquet, L.; So, K.; Ying, A. W.; Li, X.; Li, Y.; Qiu, X.; Li, R.; Singh, S.; Wu, X. S.; Liu, Q.; Qi, J.; Somerville, T. D. D.; Heiling, H.; Mazzola, E.; Lee, Y.; Zoller, T.; Vakoc, C. R.; Doench, J. G.; Forrester, W. C.; Abrams, T.; Long, H. W.; Niederst, M. J.; Kadoch, C.; Oser, M. G.\nScore: 12.0, Published: 2024-01-24 DOI: 10.1101/2024.01.21.576304\nSmall cell lung cancers (SCLC) are comprised of heterogeneous subtypes marked by lineage-specific transcription factors, including ASCL1, NEUROD1, and POU2F3. POU2F3-positive SCLC, [~]12% of all cases, are uniquely dependent on POU2F3 itself; as such, approaches to attenuate POU2F3 expression may represent new therapeutic opportunities. Here using genome-scale screens for regulators of POU2F3 expression and SCLC proliferation, we define mSWI/SNF complexes, including non-canonical BAF (ncBAF) complexes, as top dependencies specific to POU2F3-positive SCLC. Notably, clinical-grade pharmacologic mSWI/SNF inhibition attenuates proliferation of all POU2F3-positive SCLCs, while disruption of ncBAF via BRD9 degradation is uniquely effective in pure non-neuroendocrine POU2F3-SCLCs. mSWI/SNF maintains accessibility over gene loci central to POU2F3-mediated gene regulatory networks. Finally, chemical targeting of SMARCA4/2 mSWI/SNF ATPases and BRD9 decrease POU2F3-SCLC tumor growth and increase survival in vivo. Taken together, these results characterize mSWI/SNF-mediated global governance of the POU2F3 oncogenic program and suggest mSWI/SNF inhibition as a therapeutic strategy for SCLC.\nIntegrated cancer cell-specific single-cell RNA-seq datasets of immune checkpoint blockade-treated patients\nAuthors: Gondal, M. N.; Cieslik, M.; Chinnaiyan, A. M.\nScore: 6.5, Published: 2024-01-22 DOI: 10.1101/2024.01.17.576110\nImmune checkpoint blockade (ICB) therapies have emerged as a promising avenue for the treatment of various cancers. Despite their success, the efficacy of these treatments is variable across patients and cancer types. Numerous single-cell RNA-sequencing (scRNA-seq) studies have been conducted to unravel cell-specific responses to ICB treatment. However, these studies are limited in their sample sizes and require advanced coding skills for exploration. Here, we have compiled eight scRNA-seq datasets from nine cancer types, encompassing 174 patients, and 90,270 cancer cells. This compilation forms a unique resource tailored for investigating how cancer cells respond to ICB treatment across cancer types. We meticulously curated, quality-checked, pre-processed, and analyzed the data, ensuring easy access for researchers. Moreover, we designed a user-friendly interface for seamless exploration. By sharing the code and data for creating these interfaces, we aim to assist fellow researchers. These resources offer valuable support to those interested in leveraging and exploring single-cell datasets across diverse cancer types, facilitating a comprehensive understanding of ICB responses.\nMechanisms underpinning osteosarcoma genome complexity and evolution\nAuthors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.\nScore: 31.1, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573403\nOsteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood. Using multi-region whole-genome sequencing, we find that chromothripsis is an ongoing mutational process, occurring subclonally in 74% of tumours. Chromothripsis drives the acquisition of oncogenic mutations and generates highly unstable derivative chromosomes, the evolution of which drives clonal diversification and intra-tumour heterogeneity. In addition, we report a novel mechanism, loss-translocation-amplification (LTA) chromothripsis, which mediates rapid malignant transformation and punctuated evolution in about half of paediatric and adult high-grade osteosarcomas. Specifically, a single double-strand break triggers concomitant TP53 inactivation and segmental amplifications, often amplifying oncogenes to high copy numbers in extrachromosomal circular DNA elements through breakage-fusion-bridge cycles involving multiple chromosomes. LTA chromothripsis is detected at low frequency in soft-tissue sarcomas, but not in epithelial cancers, including those driven by TP53 mutation. Finally, we identify genome-wide loss of heterozygosity as a strong prognostic indicator for high-grade osteosarcoma.\nSMAC mimetics overcome apoptotic resistance in ovarian cancer through MSLN-TNF alpha axis\nAuthors: Coelho, R.; Seashore-Ludlow, B.; Schutz, S.; Christopher Lombardo, F.; Moussaud-Lamodiere, E.; Casanova, R.; Ficek-Pascual, J.; Brunhilde Labrosse, K.; Lopez-Nunez, M.; Rimmer, N.; Fedier, A.; Lima, R.; Montavon Sartorius, C.; Kurzeder, C.; Singer, F.; Bertolini, A.; The TumorProfiler Consortium, ; Ratsch, G.; Bodenmiller, B.; Kallioniemi, O.; Ostling, P.; David, L.; Heinzelmann-Schwarz, V.; Jacob, F.\nScore: 3.7, Published: 2024-01-26 DOI: 10.1101/2024.01.24.576987\nResistance to chemotherapy and PARPi inhibitors remains a critical challenge in the treatment of epithelial ovarian cancer, mainly due to disabled apoptotic responses in tumor cells. Given mesothelin's pivotal role in ovarian cancer and its restricted expression in healthy tissues, we conducted a drug-screening discovery analysis across a range of genetically modified cancer cells to unveil mesothelin's therapeutic impact. We observed enhanced cell death in low mesothelin expressing cancer cells when exposed to a second mitochondria-derived activator of caspases (SMAC) mimetics, and demonstrated a compelling synergy when combined with chemotherapy in ex vivo patient-derived cultures and zebrafish tumor xenografts. Mechanistically, the addition of the SMAC mimetics drug birinapant to either carboplatin or paclitaxel triggered the activation of the Caspase 8-dependent apoptotic program facilitated by TNF alpha signaling. Multimodal analysis of neoadjuvant-treated patient samples further revealed an association between tumor-associated macrophages and the activation of TNF alpha-related pathways. Our proposed bimodal treatment shows promise in enhancing the clinical management of patients by harnessing the potential of SMAC mimetics alongside conventional chemotherapy.\nA CANCER PERSISTENT DNA REPAIR CIRCUIT DRIVEN BY MDM2, MDM4 (MDMX), AND MUTANT P53 FOR RECRUITMENT OF MDC1 AND 53BP1 TO CHROMATIN\nAuthors: Ellison, V.; Polotskaia, A.; Leybengrub, P.; Qiu, W.; Lee, R.; Hendrickson, R. C.; Hu, W.; Bargonetti, J.\nScore: 3.4, Published: 2024-01-23 DOI: 10.1101/2024.01.20.576487\nThe influence of the metastasis promoting proteins mutant p53 (mtp53) and MDM2 on Cancer Persistent Repair (CPR) to promote cancer cell survival is understudied. Interactions between the DNA repair choice protein 53BP1 and wild type tumor suppressor protein p53 (wtp53) regulates cell cycle control. Cancer cells often express elevated levels of transcriptionally inactive missense mutant p53 (mtp53) that interacts with MDM2 and MDM4/MDMX (herein called MDMX). The ability of mtp53 to maintain a 53BP1 interaction while in the context of interactions with MDM2 and MDMX has not been described. We asked if MDM2 regulates chromatin-based phosphorylation events in the context of mtp53 by comparing the chromatin of T47D breast cancer cells with and without MDM2 in a phospho-peptide stable isotope labeling in cell culture (SILAC) screen. We found reduced phospho-53BP1 chromatin association, which we confirmed by chromatin fractionation and immunofluorescence in multiple breast cancer cell lines. We used the Proximity Ligation Assay (PLA) in breast cancer cell lines and detected 53BP1 in close proximity to mtp53, MDM2, and the DNA repair protein MDC1. Through disruption of the mtp53-MDM2 interaction, by either Nutlin 3a or a mtp53 R273H C-terminal deletion, we uncovered that mtp53 was required for MDM2-53BP1 interaction foci. Our data suggests that mtp53 works with MDM2 and 53BP1 to promote CPR and cell survival.\nCombining RASG12C(ON) inhibitor with SHP2 inhibition sensitises immune excluded lung tumours to immune checkpoint blockade: a strategy for turning cold tumours hot\nAuthors: Anastasiou, P.; Moore, C.; Rana, S.; de Castro, A.; Tomaschko, M.; Boumelha, J.; Mugarza, E.; Blaj, C.; de Carne Trecesson, S.; Goldstone, R.; Smith, J.; Quintana, E.; Molina-Arcas, M.; Downward, J.\nScore: 21.6, Published: 2024-01-16 DOI: 10.1101/2024.01.15.575765\nMutant selective drugs targeting the inactive, GDP-bound form of KRASG12C have been approved for use in lung cancer, but responses are short-lived due to rapid development of resistance. In this study we use a novel covalent tri-complex inhibitor, RMC-4998, that targets RASG12C in its active, GTP-bound form to investigate treatment of KRAS mutant lung cancer in various immune competent mouse models. While this RASG12C(ON) inhibitor was more potent than the KRASG12C(OFF) inhibitor adagrasib, rapid pathway reactivation was still observed. This could be delayed using combined treatment with a SHP2 inhibitor, RMC-4550, which not only impacted RAS pathway signalling within the tumour cells but also remodelled the tumour microenvironment (TME) to be less immunosuppressive and promoted interferon responses. In an inflamed, \"hot\", mouse model of lung cancer, RASG12C(ON) and SHP2 inhibitors in combination drive durable responses by suppressing tumour relapse and inducing development of immune memory, which can also be induced by combination of RASG12C(ON) and PD-1 inhibitors. In contrast, in an immune excluded, \"cold\", mouse model of lung cancer, combined RASG12C(ON) and SHP2 inhibition does not cause durable responses, but does sensitise tumours to immune checkpoint blockade, enabling efficient tumour rejection, accompanied by significant TME reorganization, including depletion of immunosuppressive innate immune cells and recruitment and activation of T and NK cells. These preclinical results demonstrate the potential of the combination of RASG12C(ON) inhibitors with SHP2 inhibitors to sensitize anti-PD-1 refractory tumours to immune checkpoint blockade by stimulating anti-tumour immunity as well as by targeting KRAS-driven proliferation in tumour cells.\nA Lipidomics Approach to Determine the Role of Lipids and its Crosstalk with Autophagy in Lung Cancer Metastasis\nAuthors: da Silva Rosa, S. C.; Alizadeh, J.; Vitorino, R.; Surendran, A.; Ravandi, A.; Kidane, B.; Ghavami, S.\nScore: 3.9, Published: 2024-01-24 DOI: 10.1101/2024.01.01.573842\nSummary/AbstractNon-small cell lung cancer (NSCLC) is among the most malignant tumors with high propensity for metastasis and is the leading cause of cancer-related death globally. Most patients present with regional and distant metastasis, associated with poor prognosis. Lipids may play an essential role in either activating or inhibiting detachment-induced apoptosis (anoikis), where the latter is a crucial mechanism to prevent metastasis, and it may have a cross-talk with autophagy. Autophagy has been shown to be induced in various human cancer metastasis, modulating tumor cell motility and invasion, cancer cell differentiation, resistance to anoikis, and epithelial to mesenchymal transition. Hence, it may play a crucial role in the transition of benign to malignant phenotypes, the core of metastasis initiation. Here, we provide a method we have established in our laboratory for detecting of lipids in attached and detached non-small lung cancer cells and show how to analyze lipidomics data to find its correlation with autophagy-related pathways. Graphical abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=95 SRC=\"FIGDIR/small/573842v2_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (24K): org.highwire.dtl.DTLVardef@1ad368forg.highwire.dtl.DTLVardef@d90e20org.highwire.dtl.DTLVardef@2e6accorg.highwire.dtl.DTLVardef@165b2ab_HPS_FORMAT_FIGEXP M_FIG C_FIG Workflow of A549 attached and detached cell culture. Cells are collected, lipids are extracted and subjected to LC/MS processing. Next, lipidomics resulting data is analyzed through various steps involving utilization of Metaboanlyst, HDMB, Swiss Target Prediction, and String.\n",
  "wordCount" : "2216",
  "inLanguage": "en",
  "datePublished": "2024-01-28T10:37:59Z",
  "dateModified": "2024-01-28T10:37:59Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on January 28, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.16.575968">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.16.575968" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.16.575968">
        <p class="paperTitle">Discovery of Highly Active Kynureninases for Cancer Immunotherapy through Protein Language Model</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.16.575968" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.16.575968" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eom, H.; Cho, K. S.; Lee, J.; Kim, S.; Park, S.; Kim, H.; Yang, J.; Han, Y.-H.; Lee, J.; Seok, C.; Lee, M. S.; Song, W. J.; Steinegger, M.</p>
        <p class="info">Score: 27.1, Published: 2024-01-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.16.575968' target='https://doi.org/10.1101/2024.01.16.575968'> 10.1101/2024.01.16.575968</a></p>
        <p class="abstract">Overcoming the immunosuppressive tumor microenvironment is a promising strategy in anticancer therapy. L-kynurenine, a strong immunosuppressive metabolite can be degraded through kynureninases. Through homology searches and protein language models, we identified and then experimentally determined the efficacy of four top-ranked kynureninases. The catalytically most active one nearly doubles turnover number over the prior best, reducing tumor weight by 3.42 times in mouse model comparisons, and thus, presenting substantial therapeutic potential.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576103">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576103" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576103">
        <p class="paperTitle">Chromosome 7 to the rescue: overcoming chromosome 10 loss in gliomas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576103" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576103" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nair, N. U.; Schaffer, A. A.; Gertz, M. E.; Cheng, K.; Zerbib, J.; Sahu, A. D.; Leor, G.; Shulman, E. D.; Aldape, K. D.; Ben-David, U.; Ruppin, E.</p>
        <p class="info">Score: 24.5, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576103' target='https://doi.org/10.1101/2024.01.17.576103'> 10.1101/2024.01.17.576103</a></p>
        <p class="abstract">The co-occurrence of chromosome 10 loss and chromosome 7 gain in gliomas is the most frequent loss-gain co-aneuploidy pair in human cancers, a phenomenon that has been investigated without resolution since the late 1980s. Expanding beyond previous gene-centric studies, we investigate the co-occurrence in a genome-wide manner taking an evolutionary perspective. First, by mining large tumor aneuploidy data, we predict that the more likely order is 10 loss followed by 7 gain. Second, by analyzing extensive genomic and transcriptomic data from both patients and cell lines, we find that this co-occurrence can be explained by functional rescue interactions that are highly enriched on 7, which can possibly compensate for any detrimental consequences arising from the loss of 10. Finally, by analyzing transcriptomic data from normal, non-cancerous, human brain tissues, we provide a plausible reason why this co-occurrence happens preferentially in cancers originating in certain regions of the brain.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.576566">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.576566" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.576566">
        <p class="paperTitle">The temporal evolution of cancer hallmarks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.576566" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.576566" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gourmet, L.; Ramazzotti, D.; Mallick, P.; Walker-Samuel, S.; Zapata, L.</p>
        <p class="info">Score: 14.9, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.576566' target='https://doi.org/10.1101/2024.01.21.576566'> 10.1101/2024.01.21.576566</a></p>
        <p class="abstract">Cancer hallmarks describe key physiological characteristics that distinguish cancers from normal tissues. The temporal order in which these hallmarks appear during cancer pathogenesis is of interest from both evolutionary and clinical perspectives but has not been investigated before. Here, we order hallmarks based on the allele frequency and selective advantage of mutations in cancer hallmark genes across &gt;10k untreated primary tumors and &gt;8K healthy tissues. Using this novel approach, we identified a common evolutionary trajectory for 27 of 32 cancer types with genomic instability as the first and immune evasion as the last hallmark. We demonstrated widespread positive selection in cancer and strong negative selection in normal tissues for all hallmarks. Notable exceptions to the hallmark ordering in tumours were melanomas (uveal and skin) suggesting that strong environmental factors could disrupt common evolutionary paths. Clustering of hallmark trajectories across patients revealed 2 clusters defined by early or late genomic instability, with differential prognosis. Our study is the first to identify the temporal order of cancer hallmarks during tumorigenesis and demonstrate a prognostic value that could be exploited for early detection and risk stratification across multiple cancer types.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.21.576304">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.21.576304" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.21.576304">
        <p class="paperTitle">Mammalian SWI/SNF complex activity regulates POU2F3 and constitutes a targetable dependency in small cell lung cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.21.576304" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.21.576304" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Duplaquet, L.; So, K.; Ying, A. W.; Li, X.; Li, Y.; Qiu, X.; Li, R.; Singh, S.; Wu, X. S.; Liu, Q.; Qi, J.; Somerville, T. D. D.; Heiling, H.; Mazzola, E.; Lee, Y.; Zoller, T.; Vakoc, C. R.; Doench, J. G.; Forrester, W. C.; Abrams, T.; Long, H. W.; Niederst, M. J.; Kadoch, C.; Oser, M. G.</p>
        <p class="info">Score: 12.0, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.21.576304' target='https://doi.org/10.1101/2024.01.21.576304'> 10.1101/2024.01.21.576304</a></p>
        <p class="abstract">Small cell lung cancers (SCLC) are comprised of heterogeneous subtypes marked by lineage-specific transcription factors, including ASCL1, NEUROD1, and POU2F3. POU2F3-positive SCLC, [~]12% of all cases, are uniquely dependent on POU2F3 itself; as such, approaches to attenuate POU2F3 expression may represent new therapeutic opportunities. Here using genome-scale screens for regulators of POU2F3 expression and SCLC proliferation, we define mSWI/SNF complexes, including non-canonical BAF (ncBAF) complexes, as top dependencies specific to POU2F3-positive SCLC. Notably, clinical-grade pharmacologic mSWI/SNF inhibition attenuates proliferation of all POU2F3-positive SCLCs, while disruption of ncBAF via BRD9 degradation is uniquely effective in pure non-neuroendocrine POU2F3-SCLCs. mSWI/SNF maintains accessibility over gene loci central to POU2F3-mediated gene regulatory networks. Finally, chemical targeting of SMARCA4/2 mSWI/SNF ATPases and BRD9 decrease POU2F3-SCLC tumor growth and increase survival in vivo. Taken together, these results characterize mSWI/SNF-mediated global governance of the POU2F3 oncogenic program and suggest mSWI/SNF inhibition as a therapeutic strategy for SCLC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.17.576110">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.17.576110" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.17.576110">
        <p class="paperTitle">Integrated cancer cell-specific single-cell RNA-seq datasets of immune checkpoint blockade-treated patients</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.17.576110" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.17.576110" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gondal, M. N.; Cieslik, M.; Chinnaiyan, A. M.</p>
        <p class="info">Score: 6.5, Published: 2024-01-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.17.576110' target='https://doi.org/10.1101/2024.01.17.576110'> 10.1101/2024.01.17.576110</a></p>
        <p class="abstract">Immune checkpoint blockade (ICB) therapies have emerged as a promising avenue for the treatment of various cancers. Despite their success, the efficacy of these treatments is variable across patients and cancer types. Numerous single-cell RNA-sequencing (scRNA-seq) studies have been conducted to unravel cell-specific responses to ICB treatment. However, these studies are limited in their sample sizes and require advanced coding skills for exploration. Here, we have compiled eight scRNA-seq datasets from nine cancer types, encompassing 174 patients, and 90,270 cancer cells. This compilation forms a unique resource tailored for investigating how cancer cells respond to ICB treatment across cancer types. We meticulously curated, quality-checked, pre-processed, and analyzed the data, ensuring easy access for researchers. Moreover, we designed a user-friendly interface for seamless exploration. By sharing the code and data for creating these interfaces, we aim to assist fellow researchers. These resources offer valuable support to those interested in leveraging and exploring single-cell datasets across diverse cancer types, facilitating a comprehensive understanding of ICB responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573403">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573403" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573403">
        <p class="paperTitle">Mechanisms underpinning osteosarcoma genome complexity and evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573403" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573403" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Espejo Valle-Inclan, J.; De Noon, S.; Trevers, K.; Elrick, H.; Tanguy, M.; Butters, T.; Muyas, F.; Amary, F.; Tirabosco, R.; Giess, A.; Sosinky, A.; Elgar, G.; Flanagan, A. M.; Cortes-Ciriano, I.</p>
        <p class="info">Score: 31.1, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573403' target='https://doi.org/10.1101/2023.12.29.573403'> 10.1101/2023.12.29.573403</a></p>
        <p class="abstract">Osteosarcoma is the most common primary cancer of bone with a peak incidence in children and young adults. Despite progress, the genomic aberrations underpinning osteosarcoma evolution remain poorly understood. Using multi-region whole-genome sequencing, we find that chromothripsis is an ongoing mutational process, occurring subclonally in 74% of tumours. Chromothripsis drives the acquisition of oncogenic mutations and generates highly unstable derivative chromosomes, the evolution of which drives clonal diversification and intra-tumour heterogeneity. In addition, we report a novel mechanism, loss-translocation-amplification (LTA) chromothripsis, which mediates rapid malignant transformation and punctuated evolution in about half of paediatric and adult high-grade osteosarcomas. Specifically, a single double-strand break triggers concomitant TP53 inactivation and segmental amplifications, often amplifying oncogenes to high copy numbers in extrachromosomal circular DNA elements through breakage-fusion-bridge cycles involving multiple chromosomes. LTA chromothripsis is detected at low frequency in soft-tissue sarcomas, but not in epithelial cancers, including those driven by TP53 mutation. Finally, we identify genome-wide loss of heterozygosity as a strong prognostic indicator for high-grade osteosarcoma.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.24.576987">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.24.576987" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.24.576987">
        <p class="paperTitle">SMAC mimetics overcome apoptotic resistance in ovarian cancer through MSLN-TNF alpha axis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.24.576987" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.24.576987" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coelho, R.; Seashore-Ludlow, B.; Schutz, S.; Christopher Lombardo, F.; Moussaud-Lamodiere, E.; Casanova, R.; Ficek-Pascual, J.; Brunhilde Labrosse, K.; Lopez-Nunez, M.; Rimmer, N.; Fedier, A.; Lima, R.; Montavon Sartorius, C.; Kurzeder, C.; Singer, F.; Bertolini, A.; The TumorProfiler Consortium,  ; Ratsch, G.; Bodenmiller, B.; Kallioniemi, O.; Ostling, P.; David, L.; Heinzelmann-Schwarz, V.; Jacob, F.</p>
        <p class="info">Score: 3.7, Published: 2024-01-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.24.576987' target='https://doi.org/10.1101/2024.01.24.576987'> 10.1101/2024.01.24.576987</a></p>
        <p class="abstract">Resistance to chemotherapy and PARPi inhibitors remains a critical challenge in the treatment of epithelial ovarian cancer, mainly due to disabled apoptotic responses in tumor cells. Given mesothelin&#39;s pivotal role in ovarian cancer and its restricted expression in healthy tissues, we conducted a drug-screening discovery analysis across a range of genetically modified cancer cells to unveil mesothelin&#39;s therapeutic impact. We observed enhanced cell death in low mesothelin expressing cancer cells when exposed to a second mitochondria-derived activator of caspases (SMAC) mimetics, and demonstrated a compelling synergy when combined with chemotherapy in ex vivo patient-derived cultures and zebrafish tumor xenografts. Mechanistically, the addition of the SMAC mimetics drug birinapant to either carboplatin or paclitaxel triggered the activation of the Caspase 8-dependent apoptotic program facilitated by TNF alpha signaling. Multimodal analysis of neoadjuvant-treated patient samples further revealed an association between tumor-associated macrophages and the activation of TNF alpha-related pathways. Our proposed bimodal treatment shows promise in enhancing the clinical management of patients by harnessing the potential of SMAC mimetics alongside conventional chemotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.20.576487">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.20.576487" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.20.576487">
        <p class="paperTitle">A CANCER PERSISTENT DNA REPAIR CIRCUIT DRIVEN BY MDM2, MDM4 (MDMX), AND MUTANT P53 FOR RECRUITMENT OF MDC1 AND 53BP1 TO CHROMATIN</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.20.576487" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.20.576487" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ellison, V.; Polotskaia, A.; Leybengrub, P.; Qiu, W.; Lee, R.; Hendrickson, R. C.; Hu, W.; Bargonetti, J.</p>
        <p class="info">Score: 3.4, Published: 2024-01-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.20.576487' target='https://doi.org/10.1101/2024.01.20.576487'> 10.1101/2024.01.20.576487</a></p>
        <p class="abstract">The influence of the metastasis promoting proteins mutant p53 (mtp53) and MDM2 on Cancer Persistent Repair (CPR) to promote cancer cell survival is understudied. Interactions between the DNA repair choice protein 53BP1 and wild type tumor suppressor protein p53 (wtp53) regulates cell cycle control. Cancer cells often express elevated levels of transcriptionally inactive missense mutant p53 (mtp53) that interacts with MDM2 and MDM4/MDMX (herein called MDMX). The ability of mtp53 to maintain a 53BP1 interaction while in the context of interactions with MDM2 and MDMX has not been described. We asked if MDM2 regulates chromatin-based phosphorylation events in the context of mtp53 by comparing the chromatin of T47D breast cancer cells with and without MDM2 in a phospho-peptide stable isotope labeling in cell culture (SILAC) screen. We found reduced phospho-53BP1 chromatin association, which we confirmed by chromatin fractionation and immunofluorescence in multiple breast cancer cell lines. We used the Proximity Ligation Assay (PLA) in breast cancer cell lines and detected 53BP1 in close proximity to mtp53, MDM2, and the DNA repair protein MDC1. Through disruption of the mtp53-MDM2 interaction, by either Nutlin 3a or a mtp53 R273H C-terminal deletion, we uncovered that mtp53 was required for MDM2-53BP1 interaction foci. Our data suggests that mtp53 works with MDM2 and 53BP1 to promote CPR and cell survival.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.15.575765">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.15.575765" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.15.575765">
        <p class="paperTitle">Combining RASG12C(ON) inhibitor with SHP2 inhibition sensitises immune excluded lung tumours to immune checkpoint blockade: a strategy for turning cold tumours hot</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.15.575765" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.15.575765" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Anastasiou, P.; Moore, C.; Rana, S.; de Castro, A.; Tomaschko, M.; Boumelha, J.; Mugarza, E.; Blaj, C.; de Carne Trecesson, S.; Goldstone, R.; Smith, J.; Quintana, E.; Molina-Arcas, M.; Downward, J.</p>
        <p class="info">Score: 21.6, Published: 2024-01-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.15.575765' target='https://doi.org/10.1101/2024.01.15.575765'> 10.1101/2024.01.15.575765</a></p>
        <p class="abstract">Mutant selective drugs targeting the inactive, GDP-bound form of KRASG12C have been approved for use in lung cancer, but responses are short-lived due to rapid development of resistance. In this study we use a novel covalent tri-complex inhibitor, RMC-4998, that targets RASG12C in its active, GTP-bound form to investigate treatment of KRAS mutant lung cancer in various immune competent mouse models. While this RASG12C(ON) inhibitor was more potent than the KRASG12C(OFF) inhibitor adagrasib, rapid pathway reactivation was still observed. This could be delayed using combined treatment with a SHP2 inhibitor, RMC-4550, which not only impacted RAS pathway signalling within the tumour cells but also remodelled the tumour microenvironment (TME) to be less immunosuppressive and promoted interferon responses. In an inflamed, &#34;hot&#34;, mouse model of lung cancer, RASG12C(ON) and SHP2 inhibitors in combination drive durable responses by suppressing tumour relapse and inducing development of immune memory, which can also be induced by combination of RASG12C(ON) and PD-1 inhibitors. In contrast, in an immune excluded, &#34;cold&#34;, mouse model of lung cancer, combined RASG12C(ON) and SHP2 inhibition does not cause durable responses, but does sensitise tumours to immune checkpoint blockade, enabling efficient tumour rejection, accompanied by significant TME reorganization, including depletion of immunosuppressive innate immune cells and recruitment and activation of T and NK cells. These preclinical results demonstrate the potential of the combination of RASG12C(ON) inhibitors with SHP2 inhibitors to sensitize anti-PD-1 refractory tumours to immune checkpoint blockade by stimulating anti-tumour immunity as well as by targeting KRAS-driven proliferation in tumour cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.01.573842">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.01.573842" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.01.573842">
        <p class="paperTitle">A Lipidomics Approach to Determine the Role of Lipids and its Crosstalk with Autophagy in Lung Cancer Metastasis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.01.573842" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.01.573842" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: da Silva Rosa, S. C.; Alizadeh, J.; Vitorino, R.; Surendran, A.; Ravandi, A.; Kidane, B.; Ghavami, S.</p>
        <p class="info">Score: 3.9, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.01.573842' target='https://doi.org/10.1101/2024.01.01.573842'> 10.1101/2024.01.01.573842</a></p>
        <p class="abstract">Summary/AbstractNon-small cell lung cancer (NSCLC) is among the most malignant tumors with high propensity for metastasis and is the leading cause of cancer-related death globally. Most patients present with regional and distant metastasis, associated with poor prognosis. Lipids may play an essential role in either activating or inhibiting detachment-induced apoptosis (anoikis), where the latter is a crucial mechanism to prevent metastasis, and it may have a cross-talk with autophagy. Autophagy has been shown to be induced in various human cancer metastasis, modulating tumor cell motility and invasion, cancer cell differentiation, resistance to anoikis, and epithelial to mesenchymal transition. Hence, it may play a crucial role in the transition of benign to malignant phenotypes, the core of metastasis initiation. Here, we provide a method we have established in our laboratory for detecting of lipids in attached and detached non-small lung cancer cells and show how to analyze lipidomics data to find its correlation with autophagy-related pathways.

Graphical abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=95 SRC=&#34;FIGDIR/small/573842v2_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (24K):
org.highwire.dtl.DTLVardef@1ad368forg.highwire.dtl.DTLVardef@d90e20org.highwire.dtl.DTLVardef@2e6accorg.highwire.dtl.DTLVardef@165b2ab_HPS_FORMAT_FIGEXP  M_FIG C_FIG Workflow of A549 attached and detached cell culture. Cells are collected, lipids are extracted and subjected to LC/MS processing. Next, lipidomics resulting data is analyzed through various steps involving utilization of Metaboanlyst, HDMB, Swiss Target Prediction, and String.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
