<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="A compendium of Amplification-Related Gain Of Sensitivity (ARGOS) genes in human cancer
Authors: Rendo, V.; Schubert, M.; Khuu, N.; Suarez Peredo Rodriguez, M. F.; Huang, K.; Swift, M.; He, Y.; Zerbib, J.; Smith, R.; Raaijmakers, J.; Bandopadhayay, P.; Guenther, L. M.; Hwang, J. H.; Iniguez, A.; Moody, S.; Seo, J.-H.; Stover, E.; Garraway, L.; Hahn, W. C.; Stegmaier, K.; Medema, R. H.; Chowdhury, D.; Colome-Tatche, M.; Ben-David, U.; Beroukhim, R.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="A compendium of Amplification-Related Gain Of Sensitivity (ARGOS) genes in human cancer
Authors: Rendo, V.; Schubert, M.; Khuu, N.; Suarez Peredo Rodriguez, M. F.; Huang, K.; Swift, M.; He, Y.; Zerbib, J.; Smith, R.; Raaijmakers, J.; Bandopadhayay, P.; Guenther, L. M.; Hwang, J. H.; Iniguez, A.; Moody, S.; Seo, J.-H.; Stover, E.; Garraway, L.; Hahn, W. C.; Stegmaier, K.; Medema, R. H.; Chowdhury, D.; Colome-Tatche, M.; Ben-David, U.; Beroukhim, R." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-27T10:39:30+00:00" />
<meta property="article:modified_time" content="2023-12-27T10:39:30+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="A compendium of Amplification-Related Gain Of Sensitivity (ARGOS) genes in human cancer
Authors: Rendo, V.; Schubert, M.; Khuu, N.; Suarez Peredo Rodriguez, M. F.; Huang, K.; Swift, M.; He, Y.; Zerbib, J.; Smith, R.; Raaijmakers, J.; Bandopadhayay, P.; Guenther, L. M.; Hwang, J. H.; Iniguez, A.; Moody, S.; Seo, J.-H.; Stover, E.; Garraway, L.; Hahn, W. C.; Stegmaier, K.; Medema, R. H.; Chowdhury, D.; Colome-Tatche, M.; Ben-David, U.; Beroukhim, R."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "A compendium of Amplification-Related Gain Of Sensitivity (ARGOS) genes in human cancer\nAuthors: Rendo, V.; Schubert, M.; Khuu, N.; Suarez Peredo Rodriguez, M. F.; Huang, K.; Swift, M.; He, Y.; Zerbib, J.; Smith, R.; Raaijmakers, J.; Bandopadhayay, P.; Guenther, L. M.; Hwang, J. H.; Iniguez, A.; Moody, S.; Seo, J.-H.; Stover, E.; Garraway, L.; Hahn, W. C.; Stegmaier, K.; Medema, R. H.; Chowdhury, D.; Colome-Tatche, M.; Ben-David, U.; Beroukhim, R.",
  "keywords": [
    
  ],
  "articleBody": " A compendium of Amplification-Related Gain Of Sensitivity (ARGOS) genes in human cancer\nAuthors: Rendo, V.; Schubert, M.; Khuu, N.; Suarez Peredo Rodriguez, M. F.; Huang, K.; Swift, M.; He, Y.; Zerbib, J.; Smith, R.; Raaijmakers, J.; Bandopadhayay, P.; Guenther, L. M.; Hwang, J. H.; Iniguez, A.; Moody, S.; Seo, J.-H.; Stover, E.; Garraway, L.; Hahn, W. C.; Stegmaier, K.; Medema, R. H.; Chowdhury, D.; Colome-Tatche, M.; Ben-David, U.; Beroukhim, R.; Foijer, F.\nScore: 9.1, Published: 2023-12-19 DOI: 10.1101/2023.12.16.571980\nChromosomal gains are among the most frequent somatic genetic alterations occurring in cancer. While the effect of sustained oncogene expression has been characterized, the impact of copy-number gains affecting collaterally-amplified \"bystander\" genes on cellular fitness remains less understood. To investigate this, we built a comprehensive map of dosage compensations across human cancers by integrating expression and copy number profiles from over 8,000 TCGA tumors and CCLE cell lines. Further, we analyzed the effect of gene overexpression across 17 human cancer ORF screens to provide an overview of genes that prove toxic to cancer cells when overexpressed. Combining these two independent approaches we propose a class of Amplification-Related Gain Of Sensitivity (ARGOS) genes. These genes are located in commonly amplified regions of the genome, have lower expression levels than expected by their copy-number status, and are toxic to cancer cells when overexpressed. We experimentally validated CDKN1A and RBM14 as high-confidence pan-cancer ARGOS genes in lung and breast cancer cell line models. We additionally suggest that RBM14s mechanism of toxicity involves altered DNA damage response and innate immune signaling processes following gene overexpression. Finally, we provide a comprehensive catalog of compensated, toxic, and ARGOS genes as a community resource.\nThree-dimensional assessments are necessary to determine the true spatial tissue composition of diseased tissues\nAuthors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.\nScore: 37.1, Published: 2023-12-06 DOI: 10.1101/2023.12.04.569986\nMethods for partially resolved cellular profiling has enabled in-depth quantitative tissue mapping via thinly cut sections to study inter-patient and intra-patient differences in normal human anatomy and disease onset and progression. These methods often profile extremely limited spatial regions, which may impact the evaluation of heterogeneity due to tissue sub-sampling. Here, we applied CODA, a deep learning-based tissue mapping platform, to reconstruct the 3D microanatomy of surgically resected human pancreas biospecimens obtained from patients diagnosed with pancreatic cancer. To compare differences in the inter- and intra-tumoral heterogeneity, we assessed the bulk and spatially resolved tissue composition of a cohort of two-dimensional (2D) whole slide images (WSIs), and a cohort of 3D serially sectioned and reconstructed tissues of pancreata. Here, we show the strength of using 3D as the gold standard, by measuring the information loss and sampling problems when using WSIs and TMAs. We demonstrate that spatial correlation in microanatomical tissue content decays significantly within a span of just a few microns within tumors. As a corollary, hundreds of TMAs and tens of WSIs are required to estimate spatial bulk tumor composition with \u003c10% error in any given pancreatic tumor. In sum, we demonstrate that 3D assessments are necessary to accurately assess tumor burden and tissue composition. These preliminary results show the importance of rate of sampling necessary to more reliably assess spatially resolved tissue composition.\nLarge pan-cancer cell screen coupled to (phospho-) proteomics underscores high-dose vitamin C as a potent anti-cancer agent\nAuthors: Valles Marti, A.; BoÌˆttger, F.; Yau, E.; Tejjani, K.; Meijs, L.; Sharma, S.; Mumtaz, M.; Le Large, T.; Erozenci, A.; Dekker, D.; Schelfhorst, T.; Medema, J. P.; Bijnsdorp, I. V.; Knol, J. C.; Piersma, S. R.; Pham, T. V.; Giovannetti, E.; Jimenez, C. R.\nScore: 6.0, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572293\nIncreasing preclinical and clinical evidence has positioned high-dose vitamin C as a promising anti-cancer treatment that merits more clinical attention. Multiple cytotoxicity mechanisms have been described, including pro-oxidant effects. To contribute to the preclinical understanding of the broad pan-cancer effects of high-dose vitamin C in a global manner, we determined the IC50 of a large panel of cancer cell lines (n=51) representing 7 solid tumour types and generated proteome data. The majority of cell lines were highly sensitive (IC50 range 0.036-10mM, mean 1.7 {+/-} 0.4 mM), well below a clinically achievable dose. The proteome data (\u003e5000 proteins per sample), showed that high sensitivity is associated with proliferation, as indicated by functional enrichment of cell cycle, RNA splicing and chromatin organization, while lower sensitivity is linked to extracellular vesicles, glycolysis, fatty acid metabolism and mitochondria. Moreover, (phospho-)proteome analysis of on-treatment vitamin C effects on four pancreatic ductal adenocarcinoma (PDAC) cells dosed at a range of IC50 values (Hs766 T, 2 mM; Capan-2, 0.6 mM; PANC-1, 0.14 mM and Suit-2, 0.1 mM) revealed, next to cell line specific effects, down-modulation of AKT-MTOR signalling and immune suppressive signalling, while IFN- response was enhanced upon vitamin C. Altogether, our comprehensive pharmacological and (phospho-)proteome analysis is the first to assess cancer vulnerabilities and effects of vitamin C on a large cancer cell line panel and underscores the potential of high-dose vitamin C as an anti-cancer agent.\nComprehensive chromatome profiling identifies metabolic enzymes on chromatin in healthy and cancer cells\nAuthors: Kourtis, S.; Guirola, M.; Pardo-Lorente, N.; Ghose, R.; Garcia-Cao, M.; Ganez-Zapater, A.; Haynes, S.; Fontaine, F.; Muller, A.; Sdelci, S.\nScore: 5.3, Published: 2023-12-08 DOI: 10.1101/2023.12.06.570368\nMetabolic and epigenetic rewiring are widely considered hallmarks of cancer, with emerging evidence of crosstalk between them. Anecdotal evidence of metabolic enzymes moonlighting in the chromatin environment has suggested how this crosstalk might be facilitated, but the extent of nuclear relocalization of metabolic enzymes remains elusive. Here, we provide a comprehensive chromatin proteomics resource across cancer lineages as well as healthy samples and demonstrate that metabolic enzyme moonlighting on chromatin is widespread across tissues and pathways. We show that the abundance of metabolic enzymes on chromatin is tissue-specific, with oxidative phosphorylation proteins depleted in lung cancer samples, perhaps suggesting an interplay between cell identity and nuclear metabolism. Finally, we explore metabolic functions in the chromatin environment and show that one-carbon folate enzymes are associated with DNA damage and repair processes, providing an approach to explore non-canonical functions of metabolic enzymes.\nCell Types or Cell States? An Investigation of Adrenergic and Mesenchymal Cell Phenotypes in Neuroblastoma\nAuthors: Bukkuri, A.; Andersson, S.; Brown, J. S.; Hammarlund, E. U.; Mohlin, S.\nScore: 4.3, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572368\nNeuroblastoma is a pediatric cancer that exhibits two cellular phenotypes: adrenergic (ADRN) and mesenchymal (MES). ADRN is differentiated and therapy-sensitive, while MES is less differentiated with elevated therapy resistance. To understand neuroblastoma and its treatment response, it is important to elucidate how these phenotypes impact the eco-evolutionary dynamics of cancer cell populations and whether they represent distinct cell types or dynamic cell states. Here, we show that neuroblastoma cells undergo an ADRN to a MES phenotypic switch under chemotherapy treatment. We use a strong inference approach to generate four hypotheses on how this switch may occur: cell types without resistance, cell types with resistance, cell states without resistance, and cell states with resistance. For each of these hypotheses, we create theoretical models to make qualitative predictions about their resulting eco-evolutionary dynamics. Our results provide a framework to further experimentally determine whether ADRN and MES phenotypes are distinct cell types or dynamic cell states.\nCDC20 determines the sensitivity to spindle assembly checkpoint (SAC) inhibitors\nAuthors: Zheng, S.; Raz, L.; Zhou, L.; Cohen-Sharir, Y.; Tian, R.; Wardenaar, R.; Broekhuis, M. J. C.; Suarez Peredo Rodriguez, M. F.; Wobben, S.; van den Brink, A.; Bakker, P.; Foijer, F.; Ben-David, U.\nScore: 4.0, Published: 2023-12-23 DOI: 10.1101/2023.12.21.572559\nSpindle assembly checkpoint (SAC) inhibitors are a recently developed class of drugs that perturb the regulation of chromosome segregation during division, induce chromosomal instability (CIN), and eventually lead to cell death. While they are currently in clinical trials for solid cancers, biomarkers to predict the response to SAC inhibitors are still lacking. We recently reported that aneuploid cancer cells are preferentially sensitive to SAC inhibition. Here, we investigated the molecular determinants of the response to SAC inhibition that underlies the differential sensitivity of aneuploid cells to these drugs. We found that this response was largely driven by the expression of CDC20, a main activator of the anaphase-promoting complex (APC/C), rather than by APC/C itself. Mechanistically, we discovered that CDC20 depletion prolonged metaphase duration, diminished mitotic errors, and reduced sensitivity to SAC inhibition. Aneuploid cells expressed high levels of CDC20 and experienced shorter metaphases and multiple mitotic errors, resulting in increased long-term sensitivity to SAC inhibition. Our findings propose high CDC20 expression as a favorable biomarker for SAC inhibition therapy and as an aneuploidy-induced therapeutic vulnerability.\nInactivation of p53 drives breast cancer brain metastasis by altering fatty acid metabolism\nAuthors: Laue, K.; Pozzi, S.; Cohen-Sharir, Y.; Winkler, T.; Eliezer, Y.; Israeli Dangoor, S.; Leikin-Frenkel, A.; Lange, K.; Zerbib, J.; Ricci, A. A.; Sacconi, A.; Berthelet, J.; Schaffer, A.; Shi, W.; Liao, Y.; Barshack, I.; Medyouf, H.; Merino, D.; Blandino, G.; Bertero, L.; Satchi-Fainaro, R.; Ben-David, U.\nScore: 4.0, Published: 2023-12-20 DOI: 10.1101/2023.12.20.572490\nBrain metastasis (BM) is a dire prognosis across cancer types. It is largely unknown why some tumors metastasize to the brain whereas others do not. We analyzed genomic and transcriptional data from clinical samples of breast cancer BM (BCBM) and found that nearly all of them carried p53-inactivating genetic alterations through mutations, copy-number loss, or both. Importantly, p53 pathway activity was already perturbed in primary tumors giving rise to BCBM, often by loss of the entire 17p chromosome-arm. This association was recapitulated across other carcinomas. Experimentally, p53 knockout was sufficient to drastically increase BCBM formation and growth in vivo, providing a causal link between p53 inactivation and brain tropism. Mechanistically, p53-deficient BC cells exhibited altered lipid metabolism, particularly increased fatty acid (FA) synthesis and uptake, which are characteristic of brain-metastasizing cancer cells. FA metabolism was further enhanced by astrocytes in a p53-dependent manner, as astrocyte-conditioned medium increased FASN, SCD1, and CD36 expression and activity, and enhanced the survival, proliferation and migration of p53-deficient cancer cells. Consequently, these cells were more sensitive than p53-competent cells to FA synthesis inhibitors, in isogenic cell cultures, in BCBM-derived spheroids, and across dozens of BC cell lines. Lastly, a significant association was observed between p53 inactivation, astrocyte infiltration, and SCD1 expression in clinical human BCBM samples. In summary, our study identifies p53 inactivation as a driver of BCBM and potentially of BM in general; suggests a p53-dependent effect of astrocytes on BC cell behavior; and reveals FA metabolism as an underlying, therapeutically-targetable molecular mechanism.\nA rise in double-strand breaks sensitizes tumours to oxidative metabolism inhibitors\nAuthors: Medina-Jover, F.; Figueras, A.; Lahiguera, A.; Guillen, P.; Espin, R.; Pardo, M. A.; Pujana, M. A.; Berra, E.; Villanueva, A.; Bernat, A.; Romeo, M.; Perales, J. C.; Vinals, F.\nScore: 2.9, Published: 2023-12-20 DOI: 10.1101/2023.12.19.572355\nABSTRACTDouble strand brakes (DSB) accumulate in cellular DNA as a result of deficiencies in homologous recombination repair systems, such as mutations in BRCA genes, or upon antitumoral treatments. In the present study we show that the accumulation of DSB, regardless of its origins, leads to a shift towards oxidative metabolism. We have identified that DSB-induced reactive oxygen species (ROS) promote the activation of NRF2 which downregulates the glycolytic transcription factor HIF-1. HIF-1 inhibition is a key step in this metabolic shift, because leads to the reduction of PDHK1 levels and the consequential activation of pyruvate dehydrogenase, a mitochondrial gatekeeper of cellular metabolism, promoting this metabolic shift. Remarkably, after the induction of DSBs, the tumour is more sensitive to the inhibition of oxidative metabolism since both treatments synergize in vivo, resulting in reduced tumour growth. Therefore, we demonstrate a significant feedback between DSBs induction and cancer cell metabolism that ultimately limits the cells potential for metabolic plasticity, hence sensitizing it to the action of counteracting drugs.\nPARP inhibitors enhance reovirus-mediated cell killing through the death-inducing signaling complex (DISC) with an associated NF-kappa B-regulated immune response.\nAuthors: Kyula-Currie, J.; Roulstone, V.; Wright, J.; Butera, F.; Legrand, A.; Elliott, R.; McLaughlin, M.; Bozhanova, G.; Krastev, D. B.; Pettitt, S.; Tenev, T.; Dillon, M. T. B.; Foo, S.; Patin, E. C.; Jennings, V.; Chan, C.; Appleton, E.; Pedersen, M.; Rullan, A.; Choudhary, J.; Bakal, C.; Meier, P.; Lord, C. J.; Melcher, A.; Harrington, K. J.\nScore: 2.8, Published: 2023-12-22 DOI: 10.1101/2023.12.21.572541\nOncolytic Reovirus type 3 Dearing (RT3D), is a naturally occurring double-stranded (ds) RNA virus that is under development as an oncolytic immunotherapy We used an unbiased high-throughput cytotoxicity screen of different targeted therapeutic agents with the aim of identifying potential drug-viral sensitizers to enhance RT3D tumour killing. Talazoparib, a clinical poly(ADP)-ribose polymerase 1 (PARP-1) inhibitor, was identified as a top hit and found to cause profound sensitisation to RT3D. This effect was not seen with other classes of oncolytic virus and was not mediated by enhanced viral replication or PARP inhibitor-related effects on the DNA damage response. RT3D interacts with retinoic acid-induced gene-1 (RIG-I) and activates PARP-1, with consequent PARylation of components of the extrinsic apoptosis pathway. Pharmacological and genetic inhibition of PARP-1 abrogates this PARylation and increases levels of extrinsic apoptosis, NF-kB signalling and pro-inflammatory cell death. Direct interaction between PARP-1 and RIG-I following RT3D/talazoparib treatment is a key factor in activating downstream signaling pathways that lead to IFN-{beta} and TNF-/TRAIL production which, in turn, amplify the therapeutic effect through positive feedback. Critically, it was possible to phenocopy the effect of RT3D through the use of non-viral ds-RNA therapy and RIG-I agonism. In in vivo studies, we demonstrated profound combinatorial efficacy of RT3D and talazoparib in human A375 melanoma in immunodeficient mice. More impressively, in immunocompetent mouse models of 4434 murine melanoma, we achieved 100% tumour control and protection from subsequent tumour rechallenge with the combination regimen. Correlative immunophenotyping confirmed significant innate and adaptive immune activation with the combination of RT3D and PARP inhibition. Taken together, these data provide a clear line of sight to clinical translation of combined regimens of PARP inhibition or ds-RNA agonism, with either viral or non-viral agents, in tumour types beyond the relatively narrow confines of current licensed indications for PARP inhibition.\nSpatial dissection of cellular heterogeneity and functional crosstalk in the tumor microenvironments of gastric cancers\nAuthors: Kim, T.; Lee, S. H.; Lee, D.; Choi, J.; Oh, H. J.; Ham, I.-H.; Ryu, D.; Lee, S.-Y.; Han, D.-J.; Kim, S.; Lee, S.; Hur, H.\nScore: 2.5, Published: 2023-12-25 DOI: 10.1101/2023.12.24.573275\nSpatial transcriptomics offers unprecedented challenges in characterizing tumor ecosystems, encompassing tumor cells and their surrounding tumor microenvironments (TME) along with cellular interactions therein. Here, we conducted Visium-based spatial transcriptomics on nine primary gastric cancers (GC) to translate the spatial organization of GC ecosystems into the functional landscape of cellular crosstalk across malignant, stromal and immune cells. We identified three distinct GC subtypes of 3 immunogenic, 3 epithelial and 3 stromal GCs with elevated infiltration of immune, malignant cells and fibroblasts reflecting the heterogeneity of the cell types and their interactions in GC TME. GC spatial architecture was further delineated into 6 regional compartments with varying degree of TME infiltration, among which the fibroblast-enriched TME was associated with the upregulation of epithelial-to-mesenchymal transformation (EMT) and immune response in malignant and immune cells, respectively. Cell type-specific transcriptional dynamics representing cellular crosstalk between TME cells were further identified, e.g., the infiltration of malignant and endothelial cells promote the cellular proliferations of TME cells whereas the fibroblasts and immune cells are associated with pro-cancer and anti-cancer immunity, respectively. Ligand-receptor analysis further revealed that CCL2-expressing fibroblasts promote the pro-cancer signaling of immune cells including JAK-STAT3 signaling and inflammatory response in tumor-infiltration macrophages. The CCL2+fibroblasts and macrophages with activated JAK-STAT3 signals are co-localized in spatial GC architecture and their co-abundance was associated with unfavorable clinical outcomes. Taken together, GC spatial transcriptomes can be regionally and functionally delineated into functional cellular crosstalk involving multiple cell types with potential clinical relevance, particularly the interaction between CCL2+ fibroblasts and JAK-STAT3+macrophages contributing to cancer-favoring immune contexture in GC TME.\n",
  "wordCount" : "2661",
  "inLanguage": "en",
  "datePublished": "2023-12-27T10:39:30Z",
  "dateModified": "2023-12-27T10:39:30Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 27, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.16.571980">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.16.571980" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.16.571980">
        <p class="paperTitle">A compendium of Amplification-Related Gain Of Sensitivity (ARGOS) genes in human cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.16.571980" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.16.571980" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rendo, V.; Schubert, M.; Khuu, N.; Suarez Peredo Rodriguez, M. F.; Huang, K.; Swift, M.; He, Y.; Zerbib, J.; Smith, R.; Raaijmakers, J.; Bandopadhayay, P.; Guenther, L. M.; Hwang, J. H.; Iniguez, A.; Moody, S.; Seo, J.-H.; Stover, E.; Garraway, L.; Hahn, W. C.; Stegmaier, K.; Medema, R. H.; Chowdhury, D.; Colome-Tatche, M.; Ben-David, U.; Beroukhim, R.; Foijer, F.</p>
        <p class="info">Score: 9.1, Published: 2023-12-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.16.571980' target='https://doi.org/10.1101/2023.12.16.571980'> 10.1101/2023.12.16.571980</a></p>
        <p class="abstract">Chromosomal gains are among the most frequent somatic genetic alterations occurring in cancer. While the effect of sustained oncogene expression has been characterized, the impact of copy-number gains affecting collaterally-amplified &#34;bystander&#34; genes on cellular fitness remains less understood. To investigate this, we built a comprehensive map of dosage compensations across human cancers by integrating expression and copy number profiles from over 8,000 TCGA tumors and CCLE cell lines. Further, we analyzed the effect of gene overexpression across 17 human cancer ORF screens to provide an overview of genes that prove toxic to cancer cells when overexpressed. Combining these two independent approaches we propose a class of  Amplification-Related Gain Of Sensitivity (ARGOS) genes. These genes are located in commonly amplified regions of the genome, have lower expression levels than expected by their copy-number status, and are toxic to cancer cells when overexpressed. We experimentally validated CDKN1A and RBM14 as high-confidence pan-cancer ARGOS genes in lung and breast cancer cell line models. We additionally suggest that RBM14s mechanism of toxicity involves altered DNA damage response and innate immune signaling processes following gene overexpression. Finally, we provide a comprehensive catalog of compensated, toxic, and ARGOS genes as a community resource.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569986">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569986" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569986">
        <p class="paperTitle">Three-dimensional assessments are necessary to determine the true spatial tissue composition of diseased tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569986" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569986" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.</p>
        <p class="info">Score: 37.1, Published: 2023-12-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569986' target='https://doi.org/10.1101/2023.12.04.569986'> 10.1101/2023.12.04.569986</a></p>
        <p class="abstract">Methods for partially resolved cellular profiling has enabled in-depth quantitative tissue mapping via thinly cut sections to study inter-patient and intra-patient differences in normal human anatomy and disease onset and progression. These methods often profile extremely limited spatial regions, which may impact the evaluation of heterogeneity due to tissue sub-sampling. Here, we applied CODA, a deep learning-based tissue mapping platform, to reconstruct the 3D microanatomy of surgically resected human pancreas biospecimens obtained from patients diagnosed with pancreatic cancer. To compare differences in the inter- and intra-tumoral heterogeneity, we assessed the bulk and spatially resolved tissue composition of a cohort of two-dimensional (2D) whole slide images (WSIs), and a cohort of 3D serially sectioned and reconstructed tissues of pancreata. Here, we show the strength of using 3D as the gold standard, by measuring the information loss and sampling problems when using WSIs and TMAs. We demonstrate that spatial correlation in microanatomical tissue content decays significantly within a span of just a few microns within tumors. As a corollary, hundreds of TMAs and tens of WSIs are required to estimate spatial bulk tumor composition with &lt;10% error in any given pancreatic tumor. In sum, we demonstrate that 3D assessments are necessary to accurately assess tumor burden and tissue composition. These preliminary results show the importance of rate of sampling necessary to more reliably assess spatially resolved tissue composition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572293">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572293" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572293">
        <p class="paperTitle">Large pan-cancer cell screen coupled to (phospho-) proteomics underscores high-dose vitamin C as a potent anti-cancer agent</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572293" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572293" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Valles Marti, A.; BoÌˆttger, F.; Yau, E.; Tejjani, K.; Meijs, L.; Sharma, S.; Mumtaz, M.; Le Large, T.; Erozenci, A.; Dekker, D.; Schelfhorst, T.; Medema, J. P.; Bijnsdorp, I. V.; Knol, J. C.; Piersma, S. R.; Pham, T. V.; Giovannetti, E.; Jimenez, C. R.</p>
        <p class="info">Score: 6.0, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572293' target='https://doi.org/10.1101/2023.12.19.572293'> 10.1101/2023.12.19.572293</a></p>
        <p class="abstract">Increasing preclinical and clinical evidence has positioned high-dose vitamin C as a promising anti-cancer treatment that merits more clinical attention. Multiple cytotoxicity mechanisms have been described, including pro-oxidant effects. To contribute to the preclinical understanding of the broad pan-cancer effects of high-dose vitamin C in a global manner, we determined the IC50 of a large panel of cancer cell lines (n=51) representing 7 solid tumour types and generated proteome data. The majority of cell lines were highly sensitive (IC50 range 0.036-10mM, mean 1.7 {&#43;/-} 0.4 mM), well below a clinically achievable dose. The proteome data (&gt;5000 proteins per sample), showed that high sensitivity is associated with proliferation, as indicated by functional enrichment of cell cycle, RNA splicing and chromatin organization, while lower sensitivity is linked to extracellular vesicles, glycolysis, fatty acid metabolism and mitochondria. Moreover, (phospho-)proteome analysis of on-treatment vitamin C effects on four pancreatic ductal adenocarcinoma (PDAC) cells dosed at a range of IC50 values (Hs766 T, 2 mM; Capan-2, 0.6 mM; PANC-1, 0.14 mM and Suit-2, 0.1 mM) revealed, next to cell line specific effects, down-modulation of AKT-MTOR signalling and immune suppressive signalling, while IFN- response was enhanced upon vitamin C. Altogether, our comprehensive pharmacological and (phospho-)proteome analysis is the first to assess cancer vulnerabilities and effects of vitamin C on a large cancer cell line panel and underscores the potential of high-dose vitamin C as an anti-cancer agent.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.06.570368">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.06.570368" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.06.570368">
        <p class="paperTitle">Comprehensive chromatome profiling identifies metabolic enzymes on chromatin in healthy and cancer cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.06.570368" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.06.570368" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kourtis, S.; Guirola, M.; Pardo-Lorente, N.; Ghose, R.; Garcia-Cao, M.; Ganez-Zapater, A.; Haynes, S.; Fontaine, F.; Muller, A.; Sdelci, S.</p>
        <p class="info">Score: 5.3, Published: 2023-12-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.06.570368' target='https://doi.org/10.1101/2023.12.06.570368'> 10.1101/2023.12.06.570368</a></p>
        <p class="abstract">Metabolic and epigenetic rewiring are widely considered hallmarks of cancer, with emerging evidence of crosstalk between them. Anecdotal evidence of metabolic enzymes moonlighting in the chromatin environment has suggested how this crosstalk might be facilitated, but the extent of nuclear relocalization of metabolic enzymes remains elusive. Here, we provide a comprehensive chromatin proteomics resource across cancer lineages as well as healthy samples and demonstrate that metabolic enzyme moonlighting on chromatin is widespread across tissues and pathways. We show that the abundance of metabolic enzymes on chromatin is tissue-specific, with oxidative phosphorylation proteins depleted in lung cancer samples, perhaps suggesting an interplay between cell identity and nuclear metabolism. Finally, we explore metabolic functions in the chromatin environment and show that one-carbon folate enzymes are associated with DNA damage and repair processes, providing an approach to explore non-canonical functions of metabolic enzymes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572368">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572368" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572368">
        <p class="paperTitle">Cell Types or Cell States? An Investigation of Adrenergic and Mesenchymal Cell Phenotypes in Neuroblastoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572368" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572368" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bukkuri, A.; Andersson, S.; Brown, J. S.; Hammarlund, E. U.; Mohlin, S.</p>
        <p class="info">Score: 4.3, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572368' target='https://doi.org/10.1101/2023.12.20.572368'> 10.1101/2023.12.20.572368</a></p>
        <p class="abstract">Neuroblastoma is a pediatric cancer that exhibits two cellular phenotypes: adrenergic (ADRN) and mesenchymal (MES). ADRN is differentiated and therapy-sensitive, while MES is less differentiated with elevated therapy resistance. To understand neuroblastoma and its treatment response, it is important to elucidate how these phenotypes impact the eco-evolutionary dynamics of cancer cell populations and whether they represent distinct cell types or dynamic cell states. Here, we show that neuroblastoma cells undergo an ADRN to a MES phenotypic switch under chemotherapy treatment. We use a strong inference approach to generate four hypotheses on how this switch may occur: cell types without resistance, cell types with resistance, cell states without resistance, and cell states with resistance. For each of these hypotheses, we create theoretical models to make qualitative predictions about their resulting eco-evolutionary dynamics. Our results provide a framework to further experimentally determine whether ADRN and MES phenotypes are distinct cell types or dynamic cell states.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572559">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572559" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572559">
        <p class="paperTitle">CDC20 determines the sensitivity to spindle assembly checkpoint (SAC) inhibitors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572559" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572559" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zheng, S.; Raz, L.; Zhou, L.; Cohen-Sharir, Y.; Tian, R.; Wardenaar, R.; Broekhuis, M. J. C.; Suarez Peredo Rodriguez, M. F.; Wobben, S.; van den Brink, A.; Bakker, P.; Foijer, F.; Ben-David, U.</p>
        <p class="info">Score: 4.0, Published: 2023-12-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572559' target='https://doi.org/10.1101/2023.12.21.572559'> 10.1101/2023.12.21.572559</a></p>
        <p class="abstract">Spindle assembly checkpoint (SAC) inhibitors are a recently developed class of drugs that perturb the regulation of chromosome segregation during division, induce chromosomal instability (CIN), and eventually lead to cell death. While they are currently in clinical trials for solid cancers, biomarkers to predict the response to SAC inhibitors are still lacking. We recently reported that aneuploid cancer cells are preferentially sensitive to SAC inhibition. Here, we investigated the molecular determinants of the response to SAC inhibition that underlies the differential sensitivity of aneuploid cells to these drugs. We found that this response was largely driven by the expression of CDC20, a main activator of the anaphase-promoting complex (APC/C), rather than by APC/C itself. Mechanistically, we discovered that CDC20 depletion prolonged metaphase duration, diminished mitotic errors, and reduced sensitivity to SAC inhibition. Aneuploid cells expressed high levels of CDC20 and experienced shorter metaphases and multiple mitotic errors, resulting in increased long-term sensitivity to SAC inhibition. Our findings propose high CDC20 expression as a favorable biomarker for SAC inhibition therapy and as an aneuploidy-induced therapeutic vulnerability.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.20.572490">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.20.572490" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.20.572490">
        <p class="paperTitle">Inactivation of p53 drives breast cancer brain metastasis by altering fatty acid metabolism</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.20.572490" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.20.572490" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Laue, K.; Pozzi, S.; Cohen-Sharir, Y.; Winkler, T.; Eliezer, Y.; Israeli Dangoor, S.; Leikin-Frenkel, A.; Lange, K.; Zerbib, J.; Ricci, A. A.; Sacconi, A.; Berthelet, J.; Schaffer, A.; Shi, W.; Liao, Y.; Barshack, I.; Medyouf, H.; Merino, D.; Blandino, G.; Bertero, L.; Satchi-Fainaro, R.; Ben-David, U.</p>
        <p class="info">Score: 4.0, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.20.572490' target='https://doi.org/10.1101/2023.12.20.572490'> 10.1101/2023.12.20.572490</a></p>
        <p class="abstract">Brain metastasis (BM) is a dire prognosis across cancer types. It is largely unknown why some tumors metastasize to the brain whereas others do not. We analyzed genomic and transcriptional data from clinical samples of breast cancer BM (BCBM) and found that nearly all of them carried p53-inactivating genetic alterations through mutations, copy-number loss, or both. Importantly, p53 pathway activity was already perturbed in primary tumors giving rise to BCBM, often by loss of the entire 17p chromosome-arm. This association was recapitulated across other carcinomas. Experimentally, p53 knockout was sufficient to drastically increase BCBM formation and growth in vivo, providing a causal link between p53 inactivation and brain tropism. Mechanistically, p53-deficient BC cells exhibited altered lipid metabolism, particularly increased fatty acid (FA) synthesis and uptake, which are characteristic of brain-metastasizing cancer cells. FA metabolism was further enhanced by astrocytes in a p53-dependent manner, as astrocyte-conditioned medium increased FASN, SCD1, and CD36 expression and activity, and enhanced the survival, proliferation and migration of p53-deficient cancer cells. Consequently, these cells were more sensitive than p53-competent cells to FA synthesis inhibitors, in isogenic cell cultures, in BCBM-derived spheroids, and across dozens of BC cell lines. Lastly, a significant association was observed between p53 inactivation, astrocyte infiltration, and SCD1 expression in clinical human BCBM samples. In summary, our study identifies p53 inactivation as a driver of BCBM and potentially of BM in general; suggests a p53-dependent effect of astrocytes on BC cell behavior; and reveals FA metabolism as an underlying, therapeutically-targetable molecular mechanism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.19.572355">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.19.572355" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.19.572355">
        <p class="paperTitle">A rise in double-strand breaks sensitizes tumours to oxidative metabolism inhibitors</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.19.572355" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.19.572355" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Medina-Jover, F.; Figueras, A.; Lahiguera, A.; Guillen, P.; Espin, R.; Pardo, M. A.; Pujana, M. A.; Berra, E.; Villanueva, A.; Bernat, A.; Romeo, M.; Perales, J. C.; Vinals, F.</p>
        <p class="info">Score: 2.9, Published: 2023-12-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.19.572355' target='https://doi.org/10.1101/2023.12.19.572355'> 10.1101/2023.12.19.572355</a></p>
        <p class="abstract">ABSTRACTDouble strand brakes (DSB) accumulate in cellular DNA as a result of deficiencies in homologous recombination repair systems, such as mutations in BRCA genes, or upon antitumoral treatments. In the present study we show that the accumulation of DSB, regardless of its origins, leads to a shift towards oxidative metabolism. We have identified that DSB-induced reactive oxygen species (ROS) promote the activation of NRF2 which downregulates the glycolytic transcription factor HIF-1. HIF-1 inhibition is a key step in this metabolic shift, because leads to the reduction of PDHK1 levels and the consequential activation of pyruvate dehydrogenase, a mitochondrial gatekeeper of cellular metabolism, promoting this metabolic shift. Remarkably, after the induction of DSBs, the tumour is more sensitive to the inhibition of oxidative metabolism since both treatments synergize in vivo, resulting in reduced tumour growth. Therefore, we demonstrate a significant feedback between DSBs induction and cancer cell metabolism that ultimately limits the cells potential for metabolic plasticity, hence sensitizing it to the action of counteracting drugs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.21.572541">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.21.572541" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.21.572541">
        <p class="paperTitle">PARP inhibitors enhance reovirus-mediated cell killing through the death-inducing signaling complex (DISC) with an associated NF-kappa B-regulated immune response.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.21.572541" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.21.572541" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kyula-Currie, J.; Roulstone, V.; Wright, J.; Butera, F.; Legrand, A.; Elliott, R.; McLaughlin, M.; Bozhanova, G.; Krastev, D. B.; Pettitt, S.; Tenev, T.; Dillon, M. T. B.; Foo, S.; Patin, E. C.; Jennings, V.; Chan, C.; Appleton, E.; Pedersen, M.; Rullan, A.; Choudhary, J.; Bakal, C.; Meier, P.; Lord, C. J.; Melcher, A.; Harrington, K. J.</p>
        <p class="info">Score: 2.8, Published: 2023-12-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.21.572541' target='https://doi.org/10.1101/2023.12.21.572541'> 10.1101/2023.12.21.572541</a></p>
        <p class="abstract">Oncolytic Reovirus type 3 Dearing (RT3D), is a naturally occurring double-stranded (ds) RNA virus that is under development as an oncolytic immunotherapy We used an unbiased high-throughput cytotoxicity screen of different targeted therapeutic agents with the aim of identifying potential drug-viral sensitizers to enhance RT3D tumour killing. Talazoparib, a clinical poly(ADP)-ribose polymerase 1 (PARP-1) inhibitor, was identified as a top hit and found to cause profound sensitisation to RT3D. This effect was not seen with other classes of oncolytic virus and was not mediated by enhanced viral replication or PARP inhibitor-related effects on the DNA damage response.

RT3D interacts with retinoic acid-induced gene-1 (RIG-I) and activates PARP-1, with consequent PARylation of components of the extrinsic apoptosis pathway. Pharmacological and genetic inhibition of PARP-1 abrogates this PARylation and increases levels of extrinsic apoptosis, NF-kB signalling and pro-inflammatory cell death. Direct interaction between PARP-1 and RIG-I following RT3D/talazoparib treatment is a key factor in activating downstream signaling pathways that lead to IFN-{beta} and TNF-/TRAIL production which, in turn, amplify the therapeutic effect through positive feedback. Critically, it was possible to phenocopy the effect of RT3D through the use of non-viral ds-RNA therapy and RIG-I agonism. In in vivo studies, we demonstrated profound combinatorial efficacy of RT3D and talazoparib in human A375 melanoma in immunodeficient mice. More impressively, in immunocompetent mouse models of 4434 murine melanoma, we achieved 100% tumour control and protection from subsequent tumour rechallenge with the combination regimen. Correlative immunophenotyping confirmed significant innate and adaptive immune activation with the combination of RT3D and PARP inhibition. Taken together, these data provide a clear line of sight to clinical translation of combined regimens of PARP inhibition or ds-RNA agonism, with either viral or non-viral agents, in tumour types beyond the relatively narrow confines of current licensed indications for PARP inhibition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.573275">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.573275" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.573275">
        <p class="paperTitle">Spatial dissection of cellular heterogeneity and functional crosstalk in the tumor microenvironments of gastric cancers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.573275" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.573275" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, T.; Lee, S. H.; Lee, D.; Choi, J.; Oh, H. J.; Ham, I.-H.; Ryu, D.; Lee, S.-Y.; Han, D.-J.; Kim, S.; Lee, S.; Hur, H.</p>
        <p class="info">Score: 2.5, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.573275' target='https://doi.org/10.1101/2023.12.24.573275'> 10.1101/2023.12.24.573275</a></p>
        <p class="abstract">Spatial transcriptomics offers unprecedented challenges in characterizing tumor ecosystems, encompassing tumor cells and their surrounding tumor microenvironments (TME) along with cellular interactions therein. Here, we conducted Visium-based spatial transcriptomics on nine primary gastric cancers (GC) to translate the spatial organization of GC ecosystems into the functional landscape of cellular crosstalk across malignant, stromal and immune cells. We identified three distinct GC subtypes of 3 immunogenic, 3 epithelial and 3 stromal GCs with elevated infiltration of immune, malignant cells and fibroblasts reflecting the heterogeneity of the cell types and their interactions in GC TME. GC spatial architecture was further delineated into 6 regional compartments with varying degree of TME infiltration, among which the fibroblast-enriched TME was associated with the upregulation of epithelial-to-mesenchymal transformation (EMT) and immune response in malignant and immune cells, respectively. Cell type-specific transcriptional dynamics representing cellular crosstalk between TME cells were further identified, e.g., the infiltration of malignant and endothelial cells promote the cellular proliferations of TME cells whereas the fibroblasts and immune cells are associated with pro-cancer and anti-cancer immunity, respectively. Ligand-receptor analysis further revealed that CCL2-expressing fibroblasts promote the pro-cancer signaling of immune cells including JAK-STAT3 signaling and inflammatory response in tumor-infiltration macrophages. The CCL2&#43;fibroblasts and macrophages with activated JAK-STAT3 signals are co-localized in spatial GC architecture and their co-abundance was associated with unfavorable clinical outcomes. Taken together, GC spatial transcriptomes can be regionally and functionally delineated into functional cellular crosstalk involving multiple cell types with potential clinical relevance, particularly the interaction between CCL2&#43; fibroblasts and JAK-STAT3&#43;macrophages contributing to cancer-favoring immune contexture in GC TME.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        Â· Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
