<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Epigenetic heritability of cell plasticity drives cancer drug resistance through one-to-many genotype to phenotype mapping
Authors: Fernandez-Mateos, J.; Milite, S.; Oliveira, E.; Vlachogiannis, G.; Chen, B.; Yara, E.; Cresswell, G. D.; James, C.; Patruno, L.; Ascolani, G.; Acar, A.; Heide, T.; Spiteri, I.; Graudenzi, A.; Caravagna, G.; Graham, T.; Magnani, L.; Valeri, N.; Sottoriva, A.
Score: 40.2, Published: 2023-11-16 DOI: 10.1101/2023.11.15.567140
Drug resistance is a largely unsolved problem in oncology.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Epigenetic heritability of cell plasticity drives cancer drug resistance through one-to-many genotype to phenotype mapping
Authors: Fernandez-Mateos, J.; Milite, S.; Oliveira, E.; Vlachogiannis, G.; Chen, B.; Yara, E.; Cresswell, G. D.; James, C.; Patruno, L.; Ascolani, G.; Acar, A.; Heide, T.; Spiteri, I.; Graudenzi, A.; Caravagna, G.; Graham, T.; Magnani, L.; Valeri, N.; Sottoriva, A.
Score: 40.2, Published: 2023-11-16 DOI: 10.1101/2023.11.15.567140
Drug resistance is a largely unsolved problem in oncology." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-19T10:38:34+00:00" />
<meta property="article:modified_time" content="2023-11-19T10:38:34+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Epigenetic heritability of cell plasticity drives cancer drug resistance through one-to-many genotype to phenotype mapping
Authors: Fernandez-Mateos, J.; Milite, S.; Oliveira, E.; Vlachogiannis, G.; Chen, B.; Yara, E.; Cresswell, G. D.; James, C.; Patruno, L.; Ascolani, G.; Acar, A.; Heide, T.; Spiteri, I.; Graudenzi, A.; Caravagna, G.; Graham, T.; Magnani, L.; Valeri, N.; Sottoriva, A.
Score: 40.2, Published: 2023-11-16 DOI: 10.1101/2023.11.15.567140
Drug resistance is a largely unsolved problem in oncology."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Epigenetic heritability of cell plasticity drives cancer drug resistance through one-to-many genotype to phenotype mapping\nAuthors: Fernandez-Mateos, J.; Milite, S.; Oliveira, E.; Vlachogiannis, G.; Chen, B.; Yara, E.; Cresswell, G. D.; James, C.; Patruno, L.; Ascolani, G.; Acar, A.; Heide, T.; Spiteri, I.; Graudenzi, A.; Caravagna, G.; Graham, T.; Magnani, L.; Valeri, N.; Sottoriva, A.\nScore: 40.2, Published: 2023-11-16 DOI: 10.1101/2023.11.15.567140\nDrug resistance is a largely unsolved problem in oncology.",
  "keywords": [
    
  ],
  "articleBody": " Epigenetic heritability of cell plasticity drives cancer drug resistance through one-to-many genotype to phenotype mapping\nAuthors: Fernandez-Mateos, J.; Milite, S.; Oliveira, E.; Vlachogiannis, G.; Chen, B.; Yara, E.; Cresswell, G. D.; James, C.; Patruno, L.; Ascolani, G.; Acar, A.; Heide, T.; Spiteri, I.; Graudenzi, A.; Caravagna, G.; Graham, T.; Magnani, L.; Valeri, N.; Sottoriva, A.\nScore: 40.2, Published: 2023-11-16 DOI: 10.1101/2023.11.15.567140\nDrug resistance is a largely unsolved problem in oncology. Despite the explanatory power of the genetic model of cancer initiation, most treatment resistance is unexplained by genetics alone. Even when known resistance mutations are present, they are often found in a small proportion of the cells in the tumour. So where is the cellular memory that leads to treatment failure? New evidence suggests resistance is multi-factorial, resulting from the contribution of heritable genetic and epigenetic changes, but also non-heritable phenotypic plasticity. However, cell plasticity has proven hard to study as it dynamically changes over time and needs to be distinguished from clonal evolution where cell phenotypes change because of Darwinian selective bottlenecks. Here we dissected the contribution of different evolutionary processes to drug resistance by perturbing patient-derived organoids with multiple drugs in sequence. We combined dense longitudinal tracking, single cell multi-omics, evolutionary modelling, and machine learning archetypal analysis. We found that different drugs select for distinct subclones, an essential requirement for the use of evolutionary therapy with sequential drug treatment. The data supports a model in which the cellular memory is encoded as a heritable configuration of the epigenome, which however produces multiple transcriptional programmes. Those emerge in different proportions depending on the environment, giving rise to cellular plasticity. Epigenetically encoded programmes include reactivation of developmental genes and cell regeneration. A one-to-many (epi)genotype-phenotype map explains how clonal expansions and non-heritable phenotypic plasticity manifest together, including drug tolerant states. This ensures the robustness of drug resistance subclones that can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding their selective advantage.\n2D and 3D multiplexed subcellular profiling of nuclear instability in human cancer\nAuthors: Coy, S.; Cheng, B.; Lee, J. S.; Rashid, R.; Browning, L.; Xu, Y.; Chakrabarty, S. S.; Yapp, C.; Chan, S.; Tefft, J. B.; Spektor, A.; Ligon, K. L.; Baker, G. J.; Pellman, D.; Sorger, P. K.; Santagata, S.\nScore: 15.3, Published: 2023-11-11 DOI: 10.1101/2023.11.07.566063\nNuclear atypia, including altered nuclear size, contour, and chromatin organization, is ubiquitous in cancer cells. Atypical primary nuclei and micronuclei can rupture during interphase; however, the frequency, causes, and consequences of nuclear rupture are unknown in most cancers. We demonstrate that nuclear envelope rupture is surprisingly common in many human cancers, particularly glioblastoma. Using highly-multiplexed 2D and super-resolution 3D-imaging of glioblastoma tissues and patient-derived xenografts and cells, we link primary nuclear rupture with reduced lamin A/C and micronuclear rupture with reduced lamin B1. Moreover, ruptured glioblastoma cells activate cGAS-STING-signaling involved in innate immunity. We observe that local patterning of cell states influences tumor spatial organization and is linked to both lamin expression and rupture frequency, with neural-progenitor-cell-like states exhibiting the lowest lamin A/C levels and greatest susceptibility to primary nuclear rupture. Our study reveals that nuclear instability is a core feature of cancer, and links nuclear integrity, cell state, and immune signaling.\nCRISPR-Cas13d screens identify KILR a breast cancer risk-associated lncRNA that regulates DNA replication and repair.\nAuthors: Wang, L.; Bitar, M.; Lu, X.; Jacquelin, S.; Nair, S.; Sivakumaran, H.; Hillman, K. M.; Kaufmann, S.; Ziegman, R.; Casciello, F.; Gowda, H.; Rosenbluh, J.; Edwards, S. L.; French, J. D.\nScore: 9.5, Published: 2023-11-17 DOI: 10.1101/2023.11.16.567471\nLong noncoding RNAs (lncRNAs) have surpassed the number of protein-coding genes, yet the majority have no known function. We previously discovered \u003e800 lncRNAs at regions identified by breast cancer genome-wide association studies (GWAS). Here, we performed a pooled CRISPR-Cas13d RNA knockdown screen to identify which of these lncRNAs altered cell proliferation. We found that KILR, a lncRNA that functions as a tumor suppressor, safeguards breast cells against uncontrolled proliferation. The half-life of KILR is significantly reduced by the risk haplotype, revealing an alternative mechanism by which variants alter cancer risk. We showed that KILR sequesters RPA1, a subunit of the RPA complex, required for DNA replication and repair. Reduced KILR expression promotes cell proliferation by increasing the available pool of RPA1 and the speed of DNA replication. Our findings confirm lncRNAs as mediators of breast cancer risk, emphasize the need to annotate noncoding transcripts in relevant cell types when investigating GWAS variants and provide a scalable platform for mapping phenotypes associated with lncRNAs.\neIF4A1 is essential for reprogramming the translational landscape of Wnt-driven colorectal cancers\nAuthors: Waldron, J. A.; Kanellos, G.; Smith, R. C. L.; Knight, J. R. P.; Munro, J.; Alexandrou, C.; Vlahov, N.; Pardo-Fernandez, L.; Moore, M.; Gillen, S. L.; Strathdee, D.; Stevenson, D.; Warrander, F. C.; Gilroy, K.; Nixon, C.; Cadden, B.; Powley, I.; Officer-Jones, L.; Ballantyne, F.; Hay, J.; Pennel, K.; Edwards, J.; Campbell, A. D.; Ridgway, R. A.; Coffelt, S. B.; Norman, J.; Le Quesne, J.; Bushell, M.; Sansom, O. J.\nScore: 7.7, Published: 2023-11-13 DOI: 10.1101/2023.11.10.566546\nDysregulated translation is a hallmark of cancer. Targeting the translational machinery represents a therapeutic avenue which is being actively explored. eIF4A inhibitors target both eIF4A1, which promotes translation as part of the eIF4F complex, and eIF4A2, which can repress translation via the CCR4-NOT complex. While high eIF4A1 expression is associated with poor patient outcome, the role of eIF4A2 in cancer remains unclear. Furthermore, the on-target toxicity of targeting specific eIF4A paralogues in healthy tissue is under-explored. We show that while loss of either paralogue is tolerated in the wild-type intestine, eIF4A1 is specifically required to support the translational demands of oncogenic Wnt signalling. Intestinal tumourigenesis is suppressed in colorectal cancer models following loss of eIF4A1 but accelerated following loss of eIF4A2, while eIF4A inhibition with eFT226 mimics loss of eIF4A1 in these models.\nMutational scanning reveals oncogenic CTNNB1 mutations have diverse effects on signalling and clinical traits\nAuthors: Krishna, A.; Meynert, A.; Kelder, M.; Ewing, A.; Sheraz, S.; Ferrer-Vacquer, A.; Grimes, G.; Becher, H.; Silk, R.; Semple, C. A.; Kendall, T.; Hadjantonakis, A.-K.; Bird, T.; Marsh, J. A.; Hohenstein, P.; Wood, A. J.; Ozdemir, D. D.\nScore: 7.0, Published: 2023-11-13 DOI: 10.1101/2023.11.09.566307\nCTNNB1, the gene encoding {beta}-catenin, is a frequent target for oncogenic mutations activating the canonical Wnt signalling pathway, typically via missense mutations within a degron hotspot motif in exon 3. Here, we combine saturation genome editing with a fluorescent reporter assay to quantify signalling phenotypes for all 342 missense mutations in the mutation hotspot, including 74 recurrent mutations reported in over 6000 tumours. Our data define the genetic requirements for {beta}-catenin degron function and reveal diverse levels of signal activation among known driver mutations. Tumorigenesis in different human tissues involves selection for CTNNB1 mutations spanning distinct ranges of effect size. In hepatocellular carcinoma, mutations that activate {beta}-catenin relatively weakly are associated with worse prognosis compared to stronger activating mutations, despite greater immune cell infiltration in the tumour microenvironment. Our work therefore provides a resource to understand mutational diversity within a pan-cancer mutation hotspot, with potential implications for targeted therapy.\nModeling the effect of spatial structure on solid tumor evolution and ctDNA composition\nAuthors: Rachman, T.; Bartlett, D.; Laframboise, W.; Wagner, P.; Schwartz, R.; Carja, O.\nScore: 6.0, Published: 2023-11-11 DOI: 10.1101/2023.11.10.566658\nCirculating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform on cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how elevated cell death on the edge of a tumor can simultaneously impact driver mutation accumulation and the representation of tumor clones and mutation detectability in ctDNA. We describe conditions in which invasive clones end up over-represented in ctDNA, clonal diversity can appear elevated in the blood, and spatial bias in shedding can inflate subclonal variant allele frequencies (VAFs). Additionally, we find that tumors that are mostly quiescent can display similar biases, but are far less detectable, and the extent of perceptible spatial bias strongly depends on sequence detection limits. Overall, we show that spatially structured shedding might cause liquid biopsies to provide highly biased profiles of tumor state. While this may enable more sensitive detection of expanding clones, it could also increase the risk of targeting a subclonal variant for treatment. Our results indicate that the effects and clinical consequences of spatially variable cell death on ctDNA composition present an important area for future work.\nTumor cell-derived spermidine promotes a pro-tumorigenic immune microenvironment in glioblastoma via CD8+ T cell inhibition\nAuthors: Kay, K.; Lee, J.; Hong, E. S.; Beilis, J.; Dayal, S.; Wesley, E. R.; Mitchell, S.; Wang, S. Z.; Silver, D. J.; Volovetz, J.; Johnson, S.; McGraw, M.; Grabowski, M. M.; Lu, T.; Freytag, L.; Narayana, V.; Freytag, S.; Best, S. A.; Whittle, J. R.; Wang, Z.; Reizes, O.; Yu, J. S.; Hazen, S. L.; Brown, J. M.; Bayik, D.; Lathia, J.\nScore: 5.6, Published: 2023-11-16 DOI: 10.1101/2023.11.14.567048\nThe glioblastoma microenvironment is enriched in immunosuppressive factors that potently interfere with the function of cytotoxic T lymphocytes. Cancer cells can directly impact the immune system, but the mechanisms driving these interactions are not completely clear. Here we demonstrate that the polyamine metabolite spermidine is elevated in the glioblastoma tumor microenvironment. Exogenous administration of spermidine drives tumor aggressiveness in an immune-dependent manner in pre-clinical mouse models via reduction of CD8+ T cell frequency and phenotype. Knockdown of ornithine decarboxylase, the rate-limiting enzyme in spermidine synthesis, did not impact cancer cell growth in vitro but did result in extended survival. Furthermore, glioblastoma patients with a more favorable outcome had a significant reduction in spermidine compared to patients with a poor prognosis. Our results demonstrate that spermidine functions as a cancer cell-derived metabolite that drives tumor progression by reducing CD8+ T cell number and function.\nMutation of NOTCH1 is selected within normal esophageal tissues, yet leads to selective epistasis suppressive of further evolution into cancer\nAuthors: Glasmacher, K. A.; Cannataro, V. L.; Mandell, J. D.; Jackson, M.; Fisk, J. N.; Townsend, J. P.\nScore: 10.7, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565535\nSequencing of tissues from histologically normal esophagus, among other organs, has revealed that normal tissues harbor somatic variants that are also found in cancers arising from the same tissue types. Our understanding of how somatic mutations commonly found in normal tissue can contribute to tumorigenesis is limited: common somatic mutations may or may not confer phenotypes compatible with oncogenesis. However, the strength of selection for somatic variants that appear in both normal and cancer tissues can be quantified in each context using evolutionary modeling approaches. We studied the evolutionary trajectory from normal esophageal tissue to esophageal squamous-cell carcinoma (ESCC) by analysis of 2171 sequenced samples from previous studies on normal esophageal epithelium and ESCC to reveal the stepwise contributions of somatic mutations to increased cellular division and survival. We also analyzed pairwise selective epistasis between somatically mutated genes that may lead to stepwise substitution patterns. We found that NOTCH1 substitutions are highly selected along the trajectory from embryogenesis to adult normal esophageal tissue, explaining their high prevalence in the tissue. In contrast, there is little to no positive selection for NOTCH1 mutations along the trajectory from adult normal tissue to ESCC, suggesting that NOTCH1 substitutions do not drive tumorigenesis. Furthermore, mutations in NOTCH1 exhibit antagonistic epistasis with well-known cancer drivers including TP53, reducing selection for progressive mutations in tumorigenesis. This antagonistic epistasis likely corresponds with a low likelihood of tumor progression in the presence of NOTCH1 mutations in the esophagus.\nUnsaturated fatty acid alters the immune response in non-small cell lung adenocarcinoma through regulation of HMGB1 trafficking\nAuthors: Cole-Skinner, B.; Andre, N. M.; Blankenheim, Z.; Root, K.; Simmons, G. E.\nScore: 4.5, Published: 2023-11-11 DOI: 10.1101/2023.11.08.566231\nCancer cell evasion of the immune response is critical to cancer development and metastases. The ability of clinicians to kickstart the immune system to target these rogue cells is an ever-growing area of research and medicine. In this study, we delved into the relationship between lipid metabolism, High Mobility Group Box 1 protein (HMGB1), and immune regulation within non-small cell lung adenocarcinoma (NSCLC), shedding light on novel therapeutic avenues and potential personalized approaches for patients. We found that the expression of stearoyl CoA desaturase 1 (SCD1) was decreased in NSCLC tumors compared to normal tissues. This emphasized the critical role of lipid metabolism in tumor progression. Interestingly, monounsaturated fatty acid (MUFA) availability impacted the expression of programmed death receptor ligand -1 (PD-L1), a pivotal immune checkpoint target in lung cancer cells and immune cells, suggesting a novel approach to modulating the immune response. This study uncovered a complex interplay between HMGB1, SCD1, and PD-L1, influencing the immunological sensitivity of tumors. Our work underscores the importance of understanding the intricate relationships between lipid metabolism and immune modulation to develop more effective NSCLC treatments and personalized therapies. As we continue to explore these connections, we hope to contribute to the ever-evolving field of cancer research, improving patient outcomes and advancing precision medicine in NSCLC.\nGait speed is a biomarker of cancer-associated cachexia decline and recovery\nAuthors: Roy, I.; Binder-Markey, B.; Sychowski, D.; Willbanks, A.; Phipps, T.; McAllister, D.; Bhakta, A.; Marquez, E.; D'Andrea, D.; Franz, C.; Pichika, R.; Dwinell, M.; Jayabalan, P.; Lieber, R. L.\nScore: 4.1, Published: 2023-11-15 DOI: 10.1101/2023.11.13.566852\nBackgroundProgressive functional decline is a key element of cancer-associated cachexia. No therapies have successfully translated to the clinic due to an inability to measure and improve physical function in cachectic patients. Major barriers to translating pre-clinical therapies to the clinic include lack of cancer models that accurately mimic functional decline and use of non-specific outcome measures of function, like grip strength. New approaches are needed to investigate cachexia-related function at both the basic and clinical science levels. MethodsSurvival extension studies were performed by testing multiple cell lines, dilutions, and vehicle-types in orthotopic implantation of K-rasLSL.G12D/+; Trp53R172H/+; Pdx-1-Cre (KPC) derived cells. 128 animals in this new model were then assessed for muscle wasting, inflammation, and functional decline using a battery of biochemical, physiologic, and behavioral techniques. In parallel, we analyzed a 156-subject cohort of cancer patients with a range of cachexia severity, and who required rehabilitation, to determine the relationship between gait speed via six-minute walk test (6MWT), grip strength (hGS), and functional independence measures (FIM). Cachectic patients were identified using the Weight Loss Grading Scale (WLGS), Fearon consensus criteria, and the Prognostic Nutritional Index (PNI). ResultsUsing a 100-cell dose of DT10022 KPC cells, we extended the survival of the KPC orthotopic model to 8-9 weeks post-implantation compared to higher doses used (p\u003c0.001). In this Low-dose Orthotopic (LO) model, both progressive skeletal and cardiac muscle wasting were detected in parallel to systemic inflammation; skeletal muscle atrophy at the fiber level was detected as early as 3 weeks post-implantation compared to controls (p\u003c0.001). Gait speed in LO animals declined as early 2 week post-implantation whereas grip strength change was a late event and related to end of life. Principle component analysis (PCA) revealed distinct cachectic and non-cachectic animal populations, which we leveraged to show that gait speed decline was specific to cachexia (p\u003c0.01) while grip strength decline was not (p=0.19). These data paralleled our observations in cancer patients with cachexia who required rehabilitation. In cachectic patients (identified by WLGS, Fearon criteria, or PNI, change in 6MWT correlated with motor FIM score changes while hGS did not (r2=0.18, p\u003c0.001). This relationship between 6MWT and FIM in cachectic patients was further confirmed through multivariate regression (r2=0.30, p\u003c0.001) controlling for age and cancer burden. ConclusionOutcome measures linked to gait are better associated with cachexia related function and preferred for future pre-clinical and clinical cachexia studies.\n",
  "wordCount" : "2637",
  "inLanguage": "en",
  "datePublished": "2023-11-19T10:38:34Z",
  "dateModified": "2023-11-19T10:38:34Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 19, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.15.567140">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.15.567140" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.15.567140">
        <p class="paperTitle">Epigenetic heritability of cell plasticity drives cancer drug resistance through one-to-many genotype to phenotype mapping</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.15.567140" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.15.567140" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fernandez-Mateos, J.; Milite, S.; Oliveira, E.; Vlachogiannis, G.; Chen, B.; Yara, E.; Cresswell, G. D.; James, C.; Patruno, L.; Ascolani, G.; Acar, A.; Heide, T.; Spiteri, I.; Graudenzi, A.; Caravagna, G.; Graham, T.; Magnani, L.; Valeri, N.; Sottoriva, A.</p>
        <p class="info">Score: 40.2, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.15.567140' target='https://doi.org/10.1101/2023.11.15.567140'> 10.1101/2023.11.15.567140</a></p>
        <p class="abstract">Drug resistance is a largely unsolved problem in oncology. Despite the explanatory power of the genetic model of cancer initiation, most treatment resistance is unexplained by genetics alone. Even when known resistance mutations are present, they are often found in a small proportion of the cells in the tumour. So where is the cellular memory that leads to treatment failure? New evidence suggests resistance is multi-factorial, resulting from the contribution of heritable genetic and epigenetic changes, but also non-heritable phenotypic plasticity. However, cell plasticity has proven hard to study as it dynamically changes over time and needs to be distinguished from clonal evolution where cell phenotypes change because of Darwinian selective bottlenecks. Here we dissected the contribution of different evolutionary processes to drug resistance by perturbing patient-derived organoids with multiple drugs in sequence. We combined dense longitudinal tracking, single cell multi-omics, evolutionary modelling, and machine learning archetypal analysis. We found that different drugs select for distinct subclones, an essential requirement for the use of evolutionary therapy with sequential drug treatment. The data supports a model in which the cellular memory is encoded as a heritable configuration of the epigenome, which however produces multiple transcriptional programmes. Those emerge in different proportions depending on the environment, giving rise to cellular plasticity. Epigenetically encoded programmes include reactivation of developmental genes and cell regeneration. A one-to-many (epi)genotype-phenotype map explains how clonal expansions and non-heritable phenotypic plasticity manifest together, including drug tolerant states. This ensures the robustness of drug resistance subclones that can exhibit distinct phenotypes in changing environments while still preserving the cellular memory encoding their selective advantage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566063">
        <p class="paperTitle">2D and 3D multiplexed subcellular profiling of nuclear instability in human cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coy, S.; Cheng, B.; Lee, J. S.; Rashid, R.; Browning, L.; Xu, Y.; Chakrabarty, S. S.; Yapp, C.; Chan, S.; Tefft, J. B.; Spektor, A.; Ligon, K. L.; Baker, G. J.; Pellman, D.; Sorger, P. K.; Santagata, S.</p>
        <p class="info">Score: 15.3, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566063' target='https://doi.org/10.1101/2023.11.07.566063'> 10.1101/2023.11.07.566063</a></p>
        <p class="abstract">Nuclear atypia, including altered nuclear size, contour, and chromatin organization, is ubiquitous in cancer cells. Atypical primary nuclei and micronuclei can rupture during interphase; however, the frequency, causes, and consequences of nuclear rupture are unknown in most cancers. We demonstrate that nuclear envelope rupture is surprisingly common in many human cancers, particularly glioblastoma. Using highly-multiplexed 2D and super-resolution 3D-imaging of glioblastoma tissues and patient-derived xenografts and cells, we link primary nuclear rupture with reduced lamin A/C and micronuclear rupture with reduced lamin B1. Moreover, ruptured glioblastoma cells activate cGAS-STING-signaling involved in innate immunity. We observe that local patterning of cell states influences tumor spatial organization and is linked to both lamin expression and rupture frequency, with neural-progenitor-cell-like states exhibiting the lowest lamin A/C levels and greatest susceptibility to primary nuclear rupture. Our study reveals that nuclear instability is a core feature of cancer, and links nuclear integrity, cell state, and immune signaling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.16.567471">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.16.567471" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.16.567471">
        <p class="paperTitle">CRISPR-Cas13d screens identify KILR a breast cancer risk-associated lncRNA that regulates DNA replication and repair.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.16.567471" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.16.567471" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, L.; Bitar, M.; Lu, X.; Jacquelin, S.; Nair, S.; Sivakumaran, H.; Hillman, K. M.; Kaufmann, S.; Ziegman, R.; Casciello, F.; Gowda, H.; Rosenbluh, J.; Edwards, S. L.; French, J. D.</p>
        <p class="info">Score: 9.5, Published: 2023-11-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.16.567471' target='https://doi.org/10.1101/2023.11.16.567471'> 10.1101/2023.11.16.567471</a></p>
        <p class="abstract">Long noncoding RNAs (lncRNAs) have surpassed the number of protein-coding genes, yet the majority have no known function. We previously discovered &gt;800 lncRNAs at regions identified by breast cancer genome-wide association studies (GWAS). Here, we performed a pooled CRISPR-Cas13d RNA knockdown screen to identify which of these lncRNAs altered cell proliferation. We found that KILR, a lncRNA that functions as a tumor suppressor, safeguards breast cells against uncontrolled proliferation. The half-life of KILR is significantly reduced by the risk haplotype, revealing an alternative mechanism by which variants alter cancer risk. We showed that KILR sequesters RPA1, a subunit of the RPA complex, required for DNA replication and repair. Reduced KILR expression promotes cell proliferation by increasing the available pool of RPA1 and the speed of DNA replication. Our findings confirm lncRNAs as mediators of breast cancer risk, emphasize the need to annotate noncoding transcripts in relevant cell types when investigating GWAS variants and provide a scalable platform for mapping phenotypes associated with lncRNAs.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566546">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566546" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566546">
        <p class="paperTitle">eIF4A1 is essential for reprogramming the translational landscape of Wnt-driven colorectal cancers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566546" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566546" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waldron, J. A.; Kanellos, G.; Smith, R. C. L.; Knight, J. R. P.; Munro, J.; Alexandrou, C.; Vlahov, N.; Pardo-Fernandez, L.; Moore, M.; Gillen, S. L.; Strathdee, D.; Stevenson, D.; Warrander, F. C.; Gilroy, K.; Nixon, C.; Cadden, B.; Powley, I.; Officer-Jones, L.; Ballantyne, F.; Hay, J.; Pennel, K.; Edwards, J.; Campbell, A. D.; Ridgway, R. A.; Coffelt, S. B.; Norman, J.; Le Quesne, J.; Bushell, M.; Sansom, O. J.</p>
        <p class="info">Score: 7.7, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566546' target='https://doi.org/10.1101/2023.11.10.566546'> 10.1101/2023.11.10.566546</a></p>
        <p class="abstract">Dysregulated translation is a hallmark of cancer. Targeting the translational machinery represents a therapeutic avenue which is being actively explored. eIF4A inhibitors target both eIF4A1, which promotes translation as part of the eIF4F complex, and eIF4A2, which can repress translation via the CCR4-NOT complex. While high eIF4A1 expression is associated with poor patient outcome, the role of eIF4A2 in cancer remains unclear. Furthermore, the on-target toxicity of targeting specific eIF4A paralogues in healthy tissue is under-explored. We show that while loss of either paralogue is tolerated in the wild-type intestine, eIF4A1 is specifically required to support the translational demands of oncogenic Wnt signalling. Intestinal tumourigenesis is suppressed in colorectal cancer models following loss of eIF4A1 but accelerated following loss of eIF4A2, while eIF4A inhibition with eFT226 mimics loss of eIF4A1 in these models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566307">
        <p class="paperTitle">Mutational scanning reveals oncogenic CTNNB1 mutations have diverse effects on signalling and clinical traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishna, A.; Meynert, A.; Kelder, M.; Ewing, A.; Sheraz, S.; Ferrer-Vacquer, A.; Grimes, G.; Becher, H.; Silk, R.; Semple, C. A.; Kendall, T.; Hadjantonakis, A.-K.; Bird, T.; Marsh, J. A.; Hohenstein, P.; Wood, A. J.; Ozdemir, D. D.</p>
        <p class="info">Score: 7.0, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566307' target='https://doi.org/10.1101/2023.11.09.566307'> 10.1101/2023.11.09.566307</a></p>
        <p class="abstract">CTNNB1, the gene encoding {beta}-catenin, is a frequent target for oncogenic mutations activating the canonical Wnt signalling pathway, typically via missense mutations within a degron hotspot motif in exon 3. Here, we combine saturation genome editing with a fluorescent reporter assay to quantify signalling phenotypes for all 342 missense mutations in the mutation hotspot, including 74 recurrent mutations reported in over 6000 tumours. Our data define the genetic requirements for {beta}-catenin degron function and reveal diverse levels of signal activation among known driver mutations. Tumorigenesis in different human tissues involves selection for CTNNB1 mutations spanning distinct ranges of effect size. In hepatocellular carcinoma, mutations that activate {beta}-catenin relatively weakly are associated with worse prognosis compared to stronger activating mutations, despite greater immune cell infiltration in the tumour microenvironment. Our work therefore provides a resource to understand mutational diversity within a pan-cancer mutation hotspot, with potential implications for targeted therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566658">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566658" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566658">
        <p class="paperTitle">Modeling the effect of spatial structure on solid tumor evolution and ctDNA composition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566658" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566658" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rachman, T.; Bartlett, D.; Laframboise, W.; Wagner, P.; Schwartz, R.; Carja, O.</p>
        <p class="info">Score: 6.0, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566658' target='https://doi.org/10.1101/2023.11.10.566658'> 10.1101/2023.11.10.566658</a></p>
        <p class="abstract">Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform on cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how elevated cell death on the edge of a tumor can simultaneously impact driver mutation accumulation and the representation of tumor clones and mutation detectability in ctDNA. We describe conditions in which invasive clones end up over-represented in ctDNA, clonal diversity can appear elevated in the blood, and spatial bias in shedding can inflate subclonal variant allele frequencies (VAFs). Additionally, we find that tumors that are mostly quiescent can display similar biases, but are far less detectable, and the extent of perceptible spatial bias strongly depends on sequence detection limits. Overall, we show that spatially structured shedding might cause liquid biopsies to provide highly biased profiles of tumor state. While this may enable more sensitive detection of expanding clones, it could also increase the risk of targeting a subclonal variant for treatment. Our results indicate that the effects and clinical consequences of spatially variable cell death on ctDNA composition present an important area for future work.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.14.567048">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.14.567048" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.14.567048">
        <p class="paperTitle">Tumor cell-derived spermidine promotes a pro-tumorigenic immune microenvironment in glioblastoma via CD8&#43; T cell inhibition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.14.567048" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.14.567048" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kay, K.; Lee, J.; Hong, E. S.; Beilis, J.; Dayal, S.; Wesley, E. R.; Mitchell, S.; Wang, S. Z.; Silver, D. J.; Volovetz, J.; Johnson, S.; McGraw, M.; Grabowski, M. M.; Lu, T.; Freytag, L.; Narayana, V.; Freytag, S.; Best, S. A.; Whittle, J. R.; Wang, Z.; Reizes, O.; Yu, J. S.; Hazen, S. L.; Brown, J. M.; Bayik, D.; Lathia, J.</p>
        <p class="info">Score: 5.6, Published: 2023-11-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.14.567048' target='https://doi.org/10.1101/2023.11.14.567048'> 10.1101/2023.11.14.567048</a></p>
        <p class="abstract">The glioblastoma microenvironment is enriched in immunosuppressive factors that potently interfere with the function of cytotoxic T lymphocytes. Cancer cells can directly impact the immune system, but the mechanisms driving these interactions are not completely clear. Here we demonstrate that the polyamine metabolite spermidine is elevated in the glioblastoma tumor microenvironment. Exogenous administration of spermidine drives tumor aggressiveness in an immune-dependent manner in pre-clinical mouse models via reduction of CD8&#43; T cell frequency and phenotype. Knockdown of ornithine decarboxylase, the rate-limiting enzyme in spermidine synthesis, did not impact cancer cell growth in vitro but did result in extended survival. Furthermore, glioblastoma patients with a more favorable outcome had a significant reduction in spermidine compared to patients with a poor prognosis. Our results demonstrate that spermidine functions as a cancer cell-derived metabolite that drives tumor progression by reducing CD8&#43; T cell number and function.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565535">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565535" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565535">
        <p class="paperTitle">Mutation of NOTCH1 is selected within normal esophageal tissues, yet leads to selective epistasis suppressive of further evolution into cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565535" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565535" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Glasmacher, K. A.; Cannataro, V. L.; Mandell, J. D.; Jackson, M.; Fisk, J. N.; Townsend, J. P.</p>
        <p class="info">Score: 10.7, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565535' target='https://doi.org/10.1101/2023.11.03.565535'> 10.1101/2023.11.03.565535</a></p>
        <p class="abstract">Sequencing of tissues from histologically normal esophagus, among other organs, has revealed that normal tissues harbor somatic variants that are also found in cancers arising from the same tissue types. Our understanding of how somatic mutations commonly found in normal tissue can contribute to tumorigenesis is limited: common somatic mutations may or may not confer phenotypes compatible with oncogenesis. However, the strength of selection for somatic variants that appear in both normal and cancer tissues can be quantified in each context using evolutionary modeling approaches. We studied the evolutionary trajectory from normal esophageal tissue to esophageal squamous-cell carcinoma (ESCC) by analysis of 2171 sequenced samples from previous studies on normal esophageal epithelium and ESCC to reveal the stepwise contributions of somatic mutations to increased cellular division and survival. We also analyzed pairwise selective epistasis between somatically mutated genes that may lead to stepwise substitution patterns. We found that NOTCH1 substitutions are highly selected along the trajectory from embryogenesis to adult normal esophageal tissue, explaining their high prevalence in the tissue. In contrast, there is little to no positive selection for NOTCH1 mutations along the trajectory from adult normal tissue to ESCC, suggesting that NOTCH1 substitutions do not drive tumorigenesis. Furthermore, mutations in NOTCH1 exhibit antagonistic epistasis with well-known cancer drivers including TP53, reducing selection for progressive mutations in tumorigenesis. This antagonistic epistasis likely corresponds with a low likelihood of tumor progression in the presence of NOTCH1 mutations in the esophagus.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566231">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566231" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566231">
        <p class="paperTitle">Unsaturated fatty acid alters the immune response in non-small cell lung adenocarcinoma through regulation of HMGB1 trafficking</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566231" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566231" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cole-Skinner, B.; Andre, N. M.; Blankenheim, Z.; Root, K.; Simmons, G. E.</p>
        <p class="info">Score: 4.5, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566231' target='https://doi.org/10.1101/2023.11.08.566231'> 10.1101/2023.11.08.566231</a></p>
        <p class="abstract">Cancer cell evasion of the immune response is critical to cancer development and metastases. The ability of clinicians to kickstart the immune system to target these rogue cells is an ever-growing area of research and medicine. In this study, we delved into the relationship between lipid metabolism, High Mobility Group Box 1 protein (HMGB1), and immune regulation within non-small cell lung adenocarcinoma (NSCLC), shedding light on novel therapeutic avenues and potential personalized approaches for patients. We found that the expression of stearoyl CoA desaturase 1 (SCD1) was decreased in NSCLC tumors compared to normal tissues. This emphasized the critical role of lipid metabolism in tumor progression. Interestingly, monounsaturated fatty acid (MUFA) availability impacted the expression of programmed death receptor ligand -1 (PD-L1), a pivotal immune checkpoint target in lung cancer cells and immune cells, suggesting a novel approach to modulating the immune response. This study uncovered a complex interplay between HMGB1, SCD1, and PD-L1, influencing the immunological sensitivity of tumors. Our work underscores the importance of understanding the intricate relationships between lipid metabolism and immune modulation to develop more effective NSCLC treatments and personalized therapies. As we continue to explore these connections, we hope to contribute to the ever-evolving field of cancer research, improving patient outcomes and advancing precision medicine in NSCLC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.13.566852">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.13.566852" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.13.566852">
        <p class="paperTitle">Gait speed is a biomarker of cancer-associated cachexia decline and recovery</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.13.566852" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.13.566852" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Roy, I.; Binder-Markey, B.; Sychowski, D.; Willbanks, A.; Phipps, T.; McAllister, D.; Bhakta, A.; Marquez, E.; D&#39;Andrea, D.; Franz, C.; Pichika, R.; Dwinell, M.; Jayabalan, P.; Lieber, R. L.</p>
        <p class="info">Score: 4.1, Published: 2023-11-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.13.566852' target='https://doi.org/10.1101/2023.11.13.566852'> 10.1101/2023.11.13.566852</a></p>
        <p class="abstract">BackgroundProgressive functional decline is a key element of cancer-associated cachexia. No therapies have successfully translated to the clinic due to an inability to measure and improve physical function in cachectic patients. Major barriers to translating pre-clinical therapies to the clinic include lack of cancer models that accurately mimic functional decline and use of non-specific outcome measures of function, like grip strength. New approaches are needed to investigate cachexia-related function at both the basic and clinical science levels.

MethodsSurvival extension studies were performed by testing multiple cell lines, dilutions, and vehicle-types in orthotopic implantation of K-rasLSL.G12D/&#43;; Trp53R172H/&#43;; Pdx-1-Cre (KPC) derived cells. 128 animals in this new model were then assessed for muscle wasting, inflammation, and functional decline using a battery of biochemical, physiologic, and behavioral techniques. In parallel, we analyzed a 156-subject cohort of cancer patients with a range of cachexia severity, and who required rehabilitation, to determine the relationship between gait speed via six-minute walk test (6MWT), grip strength (hGS), and functional independence measures (FIM). Cachectic patients were identified using the Weight Loss Grading Scale (WLGS), Fearon consensus criteria, and the Prognostic Nutritional Index (PNI).

ResultsUsing a 100-cell dose of DT10022 KPC cells, we extended the survival of the KPC orthotopic model to 8-9 weeks post-implantation compared to higher doses used (p&lt;0.001). In this Low-dose Orthotopic (LO) model, both progressive skeletal and cardiac muscle wasting were detected in parallel to systemic inflammation; skeletal muscle atrophy at the fiber level was detected as early as 3 weeks post-implantation compared to controls (p&lt;0.001). Gait speed in LO animals declined as early 2 week post-implantation whereas grip strength change was a late event and related to end of life. Principle component analysis (PCA) revealed distinct cachectic and non-cachectic animal populations, which we leveraged to show that gait speed decline was specific to cachexia (p&lt;0.01) while grip strength decline was not (p=0.19). These data paralleled our observations in cancer patients with cachexia who required rehabilitation. In cachectic patients (identified by WLGS, Fearon criteria, or PNI, change in 6MWT correlated with motor FIM score changes while hGS did not (r2=0.18, p&lt;0.001). This relationship between 6MWT and FIM in cachectic patients was further confirmed through multivariate regression (r2=0.30, p&lt;0.001) controlling for age and cancer burden.

ConclusionOutcome measures linked to gait are better associated with cachexia related function and preferred for future pre-clinical and clinical cachexia studies.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
