<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Major data analysis errors invalidate cancer microbiome findings
Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.
Score: 668.8, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993
We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Major data analysis errors invalidate cancer microbiome findings
Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.
Score: 668.8, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993
We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-30T10:38:23+00:00" />
<meta property="article:modified_time" content="2023-08-30T10:38:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Major data analysis errors invalidate cancer microbiome findings
Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.
Score: 668.8, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993
We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Major data analysis errors invalidate cancer microbiome findings\nAuthors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.\nScore: 668.8, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993\nWe re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier.",
  "keywords": [
    
  ],
  "articleBody": " Major data analysis errors invalidate cancer microbiome findings\nAuthors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.\nScore: 668.8, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993\nWe re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier. Each of these problems invalidates the results, leading to the conclusion that the microbiome-based classifiers for identifying cancer presented in the study are entirely wrong. These flaws have subsequently affected more than a dozen additional published studies that used the same data and whose results are likely invalid as well.\nProgressive plasticity during colorectal cancer metastasis\nAuthors: Moorman, A.; Cambuli, F.; Benitez, E.; Jiang, Q.; Xie, Y.; Mahmoud, A.; Lumish, M.; Hartner, S.; Balkaran, S.; Bermeo, J.; Asawa, S.; Firat, C.; Saxena, A.; Luthra, A.; Sgambati, V.; Luckett, K.; Wu, F.; Li, Y.; Yi, Z.; Masilionis, I.; Soares, K.; Pappou, E.; Yaeger, R.; Kingham, P.; Jarnagin, W.; Paty, P.; Weiser, M. R.; Mazutis, L.; D'Angelica, M.; Shia, J.; Garcia-Aguilar, J.; Nawy, T.; Hollman, T.; Chaligne, R.; Sanchez-Vega, F.; Sharma, R.; Pe'er, D.; Ganesh, K.\nScore: 56.7, Published: 2023-08-21 DOI: 10.1101/2023.08.18.553925\nMetastasis is the principal cause of cancer death, yet we lack an understanding of metastatic cell states, their relationship to primary tumor states, and the mechanisms by which they transition. In a cohort of biospecimen trios from same-patient normal colon, primary and metastatic colorectal cancer, we show that while primary tumors largely adopt LGR5+ intestinal stem-like states, metastases display progressive plasticity. Loss of intestinal cell states is accompanied by reprogramming into a highly conserved fetal progenitor state, followed by non-canonical differentiation into divergent squamous and neuroendocrine-like states, which is exacerbated by chemotherapy and associated with poor patient survival. Using matched patient-derived organoids, we demonstrate that metastatic cancer cells exhibit greater cell-autonomous multilineage differentiation potential in response to microenvironment cues than their intestinal lineage-restricted primary tumor counterparts. We identify PROX1 as a stabilizer of intestinal lineage in the fetal progenitor state, whose downregulation licenses non-canonical reprogramming.\nIntegration of Multiple Spatial-Omics Modalities Reveals Unique Insights into Molecular Heterogeneity of Prostate Cancer\nAuthors: Zhang, W.; Spotbeen, X.; Vanuytven, S.; Kint, S.; Sarretto, T.; Socciarelli, F.; Vandereyken, K.; Dehairs, J.; Idkowiak, J.; Wouters, D.; Alvira Larizgoitia, J. I.; Partel, G.; Ly, A.; de Laat, V.; Q. Mantas, M. J.; Gevaert, T.; Devlies, W.; Mah, C. Y.; Butler, L. M.; Loda, M.; Joniau, S.; De Moor, B.; Sifrim, A.; Ellis, S. R.; Voet, T.; Claesen, M.; Verbeeck, N.; Swinnen, J. V.\nScore: 12.9, Published: 2023-08-28 DOI: 10.1101/2023.08.28.555056\nRecent advances in spatial omics methods are revolutionising biomedical research by enabling detailed molecular analyses of cells and their interactions in their native state. As most technologies capture only a specific type of molecules, there is an unmet need to enable integration of multiple spatial-omics datasets. This, however, presents several challenges as these analyses typically operate on separate tissue sections at disparate spatial resolutions. Here, we established a spatial multi-omics integration pipeline enabling co-registration and granularity matching, and applied it to integrate spatial transcriptomics, mass spectrometry-based lipidomics, single nucleus RNA-seq and histomorphological information from human prostate cancer patient samples. This approach revealed unique correlations between lipids and gene expression profiles that are linked to distinct cell populations and histopathological disease states and uncovered molecularly different subregions not discernible by morphology alone. By its ability to correlate datasets that span across the biomolecular and spatial scale, the application of this novel spatial multi-omics integration pipeline provides unprecedented insight into the intricate interplay between different classes of molecules in a tissue context. In addition, it has unique hypothesis-generating potential, and holds promise for applications in molecular pathology, biomarker and target discovery and other tissue-based research fields.\nAn autochthonous model of lung cancer in the Naked Mole-Rat (Heterocephalus glaber)\nAuthors: Kissil, J.; Shepard, A.; Troutman, S.; Hoxha, S.; Lester, D.; Khaled, W.; Smith, E. S. J.; Park, T.; Buffenstein, R.; Dongliang, D.; Teng, M. S. J.; Crish, C.; Tsai, K.; Flores, E.; Ventura, A.\nScore: 12.0, Published: 2023-08-28 DOI: 10.1101/2023.08.28.555115\nStudies on cancer resistance in the naked mole-rat (NMR) have generally failed to interrogate possible resistance mechanisms in a physiological context. Here, we provide evidence that the NMR presents as a novel model of tumor initiation. We developed an endogenous lung cancer model in NMRs, driven by an oncogenic Eml4-Alk fusion protein introduced through CRISPR-mediated genome editing. While this is sufficient to drive tumorigenesis in mice, the development of progressive disease in NMRs required the additional loss of key tumor suppressors. Our results show that tumor initiation in NMRs more closely recapitulates that of human tumors. This suggests that the proposed resistance of NMRs to cancer development may stem from tumor initiation events that are likely to be comparable to the mechanisms in human cells.\nMutational order and epistasis determine the consequences of FBXW7 mutations during colorectal cancer evolution\nAuthors: Chan, D. K. H.; Mandal, A.; Collins, S. D.; Zhou, Y.; Owen, R.; Bundred, J.; Househam, J.; Graham, T. A.; Lu, X.; Buczacki, S. J. A.\nScore: 7.0, Published: 2023-08-27 DOI: 10.1101/2023.08.25.554836\nSomatic driver mutations, in genes such as FBXW7, have been discovered in phenotypically normal colonic tissue, however their role in cancer initiation remains elusive. Here, using patient-derived human colon organoids as models of early tumour evolution we investigate the consequences of FBXW7 mutations in normal and gene-edited organoids. We observed that FBXW7 mutations exert an epistatic effect where the transcriptional consequences of the mutation are dependent on the background mutational makeup of the cell. Specifically, we found the timing of acquiring an FBXW7 mutation relative to APC mutation, led to profound differences. When FBXW7 was mutated before APC, repression of the APC transcriptional response and maintenance of near-normal cell state was seen. However, when APC was mutated before FBXW7, cells acquired classic cancer-stem cell features. Single-cell RNA sequencing revealed that FBXW7 mutations in normal tissue also function by subtly reordering stem cell hierarchies and priming a fetal/regenerative phenotype through upregulation of YAP/TAZ signaling. Further analysis using transposase-accessible chromatin sequencing found this cellular plasticity was driven by changes in the chromatin accessibility of 36 transcriptional start site regions associated with TEAD1/TEAD2 motifs, which in turn upregulated YAP. Taken together, we demonstrate a critical role of FBXW7 mutations in preventing the initiation of colorectal cancer, and provide exemplar evidence for the importance of epistasis and mutational order in cancer biology.\nThe translation of a short open reading frame product within the human TUBA1B gene regulates cancer cell proliferation by importin-β\nAuthors: Tao, Y.; Bai, X.; Zhou, Y.; Zhao, Y.; Yang, L.; Zhang, S.; Yan, S.; Huang, X.; Schneider, E.; Zampetaki, A.; Margariti, A.; Giacca, M.; Arnold, J. N.; Zeng, L.; Cai, T.\nScore: 4.2, Published: 2023-08-27 DOI: 10.1101/2023.08.26.554759\nUnderstanding cancer biology and the molecular mechanisms underlying cancer progress are crucial for improving cancer treatment strategies. This study revealed a short open reading frame product (sORF1) alternatively translated from the human -tubulin gene (TUBA1B), which has the completely different amino acids sequence from its main ORF product, -tubulin 1B chain. sORF1 is highly expressed in cancer cell lines and positive sORF1 cells are found in gastric carcinoma. sORF1 interacts with more than one hundred proteins including importin {beta}. It also acts as a modifier like small ubiquitin-like modifier (SUMO) by covalent binding. Knockdown of sORF1 in cancer cells decreases cell proliferation and tumorigenicity. Further study reveals that the loss of sORF1-importin {beta} connections inhibits importin {beta} guided {beta}-catenin nucleus translocation, thus downregulates Wnt/{beta}-catenin pathway. These findings indicate that gene TUBA1B is a polycistronic gene, able to translate two distinct proteins and the TUBA1B-sORF1 functions as a key tumorigenesis regulator. TUBA1B-sORF1 possesses great potential to be developed as therapeutic target and diagnosis biomarker for cancer.\nAge and obesity-driven changes in the extracellular matrix of the primary tumor and metastatic site influence tumor invasion and metastatic outgrowth\nAuthors: Conner, S. J.; Guarin, J. R.; Borges, H.; Salhany, K. J.; Mensah, D. N.; Hamilton, G. A.; Le, G. H.; Oudin, M. J.\nScore: 4.0, Published: 2023-08-25 DOI: 10.1101/2023.08.24.554492\nYounger age and obesity increase the incidence and metastasis of triple-negative breast cancer (TNBC), an aggressive subtype of breast cancer. The extracellular matrix (ECM) promotes tumor invasion and metastasis. We characterized the effect of age and obesity on the ECM of mammary fat pads, lungs, and liver using a diet-induced obesity (DIO) model. At 4 week intervals, we either injected the mammary fat pads with allograft tumor cells to characterize tumor growth and metastasis or isolated the mammary fat pads and livers to characterize the ECM. Age had no effect on tumor growth but increased lung and liver metastasis after 16 weeks. Obesity increased tumor growth starting at 12 weeks, increased liver metastasis only at 4 weeks, and weight gain correlated to increased lung but not liver metastasis. Utilizing whole decellularized ECM coupled with proteomics, we found that early stages of obesity were sufficient to induce changes in the ECM composition and invasive potential of mammary fat pads with increased abundance of pro-invasive ECM proteins Collagen IV and Collagen VI. We identified cells of stromal vascular fraction and adipose stem and progenitor cells as primarily responsible for secreting Collagen IV and VI, not adipocytes. We characterized the changes in ECM in the lungs and liver, and determined that older age decreases the metastatic potential of lung and liver ECM while later-stage obesity increases the metastatic potential. These data implicate ECM changes in the primary tumor and metastatic microenvironment as mechanisms by which age and obesity contribute to breast cancer progression. SignificanceYounger age and obesity increase the incidence and metastasis of triple-negative breast cancer. Our data suggest that changes in the breast, lung and liver ECM are driving some of these effects.\nUnraveling Vulnerabilities in Endocrine Therapy-Resistant HER2+/ER+ Breast Cancer\nAuthors: Bahnassy, S.; Stires, H.; Jin, L.; Tam, S.; Mobin, D.; Balachandran, M.; Podar, M. B.; McCoy, M. D.; Beckman, R. A.; Riggins, R. B.\nScore: 4.5, Published: 2023-08-22 DOI: 10.1101/2023.08.21.554116\nBackgroundBreast tumors overexpressing human epidermal growth factor receptor (HER2) confer intrinsic resistance to endocrine therapy (ET), and patients with HER2/ estrogen receptor-positive (HER2+/HR+) breast cancer (BCa) are less responsive to ET than HER2-/ER+. However, real-world evidence reveals that a large subset of HER2+/ER+ patients receive ET as monotherapy, positioning this treatment pattern as a clinical challenge. In the present study, we developed and characterized two distinct in vitro models of ET-resistant (ETR) HER2+/ER+ BCa to identify possible therapeutic vulnerabilities. MethodsTo mimic ETR to aromatase inhibitors (AI), we developed two long-term estrogen-deprived (LTED) cell lines from BT-474 (BT474) and MDA-MB-361 (MM361). Growth assays, PAM50 molecular subtyping, genomic and transcriptomic analyses, followed by validation and functional studies, were used to identify targetable differences between ET-responsive parental and ETR-LTED HER2+/ER+ cells. ResultsCompared to their parental cells, MM361 LTEDs grew faster, lost ER, and increased HER2 expression, whereas BT474 LTEDs grew slower and maintained ER and HER2 expression. Both LTED variants had reduced responsiveness to fulvestrant. Whole-genome sequencing of the more aggressive MM361 LTED model system identified exonic mutations in genes encoding transcription factors and chromatin modifiers. Single-cell RNA sequencing demonstrated a shift towards non-luminal phenotypes, and revealed metabolic remodeling of MM361 LTEDs, with upregulated lipid metabolism and antioxidant genes associated with ferroptosis, including GPX4. Combining the GPX4 inhibitor RSL3 with anti-HER2 agents induced significant cell death in both the MM361 and BT474 LTEDs. ConclusionsThe BT474 and MM361 AI-resistant models capture distinct phenotypes of HER2+/ER+ BCa and identify altered lipid metabolism and ferroptosis remodeling as vulnerabilities of this type of ETR BCa.\nOvercome Prostate Cancer Resistance to Immune Checkpoint Therapy with Ketogenic Diet-Induced Epigenetic Reprogramming\nAuthors: Murphy, S.; Rahmy, S.; Gan, D.; Zhu, Y.; Manyak, M.; Li, J.; Lu, X.; Lu, X.\nScore: 6.0, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552383\nAdvanced prostate cancer (PCa) is overwhelmingly resistant to immune checkpoint blockade (ICB) therapy, representing a formidable clinical challenge. In this study, we developed a syngeneic murine PCa model with acquired ICB resistance. Using this model, synergistic efficacy was achieved by combining anti-PD1 and anti-CTLA4 antibodies with histone deacetylase inhibitor (HDACi) vorinostat, a cyclic ketogenic diet (CKD), or supplementation of ketone body {beta}-hydroxybutyrate (BHB, endogenous HDACi) via 1,3-butanediol-admixed food. CKD and BHB supplementation delayed PCa tumors as monotherapy, and both BHB and adaptive immunity are required for the anti-tumor activity of CKD. Single-cell transcriptomic and proteomic profiling revealed that the HDACi and ketogenesis-enhanced ICB therapy involves cancer-cell-intrinsic (upregulated MHC class I molecules) and extrinsic mechanisms (CD8+ T cell chemoattraction, M1/M2 macrophage rebalancing, monocyte differentiation toward antigen presenting cells, and diminished neutrophils). Overall, these findings underscore the potential of using HDACi and optimized KD to enhance ICB therapy for PCa.\nCaspase-2 protects against ferroptotic cell death\nAuthors: Dawar, S.; Benitez, M. C.; Lim, Y.; Dite, T. A.; Yousef, J. M.; Thio, N.; Garciaz, S.; Jackson, T. D.; Milne, J. V.; Dagley, L. F.; Phillips, W. A.; Kumar, S.; Clemons, N. J.\nScore: 2.8, Published: 2023-08-26 DOI: 10.1101/2023.08.22.553791\nCaspase-2, one of the most evolutionarily conserved member of the caspase family, is an important regulator of the cellular response to oxidative stress. Given that ferroptosis is suppressed by antioxidant defense pathways, such as that involving selenoenzyme glutathione peroxidase 4 (GPX4), we hypothesised that caspase-2 may play a role in regulating ferroptosis. This study provides the first demonstration of an important and unprecedented function of caspase-2 in protecting cancer cells from undergoing ferroptotic cell death. Specifically, we show that depletion of caspase-2 leads to downregulation of stress response genes including SESN2, HMOX1, SLC7A11 and sensitises mutant-p53 cancer cells to cell death induced by various ferroptosis inducing compounds. Importantly, the canonical catalytic activity of caspase-2 is not required for its role and suggests that caspase-2 regulates ferroptosis via non-proteolytic interaction with other proteins. Using an unbiased BioID proteomics screen, we identified novel caspase-2 interacting proteins (including heat shock proteins and co-chaperones) that regulate cellular responses to stress. Finally, we demonstrate that caspase-2 limits chaperone mediated autophagic degradation of GPX4 to promote survival of mutant-p53 cancer cells. In conclusion, we document a novel role for caspase-2 as a negative regulator of ferroptosis in cells with mutant-p53. Our results provide evidence for a novel function of caspase-2 functions in cell death regulation and open potential new avenues to exploit ferroptosis in cancer therapy.\n",
  "wordCount" : "2473",
  "inLanguage": "en",
  "datePublished": "2023-08-30T10:38:23Z",
  "dateModified": "2023-08-30T10:38:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on August 30, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550993">
        <p class="paperTitle">Major data analysis errors invalidate cancer microbiome findings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.</p>
        <p class="info">Score: 668.8, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550993' target='https://doi.org/10.1101/2023.07.28.550993'> 10.1101/2023.07.28.550993</a></p>
        <p class="abstract">We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier. Each of these problems invalidates the results, leading to the conclusion that the microbiome-based classifiers for identifying cancer presented in the study are entirely wrong. These flaws have subsequently affected more than a dozen additional published studies that used the same data and whose results are likely invalid as well.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553925">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553925" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553925">
        <p class="paperTitle">Progressive plasticity during colorectal cancer metastasis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553925" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553925" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moorman, A.; Cambuli, F.; Benitez, E.; Jiang, Q.; Xie, Y.; Mahmoud, A.; Lumish, M.; Hartner, S.; Balkaran, S.; Bermeo, J.; Asawa, S.; Firat, C.; Saxena, A.; Luthra, A.; Sgambati, V.; Luckett, K.; Wu, F.; Li, Y.; Yi, Z.; Masilionis, I.; Soares, K.; Pappou, E.; Yaeger, R.; Kingham, P.; Jarnagin, W.; Paty, P.; Weiser, M. R.; Mazutis, L.; D&#39;Angelica, M.; Shia, J.; Garcia-Aguilar, J.; Nawy, T.; Hollman, T.; Chaligne, R.; Sanchez-Vega, F.; Sharma, R.; Pe&#39;er, D.; Ganesh, K.</p>
        <p class="info">Score: 56.7, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553925' target='https://doi.org/10.1101/2023.08.18.553925'> 10.1101/2023.08.18.553925</a></p>
        <p class="abstract">Metastasis is the principal cause of cancer death, yet we lack an understanding of metastatic cell states, their relationship to primary tumor states, and the mechanisms by which they transition. In a cohort of biospecimen trios from same-patient normal colon, primary and metastatic colorectal cancer, we show that while primary tumors largely adopt LGR5&#43; intestinal stem-like states, metastases display progressive plasticity. Loss of intestinal cell states is accompanied by reprogramming into a highly conserved fetal progenitor state, followed by non-canonical differentiation into divergent squamous and neuroendocrine-like states, which is exacerbated by chemotherapy and associated with poor patient survival. Using matched patient-derived organoids, we demonstrate that metastatic cancer cells exhibit greater cell-autonomous multilineage differentiation potential in response to microenvironment cues than their intestinal lineage-restricted primary tumor counterparts. We identify PROX1 as a stabilizer of intestinal lineage in the fetal progenitor state, whose downregulation licenses non-canonical reprogramming.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555056">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555056" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555056">
        <p class="paperTitle">Integration of Multiple Spatial-Omics Modalities Reveals Unique Insights into Molecular Heterogeneity of Prostate Cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555056" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555056" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, W.; Spotbeen, X.; Vanuytven, S.; Kint, S.; Sarretto, T.; Socciarelli, F.; Vandereyken, K.; Dehairs, J.; Idkowiak, J.; Wouters, D.; Alvira Larizgoitia, J. I.; Partel, G.; Ly, A.; de Laat, V.; Q. Mantas, M. J.; Gevaert, T.; Devlies, W.; Mah, C. Y.; Butler, L. M.; Loda, M.; Joniau, S.; De Moor, B.; Sifrim, A.; Ellis, S. R.; Voet, T.; Claesen, M.; Verbeeck, N.; Swinnen, J. V.</p>
        <p class="info">Score: 12.9, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555056' target='https://doi.org/10.1101/2023.08.28.555056'> 10.1101/2023.08.28.555056</a></p>
        <p class="abstract">Recent advances in spatial omics methods are revolutionising biomedical research by enabling detailed molecular analyses of cells and their interactions in their native state. As most technologies capture only a specific type of molecules, there is an unmet need to enable integration of multiple spatial-omics datasets. This, however, presents several challenges as these analyses typically operate on separate tissue sections at disparate spatial resolutions. Here, we established a spatial multi-omics integration pipeline enabling co-registration and granularity matching, and applied it to integrate spatial transcriptomics, mass spectrometry-based lipidomics, single nucleus RNA-seq and histomorphological information from human prostate cancer patient samples. This approach revealed unique correlations between lipids and gene expression profiles that are linked to distinct cell populations and histopathological disease states and uncovered molecularly different subregions not discernible by morphology alone. By its ability to correlate datasets that span across the biomolecular and spatial scale, the application of this novel spatial multi-omics integration pipeline provides unprecedented insight into the intricate interplay between different classes of molecules in a tissue context. In addition, it has unique hypothesis-generating potential, and holds promise for applications in molecular pathology, biomarker and target discovery and other tissue-based research fields.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.28.555115">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.28.555115" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.28.555115">
        <p class="paperTitle">An autochthonous model of lung cancer in the Naked Mole-Rat (Heterocephalus glaber)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.28.555115" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.28.555115" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kissil, J.; Shepard, A.; Troutman, S.; Hoxha, S.; Lester, D.; Khaled, W.; Smith, E. S. J.; Park, T.; Buffenstein, R.; Dongliang, D.; Teng, M. S. J.; Crish, C.; Tsai, K.; Flores, E.; Ventura, A.</p>
        <p class="info">Score: 12.0, Published: 2023-08-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.28.555115' target='https://doi.org/10.1101/2023.08.28.555115'> 10.1101/2023.08.28.555115</a></p>
        <p class="abstract">Studies on cancer resistance in the naked mole-rat (NMR) have generally failed to interrogate possible resistance mechanisms in a physiological context. Here, we provide evidence that the NMR presents as a novel model of tumor initiation. We developed an endogenous lung cancer model in NMRs, driven by an oncogenic Eml4-Alk fusion protein introduced through CRISPR-mediated genome editing. While this is sufficient to drive tumorigenesis in mice, the development of progressive disease in NMRs required the additional loss of key tumor suppressors. Our results show that tumor initiation in NMRs more closely recapitulates that of human tumors. This suggests that the proposed resistance of NMRs to cancer development may stem from tumor initiation events that are likely to be comparable to the mechanisms in human cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.25.554836">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.25.554836" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.25.554836">
        <p class="paperTitle">Mutational order and epistasis determine the consequences of FBXW7 mutations during colorectal cancer evolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.25.554836" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.25.554836" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chan, D. K. H.; Mandal, A.; Collins, S. D.; Zhou, Y.; Owen, R.; Bundred, J.; Househam, J.; Graham, T. A.; Lu, X.; Buczacki, S. J. A.</p>
        <p class="info">Score: 7.0, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.25.554836' target='https://doi.org/10.1101/2023.08.25.554836'> 10.1101/2023.08.25.554836</a></p>
        <p class="abstract">Somatic driver mutations, in genes such as FBXW7, have been discovered in phenotypically normal colonic tissue, however their role in cancer initiation remains elusive. Here, using patient-derived human colon organoids as models of early tumour evolution we investigate the consequences of FBXW7 mutations in normal and gene-edited organoids. We observed that FBXW7 mutations exert an epistatic effect where the transcriptional consequences of the mutation are dependent on the background mutational makeup of the cell. Specifically, we found the timing of acquiring an FBXW7 mutation relative to APC mutation, led to profound differences. When FBXW7 was mutated before APC, repression of the APC transcriptional response and maintenance of near-normal cell state was seen. However, when APC was mutated before FBXW7, cells acquired classic cancer-stem cell features. Single-cell RNA sequencing revealed that FBXW7 mutations in normal tissue also function by subtly reordering stem cell hierarchies and priming a fetal/regenerative phenotype through upregulation of YAP/TAZ signaling. Further analysis using transposase-accessible chromatin sequencing found this cellular plasticity was driven by changes in the chromatin accessibility of 36 transcriptional start site regions associated with TEAD1/TEAD2 motifs, which in turn upregulated YAP. Taken together, we demonstrate a critical role of FBXW7 mutations in preventing the initiation of colorectal cancer, and provide exemplar evidence for the importance of epistasis and mutational order in cancer biology.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.26.554759">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.26.554759" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.26.554759">
        <p class="paperTitle">The translation of a short open reading frame product within the human TUBA1B gene regulates cancer cell proliferation by importin-β</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.26.554759" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.26.554759" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tao, Y.; Bai, X.; Zhou, Y.; Zhao, Y.; Yang, L.; Zhang, S.; Yan, S.; Huang, X.; Schneider, E.; Zampetaki, A.; Margariti, A.; Giacca, M.; Arnold, J. N.; Zeng, L.; Cai, T.</p>
        <p class="info">Score: 4.2, Published: 2023-08-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.26.554759' target='https://doi.org/10.1101/2023.08.26.554759'> 10.1101/2023.08.26.554759</a></p>
        <p class="abstract">Understanding cancer biology and the molecular mechanisms underlying cancer progress are crucial for improving cancer treatment strategies. This study revealed a short open reading frame product (sORF1) alternatively translated from the human -tubulin gene (TUBA1B), which has the completely different amino acids sequence from its main ORF product, -tubulin 1B chain. sORF1 is highly expressed in cancer cell lines and positive sORF1 cells are found in gastric carcinoma. sORF1 interacts with more than one hundred proteins including importin {beta}. It also acts as a modifier like small ubiquitin-like modifier (SUMO) by covalent binding. Knockdown of sORF1 in cancer cells decreases cell proliferation and tumorigenicity. Further study reveals that the loss of sORF1-importin {beta} connections inhibits importin {beta} guided {beta}-catenin nucleus translocation, thus downregulates Wnt/{beta}-catenin pathway. These findings indicate that gene TUBA1B is a polycistronic gene, able to translate two distinct proteins and the TUBA1B-sORF1 functions as a key tumorigenesis regulator. TUBA1B-sORF1 possesses great potential to be developed as therapeutic target and diagnosis biomarker for cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.24.554492">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.24.554492" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.24.554492">
        <p class="paperTitle">Age and obesity-driven changes in the extracellular matrix of the primary tumor and metastatic site influence tumor invasion and metastatic outgrowth</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.24.554492" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.24.554492" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Conner, S. J.; Guarin, J. R.; Borges, H.; Salhany, K. J.; Mensah, D. N.; Hamilton, G. A.; Le, G. H.; Oudin, M. J.</p>
        <p class="info">Score: 4.0, Published: 2023-08-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.24.554492' target='https://doi.org/10.1101/2023.08.24.554492'> 10.1101/2023.08.24.554492</a></p>
        <p class="abstract">Younger age and obesity increase the incidence and metastasis of triple-negative breast cancer (TNBC), an aggressive subtype of breast cancer. The extracellular matrix (ECM) promotes tumor invasion and metastasis. We characterized the effect of age and obesity on the ECM of mammary fat pads, lungs, and liver using a diet-induced obesity (DIO) model. At 4 week intervals, we either injected the mammary fat pads with allograft tumor cells to characterize tumor growth and metastasis or isolated the mammary fat pads and livers to characterize the ECM. Age had no effect on tumor growth but increased lung and liver metastasis after 16 weeks. Obesity increased tumor growth starting at 12 weeks, increased liver metastasis only at 4 weeks, and weight gain correlated to increased lung but not liver metastasis. Utilizing whole decellularized ECM coupled with proteomics, we found that early stages of obesity were sufficient to induce changes in the ECM composition and invasive potential of mammary fat pads with increased abundance of pro-invasive ECM proteins Collagen IV and Collagen VI. We identified cells of stromal vascular fraction and adipose stem and progenitor cells as primarily responsible for secreting Collagen IV and VI, not adipocytes. We characterized the changes in ECM in the lungs and liver, and determined that older age decreases the metastatic potential of lung and liver ECM while later-stage obesity increases the metastatic potential. These data implicate ECM changes in the primary tumor and metastatic microenvironment as mechanisms by which age and obesity contribute to breast cancer progression.

SignificanceYounger age and obesity increase the incidence and metastasis of triple-negative breast cancer. Our data suggest that changes in the breast, lung and liver ECM are driving some of these effects.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.21.554116">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.21.554116" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.21.554116">
        <p class="paperTitle">Unraveling Vulnerabilities in Endocrine Therapy-Resistant HER2&#43;/ER&#43; Breast Cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.21.554116" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.21.554116" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bahnassy, S.; Stires, H.; Jin, L.; Tam, S.; Mobin, D.; Balachandran, M.; Podar, M. B.; McCoy, M. D.; Beckman, R. A.; Riggins, R. B.</p>
        <p class="info">Score: 4.5, Published: 2023-08-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.21.554116' target='https://doi.org/10.1101/2023.08.21.554116'> 10.1101/2023.08.21.554116</a></p>
        <p class="abstract">BackgroundBreast tumors overexpressing human epidermal growth factor receptor (HER2) confer intrinsic resistance to endocrine therapy (ET), and patients with HER2/ estrogen receptor-positive (HER2&#43;/HR&#43;) breast cancer (BCa) are less responsive to ET than HER2-/ER&#43;. However, real-world evidence reveals that a large subset of HER2&#43;/ER&#43; patients receive ET as monotherapy, positioning this treatment pattern as a clinical challenge. In the present study, we developed and characterized two distinct in vitro models of ET-resistant (ETR) HER2&#43;/ER&#43; BCa to identify possible therapeutic vulnerabilities.

MethodsTo mimic ETR to aromatase inhibitors (AI), we developed two long-term estrogen-deprived (LTED) cell lines from BT-474 (BT474) and MDA-MB-361 (MM361). Growth assays, PAM50 molecular subtyping, genomic and transcriptomic analyses, followed by validation and functional studies, were used to identify targetable differences between ET-responsive parental and ETR-LTED HER2&#43;/ER&#43; cells.

ResultsCompared to their parental cells, MM361 LTEDs grew faster, lost ER, and increased HER2 expression, whereas BT474 LTEDs grew slower and maintained ER and HER2 expression. Both LTED variants had reduced responsiveness to fulvestrant. Whole-genome sequencing of the more aggressive MM361 LTED model system identified exonic mutations in genes encoding transcription factors and chromatin modifiers. Single-cell RNA sequencing demonstrated a shift towards non-luminal phenotypes, and revealed metabolic remodeling of MM361 LTEDs, with upregulated lipid metabolism and antioxidant genes associated with ferroptosis, including GPX4. Combining the GPX4 inhibitor RSL3 with anti-HER2 agents induced significant cell death in both the MM361 and BT474 LTEDs.

ConclusionsThe BT474 and MM361 AI-resistant models capture distinct phenotypes of HER2&#43;/ER&#43; BCa and identify altered lipid metabolism and ferroptosis remodeling as vulnerabilities of this type of ETR BCa.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552383">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552383" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552383">
        <p class="paperTitle">Overcome Prostate Cancer Resistance to Immune Checkpoint Therapy with Ketogenic Diet-Induced Epigenetic Reprogramming</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552383" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552383" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Murphy, S.; Rahmy, S.; Gan, D.; Zhu, Y.; Manyak, M.; Li, J.; Lu, X.; Lu, X.</p>
        <p class="info">Score: 6.0, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552383' target='https://doi.org/10.1101/2023.08.07.552383'> 10.1101/2023.08.07.552383</a></p>
        <p class="abstract">Advanced prostate cancer (PCa) is overwhelmingly resistant to immune checkpoint blockade (ICB) therapy, representing a formidable clinical challenge. In this study, we developed a syngeneic murine PCa model with acquired ICB resistance. Using this model, synergistic efficacy was achieved by combining anti-PD1 and anti-CTLA4 antibodies with histone deacetylase inhibitor (HDACi) vorinostat, a cyclic ketogenic diet (CKD), or supplementation of ketone body {beta}-hydroxybutyrate (BHB, endogenous HDACi) via 1,3-butanediol-admixed food. CKD and BHB supplementation delayed PCa tumors as monotherapy, and both BHB and adaptive immunity are required for the anti-tumor activity of CKD. Single-cell transcriptomic and proteomic profiling revealed that the HDACi and ketogenesis-enhanced ICB therapy involves cancer-cell-intrinsic (upregulated MHC class I molecules) and extrinsic mechanisms (CD8&#43; T cell chemoattraction, M1/M2 macrophage rebalancing, monocyte differentiation toward antigen presenting cells, and diminished neutrophils). Overall, these findings underscore the potential of using HDACi and optimized KD to enhance ICB therapy for PCa.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.22.553791">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.22.553791" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.22.553791">
        <p class="paperTitle">Caspase-2 protects against ferroptotic cell death</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.22.553791" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.22.553791" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dawar, S.; Benitez, M. C.; Lim, Y.; Dite, T. A.; Yousef, J. M.; Thio, N.; Garciaz, S.; Jackson, T. D.; Milne, J. V.; Dagley, L. F.; Phillips, W. A.; Kumar, S.; Clemons, N. J.</p>
        <p class="info">Score: 2.8, Published: 2023-08-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.22.553791' target='https://doi.org/10.1101/2023.08.22.553791'> 10.1101/2023.08.22.553791</a></p>
        <p class="abstract">Caspase-2, one of the most evolutionarily conserved member of the caspase family, is an important regulator of the cellular response to oxidative stress. Given that ferroptosis is suppressed by antioxidant defense pathways, such as that involving selenoenzyme glutathione peroxidase 4 (GPX4), we hypothesised that caspase-2 may play a role in regulating ferroptosis. This study provides the first demonstration of an important and unprecedented function of caspase-2 in protecting cancer cells from undergoing ferroptotic cell death. Specifically, we show that depletion of caspase-2 leads to downregulation of stress response genes including SESN2, HMOX1, SLC7A11 and sensitises mutant-p53 cancer cells to cell death induced by various ferroptosis inducing compounds. Importantly, the canonical catalytic activity of caspase-2 is not required for its role and suggests that caspase-2 regulates ferroptosis via non-proteolytic interaction with other proteins. Using an unbiased BioID proteomics screen, we identified novel caspase-2 interacting proteins (including heat shock proteins and co-chaperones) that regulate cellular responses to stress. Finally, we demonstrate that caspase-2 limits chaperone mediated autophagic degradation of GPX4 to promote survival of mutant-p53 cancer cells. In conclusion, we document a novel role for caspase-2 as a negative regulator of ferroptosis in cells with mutant-p53. Our results provide evidence for a novel function of caspase-2 functions in cell death regulation and open potential new avenues to exploit ferroptosis in cancer therapy.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
