<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Integrative spatial analysis reveals a multi-layered organization of glioblastoma
Authors: Greenwald, A. C.; Darnell, N. G.; Hoefflin, R.; Simkin, D.; Gonzalez-Castro, L. N.; Mount, C.; Hirsch, D.; Nomura, M.; Talpir, T.; Kedmi, M.; Goliand, I.; Medici, G.; Li, B.; Keren-Shaul, H.; Weller, M.; Addadi, Y.; Neidert, M. C.; Suva, M. L.; Tirosh, I.
Score: 74.7, Published: 2023-07-08 DOI: 10.1101/2023.07.06.547924
Glioma contains malignant cells in diverse states. Hypoxic regions are associated with a unique histology of pseudopalisading cells, while other regions appear to have limited histological organization, reflecting the diffuse nature of glioma cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Integrative spatial analysis reveals a multi-layered organization of glioblastoma
Authors: Greenwald, A. C.; Darnell, N. G.; Hoefflin, R.; Simkin, D.; Gonzalez-Castro, L. N.; Mount, C.; Hirsch, D.; Nomura, M.; Talpir, T.; Kedmi, M.; Goliand, I.; Medici, G.; Li, B.; Keren-Shaul, H.; Weller, M.; Addadi, Y.; Neidert, M. C.; Suva, M. L.; Tirosh, I.
Score: 74.7, Published: 2023-07-08 DOI: 10.1101/2023.07.06.547924
Glioma contains malignant cells in diverse states. Hypoxic regions are associated with a unique histology of pseudopalisading cells, while other regions appear to have limited histological organization, reflecting the diffuse nature of glioma cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-07-12T10:41:10+00:00" />
<meta property="article:modified_time" content="2023-07-12T10:41:10+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Integrative spatial analysis reveals a multi-layered organization of glioblastoma
Authors: Greenwald, A. C.; Darnell, N. G.; Hoefflin, R.; Simkin, D.; Gonzalez-Castro, L. N.; Mount, C.; Hirsch, D.; Nomura, M.; Talpir, T.; Kedmi, M.; Goliand, I.; Medici, G.; Li, B.; Keren-Shaul, H.; Weller, M.; Addadi, Y.; Neidert, M. C.; Suva, M. L.; Tirosh, I.
Score: 74.7, Published: 2023-07-08 DOI: 10.1101/2023.07.06.547924
Glioma contains malignant cells in diverse states. Hypoxic regions are associated with a unique histology of pseudopalisading cells, while other regions appear to have limited histological organization, reflecting the diffuse nature of glioma cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Integrative spatial analysis reveals a multi-layered organization of glioblastoma\nAuthors: Greenwald, A. C.; Darnell, N. G.; Hoefflin, R.; Simkin, D.; Gonzalez-Castro, L. N.; Mount, C.; Hirsch, D.; Nomura, M.; Talpir, T.; Kedmi, M.; Goliand, I.; Medici, G.; Li, B.; Keren-Shaul, H.; Weller, M.; Addadi, Y.; Neidert, M. C.; Suva, M. L.; Tirosh, I.\nScore: 74.7, Published: 2023-07-08 DOI: 10.1101/2023.07.06.547924\nGlioma contains malignant cells in diverse states. Hypoxic regions are associated with a unique histology of pseudopalisading cells, while other regions appear to have limited histological organization, reflecting the diffuse nature of glioma cells.",
  "keywords": [
    
  ],
  "articleBody": " Integrative spatial analysis reveals a multi-layered organization of glioblastoma\nAuthors: Greenwald, A. C.; Darnell, N. G.; Hoefflin, R.; Simkin, D.; Gonzalez-Castro, L. N.; Mount, C.; Hirsch, D.; Nomura, M.; Talpir, T.; Kedmi, M.; Goliand, I.; Medici, G.; Li, B.; Keren-Shaul, H.; Weller, M.; Addadi, Y.; Neidert, M. C.; Suva, M. L.; Tirosh, I.\nScore: 74.7, Published: 2023-07-08 DOI: 10.1101/2023.07.06.547924\nGlioma contains malignant cells in diverse states. Hypoxic regions are associated with a unique histology of pseudopalisading cells, while other regions appear to have limited histological organization, reflecting the diffuse nature of glioma cells. Here, we combine spatial transcriptomics with spatial proteomics and novel computational approaches to define glioma cellular states at high granularity and uncover their organization. We find three prominent modes of cellular organization. First, cells in any given state tend to be spatially clustered, such that tumors are composed of small local environments that are each typically enriched with one major cellular state. Second, specific pairs of states preferentially reside in proximity across multiple scales. Despite the unique composition of each tumor, this pairing of states remains largely consistent across tumors. Third, the pairwise interactions that we detect collectively define a global architecture composed of five layers. Hypoxia appears to drive this 5-layered organization, as it is both associated with unique states of surrounding cells and with a long-range organization that extends from the hypoxic core to the infiltrative edge of the tumor. Accordingly, tumor regions distant from any hypoxic foci and tumors that lack hypoxia such as IDH-mutant glioma are less organized. In summary, we provide a conceptual framework for the organization of gliomas at the resolution of cellular states and highlight the role of hypoxia as a long-range tissue organizer.\nChemical alkylation of Asp12 enables mutant selective targeting of K-Ras(G12D)\nAuthors: Zhang, Z.; Zheng, Q.; Guiley, K. Z.; Shokat, K.\nScore: 18.6, Published: 2023-07-02 DOI: 10.1101/2023.07.01.547359\nK-Ras is the most commonly mutated oncogene in human cancer, yet direct small-molecule targeting of K-Ras mutants has been mostly unsuccessful until recently. The discovery of an allosteric pocket under Switch-II with covalent cysteine-crosslinking molecules has allowed for the development of targeted therapies that selectively engage the highly reactive acquired cysteine in the K-Ras(G12C) mutation without affecting the wild-type protein. Sotorasib and adagrasib, two advanced Switch-II Pocket inhibitors, have received FDA approval to treat K-Ras(G12C)-driven non-small cell lung cancer. However, the most frequent K-Ras mutation G12D particularly prevalent in pancreatic ductal adenocarcinoma has remained untargetable with covalent drugs due to the poor nucleophilicity of the somatic aspartate residue. Here we present a set of {beta}-lactone-based electrophiles which exploit ring strain to crosslink K-Ras(G12D) at the mutant aspartate to form stable covalent complexes in cells, effectively blocking Ras-effector interaction and downstream signaling. Structural insights from x-ray crystallography and exploitation of the differential chemoselectivity and stereoelectronic requirements for attack of the {beta}-lactone electrophile allowed development of a substituted {beta}-lactone which resisted attack by aqueous buffer but enabled rapid attack by aspartic acid-12 in K-Ras. Our results demonstrate the rational design of covalent inhibitors to target a non-catalytic carboxylic acid side chain in K-Ras(G12D) which has resisted traditional drug discovery efforts.\nRobust prediction of patient outcomes with immune checkpoint blockade therapy for cancer using common clinical, pathologic, and genomic features\nAuthors: Chang, T.-G.; Cao, Y.; Sfreddo, H.; Dhruba, S. R.; Lee, S.-H.; Valero, C.; Yoo, S.-K.; Chowell, D.; Morris, L. G. T.; Ruppin, E.\nScore: 14.1, Published: 2023-07-05 DOI: 10.1101/2023.07.04.547697\nDespite the revolutionary impact of immune checkpoint blockade (ICB) in cancer treatment, accurately predicting patients responses remains elusive. We analyzed eight cohorts of 2881 ICB-treated patients across 18 solid tumor types, the largest dataset to date, examining diverse clinical, pathologic, and genomic features. We developed the LOgistic Regression-based Immunotherapy-response Score (LORIS) using a transparent, compact 6-feature logistic regression model. LORIS outperforms previous signatures in ICB response prediction and can identify responsive patients, even those with low tumor mutational burden or tumor PD-L1 expression. Importantly, LORIS consistently predicts both objective responses and short-term and long-term survival across most cancer types. Moreover, LORIS showcases a near-monotonic relationship with ICB response probability and patient survival, enabling more precise patient stratification across the board. As our method is accurate, interpretable, and only utilizes a few readily measurable features, we anticipate it will help improve clinical decision-making practices in precision medicine to maximize patient benefit.\nIntegrative analyses uncover mechanisms by which aging drives B cell lymphoma\nAuthors: Castro, J. P.; Shindyapina, A. V.; Barbieri, A.; Ying, K.; Strelkova, O. S.; Paulo, J. A.; Tyshkovskiy, A.; Meinl, R.; Kerepesi, C.; Petrashen, A. P.; Mariotti, M.; Meer, M.; Hu, Y.; Karamyshev, A.; Losyev, G.; Indzhykulian, A. A.; Gygi, S. P.; Sedivy, J. M.; Manis, J. P.; Gladyshev, V. N.\nScore: 25.9, Published: 2023-07-11 DOI: 10.1101/2021.02.23.432500\nWhile cancer is an age-related disease, many cancer studies utilize younger animal models. Here, we uncover how a cancer, B-cell lymphoma, develops as a consequence of a naturally aged system. We show that this malignancy is associated with increased cell size, splenomegaly, and a newly discovered age-associated clonal B-cell (ACBC) population. Driven by exogenous c-Myc activation, hypermethylated promoters and somatic mutations, ACBC cells clonally expand independent of germinal centers (IgM+) and show increased biological age and hypomethylation in partially methylated domains related to mitotic solo-CpGs. Epigenetic changes in transformed mouse B cells are enriched for changes observed in human B-cell lymphomas. Mechanistically, the data suggest that cancerous ACBC cells originate from age-associated B cells, in part involving CD22 protein signaling fostered by the aging microenvironment. Transplantation assays demonstrate that ACBC evolve to become self-sufficient and support malignancy when transferred into young recipients. Inhibition of mTOR or c-Myc in old mice attenuates premalignant changes in B cells during aging and emerges as a therapeutic strategy to delay the onset of age-related lymphoma. Together, we show how aging contributes to B-cell lymphoma through a previously unrecognized mechanism involving cell-intrinsic changes and the aged microenvironment, characterize a model that captures the origin and progression of spontaneous cancer during aging and identify candidate interventions against age-associated lymphoma.\nMachine-learning analysis of factors that shape cancer aneuploidy landscapes reveals an important role for negative selection\nAuthors: Jubran, J.; Slutsky, R.; Rozenblum, N.; Rokach, L.; Ben-David, U.; Yeger-Lotem, E.\nScore: 8.8, Published: 2023-07-05 DOI: 10.1101/2023.07.05.547626\nAneuploidy, an abnormal number of chromosomes within a cell, is considered a hallmark of cancer. Patterns of aneuploidy differ across cancers, yet are similar in cancers affecting closely-related tissues. The selection pressures underlying aneuploidy patterns are not fully understood, hindering our understanding of cancer development and progression. Here, we applied interpretable machine learning (ML) methods to study tissue-selective aneuploidy patterns. We defined 20 types of features of normal and cancer tissues, and used them to model gains and losses of chromosome-arms in 24 cancer types. In order to reveal the factors that shape the tissue-specific cancer aneuploidy landscapes, we interpreted the ML models by estimating the relative contribution of each feature to the models. While confirming known drivers of positive selection, our quantitative analysis highlighted the importance of negative selection for shaping the aneuploidy landscapes of human cancer. Tumor-suppressor gene density was a better predictor of gain patterns than oncogene density, and vice-versa for loss patterns. We identified the contribution of tissue-selective features and demonstrated them experimentally for chr13q gain in colon cancer. In line with an important role for negative selection in shaping the aneuploidy landscapes, we found compensation by paralogs to be a top predictor of chromosome-arm loss prevalence, and demonstrated this relationship for one such paralog interaction. Similar factors were found to shape aneuploidy patterns in human cancer cell lines, demonstrating their relevance for aneuploidy research. Overall, our quantitative, interpretable ML models improve the understanding of the genomic properties that shape cancer aneuploidy landscapes.\nSelective inhibition of interleukin-2 inducible T cell kinase (ITK) enhances anti-tumor immunity in association with Th1-skewing, cytotoxic T cell activation, and reduced T cell exhaustion\nAuthors: Hsu, L.-Y.; Rosenbaum, J. T.; Verner, E.; Jones, W. B.; Hill, C. M.; Janc, J. W.; Buggy, J. J.; Ding, N.; Reneau, J. C.; Khodadoust, M. S.; Kim, Y. H.; Wilcox, R. A.; Miller, R. A.\nScore: 7.6, Published: 2023-07-06 DOI: 10.1101/2023.07.05.547822\nITK is a tyrosine kinase expressed predominantly by T lymphocytes. In mice, selective knock-out of the ITK gene produces Th1 skewing of T helper cell differentiation. We synthesized a covalent ITK inhibitor, soquelitinib, that binds ITK with greater than 100-fold selectivity compared to binding to resting lymphocyte kinase (RLK). In vitro studies with normal or malignant T cells demonstrated that soquelitinib suppresses Th2 cytokine production preferentially with relative sparing of Th1 cytokines. Soquelitinib inhibits the in vivo growth of several syngeneic murine tumors including those that do not express ITK. Treatment with soquelitinib leads to increased tumor infiltration of normal CD8+ cells that possess enhanced T effector function. Soquelitinib inhibited expression of T cell exhaustion markers and was able to restore T effector function to exhausted cells. Pharmacologic selective ITK inhibition may represent a novel approach to cancer immunotherapy.\nFully Automated Sequential Immunofluorescence (seqIF) for Hyperplex Spatial Proteomics\nAuthors: Rivest, F.; Eroglu, D.; Pelz, B.; Kowal, J.; Kehren, A.; Procopio, M. G.; Bordignon, P.; Peres, E.; Ammann, M.; Dorel, E.; Scalmazzi, S.; Bruno, L.; Ruegg, M.; Campargue, G.; Casqueiro, G.; Arn, L.; Fischer, J.; Brajkovic, S.; Joris, P.; Cassano, M.; Dupouy, D.\nScore: 7.1, Published: 2023-07-10 DOI: 10.1101/2023.07.07.548135\nTissues are complex environments where different cell types are in constant interaction with each other and with non-cellular components. Preserving the spatial context during proteomics analyses of tissue samples has become an important objective for different applications, one of the most important being the investigation of the tumor microenvironment. Here, we describe a multiplexed protein biomarker detection method on the COMET instrument, coined sequential ImmunoFluorescence (seqIF). The fully automated method uses successive applications of antibody incubation and elution, and in-situ imaging enabled by an integrated microscope and a microfluidic chip that provides optimized optical access to the sample. We show seqIF data on different sample types such as tumor and healthy tissue, including 40-plex on a single tissue section that is obtained in less than 24 hours, using off-the-shelf antibodies. We also present extensive characterization of the developed method, including elution efficiency, epitope stability, repeatability and reproducibility, signal uniformity, and dynamic range, in addition to marker and panel optimization strategies. The streamlined workflow using off-the-shelf antibodies, data quality enabling quantitative downstream analysis, and ease of reaching hyperplex levels make seqIF suitable for immune-oncology research and other disciplines requiring spatial analysis, paving the way for its adoption in clinical settings.\nMicroenvironmental Stiffness Induces Metabolic Reprogramming in Glioblastoma\nAuthors: Sohrabi, A.; Lefebvre, A. E. Y. T.; Harrison, M. J.; Condro, M. C.; Sanazzaro, T. M.; Safarians, G.; Solomon, I.; Bastola, S.; Kordbacheh, S.; Toh, N.; Kornblum, H. I.; Digman, M. A.; Seidlits, S. K.\nScore: 7.6, Published: 2023-07-03 DOI: 10.1101/2023.07.03.547558\nThe mechanical properties of solid tumors influence tumor cell phenotype and ability to invade into surrounding tissues. Using bioengineered scaffolds to provide a matrix microenvironment for patient-derived glioblastoma (GBM) spheroids, this study demonstrates that a soft, brain-like matrix induces GBM cells to shift to a glycolysis-weighted metabolic state which supports invasive behavior. We first show that orthotopic murine GBM tumors are stiffer than peri-tumoral brain tissues, but tumor stiffness is heterogenous where tumor edges are softer than the tumor core. Then, we developed three-dimensional scaffolds with {micro}-compressive moduli resembling either stiffer, tumor core or softer, peri-tumoral brain tissue. We demonstrate that the softer matrix microenvironment induces a shift in GBM cell metabolism toward glycolysis which manifests in lower proliferation rate and increased migration activities. Finally, we show that these mechanical cues are transduced from the matrix via CD44 and integrin receptors to induce metabolic and phenotypic changes in cancer cells.\nMethNet: a robust approach to identify regulatory hubs and their distal targets in cancer\nAuthors: Sakellaropoulos, T.; Do, C.; Jiang, G.; Cova, G.; Meyn, P.; Dimartino, D.; Ramaswami, S.; Heguy, A.; Tsirigos, A.; Skok, J. A.\nScore: 5.9, Published: 2023-07-10 DOI: 10.1101/2023.07.07.548142\nAberrations in the capacity of DNA/chromatin modifiers and transcription factors to bind non-coding regions can lead to changes in gene regulation and impact disease phenotypes. However, identifying distal regulatory elements and connecting them with their target genes remains challenging. Here, we present MethNet, a pipeline that integrates large-scale DNA methylation and gene expression data across multiple cancers, to uncover novel cis regulatory elements (CREs) in a 1Mb region around every promoter in the genome. MethNet identifies clusters of highly ranked CREs, referred to as \"hubs\", which contribute to the regulation of multiple genes and significantly affect patient survival. Promoter-capture Hi-C confirmed that highly ranked associations involve physical interactions between CREs and their gene targets, and CRISPRi based scRNA Perturb-seq validated the functional impact of CREs. Thus, MethNet-identified CREs represent a valuable resource for unraveling complex mechanisms underlying gene expression, and for prioritizing the verification of predicted non-coding disease hotspots.\nSomatic mutation of Afadin leads to anchorage independent survival and metastatic growth of breast cancer through alpha E-catenin dependent destabilization of the adherens junction\nAuthors: Ratze, M. A.; Enserink, L. N.; Ishiyama, N.; Veldman, C.; Nijman, I. J.; Bernards, R. R.; van Diest, P. J.; Christgen, M.; Derksen, P. W.\nScore: 6.0, Published: 2023-07-05 DOI: 10.1101/2023.07.04.547654\nLoss of E-cadherin (CDH1) and the adherens junction (AJ) drive development and progression of invasive lobular breast cancer (ILC). However, approximately 40% retain wild type CDH1 alleles, indicating that modulation of other genes attenuates the AJ during ILC etiology. To identify alternative drivers, we performed targeted sequencing in CDH1 wild type samples, based on a defined set of 100 AJ, tight junction, and desmosome genes we designated as the Adhesome. In 146 ILC samples, we identified 62 cases (43%) with wild type CDH1 alleles in which we detected a total of 284 mutations in 36 Adhesome genes. After selection based on occurrence and potential loss of function, we identified an inactivating frameshift mutation in Afadin (AFDN; p.Lys630fs). Functional studies in E-cadherin-expressing breast cancer cells showed that Afadin knockout leads to immature AJs, and a non-cohesive phenotype accompanied by actomyosin dependent anoikis resistance, which are classical ILC hallmarks. Afadin reconstitutions show that F-actin organization critically depends on the E-catenin binding CC domain. Afadin loss in intraductal xenograft mouse breast cancer models leads to ILC-type morphologies and overt lung metastases. AFDN truncate reconstitutions revealed that deletion of the C-terminal E-catenin binding CC domain is sufficient to drive metastatic ILC. In conclusion, we identified and functionally coupled a somatic frameshift AFDN mutation in breast cancer to destabilization the epithelial AJ and the development of ILC hallmarks such as actomyosin-dependent anoikis resistance and single cell invasion. As such, Afadin represents a candidate tumor suppressor for E-cadherin-positive ILC development and progression.\n",
  "wordCount" : "2437",
  "inLanguage": "en",
  "datePublished": "2023-07-12T10:41:10Z",
  "dateModified": "2023-07-12T10:41:10Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on July 12, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.06.547924">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.06.547924" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.06.547924">
        <p class="paperTitle">Integrative spatial analysis reveals a multi-layered organization of glioblastoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.06.547924" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.06.547924" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Greenwald, A. C.; Darnell, N. G.; Hoefflin, R.; Simkin, D.; Gonzalez-Castro, L. N.; Mount, C.; Hirsch, D.; Nomura, M.; Talpir, T.; Kedmi, M.; Goliand, I.; Medici, G.; Li, B.; Keren-Shaul, H.; Weller, M.; Addadi, Y.; Neidert, M. C.; Suva, M. L.; Tirosh, I.</p>
        <p class="info">Score: 74.7, Published: 2023-07-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.06.547924' target='https://doi.org/10.1101/2023.07.06.547924'> 10.1101/2023.07.06.547924</a></p>
        <p class="abstract">Glioma contains malignant cells in diverse states. Hypoxic regions are associated with a unique histology of pseudopalisading cells, while other regions appear to have limited histological organization, reflecting the diffuse nature of glioma cells. Here, we combine spatial transcriptomics with spatial proteomics and novel computational approaches to define glioma cellular states at high granularity and uncover their organization. We find three prominent modes of cellular organization. First, cells in any given state tend to be spatially clustered, such that tumors are composed of small local environments that are each typically enriched with one major cellular state. Second, specific pairs of states preferentially reside in proximity across multiple scales. Despite the unique composition of each tumor, this pairing of states remains largely consistent across tumors. Third, the pairwise interactions that we detect collectively define a global architecture composed of five layers. Hypoxia appears to drive this 5-layered organization, as it is both associated with unique states of surrounding cells and with a long-range organization that extends from the hypoxic core to the infiltrative edge of the tumor. Accordingly, tumor regions distant from any hypoxic foci and tumors that lack hypoxia such as IDH-mutant glioma are less organized. In summary, we provide a conceptual framework for the organization of gliomas at the resolution of cellular states and highlight the role of hypoxia as a long-range tissue organizer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.01.547359">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.01.547359" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.01.547359">
        <p class="paperTitle">Chemical alkylation of Asp12 enables mutant selective targeting of K-Ras(G12D)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.01.547359" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.01.547359" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhang, Z.; Zheng, Q.; Guiley, K. Z.; Shokat, K.</p>
        <p class="info">Score: 18.6, Published: 2023-07-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.01.547359' target='https://doi.org/10.1101/2023.07.01.547359'> 10.1101/2023.07.01.547359</a></p>
        <p class="abstract">K-Ras is the most commonly mutated oncogene in human cancer, yet direct small-molecule targeting of K-Ras mutants has been mostly unsuccessful until recently. The discovery of an allosteric pocket under Switch-II with covalent cysteine-crosslinking molecules has allowed for the development of targeted therapies that selectively engage the highly reactive acquired cysteine in the K-Ras(G12C) mutation without affecting the wild-type protein. Sotorasib and adagrasib, two advanced Switch-II Pocket inhibitors, have received FDA approval to treat K-Ras(G12C)-driven non-small cell lung cancer. However, the most frequent K-Ras mutation G12D particularly prevalent in pancreatic ductal adenocarcinoma has remained untargetable with covalent drugs due to the poor nucleophilicity of the somatic aspartate residue. Here we present a set of {beta}-lactone-based electrophiles which exploit ring strain to crosslink K-Ras(G12D) at the mutant aspartate to form stable covalent complexes in cells, effectively blocking Ras-effector interaction and downstream signaling. Structural insights from x-ray crystallography and exploitation of the differential chemoselectivity and stereoelectronic requirements for attack of the {beta}-lactone electrophile allowed development of a substituted {beta}-lactone which resisted attack by aqueous buffer but enabled rapid attack by aspartic acid-12 in K-Ras. Our results demonstrate the rational design of covalent inhibitors to target a non-catalytic carboxylic acid side chain in K-Ras(G12D) which has resisted traditional drug discovery efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547697">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547697" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547697">
        <p class="paperTitle">Robust prediction of patient outcomes with immune checkpoint blockade therapy for cancer using common clinical, pathologic, and genomic features</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547697" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547697" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chang, T.-G.; Cao, Y.; Sfreddo, H.; Dhruba, S. R.; Lee, S.-H.; Valero, C.; Yoo, S.-K.; Chowell, D.; Morris, L. G. T.; Ruppin, E.</p>
        <p class="info">Score: 14.1, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547697' target='https://doi.org/10.1101/2023.07.04.547697'> 10.1101/2023.07.04.547697</a></p>
        <p class="abstract">Despite the revolutionary impact of immune checkpoint blockade (ICB) in cancer treatment, accurately predicting patients responses remains elusive. We analyzed eight cohorts of 2881 ICB-treated patients across 18 solid tumor types, the largest dataset to date, examining diverse clinical, pathologic, and genomic features. We developed the LOgistic Regression-based Immunotherapy-response Score (LORIS) using a transparent, compact 6-feature logistic regression model. LORIS outperforms previous signatures in ICB response prediction and can identify responsive patients, even those with low tumor mutational burden or tumor PD-L1 expression. Importantly, LORIS consistently predicts both objective responses and short-term and long-term survival across most cancer types. Moreover, LORIS showcases a near-monotonic relationship with ICB response probability and patient survival, enabling more precise patient stratification across the board. As our method is accurate, interpretable, and only utilizes a few readily measurable features, we anticipate it will help improve clinical decision-making practices in precision medicine to maximize patient benefit.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2021.02.23.432500">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2021.02.23.432500" aria-expanded="false" aria-controls="flush-collapse10.1101/2021.02.23.432500">
        <p class="paperTitle">Integrative analyses uncover mechanisms by which aging drives B cell lymphoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2021.02.23.432500" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2021.02.23.432500" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Castro, J. P.; Shindyapina, A. V.; Barbieri, A.; Ying, K.; Strelkova, O. S.; Paulo, J. A.; Tyshkovskiy, A.; Meinl, R.; Kerepesi, C.; Petrashen, A. P.; Mariotti, M.; Meer, M.; Hu, Y.; Karamyshev, A.; Losyev, G.; Indzhykulian, A. A.; Gygi, S. P.; Sedivy, J. M.; Manis, J. P.; Gladyshev, V. N.</p>
        <p class="info">Score: 25.9, Published: 2023-07-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2021.02.23.432500' target='https://doi.org/10.1101/2021.02.23.432500'> 10.1101/2021.02.23.432500</a></p>
        <p class="abstract">While cancer is an age-related disease, many cancer studies utilize younger animal models. Here, we uncover how a cancer, B-cell lymphoma, develops as a consequence of a naturally aged system. We show that this malignancy is associated with increased cell size, splenomegaly, and a newly discovered age-associated clonal B-cell (ACBC) population. Driven by exogenous c-Myc activation, hypermethylated promoters and somatic mutations, ACBC cells clonally expand independent of germinal centers (IgM&#43;) and show increased biological age and hypomethylation in partially methylated domains related to mitotic solo-CpGs. Epigenetic changes in transformed mouse B cells are enriched for changes observed in human B-cell lymphomas. Mechanistically, the data suggest that cancerous ACBC cells originate from age-associated B cells, in part involving CD22 protein signaling fostered by the aging microenvironment. Transplantation assays demonstrate that ACBC evolve to become self-sufficient and support malignancy when transferred into young recipients. Inhibition of mTOR or c-Myc in old mice attenuates premalignant changes in B cells during aging and emerges as a therapeutic strategy to delay the onset of age-related lymphoma. Together, we show how aging contributes to B-cell lymphoma through a previously unrecognized mechanism involving cell-intrinsic changes and the aged microenvironment, characterize a model that captures the origin and progression of spontaneous cancer during aging and identify candidate interventions against age-associated lymphoma.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547626">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547626" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547626">
        <p class="paperTitle">Machine-learning analysis of factors that shape cancer aneuploidy landscapes reveals an important role for negative selection</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547626" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547626" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jubran, J.; Slutsky, R.; Rozenblum, N.; Rokach, L.; Ben-David, U.; Yeger-Lotem, E.</p>
        <p class="info">Score: 8.8, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547626' target='https://doi.org/10.1101/2023.07.05.547626'> 10.1101/2023.07.05.547626</a></p>
        <p class="abstract">Aneuploidy, an abnormal number of chromosomes within a cell, is considered a hallmark of cancer. Patterns of aneuploidy differ across cancers, yet are similar in cancers affecting closely-related tissues. The selection pressures underlying aneuploidy patterns are not fully understood, hindering our understanding of cancer development and progression. Here, we applied interpretable machine learning (ML) methods to study tissue-selective aneuploidy patterns. We defined 20 types of features of normal and cancer tissues, and used them to model gains and losses of chromosome-arms in 24 cancer types. In order to reveal the factors that shape the tissue-specific cancer aneuploidy landscapes, we interpreted the ML models by estimating the relative contribution of each feature to the models. While confirming known drivers of positive selection, our quantitative analysis highlighted the importance of negative selection for shaping the aneuploidy landscapes of human cancer. Tumor-suppressor gene density was a better predictor of gain patterns than oncogene density, and vice-versa for loss patterns. We identified the contribution of tissue-selective features and demonstrated them experimentally for chr13q gain in colon cancer. In line with an important role for negative selection in shaping the aneuploidy landscapes, we found compensation by paralogs to be a top predictor of chromosome-arm loss prevalence, and demonstrated this relationship for one such paralog interaction. Similar factors were found to shape aneuploidy patterns in human cancer cell lines, demonstrating their relevance for aneuploidy research. Overall, our quantitative, interpretable ML models improve the understanding of the genomic properties that shape cancer aneuploidy landscapes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.05.547822">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.05.547822" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.05.547822">
        <p class="paperTitle">Selective inhibition of interleukin-2 inducible T cell kinase (ITK) enhances anti-tumor immunity in association with Th1-skewing, cytotoxic T cell activation, and reduced T cell exhaustion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.05.547822" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.05.547822" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hsu, L.-Y.; Rosenbaum, J. T.; Verner, E.; Jones, W. B.; Hill, C. M.; Janc, J. W.; Buggy, J. J.; Ding, N.; Reneau, J. C.; Khodadoust, M. S.; Kim, Y. H.; Wilcox, R. A.; Miller, R. A.</p>
        <p class="info">Score: 7.6, Published: 2023-07-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.05.547822' target='https://doi.org/10.1101/2023.07.05.547822'> 10.1101/2023.07.05.547822</a></p>
        <p class="abstract">ITK is a tyrosine kinase expressed predominantly by T lymphocytes. In mice, selective knock-out of the ITK gene produces Th1 skewing of T helper cell differentiation. We synthesized a covalent ITK inhibitor, soquelitinib, that binds ITK with greater than 100-fold selectivity compared to binding to resting lymphocyte kinase (RLK). In vitro studies with normal or malignant T cells demonstrated that soquelitinib suppresses Th2 cytokine production preferentially with relative sparing of Th1 cytokines. Soquelitinib inhibits the in vivo growth of several syngeneic murine tumors including those that do not express ITK. Treatment with soquelitinib leads to increased tumor infiltration of normal CD8&#43; cells that possess enhanced T effector function. Soquelitinib inhibited expression of T cell exhaustion markers and was able to restore T effector function to exhausted cells. Pharmacologic selective ITK inhibition may represent a novel approach to cancer immunotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548135">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548135" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548135">
        <p class="paperTitle">Fully Automated Sequential Immunofluorescence (seqIF) for Hyperplex Spatial Proteomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548135" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548135" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rivest, F.; Eroglu, D.; Pelz, B.; Kowal, J.; Kehren, A.; Procopio, M. G.; Bordignon, P.; Peres, E.; Ammann, M.; Dorel, E.; Scalmazzi, S.; Bruno, L.; Ruegg, M.; Campargue, G.; Casqueiro, G.; Arn, L.; Fischer, J.; Brajkovic, S.; Joris, P.; Cassano, M.; Dupouy, D.</p>
        <p class="info">Score: 7.1, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548135' target='https://doi.org/10.1101/2023.07.07.548135'> 10.1101/2023.07.07.548135</a></p>
        <p class="abstract">Tissues are complex environments where different cell types are in constant interaction with each other and with non-cellular components. Preserving the spatial context during proteomics analyses of tissue samples has become an important objective for different applications, one of the most important being the investigation of the tumor microenvironment. Here, we describe a multiplexed protein biomarker detection method on the COMET instrument, coined sequential ImmunoFluorescence (seqIF). The fully automated method uses successive applications of antibody incubation and elution, and in-situ imaging enabled by an integrated microscope and a microfluidic chip that provides optimized optical access to the sample. We show seqIF data on different sample types such as tumor and healthy tissue, including 40-plex on a single tissue section that is obtained in less than 24 hours, using off-the-shelf antibodies. We also present extensive characterization of the developed method, including elution efficiency, epitope stability, repeatability and reproducibility, signal uniformity, and dynamic range, in addition to marker and panel optimization strategies. The streamlined workflow using off-the-shelf antibodies, data quality enabling quantitative downstream analysis, and ease of reaching hyperplex levels make seqIF suitable for immune-oncology research and other disciplines requiring spatial analysis, paving the way for its adoption in clinical settings.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.03.547558">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.03.547558" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.03.547558">
        <p class="paperTitle">Microenvironmental Stiffness Induces Metabolic Reprogramming in Glioblastoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.03.547558" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.03.547558" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sohrabi, A.; Lefebvre, A. E. Y. T.; Harrison, M. J.; Condro, M. C.; Sanazzaro, T. M.; Safarians, G.; Solomon, I.; Bastola, S.; Kordbacheh, S.; Toh, N.; Kornblum, H. I.; Digman, M. A.; Seidlits, S. K.</p>
        <p class="info">Score: 7.6, Published: 2023-07-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.03.547558' target='https://doi.org/10.1101/2023.07.03.547558'> 10.1101/2023.07.03.547558</a></p>
        <p class="abstract">The mechanical properties of solid tumors influence tumor cell phenotype and ability to invade into surrounding tissues. Using bioengineered scaffolds to provide a matrix microenvironment for patient-derived glioblastoma (GBM) spheroids, this study demonstrates that a soft, brain-like matrix induces GBM cells to shift to a glycolysis-weighted metabolic state which supports invasive behavior. We first show that orthotopic murine GBM tumors are stiffer than peri-tumoral brain tissues, but tumor stiffness is heterogenous where tumor edges are softer than the tumor core. Then, we developed three-dimensional scaffolds with {micro}-compressive moduli resembling either stiffer, tumor core or softer, peri-tumoral brain tissue. We demonstrate that the softer matrix microenvironment induces a shift in GBM cell metabolism toward glycolysis which manifests in lower proliferation rate and increased migration activities. Finally, we show that these mechanical cues are transduced from the matrix via CD44 and integrin receptors to induce metabolic and phenotypic changes in cancer cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.07.548142">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.07.548142" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.07.548142">
        <p class="paperTitle">MethNet: a robust approach to identify regulatory hubs and their distal targets in cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.07.548142" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.07.548142" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sakellaropoulos, T.; Do, C.; Jiang, G.; Cova, G.; Meyn, P.; Dimartino, D.; Ramaswami, S.; Heguy, A.; Tsirigos, A.; Skok, J. A.</p>
        <p class="info">Score: 5.9, Published: 2023-07-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.07.548142' target='https://doi.org/10.1101/2023.07.07.548142'> 10.1101/2023.07.07.548142</a></p>
        <p class="abstract">Aberrations in the capacity of DNA/chromatin modifiers and transcription factors to bind non-coding regions can lead to changes in gene regulation and impact disease phenotypes. However, identifying distal regulatory elements and connecting them with their target genes remains challenging. Here, we present MethNet, a pipeline that integrates large-scale DNA methylation and gene expression data across multiple cancers, to uncover novel cis regulatory elements (CREs) in a 1Mb region around every promoter in the genome. MethNet identifies clusters of highly ranked CREs, referred to as &#34;hubs&#34;, which contribute to the regulation of multiple genes and significantly affect patient survival. Promoter-capture Hi-C confirmed that highly ranked associations involve physical interactions between CREs and their gene targets, and CRISPRi based scRNA Perturb-seq validated the functional impact of CREs. Thus, MethNet-identified CREs represent a valuable resource for unraveling complex mechanisms underlying gene expression, and for prioritizing the verification of predicted non-coding disease hotspots.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.04.547654">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.04.547654" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.04.547654">
        <p class="paperTitle">Somatic mutation of Afadin leads to anchorage independent survival and metastatic growth of breast cancer through alpha E-catenin dependent destabilization of the adherens junction</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.04.547654" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.04.547654" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ratze, M. A.; Enserink, L. N.; Ishiyama, N.; Veldman, C.; Nijman, I. J.; Bernards, R. R.; van Diest, P. J.; Christgen, M.; Derksen, P. W.</p>
        <p class="info">Score: 6.0, Published: 2023-07-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.04.547654' target='https://doi.org/10.1101/2023.07.04.547654'> 10.1101/2023.07.04.547654</a></p>
        <p class="abstract">Loss of E-cadherin (CDH1) and the adherens junction (AJ) drive development and progression of invasive lobular breast cancer (ILC). However, approximately 40% retain wild type CDH1 alleles, indicating that modulation of other genes attenuates the AJ during ILC etiology. To identify alternative drivers, we performed targeted sequencing in CDH1 wild type samples, based on a defined set of 100 AJ, tight junction, and desmosome genes we designated as the  Adhesome. In 146 ILC samples, we identified 62 cases (43%) with wild type CDH1 alleles in which we detected a total of 284 mutations in 36 Adhesome genes. After selection based on occurrence and potential loss of function, we identified an inactivating frameshift mutation in Afadin (AFDN; p.Lys630fs).

Functional studies in E-cadherin-expressing breast cancer cells showed that Afadin knockout leads to immature AJs, and a non-cohesive phenotype accompanied by actomyosin dependent anoikis resistance, which are classical ILC hallmarks. Afadin reconstitutions show that F-actin organization critically depends on the E-catenin binding CC domain. Afadin loss in intraductal xenograft mouse breast cancer models leads to ILC-type morphologies and overt lung metastases. AFDN truncate reconstitutions revealed that deletion of the C-terminal E-catenin binding CC domain is sufficient to drive metastatic ILC. In conclusion, we identified and functionally coupled a somatic frameshift AFDN mutation in breast cancer to destabilization the epithelial AJ and the development of ILC hallmarks such as actomyosin-dependent anoikis resistance and single cell invasion. As such, Afadin represents a candidate tumor suppressor for E-cadherin-positive ILC development and progression.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
