<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Fractal Dimension and Lacunarity Measures of Glioma Subcomponents Provide a Quantitative Platform Discriminative of IDH Status: A Radiogenomics Approach in Gliomas
Authors: Yadav, N.; Mohanty, A.; Aswin, V.; Mishrra, N.; Tiwari, V.
Score: 7.3, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573519
Background: The presence of structural and geometric variations within gliomas, even among those with similar histologic grades, reflects the phenotypic heterogeneity unique to a genetic and epigenetic landscape. Whole glioma mass comprises of various subcomponents identified on MR imaging: enhancing, nonenhancing, necrosis, and edema fractions in varied fractions across patients.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Fractal Dimension and Lacunarity Measures of Glioma Subcomponents Provide a Quantitative Platform Discriminative of IDH Status: A Radiogenomics Approach in Gliomas
Authors: Yadav, N.; Mohanty, A.; Aswin, V.; Mishrra, N.; Tiwari, V.
Score: 7.3, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573519
Background: The presence of structural and geometric variations within gliomas, even among those with similar histologic grades, reflects the phenotypic heterogeneity unique to a genetic and epigenetic landscape. Whole glioma mass comprises of various subcomponents identified on MR imaging: enhancing, nonenhancing, necrosis, and edema fractions in varied fractions across patients." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-31T10:38:19+00:00" />
<meta property="article:modified_time" content="2023-12-31T10:38:19+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Fractal Dimension and Lacunarity Measures of Glioma Subcomponents Provide a Quantitative Platform Discriminative of IDH Status: A Radiogenomics Approach in Gliomas
Authors: Yadav, N.; Mohanty, A.; Aswin, V.; Mishrra, N.; Tiwari, V.
Score: 7.3, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573519
Background: The presence of structural and geometric variations within gliomas, even among those with similar histologic grades, reflects the phenotypic heterogeneity unique to a genetic and epigenetic landscape. Whole glioma mass comprises of various subcomponents identified on MR imaging: enhancing, nonenhancing, necrosis, and edema fractions in varied fractions across patients."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Fractal Dimension and Lacunarity Measures of Glioma Subcomponents Provide a Quantitative Platform Discriminative of IDH Status: A Radiogenomics Approach in Gliomas\nAuthors: Yadav, N.; Mohanty, A.; Aswin, V.; Mishrra, N.; Tiwari, V.\nScore: 7.3, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573519\nBackground: The presence of structural and geometric variations within gliomas, even among those with similar histologic grades, reflects the phenotypic heterogeneity unique to a genetic and epigenetic landscape. Whole glioma mass comprises of various subcomponents identified on MR imaging: enhancing, nonenhancing, necrosis, and edema fractions in varied fractions across patients.",
  "keywords": [
    
  ],
  "articleBody": " Fractal Dimension and Lacunarity Measures of Glioma Subcomponents Provide a Quantitative Platform Discriminative of IDH Status: A Radiogenomics Approach in Gliomas\nAuthors: Yadav, N.; Mohanty, A.; Aswin, V.; Mishrra, N.; Tiwari, V.\nScore: 7.3, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573519\nBackground: The presence of structural and geometric variations within gliomas, even among those with similar histologic grades, reflects the phenotypic heterogeneity unique to a genetic and epigenetic landscape. Whole glioma mass comprises of various subcomponents identified on MR imaging: enhancing, nonenhancing, necrosis, and edema fractions in varied fractions across patients. The geometry of whole tumor mass and the glioma subcomponents is highly irregular. Thereby, traditional Euclidean geometry is not suitable for quantifying the geometric dimensions. Here, we employ non-Euclidean geometric measurements: Fractal Dimension and lacunarity of the glioma subcomponents as a discriminator of IDH and MGMT status of gliomas. Methods: Fractality and Lacunarity measurements were obtained using the tumor masks generated for enhancing, nonenhancing, and edema subcomponents from the preoperative T1, T1c, and T2-Flair MRI. Fractality and lacunarity measures of each subcomponent were evaluated between IDH mutant and wildtype gliomas. The fractality and lacunarity measures in IDH mutant and wildtype gliomas were further stratified for MGMT methylated and unmethylated gliomas. The fractality and lacunarities were trained and tested using supervised ML modeling as discriminators of IDH and MGMT status. Further, Cox Hazard estimations and the Kaplan-Meir investigations were performed to evaluate the impact of fractality and lacunarity measures of glioma subcomponents on the overall survival of the patients. Results: IDH wildtype gliomas had ~2-fold higher fractality for the enhancing subcomponent compared to IDH mutant enhancing subcomponent, while IDH mutant gliomas showed higher fractality for the nonenhancing subcomponent. Furthermore, the edema subcomponent did not differ for fractality or lacunarity measures between IDH mutant and wildtype gliomas. Fractal or lacunarity measures for either of the three subcomponents do not vary across MGMT methylated and unmethylated status with a given IDH mutant or wildtype gliomas. A combination of fractal measures of the enhancing and nonenhancing subcomponents together provided highly accurate and sensitive discrimination of IDH status using the supervised ML models. Moreover, fractality measure greater or equal 0.69 for the enhancing subcomponent was associated with shortened patient survival: a fractal dimension value corresponding to that of IDH wild type gliomas. However, fractality and lacunarity estimates were not sensitive for discrimination of MGMT status. Conclusion: Glioma structural heterogeneity measured as fractality and lacunarity using routine structural MRI measurements provide a noninvasive quantitative platform definitive of the molecular subtype of gliomas: IDH mutant vs. wildtype. Establishing fractality and/or lacunarity quantities as signatures of prognostic molecular events provides an avenue to bypass the need of biopsy/surgical interventions for decision-making, determining the molecular subtypes and overall clinical management of gliomas.\nPrecise single-cell transcriptomic mapping of normal and leukemic cell states reveals unconventional lineage priming in acute myeloid leukemia\nAuthors: Zeng, A. G. X.; Iacobucci, I.; Shah, S.; Mitchell, A.; Wong, G.; Bansal, S.; Gao, Q.; Kim, H.; Kennedy, J. A.; Minden, M. D.; Haferlach, T.; Mullighan, C. G.; Dick, J. E.\nScore: 6.8, Published: 2023-12-27 DOI: 10.1101/2023.12.26.573390\nInitial classification of acute leukemia involves the assignment of blasts to cell states within the hematopoietic hierarchy based on morphological and immunophenotypic features. Yet, these traditional classification approaches lack precision, especially at the level of immature blasts. Single-cell RNA-sequencing (scRNA-seq) enables precise determination of cell state using thousands of markers, thus providing an opportunity to re-examine present-day classification schemes of acute leukemia. Here, we developed a detailed reference map of human bone marrow hematopoiesis from 263,519 single-cell transcriptomes spanning 55 cellular states. Cell state annotations were benchmarked against purified cell populations, and in-depth characterization of gene expression programs underlying hematopoietic differentiation was undertaken. Projection of single-cell transcriptomes from 175 samples spanning acute myeloid leukemia (AML), mixed phenotype acute leukemia (MPAL), and acute erythroid leukemia (AEL) revealed 11 subtypes involving distinct stages of hematopoietic differentiation. These included AML subtypes with notable lymphoid or erythroid lineage priming, challenging traditional diagnostic boundaries between AML, MPAL, and AEL. Quantification of lineage priming in bulk patient cohorts revealed specific genetic alterations associated with this unconventional lineage priming. Integration of transcriptional and genetic information at the single-cell level revealed how genetic subclones can induce lineage restriction, differentiation blocks, or expansion of mature myeloid cells. Furthermore, we demonstrate that distinct cellular hierarchies can co-exist within individual patients, providing insight into AML evolution in response to varying selection pressures. Together, precise mapping of hematopoietic cell states can serve as a foundation for refining disease classification in acute leukemia and understanding response or resistance to emerging therapies.\nKMT2A oncoproteins induce epigenetic resistance to targeted therapies\nAuthors: Janssens, D. H.; Duran, M.; Otto, D. J.; Kirkey, D.; Yi, J. S.; Meshinchi, S.; Sarthy, J. F.; Ahmad, K.; Henikoff, S.\nScore: 3.6, Published: 2023-12-30 DOI: 10.1101/2023.12.29.573681\nChromosomal translocations involving the Lysine-Methyl-Tansferase-2A (KMT2A) locus generate potent oncogenes that cause highly aggressive acute leukemias. KMT2A and the most frequent translocation partners encode proteins that interact with DNA to regulate developmental gene expression. KMT2A-oncogenic fusion proteins (oncoproteins) contribute to the epigenetic mechanisms that allow KMT2A-rearranged leukemias to evade targeted therapies. By profiling the oncoprotein-target sites of 34 KMT2A-rearranged leukemia samples, we find that the genomic enrichment of oncoprotein binding is highly variable between samples. At high levels of expression, the oncoproteins preferentially activate either the lymphoid or myeloid lineage program depending on the fusion partner. These fusion-partner-dependent binding sites correspond to the frequencies of each mutation in acute lymphoid leukemia versus acute myeloid leukemia. By profiling a sample that underwent a lymphoid-to-myeloid lineage switching event in response to lymphoid-directed treatment, we find the global oncoprotein levels are reduced and the oncoprotein-target gene network changes. At lower levels of expression, the oncoprotein shifts to a non-canonical regulatory program that favors the myeloid lineage, and in a subset of resistant patients, the Menin inhibitor Revumenib induces a similar response. The dynamic shifts in KMT2A oncoproteins we describe likely contribute to epigenetic resistance of KMT2A-rearranged leukemias to targeted therapies.\nImage-Based Assessment of Natural Killer Cell Activity against Glioblastoma Stem Cells\nAuthors: Du, Y.; Metcalfe, S.; Akunapuram, S.; Richardson, A. M.; Cohen-Gadol, A. A.; Shen, J.\nScore: 3.5, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573629\nGlioblastoma (GBM) poses a significant challenge in oncology and stands as the most aggressive form of brain cancer. A primary contributor to its relentless nature is the stem-like cancer cells, called glioblastoma stem cells (GSCs). GSCs have the capacity for self-renewal and tumorigenesis, leading to frequent GBM recurrences and complicating treatment modalities. While natural killer (NK) cells exhibit potential in targeting and eliminating stem-like cancer cells, their efficacy within the GBM microenvironment is limited due to constrained infiltration and function. To address this limitation, novel investigations focusing on boosting NK cell activity against GSCs are imperative. This study presents two streamlined image-based assays assessing NK cell migration and cytotoxicity towards GSCs. It details protocols and explores the strengths and limitations of these methods. These assays could aid in identifying novel targets to enhance NK cell activity towards GSCs, facilitating the development of NK cell-based immunotherapy for improved GBM treatment.\nThe ribosomal protein L22 binds the MDM4 pre-mRNA and promotes exon skipping to activate p53 upon nucleolar stress\nAuthors: Jansen, J.; Bohnsack, K. E.; Boehlken-Fascher, S.; Bohnsack, M. T.; Dobbelstein, M.\nScore: 3.2, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573614\nThe tumor suppressor p53, along with its antagonists MDM2 and MDM4, represents a central integrator of stress signaling. While DNA damage is the most widely explored trigger of a p53 response, stress can also arise from dysbalanced assembly of ribosomes in nucleoli. Deletions of the gene encoding the ribosomal protein L22 (RPL22; eL22) correlate with the presence of full-length MDM4 mRNA in human cancer, but the mechanistic basis for this phenomenon was hitherto unknown. Here we show that L22, under conditions of ribosomal and nucleolar stress, promotes the skipping of exon 6 within the MDM4 pre-mRNA. Upon L22 depletion, far more full-length MDM4 is maintained despite treatment with nucleolar stressors, leading to diminished p53 activity and enhanced proliferation. Mechanistically, L22 binds to specific RNA elements within intron 6 of MDM4 that correspond to a stem-loop consensus, leading to the skipping of exon 6. This intronic RNA overlaps with the region responsible for splice regulation by ZMAT3. Targeted deletion of these intronic elements largely abolishes L22-mediated exon skipping and re-enables cell proliferation despite nucleolar stressors such as 5-fluorouracil. L22 also governs alternative splicing of the RPL22L1 and UBAP2L mRNAs. Thus, L22 serves as a signaling intermediate that integrates different layers of gene expression. Defects in ribosome synthesis lead to specific alternative splicing, ultimately triggering p53-mediated transcription and arresting cell proliferation.\nIdentification of nonsense-mediated decay inhibitors that alter the tumor immune landscape\nAuthors: Cook, A. L.; Sur, S.; Dobbyn, L.; Watson, E.; Cohen, J. D.; Ptak, B.; Lee, B. S.; Paul, S.; Hsiue, E.; Popoli, M.; Vogelstein, B.; Papadopoulos, N.; Bettegowda, C.; Gabrielson, K.; Zhou, S.; Kinzler, K. W.; Wyhs, N.\nScore: 3.0, Published: 2023-12-29 DOI: 10.1101/2023.12.28.573594\nDespite exciting developments in cancer immunotherapy, its broad application is limited by the paucity of targetable antigens on the tumor cell surface. As an intrinsic cellular pathway, nonsense-mediated decay (NMD) conceals neoantigens through the destruction of the RNA products from genes harboring truncating mutations. We developed and conducted a high throughput screen, based on the ratiometric analysis of transcripts, to identify critical mediators of NMD. This screen revealed disruption of kinase SMG1s phosphorylation of UPF1 as a potent disruptor of NMD. This led us to design a novel SMG1 inhibitor, KVS0001, that elevates the expression of transcripts and proteins resulting from truncating mutations in vivo and in vitro. Most importantly, KVS0001 concomitantly increased the presentation of immune-targetable HLA class I-associated peptides from NMD-downregulated proteins on the surface of cancer cells. KVS0001 provides new opportunities for studying NMD and the diseases in which NMD plays a role, including cancer and inherited diseases.\nTimp2 loss-of-function mutation and TIMP2 treatment in murine model of NSCLC: modulation of immunosuppression and oncogenic signaling\nAuthors: Peeney, D.; Kumar, S.; Singh, T. P.; Liu, Y.; Jensen, S.; Chowdhury, A.; Coates-Park, S.; Rich, J.; Gurung, S.; Fan, Y.; Meerzaman, D.; Stetler-Stevenson, W. G.\nScore: 2.5, Published: 2023-12-29 DOI: 10.1101/2023.12.29.573636\nMounting evidence suggests that the tissue inhibitor of metalloproteinases-2 (TIMP2) can reduce tumor burden and metastasis. However, the demonstration of such anti-tumor activity and associated mechanisms using in vivo tumor models is lacking. The effects of a Timp2 functional mutation and administration of recombinant TIMP2 were examined in both orthotopic and heterotopic murine models of lung cancer using C57Bl/6 syngeneic Lewis Lung 2-luciferase 2 cells (LL2-luc2) cells. Mice harboring a functional mutation of TIMP2 (mT2) display markedly increased primary lung tumor growth, increased mortality, enriched vasculature, and enhanced infiltration of pro-tumorigenic, immunosuppressive myeloid cells. Treatment with recombinant TIMP2 reduced primary tumor growth in both mutant and wild-type (wt) mice. Comparison of transcriptional profiles of lung tissues from tumor-free, wt versus mT2 mice reveals only minor changes. However, lung tumor-bearing mice of both genotypes demonstrate significant genotype-dependent changes in gene expression following treatment with TIMP. In tumor-bearing wt mice, TIMP2 treatment reduced the expression of upstream oncogenic mediators, whereas treatment of mT2 mice resulted in an immunomodulatory phenotype. A heterotopic subcutaneous model generating metastatic pulmonary tumors demonstrated that daily administration of recombinant TIMP2 significantly downregulates the expression of heat shock proteins, suggesting a reduction of cell-stress responses. In summary, we describe how TIMP2 exerts novel, anti-tumor effects in a murine model of lung cancer and that rTIMP2 treatment supports a normalizing effect on the tumor microenvironment. Our findings show that TIMP2 treatment demonstrates significant potential as an adjuvant in the treatment of NSCLC.\nSpatial dissection of cellular heterogeneity and functional crosstalk in the tumor microenvironments of gastric cancers\nAuthors: Kim, T.; Lee, S. H.; Lee, D.; Choi, J.; Oh, H. J.; Ham, I.-H.; Ryu, D.; Lee, S.-Y.; Han, D.-J.; Kim, S.; Lee, S.; Hur, H.\nScore: 2.5, Published: 2023-12-25 DOI: 10.1101/2023.12.24.573275\nSpatial transcriptomics offers unprecedented challenges in characterizing tumor ecosystems, encompassing tumor cells and their surrounding tumor microenvironments (TME) along with cellular interactions therein. Here, we conducted Visium-based spatial transcriptomics on nine primary gastric cancers (GC) to translate the spatial organization of GC ecosystems into the functional landscape of cellular crosstalk across malignant, stromal and immune cells. We identified three distinct GC subtypes of 3 immunogenic, 3 epithelial and 3 stromal GCs with elevated infiltration of immune, malignant cells and fibroblasts reflecting the heterogeneity of the cell types and their interactions in GC TME. GC spatial architecture was further delineated into 6 regional compartments with varying degree of TME infiltration, among which the fibroblast-enriched TME was associated with the upregulation of epithelial-to-mesenchymal transformation (EMT) and immune response in malignant and immune cells, respectively. Cell type-specific transcriptional dynamics representing cellular crosstalk between TME cells were further identified, e.g., the infiltration of malignant and endothelial cells promote the cellular proliferations of TME cells whereas the fibroblasts and immune cells are associated with pro-cancer and anti-cancer immunity, respectively. Ligand-receptor analysis further revealed that CCL2-expressing fibroblasts promote the pro-cancer signaling of immune cells including JAK-STAT3 signaling and inflammatory response in tumor-infiltration macrophages. The CCL2+fibroblasts and macrophages with activated JAK-STAT3 signals are co-localized in spatial GC architecture and their co-abundance was associated with unfavorable clinical outcomes. Taken together, GC spatial transcriptomes can be regionally and functionally delineated into functional cellular crosstalk involving multiple cell types with potential clinical relevance, particularly the interaction between CCL2+ fibroblasts and JAK-STAT3+macrophages contributing to cancer-favoring immune contexture in GC TME.\nOrgan specific microenvironmental MR1 expression in cutaneous melanoma\nAuthors: Gordon, P. B.; So, W. Y.; Azubuike, U. F.; Johnson, B.; Cicala, J.; Sturgess, V.; Wong, C.; Bishop, K.; Bresciani, E.; Sood, R.; Ganesan, S.; Tanner, K.\nScore: 2.2, Published: 2023-12-28 DOI: 10.1101/2023.12.28.573554\nThe microenvironment is an important regulator of intertumoral trafficking and activity of immune cells. Understanding how the immune system can be tailored to maintain anti-tumor killing responses in metastatic disease remains an important goal. Thus, immune mediated eradication of metastasis requires the consideration of organ specific microenvironmental cues. Using a xenograft model of melanoma metastasis in adult zebrafish, we perturbed the dynamic balance between the infiltrating immune cells in the metastatic setting using a suite of different transgenic zebrafish. We employed intravital imaging coupled with metabolism imaging (FLIM) to visualize and map the organ specific metabolism with near simultaneity in multiple metastatic lesions. Of all the MHC complexes examined for brain and skeletal metastases, we determined that there is an organ specific expression of mhc1uba (human ortholog, MR1) for both the melanoma cells and the resident and infiltrating immune cells. Specifically, immune clusters did not express mhc1uba in brain metastatic lesions in immune competent fish. Finally, the differential immune response drove organ specific metabolism where tumor glycolysis was increased in brain metastases compared to skeletal and parental lines as measured using fluorescence lifetime imaging microscopy (FLIM). As MR1 belongs to the MHC class I molecules and is a target of immunotherapeutic drugs, we believe that our data presents an opportunity to understand the relationship between organ specific tumor metabolism and drug efficacy in the metastatic setting.\nTissue of origin detection for cancer tumor using low-depth cfDNA samples through combination of tumor-specific methylation atlas and genome-wide methylation density in graph convolutional neural networks\nAuthors: Nguyen, T. H.; Doan, N. N. T.; Tran, T. H.; Huynh, L. A. K.; Doan, P. L.; Nguyen, T. H. H.; Nguyen, V. T. C.; Nguyen, G. T. H.; Nguyen, H.-N.; Giang, H.; Tran, L. S.; Phan, M. D.\nScore: 1.5, Published: 2023-12-25 DOI: 10.1101/2023.12.24.573277\nBackgroundCell free DNA (cfDNA)-based assays hold great potential in detecting early cancer signals yet determining the tissue-of-origin (TOO) for cancer signals remains a challenging task. Here, we investigated the contribution of a methylation atlas to TOO detection in low depth cfDNA samples. MethodsWe constructed a tumor-specific methylation atlas (TSMA) using whole-genome bisulfite sequencing (WGBS) data from five types of tumor tissues (breast, colorectal, gastric, liver and lung cancer) and paired white blood cells (WBC). TSMA was used with a non-negative least square matrix factorization (NNLS) deconvolution algorithm to identify the abundance of tumor tissue types in a WGBS sample. We showed that TSMA worked well with tumor tissue but struggled with cfDNA samples due to the overwhelming amount of WBC-derived DNA. To construct a model for TOO, we adopted the multi-modal strategy and used as inputs the combination of deconvolution scores from TSMA with other features of cfDNA. ResultsOur final model comprised of a graph convolutional neural network using deconvolution scores and genome-wide methylation density features, which achieved an accuracy of 69% in a held-out validation dataset of 239 low-depth cfDNA samples. ConclusionsIn conclusion, we have demonstrated that our TSMA in combination with other cfDNA features can improve TOO detection in low-depth cfDNA samples.\n",
  "wordCount" : "2777",
  "inLanguage": "en",
  "datePublished": "2023-12-31T10:38:19Z",
  "dateModified": "2023-12-31T10:38:19Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 31, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573519">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573519" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573519">
        <p class="paperTitle">Fractal Dimension and Lacunarity Measures of Glioma Subcomponents Provide a Quantitative Platform Discriminative of IDH Status: A Radiogenomics Approach in Gliomas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573519" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573519" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yadav, N.; Mohanty, A.; Aswin, V.; Mishrra, N.; Tiwari, V.</p>
        <p class="info">Score: 7.3, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573519' target='https://doi.org/10.1101/2023.12.28.573519'> 10.1101/2023.12.28.573519</a></p>
        <p class="abstract">Background: The presence of structural and geometric variations within gliomas, even among those with similar histologic grades, reflects the phenotypic heterogeneity unique to a genetic and epigenetic landscape. Whole glioma mass comprises of various subcomponents identified on MR imaging: enhancing, nonenhancing, necrosis, and edema fractions in varied fractions across patients. The geometry of whole tumor mass and the glioma subcomponents is highly irregular. Thereby, traditional Euclidean geometry is not suitable for quantifying the geometric dimensions. Here, we employ non-Euclidean geometric measurements: Fractal Dimension and lacunarity of the glioma subcomponents as a discriminator of IDH and MGMT status of gliomas. Methods: Fractality and Lacunarity measurements were obtained using the tumor masks generated for enhancing, nonenhancing, and edema subcomponents from the preoperative T1, T1c, and T2-Flair MRI. Fractality and lacunarity measures of each subcomponent were evaluated between IDH mutant and wildtype gliomas. The fractality and lacunarity measures in IDH mutant and wildtype gliomas were further stratified for MGMT methylated and unmethylated gliomas. The fractality and lacunarities were trained and tested using supervised ML modeling as discriminators of IDH and MGMT status. Further, Cox Hazard estimations and the Kaplan-Meir investigations were performed to evaluate the impact of fractality and lacunarity measures of glioma subcomponents on the overall survival of the patients. Results: IDH wildtype gliomas had ~2-fold higher fractality for the enhancing subcomponent compared to IDH mutant enhancing subcomponent, while IDH mutant gliomas showed higher fractality for the nonenhancing subcomponent. Furthermore, the edema subcomponent did not differ for fractality or lacunarity measures between IDH mutant and wildtype gliomas. Fractal or lacunarity measures for either of the three subcomponents do not vary across MGMT methylated and unmethylated status with a given IDH mutant or wildtype gliomas. A combination of fractal measures of the enhancing and nonenhancing subcomponents together provided highly accurate and sensitive discrimination of IDH status using the supervised ML models. Moreover, fractality measure greater or equal 0.69 for the enhancing subcomponent was associated with shortened patient survival: a fractal dimension value corresponding to that of IDH wild type gliomas. However, fractality and lacunarity estimates were not sensitive for discrimination of MGMT status. Conclusion: Glioma structural heterogeneity measured as fractality and lacunarity using routine structural MRI measurements provide a noninvasive quantitative platform definitive of the molecular subtype of gliomas: IDH mutant vs. wildtype. Establishing fractality and/or lacunarity quantities as signatures of prognostic molecular events provides an avenue to bypass the need of biopsy/surgical interventions for decision-making, determining the molecular subtypes and overall clinical management of gliomas.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.26.573390">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.26.573390" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.26.573390">
        <p class="paperTitle">Precise single-cell transcriptomic mapping of normal and leukemic cell states reveals unconventional lineage priming in acute myeloid leukemia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.26.573390" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.26.573390" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zeng, A. G. X.; Iacobucci, I.; Shah, S.; Mitchell, A.; Wong, G.; Bansal, S.; Gao, Q.; Kim, H.; Kennedy, J. A.; Minden, M. D.; Haferlach, T.; Mullighan, C. G.; Dick, J. E.</p>
        <p class="info">Score: 6.8, Published: 2023-12-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.26.573390' target='https://doi.org/10.1101/2023.12.26.573390'> 10.1101/2023.12.26.573390</a></p>
        <p class="abstract">Initial classification of acute leukemia involves the assignment of blasts to cell states within the hematopoietic hierarchy based on morphological and immunophenotypic features. Yet, these traditional classification approaches lack precision, especially at the level of immature blasts. Single-cell RNA-sequencing (scRNA-seq) enables precise determination of cell state using thousands of markers, thus providing an opportunity to re-examine present-day classification schemes of acute leukemia. Here, we developed a detailed reference map of human bone marrow hematopoiesis from 263,519 single-cell transcriptomes spanning 55 cellular states. Cell state annotations were benchmarked against purified cell populations, and in-depth characterization of gene expression programs underlying hematopoietic differentiation was undertaken. Projection of single-cell transcriptomes from 175 samples spanning acute myeloid leukemia (AML), mixed phenotype acute leukemia (MPAL), and acute erythroid leukemia (AEL) revealed 11 subtypes involving distinct stages of hematopoietic differentiation. These included AML subtypes with notable lymphoid or erythroid lineage priming, challenging traditional diagnostic boundaries between AML, MPAL, and AEL. Quantification of lineage priming in bulk patient cohorts revealed specific genetic alterations associated with this unconventional lineage priming. Integration of transcriptional and genetic information at the single-cell level revealed how genetic subclones can induce lineage restriction, differentiation blocks, or expansion of mature myeloid cells. Furthermore, we demonstrate that distinct cellular hierarchies can co-exist within individual patients, providing insight into AML evolution in response to varying selection pressures. Together, precise mapping of hematopoietic cell states can serve as a foundation for refining disease classification in acute leukemia and understanding response or resistance to emerging therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573681">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573681" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573681">
        <p class="paperTitle">KMT2A oncoproteins induce epigenetic resistance to targeted therapies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573681" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573681" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Janssens, D. H.; Duran, M.; Otto, D. J.; Kirkey, D.; Yi, J. S.; Meshinchi, S.; Sarthy, J. F.; Ahmad, K.; Henikoff, S.</p>
        <p class="info">Score: 3.6, Published: 2023-12-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573681' target='https://doi.org/10.1101/2023.12.29.573681'> 10.1101/2023.12.29.573681</a></p>
        <p class="abstract">Chromosomal translocations involving the Lysine-Methyl-Tansferase-2A (KMT2A) locus generate potent oncogenes that cause highly aggressive acute leukemias. KMT2A and the most frequent translocation partners encode proteins that interact with DNA to regulate developmental gene expression. KMT2A-oncogenic fusion proteins (oncoproteins) contribute to the epigenetic mechanisms that allow KMT2A-rearranged leukemias to evade targeted therapies. By profiling the oncoprotein-target sites of 34 KMT2A-rearranged leukemia samples, we find that the genomic enrichment of oncoprotein binding is highly variable between samples. At high levels of expression, the oncoproteins preferentially activate either the lymphoid or myeloid lineage program depending on the fusion partner. These fusion-partner-dependent binding sites correspond to the frequencies of each mutation in acute lymphoid leukemia versus acute myeloid leukemia. By profiling a sample that underwent a lymphoid-to-myeloid lineage switching event in response to lymphoid-directed treatment, we find the global oncoprotein levels are reduced and the oncoprotein-target gene network changes. At lower levels of expression, the oncoprotein shifts to a non-canonical regulatory program that favors the myeloid lineage, and in a subset of resistant patients, the Menin inhibitor Revumenib induces a similar response. The dynamic shifts in KMT2A oncoproteins we describe likely contribute to epigenetic resistance of KMT2A-rearranged leukemias to targeted therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573629">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573629" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573629">
        <p class="paperTitle">Image-Based Assessment of Natural Killer Cell Activity against Glioblastoma Stem Cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573629" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573629" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Du, Y.; Metcalfe, S.; Akunapuram, S.; Richardson, A. M.; Cohen-Gadol, A. A.; Shen, J.</p>
        <p class="info">Score: 3.5, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573629' target='https://doi.org/10.1101/2023.12.29.573629'> 10.1101/2023.12.29.573629</a></p>
        <p class="abstract">Glioblastoma (GBM) poses a significant challenge in oncology and stands as the most aggressive form of brain cancer. A primary contributor to its relentless nature is the stem-like cancer cells, called glioblastoma stem cells (GSCs). GSCs have the capacity for self-renewal and tumorigenesis, leading to frequent GBM recurrences and complicating treatment modalities. While natural killer (NK) cells exhibit potential in targeting and eliminating stem-like cancer cells, their efficacy within the GBM microenvironment is limited due to constrained infiltration and function. To address this limitation, novel investigations focusing on boosting NK cell activity against GSCs are imperative. This study presents two streamlined image-based assays assessing NK cell migration and cytotoxicity towards GSCs. It details protocols and explores the strengths and limitations of these methods. These assays could aid in identifying novel targets to enhance NK cell activity towards GSCs, facilitating the development of NK cell-based immunotherapy for improved GBM treatment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573614">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573614" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573614">
        <p class="paperTitle">The ribosomal protein L22 binds the MDM4 pre-mRNA and promotes exon skipping to activate p53 upon nucleolar stress</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573614" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573614" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jansen, J.; Bohnsack, K. E.; Boehlken-Fascher, S.; Bohnsack, M. T.; Dobbelstein, M.</p>
        <p class="info">Score: 3.2, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573614' target='https://doi.org/10.1101/2023.12.29.573614'> 10.1101/2023.12.29.573614</a></p>
        <p class="abstract">The tumor suppressor p53, along with its antagonists MDM2 and MDM4, represents a central integrator of stress signaling. While DNA damage is the most widely explored trigger of a p53 response, stress can also arise from dysbalanced assembly of ribosomes in nucleoli. Deletions of the gene encoding the ribosomal protein L22 (RPL22; eL22) correlate with the presence of full-length MDM4 mRNA in human cancer, but the mechanistic basis for this phenomenon was hitherto unknown. Here we show that L22, under conditions of ribosomal and nucleolar stress, promotes the skipping of exon 6 within the MDM4 pre-mRNA. Upon L22 depletion, far more full-length MDM4 is maintained despite treatment with nucleolar stressors, leading to diminished p53 activity and enhanced proliferation. Mechanistically, L22 binds to specific RNA elements within intron 6 of MDM4 that correspond to a stem-loop consensus, leading to the skipping of exon 6. This intronic RNA overlaps with the region responsible for splice regulation by ZMAT3. Targeted deletion of these intronic elements largely abolishes L22-mediated exon skipping and re-enables cell proliferation despite nucleolar stressors such as 5-fluorouracil. L22 also governs alternative splicing of the RPL22L1 and UBAP2L mRNAs. Thus, L22 serves as a signaling intermediate that integrates different layers of gene expression. Defects in ribosome synthesis lead to specific alternative splicing, ultimately triggering p53-mediated transcription and arresting cell proliferation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573594">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573594" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573594">
        <p class="paperTitle">Identification of nonsense-mediated decay inhibitors that alter the tumor immune landscape</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573594" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573594" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Cook, A. L.; Sur, S.; Dobbyn, L.; Watson, E.; Cohen, J. D.; Ptak, B.; Lee, B. S.; Paul, S.; Hsiue, E.; Popoli, M.; Vogelstein, B.; Papadopoulos, N.; Bettegowda, C.; Gabrielson, K.; Zhou, S.; Kinzler, K. W.; Wyhs, N.</p>
        <p class="info">Score: 3.0, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573594' target='https://doi.org/10.1101/2023.12.28.573594'> 10.1101/2023.12.28.573594</a></p>
        <p class="abstract">Despite exciting developments in cancer immunotherapy, its broad application is limited by the paucity of targetable antigens on the tumor cell surface. As an intrinsic cellular pathway, nonsense-mediated decay (NMD) conceals neoantigens through the destruction of the RNA products from genes harboring truncating mutations. We developed and conducted a high throughput screen, based on the ratiometric analysis of transcripts, to identify critical mediators of NMD. This screen revealed disruption of kinase SMG1s phosphorylation of UPF1 as a potent disruptor of NMD. This led us to design a novel SMG1 inhibitor, KVS0001, that elevates the expression of transcripts and proteins resulting from truncating mutations in vivo and in vitro. Most importantly, KVS0001 concomitantly increased the presentation of immune-targetable HLA class I-associated peptides from NMD-downregulated proteins on the surface of cancer cells. KVS0001 provides new opportunities for studying NMD and the diseases in which NMD plays a role, including cancer and inherited diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.29.573636">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.29.573636" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.29.573636">
        <p class="paperTitle">Timp2 loss-of-function mutation and TIMP2 treatment in murine model of NSCLC: modulation of immunosuppression and oncogenic signaling</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.29.573636" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.29.573636" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Peeney, D.; Kumar, S.; Singh, T. P.; Liu, Y.; Jensen, S.; Chowdhury, A.; Coates-Park, S.; Rich, J.; Gurung, S.; Fan, Y.; Meerzaman, D.; Stetler-Stevenson, W. G.</p>
        <p class="info">Score: 2.5, Published: 2023-12-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.29.573636' target='https://doi.org/10.1101/2023.12.29.573636'> 10.1101/2023.12.29.573636</a></p>
        <p class="abstract">Mounting evidence suggests that the tissue inhibitor of metalloproteinases-2 (TIMP2) can reduce tumor burden and metastasis. However, the demonstration of such anti-tumor activity and associated mechanisms using in vivo tumor models is lacking. The effects of a Timp2 functional mutation and administration of recombinant TIMP2 were examined in both orthotopic and heterotopic murine models of lung cancer using C57Bl/6 syngeneic Lewis Lung 2-luciferase 2 cells (LL2-luc2) cells. Mice harboring a functional mutation of TIMP2 (mT2) display markedly increased primary lung tumor growth, increased mortality, enriched vasculature, and enhanced infiltration of pro-tumorigenic, immunosuppressive myeloid cells. Treatment with recombinant TIMP2 reduced primary tumor growth in both mutant and wild-type (wt) mice. Comparison of transcriptional profiles of lung tissues from tumor-free, wt versus mT2 mice reveals only minor changes. However, lung tumor-bearing mice of both genotypes demonstrate significant genotype-dependent changes in gene expression following treatment with TIMP. In tumor-bearing wt mice, TIMP2 treatment reduced the expression of upstream oncogenic mediators, whereas treatment of mT2 mice resulted in an immunomodulatory phenotype. A heterotopic subcutaneous model generating metastatic pulmonary tumors demonstrated that daily administration of recombinant TIMP2 significantly downregulates the expression of heat shock proteins, suggesting a reduction of cell-stress responses. In summary, we describe how TIMP2 exerts novel, anti-tumor effects in a murine model of lung cancer and that rTIMP2 treatment supports a normalizing effect on the tumor microenvironment. Our findings show that TIMP2 treatment demonstrates significant potential as an adjuvant in the treatment of NSCLC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.573275">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.573275" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.573275">
        <p class="paperTitle">Spatial dissection of cellular heterogeneity and functional crosstalk in the tumor microenvironments of gastric cancers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.573275" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.573275" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kim, T.; Lee, S. H.; Lee, D.; Choi, J.; Oh, H. J.; Ham, I.-H.; Ryu, D.; Lee, S.-Y.; Han, D.-J.; Kim, S.; Lee, S.; Hur, H.</p>
        <p class="info">Score: 2.5, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.573275' target='https://doi.org/10.1101/2023.12.24.573275'> 10.1101/2023.12.24.573275</a></p>
        <p class="abstract">Spatial transcriptomics offers unprecedented challenges in characterizing tumor ecosystems, encompassing tumor cells and their surrounding tumor microenvironments (TME) along with cellular interactions therein. Here, we conducted Visium-based spatial transcriptomics on nine primary gastric cancers (GC) to translate the spatial organization of GC ecosystems into the functional landscape of cellular crosstalk across malignant, stromal and immune cells. We identified three distinct GC subtypes of 3 immunogenic, 3 epithelial and 3 stromal GCs with elevated infiltration of immune, malignant cells and fibroblasts reflecting the heterogeneity of the cell types and their interactions in GC TME. GC spatial architecture was further delineated into 6 regional compartments with varying degree of TME infiltration, among which the fibroblast-enriched TME was associated with the upregulation of epithelial-to-mesenchymal transformation (EMT) and immune response in malignant and immune cells, respectively. Cell type-specific transcriptional dynamics representing cellular crosstalk between TME cells were further identified, e.g., the infiltration of malignant and endothelial cells promote the cellular proliferations of TME cells whereas the fibroblasts and immune cells are associated with pro-cancer and anti-cancer immunity, respectively. Ligand-receptor analysis further revealed that CCL2-expressing fibroblasts promote the pro-cancer signaling of immune cells including JAK-STAT3 signaling and inflammatory response in tumor-infiltration macrophages. The CCL2&#43;fibroblasts and macrophages with activated JAK-STAT3 signals are co-localized in spatial GC architecture and their co-abundance was associated with unfavorable clinical outcomes. Taken together, GC spatial transcriptomes can be regionally and functionally delineated into functional cellular crosstalk involving multiple cell types with potential clinical relevance, particularly the interaction between CCL2&#43; fibroblasts and JAK-STAT3&#43;macrophages contributing to cancer-favoring immune contexture in GC TME.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.28.573554">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.28.573554" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.28.573554">
        <p class="paperTitle">Organ specific microenvironmental MR1 expression in cutaneous melanoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.28.573554" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.28.573554" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gordon, P. B.; So, W. Y.; Azubuike, U. F.; Johnson, B.; Cicala, J.; Sturgess, V.; Wong, C.; Bishop, K.; Bresciani, E.; Sood, R.; Ganesan, S.; Tanner, K.</p>
        <p class="info">Score: 2.2, Published: 2023-12-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.28.573554' target='https://doi.org/10.1101/2023.12.28.573554'> 10.1101/2023.12.28.573554</a></p>
        <p class="abstract">The microenvironment is an important regulator of intertumoral trafficking and activity of immune cells. Understanding how the immune system can be tailored to maintain anti-tumor killing responses in metastatic disease remains an important goal. Thus, immune mediated eradication of metastasis requires the consideration of organ specific microenvironmental cues. Using a xenograft model of melanoma metastasis in adult zebrafish, we perturbed the dynamic balance between the infiltrating immune cells in the metastatic setting using a suite of different transgenic zebrafish. We employed intravital imaging coupled with metabolism imaging (FLIM) to visualize and map the organ specific metabolism with near simultaneity in multiple metastatic lesions. Of all the MHC complexes examined for brain and skeletal metastases, we determined that there is an organ specific expression of mhc1uba (human ortholog, MR1) for both the melanoma cells and the resident and infiltrating immune cells. Specifically, immune clusters did not express mhc1uba in brain metastatic lesions in immune competent fish. Finally, the differential immune response drove organ specific metabolism where tumor glycolysis was increased in brain metastases compared to skeletal and parental lines as measured using fluorescence lifetime imaging microscopy (FLIM). As MR1 belongs to the MHC class I molecules and is a target of immunotherapeutic drugs, we believe that our data presents an opportunity to understand the relationship between organ specific tumor metabolism and drug efficacy in the metastatic setting.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.24.573277">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.24.573277" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.24.573277">
        <p class="paperTitle">Tissue of origin detection for cancer tumor using low-depth cfDNA samples through combination of tumor-specific methylation atlas and genome-wide methylation density in graph convolutional neural networks</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.24.573277" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.24.573277" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nguyen, T. H.; Doan, N. N. T.; Tran, T. H.; Huynh, L. A. K.; Doan, P. L.; Nguyen, T. H. H.; Nguyen, V. T. C.; Nguyen, G. T. H.; Nguyen, H.-N.; Giang, H.; Tran, L. S.; Phan, M. D.</p>
        <p class="info">Score: 1.5, Published: 2023-12-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.24.573277' target='https://doi.org/10.1101/2023.12.24.573277'> 10.1101/2023.12.24.573277</a></p>
        <p class="abstract">BackgroundCell free DNA (cfDNA)-based assays hold great potential in detecting early cancer signals yet determining the tissue-of-origin (TOO) for cancer signals remains a challenging task. Here, we investigated the contribution of a methylation atlas to TOO detection in low depth cfDNA samples.

MethodsWe constructed a tumor-specific methylation atlas (TSMA) using whole-genome bisulfite sequencing (WGBS) data from five types of tumor tissues (breast, colorectal, gastric, liver and lung cancer) and paired white blood cells (WBC). TSMA was used with a non-negative least square matrix factorization (NNLS) deconvolution algorithm to identify the abundance of tumor tissue types in a WGBS sample. We showed that TSMA worked well with tumor tissue but struggled with cfDNA samples due to the overwhelming amount of WBC-derived DNA. To construct a model for TOO, we adopted the multi-modal strategy and used as inputs the combination of deconvolution scores from TSMA with other features of cfDNA.

ResultsOur final model comprised of a graph convolutional neural network using deconvolution scores and genome-wide methylation density features, which achieved an accuracy of 69% in a held-out validation dataset of 239 low-depth cfDNA samples.

ConclusionsIn conclusion, we have demonstrated that our TSMA in combination with other cfDNA features can improve TOO detection in low-depth cfDNA samples.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
