<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues
Authors: Sanghvi, N.; Calvo-Alcaniz, C.; Rajagopal, P.; Scalera, S.; Canu, V.; Sinha, S.; Schischlik, F.; Wang, K.; Madan, S.; Shulman, E.; Papanicolau-Sengos, A.; Blandino, G.; Ruppin, E.; Nair, N.
Score: 32.5, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564810
Metastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to target tissues is challenging.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues
Authors: Sanghvi, N.; Calvo-Alcaniz, C.; Rajagopal, P.; Scalera, S.; Canu, V.; Sinha, S.; Schischlik, F.; Wang, K.; Madan, S.; Shulman, E.; Papanicolau-Sengos, A.; Blandino, G.; Ruppin, E.; Nair, N.
Score: 32.5, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564810
Metastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to target tissues is challenging." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-08T10:39:23+00:00" />
<meta property="article:modified_time" content="2023-11-08T10:39:23+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues
Authors: Sanghvi, N.; Calvo-Alcaniz, C.; Rajagopal, P.; Scalera, S.; Canu, V.; Sinha, S.; Schischlik, F.; Wang, K.; Madan, S.; Shulman, E.; Papanicolau-Sengos, A.; Blandino, G.; Ruppin, E.; Nair, N.
Score: 32.5, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564810
Metastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to target tissues is challenging."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues\nAuthors: Sanghvi, N.; Calvo-Alcaniz, C.; Rajagopal, P.; Scalera, S.; Canu, V.; Sinha, S.; Schischlik, F.; Wang, K.; Madan, S.; Shulman, E.; Papanicolau-Sengos, A.; Blandino, G.; Ruppin, E.; Nair, N.\nScore: 32.5, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564810\nMetastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to target tissues is challenging.",
  "keywords": [
    
  ],
  "articleBody": " Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues\nAuthors: Sanghvi, N.; Calvo-Alcaniz, C.; Rajagopal, P.; Scalera, S.; Canu, V.; Sinha, S.; Schischlik, F.; Wang, K.; Madan, S.; Shulman, E.; Papanicolau-Sengos, A.; Blandino, G.; Ruppin, E.; Nair, N.\nScore: 32.5, Published: 2023-11-02 DOI: 10.1101/2023.10.30.564810\nMetastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to target tissues is challenging. To address this, we assessed whether primary and metastatic tumors resemble their tissue of origin or target tissue in terms of gene expression. We analyzed gene expression profiles from various cancer types, including single-cell and bulk RNA-seq data, in both paired and unpaired primary and metastatic patient cohorts. We quantified the transcriptomic distances between tumor samples and their normal tissues, revealing that primary tumors are more similar to their tissue of origin, while metastases shift towards the target tissue. Pathway-level analysis highlighted critical transcriptomic changes during metastasis. Notably, primary cancers exhibited higher activity in cancer hallmarks, including Activating Invasion and Metastasis, compared to metastatic cancers. This comprehensive landscape analysis provides insight into how cancer tumors adapt to their metastatic environments, providing a transcriptome-wide view of the processes involved.\nOncogenic RAS-Pathway Activation Drives Oncofetal Reprogramming and Creates Therapeutic Vulnerabilities in Juvenile Myelomonocytic Leukemia\nAuthors: Hartmann, M.; Schoenung, M.; Rajak, J.; Maurer, V.; Hai, L.; Bauer, K.; Hakobyan, M.; Staeble, S.; Langstein, J.; Jardine, L.; Roelz, R.; Bohler, S.; Khabirova, E.; Maag, A.-H.; Vonficht, D.; Lebrecht, D.; Bernt, K. M.; Tan, K.; Chen, C.; Alikarami, F.; Boch, T.; Flore, V.; Lutsik, P.; Milsom, M. D.; Raffel, S.; Buske, C.; Haas, S.; Haniffa, M.; Mallm, J.-P.; Behjati, S.; Bonder, M.-J.; Froehling, S.; Niemeyer, C. M.; Hey, J.; Flotho, C.; Plass, C.; Erlacher, M.; Schlesner, M.; Lipka, D. B.\nScore: 24.0, Published: 2023-11-01 DOI: 10.1101/2023.10.27.563754\nAberrant fetal gene expression facilitates tumor-specific cellular plasticity by hijacking molecular programs of embryogenesis1. Persistent fetal gene signatures in childhood malignancies are typically explained by their prenatal origins2-6. In contrast, reactivation of fetal gene expression is considered a consequence of oncofetal reprogramming (OFR) in adult malignancies and is associated with aggressive disease7-10. To date, OFR has not been described in the context of childhood malignancies. Here, we performed a comprehensive multi-layered molecular characterization of juvenile myelomonocytic leukemia (JMML) and identified OFR as a hallmark of aggressive JMML. We observed that hematopoietic stem cells (HSCs) aberrantly express mixed developmental programs in JMML. Expression of fetal gene signatures combined with a postnatal epigenetic landscape suggested OFR, which was validated in a JMML mouse model, demonstrating that postnatal activation of RAS signaling is sufficient to induce fetal gene signatures. Integrative analysis identified the fetal HSC maturation marker CD52 as a novel therapeutic target for aggressive JMML. Anti-CD52 treatment depleted human JMML HSCs and disrupted disease propagation in vivo. In summary, this study implicates OFR, defined as postnatal acquisition of fetal transcription signatures, in the pathobiology of a childhood malignancy. We provide evidence for the direct involvement of oncogenic RAS signaling in OFR. Finally, we demonstrate how OFR can be leveraged for the development of novel treatment strategies. Highlights{blacksquare} Epigenomic and transcriptomic landscape of juvenile myelomonocytic leukemia (JMML) in the context of hematopoietic development. {blacksquare}The presence of fetal transcription signatures in childhood malignancies is not indicative of a developmental maturation block. {blacksquare}High-risk JMML is characterized by oncofetal reprogramming of postnatal hematopoietic stem cells (HSCs). {blacksquare}RAS-pathway mutations induce fetal-like gene expression signatures in murine postnatal HSCs. {blacksquare}The fetal maturation marker CD52 is a novel therapeutic target in high-risk JMML.\nMethylation of histone H3 lysine 36 is a barrier for therapeutic interventions of head and neck squamous cell carcinoma.\nAuthors: Caeiro, L.; Nakata, Y.; Borges, R. L.; Garcia-Martinez, L.; Banuelos, C.; Stransky, S.; Chan, H. L.; Brabson, J.; Dominguez, D.; Zhang, Y.; Lewis, P. W.; Benitah, S. A.; Cimmino, L.; Bilbao, D.; Sidoli, S.; Verdun, R.; Morey, L.\nScore: 11.7, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565847\nApproximately 20% of head and neck squamous cell carcinomas (HNSCC) exhibit reduced methylation on lysine 36 of histone H3 (H3K36me) due to mutations in histone methylase NSD1 or a lysine-to-methionine mutation in histone H3 (H3K36M). Whether such alterations of H3K36me can be exploited for therapeutic interventions is still unknown. Here, we show that HNSCC models expressing H3K36M can be divided into two groups: those that display aberrant accumulation of H3K27me3 and those that maintain steady levels of H3K27me3. The first group shows decreased proliferation, genome instability, and increased sensitivity to genotoxic agents, such as PARP1/2 inhibitors. In contrast, the H3K36M HNSCC models with steady H3K27me3 levels do not exhibit these characteristics unless H3K27me3 levels are elevated, either by DNA hypomethylating agents or by inhibiting the H3K27me3 demethylases KDM6A/B. Mechanistically, we found that H3K36M reduces H3K36me by directly impeding the activities of the histone methyltransferase NSD3 and the histone demethylase LSD2. Notably, we found that aberrant H3K27me3 levels induced by H3K36M expression is not a bona fide epigenetic mark in HNSCC since it requires continuous expression of H3K36M to be inherited. Moreover, increased sensitivity of H3K36M HNSCC models to PARP1/2 inhibitors solely depends on the increased H3K27me3 levels. Indeed, aberrantly high H3K27me3 levels decrease BRCA1 and FANCD2-dependent DNA repair, resulting in higher sensitivity to DNA breaks and replication stress. Finally, in support of our in vitro findings, a PARP1/2 inhibitor alone reduce tumor burden in a H3K36M HNSCC xenograft model with elevated H3K27me3, whereas in a H3K36M HNSCC xenograft model with consistent H3K27me3 levels, a combination of PARP1/2 inhibitors and agents that upregulate H3K27me3 proves to be successful. In conclusion, our findings underscore a delicate balance between H3K36 and H3K27 methylation, essential for maintaining genome stability. This equilibrium presents promising therapeutic opportunities for patients with H3K36me-deficient tumors.\nDissection and reconstruction of the colorectal cancer tumor microenvironment\nAuthors: Broguiere, N.; Lorenzo-Martin, L. F.; Ragusa, S.; Rivest, F.; Brandenberg, N.; Hohnel, S.; Dutta, D.; Gueye, M.; Alpern, D.; Deplancke, B.; Kandalaft, L.; Coukos, G.; Homicsko, K.; Lutolf, M. P.\nScore: 8.9, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565169\nPatient-derived organoids (PDOs) are the reference in vitro human disease models. However, the utility of colorectal cancer (CRC) PDOs is hindered by the lack of a tumor microenvironment (TME). To address this limitation, we built a living biobank of CRC PDOs with autologous stromal and immune TME. We characterized the original tumors and traditional monocultures using single-cell RNA-seq (scRNA-seq) and whole exome sequencing (WES) to obtain insights into cell type selection and phenotypic drift in culture. Subsequently, we developed culture conditions supporting all cell types to recapitulate the CRC-TME around PDOs. From the transcriptomes of \u003e180k cells obtained from 260 such co-cultures, we illuminated the mutual influence of cells within CRC tumors. Based on original tumor data, atlases of predicted interactions and transcriptional networks elucidated why monocultures were altered and suggested that TME reconstruction more accurately reflected original tumor behavior. We found that inflammatory signals were absent in vitro and recovered upon co-culture with tumor-infiltrating lymphocytes (TILs). We also functionally confirmed that stromal, not cancer cells, mediated immune evasion. Additionally, stroma induced an invasive phenotype in cancer cells. From this deep dive into CRC-TME interactions, we built the human CRC-TME atlas (https://crc-tme.com/), an online portal for interactive exploration of gene expression data, prediction of cell-cell interactions at the pathway and receptor/ligand levels, transcriptional networks, and more. We anticipate PDO cultures with reconstructed TMEs will be valuable for discovery efforts, preclinical studies, and personalized medicine, with the atlas as a framework and inspiration for future CRC-TME studies.\nProteomics of Patient-derived Breast Tumours Identifies a Pro-migratory Osteomodulin-Cyclin Dependent Kinase 1 Axis which Drives Bone Metastasis\nAuthors: Parsons, J.; Harrison, H.; Kedward, T.; Fullwood, P.; Cabral, S.; Spence, K.; Barden, D.; Haworth, J.; Watson, J.; Tsafou, K.; Behan, C.; Ali, N.; Smith, M.; O'Brien, C.; Clarke, R.; Francavilla, C.\nScore: 8.2, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565489\nBreast cancer remains a leading cause of mortality, predominantly due to the development of metastases to vital organs. At present, predictive biomarkers of organ specific metastasis and therapies targeted to the metastatic niche are limited. Here, to identify the molecular determinants of breast cancer metastasis we analysed patient-derived breast tumours by combining quantitative proteomics, bioinformatics, and functional assays in vitro and in vivo. We identified elevated levels of the protein Osteomodulin (OMD) associated with breast cancer bone metastases in patient-derived samples. OMD overexpression in the breast cancer MDA-MB-231 cell model significantly increases cell migration in vitro and promotes the formation of bone metastases in vivo. Phosphoproteomics analysis of MDA-MB-231 cells expressing OMD identifies active Cyclin-dependent kinase 1 (CDK1) downstream of OMD. The importance of the OMD-CDK1 axis was validated using two independent phosphoproteomics datasets analysing patient-derived breast cancer samples. We also show that the OMD-CDK1 axis drives cell migration and cell viability in vitro and the formation of bone metastases in vivo. Finally, CDK1 inhibition reduces in vitro cell viability of an independent cohort of metastatic patient samples showing high CDK1 activity. Therefore, the OMD-CDK1 axis will determine which breast cancer patients develop bone metastases and is a therapeutic target to treat or prevent breast cancer bone metastases.\nSpatial tumor immune heterogeneity facilitates subtype co-existence and therapy response via AP1 dichotomy in pancreatic cancer\nAuthors: Klein, L.; Tu, M.; Krebs, N.; Urbach, L.; Grimm, D. G.; Latif, M. U.; Penz, F.; Chan, N.; Aliar, K.; Vyas, F.; Kishore, U.; Hessmann, E.; Trumpp, A.; Espinet, E.; Papantonis, A.; Khokha, R.; Ellenrieder, V.; Gruenwald, B. T.; Singh, S. K.\nScore: 6.8, Published: 2023-11-02 DOI: 10.1101/2023.10.30.563552\nPancreatic ductal adenocarcinoma (PDAC) displays a high degree of spatial subtype heterogeneity. This intratumoral co-existence of classical and basal-like programs is evident in multi-scale transcriptomic and spatial analyses of resected, advanced-stage and chemotherapy-treated specimens and reciprocally linked to a diverse stromal immune microenvironment as well as worse clinical outcome. However, the underlying mechanisms of intratumoral subtype heterogeneity remain largely unclear. Here, by combining preclinical models, multi-center clinical, bulk and compartment-specific transcriptomic, proteomic, and bioimaging data from human specimens, we identified an interplay between neoplastic intrinsic AP1 transcription factor dichotomy and extrinsic CD68+ macrophages as a driver of intratumoral subtype co-existence along with an immunosuppressive tumor microenvironment with T cell exclusion. Our ATAC-, ChIP-, and RNA-seq analyses revealed that JUNB/AP1- and HDAC-mediated epigenetic programs repress pro-inflammatory immune signatures in tumor cells, antagonizing cJUN/AP1 signaling to favor a therapy-responsive classical neoplastic identity. Through the tumor microenvironment, this dichotomous regulation was further amplified via regional macrophage populations. Moreover, CD68+/TNF-+ cells associated with a reactive phenotype and reduced CD8+ T cell infiltration in human PDAC tumors. Consequently, combined anti-TNF- immunotherapy and chemotherapy resulted in reduced macrophage counts and promoted CD3+/CD8+ T cell infiltration in basal-like PDAC, leading to improved survival in preclinical murine models. We conclude that tumor cell intrinsic epigenetic programs, together with extrinsic microenvironmental cues, facilitate intratumoral subtype heterogeneity and disease progression.\nBreast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization\nAuthors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.\nScore: 7.8, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563749\nTumor-infiltrating macrophages support critical steps in tumor progression, and their accumulation in the tumor microenvironment (TME) is associated with adverse outcomes and therapeutic resistance across human cancers. In the TME, macrophages adopt diverse phenotypic alterations, giving rise to heterogeneous immune activation states and induction of cell cycle. While the transcriptional profiles of these activation states are well-annotated across human cancers, the underlying signals that regulate macrophage heterogeneity and accumulation remain incompletely understood. Here, we leveraged a novel ex vivo organotypic TME (oTME) model of breast cancer, in vivo murine models, and human samples to map the determinants of functional heterogeneity of TME macrophages. We identified a subset of F4/80highSca-1+ self-renewing macrophages maintained by type-I interferon (IFN) signaling and requiring physical contact with cancer-associated fibroblasts. We discovered that the contact-dependent self-renewal of TME macrophages is mediated via Notch4, and its inhibition abrogated tumor growth of breast and ovarian carcinomas in vivo, as well as lung dissemination in a PDX model of triple-negative breast cancer (TNBC). Through spatial multi-omic profiling of protein markers and transcriptomes, we found that the localization of macrophages further dictates functionally distinct but reversible phenotypes, regardless of their ontogeny. Whereas immune-stimulatory macrophages (CD11C+CD86+) populated the tumor epithelial nests, the stroma-associated macrophages (SAMs) were proliferative, immunosuppressive (Sca-1+CD206+PD-L1+), resistant to CSF-1R depletion, and associated with worse patient outcomes. Notably, following cessation of CSF-1R depletion, macrophages rebounded primarily to the SAM phenotype, which was associated with accelerated growth of mammary tumors. Our work reveals the spatial determinants of macrophage heterogeneity in breast cancer and highlights the disruption of macrophage self-renewal as a potential new therapeutic strategy.\nATF6 activation alters colonic lipid metabolism causing tumor-associated microbial adaptation\nAuthors: Coleman, O. I.; Sorbie, A.; Bierwirth, S.; Koevilein, J.; von Stern, M.; Koehler, N.; Wirbel, J.; Schmidt, C.; Kacprowski, T.; Dunkel, A.; Pauling, J. K.; Plagge, J.; Miedel-Cuadra, D.; Wagner, S.; Peng, T.; Metzler, T.; Schafmayer, C.; Hinz, S.; Roeder, C.; Roecken, C.; Stecher, B.; Rosenstiel, P.; Steiger, K.; Jesinghaus, M.; Liebisch, G.; Ecker, J.; Zeller, G.; Janssen, K.-P.; Haller, D.\nScore: 6.1, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565267\nThe induction of endoplasmic reticulum unfolded protein responses (UPRER) contributes to cancer development and progression. We recently linked microbiota-related triggers to the tumor-promoting role of signal transducer activating transcription factor 6 (ATF6) in the colon. Here we substantiate the clinical relevance of ATF6 and related bacterial genera in colorectal cancer patient cohorts. Spatial and longitudinal bacterial profiling in ATF6 transgenic mice (nATF6IEC) identified tumor-initiating and tumor-progressing shifts in the mucosa-associated microbiota. Transcriptional analysis in intestinal epithelial cells (IEC) of germ-free and specific pathogen-free nATF6IEC mice defined bacteria-specific changes in cellular metabolism enriched for fatty acid biosynthesis. Untargeted metabolomics, isotope-labeling in intestinal organoids and FASN inhibition confirmed ATF6-mediated involvement of long-chain fatty acids in tumorigenesis. Multi-omics data integration identified a bacteria-lipid network characterized by fatty acid efflux, catabolism and detoxification. We postulate chronic ATF6 signaling to drive a clinically relevant pathologic response altering lipid metabolism to select for a tumor-promoting microbiota.\nHPV and p53 status as precision determinants of head and neck cancer response to DNA-PKcs inhibition in combination with irradiation\nAuthors: Hayrapetyan, L.; Roth, S. M.; Hovhannisyan, L.; Medo, M.; Quintin, A.; Ott, J.; Albers, J.; Aebersold, D. M.; Zimmer, Y.; Medova, M.\nScore: 3.9, Published: 2023-11-04 DOI: 10.1101/2023.11.02.565300\nMajor risk factors of head and neck squamous cell carcinoma (HNSCC) are tobacco use and human papillomavirus (HPV). HPV E6 oncoprotein leads to the degradation of the p53 protein, whereas HPV-negative cancers are frequently associated with TP53 mutations. Peposertib is a potent and selective, orally administered small-molecule inhibitor of the catalytic subunit of the DNA-dependent kinase (DNA-PKcs), a key regulator of non-homologous end joining (NHEJ). NHEJ inhibition along with irradiation (IR)-induced DNA double-strand breaks has the potential to increase antitumor treatment efficacy. Here, we investigated the responses of HNSCC models with distinct HPV and p53 status to treatments with IR, DNA-PKcs inhibition, and their combination. We observed that IR-induced DNA damage combined with peposertib administration shortly before IR results in decreased cell viability and proliferation and causes DNA repair delay in all the studied HNSCC cell lines. However, our data confirm that the actual cell fate upon this treatment is strongly dependent on cellular p53/HPV status. Cells lacking functional p53 due to its degradation by HPV or due to the presence of a loss-of-function mutation are arrested in the G2 phase of the cell cycle and eliminated by apoptosis whereas p53-proficient HNSCC cell lines undergo senescence. Consequently, HPV+ cancer cell lines and xenografts display stronger and more durable responses and seem to benefit from the combined treatment more than p53-proficient HNSCCs. In conclusion, DNA-PKcs inhibitor peposertib should be further studied as a potential radiosensitizer for HNSCCs, taking into consideration the genetic background and the HPV status of a particular tumor.\nMolecular subtypes of high grade serous ovarian cancer across racial groups and gene expression platforms\nAuthors: Davidson, N. R.; Barnard, M. E.; Hippen, A. A.; Campbell, A.; Johnson, C.; Way, G. P.; Dalley, B. K.; Berchuck, A.; Salas, L. A.; Peres, L. C.; Marks, J. R.; Schildkraut, J. M.; Greene, C. S.; Doherty, J. A.\nScore: 3.8, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565179\nIntroduction: Black individuals with ovarian cancer experience poorer survival compared to non-Hispanic White individuals. High-grade serous carcinoma (HGSC) is the most common ovarian cancer histotype, with gene expression subtypes that are associated with differential survival. We characterized HGSC gene expression in Black individuals and considered whether gene expression differences by race may contribute to disparities in survival. Methods: We performed gene expression analyses using RNA-Seq data from Black and White individuals with HGSC, as well as array-based genotyping data from four existing studies of HGSC. Our main analysis assigned subtypes by identifying dataset-specific clusters using K-means clustering for K=2-4. The cluster- and dataset-specific gene expression patterns were summarized by moderated t-scores that differentiate an individual cluster from all other clusters within each dataset. We compared the calculated gene expression patterns for each cluster across datasets by calculating the Pearson correlation between the two summarized vectors of moderated t-scores. Following K=4 subtype assignment and mapping to The Cancer Genome Atlas (TCGA)-derived HGSC subtypes, we used multivariable-adjusted Cox proportional hazards models to estimate subtype-specific survival separately for each dataset. Results: When we compared HGSC gene expression between RNA-Seq data and array-based gene expression data, cluster-specific gene expression was similar across data types. Comparing the Black study population to White and Japanese study populations, the immunoreactive subtype was more common (39% versus 23%-28%) and the differentiated subtype less common (7% versus 22%-31%). Patterns of subtype-specific survival were similar between the Black and White populations with RNAseq data; when compared to mesenchymal HGSC, the risk of death was similar for proliferative and differentiated cases and lower, but not statistically significantly lower, for immunoreactive cases (HR=0.79 [0.55, 1.13] in the Black population and HR=0.86 [0.62, 1.19] in the White population). Patterns of subtype-specific survival were also similar in a Japanese population and in two of three predominantly White populations with array-based gene expression data. Conclusions: A single pipeline can be used to assign gene expression subtypes to HGSC with array-based or RNA-Seq gene expression data. We observed that while the prevalence of the immunoreactive subtype was higher, and the prevalence of the differentiated subtype was lower among Black individuals compared to White individuals, subtype-specific survival was similar.\n",
  "wordCount" : "3079",
  "inLanguage": "en",
  "datePublished": "2023-11-08T10:39:23Z",
  "dateModified": "2023-11-08T10:39:23Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 8, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.564810">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.564810" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.564810">
        <p class="paperTitle">Charting the transcriptomic landscape of primary and metastatic cancers in relation to their origin and target normal tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.564810" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.564810" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sanghvi, N.; Calvo-Alcaniz, C.; Rajagopal, P.; Scalera, S.; Canu, V.; Sinha, S.; Schischlik, F.; Wang, K.; Madan, S.; Shulman, E.; Papanicolau-Sengos, A.; Blandino, G.; Ruppin, E.; Nair, N.</p>
        <p class="info">Score: 32.5, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.564810' target='https://doi.org/10.1101/2023.10.30.564810'> 10.1101/2023.10.30.564810</a></p>
        <p class="abstract">Metastasis is a leading cause of cancer-related deaths, yet understanding how metastatic tumors adapt from their origin to target tissues is challenging. To address this, we assessed whether primary and metastatic tumors resemble their tissue of origin or target tissue in terms of gene expression. We analyzed gene expression profiles from various cancer types, including single-cell and bulk RNA-seq data, in both paired and unpaired primary and metastatic patient cohorts. We quantified the transcriptomic distances between tumor samples and their normal tissues, revealing that primary tumors are more similar to their tissue of origin, while metastases shift towards the target tissue. Pathway-level analysis highlighted critical transcriptomic changes during metastasis. Notably, primary cancers exhibited higher activity in cancer hallmarks, including Activating Invasion and Metastasis, compared to metastatic cancers. This comprehensive landscape analysis provides insight into how cancer tumors adapt to their metastatic environments, providing a transcriptome-wide view of the processes involved.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.27.563754">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.27.563754" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.27.563754">
        <p class="paperTitle">Oncogenic RAS-Pathway Activation Drives Oncofetal Reprogramming and Creates Therapeutic Vulnerabilities in Juvenile Myelomonocytic Leukemia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.27.563754" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.27.563754" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hartmann, M.; Schoenung, M.; Rajak, J.; Maurer, V.; Hai, L.; Bauer, K.; Hakobyan, M.; Staeble, S.; Langstein, J.; Jardine, L.; Roelz, R.; Bohler, S.; Khabirova, E.; Maag, A.-H.; Vonficht, D.; Lebrecht, D.; Bernt, K. M.; Tan, K.; Chen, C.; Alikarami, F.; Boch, T.; Flore, V.; Lutsik, P.; Milsom, M. D.; Raffel, S.; Buske, C.; Haas, S.; Haniffa, M.; Mallm, J.-P.; Behjati, S.; Bonder, M.-J.; Froehling, S.; Niemeyer, C. M.; Hey, J.; Flotho, C.; Plass, C.; Erlacher, M.; Schlesner, M.; Lipka, D. B.</p>
        <p class="info">Score: 24.0, Published: 2023-11-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.27.563754' target='https://doi.org/10.1101/2023.10.27.563754'> 10.1101/2023.10.27.563754</a></p>
        <p class="abstract">Aberrant fetal gene expression facilitates tumor-specific cellular plasticity by hijacking molecular programs of embryogenesis1. Persistent fetal gene signatures in childhood malignancies are typically explained by their prenatal origins2-6. In contrast, reactivation of fetal gene expression is considered a consequence of oncofetal reprogramming (OFR) in adult malignancies and is associated with aggressive disease7-10. To date, OFR has not been described in the context of childhood malignancies. Here, we performed a comprehensive multi-layered molecular characterization of juvenile myelomonocytic leukemia (JMML) and identified OFR as a hallmark of aggressive JMML. We observed that hematopoietic stem cells (HSCs) aberrantly express mixed developmental programs in JMML. Expression of fetal gene signatures combined with a postnatal epigenetic landscape suggested OFR, which was validated in a JMML mouse model, demonstrating that postnatal activation of RAS signaling is sufficient to induce fetal gene signatures. Integrative analysis identified the fetal HSC maturation marker CD52 as a novel therapeutic target for aggressive JMML. Anti-CD52 treatment depleted human JMML HSCs and disrupted disease propagation in vivo. In summary, this study implicates OFR, defined as postnatal acquisition of fetal transcription signatures, in the pathobiology of a childhood malignancy. We provide evidence for the direct involvement of oncogenic RAS signaling in OFR. Finally, we demonstrate how OFR can be leveraged for the development of novel treatment strategies.

Highlights{blacksquare} Epigenomic and transcriptomic landscape of juvenile myelomonocytic leukemia (JMML) in the context of hematopoietic development.
{blacksquare}The presence of fetal transcription signatures in childhood malignancies is not indicative of a developmental maturation block.
{blacksquare}High-risk JMML is characterized by oncofetal reprogramming of postnatal hematopoietic stem cells (HSCs).
{blacksquare}RAS-pathway mutations induce fetal-like gene expression signatures in murine postnatal HSCs.
{blacksquare}The fetal maturation marker CD52 is a novel therapeutic target in high-risk JMML.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565847">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565847" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565847">
        <p class="paperTitle">Methylation of histone H3 lysine 36 is a barrier for therapeutic interventions of head and neck squamous cell carcinoma.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565847" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565847" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caeiro, L.; Nakata, Y.; Borges, R. L.; Garcia-Martinez, L.; Banuelos, C.; Stransky, S.; Chan, H. L.; Brabson, J.; Dominguez, D.; Zhang, Y.; Lewis, P. W.; Benitah, S. A.; Cimmino, L.; Bilbao, D.; Sidoli, S.; Verdun, R.; Morey, L.</p>
        <p class="info">Score: 11.7, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565847' target='https://doi.org/10.1101/2023.11.06.565847'> 10.1101/2023.11.06.565847</a></p>
        <p class="abstract">Approximately 20% of head and neck squamous cell carcinomas (HNSCC) exhibit reduced methylation on lysine 36 of histone H3 (H3K36me) due to mutations in histone methylase NSD1 or a lysine-to-methionine mutation in histone H3 (H3K36M). Whether such alterations of H3K36me can be exploited for therapeutic interventions is still unknown. Here, we show that HNSCC models expressing H3K36M can be divided into two groups: those that display aberrant accumulation of H3K27me3 and those that maintain steady levels of H3K27me3. The first group shows decreased proliferation, genome instability, and increased sensitivity to genotoxic agents, such as PARP1/2 inhibitors. In contrast, the H3K36M HNSCC models with steady H3K27me3 levels do not exhibit these characteristics unless H3K27me3 levels are elevated, either by DNA hypomethylating agents or by inhibiting the H3K27me3 demethylases KDM6A/B. Mechanistically, we found that H3K36M reduces H3K36me by directly impeding the activities of the histone methyltransferase NSD3 and the histone demethylase LSD2. Notably, we found that aberrant H3K27me3 levels induced by H3K36M expression is not a bona fide epigenetic mark in HNSCC since it requires continuous expression of H3K36M to be inherited. Moreover, increased sensitivity of H3K36M HNSCC models to PARP1/2 inhibitors solely depends on the increased H3K27me3 levels. Indeed, aberrantly high H3K27me3 levels decrease BRCA1 and FANCD2-dependent DNA repair, resulting in higher sensitivity to DNA breaks and replication stress. Finally, in support of our in vitro findings, a PARP1/2 inhibitor alone reduce tumor burden in a H3K36M HNSCC xenograft model with elevated H3K27me3, whereas in a H3K36M HNSCC xenograft model with consistent H3K27me3 levels, a combination of PARP1/2 inhibitors and agents that upregulate H3K27me3 proves to be successful. In conclusion, our findings underscore a delicate balance between H3K36 and H3K27 methylation, essential for maintaining genome stability. This equilibrium presents promising therapeutic opportunities for patients with H3K36me-deficient tumors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565169">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565169" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565169">
        <p class="paperTitle">Dissection and reconstruction of the colorectal cancer tumor microenvironment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565169" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565169" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Broguiere, N.; Lorenzo-Martin, L. F.; Ragusa, S.; Rivest, F.; Brandenberg, N.; Hohnel, S.; Dutta, D.; Gueye, M.; Alpern, D.; Deplancke, B.; Kandalaft, L.; Coukos, G.; Homicsko, K.; Lutolf, M. P.</p>
        <p class="info">Score: 8.9, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565169' target='https://doi.org/10.1101/2023.11.01.565169'> 10.1101/2023.11.01.565169</a></p>
        <p class="abstract">Patient-derived organoids (PDOs) are the reference in vitro human disease models. However, the utility of colorectal cancer (CRC) PDOs is hindered by the lack of a tumor microenvironment (TME). To address this limitation, we built a living biobank of CRC PDOs with autologous stromal and immune TME. We characterized the original tumors and traditional monocultures using single-cell RNA-seq (scRNA-seq) and whole exome sequencing (WES) to obtain insights into cell type selection and phenotypic drift in culture. Subsequently, we developed culture conditions supporting all cell types to recapitulate the CRC-TME around PDOs. From the transcriptomes of &gt;180k cells obtained from 260 such co-cultures, we illuminated the mutual influence of cells within CRC tumors. Based on original tumor data, atlases of predicted interactions and transcriptional networks elucidated why monocultures were altered and suggested that TME reconstruction more accurately reflected original tumor behavior. We found that inflammatory signals were absent in vitro and recovered upon co-culture with tumor-infiltrating lymphocytes (TILs). We also functionally confirmed that stromal, not cancer cells, mediated immune evasion. Additionally, stroma induced an invasive phenotype in cancer cells. From this deep dive into CRC-TME interactions, we built the human CRC-TME atlas (https://crc-tme.com/), an online portal for interactive exploration of gene expression data, prediction of cell-cell interactions at the pathway and receptor/ligand levels, transcriptional networks, and more. We anticipate PDO cultures with reconstructed TMEs will be valuable for discovery efforts, preclinical studies, and personalized medicine, with the atlas as a framework and inspiration for future CRC-TME studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565489">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565489" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565489">
        <p class="paperTitle">Proteomics of Patient-derived Breast Tumours Identifies a Pro-migratory Osteomodulin-Cyclin Dependent Kinase 1 Axis which Drives Bone Metastasis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565489" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565489" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parsons, J.; Harrison, H.; Kedward, T.; Fullwood, P.; Cabral, S.; Spence, K.; Barden, D.; Haworth, J.; Watson, J.; Tsafou, K.; Behan, C.; Ali, N.; Smith, M.; O&#39;Brien, C.; Clarke, R.; Francavilla, C.</p>
        <p class="info">Score: 8.2, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565489' target='https://doi.org/10.1101/2023.11.03.565489'> 10.1101/2023.11.03.565489</a></p>
        <p class="abstract">Breast cancer remains a leading cause of mortality, predominantly due to the development of metastases to vital organs. At present, predictive biomarkers of organ specific metastasis and therapies targeted to the metastatic niche are limited. Here, to identify the molecular determinants of breast cancer metastasis we analysed patient-derived breast tumours by combining quantitative proteomics, bioinformatics, and functional assays in vitro and in vivo. We identified elevated levels of the protein Osteomodulin (OMD) associated with breast cancer bone metastases in patient-derived samples. OMD overexpression in the breast cancer MDA-MB-231 cell model significantly increases cell migration in vitro and promotes the formation of bone metastases in vivo. Phosphoproteomics analysis of MDA-MB-231 cells expressing OMD identifies active Cyclin-dependent kinase 1 (CDK1) downstream of OMD. The importance of the OMD-CDK1 axis was validated using two independent phosphoproteomics datasets analysing patient-derived breast cancer samples. We also show that the OMD-CDK1 axis drives cell migration and cell viability in vitro and the formation of bone metastases in vivo. Finally, CDK1 inhibition reduces in vitro cell viability of an independent cohort of metastatic patient samples showing high CDK1 activity. Therefore, the OMD-CDK1 axis will determine which breast cancer patients develop bone metastases and is a therapeutic target to treat or prevent breast cancer bone metastases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.30.563552">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.30.563552" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.30.563552">
        <p class="paperTitle">Spatial tumor immune heterogeneity facilitates subtype co-existence and therapy response via AP1 dichotomy in pancreatic cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.30.563552" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.30.563552" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Klein, L.; Tu, M.; Krebs, N.; Urbach, L.; Grimm, D. G.; Latif, M. U.; Penz, F.; Chan, N.; Aliar, K.; Vyas, F.; Kishore, U.; Hessmann, E.; Trumpp, A.; Espinet, E.; Papantonis, A.; Khokha, R.; Ellenrieder, V.; Gruenwald, B. T.; Singh, S. K.</p>
        <p class="info">Score: 6.8, Published: 2023-11-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.30.563552' target='https://doi.org/10.1101/2023.10.30.563552'> 10.1101/2023.10.30.563552</a></p>
        <p class="abstract">Pancreatic ductal adenocarcinoma (PDAC) displays a high degree of spatial subtype heterogeneity. This intratumoral co-existence of classical and basal-like programs is evident in multi-scale transcriptomic and spatial analyses of resected, advanced-stage and chemotherapy-treated specimens and reciprocally linked to a diverse stromal immune microenvironment as well as worse clinical outcome. However, the underlying mechanisms of intratumoral subtype heterogeneity remain largely unclear. Here, by combining preclinical models, multi-center clinical, bulk and compartment-specific transcriptomic, proteomic, and bioimaging data from human specimens, we identified an interplay between neoplastic intrinsic AP1 transcription factor dichotomy and extrinsic CD68&#43; macrophages as a driver of intratumoral subtype co-existence along with an immunosuppressive tumor microenvironment with T cell exclusion. Our ATAC-, ChIP-, and RNA-seq analyses revealed that JUNB/AP1- and HDAC-mediated epigenetic programs repress pro-inflammatory immune signatures in tumor cells, antagonizing cJUN/AP1 signaling to favor a therapy-responsive classical neoplastic identity. Through the tumor microenvironment, this dichotomous regulation was further amplified via regional macrophage populations. Moreover, CD68&#43;/TNF-&#43; cells associated with a reactive phenotype and reduced CD8&#43; T cell infiltration in human PDAC tumors. Consequently, combined anti-TNF- immunotherapy and chemotherapy resulted in reduced macrophage counts and promoted CD3&#43;/CD8&#43; T cell infiltration in basal-like PDAC, leading to improved survival in preclinical murine models. We conclude that tumor cell intrinsic epigenetic programs, together with extrinsic microenvironmental cues, facilitate intratumoral subtype heterogeneity and disease progression.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563749">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563749" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563749">
        <p class="paperTitle">Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563749" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563749" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.</p>
        <p class="info">Score: 7.8, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563749' target='https://doi.org/10.1101/2023.10.24.563749'> 10.1101/2023.10.24.563749</a></p>
        <p class="abstract">Tumor-infiltrating macrophages support critical steps in tumor progression, and their accumulation in the tumor microenvironment (TME) is associated with adverse outcomes and therapeutic resistance across human cancers. In the TME, macrophages adopt diverse phenotypic alterations, giving rise to heterogeneous immune activation states and induction of cell cycle. While the transcriptional profiles of these activation states are well-annotated across human cancers, the underlying signals that regulate macrophage heterogeneity and accumulation remain incompletely understood. Here, we leveraged a novel ex vivo organotypic TME (oTME) model of breast cancer, in vivo murine models, and human samples to map the determinants of functional heterogeneity of TME macrophages. We identified a subset of F4/80highSca-1&#43; self-renewing macrophages maintained by type-I interferon (IFN) signaling and requiring physical contact with cancer-associated fibroblasts. We discovered that the contact-dependent self-renewal of TME macrophages is mediated via Notch4, and its inhibition abrogated tumor growth of breast and ovarian carcinomas in vivo, as well as lung dissemination in a PDX model of triple-negative breast cancer (TNBC). Through spatial multi-omic profiling of protein markers and transcriptomes, we found that the localization of macrophages further dictates functionally distinct but reversible phenotypes, regardless of their ontogeny. Whereas immune-stimulatory macrophages (CD11C&#43;CD86&#43;) populated the tumor epithelial nests, the stroma-associated macrophages (SAMs) were proliferative, immunosuppressive (Sca-1&#43;CD206&#43;PD-L1&#43;), resistant to CSF-1R depletion, and associated with worse patient outcomes. Notably, following cessation of CSF-1R depletion, macrophages rebounded primarily to the SAM phenotype, which was associated with accelerated growth of mammary tumors. Our work reveals the spatial determinants of macrophage heterogeneity in breast cancer and highlights the disruption of macrophage self-renewal as a potential new therapeutic strategy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565267">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565267" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565267">
        <p class="paperTitle">ATF6 activation alters colonic lipid metabolism causing tumor-associated microbial adaptation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565267" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565267" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coleman, O. I.; Sorbie, A.; Bierwirth, S.; Koevilein, J.; von Stern, M.; Koehler, N.; Wirbel, J.; Schmidt, C.; Kacprowski, T.; Dunkel, A.; Pauling, J. K.; Plagge, J.; Miedel-Cuadra, D.; Wagner, S.; Peng, T.; Metzler, T.; Schafmayer, C.; Hinz, S.; Roeder, C.; Roecken, C.; Stecher, B.; Rosenstiel, P.; Steiger, K.; Jesinghaus, M.; Liebisch, G.; Ecker, J.; Zeller, G.; Janssen, K.-P.; Haller, D.</p>
        <p class="info">Score: 6.1, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565267' target='https://doi.org/10.1101/2023.11.03.565267'> 10.1101/2023.11.03.565267</a></p>
        <p class="abstract">The induction of endoplasmic reticulum unfolded protein responses (UPRER) contributes to cancer development and progression. We recently linked microbiota-related triggers to the tumor-promoting role of signal transducer activating transcription factor 6 (ATF6) in the colon. Here we substantiate the clinical relevance of ATF6 and related bacterial genera in colorectal cancer patient cohorts. Spatial and longitudinal bacterial profiling in ATF6 transgenic mice (nATF6IEC) identified tumor-initiating and tumor-progressing shifts in the mucosa-associated microbiota. Transcriptional analysis in intestinal epithelial cells (IEC) of germ-free and specific pathogen-free nATF6IEC mice defined bacteria-specific changes in cellular metabolism enriched for fatty acid biosynthesis. Untargeted metabolomics, isotope-labeling in intestinal organoids and FASN inhibition confirmed ATF6-mediated involvement of long-chain fatty acids in tumorigenesis. Multi-omics data integration identified a bacteria-lipid network characterized by fatty acid efflux, catabolism and detoxification. We postulate chronic ATF6 signaling to drive a clinically relevant pathologic response altering lipid metabolism to select for a tumor-promoting microbiota.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.02.565300">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.02.565300" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.02.565300">
        <p class="paperTitle">HPV and p53 status as precision determinants of head and neck cancer response to DNA-PKcs inhibition in combination with irradiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.02.565300" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.02.565300" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hayrapetyan, L.; Roth, S. M.; Hovhannisyan, L.; Medo, M.; Quintin, A.; Ott, J.; Albers, J.; Aebersold, D. M.; Zimmer, Y.; Medova, M.</p>
        <p class="info">Score: 3.9, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.02.565300' target='https://doi.org/10.1101/2023.11.02.565300'> 10.1101/2023.11.02.565300</a></p>
        <p class="abstract">Major risk factors of head and neck squamous cell carcinoma (HNSCC) are tobacco use and human papillomavirus (HPV). HPV E6 oncoprotein leads to the degradation of the p53 protein, whereas HPV-negative cancers are frequently associated with TP53 mutations. Peposertib is a potent and selective, orally administered small-molecule inhibitor of the catalytic subunit of the DNA-dependent kinase (DNA-PKcs), a key regulator of non-homologous end joining (NHEJ). NHEJ inhibition along with irradiation (IR)-induced DNA double-strand breaks has the potential to increase antitumor treatment efficacy. Here, we investigated the responses of HNSCC models with distinct HPV and p53 status to treatments with IR, DNA-PKcs inhibition, and their combination. We observed that IR-induced DNA damage combined with peposertib administration shortly before IR results in decreased cell viability and proliferation and causes DNA repair delay in all the studied HNSCC cell lines. However, our data confirm that the actual cell fate upon this treatment is strongly dependent on cellular p53/HPV status. Cells lacking functional p53 due to its degradation by HPV or due to the presence of a loss-of-function mutation are arrested in the G2 phase of the cell cycle and eliminated by apoptosis whereas p53-proficient HNSCC cell lines undergo senescence. Consequently, HPV&#43; cancer cell lines and xenografts display stronger and more durable responses and seem to benefit from the combined treatment more than p53-proficient HNSCCs. In conclusion, DNA-PKcs inhibitor peposertib should be further studied as a potential radiosensitizer for HNSCCs, taking into consideration the genetic background and the HPV status of a particular tumor.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565179">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565179" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565179">
        <p class="paperTitle">Molecular subtypes of high grade serous ovarian cancer across racial groups and gene expression platforms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565179" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565179" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davidson, N. R.; Barnard, M. E.; Hippen, A. A.; Campbell, A.; Johnson, C.; Way, G. P.; Dalley, B. K.; Berchuck, A.; Salas, L. A.; Peres, L. C.; Marks, J. R.; Schildkraut, J. M.; Greene, C. S.; Doherty, J. A.</p>
        <p class="info">Score: 3.8, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565179' target='https://doi.org/10.1101/2023.11.01.565179'> 10.1101/2023.11.01.565179</a></p>
        <p class="abstract">Introduction: Black individuals with ovarian cancer experience poorer survival compared to non-Hispanic White individuals. High-grade serous carcinoma (HGSC) is the most common ovarian cancer histotype, with gene expression subtypes that are associated with differential survival. We characterized HGSC gene expression in Black individuals and considered whether gene expression differences by race may contribute to disparities in survival. Methods: We performed gene expression analyses using RNA-Seq data from Black and White individuals with HGSC, as well as array-based genotyping data from four existing studies of HGSC. Our main analysis assigned subtypes by identifying dataset-specific clusters using K-means clustering for K=2-4. The cluster- and dataset-specific gene expression patterns were summarized by moderated t-scores that differentiate an individual cluster from all other clusters within each dataset. We compared the calculated gene expression patterns for each cluster across datasets by calculating the Pearson correlation between the two summarized vectors of moderated t-scores. Following K=4 subtype assignment and mapping to The Cancer Genome Atlas (TCGA)-derived HGSC subtypes, we used multivariable-adjusted Cox proportional hazards models to estimate subtype-specific survival separately for each dataset. Results: When we compared HGSC gene expression between RNA-Seq data and array-based gene expression data, cluster-specific gene expression was similar across data types. Comparing the Black study population to White and Japanese study populations, the immunoreactive subtype was more common (39% versus 23%-28%) and the differentiated subtype less common (7% versus 22%-31%). Patterns of subtype-specific survival were similar between the Black and White populations with RNAseq data; when compared to mesenchymal HGSC, the risk of death was similar for proliferative and differentiated cases and lower, but not statistically significantly lower, for immunoreactive cases (HR=0.79 [0.55, 1.13] in the Black population and HR=0.86 [0.62, 1.19] in the White population). Patterns of subtype-specific survival were also similar in a Japanese population and in two of three predominantly White populations with array-based gene expression data. Conclusions: A single pipeline can be used to assign gene expression subtypes to HGSC with array-based or RNA-Seq gene expression data. We observed that while the prevalence of the immunoreactive subtype was higher, and the prevalence of the differentiated subtype was lower among Black individuals compared to White individuals, subtype-specific survival was similar.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
