<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Metabolism-focused CRISPR screen unveils Mitochondrial Pyruvate Carrier 1 as a critical driver for PARP inhibitor resistance in lung cancer
Authors: Furusawa, T.; Cavero, R.; Liu, Y.; Li, H.; Xu, X.; Andresson, T.; Reinhold, W.; White, O.; Boufraqech, M.; Meyer, T. J.; Hartmann, O.; Diefenbacher, M. E.; Pommier, Y.; Weyemi, U.
Score: 10.1, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569226
Homologous recombination (HR) and poly ADP-ribosylation are partially redundant pathways for repair of DNA damage in normal and cancer cells.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Metabolism-focused CRISPR screen unveils Mitochondrial Pyruvate Carrier 1 as a critical driver for PARP inhibitor resistance in lung cancer
Authors: Furusawa, T.; Cavero, R.; Liu, Y.; Li, H.; Xu, X.; Andresson, T.; Reinhold, W.; White, O.; Boufraqech, M.; Meyer, T. J.; Hartmann, O.; Diefenbacher, M. E.; Pommier, Y.; Weyemi, U.
Score: 10.1, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569226
Homologous recombination (HR) and poly ADP-ribosylation are partially redundant pathways for repair of DNA damage in normal and cancer cells." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-12-03T10:38:01+00:00" />
<meta property="article:modified_time" content="2023-12-03T10:38:01+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Metabolism-focused CRISPR screen unveils Mitochondrial Pyruvate Carrier 1 as a critical driver for PARP inhibitor resistance in lung cancer
Authors: Furusawa, T.; Cavero, R.; Liu, Y.; Li, H.; Xu, X.; Andresson, T.; Reinhold, W.; White, O.; Boufraqech, M.; Meyer, T. J.; Hartmann, O.; Diefenbacher, M. E.; Pommier, Y.; Weyemi, U.
Score: 10.1, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569226
Homologous recombination (HR) and poly ADP-ribosylation are partially redundant pathways for repair of DNA damage in normal and cancer cells."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Metabolism-focused CRISPR screen unveils Mitochondrial Pyruvate Carrier 1 as a critical driver for PARP inhibitor resistance in lung cancer\nAuthors: Furusawa, T.; Cavero, R.; Liu, Y.; Li, H.; Xu, X.; Andresson, T.; Reinhold, W.; White, O.; Boufraqech, M.; Meyer, T. J.; Hartmann, O.; Diefenbacher, M. E.; Pommier, Y.; Weyemi, U.\nScore: 10.1, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569226\nHomologous recombination (HR) and poly ADP-ribosylation are partially redundant pathways for repair of DNA damage in normal and cancer cells.",
  "keywords": [
    
  ],
  "articleBody": " Metabolism-focused CRISPR screen unveils Mitochondrial Pyruvate Carrier 1 as a critical driver for PARP inhibitor resistance in lung cancer\nAuthors: Furusawa, T.; Cavero, R.; Liu, Y.; Li, H.; Xu, X.; Andresson, T.; Reinhold, W.; White, O.; Boufraqech, M.; Meyer, T. J.; Hartmann, O.; Diefenbacher, M. E.; Pommier, Y.; Weyemi, U.\nScore: 10.1, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569226\nHomologous recombination (HR) and poly ADP-ribosylation are partially redundant pathways for repair of DNA damage in normal and cancer cells. In cell lines that are deficient in HR, inhibition of poly (ADP-ribose) polymerase (PARP1/2) is a proven target with several PARP inhibitors (PARPi) currently in clinical use. Resistance to PARP inhibitors often develops, usually involving genetic alterations in DNA repair signaling cascades, but also metabolic rewiring particularly in HR-proficient cells. PARP1/2 utilize NAD+ (nicotinamide adenine dinucleotide), an essential substrate not only for PARPs but also for multiple pathways in cellular metabolism, TCA cycle and mitochondrial functions. Thus, NAD+ is central to many key cellular functions. Both activation of PARPs by DNA damage and their inhibition by drugs such as olaparib affect NAD+ consumption. We surmised that alterations in NAD+ metabolism by cancer drugs such as Olaparib might be involved in the development of resistance to drug therapy. To test this hypothesis, we conducted a metabolism-focused CRISPR knockout (KO) screen to identify genes which undergo alterations during treatment of tumor cells with PARP inhibitors. Of about 3000 genes in the screen, our data revealed that mitochondrial pyruvate carrier 1 (MPC1) is an essential factor in desensitizing NSCLC lung cancer lines to PARP inhibition. In contrast to NSCLC lung cancer cells, triple negative breast cancer cells do not exhibit such desensitization following MPC1 loss and reprogram the TCA cycle and oxidative phosphorylation pathways to overcome PARP inhibitor treatment. Our findings unveil a previously unknown synergistic response between MPC1 loss and PARP inhibition in lung cancer cells.\nNext-Gen Profiling of Tumor-resident Stem Cells using Machine learning\nAuthors: Chowdhury, D.; Neekhra, B.; Priyadarshi, S.; Mukherjee, S.; Maity, D.; Gupta, D.; Haldar, S.\nScore: 4.8, Published: 2023-11-25 DOI: 10.1101/2023.11.24.568600\nTumor-resident stem cells, also known as cancer stem cells (CSCs), constitute a subgroup within tumors, play a crucial role in fostering resistance to treatment and the recurrence of tumors, and pose significant challenges for conventional therapeutic methods. Existing approaches for identifying CSCs face notable hurdles related to scalability, reproducibility, and technical consistency across different cancer types due to the adaptable nature of CSCs. In this context, we introduce OSCORP, an innovative machine-learning-driven approach. This methodology quantifies and identifies CSCs, achieving almost 99% accuracy using biopsy bulk RNAseq data. OSCORP leverages genetic similarities between normal and cancer stem cells. By categorizing CSCs into four distinct yet dynamic potency states, this approach provides insights into the differentiation landscape of CSCs, unveiling previously undisclosed facets of tumor heterogeneity. In evaluations conducted on patient samples across 22 cancer types, OSCORP revealed clinical, transcriptomic, and immunological signatures associated with each CSC state. It has emerged as a comprehensive tool for understanding and addressing the complexities of cancer stem cells. Ultimately, OSCORP opens up new possibilities for more effective personalized cancer therapies and holds the potential to serve as a clinical tool for monitoring patient-specific CSC changes during treatment or follow-up care.\nTemporal genomic analysis of melanoma rejection identifies regulators of tumor immune evasion\nAuthors: Shvefel, S. C.; Pai, J.; Cao, Y.; Pal, L.; Levy, R.; Yao, W.; Cheng, K.; Zemanek, M.; Bartok, O.; Weller, C.; Yin, Y.; Du, P.; Yakubovich, E.; Orr, I.; Ben-Dor, S.; Oren, R.; Fellus-Alyagor, L.; Golani, O.; Goliand, I.; Ranmar, D.; Savchenko, I.; Ketrarou, N.; Schaffer, A. A.; Ruppin, E.; Satpathy, A.; Samuels, Y.\nScore: 4.8, Published: 2023-11-29 DOI: 10.1101/2023.11.29.569032\nDecreased intra-tumor heterogeneity (ITH) correlates with increased patient survival and immunotherapy response. However, even highly homogenous tumors may display variability in their aggressiveness, and how immunologic-factors impinge on their aggressiveness remains understudied. Here we studied the mechanisms responsible for the immune-escape of murine tumors with low ITH. We compared the temporal growth of homogeneous, genetically-similar single-cell clones that are rejected vs. those that are not- rejected after transplantation in-vivo using single-cell RNA sequencing and immunophenotyping. Non- rejected clones showed high infiltration of tumor-associated-macrophages (TAMs), lower T-cell infiltration, and increased T-cell exhaustion compared to rejected clones. Comparative analysis of rejection- associated gene expression programs, combined with in-vivo CRISPR knockout screens of candidate mediators, identified Mif (macrophage migration inhibitory factor) as a regulator of immune rejection. Mif knockout led to smaller tumors and reversed non-rejection-associated immune composition, particularly, leading to the reduction of immunosuppressive macrophage infiltration. Finally, we validated these results in melanoma patient data. Statement of significanceHere, we uncover the association of Mif expression with tumor growth and aggressiveness, specifically in low ITH tumors. These findings could facilitate the development of new strategies to treat patients with homogeneous, high-MIF expressing tumors that are unresponsive to immune checkpoint therapy.\nH2A.Z chaperones converge on histone H4 acetylation for melanoma cell proliferation\nAuthors: Jostes, S.; Vardabasso, C.; Dong, J.; Carcamo, S.; Singh, R.; Phelps, R.; Meadows, A.; Hasson, D.; Bernstein, E.\nScore: 4.2, Published: 2023-11-27 DOI: 10.1101/2023.11.26.568747\nHigh levels of H2A.Z promote melanoma cell proliferation and correlate with poor prognosis. However, the role of the two distinct H2A.Z histone chaperone complexes, SRCAP and P400-TIP60, in melanoma remains unclear. Here, we show that individual depletion of SRCAP, P400, and VPS72 (YL1) not only results in loss of H2A.Z deposition into chromatin, but also a striking reduction of H4 acetylation in melanoma cells. This loss of H4 acetylation is found at the promoters of cell cycle genes directly bound by H2A.Z and its chaperones, suggesting a highly coordinated regulation between H2A.Z deposition and H4 acetylation to promote their expression. Knockdown of each of the three subunits downregulates E2F1 and its targets, resulting in a cell cycle arrest akin to H2A.Z depletion. However, unlike H2A.Z deficiency, loss of the shared H2A.Z chaperone subunit YL1 induces apoptosis. Furthermore, YL1 is overexpressed in melanoma tissues, and its upregulation is associated with poor patient outcome. Together, these findings provide a rationale for future targeting of H2A.Z chaperones as an epigenetic strategy for melanoma treatment.\nCancer-associated fibroblasts serve as decoys to suppress NK cell anti-cancer cytotoxicity\nAuthors: Ben-Shmuel, A.; Gruper, Y.; Levi-Galibov, O.; Rosenberg-Fogler, H.; Carradori, G.; Stein, Y.; Dadiani, M.; Naumova, M.; Nevo, R.; Morzaev-Sulzbach, D.; Yagel, G.; Mayer, S.; Gal-Yam, E.; Scherz-Shouval, R.\nScore: 20.7, Published: 2023-11-23 DOI: 10.1101/2023.11.23.568355\nCancer associated fibroblasts (CAFs) are among the most abundant components of the breast tumor microenvironment (TME) and major contributors to immune modulation. CAFs are well-known to regulate the activity of diverse types of immune cells including T cells, macrophages and dendritic cells, however little is known about their interaction with Natural killer (NK) cells, which constitute an important arm of anti-tumor immunity. Here we find, using mouse models of cancer and ex-vivo co-cultures, that CAFs inhibit NK cell cytotoxicity towards cancer cells. We unravel the mechanism by which this suppression occurs, through ligand-receptor engagement between NK cells and CAFs leading to CAF cytolysis, which in turn diminishes the expression of activating receptors on NK cells, promoting cancer escape from NK cell surveillance. Analysis of breast cancer patient samples reveals enrichment of NK cells in CAF-rich regions, and upregulation of NK binding ligands on CAFs which is correlated with poor disease outcome. These results reveal a CAF-mediated immunosuppressive decoy mechanism with implications for treatment of solid tumors.\nHypermetabolic state is associated with circadian rhythm disruption in mouse and human cancer cells\nAuthors: Iascone, D. M.; Zhang, X.; Bafford, P.; Mesaros, C.; Sela, Y.; Hofbauer, S.; Zhang, S. L.; Cook, K.; Pivarshev, P.; Stanger, B.; Anderson, S. A.; Dang, C. V.; Sehgal, A.\nScore: 7.1, Published: 2023-11-13 DOI: 10.1101/2023.11.08.566310\nCrosstalk between cellular metabolism and circadian rhythms is a fundamental building block of multicellular life, and disruption of this reciprocal communication could be relevant to degenerative disease, including cancer. Here, we investigated whether maintenance of circadian rhythms depends upon specific metabolic pathways, particularly in the context of cancer. We found that in adult mouse fibroblasts, ATP levels were a major contributor to overall levels of a clock gene luciferase reporter, although not necessarily to the strength of circadian cycling. In contrast, we identified significant metabolic control of circadian function in an in vitro mouse model of pancreatic adenocarcinoma. Metabolic profiling of a library of congenic tumor cell clones revealed significant differences in levels of lactate, pyruvate, ATP, and other crucial metabolites that we used to identify candidate clones with which to generate circadian reporter lines. Despite the shared genetic background of the clones, we observed diverse circadian profiles among these lines that varied with their metabolic phenotype: the most hypometabolic line had the strongest circadian rhythms while the most hypermetabolic line had the weakest rhythms. Treatment of these tumor cell lines with bezafibrate, a peroxisome proliferator-activated receptor (PPAR) agonist shown to increase OxPhos, decreased the amplitude of circadian oscillation in a subset of tumor cell lines. Strikingly, treatment with the Complex I antagonist rotenone enhanced circadian rhythms only in the tumor cell line in which glycolysis was also low, thereby establishing a hypometabolic state. We further analyzed metabolic and circadian phenotypes across a panel of human patient-derived melanoma cell lines and observed a significant negative association between metabolic activity and circadian cycling strength. Together, these findings suggest that metabolic heterogeneity in cancer directly contributes to circadian function, and that high levels of glycolysis or OxPhos independently disrupt circadian rhythms in these cells.\nMultivariate modeling of metabolic state vulnerabilities across diverse cancer contexts reveals synthetically lethal associations\nAuthors: Abecunas, C.; Fallahi-Sichani, M.\nScore: 3.5, Published: 2023-11-29 DOI: 10.1101/2023.11.28.569098\nTargeting the distinct metabolic needs of tumor cells has recently emerged as a promising strategy for cancer therapy. The heterogeneous, context-dependent nature of cancer cell metabolism, however, poses challenges in identifying effective therapeutic interventions. Here, we utilize various unsupervised and supervised multivariate modeling approaches to systematically pinpoint recurrent metabolic states within hundreds of cancer cell lines, elucidate their association with tissue lineage and growth environments, and uncover vulnerabilities linked to their metabolic states across diverse genetic and tissue contexts. We validate key findings using data from an independent set of cell lines, pharmacological screens, and via single-cell analysis of patient-derived tumors. Our analysis uncovers new synthetically lethal associations between the tumor metabolic state (e.g., oxidative phosphorylation), driver mutations (e.g., loss of tumor suppressor PTEN), and actionable biological targets (e.g., mitochondrial electron transport chain). Investigating these relationships could inform the development of more precise and context-specific, metabolism-targeted cancer therapies.\nMutational scanning reveals oncogenic CTNNB1 mutations have diverse effects on signalling and clinical traits\nAuthors: Krishna, A.; Meynert, A.; Kelder, M.; Ewing, A.; Sheraz, S.; Ferrer-Vacquer, A.; Grimes, G.; Becher, H.; Silk, R.; Semple, C. A.; Kendall, T.; Hadjantonakis, A.-K.; Bird, T.; Marsh, J. A.; Hohenstein, P.; Wood, A. J.; Ozdemir, D. D.\nScore: 10.5, Published: 2023-11-13 DOI: 10.1101/2023.11.09.566307\nCTNNB1, the gene encoding {beta}-catenin, is a frequent target for oncogenic mutations activating the canonical Wnt signalling pathway, typically via missense mutations within a degron hotspot motif in exon 3. Here, we combine saturation genome editing with a fluorescent reporter assay to quantify signalling phenotypes for all 342 missense mutations in the mutation hotspot, including 74 recurrent mutations reported in over 6000 tumours. Our data define the genetic requirements for {beta}-catenin degron function and reveal diverse levels of signal activation among known driver mutations. Tumorigenesis in different human tissues involves selection for CTNNB1 mutations spanning distinct ranges of effect size. In hepatocellular carcinoma, mutations that activate {beta}-catenin relatively weakly are associated with worse prognosis compared to stronger activating mutations, despite greater immune cell infiltration in the tumour microenvironment. Our work therefore provides a resource to understand mutational diversity within a pan-cancer mutation hotspot, with potential implications for targeted therapy.\nProtein tyrosine phosphatase receptor kappa regulates glycolysis and de novo lipogenesis to promote hepatocyte metabolic reprogramming in obesity\nAuthors: Gilglioni, E. H.; Li, A.; Saint-Pierre Wijckmans, W.; Shen, T.-K.; Perez-Chavez, I.; Hovhannisyan, G.; Lisjak, M.; Negueruela, J.; Vandenbempt, V.; Bauza-Martinez, J.; Herranz, J. M.; Ezerina, D.; Demine, S.; Feng, Z.; Vignane, T.; Otero-Sanchez, L.; Lambertucci, F.; Prasnicka, A.; Deviere, J.; Hay, D. C.; Encinar, J. A. N.; Singh, S. P.; Messens, J.; Filipovic, M. R.; Sharpe, H. J.; Trepo, E.; Wu, W.; Gurzov, E. N.\nScore: 3.1, Published: 2023-12-01 DOI: 10.1101/2023.12.01.569004\nFat accumulation, de novo lipogenesis, and glycolysis are key drivers of hepatocyte reprogramming and the consequent metabolic dysfunction-associated steatotic liver disease (MASLD). Here we report that obesity leads to dysregulated expression of hepatic protein-tyrosine phosphatases (PTPs). PTPRK was found to be increased in steatotic hepatocytes in both humans and mice, and positively correlated with PPAR{gamma}-induced lipogenic signalling. High-fat-fed PTPRK knockout mice displayed reduced weight gain and hepatic fat accumulation. Phosphoproteomic analysis in primary hepatocytes and hepatic metabolomics identified fructose-1,6 bisphosphatase-1 and glycolysis as PTPRK targets in metabolic reprogramming. Silencing PTPRK in hepatoma cell lines resulted in reduced colony-forming ability, and PTPRK knockout mice developed smaller tumours after diethylnitrosamine-induced hepatocarcinogenesis. Our study defines a novel role for PTPRK in regulating hepatic glycolysis, lipid metabolism, and tumour development. PTPRK inhibition may provide therapeutic possibilities in obesity-associated liver diseases.\nEpigenetic Small-Molecule Screen for Inhibition and Reversal of Acinar Ductal Metaplasia in Mouse Pancreatic Organoids\nAuthors: Atanasova, K. R.; Perkins, C. M.; Ratnayake, R.; Jiang, J.; Chen, Q.-Y.; Schmittgen, T. D.; Luesch, H.\nScore: 2.9, Published: 2023-11-27 DOI: 10.1101/2023.11.27.567685\nBackgroundAcinar ductal metaplasia (ADM) is among the earliest initiating events in pancreatic ductal adenocarcinoma (PDAC) development. MethodsWe developed a novel morphology-based screen using organoids from wildtype and p48Cre/+ (Cre) mice to discover epigenetic modulators that inhibit or reverse pancreatic ADM more effectively than the broad-spectrum HDAC inhibitor trichostatin A (TSA). ResultsOf the 144 compounds screened, nine hits and two additional natural product HDAC inhibitors were validated by dose-response analysis. The class I HDAC inhibitors apicidin and FK228, and the histone methyltransferase inhibitor chaetocin demonstrated pronounced ADM inhibition and reversal without inducing significant cytotoxicity at 1 {micro}M. Thioester prodrug class I HDAC inhibitor largazole attenuated ADM while its disulfide homodimer was effective in both ADM inhibition and reversal. Prioritized compounds were validated for ADM reversal in p48Cre/+;LSL-KrasG12D/+ (KC) mouse organoids using both morphological and molecular endpoints. Molecular index analysis of ADM reversal in KC mouse organoids demonstrated improved activity compared to TSA. Improved prodrug stability translated into a stronger phenotypic and molecular response. RNA-sequencing indicated that angiotensinogen was the top inhibited pathway during ADM reversal. ConclusionOur findings demonstrate a unique epigenetic mechanism and suggest that the phenotypic screen developed here may be applied to discover potential treatments for PDAC.\n",
  "wordCount" : "2413",
  "inLanguage": "en",
  "datePublished": "2023-12-03T10:38:01Z",
  "dateModified": "2023-12-03T10:38:01Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on December 3, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569226">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569226" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569226">
        <p class="paperTitle">Metabolism-focused CRISPR screen unveils Mitochondrial Pyruvate Carrier 1 as a critical driver for PARP inhibitor resistance in lung cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569226" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569226" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Furusawa, T.; Cavero, R.; Liu, Y.; Li, H.; Xu, X.; Andresson, T.; Reinhold, W.; White, O.; Boufraqech, M.; Meyer, T. J.; Hartmann, O.; Diefenbacher, M. E.; Pommier, Y.; Weyemi, U.</p>
        <p class="info">Score: 10.1, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569226' target='https://doi.org/10.1101/2023.11.29.569226'> 10.1101/2023.11.29.569226</a></p>
        <p class="abstract">Homologous recombination (HR) and poly ADP-ribosylation are partially redundant pathways for repair of DNA damage in normal and cancer cells. In cell lines that are deficient in HR, inhibition of poly (ADP-ribose) polymerase (PARP1/2) is a proven target with several PARP inhibitors (PARPi) currently in clinical use. Resistance to PARP inhibitors often develops, usually involving genetic alterations in DNA repair signaling cascades, but also metabolic rewiring particularly in HR-proficient cells. PARP1/2 utilize NAD&#43; (nicotinamide adenine dinucleotide), an essential substrate not only for PARPs but also for multiple pathways in cellular metabolism, TCA cycle and mitochondrial functions. Thus, NAD&#43; is central to many key cellular functions. Both activation of PARPs by DNA damage and their inhibition by drugs such as olaparib affect NAD&#43; consumption. We surmised that alterations in NAD&#43; metabolism by cancer drugs such as Olaparib might be involved in the development of resistance to drug therapy. To test this hypothesis, we conducted a metabolism-focused CRISPR knockout (KO) screen to identify genes which undergo alterations during treatment of tumor cells with PARP inhibitors. Of about 3000 genes in the screen, our data revealed that mitochondrial pyruvate carrier 1 (MPC1) is an essential factor in desensitizing NSCLC lung cancer lines to PARP inhibition. In contrast to NSCLC lung cancer cells, triple negative breast cancer cells do not exhibit such desensitization following MPC1 loss and reprogram the TCA cycle and oxidative phosphorylation pathways to overcome PARP inhibitor treatment. Our findings unveil a previously unknown synergistic response between MPC1 loss and PARP inhibition in lung cancer cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.24.568600">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.24.568600" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.24.568600">
        <p class="paperTitle">Next-Gen Profiling of Tumor-resident Stem Cells using Machine learning</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.24.568600" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.24.568600" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chowdhury, D.; Neekhra, B.; Priyadarshi, S.; Mukherjee, S.; Maity, D.; Gupta, D.; Haldar, S.</p>
        <p class="info">Score: 4.8, Published: 2023-11-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.24.568600' target='https://doi.org/10.1101/2023.11.24.568600'> 10.1101/2023.11.24.568600</a></p>
        <p class="abstract">Tumor-resident stem cells, also known as cancer stem cells (CSCs), constitute a subgroup within tumors, play a crucial role in fostering resistance to treatment and the recurrence of tumors, and pose significant challenges for conventional therapeutic methods. Existing approaches for identifying CSCs face notable hurdles related to scalability, reproducibility, and technical consistency across different cancer types due to the adaptable nature of CSCs. In this context, we introduce OSCORP, an innovative machine-learning-driven approach. This methodology quantifies and identifies CSCs, achieving almost 99% accuracy using biopsy bulk RNAseq data. OSCORP leverages genetic similarities between normal and cancer stem cells. By categorizing CSCs into four distinct yet dynamic potency states, this approach provides insights into the differentiation landscape of CSCs, unveiling previously undisclosed facets of tumor heterogeneity. In evaluations conducted on patient samples across 22 cancer types, OSCORP revealed clinical, transcriptomic, and immunological signatures associated with each CSC state. It has emerged as a comprehensive tool for understanding and addressing the complexities of cancer stem cells. Ultimately, OSCORP opens up new possibilities for more effective personalized cancer therapies and holds the potential to serve as a clinical tool for monitoring patient-specific CSC changes during treatment or follow-up care.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.29.569032">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.29.569032" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.29.569032">
        <p class="paperTitle">Temporal genomic analysis of melanoma rejection identifies regulators of tumor immune evasion</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.29.569032" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.29.569032" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Shvefel, S. C.; Pai, J.; Cao, Y.; Pal, L.; Levy, R.; Yao, W.; Cheng, K.; Zemanek, M.; Bartok, O.; Weller, C.; Yin, Y.; Du, P.; Yakubovich, E.; Orr, I.; Ben-Dor, S.; Oren, R.; Fellus-Alyagor, L.; Golani, O.; Goliand, I.; Ranmar, D.; Savchenko, I.; Ketrarou, N.; Schaffer, A. A.; Ruppin, E.; Satpathy, A.; Samuels, Y.</p>
        <p class="info">Score: 4.8, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.29.569032' target='https://doi.org/10.1101/2023.11.29.569032'> 10.1101/2023.11.29.569032</a></p>
        <p class="abstract">Decreased intra-tumor heterogeneity (ITH) correlates with increased patient survival and immunotherapy response. However, even highly homogenous tumors may display variability in their aggressiveness, and how immunologic-factors impinge on their aggressiveness remains understudied. Here we studied the mechanisms responsible for the immune-escape of murine tumors with low ITH. We compared the temporal growth of homogeneous, genetically-similar single-cell clones that are rejected vs. those that are not- rejected after transplantation in-vivo using single-cell RNA sequencing and immunophenotyping. Non- rejected clones showed high infiltration of tumor-associated-macrophages (TAMs), lower T-cell infiltration, and increased T-cell exhaustion compared to rejected clones. Comparative analysis of rejection- associated gene expression programs, combined with in-vivo CRISPR knockout screens of candidate mediators, identified Mif (macrophage migration inhibitory factor) as a regulator of immune rejection. Mif knockout led to smaller tumors and reversed non-rejection-associated immune composition, particularly, leading to the reduction of immunosuppressive macrophage infiltration. Finally, we validated these results in melanoma patient data.

Statement of significanceHere, we uncover the association of Mif expression with tumor growth and aggressiveness, specifically in low ITH tumors. These findings could facilitate the development of new strategies to treat patients with homogeneous, high-MIF expressing tumors that are unresponsive to immune checkpoint therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.26.568747">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.26.568747" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.26.568747">
        <p class="paperTitle">H2A.Z chaperones converge on histone H4 acetylation for melanoma cell proliferation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.26.568747" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.26.568747" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jostes, S.; Vardabasso, C.; Dong, J.; Carcamo, S.; Singh, R.; Phelps, R.; Meadows, A.; Hasson, D.; Bernstein, E.</p>
        <p class="info">Score: 4.2, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.26.568747' target='https://doi.org/10.1101/2023.11.26.568747'> 10.1101/2023.11.26.568747</a></p>
        <p class="abstract">High levels of H2A.Z promote melanoma cell proliferation and correlate with poor prognosis. However, the role of the two distinct H2A.Z histone chaperone complexes, SRCAP and P400-TIP60, in melanoma remains unclear. Here, we show that individual depletion of SRCAP, P400, and VPS72 (YL1) not only results in loss of H2A.Z deposition into chromatin, but also a striking reduction of H4 acetylation in melanoma cells. This loss of H4 acetylation is found at the promoters of cell cycle genes directly bound by H2A.Z and its chaperones, suggesting a highly coordinated regulation between H2A.Z deposition and H4 acetylation to promote their expression. Knockdown of each of the three subunits downregulates E2F1 and its targets, resulting in a cell cycle arrest akin to H2A.Z depletion. However, unlike H2A.Z deficiency, loss of the shared H2A.Z chaperone subunit YL1 induces apoptosis. Furthermore, YL1 is overexpressed in melanoma tissues, and its upregulation is associated with poor patient outcome. Together, these findings provide a rationale for future targeting of H2A.Z chaperones as an epigenetic strategy for melanoma treatment.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.23.568355">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.23.568355" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.23.568355">
        <p class="paperTitle">Cancer-associated fibroblasts serve as decoys to suppress NK cell anti-cancer cytotoxicity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.23.568355" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.23.568355" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ben-Shmuel, A.; Gruper, Y.; Levi-Galibov, O.; Rosenberg-Fogler, H.; Carradori, G.; Stein, Y.; Dadiani, M.; Naumova, M.; Nevo, R.; Morzaev-Sulzbach, D.; Yagel, G.; Mayer, S.; Gal-Yam, E.; Scherz-Shouval, R.</p>
        <p class="info">Score: 20.7, Published: 2023-11-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.23.568355' target='https://doi.org/10.1101/2023.11.23.568355'> 10.1101/2023.11.23.568355</a></p>
        <p class="abstract">Cancer associated fibroblasts (CAFs) are among the most abundant components of the breast tumor microenvironment (TME) and major contributors to immune modulation. CAFs are well-known to regulate the activity of diverse types of immune cells including T cells, macrophages and dendritic cells, however little is known about their interaction with Natural killer (NK) cells, which constitute an important arm of anti-tumor immunity. Here we find, using mouse models of cancer and ex-vivo co-cultures, that CAFs inhibit NK cell cytotoxicity towards cancer cells. We unravel the mechanism by which this suppression occurs, through ligand-receptor engagement between NK cells and CAFs leading to CAF cytolysis, which in turn diminishes the expression of activating receptors on NK cells, promoting cancer escape from NK cell surveillance. Analysis of breast cancer patient samples reveals enrichment of NK cells in CAF-rich regions, and upregulation of NK binding ligands on CAFs which is correlated with poor disease outcome. These results reveal a CAF-mediated immunosuppressive decoy mechanism with implications for treatment of solid tumors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.08.566310">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.08.566310" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.08.566310">
        <p class="paperTitle">Hypermetabolic state is associated with circadian rhythm disruption in mouse and human cancer cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.08.566310" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.08.566310" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Iascone, D. M.; Zhang, X.; Bafford, P.; Mesaros, C.; Sela, Y.; Hofbauer, S.; Zhang, S. L.; Cook, K.; Pivarshev, P.; Stanger, B.; Anderson, S. A.; Dang, C. V.; Sehgal, A.</p>
        <p class="info">Score: 7.1, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.08.566310' target='https://doi.org/10.1101/2023.11.08.566310'> 10.1101/2023.11.08.566310</a></p>
        <p class="abstract">Crosstalk between cellular metabolism and circadian rhythms is a fundamental building block of multicellular life, and disruption of this reciprocal communication could be relevant to degenerative disease, including cancer. Here, we investigated whether maintenance of circadian rhythms depends upon specific metabolic pathways, particularly in the context of cancer. We found that in adult mouse fibroblasts, ATP levels were a major contributor to overall levels of a clock gene luciferase reporter, although not necessarily to the strength of circadian cycling. In contrast, we identified significant metabolic control of circadian function in an in vitro mouse model of pancreatic adenocarcinoma. Metabolic profiling of a library of congenic tumor cell clones revealed significant differences in levels of lactate, pyruvate, ATP, and other crucial metabolites that we used to identify candidate clones with which to generate circadian reporter lines. Despite the shared genetic background of the clones, we observed diverse circadian profiles among these lines that varied with their metabolic phenotype: the most hypometabolic line had the strongest circadian rhythms while the most hypermetabolic line had the weakest rhythms. Treatment of these tumor cell lines with bezafibrate, a peroxisome proliferator-activated receptor (PPAR) agonist shown to increase OxPhos, decreased the amplitude of circadian oscillation in a subset of tumor cell lines. Strikingly, treatment with the Complex I antagonist rotenone enhanced circadian rhythms only in the tumor cell line in which glycolysis was also low, thereby establishing a hypometabolic state. We further analyzed metabolic and circadian phenotypes across a panel of human patient-derived melanoma cell lines and observed a significant negative association between metabolic activity and circadian cycling strength. Together, these findings suggest that metabolic heterogeneity in cancer directly contributes to circadian function, and that high levels of glycolysis or OxPhos independently disrupt circadian rhythms in these cells.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.28.569098">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.28.569098" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.28.569098">
        <p class="paperTitle">Multivariate modeling of metabolic state vulnerabilities across diverse cancer contexts reveals synthetically lethal associations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.28.569098" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.28.569098" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Abecunas, C.; Fallahi-Sichani, M.</p>
        <p class="info">Score: 3.5, Published: 2023-11-29 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.28.569098' target='https://doi.org/10.1101/2023.11.28.569098'> 10.1101/2023.11.28.569098</a></p>
        <p class="abstract">Targeting the distinct metabolic needs of tumor cells has recently emerged as a promising strategy for cancer therapy. The heterogeneous, context-dependent nature of cancer cell metabolism, however, poses challenges in identifying effective therapeutic interventions. Here, we utilize various unsupervised and supervised multivariate modeling approaches to systematically pinpoint recurrent metabolic states within hundreds of cancer cell lines, elucidate their association with tissue lineage and growth environments, and uncover vulnerabilities linked to their metabolic states across diverse genetic and tissue contexts. We validate key findings using data from an independent set of cell lines, pharmacological screens, and via single-cell analysis of patient-derived tumors. Our analysis uncovers new synthetically lethal associations between the tumor metabolic state (e.g., oxidative phosphorylation), driver mutations (e.g., loss of tumor suppressor PTEN), and actionable biological targets (e.g., mitochondrial electron transport chain). Investigating these relationships could inform the development of more precise and context-specific, metabolism-targeted cancer therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566307">
        <p class="paperTitle">Mutational scanning reveals oncogenic CTNNB1 mutations have diverse effects on signalling and clinical traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishna, A.; Meynert, A.; Kelder, M.; Ewing, A.; Sheraz, S.; Ferrer-Vacquer, A.; Grimes, G.; Becher, H.; Silk, R.; Semple, C. A.; Kendall, T.; Hadjantonakis, A.-K.; Bird, T.; Marsh, J. A.; Hohenstein, P.; Wood, A. J.; Ozdemir, D. D.</p>
        <p class="info">Score: 10.5, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566307' target='https://doi.org/10.1101/2023.11.09.566307'> 10.1101/2023.11.09.566307</a></p>
        <p class="abstract">CTNNB1, the gene encoding {beta}-catenin, is a frequent target for oncogenic mutations activating the canonical Wnt signalling pathway, typically via missense mutations within a degron hotspot motif in exon 3. Here, we combine saturation genome editing with a fluorescent reporter assay to quantify signalling phenotypes for all 342 missense mutations in the mutation hotspot, including 74 recurrent mutations reported in over 6000 tumours. Our data define the genetic requirements for {beta}-catenin degron function and reveal diverse levels of signal activation among known driver mutations. Tumorigenesis in different human tissues involves selection for CTNNB1 mutations spanning distinct ranges of effect size. In hepatocellular carcinoma, mutations that activate {beta}-catenin relatively weakly are associated with worse prognosis compared to stronger activating mutations, despite greater immune cell infiltration in the tumour microenvironment. Our work therefore provides a resource to understand mutational diversity within a pan-cancer mutation hotspot, with potential implications for targeted therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.01.569004">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.01.569004" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.01.569004">
        <p class="paperTitle">Protein tyrosine phosphatase receptor kappa regulates glycolysis and de novo lipogenesis to promote hepatocyte metabolic reprogramming in obesity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.01.569004" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.01.569004" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gilglioni, E. H.; Li, A.; Saint-Pierre Wijckmans, W.; Shen, T.-K.; Perez-Chavez, I.; Hovhannisyan, G.; Lisjak, M.; Negueruela, J.; Vandenbempt, V.; Bauza-Martinez, J.; Herranz, J. M.; Ezerina, D.; Demine, S.; Feng, Z.; Vignane, T.; Otero-Sanchez, L.; Lambertucci, F.; Prasnicka, A.; Deviere, J.; Hay, D. C.; Encinar, J. A. N.; Singh, S. P.; Messens, J.; Filipovic, M. R.; Sharpe, H. J.; Trepo, E.; Wu, W.; Gurzov, E. N.</p>
        <p class="info">Score: 3.1, Published: 2023-12-01 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.01.569004' target='https://doi.org/10.1101/2023.12.01.569004'> 10.1101/2023.12.01.569004</a></p>
        <p class="abstract">Fat accumulation, de novo lipogenesis, and glycolysis are key drivers of hepatocyte reprogramming and the consequent metabolic dysfunction-associated steatotic liver disease (MASLD). Here we report that obesity leads to dysregulated expression of hepatic protein-tyrosine phosphatases (PTPs). PTPRK was found to be increased in steatotic hepatocytes in both humans and mice, and positively correlated with PPAR{gamma}-induced lipogenic signalling. High-fat-fed PTPRK knockout mice displayed reduced weight gain and hepatic fat accumulation. Phosphoproteomic analysis in primary hepatocytes and hepatic metabolomics identified fructose-1,6 bisphosphatase-1 and glycolysis as PTPRK targets in metabolic reprogramming. Silencing PTPRK in hepatoma cell lines resulted in reduced colony-forming ability, and PTPRK knockout mice developed smaller tumours after diethylnitrosamine-induced hepatocarcinogenesis. Our study defines a novel role for PTPRK in regulating hepatic glycolysis, lipid metabolism, and tumour development. PTPRK inhibition may provide therapeutic possibilities in obesity-associated liver diseases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.27.567685">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.27.567685" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.27.567685">
        <p class="paperTitle">Epigenetic Small-Molecule Screen for Inhibition and Reversal of Acinar Ductal Metaplasia in Mouse Pancreatic Organoids</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.27.567685" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.27.567685" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Atanasova, K. R.; Perkins, C. M.; Ratnayake, R.; Jiang, J.; Chen, Q.-Y.; Schmittgen, T. D.; Luesch, H.</p>
        <p class="info">Score: 2.9, Published: 2023-11-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.27.567685' target='https://doi.org/10.1101/2023.11.27.567685'> 10.1101/2023.11.27.567685</a></p>
        <p class="abstract">BackgroundAcinar ductal metaplasia (ADM) is among the earliest initiating events in pancreatic ductal adenocarcinoma (PDAC) development.

MethodsWe developed a novel morphology-based screen using organoids from wildtype and p48Cre/&#43; (Cre) mice to discover epigenetic modulators that inhibit or reverse pancreatic ADM more effectively than the broad-spectrum HDAC inhibitor trichostatin A (TSA).

ResultsOf the 144 compounds screened, nine hits and two additional natural product HDAC inhibitors were validated by dose-response analysis. The class I HDAC inhibitors apicidin and FK228, and the histone methyltransferase inhibitor chaetocin demonstrated pronounced ADM inhibition and reversal without inducing significant cytotoxicity at 1 {micro}M. Thioester prodrug class I HDAC inhibitor largazole attenuated ADM while its disulfide homodimer was effective in both ADM inhibition and reversal. Prioritized compounds were validated for ADM reversal in p48Cre/&#43;;LSL-KrasG12D/&#43; (KC) mouse organoids using both morphological and molecular endpoints. Molecular index analysis of ADM reversal in KC mouse organoids demonstrated improved activity compared to TSA. Improved prodrug stability translated into a stronger phenotypic and molecular response. RNA-sequencing indicated that angiotensinogen was the top inhibited pathway during ADM reversal.

ConclusionOur findings demonstrate a unique epigenetic mechanism and suggest that the phenotypic screen developed here may be applied to discover potential treatments for PDAC.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
