<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Deep intravital brain tumor imaging enabled by tailored three-photon microscopy and analysis
Authors: Schubert, M. C.; Soyka, S. J.; Tamimi, A.; Maus, E.; Denninger, R.; Wissmann, N.; Reyhan, E.; Tetzlaff, S. K.; Beretta, C.; Drumm, M.; Schroers, J.; Steffens, A.; Walshon, J.; McCortney, K.; Heiland, S.; Golebiewska, A.; Kurz, F. T.; Wick, W.; Winkler, F.; Kreshuk, A.; Kuner, T.; Horbinski, C.; Prevedel, R.; Venkataramani, V.
Score: 44.5, Published: 2023-06-21 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Deep intravital brain tumor imaging enabled by tailored three-photon microscopy and analysis
Authors: Schubert, M. C.; Soyka, S. J.; Tamimi, A.; Maus, E.; Denninger, R.; Wissmann, N.; Reyhan, E.; Tetzlaff, S. K.; Beretta, C.; Drumm, M.; Schroers, J.; Steffens, A.; Walshon, J.; McCortney, K.; Heiland, S.; Golebiewska, A.; Kurz, F. T.; Wick, W.; Winkler, F.; Kreshuk, A.; Kuner, T.; Horbinski, C.; Prevedel, R.; Venkataramani, V.
Score: 44.5, Published: 2023-06-21 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-06-28T11:45:43+00:00" />
<meta property="article:modified_time" content="2023-06-28T11:45:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Deep intravital brain tumor imaging enabled by tailored three-photon microscopy and analysis
Authors: Schubert, M. C.; Soyka, S. J.; Tamimi, A.; Maus, E.; Denninger, R.; Wissmann, N.; Reyhan, E.; Tetzlaff, S. K.; Beretta, C.; Drumm, M.; Schroers, J.; Steffens, A.; Walshon, J.; McCortney, K.; Heiland, S.; Golebiewska, A.; Kurz, F. T.; Wick, W.; Winkler, F.; Kreshuk, A.; Kuner, T.; Horbinski, C.; Prevedel, R.; Venkataramani, V.
Score: 44.5, Published: 2023-06-21 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Deep intravital brain tumor imaging enabled by tailored three-photon microscopy and analysis\nAuthors: Schubert, M. C.; Soyka, S. J.; Tamimi, A.; Maus, E.; Denninger, R.; Wissmann, N.; Reyhan, E.; Tetzlaff, S. K.; Beretta, C.; Drumm, M.; Schroers, J.; Steffens, A.; Walshon, J.; McCortney, K.; Heiland, S.; Golebiewska, A.; Kurz, F. T.; Wick, W.; Winkler, F.; Kreshuk, A.; Kuner, T.; Horbinski, C.; Prevedel, R.; Venkataramani, V.\nScore: 44.5, Published: 2023-06-21 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Deep intravital brain tumor imaging enabled by tailored three-photon microscopy and analysis\nAuthors: Schubert, M. C.; Soyka, S. J.; Tamimi, A.; Maus, E.; Denninger, R.; Wissmann, N.; Reyhan, E.; Tetzlaff, S. K.; Beretta, C.; Drumm, M.; Schroers, J.; Steffens, A.; Walshon, J.; McCortney, K.; Heiland, S.; Golebiewska, A.; Kurz, F. T.; Wick, W.; Winkler, F.; Kreshuk, A.; Kuner, T.; Horbinski, C.; Prevedel, R.; Venkataramani, V.\nScore: 44.5, Published: 2023-06-21 DOI: 10.1101/2023.06.17.545350\nIntravital two-photon microscopy has emerged as a powerful technology to study brain tumor biology and its temporal dynamics, including invasion, proliferation and therapeutic resistance in the superficial layers of the mouse cortex. However, intravital microscopy of deeper cortical layers and especially the subcortical white matter, an important route of glioblastoma invasion and recurrence, has not yet been feasible due to low signal-to-noise ratios, missing spatiotemporal resolution and the inability to delineate myelinated axonal tracts. Here, we present a tailored intravital microscopy and artificial intelligence-based analysis methodology and workflow that enables routine deep imaging of glioblastoma over extended time periods, named Deep3P. We show that three-photon microscopy, adaptive optics, as well as customized deep learning-based denoising and machine learning segmentation together allow for deep brain intravital investigation of tumor biology up to 1.2 mm depth. Leveraging this approach, we find that perivascular invasion is a preferred invasion route into the corpus callosum as compared to intracortical glioblastoma invasion and uncover two vascular mechanisms of glioblastoma migration in the white matter. Furthermore, we can define an imaging biomarker of white matter disruption during early glioblastoma colonization. Taken together, Deep3P allows for an efficient and non-invasive investigation of brain tumor biology and its tumor microenvironment in unprecedented deep white and gray matter of the living mouse, opening up novel opportunities for studying the neuroscience of brain tumors and other model systems.\nImmortalization and transformation of primary cells mediated by engineered ecDNAs\nAuthors: Pradella, D.; Zhang, M.; Gao, R.; Melissa, Y. A.; Gluchowska, K. M.; Cendon Florez, Y.; Mishra, T.; La Rocca, G.; Weigl, M.; Jiao, Z.; Nguyen, H. H. M.; Grimm, F.; Lisi, M.; Mastroleo, C.; Chen, K.; Luebeck, J.; Bafna, V.; Antonescu, C. R.; Ventura, A.\nScore: 37.1, Published: 2023-06-26 DOI: 10.1101/2023.06.25.546239\nFocal gene amplifications are among the most common cancer-associated mutations, but their evolution and contribution to tumorigenesis have proven challenging to recapitulate in primary cells and model organisms. Here we describe a general approach to engineer large (\u003e1 Mbp) focal amplifications mediated by extrachromosomal circular DNAs (ecDNAs, also known as 'double minutes') in a spatiotemporally controlled manner in cancer cell lines and in primary cells derived from genetically engineered mice. With this strategy, ecDNA formation can be coupled with expression of fluorescent reporters or other selectable markers to enable the identification and tracking of ecDNA-containing cells. We demonstrate the feasibility of this approach by engineering MDM2-containing ecDNAs in near-diploid human cells, showing that GFP expression can be used to track ecDNA dynamics under physiological conditions or in the presence of specific selective pressures. We also apply this approach to generate mice harboring inducible Myc- and Mdm2-containing ecDNAs analogous to those spontaneously occurring in human cancers. We show that the engineered ecDNAs rapidly accumulate in primary cells derived from these animals, promoting proliferation, immortalization, and transformation.\nPremature Aging and Reduced Cancer Incidence Associated with Near-Complete Body-Wide Myc Inactivation\nAuthors: Wang, H.; Lu, J.; Stevens, T.; Roberts, A.; Mandel, J.; Avula, R.; Ma, B.; Wu, Y.; Wang, J.; Finkel, T.; Van't Land, C.; Vockley, J. E.; Airik, M.; Airik, R.; Muzumdar, R.; Gong, Z.; Torbenson, M. S.; Prochownik, E. V.\nScore: 43.3, Published: 2023-06-19 DOI: 10.1101/2023.01.30.526354\nMYC proto-oncogene dysregulation alters metabolism, translation and other functions in ways that support tumor induction and maintenance. Although Myc+/- mice are healthier and longer-lived than control mice, the long-term ramifications of more complete Myc loss remain unknown. We now describe the chronic consequences of body-wide Myc inactivation initiated postnatally. \"MycKO\" mice acquire numerous features of premature aging including altered body composition and habitus, metabolic dysfunction, hepatic steatosis and the dysregulation of numerous gene sets involved in functions that normally deteriorate with aging. Yet, MycKO mice have extended life spans that correlate with a 3-4-fold lower lifetime cancer incidence. Aging tissues from normal mice and humans also down-regulate Myc and gradually deregulate many of the same Myc target gene sets that are dysregulated in MycKO mice. Normal aging and its associated cancer predisposition are thus highly linked via Myc and its target genes and can be genetically separated.\nSecond-order effects of chemotherapy pharmacodynamics and pharmacokinetics on tumor regression and cachexia\nAuthors: Pierik, L.; McDonald, P.; Anderson, A. R. A.; West, J.\nScore: 12.2, Published: 2023-06-15 DOI: 10.1101/2023.06.14.544974\nDrug dose response curves are ubiquitous in cancer biology, but these curves are often used to measure differential response in first-order effects: the effectiveness of increasing the cumulative dose delivered. In contrast, second-order effects (the variance of drug dose) are often ignored. Knowledge of second-order effects may improve the design of chemotherapy scheduling protocols, leading to improvements in tumor response without changing the total dose delivered. By considering treatment schedules with identical cumulative dose delivered, we optimize treatment by comparing high variance schedules (e.g. high dose, low dose) with low variance schedules (constant dose). We extend a previous framework used to quantify second-order effects, known as antifragility theory, to investigate the role of drug pharmacokinetics. Using a simple one-compartment model, we find that high variance schedules are effective for a wide range of cumulative dose values. Next, using a mouse-parameterized two-compartment model of 5-fluorouracil, we show that the optimal schedule depends on initial tumor volume. Finally, we illustrate the trade-off between tumor response and lean mass preservation. Mathematical modeling indicates that high variance dose schedules provide a potential path forward in mitigating the risk of chemotherapy-associated cachexia by preserving lean mass without sacrificing tumor response.\nLineage-specific intolerance to oncogenic drivers restricts histological transformation\nAuthors: Gardner, E. E.; Earlie, E. M.; Li, K.; Thomas, J.; Hubisz, M. J.; Stein, B. D.; Zhang, C.; Cantley, L. C.; Laughney, A. M.; Varmus, H.\nScore: 9.1, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545980\nLung adenocarcinoma (LUAD) and small cell lung cancer (SCLC) are thought to originate from different epithelial cell types in the lung. Intriguingly, LUAD can histologically transform into SCLC following treatment with targeted therapies. Here we designed models to follow the conversion of LUAD to SCLC and found the barrier to histological transformation converges on tolerance to Myc, which we implicate as a lineage-specific driver of the pulmonary neuroendocrine cell. Histological transformations are frequently accompanied by activation of the Akt pathway. Manipulating this pathway permitted tolerance to Myc as an oncogenic driver, producing rare, stem-like cells, transcriptionally resembling the pulmonary basal lineage. These findings suggest histological transformation may require the plasticity inherent to the basal stem cell, enabling tolerance to previously incompatible oncogenic driver programs. One-Sentence SummaryBy modeling histological transformation of lung cancer, we uncover neuroendocrine-specific tolerance to Myc as an oncogenic driver.\nA survey of CIN measures across mechanistic models\nAuthors: Lynch, A. R.; Bradford, S.; Zhou, A. S.; Oxendine, K.; Henderson, L.; Horner, V. L.; Weaver, B. A.; Burkard, M. E.\nScore: 12.5, Published: 2023-06-15 DOI: 10.1101/2023.06.15.544840\nChromosomal instability (CIN) is the persistent reshuffling of cancer karyotypes via chromosome mis-segregation during cell division. In cancer, CIN exists at varying levels that have differential effects on tumor progression. However, mis-segregation rates remain challenging to assess in human cancer despite an array of available measures. We evaluated measures of CIN by comparing quantitative methods using specific, inducible phenotypic CIN models of chromosome bridges, pseudobipolar spindles, multipolar spindles, and polar chromosomes. For each, we measured CIN fixed and timelapse fluorescence microscopy, chromosome spreads, 6-centromere FISH, bulk transcriptomics, and single cell DNA sequencing (scDNAseq). As expected, microscopy of tumor cells in live and fixed samples correlated well (R=0.77; p\u003c0.01) and sensitively detect CIN. Cytogenetics approaches include chromosome spreads and 6-centromere FISH, which also correlate well (R=0.77; p\u003c0.01) but had limited sensitivity for lower rates of CIN. Bulk genomic DNA signatures and bulk transcriptomic scores, CIN70 and HET70, did not detect CIN. By contrast, single-cell DNA sequencing (scDNAseq) detects CIN with high sensitivity, and correlates very well with imaging methods (R=0.83; p\u003c0.01). In summary, single-cell methods such as imaging, cytogenetics, and scDNAseq can measure CIN, with the latter being the most comprehensive method accessible to clinical samples. To facilitate comparison of CIN rates between phenotypes and methods, we propose a standardized unit of CIN: Mis-segregations per Diploid Division (MDD). This systematic analysis of common CIN measures highlights the superiority of single-cell methods and provides guidance for measuring CIN in the clinical setting. SignificanceCancer relies on genomic changes to drive evolution. One type of change, Chromosomal INstability (CIN), promotes plasticity and heterogeneity of chromosome sets via ongoing errors in mitosis. The rate of these errors informs patient prognosis, drug response, and risk of metastasis. However, measuring CIN in patient tissues is challenging, hindering the emergence of CIN rate as a prognostic and predictive clinical biomarker. To advance clinical measures of CIN, we quantitatively tested the relative performance of several CIN measures in tandem using four well-defined, inducible CIN models. This survey revealed poor sensitivity in several common CIN assays and highlights the primacy of single-cell approaches. Further, we propose a standard, normalized unit of CIN, permitting comparison across methods and studies.\nEpigenomic analysis of Formalin-Fixed Paraffin-Embedded samples by CUT\u0026Tag\nAuthors: Henikoff, S.; Henikoff, J. G.; Ahmad, K.; Paranal, R. M.; Janssens, D. H.; Russell, Z. R.; Szulzewsky, F.; Kugel, S.; Holland, E. C.\nScore: 7.6, Published: 2023-06-22 DOI: 10.1101/2023.06.20.545743\nFor more than a century, Formalin Fixed Paraffin Embedded (FFPE) sample preparation has been the preferred method for long-term preservation of biological material. However, the use of FFPE samples for epigenomic studies has been difficult because of chromatin damage from long exposure to high concentrations of formaldehyde. Previously, we introduced Cleavage Under Targeted Accessible Chromatin (CUTAC), an antibody-targeted chromatin accessibility mapping protocol based on CUT\u0026Tag. Here we show that simple modifications of our single-tube CUTAC protocol are sufficient to produce high-resolution maps of paused RNA Polymerase II (RNAPII) at enhancers and promoters using FFPE samples. We find that transcriptional regulatory element differences produced by FFPE-CUTAC distinguish between mouse brain tumor specimens and identify regulatory element markers with high confidence and precision. Our simple work-flow is suitable for automation, making possible affordable epigenomic profiling of archived biological samples for biomarker identification, clinical applications and retrospective studies.\nThe T766M-EGFR lung cancer mutation promotes tumor growth by exploiting newfound assembly mechanisms in ligand-free EGFR oligomers\nAuthors: Iyer, R. S.; Needham, S. R.; Galdadas, I.; Davis, B. M.; Roberts, S. K.; Man, R. C.; Zanetti-Domingues, L. C.; Clarke, D. T.; Fruhwirth, G. O.; Parker, P. J.; Rolfe, D. J.; Gervasio, F. L.; Martin-Fernandez, M. L.\nScore: 7.3, Published: 2023-06-20 DOI: 10.1101/2023.06.16.545259\nEpidermal growth factor receptor (EGFR) is central to cell growth in physiology and pathophysiologies, including non-small cell lung cancer (NSCLC). EGFR has been successfully targeted with tyrosine kinase inhibitor generations, but the missense secondary T766M mutation is a common cause of resistance. Overcoming this therapeutic challenge has been hindered by poor understanding of how T766M dysregulates EGFR function leading to tumor progression. Here we show that T766M amplifies tumor growth in vivo by exploiting newly discovered oligomer assembly mechanisms employed by wild type (WT)-EGFR to maintain ligand-independent basal phosphorylation. These mechanisms, also shared by drug-resistant exon 20 EGFR insertions, reveal tumor growth promoting functions for hitherto orphan transmembrane and kinase interfaces and for the ectodomain tethered conformation of EGFR. Placing our findings into the context of a ligand-free oligomer structure model, we provide a framework for future drug discovery directed at tackling EGFR mutations in cancer by disabling oligomer-assembling interactions.\nKetogenic diet as a metabolic vehicle for enhancing the therapeutic efficacy of mebendazole and devimistat in preclinical pediatric glioma\nAuthors: Mukherjee, P.; Greenwood, B.; Henao, J.; Kiebish, M. A.; Seyfried, T. N.\nScore: 24.4, Published: 2023-06-11 DOI: 10.1101/2023.06.09.544252\nInvasion of high-grade glioma (HGG) cells through the brain and spinal cord is a leading cause of cancer death in children. Despite advances in treatment, survivors often suffer from life-long adverse effects of the toxic therapies. This study investigated the influence of nutritional ketosis on the therapeutic action of mebendazole (MBZ) and devimistat (CPI-613) against the highly invasive VM-M3 glioblastoma cells in juvenile syngeneic p20-p25 mice; a preclinical model of pediatric HGG. Cerebral implantation of the VM-M3 glioblastoma cells invaded throughout the brain and the spinal column similar to that seen commonly in children with malignant glioma. The maximum therapeutic benefit of MBZ and CPI-613 on tumour invasion and mouse survival occurred only when the drugs were administered together with a ketogenic diet (KD). MBZ reduced VM-M3 tumour cell growth and invasion when evaluated under in-vitro and in-vivo conditions through inhibition of both the glutaminolysis and the glycolysis pathways. Moreover, administration of the drugs with the KD allowed a low dosing for the juvenile mice, which minimized toxicity while improving overall survival. This preclinical study in juvenile mice highlights the potential importance of a diet/drug therapeutic strategy for managing childhood brain cancer.\nEx vivo quantification of anti-tumor T-cell activity upon anti-PD-1 treatment in patient-derived lung tumor-on-chip\nAuthors: veith, i.; mencattini, a.; nurmik, m.; damei, i.; lansche, c.; brosseau, s.; gropplero, g.; corgnac, s.; filippi, j.; pote, n.; mordant, p.; tosello, j.; sedlik, c.; piaggio, e.; girard, n.; camonis, j.; shirvani, h.; mami-chouaib, f.; mechta-grigoriou, f.; descroix, s.; martinelli, e.; zalcman, g.; PARRINI, M. C.\nScore: 3.1, Published: 2023-06-22 DOI: 10.1101/2023.06.21.545960\nThere is a compelling need for new approaches to predict efficacy of immunotherapy drugs. Tumor-on-chip technology exploits microfluidics to generate 3D cell co-cultures embedded in hydrogels that recapitulate immune and stromal characteristics of a simplified tumor ecosystem. Here, we present the development and validation of lung-tumor-on-chip platforms to quickly and precisely measure ex vivo the effects of immune check-point inhibitors on T-cell-mediated cancer cell death, by exploiting the power of live imaging and advanced image analysis algorithms. These tumor-on-chips were generated with patient-derived autologous primary cells isolated from fresh lung cancer samples, opening the path for applications in personalized medicine. Moreover, cancer-associated fibroblasts were shown to impair the response to anti-PD-1, indicating that tumor-on-chips are capable of recapitulating stroma-dependent mechanisms of immunotherapy resistance. This interdisciplinary combination of microfluidic devices, clinically-relevant cell models, and advanced computational methods, can innovatively improve both the fundamental understanding and clinical efficacy of immuno-oncology drugs.\n",
  "wordCount" : "2385",
  "inLanguage": "en",
  "datePublished": "2023-06-28T11:45:43Z",
  "dateModified": "2023-06-28T11:45:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on June 28, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.17.545350">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.17.545350" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.17.545350">
        <p class="paperTitle">Deep intravital brain tumor imaging enabled by tailored three-photon microscopy and analysis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.17.545350" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.17.545350" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Schubert, M. C.; Soyka, S. J.; Tamimi, A.; Maus, E.; Denninger, R.; Wissmann, N.; Reyhan, E.; Tetzlaff, S. K.; Beretta, C.; Drumm, M.; Schroers, J.; Steffens, A.; Walshon, J.; McCortney, K.; Heiland, S.; Golebiewska, A.; Kurz, F. T.; Wick, W.; Winkler, F.; Kreshuk, A.; Kuner, T.; Horbinski, C.; Prevedel, R.; Venkataramani, V.</p>
        <p class="info">Score: 44.5, Published: 2023-06-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.17.545350' target='https://doi.org/10.1101/2023.06.17.545350'> 10.1101/2023.06.17.545350</a></p>
        <p class="abstract">Intravital two-photon microscopy has emerged as a powerful technology to study brain tumor biology and its temporal dynamics, including invasion, proliferation and therapeutic resistance in the superficial layers of the mouse cortex. However, intravital microscopy of deeper cortical layers and especially the subcortical white matter, an important route of glioblastoma invasion and recurrence, has not yet been feasible due to low signal-to-noise ratios, missing spatiotemporal resolution and the inability to delineate myelinated axonal tracts. Here, we present a tailored intravital microscopy and artificial intelligence-based analysis methodology and workflow that enables routine deep imaging of glioblastoma over extended time periods, named Deep3P. We show that three-photon microscopy, adaptive optics, as well as customized deep learning-based denoising and machine learning segmentation together allow for deep brain intravital investigation of tumor biology up to 1.2 mm depth. Leveraging this approach, we find that perivascular invasion is a preferred invasion route into the corpus callosum as compared to intracortical glioblastoma invasion and uncover two vascular mechanisms of glioblastoma migration in the white matter. Furthermore, we can define an imaging biomarker of white matter disruption during early glioblastoma colonization. Taken together, Deep3P allows for an efficient and non-invasive investigation of brain tumor biology and its tumor microenvironment in unprecedented deep white and gray matter of the living mouse, opening up novel opportunities for studying the neuroscience of brain tumors and other model systems.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.25.546239">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.25.546239" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.25.546239">
        <p class="paperTitle">Immortalization and transformation of primary cells mediated by engineered ecDNAs</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.25.546239" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.25.546239" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pradella, D.; Zhang, M.; Gao, R.; Melissa, Y. A.; Gluchowska, K. M.; Cendon Florez, Y.; Mishra, T.; La Rocca, G.; Weigl, M.; Jiao, Z.; Nguyen, H. H. M.; Grimm, F.; Lisi, M.; Mastroleo, C.; Chen, K.; Luebeck, J.; Bafna, V.; Antonescu, C. R.; Ventura, A.</p>
        <p class="info">Score: 37.1, Published: 2023-06-26 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.25.546239' target='https://doi.org/10.1101/2023.06.25.546239'> 10.1101/2023.06.25.546239</a></p>
        <p class="abstract">Focal gene amplifications are among the most common cancer-associated mutations, but their evolution and contribution to tumorigenesis have proven challenging to recapitulate in primary cells and model organisms. Here we describe a general approach to engineer large (&gt;1 Mbp) focal amplifications mediated by extrachromosomal circular DNAs (ecDNAs, also known as &#39;double minutes&#39;) in a spatiotemporally controlled manner in cancer cell lines and in primary cells derived from genetically engineered mice. With this strategy, ecDNA formation can be coupled with expression of fluorescent reporters or other selectable markers to enable the identification and tracking of ecDNA-containing cells. We demonstrate the feasibility of this approach by engineering MDM2-containing ecDNAs in near-diploid human cells, showing that GFP expression can be used to track ecDNA dynamics under physiological conditions or in the presence of specific selective pressures. We also apply this approach to generate mice harboring inducible Myc- and Mdm2-containing ecDNAs analogous to those spontaneously occurring in human cancers. We show that the engineered ecDNAs rapidly accumulate in primary cells derived from these animals, promoting proliferation, immortalization, and transformation.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.01.30.526354">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.01.30.526354" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.01.30.526354">
        <p class="paperTitle">Premature Aging and Reduced Cancer Incidence Associated with Near-Complete Body-Wide Myc Inactivation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.01.30.526354" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.01.30.526354" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Wang, H.; Lu, J.; Stevens, T.; Roberts, A.; Mandel, J.; Avula, R.; Ma, B.; Wu, Y.; Wang, J.; Finkel, T.; Van&#39;t Land, C.; Vockley, J. E.; Airik, M.; Airik, R.; Muzumdar, R.; Gong, Z.; Torbenson, M. S.; Prochownik, E. V.</p>
        <p class="info">Score: 43.3, Published: 2023-06-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.01.30.526354' target='https://doi.org/10.1101/2023.01.30.526354'> 10.1101/2023.01.30.526354</a></p>
        <p class="abstract">MYC proto-oncogene dysregulation alters metabolism, translation and other functions in ways that support tumor induction and maintenance. Although Myc&#43;/- mice are healthier and longer-lived than control mice, the long-term ramifications of more complete Myc loss remain unknown. We now describe the chronic consequences of body-wide Myc inactivation initiated postnatally. &#34;MycKO&#34; mice acquire numerous features of premature aging including altered body composition and habitus, metabolic dysfunction, hepatic steatosis and the dysregulation of numerous gene sets involved in functions that normally deteriorate with aging. Yet, MycKO mice have extended life spans that correlate with a 3-4-fold lower lifetime cancer incidence. Aging tissues from normal mice and humans also down-regulate Myc and gradually deregulate many of the same Myc target gene sets that are dysregulated in MycKO mice. Normal aging and its associated cancer predisposition are thus highly linked via Myc and its target genes and can be genetically separated.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.14.544974">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.14.544974" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.14.544974">
        <p class="paperTitle">Second-order effects of chemotherapy pharmacodynamics and pharmacokinetics on tumor regression and cachexia</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.14.544974" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.14.544974" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pierik, L.; McDonald, P.; Anderson, A. R. A.; West, J.</p>
        <p class="info">Score: 12.2, Published: 2023-06-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.14.544974' target='https://doi.org/10.1101/2023.06.14.544974'> 10.1101/2023.06.14.544974</a></p>
        <p class="abstract">Drug dose response curves are ubiquitous in cancer biology, but these curves are often used to measure differential response in first-order effects: the effectiveness of increasing the cumulative dose delivered. In contrast, second-order effects (the variance of drug dose) are often ignored. Knowledge of second-order effects may improve the design of chemotherapy scheduling protocols, leading to improvements in tumor response without changing the total dose delivered. By considering treatment schedules with identical cumulative dose delivered, we optimize treatment by comparing high variance schedules (e.g. high dose, low dose) with low variance schedules (constant dose). We extend a previous framework used to quantify second-order effects, known as antifragility theory, to investigate the role of drug pharmacokinetics. Using a simple one-compartment model, we find that high variance schedules are effective for a wide range of cumulative dose values. Next, using a mouse-parameterized two-compartment model of 5-fluorouracil, we show that the optimal schedule depends on initial tumor volume. Finally, we illustrate the trade-off between tumor response and lean mass preservation. Mathematical modeling indicates that high variance dose schedules provide a potential path forward in mitigating the risk of chemotherapy-associated cachexia by preserving lean mass without sacrificing tumor response.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545980">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545980" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545980">
        <p class="paperTitle">Lineage-specific intolerance to oncogenic drivers restricts histological transformation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545980" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545980" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gardner, E. E.; Earlie, E. M.; Li, K.; Thomas, J.; Hubisz, M. J.; Stein, B. D.; Zhang, C.; Cantley, L. C.; Laughney, A. M.; Varmus, H.</p>
        <p class="info">Score: 9.1, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545980' target='https://doi.org/10.1101/2023.06.21.545980'> 10.1101/2023.06.21.545980</a></p>
        <p class="abstract">Lung adenocarcinoma (LUAD) and small cell lung cancer (SCLC) are thought to originate from different epithelial cell types in the lung. Intriguingly, LUAD can histologically transform into SCLC following treatment with targeted therapies. Here we designed models to follow the conversion of LUAD to SCLC and found the barrier to histological transformation converges on tolerance to Myc, which we implicate as a lineage-specific driver of the pulmonary neuroendocrine cell. Histological transformations are frequently accompanied by activation of the Akt pathway. Manipulating this pathway permitted tolerance to Myc as an oncogenic driver, producing rare, stem-like cells, transcriptionally resembling the pulmonary basal lineage. These findings suggest histological transformation may require the plasticity inherent to the basal stem cell, enabling tolerance to previously incompatible oncogenic driver programs.

One-Sentence SummaryBy modeling histological transformation of lung cancer, we uncover neuroendocrine-specific tolerance to Myc as an oncogenic driver.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.15.544840">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.15.544840" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.15.544840">
        <p class="paperTitle">A survey of CIN measures across mechanistic models</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.15.544840" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.15.544840" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lynch, A. R.; Bradford, S.; Zhou, A. S.; Oxendine, K.; Henderson, L.; Horner, V. L.; Weaver, B. A.; Burkard, M. E.</p>
        <p class="info">Score: 12.5, Published: 2023-06-15 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.15.544840' target='https://doi.org/10.1101/2023.06.15.544840'> 10.1101/2023.06.15.544840</a></p>
        <p class="abstract">Chromosomal instability (CIN) is the persistent reshuffling of cancer karyotypes via chromosome mis-segregation during cell division. In cancer, CIN exists at varying levels that have differential effects on tumor progression. However, mis-segregation rates remain challenging to assess in human cancer despite an array of available measures. We evaluated measures of CIN by comparing quantitative methods using specific, inducible phenotypic CIN models of chromosome bridges, pseudobipolar spindles, multipolar spindles, and polar chromosomes. For each, we measured CIN fixed and timelapse fluorescence microscopy, chromosome spreads, 6-centromere FISH, bulk transcriptomics, and single cell DNA sequencing (scDNAseq). As expected, microscopy of tumor cells in live and fixed samples correlated well (R=0.77; p&lt;0.01) and sensitively detect CIN. Cytogenetics approaches include chromosome spreads and 6-centromere FISH, which also correlate well (R=0.77; p&lt;0.01) but had limited sensitivity for lower rates of CIN. Bulk genomic DNA signatures and bulk transcriptomic scores, CIN70 and HET70, did not detect CIN. By contrast, single-cell DNA sequencing (scDNAseq) detects CIN with high sensitivity, and correlates very well with imaging methods (R=0.83; p&lt;0.01). In summary, single-cell methods such as imaging, cytogenetics, and scDNAseq can measure CIN, with the latter being the most comprehensive method accessible to clinical samples. To facilitate comparison of CIN rates between phenotypes and methods, we propose a standardized unit of CIN: Mis-segregations per Diploid Division (MDD). This systematic analysis of common CIN measures highlights the superiority of single-cell methods and provides guidance for measuring CIN in the clinical setting.

SignificanceCancer relies on genomic changes to drive evolution. One type of change, Chromosomal INstability (CIN), promotes plasticity and heterogeneity of chromosome sets via ongoing errors in mitosis. The rate of these errors informs patient prognosis, drug response, and risk of metastasis. However, measuring CIN in patient tissues is challenging, hindering the emergence of CIN rate as a prognostic and predictive clinical biomarker. To advance clinical measures of CIN, we quantitatively tested the relative performance of several CIN measures in tandem using four well-defined, inducible CIN models. This survey revealed poor sensitivity in several common CIN assays and highlights the primacy of single-cell approaches. Further, we propose a standard, normalized unit of CIN, permitting comparison across methods and studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.20.545743">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.20.545743" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.20.545743">
        <p class="paperTitle">Epigenomic analysis of Formalin-Fixed Paraffin-Embedded samples by CUT&amp;Tag</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.20.545743" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.20.545743" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Henikoff, S.; Henikoff, J. G.; Ahmad, K.; Paranal, R. M.; Janssens, D. H.; Russell, Z. R.; Szulzewsky, F.; Kugel, S.; Holland, E. C.</p>
        <p class="info">Score: 7.6, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.20.545743' target='https://doi.org/10.1101/2023.06.20.545743'> 10.1101/2023.06.20.545743</a></p>
        <p class="abstract">For more than a century, Formalin Fixed Paraffin Embedded (FFPE) sample preparation has been the preferred method for long-term preservation of biological material. However, the use of FFPE samples for epigenomic studies has been difficult because of chromatin damage from long exposure to high concentrations of formaldehyde. Previously, we introduced Cleavage Under Targeted Accessible Chromatin (CUTAC), an antibody-targeted chromatin accessibility mapping protocol based on CUT&amp;Tag. Here we show that simple modifications of our single-tube CUTAC protocol are sufficient to produce high-resolution maps of paused RNA Polymerase II (RNAPII) at enhancers and promoters using FFPE samples. We find that transcriptional regulatory element differences produced by FFPE-CUTAC distinguish between mouse brain tumor specimens and identify regulatory element markers with high confidence and precision. Our simple work-flow is suitable for automation, making possible affordable epigenomic profiling of archived biological samples for biomarker identification, clinical applications and retrospective studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.16.545259">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.16.545259" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.16.545259">
        <p class="paperTitle">The T766M-EGFR lung cancer mutation promotes tumor growth by exploiting newfound assembly mechanisms in ligand-free EGFR oligomers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.16.545259" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.16.545259" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Iyer, R. S.; Needham, S. R.; Galdadas, I.; Davis, B. M.; Roberts, S. K.; Man, R. C.; Zanetti-Domingues, L. C.; Clarke, D. T.; Fruhwirth, G. O.; Parker, P. J.; Rolfe, D. J.; Gervasio, F. L.; Martin-Fernandez, M. L.</p>
        <p class="info">Score: 7.3, Published: 2023-06-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.16.545259' target='https://doi.org/10.1101/2023.06.16.545259'> 10.1101/2023.06.16.545259</a></p>
        <p class="abstract">Epidermal growth factor receptor (EGFR) is central to cell growth in physiology and pathophysiologies, including non-small cell lung cancer (NSCLC). EGFR has been successfully targeted with tyrosine kinase inhibitor generations, but the missense secondary T766M mutation is a common cause of resistance. Overcoming this therapeutic challenge has been hindered by poor understanding of how T766M dysregulates EGFR function leading to tumor progression. Here we show that T766M amplifies tumor growth in vivo by exploiting newly discovered oligomer assembly mechanisms employed by wild type (WT)-EGFR to maintain ligand-independent basal phosphorylation. These mechanisms, also shared by drug-resistant exon 20 EGFR insertions, reveal tumor growth promoting functions for hitherto orphan transmembrane and kinase interfaces and for the ectodomain tethered conformation of EGFR. Placing our findings into the context of a ligand-free oligomer structure model, we provide a framework for future drug discovery directed at tackling EGFR mutations in cancer by disabling oligomer-assembling interactions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.09.544252">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.09.544252" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.09.544252">
        <p class="paperTitle">Ketogenic diet as a metabolic vehicle for enhancing the therapeutic efficacy of mebendazole and devimistat in preclinical pediatric glioma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.09.544252" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.09.544252" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mukherjee, P.; Greenwood, B.; Henao, J.; Kiebish, M. A.; Seyfried, T. N.</p>
        <p class="info">Score: 24.4, Published: 2023-06-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.09.544252' target='https://doi.org/10.1101/2023.06.09.544252'> 10.1101/2023.06.09.544252</a></p>
        <p class="abstract">Invasion of high-grade glioma (HGG) cells through the brain and spinal cord is a leading cause of cancer death in children. Despite advances in treatment, survivors often suffer from life-long adverse effects of the toxic therapies. This study investigated the influence of nutritional ketosis on the therapeutic action of mebendazole (MBZ) and devimistat (CPI-613) against the highly invasive VM-M3 glioblastoma cells in juvenile syngeneic p20-p25 mice; a preclinical model of pediatric HGG. Cerebral implantation of the VM-M3 glioblastoma cells invaded throughout the brain and the spinal column similar to that seen commonly in children with malignant glioma. The maximum therapeutic benefit of MBZ and CPI-613 on tumour invasion and mouse survival occurred only when the drugs were administered together with a ketogenic diet (KD). MBZ reduced VM-M3 tumour cell growth and invasion when evaluated under in-vitro and in-vivo conditions through inhibition of both the glutaminolysis and the glycolysis pathways.

Moreover, administration of the drugs with the KD allowed a low dosing for the juvenile mice, which minimized toxicity while improving overall survival. This preclinical study in juvenile mice highlights the potential importance of a diet/drug therapeutic strategy for managing childhood brain cancer.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.06.21.545960">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.06.21.545960" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.06.21.545960">
        <p class="paperTitle">Ex vivo quantification of anti-tumor T-cell activity upon anti-PD-1 treatment in patient-derived lung tumor-on-chip</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.06.21.545960" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.06.21.545960" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: veith, i.; mencattini, a.; nurmik, m.; damei, i.; lansche, c.; brosseau, s.; gropplero, g.; corgnac, s.; filippi, j.; pote, n.; mordant, p.; tosello, j.; sedlik, c.; piaggio, e.; girard, n.; camonis, j.; shirvani, h.; mami-chouaib, f.; mechta-grigoriou, f.; descroix, s.; martinelli, e.; zalcman, g.; PARRINI, M. C.</p>
        <p class="info">Score: 3.1, Published: 2023-06-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.06.21.545960' target='https://doi.org/10.1101/2023.06.21.545960'> 10.1101/2023.06.21.545960</a></p>
        <p class="abstract">There is a compelling need for new approaches to predict efficacy of immunotherapy drugs. Tumor-on-chip technology exploits microfluidics to generate 3D cell co-cultures embedded in hydrogels that recapitulate immune and stromal characteristics of a simplified tumor ecosystem. Here, we present the development and validation of lung-tumor-on-chip platforms to quickly and precisely measure ex vivo the effects of immune check-point inhibitors on T-cell-mediated cancer cell death, by exploiting the power of live imaging and advanced image analysis algorithms. These tumor-on-chips were generated with patient-derived autologous primary cells isolated from fresh lung cancer samples, opening the path for applications in personalized medicine. Moreover, cancer-associated fibroblasts were shown to impair the response to anti-PD-1, indicating that tumor-on-chips are capable of recapitulating stroma-dependent mechanisms of immunotherapy resistance. This interdisciplinary combination of microfluidic devices, clinically-relevant cell models, and advanced computational methods, can innovatively improve both the fundamental understanding and clinical efficacy of immuno-oncology drugs.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
