<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Gene regulatory network landscape of Group 3/4 medulloblastoma
Authors: Joshi, P.; Stelzer, T.; Okonechnikov, K.; Sarropoulos, I.; Sepp, M.; Pour-Jamnani, M. V.; Rademacher, A.; Yamada-Saito, T.; Schneider, C.; Schmidt, J.; Schaefer, P.; Leiss, K.; Bortolomeazzi, M.; Mallm, J.-P.; da Silva, P. B. G.; Statz, B.; Wittmann, A.; Schramm, K.; Blattner-Johnson, M.; Fiesel, P.; Jones, B.; Milde, T.; Pajtler, K.; van Tilburg, C. M.; Witt, O.; Rippe, K.; Korshunov, A.; Jones, D.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Gene regulatory network landscape of Group 3/4 medulloblastoma
Authors: Joshi, P.; Stelzer, T.; Okonechnikov, K.; Sarropoulos, I.; Sepp, M.; Pour-Jamnani, M. V.; Rademacher, A.; Yamada-Saito, T.; Schneider, C.; Schmidt, J.; Schaefer, P.; Leiss, K.; Bortolomeazzi, M.; Mallm, J.-P.; da Silva, P. B. G.; Statz, B.; Wittmann, A.; Schramm, K.; Blattner-Johnson, M.; Fiesel, P.; Jones, B.; Milde, T.; Pajtler, K.; van Tilburg, C. M.; Witt, O.; Rippe, K.; Korshunov, A.; Jones, D." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-18T10:38:00+00:00" />
<meta property="article:modified_time" content="2024-02-18T10:38:00+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Gene regulatory network landscape of Group 3/4 medulloblastoma
Authors: Joshi, P.; Stelzer, T.; Okonechnikov, K.; Sarropoulos, I.; Sepp, M.; Pour-Jamnani, M. V.; Rademacher, A.; Yamada-Saito, T.; Schneider, C.; Schmidt, J.; Schaefer, P.; Leiss, K.; Bortolomeazzi, M.; Mallm, J.-P.; da Silva, P. B. G.; Statz, B.; Wittmann, A.; Schramm, K.; Blattner-Johnson, M.; Fiesel, P.; Jones, B.; Milde, T.; Pajtler, K.; van Tilburg, C. M.; Witt, O.; Rippe, K.; Korshunov, A.; Jones, D."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Gene regulatory network landscape of Group 3/4 medulloblastoma\nAuthors: Joshi, P.; Stelzer, T.; Okonechnikov, K.; Sarropoulos, I.; Sepp, M.; Pour-Jamnani, M. V.; Rademacher, A.; Yamada-Saito, T.; Schneider, C.; Schmidt, J.; Schaefer, P.; Leiss, K.; Bortolomeazzi, M.; Mallm, J.-P.; da Silva, P. B. G.; Statz, B.; Wittmann, A.; Schramm, K.; Blattner-Johnson, M.; Fiesel, P.; Jones, B.; Milde, T.; Pajtler, K.; van Tilburg, C. M.; Witt, O.; Rippe, K.; Korshunov, A.; Jones, D.",
  "keywords": [
    
  ],
  "articleBody": " Gene regulatory network landscape of Group 3/4 medulloblastoma\nAuthors: Joshi, P.; Stelzer, T.; Okonechnikov, K.; Sarropoulos, I.; Sepp, M.; Pour-Jamnani, M. V.; Rademacher, A.; Yamada-Saito, T.; Schneider, C.; Schmidt, J.; Schaefer, P.; Leiss, K.; Bortolomeazzi, M.; Mallm, J.-P.; da Silva, P. B. G.; Statz, B.; Wittmann, A.; Schramm, K.; Blattner-Johnson, M.; Fiesel, P.; Jones, B.; Milde, T.; Pajtler, K.; van Tilburg, C. M.; Witt, O.; Rippe, K.; Korshunov, A.; Jones, D. T. W.; Hovestadt, V.; Northcott, P. A.; Thongjuea, S.; Jaeger, N.; Kaessmann, H.; Pfister, S. M.; Kutscher, L. M.\nScore: 18.6, Published: 2024-02-10 DOI: 10.1101/2024.02.09.579680\nResolving the molecular mechanisms driving childhood brain tumors will uncover tumor-specific vulnerabilities and advance mechanism-of-action-based therapies. Here we describe a continuum of cell-states in Group 3/4 medulloblastomas, the most frequent and fatal cerebellar embryonal tumor subgroups, based on the differential activity of transcription-factor-driven gene networks derived using a comprehensive single-nucleus multi-omic medulloblastoma atlas. We show that Group 3/4 tumor diversity stems from enriched cell-states along four molecular identity axes: photoreceptor, MYC, precursor, and unipolar brush cell-like. We identified a potential role of PAX6 in driving dual Group 3- and Group 4-like tumor trajectories in subtype VII tumors. Our study demonstrates how oncogenic events together with lineage determinants drive Group 3/4 tumor identity away from their original source in the cerebellar unipolar brush cell lineage.\nEpigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer\nAuthors: Lakatos, E.; Gunasri, V.; Zapata, L.; Househam, J.; Heide, T.; Trahearn, N.; Swinyard, O.; Cisneros, L.; Lynn, C.; Mossner, M.; Kimberley, C.; Spiteri, I.; Cresswell, G. D.; Llibre-Palomar, G.; Mitchison, M.; Maley, C. C.; Jansen, M.; Rodriguez-Justo, M.; Bridgewater, J.; Baker, A.-M.; Sottoriva, A.; Graham, T. A.\nScore: 9.1, Published: 2024-02-14 DOI: 10.1101/2024.02.12.579956\nImmune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we supplement with high-resolution spatially-resolved neoantigen sequencing data and highly multiplexed imaging of the tumour microenvironment (TME). Analysis of somatic chromatin accessibility alterations (SCAAs) reveals frequent somatic loss of accessibility at antigen presenting genes, and that SCAAs contribute to silencing of neoantigens. We observe that strong immune escape and exclusion occur at the outset of CRC formation, and that within tumours, including at the microscopic level of individual tumour glands, additional immune escape alterations have negligible consequences for the immunophenotype of cancer cells. Further minor immuno-editing occurs during local invasion and is associated with TME reorganisation, but that evolutionary bottleneck is relatively weak. Collectively, we show that immune evasion in CRC follows a \"Big Bang\" evolutionary pattern, whereby genetic, epigenetic and TME-driven immune evasion acquired by the time of transformation defines subsequent cancer-immune evolution.\nMedulloblastoma oncogene aberrations are not involved in tumor initiation, but essential for disease progression and therapy resistance\nAuthors: Okonechnikov, K.; Joshi, P.; Koerber, V.; Rademacher, A.; Bortolomeazzi, M.; Mallm, J.-P.; da Silva, P. B. G.; Statz, B.; Sepp, M.; Sarropoulos, I.; Yamada-Saito, T.; Vaillant, J.; Wittmann, A.; Schramm, K.; Blattner-Johnson, M.; Fiesel, P.; Jones, B.; Milde, T.; Pajtler, K.; van Tilburg, C. M.; Witt, O.; Bochennek, K.; Weber, K. J.; Nonnenmacher, L.; Reimann, C.; Schueller, U.; Mynarek, M.; Rutkowski, S.; Jones, D. T. W.; Korshunov, A.; Rippe, K.; Westermann, F.; Thongjuea, S.; Hoefer, T.; Kaessmann, H.; Kutscher, L. M.; Pfister, S. M.\nScore: 7.9, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579690\nDespite recent advances in understanding disease biology, treatment of Group 3/4 medulloblastoma remains a therapeutic challenge in pediatric neuro-oncology. Bulk-omics approaches have identified considerable intertumoral heterogeneity in Group 3/4 medulloblastoma, including the presence of clear single-gene oncogenic drivers in only a subset of cases, whereas in the majority of cases, large-scale copy-number aberrations prevail. However, intratumoral heterogeneity, the role of oncogene aberrations, and broad CNVs in tumor evolution and treatment resistance remain poorly understood. To dissect this interplay, we used single-cell technologies (snRNA-seq, snATAC-seq, spatial transcriptomics) on a cohort of Group 3/4 medulloblastoma with known alterations in the oncogenes MYC, MYCN, and PRDM6. We show that large-scale chromosomal aberrations are early tumor initiating events, while the single-gene oncogenic events arise late and are typically sub-clonal, but MYC can become clonal upon disease progression to drive further tumor development and therapy resistance. We identify that the subclones are mostly interspersed across tumor tissue using spatial transcriptomics, but clear segregation is also present. Using a population genetics model, we estimate medulloblastoma initiation in the cerebellar unipolar brush cell-lineage starting from the first gestational trimester. Our findings demonstrate how single-cell technologies can be applied for early detection and diagnosis of this fatal disease.\nCoRAL accurately resolves extrachromosomal DNA genomestructures with long-read sequencing\nAuthors: Zhu, K.; Jones, M. G.; Luebeck, J.; Bu, X.; Yi, H.; Hung, K. L.; Wong, I. T.-L.; Zhang, S.; Mischel, P. S.; Chang, H. Y.; Bafna, V.\nScore: 6.4, Published: 2024-02-16 DOI: 10.1101/2024.02.15.580594\nExtrachromosomal DNA (ecDNA) is a central mechanism for focal oncogene amplification in cancer, occurring in approximately 15% of early stage cancers and 30% of late-stage cancers. EcDNAs drive tumor formation, evolution, and drug resistance by dynamically modulating oncogene copy-number and rewiring gene-regulatory networks. Elucidating the genomic architecture of ecDNA amplifications is critical for understanding tumor pathology and developing more effective therapies. Paired-end short-read (Illumina) sequencing and mapping have been utilized to represent ecDNA amplifications using a breakpoint graph, where the inferred architecture of ecDNA is encoded as a cycle in the graph. Traversals of breakpoint graph have been used to successfully predict ecDNA presence in cancer samples. However, short-read technologies are intrinsically limited in the identification of breakpoints, phasing together of complex rearrangements and internal duplications, and deconvolution of cell-to-cell heterogeneity of ecDNA structures. Long-read technologies, such as from Oxford Nanopore Technologies, have the potential to improve inference as the longer reads are better at mapping structural variants and are more likely to span rearranged or duplicated regions. Here, we propose CoRAL (Complete Reconstruction of Amplifications with Long reads), for reconstructing ecDNA architectures using long-read data. CoRAL reconstructs likely cyclic architectures using quadratic programming that simultaneously optimizes parsimony of reconstruction, explained copy number, and consistency of long-read mapping. CoRAL substantially improves reconstructions in extensive simulations and 9 datasets from previously-characterized cell-lines as compared to previous short-read-based tools. As long-read usage becomes wide-spread, we anticipate that CoRAL will be a valuable tool for profiling the landscape and evolution of focal amplifications in tumors.\nAlternative splice variants of the mitochondrial fission protein DNM1L/Drp1 regulate mitochondrial dynamics and cell fate in ovarian cancer.\nAuthors: Javed, Z.; Shin, D. H.; Pan, W.; White, S. R.; Kim, Y. S.; Elhaw, A. T.; Kamlapurkar, S.; Cheng, Y.-Y.; Benson, J. C.; Abdelnaby, A. E.; Phaeton, R.; Wang, H.-G.; Yang, S.; Sullivan, M. L. G.; St. Croix, C.; Watkins, S. C.; Mullett, S. J.; Gelhaus, S. L.; Lee, N. Y.; Coffman, L.; Aird, K. M.; Trebak, M.; Karthikeyan, M.; Walter, V.; Hempel, N.\nScore: 6.4, Published: 2024-01-24 DOI: 10.1101/2023.09.20.558501\nAberrant mitochondrial fission/fusion dynamics have been reported in cancer cells. While post translational modifications are known regulators of the mitochondrial fission/fusion machinery, we show that alternative splice variants of the fission protein Drp1 (DNM1L) have specific and unique roles in cancer, adding to the complexity of mitochondrial fission/fusion regulation in tumor cells. Ovarian cancer specimens express an alternative splice transcript variant of Drp1 lacking exon 16 of the variable domain, and high expression of this splice variant relative to other transcripts is associated with poor patient outcome. Unlike the full-length variant, expression of Drp1 lacking exon 16 leads to decreased association of Drp1 to mitochondrial fission sites, more fused mitochondrial networks, enhanced respiration, and TCA cycle metabolites, and is associated with a more metastatic phenotype in vitro and in vivo. These pro-tumorigenic effects can also be inhibited by specific siRNA-mediated inhibition of the endogenously expressed transcript lacking exon 16. Moreover, lack of exon 16 abrogates mitochondrial fission in response to pro-apoptotic stimuli and leads to decreased sensitivity to chemotherapeutics. These data emphasize the significance of the pathophysiological consequences of Drp1 alternative splicing and divergent functions of Drp1 splice variants, and strongly warrant consideration of Drp1 splicing in future studies.\nDisrupting CD38-driven T cell dysfunction restores sensitivity to cancer immunotherapy\nAuthors: Revach, O.-Y.; Cicerchia, A. M.; Shorer, O.; Petrova, B.; Anderson, S.; Park, J.; Chen, L.; Mehta, A.; Wright, S. J.; McNamee, N.; Tal-Mason, A.; Cattaneo, G.; Tiwari, P.; Xie, H.; Sweere, J. M.; Cheng, L.-C.; Sigal, N.; Enrico, E.; Miljkovic, M.; Evans, S. A.; Nguyen, N.; Whidden, M. E.; Srinivasan, R.; Spitzer, M. H.; Sun, Y.; Sharova, T.; Lawless, A.; Michaud, W. A.; Rasmussen, M. Q.; Fang, J.; Palin, C.; Chen, F.; Wang, X.; Ferrone, C. R.; Lawrence, D. P.; Sullivan, R. J.; Liu, D.; Sachdeva, U. M.; Sen, D. R.; Flaherty, K. T.; Manguso, R. T.; Bod, L.; Kellis, M.; Boland, G. M.; Yizhak, K.;\nScore: 4.8, Published: 2024-02-17 DOI: 10.1101/2024.02.12.579184\nA central problem in cancer immunotherapy with immune checkpoint blockade (ICB) is the development of resistance, which affects 50% of patients with metastatic melanoma1,2. T cell exhaustion, resulting from chronic antigen exposure in the tumour microenvironment, is a major driver of ICB resistance3. Here, we show that CD38, an ecto-enzyme involved in nicotinamide adenine dinucleotide (NAD+) catabolism, is highly expressed in exhausted CD8+ T cells in melanoma and is associated with ICB resistance. Tumour-derived CD38hiCD8+ T cells are dysfunctional, characterised by impaired proliferative capacity, effector function, and dysregulated mitochondrial bioenergetics. Genetic and pharmacological blockade of CD38 in murine and patient-derived organotypic tumour models (MDOTS/PDOTS) enhanced tumour immunity and overcame ICB resistance. Mechanistically, disrupting CD38 activity in T cells restored cellular NAD+ pools, improved mitochondrial function, increased proliferation, augmented effector function, and restored ICB sensitivity. Taken together, these data demonstrate a role for the CD38-NAD+ axis in promoting T cell exhaustion and ICB resistance and establish the efficacy of CD38 directed therapeutic strategies to overcome ICB resistance using clinically relevant, patient-derived 3D tumour models.\nCoordinated Immune Cell Networks in the Bone Marrow Microenvironment Define the Graft versus Leukemia Response with Adoptive Cellular Therapy\nAuthors: Maurer, K.; Park, C.; Mani, S.; Borji, M.; Penter, L.; Jin, Y.; Zhang, J. Y.; Shin, C.; Brenner, J. R.; Southard, J.; Krishna, S.; Lu, W.; Lyu, H.; Abbondanza, D.; Mangum, C.; Olsen, L. R.; Neuberg, D. S.; Bachireddy, P.; Farhi, S. L.; Li, S.; Livak, K. J.; Ritz, J.; Soiffer, R. J.; Wu, C. J.; Azizi, E.\nScore: 4.8, Published: 2024-02-12 DOI: 10.1101/2024.02.09.579677\nUnderstanding how intra-tumoral immune populations coordinate to generate anti-tumor responses following therapy can guide precise treatment prioritization. We performed systematic dissection of an established adoptive cellular therapy, donor lymphocyte infusion (DLI), by analyzing 348,905 single-cell transcriptomes from 74 longitudinal bone-marrow samples of 25 patients with relapsed myeloid leukemia; a subset was evaluated by protein-based spatial analysis. In acute myelogenous leukemia (AML) responders, diverse immune cell types within the bone-marrow microenvironment (BME) were predicted to interact with a clonally expanded population of ZNF683+GZMB+ CD8+ cytotoxic T lymphocytes (CTLs) which demonstrated in vitro specificity for autologous leukemia. This population, originating predominantly from the DLI product, expanded concurrently with NK and B cells. AML nonresponder BME revealed a paucity of crosstalk and elevated TIGIT expression in CD8+ CTLs. Our study highlights recipient BME differences as a key determinant of effective anti-leukemia response and opens new opportunities to modulate cell-based leukemia-directed therapy.\nMicroenvironment Shapes Cell State, Plasticity, and Heterogeneity of Small Cell Lung Cancer\nAuthors: Desai, P. A.; Takahashi, N.; Kumar, R.; Nichols, S.; Malin, J.; Hunt, A.; Schultz, C.; Cao, Y.; Tillo, D.; Nousome, D.; Chauhan, L.; Sciuto, L.; Jordan, K.; Rajapakse, V.; Lissa, D.; Zhang, Y.; Kumar, S.; Pongor, L.; Singh, A.; Schroeder, B.; Sharma, A.; Chang, T.; Vilimas, R.; Pinkiert, D.; Graham, C.; Butcher, D.; Warmer, A.; Sebastian, R.; Mahon, M.; Baker, K.; Cheng, J.; Berger, A.; Lake, R.; Abel, M.; Krishnamurthy, M.; Chrisaffis, G.; Upadhyay, D.; Fitzgerald, P.; Nirula, M.; Goyal, S.; Atkinson, D.; Bateman, N. W.; Abulez, T.; Nair, G.; Apolo, A.; Guha, U.; Baktiar, K.; El Meskini, R.; Score: 4.7, Published: 2024-02-14 DOI: 10.1101/2024.02.09.579028\nSmall-cell lung cancer (SCLC) is the most fatal form of lung cancer. Intra-tumoral heterogeneity, marked by neuroendocrine (NE) and non-neuroendocrine (non-NE) cell states, defines SCLC, but the drivers of SCLC plasticity are poorly understood. To map the landscape of SCLC tumor microenvironment (TME), we applied spatially resolved transcriptomics and quantitative mass spectrometry-based proteomics to metastatic SCLC tumors obtained via rapid autopsy. The phenotype and overall composition of non-malignant cells in the tumor microenvironment (TME) exhibits substantial variability, closely mirroring the tumor phenotype, suggesting TME-driven reprogramming of NE cell states. We identify cancer-associated fibroblasts (CAF) as a crucial element of SCLC TME heterogeneity, contributing to immune exclusion, and predicting an exceptionally poor prognosis. Together, our work provides the first comprehensive map of SCLC tumor and TME ecosystems, emphasizing their pivotal role in SCLCs adaptable nature, opening possibilities for re-programming the intercellular communications that shape SCLC tumor states. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC=\"FIGDIR/small/579028v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (74K): org.highwire.dtl.DTLVardef@dd489eorg.highwire.dtl.DTLVardef@1ca8ce6org.highwire.dtl.DTLVardef@e879fdorg.highwire.dtl.DTLVardef@a0f670_HPS_FORMAT_FIGEXP M_FIG C_FIG\nAtox1-Cyclin D1 Loop Activity is Critical for Survival of Tumors with Inactivated TP53\nAuthors: Kuchur, O. A.; Pogodaeva, S. S.; Scherbakova, A. V.; Tsymbal, S. A.\nScore: 3.5, Published: 2024-02-16 DOI: 10.1101/2024.02.14.580254\nThe search for relevant molecular targets is one of the main tasks of modern tumor chemotherapy. To successfully achieve this, it is necessary to have the most complete understanding of the functioning of the cell's transcriptional apparatus, particularly related to proliferation. The p53 protein plays an important role in regulating processes such as apoptosis, repair, and cell division, and the loss of its functionality often accompanies various types of tumors and contributes to the development of chemoresistance. Additionally, the proliferative activity of tumor cells is closely related to the metabolism of transition metals. For example, the metallochaperone Atox1, a copper transporter protein, acts as a transcription activator for cyclin D1, promoting progression through the G1/S phase of the cell cycle. On the other hand, p53 suppresses cyclin D1 at the transcriptional level, thereby these proteins have divergent effects on cell cycle progression. However, the contribution of the interaction between these proteins to cell survival is poorly understood. This work demonstrates that there not only exists a positive feedback loop between Atox1 and cyclin D1, but also that the activity of this loop depends on the status of the TP53 gene. Upon inactivation of TP53 in A549 and HepG2 cell lines, the expression of ATOX1 and CCND1 genes is enhanced, and their suppression in these cells leads to pronounced apoptosis. This fundamental observation may be useful in selecting more precise interventions for combined therapy of p53-negative tumors.\nR-Loop Functions in Brca1-Associated Mammary Tumorigenesis\nAuthors: Chiang, H.-C.; Qi, L.; Mitra, P.; Hu, Y.; Li, R.\nScore: 3.1, Published: 2024-02-16 DOI: 10.1101/2024.02.14.580374\nExcessive R-loops, a DNA-RNA hybrid structure, are associated with genome instability and BRCA1 mutation-related breast cancer. Yet the causality of R-loops in tumorigenesis remains unclear. Here we show that R-loop removal by Rnaseh1 overexpression (Rh1-OE) in Brca1-knockout (BKO) mouse mammary epithelium exacerbates DNA replication stress without affecting homology-directed DNA repair. R-loop removal also diminishes luminal progenitors, the cell of origin for estrogen receptor (ER)-negative BKO tumors. However, R-loop reduction does not dampen spontaneous BKO tumor incidence. Rather, it gives rise to a significant percentage of ER-expressing BKO tumors. Thus, R-loops reshape mammary tumor subtype rather than promoting tumorigenesis.\n",
  "wordCount" : "2571",
  "inLanguage": "en",
  "datePublished": "2024-02-18T10:38:00Z",
  "dateModified": "2024-02-18T10:38:00Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 18, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579680">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579680" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579680">
        <p class="paperTitle">Gene regulatory network landscape of Group 3/4 medulloblastoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579680" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579680" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Joshi, P.; Stelzer, T.; Okonechnikov, K.; Sarropoulos, I.; Sepp, M.; Pour-Jamnani, M. V.; Rademacher, A.; Yamada-Saito, T.; Schneider, C.; Schmidt, J.; Schaefer, P.; Leiss, K.; Bortolomeazzi, M.; Mallm, J.-P.; da Silva, P. B. G.; Statz, B.; Wittmann, A.; Schramm, K.; Blattner-Johnson, M.; Fiesel, P.; Jones, B.; Milde, T.; Pajtler, K.; van Tilburg, C. M.; Witt, O.; Rippe, K.; Korshunov, A.; Jones, D. T. W.; Hovestadt, V.; Northcott, P. A.; Thongjuea, S.; Jaeger, N.; Kaessmann, H.; Pfister, S. M.; Kutscher, L. M.</p>
        <p class="info">Score: 18.6, Published: 2024-02-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579680' target='https://doi.org/10.1101/2024.02.09.579680'> 10.1101/2024.02.09.579680</a></p>
        <p class="abstract">Resolving the molecular mechanisms driving childhood brain tumors will uncover tumor-specific vulnerabilities and advance mechanism-of-action-based therapies. Here we describe a continuum of cell-states in Group 3/4 medulloblastomas, the most frequent and fatal cerebellar embryonal tumor subgroups, based on the differential activity of transcription-factor-driven gene networks derived using a comprehensive single-nucleus multi-omic medulloblastoma atlas. We show that Group 3/4 tumor diversity stems from enriched cell-states along four molecular identity axes: photoreceptor, MYC, precursor, and unipolar brush cell-like. We identified a potential role of PAX6 in driving dual Group 3- and Group 4-like tumor trajectories in subtype VII tumors. Our study demonstrates how oncogenic events together with lineage determinants drive Group 3/4 tumor identity away from their original source in the cerebellar unipolar brush cell lineage.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.579956">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.579956" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.579956">
        <p class="paperTitle">Epigenome and early selection determine the tumour-immune evolutionary trajectory of colorectal cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.579956" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.579956" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lakatos, E.; Gunasri, V.; Zapata, L.; Househam, J.; Heide, T.; Trahearn, N.; Swinyard, O.; Cisneros, L.; Lynn, C.; Mossner, M.; Kimberley, C.; Spiteri, I.; Cresswell, G. D.; Llibre-Palomar, G.; Mitchison, M.; Maley, C. C.; Jansen, M.; Rodriguez-Justo, M.; Bridgewater, J.; Baker, A.-M.; Sottoriva, A.; Graham, T. A.</p>
        <p class="info">Score: 9.1, Published: 2024-02-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.579956' target='https://doi.org/10.1101/2024.02.12.579956'> 10.1101/2024.02.12.579956</a></p>
        <p class="abstract">Immune system control is a major hurdle that cancer evolution must circumvent. The relative timing and evolutionary dynamics of subclones that have escaped immune control remain incompletely characterized, and how immune-mediated selection shapes the epigenome has received little attention. Here, we infer the genome- and epigenome-driven evolutionary dynamics of tumour-immune coevolution within primary colorectal cancers (CRCs). We utilise our existing CRC multi-region multi-omic dataset that we supplement with high-resolution spatially-resolved neoantigen sequencing data and highly multiplexed imaging of the tumour microenvironment (TME). Analysis of somatic chromatin accessibility alterations (SCAAs) reveals frequent somatic loss of accessibility at antigen presenting genes, and that SCAAs contribute to silencing of neoantigens. We observe that strong immune escape and exclusion occur at the outset of CRC formation, and that within tumours, including at the microscopic level of individual tumour glands, additional immune escape alterations have negligible consequences for the immunophenotype of cancer cells. Further minor immuno-editing occurs during local invasion and is associated with TME reorganisation, but that evolutionary bottleneck is relatively weak. Collectively, we show that immune evasion in CRC follows a &#34;Big Bang&#34; evolutionary pattern, whereby genetic, epigenetic and TME-driven immune evasion acquired by the time of transformation defines subsequent cancer-immune evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579690">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579690" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579690">
        <p class="paperTitle">Medulloblastoma oncogene aberrations are not involved in tumor initiation, but essential for disease progression and therapy resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579690" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579690" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Okonechnikov, K.; Joshi, P.; Koerber, V.; Rademacher, A.; Bortolomeazzi, M.; Mallm, J.-P.; da Silva, P. B. G.; Statz, B.; Sepp, M.; Sarropoulos, I.; Yamada-Saito, T.; Vaillant, J.; Wittmann, A.; Schramm, K.; Blattner-Johnson, M.; Fiesel, P.; Jones, B.; Milde, T.; Pajtler, K.; van Tilburg, C. M.; Witt, O.; Bochennek, K.; Weber, K. J.; Nonnenmacher, L.; Reimann, C.; Schueller, U.; Mynarek, M.; Rutkowski, S.; Jones, D. T. W.; Korshunov, A.; Rippe, K.; Westermann, F.; Thongjuea, S.; Hoefer, T.; Kaessmann, H.; Kutscher, L. M.; Pfister, S. M.</p>
        <p class="info">Score: 7.9, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579690' target='https://doi.org/10.1101/2024.02.09.579690'> 10.1101/2024.02.09.579690</a></p>
        <p class="abstract">Despite recent advances in understanding disease biology, treatment of Group 3/4 medulloblastoma remains a therapeutic challenge in pediatric neuro-oncology. Bulk-omics approaches have identified considerable intertumoral heterogeneity in Group 3/4 medulloblastoma, including the presence of clear single-gene oncogenic drivers in only a subset of cases, whereas in the majority of cases, large-scale copy-number aberrations prevail. However, intratumoral heterogeneity, the role of oncogene aberrations, and broad CNVs in tumor evolution and treatment resistance remain poorly understood. To dissect this interplay, we used single-cell technologies (snRNA-seq, snATAC-seq, spatial transcriptomics) on a cohort of Group 3/4 medulloblastoma with known alterations in the oncogenes MYC, MYCN, and PRDM6. We show that large-scale chromosomal aberrations are early tumor initiating events, while the single-gene oncogenic events arise late and are typically sub-clonal, but MYC can become clonal upon disease progression to drive further tumor development and therapy resistance. We identify that the subclones are mostly interspersed across tumor tissue using spatial transcriptomics, but clear segregation is also present. Using a population genetics model, we estimate medulloblastoma initiation in the cerebellar unipolar brush cell-lineage starting from the first gestational trimester. Our findings demonstrate how single-cell technologies can be applied for early detection and diagnosis of this fatal disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.15.580594">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.15.580594" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.15.580594">
        <p class="paperTitle">CoRAL accurately resolves extrachromosomal DNA genomestructures with long-read sequencing</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.15.580594" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.15.580594" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zhu, K.; Jones, M. G.; Luebeck, J.; Bu, X.; Yi, H.; Hung, K. L.; Wong, I. T.-L.; Zhang, S.; Mischel, P. S.; Chang, H. Y.; Bafna, V.</p>
        <p class="info">Score: 6.4, Published: 2024-02-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.15.580594' target='https://doi.org/10.1101/2024.02.15.580594'> 10.1101/2024.02.15.580594</a></p>
        <p class="abstract">Extrachromosomal DNA (ecDNA) is a central mechanism for focal oncogene amplification in cancer, occurring in approximately 15% of early stage cancers and 30% of late-stage cancers. EcDNAs drive tumor formation, evolution, and drug resistance by dynamically modulating oncogene copy-number and rewiring gene-regulatory networks. Elucidating the genomic architecture of ecDNA amplifications is critical for understanding tumor pathology and developing more effective therapies. Paired-end short-read (Illumina) sequencing and mapping have been utilized to represent ecDNA amplifications using a breakpoint graph, where the inferred architecture of ecDNA is encoded as a cycle in the graph. Traversals of breakpoint graph have been used to successfully predict ecDNA presence in cancer samples. However, short-read technologies are intrinsically limited in the identification of breakpoints, phasing together of complex rearrangements and internal duplications, and deconvolution of cell-to-cell heterogeneity of ecDNA structures. Long-read technologies, such as from Oxford Nanopore Technologies, have the potential to improve inference as the longer reads are better at mapping structural variants and are more likely to span rearranged or duplicated regions. Here, we propose CoRAL (Complete Reconstruction of Amplifications with Long reads), for reconstructing ecDNA architectures using long-read data. CoRAL reconstructs likely cyclic architectures using quadratic programming that simultaneously optimizes parsimony of reconstruction, explained copy number, and consistency of long-read mapping. CoRAL substantially improves reconstructions in extensive simulations and 9 datasets from previously-characterized cell-lines as compared to previous short-read-based tools. As long-read usage becomes wide-spread, we anticipate that CoRAL will be a valuable tool for profiling the landscape and evolution of focal amplifications in tumors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.20.558501">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.20.558501" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.20.558501">
        <p class="paperTitle">Alternative splice variants of the mitochondrial fission protein DNM1L/Drp1 regulate mitochondrial dynamics and cell fate in ovarian cancer.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.20.558501" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.20.558501" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Javed, Z.; Shin, D. H.; Pan, W.; White, S. R.; Kim, Y. S.; Elhaw, A. T.; Kamlapurkar, S.; Cheng, Y.-Y.; Benson, J. C.; Abdelnaby, A. E.; Phaeton, R.; Wang, H.-G.; Yang, S.; Sullivan, M. L. G.; St. Croix, C.; Watkins, S. C.; Mullett, S. J.; Gelhaus, S. L.; Lee, N. Y.; Coffman, L.; Aird, K. M.; Trebak, M.; Karthikeyan, M.; Walter, V.; Hempel, N.</p>
        <p class="info">Score: 6.4, Published: 2024-01-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.20.558501' target='https://doi.org/10.1101/2023.09.20.558501'> 10.1101/2023.09.20.558501</a></p>
        <p class="abstract">Aberrant mitochondrial fission/fusion dynamics have been reported in cancer cells. While post translational modifications are known regulators of the mitochondrial fission/fusion machinery, we show that alternative splice variants of the fission protein Drp1 (DNM1L) have specific and unique roles in cancer, adding to the complexity of mitochondrial fission/fusion regulation in tumor cells. Ovarian cancer specimens express an alternative splice transcript variant of Drp1 lacking exon 16 of the variable domain, and high expression of this splice variant relative to other transcripts is associated with poor patient outcome. Unlike the full-length variant, expression of Drp1 lacking exon 16 leads to decreased association of Drp1 to mitochondrial fission sites, more fused mitochondrial networks, enhanced respiration, and TCA cycle metabolites, and is associated with a more metastatic phenotype in vitro and in vivo. These pro-tumorigenic effects can also be inhibited by specific siRNA-mediated inhibition of the endogenously expressed transcript lacking exon 16. Moreover, lack of exon 16 abrogates mitochondrial fission in response to pro-apoptotic stimuli and leads to decreased sensitivity to chemotherapeutics. These data emphasize the significance of the pathophysiological consequences of Drp1 alternative splicing and divergent functions of Drp1 splice variants, and strongly warrant consideration of Drp1 splicing in future studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.12.579184">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.12.579184" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.12.579184">
        <p class="paperTitle">Disrupting CD38-driven T cell dysfunction restores sensitivity to cancer immunotherapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.12.579184" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.12.579184" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Revach, O.-Y.; Cicerchia, A. M.; Shorer, O.; Petrova, B.; Anderson, S.; Park, J.; Chen, L.; Mehta, A.; Wright, S. J.; McNamee, N.; Tal-Mason, A.; Cattaneo, G.; Tiwari, P.; Xie, H.; Sweere, J. M.; Cheng, L.-C.; Sigal, N.; Enrico, E.; Miljkovic, M.; Evans, S. A.; Nguyen, N.; Whidden, M. E.; Srinivasan, R.; Spitzer, M. H.; Sun, Y.; Sharova, T.; Lawless, A.; Michaud, W. A.; Rasmussen, M. Q.; Fang, J.; Palin, C.; Chen, F.; Wang, X.; Ferrone, C. R.; Lawrence, D. P.; Sullivan, R. J.; Liu, D.; Sachdeva, U. M.; Sen, D. R.; Flaherty, K. T.; Manguso, R. T.; Bod, L.; Kellis, M.; Boland, G. M.; Yizhak, K.;</p>
        <p class="info">Score: 4.8, Published: 2024-02-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.12.579184' target='https://doi.org/10.1101/2024.02.12.579184'> 10.1101/2024.02.12.579184</a></p>
        <p class="abstract">A central problem in cancer immunotherapy with immune checkpoint blockade (ICB) is the development of resistance, which affects 50% of patients with metastatic melanoma1,2. T cell exhaustion, resulting from chronic antigen exposure in the tumour microenvironment, is a major driver of ICB resistance3. Here, we show that CD38, an ecto-enzyme involved in nicotinamide adenine dinucleotide (NAD&#43;) catabolism, is highly expressed in exhausted CD8&#43; T cells in melanoma and is associated with ICB resistance. Tumour-derived CD38hiCD8&#43; T cells are dysfunctional, characterised by impaired proliferative capacity, effector function, and dysregulated mitochondrial bioenergetics. Genetic and pharmacological blockade of CD38 in murine and patient-derived organotypic tumour models (MDOTS/PDOTS) enhanced tumour immunity and overcame ICB resistance. Mechanistically, disrupting CD38 activity in T cells restored cellular NAD&#43; pools, improved mitochondrial function, increased proliferation, augmented effector function, and restored ICB sensitivity. Taken together, these data demonstrate a role for the CD38-NAD&#43; axis in promoting T cell exhaustion and ICB resistance and establish the efficacy of CD38 directed therapeutic strategies to overcome ICB resistance using clinically relevant, patient-derived 3D tumour models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579677">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579677" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579677">
        <p class="paperTitle">Coordinated Immune Cell Networks in the Bone Marrow Microenvironment Define the Graft versus Leukemia Response with Adoptive Cellular Therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579677" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579677" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Maurer, K.; Park, C.; Mani, S.; Borji, M.; Penter, L.; Jin, Y.; Zhang, J. Y.; Shin, C.; Brenner, J. R.; Southard, J.; Krishna, S.; Lu, W.; Lyu, H.; Abbondanza, D.; Mangum, C.; Olsen, L. R.; Neuberg, D. S.; Bachireddy, P.; Farhi, S. L.; Li, S.; Livak, K. J.; Ritz, J.; Soiffer, R. J.; Wu, C. J.; Azizi, E.</p>
        <p class="info">Score: 4.8, Published: 2024-02-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579677' target='https://doi.org/10.1101/2024.02.09.579677'> 10.1101/2024.02.09.579677</a></p>
        <p class="abstract">Understanding how intra-tumoral immune populations coordinate to generate anti-tumor responses following therapy can guide precise treatment prioritization. We performed systematic dissection of an established adoptive cellular therapy, donor lymphocyte infusion (DLI), by analyzing 348,905 single-cell transcriptomes from 74 longitudinal bone-marrow samples of 25 patients with relapsed myeloid leukemia; a subset was evaluated by protein-based spatial analysis. In acute myelogenous leukemia (AML) responders, diverse immune cell types within the bone-marrow microenvironment (BME) were predicted to interact with a clonally expanded population of ZNF683&#43;GZMB&#43; CD8&#43; cytotoxic T lymphocytes (CTLs) which demonstrated in vitro specificity for autologous leukemia. This population, originating predominantly from the DLI product, expanded concurrently with NK and B cells. AML nonresponder BME revealed a paucity of crosstalk and elevated TIGIT expression in CD8&#43; CTLs. Our study highlights recipient BME differences as a key determinant of effective anti-leukemia response and opens new opportunities to modulate cell-based leukemia-directed therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.09.579028">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.09.579028" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.09.579028">
        <p class="paperTitle">Microenvironment Shapes Cell State, Plasticity, and Heterogeneity of Small Cell Lung Cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.09.579028" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.09.579028" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Desai, P. A.; Takahashi, N.; Kumar, R.; Nichols, S.; Malin, J.; Hunt, A.; Schultz, C.; Cao, Y.; Tillo, D.; Nousome, D.; Chauhan, L.; Sciuto, L.; Jordan, K.; Rajapakse, V.; Lissa, D.; Zhang, Y.; Kumar, S.; Pongor, L.; Singh, A.; Schroeder, B.; Sharma, A.; Chang, T.; Vilimas, R.; Pinkiert, D.; Graham, C.; Butcher, D.; Warmer, A.; Sebastian, R.; Mahon, M.; Baker, K.; Cheng, J.; Berger, A.; Lake, R.; Abel, M.; Krishnamurthy, M.; Chrisaffis, G.; Upadhyay, D.; Fitzgerald, P.; Nirula, M.; Goyal, S.; Atkinson, D.; Bateman, N. W.; Abulez, T.; Nair, G.; Apolo, A.; Guha, U.; Baktiar, K.; El Meskini, R.; </p>
        <p class="info">Score: 4.7, Published: 2024-02-14 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.09.579028' target='https://doi.org/10.1101/2024.02.09.579028'> 10.1101/2024.02.09.579028</a></p>
        <p class="abstract">Small-cell lung cancer (SCLC) is the most fatal form of lung cancer. Intra-tumoral heterogeneity, marked by neuroendocrine (NE) and non-neuroendocrine (non-NE) cell states, defines SCLC, but the drivers of SCLC plasticity are poorly understood. To map the landscape of SCLC tumor microenvironment (TME), we applied spatially resolved transcriptomics and quantitative mass spectrometry-based proteomics to metastatic SCLC tumors obtained via rapid autopsy. The phenotype and overall composition of non-malignant cells in the tumor microenvironment (TME) exhibits substantial variability, closely mirroring the tumor phenotype, suggesting TME-driven reprogramming of NE cell states. We identify cancer-associated fibroblasts (CAF) as a crucial element of SCLC TME heterogeneity, contributing to immune exclusion, and predicting an exceptionally poor prognosis. Together, our work provides the first comprehensive map of SCLC tumor and TME ecosystems, emphasizing their pivotal role in SCLCs adaptable nature, opening possibilities for re-programming the intercellular communications that shape SCLC tumor states.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=200 SRC=&#34;FIGDIR/small/579028v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (74K):
org.highwire.dtl.DTLVardef@dd489eorg.highwire.dtl.DTLVardef@1ca8ce6org.highwire.dtl.DTLVardef@e879fdorg.highwire.dtl.DTLVardef@a0f670_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.580254">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.580254" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.580254">
        <p class="paperTitle">Atox1-Cyclin D1 Loop Activity is Critical for Survival of Tumors with Inactivated TP53</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.580254" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.580254" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kuchur, O. A.; Pogodaeva, S. S.; Scherbakova, A. V.; Tsymbal, S. A.</p>
        <p class="info">Score: 3.5, Published: 2024-02-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.580254' target='https://doi.org/10.1101/2024.02.14.580254'> 10.1101/2024.02.14.580254</a></p>
        <p class="abstract">The search for relevant molecular targets is one of the main tasks of modern tumor chemotherapy. To successfully achieve this, it is necessary to have the most complete understanding of the functioning of the cell&#39;s transcriptional apparatus, particularly related to proliferation. The p53 protein plays an important role in regulating processes such as apoptosis, repair, and cell division, and the loss of its functionality often accompanies various types of tumors and contributes to the development of chemoresistance. Additionally, the proliferative activity of tumor cells is closely related to the metabolism of transition metals. For example, the metallochaperone Atox1, a copper transporter protein, acts as a transcription activator for cyclin D1, promoting progression through the G1/S phase of the cell cycle. On the other hand, p53 suppresses cyclin D1 at the transcriptional level, thereby these proteins have divergent effects on cell cycle progression. However, the contribution of the interaction between these proteins to cell survival is poorly understood. This work demonstrates that there not only exists a positive feedback loop between Atox1 and cyclin D1, but also that the activity of this loop depends on the status of the TP53 gene. Upon inactivation of TP53 in A549 and HepG2 cell lines, the expression of ATOX1 and CCND1 genes is enhanced, and their suppression in these cells leads to pronounced apoptosis. This fundamental observation may be useful in selecting more precise interventions for combined therapy of p53-negative tumors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.14.580374">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.14.580374" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.14.580374">
        <p class="paperTitle">R-Loop Functions in Brca1-Associated Mammary Tumorigenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.14.580374" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.14.580374" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chiang, H.-C.; Qi, L.; Mitra, P.; Hu, Y.; Li, R.</p>
        <p class="info">Score: 3.1, Published: 2024-02-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.14.580374' target='https://doi.org/10.1101/2024.02.14.580374'> 10.1101/2024.02.14.580374</a></p>
        <p class="abstract">Excessive R-loops, a DNA-RNA hybrid structure, are associated with genome instability and BRCA1 mutation-related breast cancer. Yet the causality of R-loops in tumorigenesis remains unclear. Here we show that R-loop removal by Rnaseh1 overexpression (Rh1-OE) in Brca1-knockout (BKO) mouse mammary epithelium exacerbates DNA replication stress without affecting homology-directed DNA repair. R-loop removal also diminishes luminal progenitors, the cell of origin for estrogen receptor  (ER)-negative BKO tumors. However, R-loop reduction does not dampen spontaneous BKO tumor incidence. Rather, it gives rise to a significant percentage of ER-expressing BKO tumors. Thus, R-loops reshape mammary tumor subtype rather than promoting tumorigenesis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
