<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Progressive plasticity during colorectal cancer metastasis
Authors: Moorman, A.; Cambuli, F.; Benitez, E.; Jiang, Q.; Xie, Y.; Mahmoud, A.; Lumish, M.; Hartner, S.; Balkaran, S.; Bermeo, J.; Asawa, S.; Firat, C.; Saxena, A.; Luthra, A.; Sgambati, V.; Luckett, K.; Wu, F.; Li, Y.; Yi, Z.; Masilionis, I.; Soares, K.; Pappou, E.; Yaeger, R.; Kingham, P.; Jarnagin, W.; Paty, P.; Weiser, M. R.; Mazutis, L.; D&#39;Angelica, M.; Shia, J.; Garcia-Aguilar, J.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Progressive plasticity during colorectal cancer metastasis
Authors: Moorman, A.; Cambuli, F.; Benitez, E.; Jiang, Q.; Xie, Y.; Mahmoud, A.; Lumish, M.; Hartner, S.; Balkaran, S.; Bermeo, J.; Asawa, S.; Firat, C.; Saxena, A.; Luthra, A.; Sgambati, V.; Luckett, K.; Wu, F.; Li, Y.; Yi, Z.; Masilionis, I.; Soares, K.; Pappou, E.; Yaeger, R.; Kingham, P.; Jarnagin, W.; Paty, P.; Weiser, M. R.; Mazutis, L.; D&#39;Angelica, M.; Shia, J.; Garcia-Aguilar, J." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-23T10:37:56+00:00" />
<meta property="article:modified_time" content="2023-08-23T10:37:56+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Progressive plasticity during colorectal cancer metastasis
Authors: Moorman, A.; Cambuli, F.; Benitez, E.; Jiang, Q.; Xie, Y.; Mahmoud, A.; Lumish, M.; Hartner, S.; Balkaran, S.; Bermeo, J.; Asawa, S.; Firat, C.; Saxena, A.; Luthra, A.; Sgambati, V.; Luckett, K.; Wu, F.; Li, Y.; Yi, Z.; Masilionis, I.; Soares, K.; Pappou, E.; Yaeger, R.; Kingham, P.; Jarnagin, W.; Paty, P.; Weiser, M. R.; Mazutis, L.; D&#39;Angelica, M.; Shia, J.; Garcia-Aguilar, J."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Progressive plasticity during colorectal cancer metastasis\nAuthors: Moorman, A.; Cambuli, F.; Benitez, E.; Jiang, Q.; Xie, Y.; Mahmoud, A.; Lumish, M.; Hartner, S.; Balkaran, S.; Bermeo, J.; Asawa, S.; Firat, C.; Saxena, A.; Luthra, A.; Sgambati, V.; Luckett, K.; Wu, F.; Li, Y.; Yi, Z.; Masilionis, I.; Soares, K.; Pappou, E.; Yaeger, R.; Kingham, P.; Jarnagin, W.; Paty, P.; Weiser, M. R.; Mazutis, L.; D\u0026#39;Angelica, M.; Shia, J.; Garcia-Aguilar, J.",
  "keywords": [
    
  ],
  "articleBody": " Progressive plasticity during colorectal cancer metastasis\nAuthors: Moorman, A.; Cambuli, F.; Benitez, E.; Jiang, Q.; Xie, Y.; Mahmoud, A.; Lumish, M.; Hartner, S.; Balkaran, S.; Bermeo, J.; Asawa, S.; Firat, C.; Saxena, A.; Luthra, A.; Sgambati, V.; Luckett, K.; Wu, F.; Li, Y.; Yi, Z.; Masilionis, I.; Soares, K.; Pappou, E.; Yaeger, R.; Kingham, P.; Jarnagin, W.; Paty, P.; Weiser, M. R.; Mazutis, L.; D'Angelica, M.; Shia, J.; Garcia-Aguilar, J.; Nawy, T.; Hollman, T.; Chaligne, R.; Sanchez-Vega, F.; Sharma, R.; Pe'er, D.; Ganesh, K.\nScore: 48.6, Published: 2023-08-21 DOI: 10.1101/2023.08.18.553925\nMetastasis is the principal cause of cancer death, yet we lack an understanding of metastatic cell states, their relationship to primary tumor states, and the mechanisms by which they transition. In a cohort of biospecimen trios from same-patient normal colon, primary and metastatic colorectal cancer, we show that while primary tumors largely adopt Lgr5+ intestinal stem-like states, metastases display progressive plasticity. Loss of intestinal cell states is accompanied by reprogramming into a highly conserved fetal progenitor state, followed by non-canonical differentiation into divergent squamous and neuroendocrine-like states, which is exacerbated by chemotherapy and associated with poor patient survival. Using matched patient-derived organoids, we demonstrate that metastatic cancer cells exhibit greater cell-autonomous multilineage differentiation potential in response to microenvironment cues than their intestinal lineage-restricted primary tumor counterparts. We identify PROX1 as a stabilizer of intestinal lineage in the fetal progenitor state, whose downregulation licenses non-canonical reprogramming.\nMajor data analysis errors invalidate cancer microbiome findings\nAuthors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.\nScore: 654.8, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993\nWe re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier. Each of these problems invalidates the results, leading to the conclusion that the microbiome-based classifiers for identifying cancer presented in the study are entirely wrong. These flaws have subsequently affected more than a dozen additional published studies that used the same data and whose results are likely invalid as well.\nAdipose contributes to CXCL5 regulated immune evasion in pancreatic cancer\nAuthors: Walsh, R. M.; Ambrose, J.; Messaggio, F.; Bye, B.; Eades, A.; Jack, J.; Ruckert, M.; Olou, A.; Chalise, P.; Pei, D.; VanSaun, M.\nScore: 13.1, Published: 2023-08-16 DOI: 10.1101/2023.08.15.553432\nBackgroundCXCR1/2 inhibitors are being implemented with immunotherapies in PDAC clinical trials. Cytokines responsible for stimulating these receptors include CXCL ligands, typically secreted by activated immune cells, fibroblasts, and even adipocytes. Obesity has been linked to poor patient outcome and altered anti-tumor immunity. Adipose-derived cytokines and chemokines have been implicated as potential drivers of tumor cell immune evasion, suggesting a possibility of susceptibility to targeting specifically in the context of obesity. MethodsRNA-sequencing of human PDAC cell lines was used to assess differential influences on the cancer cell transcriptome after treatment with conditioned media from peri-pancreatic adipose tissue of lean and obese PDAC patients. The adipose-induced secretome of PDAC cells was then assessed by cytokine arrays and ELISAs. Lentiviral transduction and CRISPR-Cas9 was used to knock out CXCL5 from a murine PDAC cell line for orthotopic tumor studies in diet-induced obese, syngeneic mice. Flow cytometry was used to define the immune profiles of tumors. Anti-PD-1 immune checkpoint blockade therapy was administered to alleviate T cell exhaustion and invoke an immune response, while the mice were monitored at endpoint for differences in tumor size. ResultsThe chemokine CXCL5 was secreted in response to stimulation of PDAC cells with human adipose conditioned media (hAT-CM). PDAC CXCL5 secretion was induced by either IL-1{beta} or TNF, but neutralization of both was required to limit secretion. Ablation of CXCL5 from tumors promoted an immune phenotype susceptible to PD-1 inhibitor therapy. While application of anti-PD-1 treatment to control tumors failed to alter tumor growth, knockout CXCL5 tumors were diminished. ConclusionsIn summary, our findings show that known adipokines TNF and IL-1{beta} can stimulate CXCL5 release from PDAC cells in vitro. In vivo, CXCL5 depletion alone is sufficient to promote T cell infiltration into tumors in an obese setting, but requires checkpoint blockade inhibition to alleviate tumor burden. DATA AVAILABILITY STATEMENTRaw and processed RNAseq data will be further described in the GEO accession database (awaiting approval from GEO for PRJ number). Additional raw data is included in the supplemental material and available upon reasonable request. WHAT IS ALREADY KNOWN ON THIS TOPICObesity is linked to a worsened patient outcome and immunogenic tumor profile in PDAC. CXCR1/2 inhibitors have begun to be implemented in combination with immune checkpoint blockade therapies to promote T cell infiltration under the premise of targeting the myeloid rich TME. WHAT THIS STUDY ADDSUsing in vitro/ex vivo cell and tissue culture-based assays with in vivo mouse models we have identified that adipose derived IL-1{beta} and TNF can promote tumor secretion of CXCL5 which acts as a critical deterrent to CD8 T cell tumor infiltration, but loss of CXCL5 also leads to a more immune suppressive myeloid profile. HOW THIS STUDY MIGHT AFFECT RESEARCH, PRACTICE, OR POLICYThis study highlights a mechanism and emphasizes the efficacy of single CXCR1/2 ligand targeting that could be beneficial to overcoming tumor immune-evasion even in the obese PDAC patient population.\nMorphometry and mechanical instability at the onset of epithelial bladder cancer\nAuthors: Lampart, F. L.; Vetter, R.; Wang, Y.; Yamauchi, K. A.; Strohmeyer, N.; Meer, F.; Hussherr, M.-D.; Camenisch, G.; Seifert, H.-H.; Rentsch, C. A.; Le Magnen, C.; Mueller, D. J.; Bubendorf, L.; Iber, D.\nScore: 11.6, Published: 2023-08-19 DOI: 10.1101/2023.08.17.553533\nMalignancies of epithelial tissues, called carcinomas, account for the majority of cancer cases. Much cancer research has focused on genetic alterations and their relation to different carcinoma phenotypes. Besides a rewiring in the signalling networks, carcinoma progression is accompanied by mechanical changes in the epithelial cells and the extracellular matrix. Here, we reveal intricate morphologies in the basement membrane at the onset of bladder cancer, and propose that they emerge from a mechanical buckling instability upon epithelial overgrowth. Using a combination of microscopy imaging of the mouse and human bladder tissue, elasticity theory, and numerical simulations of differential growth in the bladder mucosa, we find that aberrant tissue morphologies can emerge through stiffness changes in the different mucosa layers. The resulting thickening, wrinkles and folds exhibit qualitative and quantitative similarity with imaged early papillary tumors and carcinomas in situ. Atomic force microscopy indeed reveals local stiffness changes in the pathological basement membrane. Our findings suggest a mechanical origin of the different carcinoma subtypes in the bladder, which have vastly different clinical prognosis. They might provide the basis for a new line of attack in medical carcinoma treatment and prophylaxis.\nIdentification of resistance mechanisms to small-molecule inhibition of TEAD-regulated transcription\nAuthors: Kulkarni, A.; Mohan, V.; Tang, T.; Post, L.; Manning, M.; Thio, N.; Parker, B. L.; Rosenbluh, J.; Vissers, J. H. A.; Harvey, K. F.\nScore: 9.6, Published: 2023-08-16 DOI: 10.1101/2023.08.16.553491\nThe Hippo tumour suppressor pathway controls transcription by regulating nuclear abundance of YAP and TAZ, which activate transcription with the TEAD1-TEAD4 DNA-binding proteins. Recently, several small-molecule inhibitors of YAP and TEADs have been reported, with some now entering clinical trials for different cancers. Here, we investigated the cellular response to TEAD palmitoylation inhibitors, using a combination of genomic and genetic strategies. Genome-wide CRISPR/Cas9 screens identified genes that modulate the cellular response to TEAD inhibition, including members of the Hippo, MAPK and JAK-STAT signaling pathways. By exploring gene expression programs of mutant cells, we found that MAPK pathway hyperactivation confers resistance to TEAD inhibition by reinstating expression of a subset of YAP/TEAD target genes. Consistent with this, combined inhibition of TEAD and the MAPK protein MEK, synergistically blocked proliferation of several mesothelioma and lung cancer cell lines and more potently reduced the growth of patient-derived lung cancers in vivo. Collectively, we reveal mechanisms by which cells can overcome small-molecule inhibition of TEADs and potential strategies to enhance the anti-tumor activity of emerging Hippo pathway targeted therapies.\nCell-specific crosstalk proteomics reveals cathepsin B signaling as a driver of glioblastoma malignancy near the subventricular zone\nAuthors: Norton, E. S.; Whaley, L. A.; Jones, V. K.; Brooks, M. M.; Russo, M. N.; Morderer, D.; Jessen, E.; Schiapparelli, P.; Ramos-Fresnedo, A.; Zarco, N.; Carrano, A.; Rossoll, W.; Asmann, Y. W.; Lam, T. T.; Chaichana, K. L.; Anastasiadis, P. Z.; Quinones-Hinojosa, A.; Guerrero-Cazares, H.\nScore: 7.7, Published: 2023-08-21 DOI: 10.1101/2023.08.19.553966\nGlioblastoma (GBM) is the most prevalent and aggressive malignant primary brain tumor. GBM proximal to the lateral ventricles (LVs) is more aggressive, potentially due to subventricular zone (SVZ) contact. Despite this, crosstalk between GBM and neural stem/progenitor cells (NSC/NPCs) is not well understood. Using cell-specific proteomics, we show that LV-proximal GBM prevents neuronal maturation of NSCs through induction of senescence. Additionally, GBM brain tumor initiating cells (BTICs) increase expression of CTSB upon interaction with NPCs. Lentiviral knockdown and recombinant protein experiments reveal both cell-intrinsic and soluble CTSB promote malignancy-associated phenotypes in BTICs. Soluble CTSB stalls neuronal maturation in NPCs while promoting senescence, providing a link between LV-tumor proximity and neurogenesis disruption. Finally, we show LV-proximal CTSB upregulation in patients, showing the relevance of this crosstalk in human GBM biology. These results demonstrate the value of proteomic analysis in tumor microenvironment research and provide direction for new therapeutic strategies in GBM.\nThe eIF4A2 negative regulator of mRNA translation promotes extracellular matrix deposition to accelerate hepatocellular carcinoma initiation\nAuthors: Moore, M.; Pardo, L.; Mitchell, L.; Schmidt, T.; May, S.; Mueller, M.; Strathdee, D.; Bryson, S.; Hodge, K.; Lilla, S.; Zanivan, S.; Waldron, J.; McGarry, L.; Peter-Durairaj, R.; Kanellos, G.; Nixon, C.; Ballantyne, F.; LeQuesne, J.; Sansom, O. J.; Bird, T.; Bushell, M.; Norman, J. C.\nScore: 7.6, Published: 2023-08-16 DOI: 10.1101/2023.08.16.553544\nIncreased protein synthesis enables proliferation of established tumors. However, how mRNA translation contributes to early tumorigenesis remains unclear. Following oncogene activation, hepatocytes enter a non-proliferative/senescent-like phase characterised by deposition of extracellular matrix (ECM) niches which then promote subsequent progression to proliferating hepatocellular carcinoma (HCC). We demonstrate that removal of eIF4A2, a negative regulator of mRNA translation, upregulates synthesis of membrane/secretory proteins. This leads to a compensatory increase of integrin degradation which, in turn, compromises generation of ECM-rich tumour initiation niches and progression to proliferating HCC. Conversely, pharmacological inhibition of mRNA translation following eIF4A2 deletion restores ECM deposition and reinstates HCC progression. Our results highlight how the way in which cells respond to dysregulated mRNA translation in early tumorigenesis influences cancer outcomes. One-Sentence SummaryEnhancing rates of secretory protein synthesis during early oncogenesis retards cancer initiation by inhibiting ECM deposition.\nColibactin-induced genotoxicity and colorectal cancer exacerbation critically depends on adhesin-mediated epithelial binding\nAuthors: Jans, M.; Kolata, M.; Blancke, G.; Ciers, M.; Dohlman, A. B.; Kusakabe, T.; Sze, M.; Thiran, A.; Berx, G.; Tejpar, S.; van Loo, G.; Iliev, I. D.; Remaut, H.; Vereecke, L.\nScore: 6.6, Published: 2023-08-18 DOI: 10.1101/2023.08.16.553526\nVarious bacteria are suggested to contribute to colorectal cancer (CRC) development, including pks+ E. coli which produce the genotoxin colibactin that induces characteristic mutational signatures in host epithelial cells. It remains unclear how the highly unstable colibactin molecule is able to access host epithelial cells and its DNA to cause harm. Using the microbiota-dependent ZEB2-transgenic mouse model of invasive CRC, we found that pks+ E. coli drives CRC exacerbation and tissue invasion in a colibactin-dependent manner. Using isogenic mutant strains, we further demonstrate that CRC exacerbation critically depends on expression of the E. coli type-1 pilus adhesin FimH and the F9-pilus adhesin FmlH. Blocking bacterial adhesion using a pharmacological FimH inhibitor attenuates colibactin-mediated genotoxicity and CRC exacerbation. Together, we show that the oncogenic potential of pks+ E. coli critically depends on bacterial adhesion to host epithelial cells and is critically mediated by specific bacterial adhesins. Adhesin-mediated epithelial binding subsequently allows production of the genotoxin colibactin in close proximity to host epithelial cells, which promotes DNA damage and drives CRC development. These findings present promising therapeutic avenues for the development of anti-adhesive therapies aiming at mitigating colibactin-induced DNA damage and inhibiting the initiation and progression of CRC, particularly in individuals at risk for developing CRC.\nEZH2 synergizes with BRD4-NUT to drive NUT carcinoma growth through silencing of key tumor suppressor genes\nAuthors: Huang, Y.; Durall, R. T.; Luong, N. M.; Hertzler, H. J.; Huang, J.; Gokhale, P. C.; Leeper, B. A.; Persky, N. S.; Root, D. E.; Anekal, P. V.; Montero Llopis, P. D.; David, C. N.; Kutok, J. L.; Raimondi, A.; Saluja, K.; Luo, J.; Zahnow, C. A.; Adane, B.; Stegmaier, K.; Hawkins, C. E.; Ponne, C.; Le, Q.; Shapiro, G. I.; Lemieux, M. E.; Eagen, K. P.; French, C. A.\nScore: 5.8, Published: 2023-08-16 DOI: 10.1101/2023.08.15.553204\nNUT carcinoma (NC) is an aggressive carcinoma driven by the BRD4-NUT fusion oncoprotein, which activates chromatin to promote expression of pro-growth genes. BET bromodomain inhibitors (BETi) impede BRD4-NUTs ability to activate genes and are thus a promising treatment but limited as monotherapy. The role of gene repression in NC is unknown. Here, we demonstrate that EZH2, which silences genes through establishment of repressive chromatin, is a dependency in NC. Inhibition of EZH2 with the clinical compound tazemetostat (taz) potently blocked growth of NC cells. Epigenetic and transcriptomic analysis revealed that taz reversed the EZH2-specific H3K27me3 silencing mark, and restored expression of multiple tumor suppressor genes while having no effect on key oncogenic BRD4- NUT-regulated genes. CDKN2A was identified as the only gene amongst all taz-derepressed genes to confer resistance to taz in a CRISPR-Cas9 screen. Combined EZH2 inhibition and BET inhibition synergized to downregulate cell proliferation genes resulting in more pronounced growth arrest and differentiation than either inhibitor alone. In pre-clinical models, combined taz and BETi synergistically blocked growth and prolonged survival of NC-xenografted mice, with all mice cured in one cohort. STATEMENT OF SIGNIFICANCEIdentification of EZH2 as a dependency in NC substantiates the reliance of NC tumor cells on epigenetic dysregulation of functionally opposite, yet highly complementary chromatin regulatory pathways to maintain NC growth. In particular, repression of CDKN2A expression by EZH2 provides a mechanistic rationale for combining EZH2i with BETi for the clinical treatment of NC.\nMathematical Modeling of Field Cancerization through the Lens of Cancer Behavioral Ecology\nAuthors: Bukkuri, A.; Adler, F.\nScore: 9.3, Published: 2023-08-08 DOI: 10.1101/2023.08.07.552382\nField cancerization is a process in which a normal tissue is replaced with pre-cancerous but histologically normal tissue. This transformed field can give rise to malignancy and contribute to tumor relapse. In this paper, we create a mathematical model of field cancerization from the perspective of cancer behavioral ecology. In our model, field cancerization arises from a breakdown in signaling integrity and control, and investigate implications for acute wounding, chronic wounding, aging, and therapeutic interventions. We find that restoration of communication networks can lead to cancer regression in the context of acute injury. Conversely, long term loss of controls, such as through chronic wounding or aging, can promote oncogenesis. These results are paralleled in therapeutic interventions: those that simply target cells in cancerous states may be less effective than those that reestablish signaling integrity. Viewing cancer as a corruption of communication systems rather than as a corruption of individual cells may lead to novel approaches for understanding and treating this disease.\n",
  "wordCount" : "2613",
  "inLanguage": "en",
  "datePublished": "2023-08-23T10:37:56Z",
  "dateModified": "2023-08-23T10:37:56Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on August 23, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.18.553925">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.18.553925" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.18.553925">
        <p class="paperTitle">Progressive plasticity during colorectal cancer metastasis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.18.553925" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.18.553925" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moorman, A.; Cambuli, F.; Benitez, E.; Jiang, Q.; Xie, Y.; Mahmoud, A.; Lumish, M.; Hartner, S.; Balkaran, S.; Bermeo, J.; Asawa, S.; Firat, C.; Saxena, A.; Luthra, A.; Sgambati, V.; Luckett, K.; Wu, F.; Li, Y.; Yi, Z.; Masilionis, I.; Soares, K.; Pappou, E.; Yaeger, R.; Kingham, P.; Jarnagin, W.; Paty, P.; Weiser, M. R.; Mazutis, L.; D&#39;Angelica, M.; Shia, J.; Garcia-Aguilar, J.; Nawy, T.; Hollman, T.; Chaligne, R.; Sanchez-Vega, F.; Sharma, R.; Pe&#39;er, D.; Ganesh, K.</p>
        <p class="info">Score: 48.6, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.18.553925' target='https://doi.org/10.1101/2023.08.18.553925'> 10.1101/2023.08.18.553925</a></p>
        <p class="abstract">Metastasis is the principal cause of cancer death, yet we lack an understanding of metastatic cell states, their relationship to primary tumor states, and the mechanisms by which they transition. In a cohort of biospecimen trios from same-patient normal colon, primary and metastatic colorectal cancer, we show that while primary tumors largely adopt Lgr5&#43; intestinal stem-like states, metastases display progressive plasticity. Loss of intestinal cell states is accompanied by reprogramming into a highly conserved fetal progenitor state, followed by non-canonical differentiation into divergent squamous and neuroendocrine-like states, which is exacerbated by chemotherapy and associated with poor patient survival. Using matched patient-derived organoids, we demonstrate that metastatic cancer cells exhibit greater cell-autonomous multilineage differentiation potential in response to microenvironment cues than their intestinal lineage-restricted primary tumor counterparts. We identify PROX1 as a stabilizer of intestinal lineage in the fetal progenitor state, whose downregulation licenses non-canonical reprogramming.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550993">
        <p class="paperTitle">Major data analysis errors invalidate cancer microbiome findings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.</p>
        <p class="info">Score: 654.8, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550993' target='https://doi.org/10.1101/2023.07.28.550993'> 10.1101/2023.07.28.550993</a></p>
        <p class="abstract">We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier. Each of these problems invalidates the results, leading to the conclusion that the microbiome-based classifiers for identifying cancer presented in the study are entirely wrong. These flaws have subsequently affected more than a dozen additional published studies that used the same data and whose results are likely invalid as well.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553432">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553432" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553432">
        <p class="paperTitle">Adipose contributes to CXCL5 regulated immune evasion in pancreatic cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553432" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553432" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Walsh, R. M.; Ambrose, J.; Messaggio, F.; Bye, B.; Eades, A.; Jack, J.; Ruckert, M.; Olou, A.; Chalise, P.; Pei, D.; VanSaun, M.</p>
        <p class="info">Score: 13.1, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553432' target='https://doi.org/10.1101/2023.08.15.553432'> 10.1101/2023.08.15.553432</a></p>
        <p class="abstract">BackgroundCXCR1/2 inhibitors are being implemented with immunotherapies in PDAC clinical trials. Cytokines responsible for stimulating these receptors include CXCL ligands, typically secreted by activated immune cells, fibroblasts, and even adipocytes. Obesity has been linked to poor patient outcome and altered anti-tumor immunity. Adipose-derived cytokines and chemokines have been implicated as potential drivers of tumor cell immune evasion, suggesting a possibility of susceptibility to targeting specifically in the context of obesity.

MethodsRNA-sequencing of human PDAC cell lines was used to assess differential influences on the cancer cell transcriptome after treatment with conditioned media from peri-pancreatic adipose tissue of lean and obese PDAC patients. The adipose-induced secretome of PDAC cells was then assessed by cytokine arrays and ELISAs. Lentiviral transduction and CRISPR-Cas9 was used to knock out CXCL5 from a murine PDAC cell line for orthotopic tumor studies in diet-induced obese, syngeneic mice. Flow cytometry was used to define the immune profiles of tumors. Anti-PD-1 immune checkpoint blockade therapy was administered to alleviate T cell exhaustion and invoke an immune response, while the mice were monitored at endpoint for differences in tumor size.

ResultsThe chemokine CXCL5 was secreted in response to stimulation of PDAC cells with human adipose conditioned media (hAT-CM). PDAC CXCL5 secretion was induced by either IL-1{beta} or TNF, but neutralization of both was required to limit secretion. Ablation of CXCL5 from tumors promoted an immune phenotype susceptible to PD-1 inhibitor therapy. While application of anti-PD-1 treatment to control tumors failed to alter tumor growth, knockout CXCL5 tumors were diminished.

ConclusionsIn summary, our findings show that known adipokines TNF and IL-1{beta} can stimulate CXCL5 release from PDAC cells in vitro. In vivo, CXCL5 depletion alone is sufficient to promote T cell infiltration into tumors in an obese setting, but requires checkpoint blockade inhibition to alleviate tumor burden.

DATA AVAILABILITY STATEMENTRaw and processed RNAseq data will be further described in the GEO accession database (awaiting approval from GEO for PRJ number). Additional raw data is included in the supplemental material and available upon reasonable request.

WHAT IS ALREADY KNOWN ON THIS TOPICObesity is linked to a worsened patient outcome and immunogenic tumor profile in PDAC. CXCR1/2 inhibitors have begun to be implemented in combination with immune checkpoint blockade therapies to promote T cell infiltration under the premise of targeting the myeloid rich TME.

WHAT THIS STUDY ADDSUsing in vitro/ex vivo cell and tissue culture-based assays with in vivo mouse models we have identified that adipose derived IL-1{beta} and TNF can promote tumor secretion of CXCL5 which acts as a critical deterrent to CD8 T cell tumor infiltration, but loss of CXCL5 also leads to a more immune suppressive myeloid profile.

HOW THIS STUDY MIGHT AFFECT RESEARCH, PRACTICE, OR POLICYThis study highlights a mechanism and emphasizes the efficacy of single CXCR1/2 ligand targeting that could be beneficial to overcoming tumor immune-evasion even in the obese PDAC patient population.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.17.553533">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.17.553533" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.17.553533">
        <p class="paperTitle">Morphometry and mechanical instability at the onset of epithelial bladder cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.17.553533" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.17.553533" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lampart, F. L.; Vetter, R.; Wang, Y.; Yamauchi, K. A.; Strohmeyer, N.; Meer, F.; Hussherr, M.-D.; Camenisch, G.; Seifert, H.-H.; Rentsch, C. A.; Le Magnen, C.; Mueller, D. J.; Bubendorf, L.; Iber, D.</p>
        <p class="info">Score: 11.6, Published: 2023-08-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.17.553533' target='https://doi.org/10.1101/2023.08.17.553533'> 10.1101/2023.08.17.553533</a></p>
        <p class="abstract">Malignancies of epithelial tissues, called carcinomas, account for the majority of cancer cases. Much cancer research has focused on genetic alterations and their relation to different carcinoma phenotypes. Besides a rewiring in the signalling networks, carcinoma progression is accompanied by mechanical changes in the epithelial cells and the extracellular matrix. Here, we reveal intricate morphologies in the basement membrane at the onset of bladder cancer, and propose that they emerge from a mechanical buckling instability upon epithelial overgrowth. Using a combination of microscopy imaging of the mouse and human bladder tissue, elasticity theory, and numerical simulations of differential growth in the bladder mucosa, we find that aberrant tissue morphologies can emerge through stiffness changes in the different mucosa layers. The resulting thickening, wrinkles and folds exhibit qualitative and quantitative similarity with imaged early papillary tumors and carcinomas in situ. Atomic force microscopy indeed reveals local stiffness changes in the pathological basement membrane. Our findings suggest a mechanical origin of the different carcinoma subtypes in the bladder, which have vastly different clinical prognosis. They might provide the basis for a new line of attack in medical carcinoma treatment and prophylaxis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553491">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553491" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553491">
        <p class="paperTitle">Identification of resistance mechanisms to small-molecule inhibition of TEAD-regulated transcription</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553491" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553491" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kulkarni, A.; Mohan, V.; Tang, T.; Post, L.; Manning, M.; Thio, N.; Parker, B. L.; Rosenbluh, J.; Vissers, J. H. A.; Harvey, K. F.</p>
        <p class="info">Score: 9.6, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553491' target='https://doi.org/10.1101/2023.08.16.553491'> 10.1101/2023.08.16.553491</a></p>
        <p class="abstract">The Hippo tumour suppressor pathway controls transcription by regulating nuclear abundance of YAP and TAZ, which activate transcription with the TEAD1-TEAD4 DNA-binding proteins. Recently, several small-molecule inhibitors of YAP and TEADs have been reported, with some now entering clinical trials for different cancers. Here, we investigated the cellular response to TEAD palmitoylation inhibitors, using a combination of genomic and genetic strategies. Genome-wide CRISPR/Cas9 screens identified genes that modulate the cellular response to TEAD inhibition, including members of the Hippo, MAPK and JAK-STAT signaling pathways. By exploring gene expression programs of mutant cells, we found that MAPK pathway hyperactivation confers resistance to TEAD inhibition by reinstating expression of a subset of YAP/TEAD target genes. Consistent with this, combined inhibition of TEAD and the MAPK protein MEK, synergistically blocked proliferation of several mesothelioma and lung cancer cell lines and more potently reduced the growth of patient-derived lung cancers in vivo. Collectively, we reveal mechanisms by which cells can overcome small-molecule inhibition of TEADs and potential strategies to enhance the anti-tumor activity of emerging Hippo pathway targeted therapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.19.553966">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.19.553966" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.19.553966">
        <p class="paperTitle">Cell-specific crosstalk proteomics reveals cathepsin B signaling as a driver of glioblastoma malignancy near the subventricular zone</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.19.553966" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.19.553966" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Norton, E. S.; Whaley, L. A.; Jones, V. K.; Brooks, M. M.; Russo, M. N.; Morderer, D.; Jessen, E.; Schiapparelli, P.; Ramos-Fresnedo, A.; Zarco, N.; Carrano, A.; Rossoll, W.; Asmann, Y. W.; Lam, T. T.; Chaichana, K. L.; Anastasiadis, P. Z.; Quinones-Hinojosa, A.; Guerrero-Cazares, H.</p>
        <p class="info">Score: 7.7, Published: 2023-08-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.19.553966' target='https://doi.org/10.1101/2023.08.19.553966'> 10.1101/2023.08.19.553966</a></p>
        <p class="abstract">Glioblastoma (GBM) is the most prevalent and aggressive malignant primary brain tumor. GBM proximal to the lateral ventricles (LVs) is more aggressive, potentially due to subventricular zone (SVZ) contact. Despite this, crosstalk between GBM and neural stem/progenitor cells (NSC/NPCs) is not well understood. Using cell-specific proteomics, we show that LV-proximal GBM prevents neuronal maturation of NSCs through induction of senescence. Additionally, GBM brain tumor initiating cells (BTICs) increase expression of CTSB upon interaction with NPCs. Lentiviral knockdown and recombinant protein experiments reveal both cell-intrinsic and soluble CTSB promote malignancy-associated phenotypes in BTICs. Soluble CTSB stalls neuronal maturation in NPCs while promoting senescence, providing a link between LV-tumor proximity and neurogenesis disruption. Finally, we show LV-proximal CTSB upregulation in patients, showing the relevance of this crosstalk in human GBM biology. These results demonstrate the value of proteomic analysis in tumor microenvironment research and provide direction for new therapeutic strategies in GBM.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553544">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553544" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553544">
        <p class="paperTitle">The eIF4A2 negative regulator of mRNA translation promotes extracellular matrix deposition to accelerate hepatocellular carcinoma initiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553544" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553544" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Moore, M.; Pardo, L.; Mitchell, L.; Schmidt, T.; May, S.; Mueller, M.; Strathdee, D.; Bryson, S.; Hodge, K.; Lilla, S.; Zanivan, S.; Waldron, J.; McGarry, L.; Peter-Durairaj, R.; Kanellos, G.; Nixon, C.; Ballantyne, F.; LeQuesne, J.; Sansom, O. J.; Bird, T.; Bushell, M.; Norman, J. C.</p>
        <p class="info">Score: 7.6, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553544' target='https://doi.org/10.1101/2023.08.16.553544'> 10.1101/2023.08.16.553544</a></p>
        <p class="abstract">Increased protein synthesis enables proliferation of established tumors. However, how mRNA translation contributes to early tumorigenesis remains unclear. Following oncogene activation, hepatocytes enter a non-proliferative/senescent-like phase characterised by deposition of extracellular matrix (ECM) niches which then promote subsequent progression to proliferating hepatocellular carcinoma (HCC). We demonstrate that removal of eIF4A2, a negative regulator of mRNA translation, upregulates synthesis of membrane/secretory proteins. This leads to a compensatory increase of integrin degradation which, in turn, compromises generation of ECM-rich tumour initiation niches and progression to proliferating HCC. Conversely, pharmacological inhibition of mRNA translation following eIF4A2 deletion restores ECM deposition and reinstates HCC progression. Our results highlight how the way in which cells respond to dysregulated mRNA translation in early tumorigenesis influences cancer outcomes.

One-Sentence SummaryEnhancing rates of secretory protein synthesis during early oncogenesis retards cancer initiation by inhibiting ECM deposition.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.16.553526">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.16.553526" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.16.553526">
        <p class="paperTitle">Colibactin-induced genotoxicity and colorectal cancer exacerbation critically depends on adhesin-mediated epithelial binding</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.16.553526" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.16.553526" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Jans, M.; Kolata, M.; Blancke, G.; Ciers, M.; Dohlman, A. B.; Kusakabe, T.; Sze, M.; Thiran, A.; Berx, G.; Tejpar, S.; van Loo, G.; Iliev, I. D.; Remaut, H.; Vereecke, L.</p>
        <p class="info">Score: 6.6, Published: 2023-08-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.16.553526' target='https://doi.org/10.1101/2023.08.16.553526'> 10.1101/2023.08.16.553526</a></p>
        <p class="abstract">Various bacteria are suggested to contribute to colorectal cancer (CRC) development, including pks&#43; E. coli which produce the genotoxin colibactin that induces characteristic mutational signatures in host epithelial cells. It remains unclear how the highly unstable colibactin molecule is able to access host epithelial cells and its DNA to cause harm. Using the microbiota-dependent ZEB2-transgenic mouse model of invasive CRC, we found that pks&#43; E. coli drives CRC exacerbation and tissue invasion in a colibactin-dependent manner. Using isogenic mutant strains, we further demonstrate that CRC exacerbation critically depends on expression of the E. coli type-1 pilus adhesin FimH and the F9-pilus adhesin FmlH. Blocking bacterial adhesion using a pharmacological FimH inhibitor attenuates colibactin-mediated genotoxicity and CRC exacerbation. Together, we show that the oncogenic potential of pks&#43; E. coli critically depends on bacterial adhesion to host epithelial cells and is critically mediated by specific bacterial adhesins. Adhesin-mediated epithelial binding subsequently allows production of the genotoxin colibactin in close proximity to host epithelial cells, which promotes DNA damage and drives CRC development. These findings present promising therapeutic avenues for the development of anti-adhesive therapies aiming at mitigating colibactin-induced DNA damage and inhibiting the initiation and progression of CRC, particularly in individuals at risk for developing CRC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.15.553204">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.15.553204" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.15.553204">
        <p class="paperTitle">EZH2 synergizes with BRD4-NUT to drive NUT carcinoma growth through silencing of key tumor suppressor genes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.15.553204" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.15.553204" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Huang, Y.; Durall, R. T.; Luong, N. M.; Hertzler, H. J.; Huang, J.; Gokhale, P. C.; Leeper, B. A.; Persky, N. S.; Root, D. E.; Anekal, P. V.; Montero Llopis, P. D.; David, C. N.; Kutok, J. L.; Raimondi, A.; Saluja, K.; Luo, J.; Zahnow, C. A.; Adane, B.; Stegmaier, K.; Hawkins, C. E.; Ponne, C.; Le, Q.; Shapiro, G. I.; Lemieux, M. E.; Eagen, K. P.; French, C. A.</p>
        <p class="info">Score: 5.8, Published: 2023-08-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.15.553204' target='https://doi.org/10.1101/2023.08.15.553204'> 10.1101/2023.08.15.553204</a></p>
        <p class="abstract">NUT carcinoma (NC) is an aggressive carcinoma driven by the BRD4-NUT fusion oncoprotein, which activates chromatin to promote expression of pro-growth genes. BET bromodomain inhibitors (BETi) impede BRD4-NUTs ability to activate genes and are thus a promising treatment but limited as monotherapy. The role of gene repression in NC is unknown. Here, we demonstrate that EZH2, which silences genes through establishment of repressive chromatin, is a dependency in NC. Inhibition of EZH2 with the clinical compound tazemetostat (taz) potently blocked growth of NC cells. Epigenetic and transcriptomic analysis revealed that taz reversed the EZH2-specific H3K27me3 silencing mark, and restored expression of multiple tumor suppressor genes while having no effect on key oncogenic BRD4- NUT-regulated genes. CDKN2A was identified as the only gene amongst all taz-derepressed genes to confer resistance to taz in a CRISPR-Cas9 screen. Combined EZH2 inhibition and BET inhibition synergized to downregulate cell proliferation genes resulting in more pronounced growth arrest and differentiation than either inhibitor alone. In pre-clinical models, combined taz and BETi synergistically blocked growth and prolonged survival of NC-xenografted mice, with all mice cured in one cohort.

STATEMENT OF SIGNIFICANCEIdentification of EZH2 as a dependency in NC substantiates the reliance of NC tumor cells on epigenetic dysregulation of functionally opposite, yet highly complementary chromatin regulatory pathways to maintain NC growth. In particular, repression of CDKN2A expression by EZH2 provides a mechanistic rationale for combining EZH2i with BETi for the clinical treatment of NC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.07.552382">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.07.552382" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.07.552382">
        <p class="paperTitle">Mathematical Modeling of Field Cancerization through the Lens of Cancer Behavioral Ecology</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.07.552382" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.07.552382" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bukkuri, A.; Adler, F.</p>
        <p class="info">Score: 9.3, Published: 2023-08-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.07.552382' target='https://doi.org/10.1101/2023.08.07.552382'> 10.1101/2023.08.07.552382</a></p>
        <p class="abstract">Field cancerization is a process in which a normal tissue is replaced with pre-cancerous but histologically normal tissue. This transformed field can give rise to malignancy and contribute to tumor relapse. In this paper, we create a mathematical model of field cancerization from the perspective of cancer behavioral ecology. In our model, field cancerization arises from a breakdown in signaling integrity and control, and investigate implications for acute wounding, chronic wounding, aging, and therapeutic interventions. We find that restoration of communication networks can lead to cancer regression in the context of acute injury. Conversely, long term loss of controls, such as through chronic wounding or aging, can promote oncogenesis.

These results are paralleled in therapeutic interventions: those that simply target cells in cancerous states may be less effective than those that reestablish signaling integrity. Viewing cancer as a corruption of communication systems rather than as a corruption of individual cells may lead to novel approaches for understanding and treating this disease.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
