<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Major data analysis errors invalidate cancer microbiome findings
Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.
Score: 361.6, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993
We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Major data analysis errors invalidate cancer microbiome findings
Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.
Score: 361.6, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993
We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-06T10:36:48+00:00" />
<meta property="article:modified_time" content="2023-08-06T10:36:48+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Major data analysis errors invalidate cancer microbiome findings
Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.
Score: 361.6, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993
We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Major data analysis errors invalidate cancer microbiome findings\nAuthors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.\nScore: 361.6, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993\nWe re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier.",
  "keywords": [
    
  ],
  "articleBody": " Major data analysis errors invalidate cancer microbiome findings\nAuthors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.\nScore: 361.6, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993\nWe re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier. Each of these problems invalidates the results, leading to the conclusion that the microbiome-based classifiers for identifying cancer presented in the study are entirely wrong. These flaws have subsequently affected more than a dozen additional published studies that used the same data and whose results are likely invalid as well.\nComprehensive mapping of cell fates in microsatellite unstable cancer cells support dual targeting of WRN and ATR\nAuthors: Zong, D.; Koussa, N. C.; Cornwell, J. A.; Pankajam, A. V.; Kruhlak, M.; Wong, N.; Chari, R.; Cappell, S. D.; Nussenzweig, A.\nScore: 10.5, Published: 2023-07-30 DOI: 10.1101/2023.07.28.550976\nAddiction to the WRN helicase is a unique vulnerability of human cancers with high levels of microsatellite instability (MSI-H). However, while prolonged loss of WRN ultimately leads to cell death, little is known about how MSI-H cancers initially respond to acute loss of WRN, knowledge that would be helpful for informing clinical development of WRN-targeting therapy, predicting possible resistance mechanisms, and identifying useful biomarkers of successful WRN inhibition. Here, we report the construction of an inducible ligand-mediated degradation system wherein the stability of endogenous WRN protein can be rapidly and specifically tuned, enabling us to track the complete sequence of cellular events elicited by acute loss of WRN function. We find that WRN degradation leads to immediate accrual of DNA damage in a replication-dependent manner that curiously did not robustly engage checkpoint mechanisms to halt DNA synthesis. As a result, WRN-degraded MSI-H cancer cells accumulate DNA damage across multiple replicative cycles and undergo successive rounds of increasingly aberrant mitoses, ultimately triggering cell death. Of potential therapeutic importance, we find no evidence of any generalized mechanism by which MSI-H cancers could adapt to near-complete loss of WRN. However, under conditions of partial WRN degradation, addition of low dose ATR inhibitor significantly increased their combined efficacy to levels approaching full inactivation of WRN. Overall, our results provided the first comprehensive view of molecular events linking upstream inhibition of WRN to subsequent cell death and suggested a potential therapeutical rationale for dual targeting of WRN and ATR.\nAn integrative proteomics approach identifies tyrosine kinase KIT as a novel therapeutic target for SPINK1-positive prostate cancer.\nAuthors: Manzar, N.; Khan, U. K.; Goel, A.; Carskadon, S.; Gupta, N.; Palanisamy, N.; Ateeq, B.\nScore: 9.7, Published: 2023-07-25 DOI: 10.1101/2023.07.24.550265\nElevated Serine Peptidase Inhibitor, Kazal type 1 (SPINK1) levels in [~]10-25% of prostate cancer (PCa) patients associate with aggressive phenotype, for which there are limited treatment choices and dismal clinical outcomes. Using an integrative proteomics approach involving label-free phosphoproteome and proteome profiling, we delineated the downstream signaling pathways involved in SPINK1-mediated tumorigenesis in PCa, and identified tyrosine kinase KIT as a highly enriched kinase. Furthermore, high to moderate levels of KIT expression was detected in [~]85% of SPINK1-positive PCa specimens. KIT signaling regulates SPINK1-associated oncogenesis, and treatment with KIT inhibitor reduces tumor growth and distant metastases in preclinical mice models. Mechanistically, KIT signaling modulates WNT/{beta}-catenin pathway and confers stemness-related features in PCa. Notably, inhibiting KIT signaling restores AR/REST levels, forming a feedback loop enabling SPINK1 repression. Overall, we uncover the role of KIT signaling downstream of SPINK1 in maintaining lineage plasticity and provide new treatment modalities for advanced-stage SPINK1-positive subtype.\nThe relationship between diet, plasma glucose, and cancer prevalence across vertebrates\nAuthors: Kapsetaki, S. E.; Basile, A. J.; Compton, Z. T.; Rupp, S. M.; Duke, E. G.; Boddy, A. M.; Harrison, T. M.; Sweazea, K. L.; Maley, C. C.\nScore: 6.6, Published: 2023-08-02 DOI: 10.1101/2023.07.31.551378\nCould diet and mean plasma glucose concentration (MPGluC) explain the variation in cancer prevalence across species? We collected diet, MPGluC, and neoplasia data for 160 vertebrate species from existing databases. We found that MPGluC negatively correlates with cancer and neoplasia prevalence, mostly of gastrointestinal organs. Trophic level positively correlates with cancer and neoplasia prevalence even after controlling for species MPGluC. Most species with high MPGluC (50/78 species = 64.1%) were birds. Most species in high trophic levels (42/53 species = 79.2%) were reptiles and mammals. Our results may be explained by the evolution of insulin resistance in birds which selected for loss or downregulation of genes related to insulin-mediated glucose import in cells. This led to higher MPGluC, intracellular caloric restriction, production of fewer reactive oxygen species and inflammatory cytokines, and longer telomeres contributing to longer longevity and lower neoplasia prevalence in extant birds relative to other vertebrates.\nDrug screening in human physiologic medium identifies uric acid as an inhibitor of rigosertib efficacy\nAuthors: Rawat, V.; DeLear, P.; Prashanth, P.; Ozgurses, M. E.; Tebeje, A.; Burns, P. A.; Conger, K. O.; Solis, C.; Hasnain, Y.; Novikova, A.; Endress, J. E.; Gonzalez-Sanchez, P.; Dong, W.; Stephanopoulos, G.; DeNicola, G. M.; Harris, I. S.; Sept, D. S.; Mason, F. M.; Coloff, J. L.\nScore: 35.1, Published: 2023-07-28 DOI: 10.1101/2023.07.26.550731\nThe non-physiological nutrient levels found in traditional culture media have been shown to affect numerous aspects of cancer cell physiology, including how cells respond to certain therapeutic agents. Here, we comprehensively evaluated how physiological nutrient levels impact therapeutic response by performing drug screening in human plasma-like medium (HPLM). We observed dramatic nutrient-dependent changes in sensitivity to a variety of FDA-approved and clinically trialed compounds, including rigosertib, an experimental cancer therapeutic that has recently failed in phase 3 clinical trials. Mechanistically, we found that the ability of rigosertib to destabilize microtubules is strongly inhibited by the purine metabolism waste product uric acid, which is uniquely abundant in humans relative to traditional in vitro and in vivo cancer models. Structural modelling studies suggest that uric acid interacts with the tubulin-rigosertib complex and may act as an uncompetitive inhibitor of rigosertib. These results offer a possible explanation for the failure of rigosertib in clinical trials and demonstrate the utility of physiological media to achieve in vitro results that better represent human therapeutic responses.\nTargeting the dependence on PIK3C3-mTORC1 signaling in dormancy-prone breast cancer cells blunts metastasis initiation\nAuthors: Elkholi, I. E.; Robert, A.; Kuasne, H.; Drapela, S.; Macleod, G.; Hebert, S.; Pacis, A.; Calderon, V.; Kleinman, C. L.; Gomes, A. P.; Aguirre-Ghiso, J. A.; Park, M.; Angers, S.; Cote, J.-F.\nScore: 4.3, Published: 2023-08-05 DOI: 10.1101/2023.08.02.551681\nHalting breast cancer metastatic relapses following primary tumor removal and the clinical dormant phase, remains challenging, due to a lack of specific vulnerabilities to target during dormancy. To address this, we conducted genome-wide CRISPR screens on two breast cancer cell lines with distinct dormancy properties: 4T1 (short-term dormancy) and 4T07 (prolonged dormancy). We discovered that loss of class-III PI3K, Pik3c3, revealed a unique vulnerability in 4T07 cells. Surprisingly, dormancy-prone 4T07 cells exhibited higher mTORC1 activity than 4T1 cells, due to lysosome-dependent signaling occurring at the cell periphery. Pharmacological inhibition of Pik3c3 counteracted this phenotype in 4T07 cells, and selectively reduced metastasis burden only in the 4T07 dormancy prone model. This mechanism was also detected in xenografts from human breast cancer patients, supporting that it may be relevant in humans. Our findings suggest dormant cancer cell-initiated metastasis may be prevented in patients carrying tumor cells that display PIK3C3-peripheral lysosomal signaling to mTORC1.\nDistinguishing preferences of human APOBEC3A and APOBEC3B for cytosines in hairpin loops, and reflection of these preferences in APOBEC-signature cancer genome mutations\nAuthors: Bhagwat, A. S.; Butt, Y.; Sakhtemani, R.; Mohamad-Ramshan, R.; Lawrence, M. S.\nScore: 3.0, Published: 2023-08-02 DOI: 10.1101/2023.08.01.551518\nThe APOBEC3 family of enzymes convert cytosines in single-stranded DNA to uracils thereby causing mutations. These enzymes protect human cells against viruses and retrotransposons, but in many cancers they contribute to mutations that diversify the tumors and help them escape anticancer drug treatments. To understand the mechanism of mutagenesis by APOBEC3B, we expressed the complete enzyme or its catalytic carboxy-terminal domain (CTD) in repair-deficient Eschericia coli and mapped the resulting uracils using uracil pull-down and sequencing technology. The uracilomes of A3B-full and A3B-CTD showed peaks in many of the same regions where APOBEC3A also created uracilation peaks. Like A3A, the two A3B enzymes also preferred to deaminate cytosines near transcription start sites and in the lagging-strand template at replication forks. In contrast to an earlier report that A3B does not favor hairpin loops over linear DNA, we found that both A3B-full and A3B-CTD showed a strong preference for cytosines in hairpin loops. The major difference between A3A and A3B was that while the former enzyme prefers 3 nt loops the best, A3B prefers loops of 4 nt over those of other lengths. Furthermore, within 5 nt loops, A3A prefers cytosine to be in the penultimate position, while A3B prefers it to be at the 3 end of the loop. We confirmed these loop size and sequence preferences experimentally using purified A3A and A3B-CTD proteins. Reanalysis of hairpin loop mutations in human tumors using the size, sequence and position preferences of the two enzymes found that the tumors displayed mutations with intrinsic characteristics of both the enzymes with a stronger contribution from A3A.\nSingle-cell analysis characterizes PLK1 as a promoter of immunosuppressive tumor microenvironment in LUAD\nAuthors: Kong, Y.; Li, C.; Liu, J.; Zhang, M.; Allison, D. B.; Hassan, F.; He, D.; Wang, X.; Mao, F.; Zhang, Q.; Zhang, Y.; Li, Z.; Wu, S.; Wang, C.; Liu, X.\nScore: 2.8, Published: 2023-08-05 DOI: 10.1101/2023.08.02.551692\nPLK1 (Polo-like kinase 1) plays a critical role in cancer progression of lung adenocarcinoma (LUAD). Recent studies have demonstrated that targeting PLK1 improves the efficacy of immunotherapy, indicating its important role in the regulation of tumor immunity. However, the correlation between PLK1 and tumor microenvironment (TME) is poorly understood. Here, using genetically engineered mouse model and single-cell RNA-seq analysis, we report that PLK1 promotes a general immunosuppressive TME in LUAD characterized with impaired tumor-infiltrating lymphocytes, enhanced M2 polarization and suppression of antigen presentation process. Mechanistically, elevated PLK1 coincides with increased secretion of CXCL2 cytokine, which promotes M2 polarization of macrophages and decreases expression of class II major histocompatibility complex (MHC-II) in professional antigen-presenting cells. Furthermore, PLK1 negatively regulates MHC-II expression in cancer cells, which has been shown to be associated with poor tumor immunity and patients' outcomes. Taken together, our results reveal PLK1 as a novel modulator of TME in LUAD and provide possible therapeutic interventions.\nEnhancer plasticity in endometrial tumorigenesis demarcates non-coding driver mutations and alterations in 3D genome organization to stimulate oncogene expression\nAuthors: Gregoricchio, S.; Kojic, A.; Hoogstraat, M.; Schuurman, K.; Stelloo, S.; Severson, T. M.; O'Mara, T. A.; Droog, M.; Singh, A. A.; Glubb, D. M.; Wessels, L. F. A.; Vermeulen, M.; van Leeuwen, F.; Zwart, W.\nScore: 7.2, Published: 2023-07-27 DOI: 10.1101/2023.07.25.550368\nAO_SCPLOWBSTRACTC_SCPLOWThe incidence and mortality of Endometrial Cancer (EC) is on the rise. 85% of ECs depend on Estrogen Receptor alpha (ER) for proliferation, but little is known about its transcriptional regulation in these tumors. We generated epigenomics and Hi-C data streams in healthy and tumor endometrial tissues, identifying robust ER reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with WGS data from metastatic samples revealed a striking enrichment of non-coding somatic mutations at tumor-enriched ER sites. Through machine learning-based predictions and interaction proteomics analyses, we identified an enhancer mutation which alters 3D genome organization, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC. In summary, we identified a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ER.\nAdaptive therapy achieves long-term control of chemotherapy resistance in high grade ovarian cancer\nAuthors: Hockings, H.; Lakatos, E.; Huang, W.; Mossner, M.; Khan, M. A.; Metcalf, S.; Nicolini, F.; Smith, K.; Baker, A.-M.; Graham, T.; Lockley, M.\nScore: 16.2, Published: 2023-07-25 DOI: 10.1101/2023.07.21.549688\nDrug resistance results in poor outcomes for most patients with metastatic cancer. Adaptive Therapy (AT) proposes to address this by exploiting presumed fitness costs incurred by drug-resistant cells when drug is absent, and prescribing dose reductions to allow fitter, sensitive cells to re-grow and re- sensitise the tumour. However, empirical evidence for treatment-induced fitness change is lacking. We show that fitness costs in chemotherapy-resistant ovarian cancer cause selective decline and apoptosis of resistant populations in low-resource conditions. Moreover, carboplatin AT caused fluctuations in sensitive/resistant tumour population size in vitro and significantly extended survival of tumour-bearing mice. In sequential blood-derived cell-free DNA and tumour samples obtained longitudinally from ovarian cancer patients during treatment, we inferred resistant cancer cell population size through therapy and observed it correlated strongly with disease burden. These data have enabled us to launch a multicentre, phase 2 randomised controlled trial (ACTOv) to evaluate AT in ovarian cancer.\n",
  "wordCount" : "2237",
  "inLanguage": "en",
  "datePublished": "2023-08-06T10:36:48Z",
  "dateModified": "2023-08-06T10:36:48Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on August 6, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550993">
        <p class="paperTitle">Major data analysis errors invalidate cancer microbiome findings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.</p>
        <p class="info">Score: 361.6, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550993' target='https://doi.org/10.1101/2023.07.28.550993'> 10.1101/2023.07.28.550993</a></p>
        <p class="abstract">We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier. Each of these problems invalidates the results, leading to the conclusion that the microbiome-based classifiers for identifying cancer presented in the study are entirely wrong. These flaws have subsequently affected more than a dozen additional published studies that used the same data and whose results are likely invalid as well.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550976">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550976" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550976">
        <p class="paperTitle">Comprehensive mapping of cell fates in microsatellite unstable cancer cells support dual targeting of WRN and ATR</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550976" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550976" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Zong, D.; Koussa, N. C.; Cornwell, J. A.; Pankajam, A. V.; Kruhlak, M.; Wong, N.; Chari, R.; Cappell, S. D.; Nussenzweig, A.</p>
        <p class="info">Score: 10.5, Published: 2023-07-30 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550976' target='https://doi.org/10.1101/2023.07.28.550976'> 10.1101/2023.07.28.550976</a></p>
        <p class="abstract">Addiction to the WRN helicase is a unique vulnerability of human cancers with high levels of microsatellite instability (MSI-H). However, while prolonged loss of WRN ultimately leads to cell death, little is known about how MSI-H cancers initially respond to acute loss of WRN, knowledge that would be helpful for informing clinical development of WRN-targeting therapy, predicting possible resistance mechanisms, and identifying useful biomarkers of successful WRN inhibition. Here, we report the construction of an inducible ligand-mediated degradation system wherein the stability of endogenous WRN protein can be rapidly and specifically tuned, enabling us to track the complete sequence of cellular events elicited by acute loss of WRN function. We find that WRN degradation leads to immediate accrual of DNA damage in a replication-dependent manner that curiously did not robustly engage checkpoint mechanisms to halt DNA synthesis. As a result, WRN-degraded MSI-H cancer cells accumulate DNA damage across multiple replicative cycles and undergo successive rounds of increasingly aberrant mitoses, ultimately triggering cell death. Of potential therapeutic importance, we find no evidence of any generalized mechanism by which MSI-H cancers could adapt to near-complete loss of WRN. However, under conditions of partial WRN degradation, addition of low dose ATR inhibitor significantly increased their combined efficacy to levels approaching full inactivation of WRN. Overall, our results provided the first comprehensive view of molecular events linking upstream inhibition of WRN to subsequent cell death and suggested a potential therapeutical rationale for dual targeting of WRN and ATR.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.24.550265">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.24.550265" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.24.550265">
        <p class="paperTitle">An integrative proteomics approach identifies tyrosine kinase KIT as a novel therapeutic target for SPINK1-positive prostate cancer.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.24.550265" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.24.550265" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Manzar, N.; Khan, U. K.; Goel, A.; Carskadon, S.; Gupta, N.; Palanisamy, N.; Ateeq, B.</p>
        <p class="info">Score: 9.7, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.24.550265' target='https://doi.org/10.1101/2023.07.24.550265'> 10.1101/2023.07.24.550265</a></p>
        <p class="abstract">Elevated Serine Peptidase Inhibitor, Kazal type 1 (SPINK1) levels in [~]10-25% of prostate cancer (PCa) patients associate with aggressive phenotype, for which there are limited treatment choices and dismal clinical outcomes. Using an integrative proteomics approach involving label-free phosphoproteome and proteome profiling, we delineated the downstream signaling pathways involved in SPINK1-mediated tumorigenesis in PCa, and identified tyrosine kinase KIT as a highly enriched kinase. Furthermore, high to moderate levels of KIT expression was detected in [~]85% of SPINK1-positive PCa specimens. KIT signaling regulates SPINK1-associated oncogenesis, and treatment with KIT inhibitor reduces tumor growth and distant metastases in preclinical mice models. Mechanistically, KIT signaling modulates WNT/{beta}-catenin pathway and confers stemness-related features in PCa. Notably, inhibiting KIT signaling restores AR/REST levels, forming a feedback loop enabling SPINK1 repression. Overall, we uncover the role of KIT signaling downstream of SPINK1 in maintaining lineage plasticity and provide new treatment modalities for advanced-stage SPINK1-positive subtype.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551378">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551378" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551378">
        <p class="paperTitle">The relationship between diet, plasma glucose, and cancer prevalence across vertebrates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551378" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551378" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kapsetaki, S. E.; Basile, A. J.; Compton, Z. T.; Rupp, S. M.; Duke, E. G.; Boddy, A. M.; Harrison, T. M.; Sweazea, K. L.; Maley, C. C.</p>
        <p class="info">Score: 6.6, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551378' target='https://doi.org/10.1101/2023.07.31.551378'> 10.1101/2023.07.31.551378</a></p>
        <p class="abstract">Could diet and mean plasma glucose concentration (MPGluC) explain the variation in cancer prevalence across species? We collected diet, MPGluC, and neoplasia data for 160 vertebrate species from existing databases. We found that MPGluC negatively correlates with cancer and neoplasia prevalence, mostly of gastrointestinal organs. Trophic level positively correlates with cancer and neoplasia prevalence even after controlling for species MPGluC. Most species with high MPGluC (50/78 species = 64.1%) were birds. Most species in high trophic levels (42/53 species = 79.2%) were reptiles and mammals. Our results may be explained by the evolution of insulin resistance in birds which selected for loss or downregulation of genes related to insulin-mediated glucose import in cells. This led to higher MPGluC, intracellular caloric restriction, production of fewer reactive oxygen species and inflammatory cytokines, and longer telomeres contributing to longer longevity and lower neoplasia prevalence in extant birds relative to other vertebrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.26.550731">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.26.550731" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.26.550731">
        <p class="paperTitle">Drug screening in human physiologic medium identifies uric acid as an inhibitor of rigosertib efficacy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.26.550731" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.26.550731" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rawat, V.; DeLear, P.; Prashanth, P.; Ozgurses, M. E.; Tebeje, A.; Burns, P. A.; Conger, K. O.; Solis, C.; Hasnain, Y.; Novikova, A.; Endress, J. E.; Gonzalez-Sanchez, P.; Dong, W.; Stephanopoulos, G.; DeNicola, G. M.; Harris, I. S.; Sept, D. S.; Mason, F. M.; Coloff, J. L.</p>
        <p class="info">Score: 35.1, Published: 2023-07-28 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.26.550731' target='https://doi.org/10.1101/2023.07.26.550731'> 10.1101/2023.07.26.550731</a></p>
        <p class="abstract">The non-physiological nutrient levels found in traditional culture media have been shown to affect numerous aspects of cancer cell physiology, including how cells respond to certain therapeutic agents. Here, we comprehensively evaluated how physiological nutrient levels impact therapeutic response by performing drug screening in human plasma-like medium (HPLM). We observed dramatic nutrient-dependent changes in sensitivity to a variety of FDA-approved and clinically trialed compounds, including rigosertib, an experimental cancer therapeutic that has recently failed in phase 3 clinical trials. Mechanistically, we found that the ability of rigosertib to destabilize microtubules is strongly inhibited by the purine metabolism waste product uric acid, which is uniquely abundant in humans relative to traditional in vitro and in vivo cancer models. Structural modelling studies suggest that uric acid interacts with the tubulin-rigosertib complex and may act as an uncompetitive inhibitor of rigosertib. These results offer a possible explanation for the failure of rigosertib in clinical trials and demonstrate the utility of physiological media to achieve in vitro results that better represent human therapeutic responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551681">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551681" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551681">
        <p class="paperTitle">Targeting the dependence on PIK3C3-mTORC1 signaling in dormancy-prone breast cancer cells blunts metastasis initiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551681" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551681" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Elkholi, I. E.; Robert, A.; Kuasne, H.; Drapela, S.; Macleod, G.; Hebert, S.; Pacis, A.; Calderon, V.; Kleinman, C. L.; Gomes, A. P.; Aguirre-Ghiso, J. A.; Park, M.; Angers, S.; Cote, J.-F.</p>
        <p class="info">Score: 4.3, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551681' target='https://doi.org/10.1101/2023.08.02.551681'> 10.1101/2023.08.02.551681</a></p>
        <p class="abstract">Halting breast cancer metastatic relapses following primary tumor removal and the clinical dormant phase, remains challenging, due to a lack of specific vulnerabilities to target during dormancy. To address this, we conducted genome-wide CRISPR screens on two breast cancer cell lines with distinct dormancy properties: 4T1 (short-term dormancy) and 4T07 (prolonged dormancy). We discovered that loss of class-III PI3K, Pik3c3, revealed a unique vulnerability in 4T07 cells. Surprisingly, dormancy-prone 4T07 cells exhibited higher mTORC1 activity than 4T1 cells, due to lysosome-dependent signaling occurring at the cell periphery. Pharmacological inhibition of Pik3c3 counteracted this phenotype in 4T07 cells, and selectively reduced metastasis burden only in the 4T07 dormancy prone model. This mechanism was also detected in xenografts from human breast cancer patients, supporting that it may be relevant in humans. Our findings suggest dormant cancer cell-initiated metastasis may be prevented in patients carrying tumor cells that display PIK3C3-peripheral lysosomal signaling to mTORC1.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.01.551518">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.01.551518" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.01.551518">
        <p class="paperTitle">Distinguishing preferences of human APOBEC3A and APOBEC3B for cytosines in hairpin loops, and reflection of these preferences in APOBEC-signature cancer genome mutations</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.01.551518" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.01.551518" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhagwat, A. S.; Butt, Y.; Sakhtemani, R.; Mohamad-Ramshan, R.; Lawrence, M. S.</p>
        <p class="info">Score: 3.0, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.01.551518' target='https://doi.org/10.1101/2023.08.01.551518'> 10.1101/2023.08.01.551518</a></p>
        <p class="abstract">The APOBEC3 family of enzymes convert cytosines in single-stranded DNA to uracils thereby causing mutations. These enzymes protect human cells against viruses and retrotransposons, but in many cancers they contribute to mutations that diversify the tumors and help them escape anticancer drug treatments. To understand the mechanism of mutagenesis by APOBEC3B, we expressed the complete enzyme or its catalytic carboxy-terminal domain (CTD) in repair-deficient Eschericia coli and mapped the resulting uracils using uracil pull-down and sequencing technology. The uracilomes of A3B-full and A3B-CTD showed peaks in many of the same regions where APOBEC3A also created uracilation peaks. Like A3A, the two A3B enzymes also preferred to deaminate cytosines near transcription start sites and in the lagging-strand template at replication forks. In contrast to an earlier report that A3B does not favor hairpin loops over linear DNA, we found that both A3B-full and A3B-CTD showed a strong preference for cytosines in hairpin loops. The major difference between A3A and A3B was that while the former enzyme prefers 3 nt loops the best, A3B prefers loops of 4 nt over those of other lengths. Furthermore, within 5 nt loops, A3A prefers cytosine to be in the penultimate position, while A3B prefers it to be at the 3 end of the loop. We confirmed these loop size and sequence preferences experimentally using purified A3A and A3B-CTD proteins. Reanalysis of hairpin loop mutations in human tumors using the size, sequence and position preferences of the two enzymes found that the tumors displayed mutations with intrinsic characteristics of both the enzymes with a stronger contribution from A3A.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551692">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551692" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551692">
        <p class="paperTitle">Single-cell analysis characterizes PLK1 as a promoter of immunosuppressive tumor microenvironment in LUAD</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551692" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551692" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kong, Y.; Li, C.; Liu, J.; Zhang, M.; Allison, D. B.; Hassan, F.; He, D.; Wang, X.; Mao, F.; Zhang, Q.; Zhang, Y.; Li, Z.; Wu, S.; Wang, C.; Liu, X.</p>
        <p class="info">Score: 2.8, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551692' target='https://doi.org/10.1101/2023.08.02.551692'> 10.1101/2023.08.02.551692</a></p>
        <p class="abstract">PLK1 (Polo-like kinase 1) plays a critical role in cancer progression of lung adenocarcinoma (LUAD). Recent studies have demonstrated that targeting PLK1 improves the efficacy of immunotherapy, indicating its important role in the regulation of tumor immunity. However, the correlation between PLK1 and tumor microenvironment (TME) is poorly understood. Here, using genetically engineered mouse model and single-cell RNA-seq analysis, we report that PLK1 promotes a general immunosuppressive TME in LUAD characterized with impaired tumor-infiltrating lymphocytes, enhanced M2 polarization and suppression of antigen presentation process. Mechanistically, elevated PLK1 coincides with increased secretion of CXCL2 cytokine, which promotes M2 polarization of macrophages and decreases expression of class II major histocompatibility complex (MHC-II) in professional antigen-presenting cells. Furthermore, PLK1 negatively regulates MHC-II expression in cancer cells, which has been shown to be associated with poor tumor immunity and patients&#39; outcomes. Taken together, our results reveal PLK1 as a novel modulator of TME in LUAD and provide possible therapeutic interventions.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.25.550368">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.25.550368" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.25.550368">
        <p class="paperTitle">Enhancer plasticity in endometrial tumorigenesis demarcates non-coding driver mutations and alterations in 3D genome organization to stimulate oncogene expression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.25.550368" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.25.550368" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gregoricchio, S.; Kojic, A.; Hoogstraat, M.; Schuurman, K.; Stelloo, S.; Severson, T. M.; O&#39;Mara, T. A.; Droog, M.; Singh, A. A.; Glubb, D. M.; Wessels, L. F. A.; Vermeulen, M.; van Leeuwen, F.; Zwart, W.</p>
        <p class="info">Score: 7.2, Published: 2023-07-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.25.550368' target='https://doi.org/10.1101/2023.07.25.550368'> 10.1101/2023.07.25.550368</a></p>
        <p class="abstract">AO_SCPLOWBSTRACTC_SCPLOWThe incidence and mortality of Endometrial Cancer (EC) is on the rise. 85% of ECs depend on Estrogen Receptor alpha (ER) for proliferation, but little is known about its transcriptional regulation in these tumors.

We generated epigenomics and Hi-C data streams in healthy and tumor endometrial tissues, identifying robust ER reprogramming and profound alterations in 3D genome organization that lead to a gain of tumor-specific enhancer activity during EC development. Integration with WGS data from metastatic samples revealed a striking enrichment of non-coding somatic mutations at tumor-enriched ER sites. Through machine learning-based predictions and interaction proteomics analyses, we identified an enhancer mutation which alters 3D genome organization, impairing recruitment of the transcriptional repressor EHMT2/G9a/KMT1C, thereby alleviating transcriptional repression of ESR1 in EC.

In summary, we identified a complex genomic-epigenomic interplay in EC development and progression, altering 3D genome organization to enhance expression of the critical driver ER.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.21.549688">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.21.549688" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.21.549688">
        <p class="paperTitle">Adaptive therapy achieves long-term control of chemotherapy resistance in high grade ovarian cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.21.549688" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.21.549688" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hockings, H.; Lakatos, E.; Huang, W.; Mossner, M.; Khan, M. A.; Metcalf, S.; Nicolini, F.; Smith, K.; Baker, A.-M.; Graham, T.; Lockley, M.</p>
        <p class="info">Score: 16.2, Published: 2023-07-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.21.549688' target='https://doi.org/10.1101/2023.07.21.549688'> 10.1101/2023.07.21.549688</a></p>
        <p class="abstract">Drug resistance results in poor outcomes for most patients with metastatic cancer. Adaptive Therapy (AT) proposes to address this by exploiting presumed fitness costs incurred by drug-resistant cells when drug is absent, and prescribing dose reductions to allow fitter, sensitive cells to re-grow and re- sensitise the tumour. However, empirical evidence for treatment-induced fitness change is lacking. We show that fitness costs in chemotherapy-resistant ovarian cancer cause selective decline and apoptosis of resistant populations in low-resource conditions. Moreover, carboplatin AT caused fluctuations in sensitive/resistant tumour population size in vitro and significantly extended survival of tumour-bearing mice. In sequential blood-derived cell-free DNA and tumour samples obtained longitudinally from ovarian cancer patients during treatment, we inferred resistant cancer cell population size through therapy and observed it correlated strongly with disease burden. These data have enabled us to launch a multicentre, phase 2 randomised controlled trial (ACTOv) to evaluate AT in ovarian cancer.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
