<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization
Authors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.
Score: 32.4, Published: 2023-10-27 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization
Authors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.
Score: 32.4, Published: 2023-10-27 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-15T10:40:26+00:00" />
<meta property="article:modified_time" content="2023-11-15T10:40:26+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization
Authors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.
Score: 32.4, Published: 2023-10-27 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization\nAuthors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.\nScore: 32.4, Published: 2023-10-27 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization\nAuthors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.\nScore: 32.4, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563749\nTumor-infiltrating macrophages support critical steps in tumor progression, and their accumulation in the tumor microenvironment (TME) is associated with adverse outcomes and therapeutic resistance across human cancers. In the TME, macrophages adopt diverse phenotypic alterations, giving rise to heterogeneous immune activation states and induction of cell cycle. While the transcriptional profiles of these activation states are well-annotated across human cancers, the underlying signals that regulate macrophage heterogeneity and accumulation remain incompletely understood. Here, we leveraged a novel ex vivo organotypic TME (oTME) model of breast cancer, in vivo murine models, and human samples to map the determinants of functional heterogeneity of TME macrophages. We identified a subset of F4/80highSca-1+ self-renewing macrophages maintained by type-I interferon (IFN) signaling and requiring physical contact with cancer-associated fibroblasts. We discovered that the contact-dependent self-renewal of TME macrophages is mediated via Notch4, and its inhibition abrogated tumor growth of breast and ovarian carcinomas in vivo, as well as lung dissemination in a PDX model of triple-negative breast cancer (TNBC). Through spatial multi-omic profiling of protein markers and transcriptomes, we found that the localization of macrophages further dictates functionally distinct but reversible phenotypes, regardless of their ontogeny. Whereas immune-stimulatory macrophages (CD11C+CD86+) populated the tumor epithelial nests, the stroma-associated macrophages (SAMs) were proliferative, immunosuppressive (Sca-1+CD206+PD-L1+), resistant to CSF-1R depletion, and associated with worse patient outcomes. Notably, following cessation of CSF-1R depletion, macrophages rebounded primarily to the SAM phenotype, which was associated with accelerated growth of mammary tumors. Our work reveals the spatial determinants of macrophage heterogeneity in breast cancer and highlights the disruption of macrophage self-renewal as a potential new therapeutic strategy.\n2D and 3D multiplexed subcellular profiling of nuclear instability in human cancer\nAuthors: Coy, S.; Cheng, B.; Lee, J. S.; Rashid, R.; Browning, L.; Xu, Y.; Chakrabarty, S. S.; Yapp, C.; Chan, S.; Tefft, J. B.; Spektor, A.; Ligon, K. L.; Baker, G. J.; Pellman, D.; Sorger, P. K.; Santagata, S.\nScore: 14.8, Published: 2023-11-11 DOI: 10.1101/2023.11.07.566063\nNuclear atypia, including altered nuclear size, contour, and chromatin organization, is ubiquitous in cancer cells. Atypical primary nuclei and micronuclei can rupture during interphase; however, the frequency, causes, and consequences of nuclear rupture are unknown in most cancers. We demonstrate that nuclear envelope rupture is surprisingly common in many human cancers, particularly glioblastoma. Using highly-multiplexed 2D and super-resolution 3D-imaging of glioblastoma tissues and patient-derived xenografts and cells, we link primary nuclear rupture with reduced lamin A/C and micronuclear rupture with reduced lamin B1. Moreover, ruptured glioblastoma cells activate cGAS-STING-signaling involved in innate immunity. We observe that local patterning of cell states influences tumor spatial organization and is linked to both lamin expression and rupture frequency, with neural-progenitor-cell-like states exhibiting the lowest lamin A/C levels and greatest susceptibility to primary nuclear rupture. Our study reveals that nuclear instability is a core feature of cancer, and links nuclear integrity, cell state, and immune signaling.\nAtaxia-telangiectasia mutated (Atm) disruption sensitizes spatially-directed H3.3K27M/TP53 diffuse midline gliomas to radiation therapy\nAuthors: Mangoli, A.; Wu, S.; Liu, H. Q.; Aksu, M.; Jain, V. J.; Foreman, B. E.; Regal, J. A.; Weidenhammer, L. B.; Stewart, C. E.; Guerra Garcia, M. E.; Hocke, E.; Abramson, K.; Williams, N. T.; Luo, L.; Deland, K.; Attardi, L.; Abe, K.; Hashizume, R.; Ashley, D. M.; Becher, O. J.; Kirsch, D. G.; Gregory, S. G.; Reitman, Z. J.\nScore: 292.4, Published: 2023-10-20 DOI: 10.1101/2023.10.18.562892\nDiffuse midline gliomas (DMGs) are lethal brain tumors characterized by p53-inactivating mutations and oncohistone H3.3K27M mutations that rewire the cellular response to genotoxic stress, which presents therapeutic opportunities. We used RCAS/tv-a retroviruses and Cre recombinase to inactivate p53 and induce K27M in the native H3f3a allele in a lineage- and spatially-directed manner, yielding primary mouse DMGs. Genetic or pharmacologic disruption of the DNA damage response kinase Ataxia-telangiectasia mutated (ATM) enhanced the efficacy of focal brain irradiation, extending mouse survival. This finding suggests that targeting ATM will enhance the efficacy of radiation therapy for p53-mutant DMG but not p53-wildtype DMG. We used spatial in situ transcriptomics and an allelic series of primary murine DMG models with different p53 mutations to identify transactivation-independent p53 activity as a key mediator of such radiosensitivity. These studies deeply profile a genetically faithful and versatile model of a lethal brain tumor to identify resistance mechanisms for a therapeutic strategy currently in clinical trials.\nA single-cell multi-omic and spatial atlas of nodal B-cell lymphomas reveals B-cell maturation drives intratumor heterogeneity\nAuthors: Fitzgerald, D.; Roider, T.; Baertsch, M. A.; Kibler, A.; Horlova, A.; Chung, E. K.; Voehringer, H.; Mathioudaki, A.; Budeus, B.; Passerini, V.; Knoll, M.; Mammen, J.; Li, L.; Caille, L.; Czernilofsky, F.; Bruch, P. M.; Liebers, N.; Meyer-Bender, M.; Weigert, O.; Zaugg, J. B.; Nolan, G. P.; Seifert, M.; Huber, W.; Dietrich, S.\nScore: 10.1, Published: 2023-11-07 DOI: 10.1101/2023.11.06.565756\nIntratumor heterogeneity is intrinsic to cancer pathogenesis and evolution, although little is known about how it relates to the differentiation trajectories of the tumors cell-of-origin. Nodal B-cell non-Hodgkin lymphomas are a diverse set of cancers thought to originate from distinct stages of B-cell maturation. Through a single-cell multi-omic and spatial atlas of diffuse large B-cell, mantle cell, follicular, and marginal zone lymphomas along with reactive lymph nodes (n=51), we found multiple B-cell maturation states coexist within the same tumors. Intratumor maturation states emerged from the same cell-of-origin, revealing that maturation remains plastic in malignancy. The maturation state composition varied across entities and tumors, which included mixtures of cell-of-origin subtypes. Intratumor maturation states inhabited unique spatial niches, which typically retained their maturation-associated cellular interactions and regulatory networks. Intratumor maturation states showed distinct expression patterns of genetic variants, suggesting that maturation and genetic aberrations are intertwined. Our findings put forward a transformative model for cancer pathogenesis, where differentiation continues in malignancy and is central to tumor heterogeneity and evolution.\nMutational scanning reveals oncogenic CTNNB1 mutations have diverse effects on signalling and clinical traits\nAuthors: Krishna, A.; Meynert, A.; Kelder, M.; Ewing, A.; Sheraz, S.; Ferrer-Vacquer, A.; Grimes, G.; Becher, H.; Silk, R.; Semple, C. A.; Kendall, T.; Hadjantonakis, A.-K.; Bird, T.; Marsh, J. A.; Hohenstein, P.; Wood, A. J.; Ozdemir, D. D.\nScore: 6.1, Published: 2023-11-13 DOI: 10.1101/2023.11.09.566307\nAbstract CTNNB1, the gene encoding beta-catenin, is a frequent target for oncogenic mutations activating the canonical Wnt signalling pathway, typically via missense mutations within a degron hotspot motif in exon 3. Here, we combine saturation genome editing with a fluorescent reporter assay to quantify signalling phenotypes for all 342 missense mutations in the mutation hotspot, including 74 recurrent mutations reported in over 6000 tumours. Our data define the genetic requirements for beta-catenin degron function and reveal diverse levels of signal activation among known driver mutations. Tumorigenesis in different human tissues involves selection for CTNNB1 mutations spanning distinct ranges of effect size. In hepatocellular carcinoma, mutations that activate beta-catenin relatively weakly are associated with worse prognosis compared to stronger activating mutations, despite greater immune cell infiltration in the tumour microenvironment. Our work therefore provides a resource to understand mutational diversity within a pan-cancer mutation hotspot, with potential implications for targeted therapy.\nModeling the effect of spatial structure on solid tumor evolution and ctDNA composition\nAuthors: Rachman, T.; Bartlett, D.; Laframboise, W.; Wagner, P.; Schwartz, R.; Carja, O.\nScore: 6.0, Published: 2023-11-11 DOI: 10.1101/2023.11.10.566658\nCirculating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform on cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how elevated cell death on the edge of a tumor can simultaneously impact driver mutation accumulation and the representation of tumor clones and mutation detectability in ctDNA. We describe conditions in which invasive clones end up over-represented in ctDNA, clonal diversity can appear elevated in the blood, and spatial bias in shedding can inflate subclonal variant allele frequencies (VAFs). Additionally, we find that tumors that are mostly quiescent can display similar biases, but are far less detectable, and the extent of perceptible spatial bias strongly depends on sequence detection limits. Overall, we show that spatially structured shedding might cause liquid biopsies to provide highly biased profiles of tumor state. While this may enable more sensitive detection of expanding clones, it could also increase the risk of targeting a subclonal variant for treatment. Our results indicate that the effects and clinical consequences of spatially variable cell death on ctDNA composition present an important area for future work.\neIF4A1 is essential for reprogramming the translational landscape of Wnt-driven colorectal cancers\nAuthors: Waldron, J. A.; Kanellos, G.; Smith, R. C. L.; Knight, J. R. P.; Munro, J.; Alexandrou, C.; Vlahov, N.; Pardo-Fernandez, L.; Moore, M.; Gillen, S. L.; Strathdee, D.; Stevenson, D.; Warrander, F. C.; Gilroy, K.; Nixon, C.; Cadden, B.; Powley, I.; Officer-Jones, L.; Ballantyne, F.; Hay, J.; Pennel, K.; Edwards, J.; Campbell, A. D.; Ridgway, R. A.; Coffelt, S. B.; Norman, J.; Le Quesne, J.; Bushell, M.; Sansom, O. J.\nScore: 5.6, Published: 2023-11-13 DOI: 10.1101/2023.11.10.566546\nDysregulated translation is a hallmark of cancer. Targeting the translational machinery represents a therapeutic avenue which is being actively explored. eIF4A inhibitors target both eIF4A1, which promotes translation as part of the eIF4F complex, and eIF4A2, which can repress translation via the CCR4 NOT complex. While high eIF4A1 expression is associated with poor patient outcome, the role of eIF4A2 in cancer remains unclear. Furthermore, the on-target toxicity of targeting specific eIF4A paralogues in healthy tissue is under explored. We show that while loss of either paralogue is tolerated in the wild-type intestine, eIF4A1 is specifically required to support the translational demands of oncogenic Wnt signalling. Intestinal tumourigenesis is suppressed in colorectal cancer models following loss of eIF4A1 but accelerated following loss of eIF4A2, while eIF4A inhibition with eFT226 mimics loss of eIF4A1 in these models.\nComparison of Tug-of-War Models Assuming Moran versus Branching Process Population Dynamics\nAuthors: Dinh, K. N.; Kurpas, M. K.; Kimmel, M.\nScore: 8.9, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563302\nMutations arising during cancer evolution are typically categorized as either drivers or passengers, depending on whether they increase the cell fitness. Recently, McFarland et al. introduced the tug-of-war model for the joint effect of rare advantageous drivers and frequent but deleterious passengers. We examine this model under two common but distinct frameworks, the Moran model and the branching process. We show that frequently used statistics are similar between a version of the Moran model and the branching process conditioned on the final cell count, under different selection scenarios. We infer the selection coefficients for three breast cancer samples, resulting in good fits of the shape of their Site Frequency Spectra. All fitted values for the selective disadvantage of passenger mutations are nonzero, supporting the view that they exert deleterious selection during tumorigenesis that driver mutations must compensate.\nInteractions between ploidy and resource availability shape clonal interference at initiation and recurrence of glioblastoma\nAuthors: Nowicka, Z.; Rentzeperis, F.; Beck, R.; Tagal, V.; Forero Pinto, A. M.; Scanu, E.; Veith, T.; Cole, J.; Ilter, D.; Dominguez Viqueira, W.; Teer, J. K.; Maksin, K.; Pasetto, S.; Abdalah, M. A.; Fiandaca, G.; Prabhakaran, S.; Schultz, A.; Ojwang, M.; Barnholtz-Sloan, J. S.; Joaquim, F. M.; Gomes, A. P.; Katira, P.; Andor, N.\nScore: 15.7, Published: 2023-10-20 DOI: 10.1101/2023.10.17.562670\nGlioblastoma (GBM) is the most aggressive form of primary brain tumor. Complete surgical resection of GBM is almost impossible due to the infiltrative nature of the cancer. While no evidence for recent selection events have been found after diagnosis, the selective forces that govern gliomagenesis are strong, shaping the tumors cell composition during the initial progression to malignancy with late consequences for invasiveness and therapy response. We present a mathematical model that simulates the growth and invasion of a glioma, given its ploidy level and the nature of its brain tissue micro-environment (TME), and use it to make inferences about GBM initiation and response to standard-of-care treatment. We approximate the spatial distribution of resource access in the TME through integration of in-silico modelling, multi-omics data and image analysis of primary and recurrent GBM. In the pre-malignant setting, our in-silico results suggest that low ploidy cancer cells are more resistant to starvation-induced cell death. In the malignant setting, between first and second surgery, simulated tumors with different ploidy compositions progressed at different rates. Whether higher ploidy predicted fast recurrence, however, depended on the TME. Historical data supports this dependence on TME resources, as shown by a significant correlation between the median glucose uptake rates in human tissues and the median ploidy of cancer types that arise in the respective tissues (Spearman r = -0.70; P = 0.026). Taken together our findings suggest that availability of metabolic substrates in the TME drives different cell fate decisions for cancer cells with different ploidy and shapes GBM disease initiation and relapse characteristics.\nMolecular subtypes of high grade serous ovarian cancer across racial groups and gene expression platforms\nAuthors: Davidson, N. R.; Barnard, M. E.; Hippen, A. A.; Campbell, A.; Johnson, C.; Way, G. P.; Dalley, B. K.; Berchuck, A.; Salas, L. A.; Peres, L. C.; Marks, J. R.; Schildkraut, J. M.; Greene, C. S.; Doherty, J. A.\nScore: 7.7, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565179\nIntroductionBlack individuals with ovarian cancer experience poorer survival compared to non-Hispanic White individuals. High-grade serous carcinoma (HGSC) is the most common ovarian cancer histotype, with gene expression subtypes that are associated with differential survival. We characterized HGSC gene expression in Black individuals and considered whether gene expression differences by race may contribute to disparities in survival. MethodsWe performed gene expression analyses using RNA-Seq data from Black and White individuals with HGSC, as well as array-based genotyping data from four existing studies of HGSC. Our main analysis assigned subtypes by identifying dataset-specific clusters using K-means clustering for K=2-4. The cluster- and dataset-specific gene expression patterns were summarized by moderated t-scores that differentiate an individual cluster from all other clusters within each dataset. We compared the calculated gene expression patterns for each cluster across datasets by calculating the Pearson correlation between the two summarized vectors of moderated t-scores. Following K=4 subtype assignment and mapping to The Cancer Genome Atlas (TCGA)-derived HGSC subtypes, we used multivariable-adjusted Cox proportional hazards models to estimate subtype-specific survival separately for each dataset. ResultsWhen we compared HGSC gene expression between RNA-Seq data and array-based gene expression data, cluster-specific gene expression was similar across data types. Comparing the Black study population to White and Japanese study populations, the immunoreactive subtype was more common (39% versus 23%-28%) and the differentiated subtype less common (7% versus 22%-31%). Patterns of subtype-specific survival were similar between the Black and White populations with RNAseq data; when compared to mesenchymal HGSC, the risk of death was similar for proliferative and differentiated cases and lower, but not statistically significantly lower, for immunoreactive cases (HR=0.79 [0.55, 1.13] in the Black population and HR=0.86 [0.62, 1.19] in the White population). Patterns of subtype-specific survival were also similar in a Japanese population and in two of three predominantly White populations with array-based gene expression data. ConclusionsA single pipeline can be used to assign gene expression subtypes to HGSC with array-based or RNA-Seq gene expression data. We observed that while the prevalence of the immunoreactive subtype was higher, and the prevalence of the differentiated subtype was lower among Black individuals compared to White individuals, subtype-specific survival was similar.\n",
  "wordCount" : "2637",
  "inLanguage": "en",
  "datePublished": "2023-11-15T10:40:26Z",
  "dateModified": "2023-11-15T10:40:26Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 15, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563749">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563749" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563749">
        <p class="paperTitle">Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563749" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563749" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.</p>
        <p class="info">Score: 32.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563749' target='https://doi.org/10.1101/2023.10.24.563749'> 10.1101/2023.10.24.563749</a></p>
        <p class="abstract">Tumor-infiltrating macrophages support critical steps in tumor progression, and their accumulation in the tumor microenvironment (TME) is associated with adverse outcomes and therapeutic resistance across human cancers. In the TME, macrophages adopt diverse phenotypic alterations, giving rise to heterogeneous immune activation states and induction of cell cycle. While the transcriptional profiles of these activation states are well-annotated across human cancers, the underlying signals that regulate macrophage heterogeneity and accumulation remain incompletely understood. Here, we leveraged a novel ex vivo organotypic TME (oTME) model of breast cancer, in vivo murine models, and human samples to map the determinants of functional heterogeneity of TME macrophages. We identified a subset of F4/80highSca-1&#43; self-renewing macrophages maintained by type-I interferon (IFN) signaling and requiring physical contact with cancer-associated fibroblasts. We discovered that the contact-dependent self-renewal of TME macrophages is mediated via Notch4, and its inhibition abrogated tumor growth of breast and ovarian carcinomas in vivo, as well as lung dissemination in a PDX model of triple-negative breast cancer (TNBC). Through spatial multi-omic profiling of protein markers and transcriptomes, we found that the localization of macrophages further dictates functionally distinct but reversible phenotypes, regardless of their ontogeny. Whereas immune-stimulatory macrophages (CD11C&#43;CD86&#43;) populated the tumor epithelial nests, the stroma-associated macrophages (SAMs) were proliferative, immunosuppressive (Sca-1&#43;CD206&#43;PD-L1&#43;), resistant to CSF-1R depletion, and associated with worse patient outcomes. Notably, following cessation of CSF-1R depletion, macrophages rebounded primarily to the SAM phenotype, which was associated with accelerated growth of mammary tumors. Our work reveals the spatial determinants of macrophage heterogeneity in breast cancer and highlights the disruption of macrophage self-renewal as a potential new therapeutic strategy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566063">
        <p class="paperTitle">2D and 3D multiplexed subcellular profiling of nuclear instability in human cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coy, S.; Cheng, B.; Lee, J. S.; Rashid, R.; Browning, L.; Xu, Y.; Chakrabarty, S. S.; Yapp, C.; Chan, S.; Tefft, J. B.; Spektor, A.; Ligon, K. L.; Baker, G. J.; Pellman, D.; Sorger, P. K.; Santagata, S.</p>
        <p class="info">Score: 14.8, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566063' target='https://doi.org/10.1101/2023.11.07.566063'> 10.1101/2023.11.07.566063</a></p>
        <p class="abstract">Nuclear atypia, including altered nuclear size, contour, and chromatin organization, is ubiquitous in cancer cells. Atypical primary nuclei and micronuclei can rupture during interphase; however, the frequency, causes, and consequences of nuclear rupture are unknown in most cancers. We demonstrate that nuclear envelope rupture is surprisingly common in many human cancers, particularly glioblastoma. Using highly-multiplexed 2D and super-resolution 3D-imaging of glioblastoma tissues and patient-derived xenografts and cells, we link primary nuclear rupture with reduced lamin A/C and micronuclear rupture with reduced lamin B1. Moreover, ruptured glioblastoma cells activate cGAS-STING-signaling involved in innate immunity. We observe that local patterning of cell states influences tumor spatial organization and is linked to both lamin expression and rupture frequency, with neural-progenitor-cell-like states exhibiting the lowest lamin A/C levels and greatest susceptibility to primary nuclear rupture. Our study reveals that nuclear instability is a core feature of cancer, and links nuclear integrity, cell state, and immune signaling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562892">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562892" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562892">
        <p class="paperTitle">Ataxia-telangiectasia mutated (Atm) disruption sensitizes spatially-directed H3.3K27M/TP53 diffuse midline gliomas to radiation therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562892" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562892" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mangoli, A.; Wu, S.; Liu, H. Q.; Aksu, M.; Jain, V. J.; Foreman, B. E.; Regal, J. A.; Weidenhammer, L. B.; Stewart, C. E.; Guerra Garcia, M. E.; Hocke, E.; Abramson, K.; Williams, N. T.; Luo, L.; Deland, K.; Attardi, L.; Abe, K.; Hashizume, R.; Ashley, D. M.; Becher, O. J.; Kirsch, D. G.; Gregory, S. G.; Reitman, Z. J.</p>
        <p class="info">Score: 292.4, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562892' target='https://doi.org/10.1101/2023.10.18.562892'> 10.1101/2023.10.18.562892</a></p>
        <p class="abstract">Diffuse midline gliomas (DMGs) are lethal brain tumors characterized by p53-inactivating mutations and oncohistone H3.3K27M mutations that rewire the cellular response to genotoxic stress, which presents therapeutic opportunities. We used RCAS/tv-a retroviruses and Cre recombinase to inactivate p53 and induce K27M in the native H3f3a allele in a lineage- and spatially-directed manner, yielding primary mouse DMGs. Genetic or pharmacologic disruption of the DNA damage response kinase Ataxia-telangiectasia mutated (ATM) enhanced the efficacy of focal brain irradiation, extending mouse survival. This finding suggests that targeting ATM will enhance the efficacy of radiation therapy for p53-mutant DMG but not p53-wildtype DMG. We used spatial in situ transcriptomics and an allelic series of primary murine DMG models with different p53 mutations to identify transactivation-independent p53 activity as a key mediator of such radiosensitivity. These studies deeply profile a genetically faithful and versatile model of a lethal brain tumor to identify resistance mechanisms for a therapeutic strategy currently in clinical trials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565756" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565756">
        <p class="paperTitle">A single-cell multi-omic and spatial atlas of nodal B-cell lymphomas reveals B-cell maturation drives intratumor heterogeneity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fitzgerald, D.; Roider, T.; Baertsch, M. A.; Kibler, A.; Horlova, A.; Chung, E. K.; Voehringer, H.; Mathioudaki, A.; Budeus, B.; Passerini, V.; Knoll, M.; Mammen, J.; Li, L.; Caille, L.; Czernilofsky, F.; Bruch, P. M.; Liebers, N.; Meyer-Bender, M.; Weigert, O.; Zaugg, J. B.; Nolan, G. P.; Seifert, M.; Huber, W.; Dietrich, S.</p>
        <p class="info">Score: 10.1, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565756' target='https://doi.org/10.1101/2023.11.06.565756'> 10.1101/2023.11.06.565756</a></p>
        <p class="abstract">Intratumor heterogeneity is intrinsic to cancer pathogenesis and evolution, although little is known about how it relates to the differentiation trajectories of the tumors cell-of-origin. Nodal B-cell non-Hodgkin lymphomas are a diverse set of cancers thought to originate from distinct stages of B-cell maturation. Through a single-cell multi-omic and spatial atlas of diffuse large B-cell, mantle cell, follicular, and marginal zone lymphomas along with reactive lymph nodes (n=51), we found multiple B-cell maturation states coexist within the same tumors. Intratumor maturation states emerged from the same cell-of-origin, revealing that maturation remains plastic in malignancy. The maturation state composition varied across entities and tumors, which included mixtures of cell-of-origin subtypes. Intratumor maturation states inhabited unique spatial niches, which typically retained their maturation-associated cellular interactions and regulatory networks. Intratumor maturation states showed distinct expression patterns of genetic variants, suggesting that maturation and genetic aberrations are intertwined. Our findings put forward a transformative model for cancer pathogenesis, where differentiation continues in malignancy and is central to tumor heterogeneity and evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.09.566307">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.09.566307" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.09.566307">
        <p class="paperTitle">Mutational scanning reveals oncogenic CTNNB1 mutations have diverse effects on signalling and clinical traits</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.09.566307" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.09.566307" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Krishna, A.; Meynert, A.; Kelder, M.; Ewing, A.; Sheraz, S.; Ferrer-Vacquer, A.; Grimes, G.; Becher, H.; Silk, R.; Semple, C. A.; Kendall, T.; Hadjantonakis, A.-K.; Bird, T.; Marsh, J. A.; Hohenstein, P.; Wood, A. J.; Ozdemir, D. D.</p>
        <p class="info">Score: 6.1, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.09.566307' target='https://doi.org/10.1101/2023.11.09.566307'> 10.1101/2023.11.09.566307</a></p>
        <p class="abstract">Abstract CTNNB1, the gene encoding beta-catenin, is a frequent target for oncogenic mutations activating the canonical Wnt signalling pathway, typically via missense mutations within a degron hotspot motif in exon 3. Here, we combine saturation genome editing with a fluorescent reporter assay to quantify signalling phenotypes for all 342 missense mutations in the mutation hotspot, including 74 recurrent mutations reported in over 6000 tumours. Our data define the genetic requirements for beta-catenin degron function and reveal diverse levels of signal activation among known driver mutations. Tumorigenesis in different human tissues involves selection for CTNNB1 mutations spanning distinct ranges of effect size. In hepatocellular carcinoma, mutations that activate beta-catenin relatively weakly are associated with worse prognosis compared to stronger activating mutations, despite greater immune cell infiltration in the tumour microenvironment. Our work therefore provides a resource to understand mutational diversity within a pan-cancer mutation hotspot, with potential implications for targeted therapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566658">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566658" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566658">
        <p class="paperTitle">Modeling the effect of spatial structure on solid tumor evolution and ctDNA composition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566658" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566658" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Rachman, T.; Bartlett, D.; Laframboise, W.; Wagner, P.; Schwartz, R.; Carja, O.</p>
        <p class="info">Score: 6.0, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566658' target='https://doi.org/10.1101/2023.11.10.566658'> 10.1101/2023.11.10.566658</a></p>
        <p class="abstract">Circulating tumor DNA (ctDNA) monitoring, while sufficiently advanced to reflect tumor evolution in real time and inform on cancer diagnosis, treatment, and prognosis, mainly relies on DNA that originates from cell death via apoptosis or necrosis. In solid tumors, chemotherapy and immune infiltration can induce spatially variable rates of cell death, with the potential to bias and distort the clonal composition of ctDNA. Using a stochastic evolutionary model of boundary-driven growth, we study how elevated cell death on the edge of a tumor can simultaneously impact driver mutation accumulation and the representation of tumor clones and mutation detectability in ctDNA. We describe conditions in which invasive clones end up over-represented in ctDNA, clonal diversity can appear elevated in the blood, and spatial bias in shedding can inflate subclonal variant allele frequencies (VAFs). Additionally, we find that tumors that are mostly quiescent can display similar biases, but are far less detectable, and the extent of perceptible spatial bias strongly depends on sequence detection limits. Overall, we show that spatially structured shedding might cause liquid biopsies to provide highly biased profiles of tumor state. While this may enable more sensitive detection of expanding clones, it could also increase the risk of targeting a subclonal variant for treatment. Our results indicate that the effects and clinical consequences of spatially variable cell death on ctDNA composition present an important area for future work.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.10.566546">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.10.566546" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.10.566546">
        <p class="paperTitle">eIF4A1 is essential for reprogramming the translational landscape of Wnt-driven colorectal cancers</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.10.566546" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.10.566546" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Waldron, J. A.; Kanellos, G.; Smith, R. C. L.; Knight, J. R. P.; Munro, J.; Alexandrou, C.; Vlahov, N.; Pardo-Fernandez, L.; Moore, M.; Gillen, S. L.; Strathdee, D.; Stevenson, D.; Warrander, F. C.; Gilroy, K.; Nixon, C.; Cadden, B.; Powley, I.; Officer-Jones, L.; Ballantyne, F.; Hay, J.; Pennel, K.; Edwards, J.; Campbell, A. D.; Ridgway, R. A.; Coffelt, S. B.; Norman, J.; Le Quesne, J.; Bushell, M.; Sansom, O. J.</p>
        <p class="info">Score: 5.6, Published: 2023-11-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.10.566546' target='https://doi.org/10.1101/2023.11.10.566546'> 10.1101/2023.11.10.566546</a></p>
        <p class="abstract">Dysregulated translation is a hallmark of cancer. Targeting the translational machinery represents a therapeutic avenue which is being actively explored. eIF4A inhibitors target both eIF4A1, which promotes translation as part of the eIF4F complex, and eIF4A2, which can repress translation via the CCR4 NOT complex. While high eIF4A1 expression is associated with poor patient outcome, the role of eIF4A2 in cancer remains unclear. Furthermore, the on-target toxicity of targeting specific eIF4A paralogues in healthy tissue is under explored. We show that while loss of either paralogue is tolerated in the wild-type intestine, eIF4A1 is specifically required to support the translational demands of oncogenic Wnt signalling. Intestinal tumourigenesis is suppressed in colorectal cancer models following loss of eIF4A1 but accelerated following loss of eIF4A2, while eIF4A inhibition with eFT226 mimics loss of eIF4A1 in these models.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563302">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563302" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563302">
        <p class="paperTitle">Comparison of Tug-of-War Models Assuming Moran versus Branching Process Population Dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563302" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563302" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dinh, K. N.; Kurpas, M. K.; Kimmel, M.</p>
        <p class="info">Score: 8.9, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563302' target='https://doi.org/10.1101/2023.10.20.563302'> 10.1101/2023.10.20.563302</a></p>
        <p class="abstract">Mutations arising during cancer evolution are typically categorized as either  drivers or  passengers, depending on whether they increase the cell fitness. Recently, McFarland et al. introduced the tug-of-war model for the joint effect of rare advantageous drivers and frequent but deleterious passengers. We examine this model under two common but distinct frameworks, the Moran model and the branching process. We show that frequently used statistics are similar between a version of the Moran model and the branching process conditioned on the final cell count, under different selection scenarios. We infer the selection coefficients for three breast cancer samples, resulting in good fits of the shape of their Site Frequency Spectra. All fitted values for the selective disadvantage of passenger mutations are nonzero, supporting the view that they exert deleterious selection during tumorigenesis that driver mutations must compensate.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562670">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562670" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562670">
        <p class="paperTitle">Interactions between ploidy and resource availability shape clonal interference at initiation and recurrence of glioblastoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562670" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562670" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nowicka, Z.; Rentzeperis, F.; Beck, R.; Tagal, V.; Forero Pinto, A. M.; Scanu, E.; Veith, T.; Cole, J.; Ilter, D.; Dominguez Viqueira, W.; Teer, J. K.; Maksin, K.; Pasetto, S.; Abdalah, M. A.; Fiandaca, G.; Prabhakaran, S.; Schultz, A.; Ojwang, M.; Barnholtz-Sloan, J. S.; Joaquim, F. M.; Gomes, A. P.; Katira, P.; Andor, N.</p>
        <p class="info">Score: 15.7, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562670' target='https://doi.org/10.1101/2023.10.17.562670'> 10.1101/2023.10.17.562670</a></p>
        <p class="abstract">Glioblastoma (GBM) is the most aggressive form of primary brain tumor. Complete surgical resection of GBM is almost impossible due to the infiltrative nature of the cancer. While no evidence for recent selection events have been found after diagnosis, the selective forces that govern gliomagenesis are strong, shaping the tumors cell composition during the initial progression to malignancy with late consequences for invasiveness and therapy response. We present a mathematical model that simulates the growth and invasion of a glioma, given its ploidy level and the nature of its brain tissue micro-environment (TME), and use it to make inferences about GBM initiation and response to standard-of-care treatment. We approximate the spatial distribution of resource access in the TME through integration of in-silico modelling, multi-omics data and image analysis of primary and recurrent GBM. In the pre-malignant setting, our in-silico results suggest that low ploidy cancer cells are more resistant to starvation-induced cell death. In the malignant setting, between first and second surgery, simulated tumors with different ploidy compositions progressed at different rates. Whether higher ploidy predicted fast recurrence, however, depended on the TME. Historical data supports this dependence on TME resources, as shown by a significant correlation between the median glucose uptake rates in human tissues and the median ploidy of cancer types that arise in the respective tissues (Spearman r = -0.70; P = 0.026). Taken together our findings suggest that availability of metabolic substrates in the TME drives different cell fate decisions for cancer cells with different ploidy and shapes GBM disease initiation and relapse characteristics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565179">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565179" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565179">
        <p class="paperTitle">Molecular subtypes of high grade serous ovarian cancer across racial groups and gene expression platforms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565179" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565179" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davidson, N. R.; Barnard, M. E.; Hippen, A. A.; Campbell, A.; Johnson, C.; Way, G. P.; Dalley, B. K.; Berchuck, A.; Salas, L. A.; Peres, L. C.; Marks, J. R.; Schildkraut, J. M.; Greene, C. S.; Doherty, J. A.</p>
        <p class="info">Score: 7.7, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565179' target='https://doi.org/10.1101/2023.11.01.565179'> 10.1101/2023.11.01.565179</a></p>
        <p class="abstract">IntroductionBlack individuals with ovarian cancer experience poorer survival compared to non-Hispanic White individuals. High-grade serous carcinoma (HGSC) is the most common ovarian cancer histotype, with gene expression subtypes that are associated with differential survival. We characterized HGSC gene expression in Black individuals and considered whether gene expression differences by race may contribute to disparities in survival.

MethodsWe performed gene expression analyses using RNA-Seq data from Black and White individuals with HGSC, as well as array-based genotyping data from four existing studies of HGSC. Our main analysis assigned subtypes by identifying dataset-specific clusters using K-means clustering for K=2-4. The cluster- and dataset-specific gene expression patterns were summarized by moderated t-scores that differentiate an individual cluster from all other clusters within each dataset. We compared the calculated gene expression patterns for each cluster across datasets by calculating the Pearson correlation between the two summarized vectors of moderated t-scores. Following K=4 subtype assignment and mapping to The Cancer Genome Atlas (TCGA)-derived HGSC subtypes, we used multivariable-adjusted Cox proportional hazards models to estimate subtype-specific survival separately for each dataset.

ResultsWhen we compared HGSC gene expression between RNA-Seq data and array-based gene expression data, cluster-specific gene expression was similar across data types. Comparing the Black study population to White and Japanese study populations, the immunoreactive subtype was more common (39% versus 23%-28%) and the differentiated subtype less common (7% versus 22%-31%). Patterns of subtype-specific survival were similar between the Black and White populations with RNAseq data; when compared to mesenchymal HGSC, the risk of death was similar for proliferative and differentiated cases and lower, but not statistically significantly lower, for immunoreactive cases (HR=0.79 [0.55, 1.13] in the Black population and HR=0.86 [0.62, 1.19] in the White population). Patterns of subtype-specific survival were also similar in a Japanese population and in two of three predominantly White populations with array-based gene expression data.

ConclusionsA single pipeline can be used to assign gene expression subtypes to HGSC with array-based or RNA-Seq gene expression data. We observed that while the prevalence of the immunoreactive subtype was higher, and the prevalence of the differentiated subtype was lower among Black individuals compared to White individuals, subtype-specific survival was similar.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
         Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
