<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Ataxia-telangiectasia mutated (Atm) disruption sensitizes spatially-directed H3.3K27M/TP53 diffuse midline gliomas to radiation therapy
Authors: Mangoli, A.; Wu, S.; Liu, H. Q.; Aksu, M.; Jain, V. J.; Foreman, B. E.; Regal, J. A.; Weidenhammer, L. B.; Stewart, C. E.; Guerra Garcia, M. E.; Hocke, E.; Abramson, K.; Williams, N. T.; Luo, L.; Deland, K.; Attardi, L.; Abe, K.; Hashizume, R.; Ashley, D. M.; Becher, O. J.; Kirsch, D. G.; Gregory, S.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Ataxia-telangiectasia mutated (Atm) disruption sensitizes spatially-directed H3.3K27M/TP53 diffuse midline gliomas to radiation therapy
Authors: Mangoli, A.; Wu, S.; Liu, H. Q.; Aksu, M.; Jain, V. J.; Foreman, B. E.; Regal, J. A.; Weidenhammer, L. B.; Stewart, C. E.; Guerra Garcia, M. E.; Hocke, E.; Abramson, K.; Williams, N. T.; Luo, L.; Deland, K.; Attardi, L.; Abe, K.; Hashizume, R.; Ashley, D. M.; Becher, O. J.; Kirsch, D. G.; Gregory, S." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-29T10:37:40+00:00" />
<meta property="article:modified_time" content="2023-10-29T10:37:40+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Ataxia-telangiectasia mutated (Atm) disruption sensitizes spatially-directed H3.3K27M/TP53 diffuse midline gliomas to radiation therapy
Authors: Mangoli, A.; Wu, S.; Liu, H. Q.; Aksu, M.; Jain, V. J.; Foreman, B. E.; Regal, J. A.; Weidenhammer, L. B.; Stewart, C. E.; Guerra Garcia, M. E.; Hocke, E.; Abramson, K.; Williams, N. T.; Luo, L.; Deland, K.; Attardi, L.; Abe, K.; Hashizume, R.; Ashley, D. M.; Becher, O. J.; Kirsch, D. G.; Gregory, S."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Ataxia-telangiectasia mutated (Atm) disruption sensitizes spatially-directed H3.3K27M/TP53 diffuse midline gliomas to radiation therapy\nAuthors: Mangoli, A.; Wu, S.; Liu, H. Q.; Aksu, M.; Jain, V. J.; Foreman, B. E.; Regal, J. A.; Weidenhammer, L. B.; Stewart, C. E.; Guerra Garcia, M. E.; Hocke, E.; Abramson, K.; Williams, N. T.; Luo, L.; Deland, K.; Attardi, L.; Abe, K.; Hashizume, R.; Ashley, D. M.; Becher, O. J.; Kirsch, D. G.; Gregory, S.",
  "keywords": [
    
  ],
  "articleBody": " Ataxia-telangiectasia mutated (Atm) disruption sensitizes spatially-directed H3.3K27M/TP53 diffuse midline gliomas to radiation therapy\nAuthors: Mangoli, A.; Wu, S.; Liu, H. Q.; Aksu, M.; Jain, V. J.; Foreman, B. E.; Regal, J. A.; Weidenhammer, L. B.; Stewart, C. E.; Guerra Garcia, M. E.; Hocke, E.; Abramson, K.; Williams, N. T.; Luo, L.; Deland, K.; Attardi, L.; Abe, K.; Hashizume, R.; Ashley, D. M.; Becher, O. J.; Kirsch, D. G.; Gregory, S. G.; Reitman, Z. J.\nScore: 282.8, Published: 2023-10-20 DOI: 10.1101/2023.10.18.562892\nDiffuse midline gliomas (DMGs) are lethal brain tumors characterized by p53-inactivating mutations and oncohistone H3.3K27M mutations that rewire the cellular response to genotoxic stress, which presents therapeutic opportunities. We used RCAS/tv-a retroviruses and Cre recombinase to inactivate p53 and induce K27M in the native H3f3a allele in a lineage- and spatially-directed manner, yielding primary mouse DMGs. Genetic or pharmacologic disruption of the DNA damage response kinase Ataxia-telangiectasia mutated (ATM) enhanced the efficacy of focal brain irradiation, extending mouse survival. This finding suggests that targeting ATM will enhance the efficacy of radiation therapy for p53-mutant DMG but not p53-wildtype DMG. We used spatial in situ transcriptomics and an allelic series of primary murine DMG models with different p53 mutations to identify transactivation-independent p53 activity as a key mediator of such radiosensitivity. These studies deeply profile a genetically faithful and versatile model of a lethal brain tumor to identify resistance mechanisms for a therapeutic strategy currently in clinical trials.\nPrograms, Origins, and Niches of Immunomodulatory Myeloid Cells in Gliomas\nAuthors: Miller, T. E.; El Farran, C. A. K.; Couturier, C. P.; Chen, Z.; D'Antonio, J. P.; Verga, J.; Villanueva, M. A.; Castro, L. N. G.; Tong, Y. E.; Al-Saadi, T.; Chiocca, A. N.; Fischer, D. S.; Heiland, D. H.; Guerriero, J. L.; Petrecca, K.; Suva, M. L.; Shalek, A. K.; Bernstein, B. E.\nScore: 15.4, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563466\nGliomas are incurable malignancies notable for an immunosuppressive microenvironment with abundant myeloid cells whose immunomodulatory properties remain poorly defined. Here, utilizing scRNA-seq data for 183,062 myeloid cells from 85 human tumors, we discover that nearly all glioma-associated myeloid cells express at least one of four immunomodulatory activity programs: Scavenger Immunosuppressive, C1Q Immunosuppressive, CXCR4 Inflammatory, and IL1B Inflammatory. All four programs are present in IDH1 mutant and wild-type gliomas and are expressed in macrophages, monocytes, and microglia whether of blood or resident myeloid cell origins. Integrating our scRNA-seq data with mitochondrial DNA-based lineage tracing, spatial transcriptomics, and organoid explant systems that model peripheral monocyte infiltration, we show that these programs are driven by microenvironmental cues and therapies rather than myeloid cell type, origin, or mutation status. The C1Q Immunosuppressive program is driven by routinely administered dexamethasone. The Scavenger Immunosuppressive program includes ligands with established roles in T-cell suppression, is induced in hypoxic regions, and is associated with immunotherapy resistance. Both immunosuppressive programs are less prevalent in lower-grade gliomas, which are instead enriched for the CXCR4 Inflammatory program. Our study provides a framework to understand immunomodulatory myeloid cells in glioma, and a foundation to develop more effective immunotherapies.\nBorrowing Transcriptional Kinases to Activate Apoptosis\nAuthors: Sarott, R. C.; Gourisankar, S. P.; Karim, B. A.; Nettles, S.; Yang, H.; Dwyer, B.; Simanauskaite, J. M.; Tse, J.; Abuzaid, H.; Krokhotin, A.; Zhang, T.; Hinshaw, S. M.; Green, M. R.; Crabtree, G. R.; Gray, N.\nScore: 15.4, Published: 2023-10-25 DOI: 10.1101/2023.10.23.563687\nProtein kinases are disease drivers whose therapeutic targeting traditionally centers on inhibition of enzymatic activity. Here chemically induced proximity is leveraged to convert kinase inhibitors into context-specific activators of therapeutic genes. Bivalent molecules that link ligands of the transcription factor B-cell lymphoma 6 (BCL6) to ATP-competitive inhibitors of cyclin-dependent kinases (CDKs) were developed to re-localize CDK to BCL6-bound loci on chromatin and direct phosphorylation of RNA Pol II. The resulting BCL6-target proapoptotic gene expression translated into killing of diffuse large B-cell lymphoma (DLBCL) cells at 72 h with EC50s of 0.9 - 10 nM and highly specific ablation of the BCL6-regulated germinal center response in mice. The molecules exhibited 10,000-fold lower cytotoxicity in normal lymphocytes and are well tolerated in mice. Genomic and proteomic evidence corroborated a gain-of-function mechanism where, instead of global enzyme inhibition, a fraction of total kinase activity is borrowed and re-localized to BCL6-bound loci. The strategy demonstrates how kinase inhibitors can be used to context-specifically activate transcription, accessing new therapeutic space.\nTumor-wide RNA splicing aberrations generate immunogenic public neoantigens\nAuthors: Kwok, D. W.; Stevers, N. O.; Nejo, T.; Chen, L. H.; Etxeberria, I.; Jung, J.; Okada, K.; Colton Cove, M.; Lakshmanachetty, S.; Gallus, M.; Barpanda, A.; Hong, C.; Chan, G. K. L.; Wu, S. H.; Ramos, E.; Yamamichi, A.; Liu, J. Y. Z.; Watchmaker, P.; Ogino, H.; Saijo, A.; Du, A.; Grishanina, N.; Woo, J.; Diaz, A.; Chang, S. M.; Phillips, J. J.; Wiita, A.; Klebanoff, C. A.; Costello, J. F.; Okada, H.\nScore: 17.1, Published: 2023-10-20 DOI: 10.1101/2023.10.19.563178\nT-cell-mediated immunotherapies are limited by the extent to which cancer-specific antigens are homogenously expressed throughout a tumor. We reasoned that recurrent splicing aberrations in cancer represent a potential source of tumor-wide and public neoantigens, and to test this possibility, we developed a novel pipeline for identifying neojunctions expressed uniformly within a tumor across diverse cancer types. Our analyses revealed multiple neojunctions that recur across patients and either exhibited intratumor heterogeneity or, in some cases, were tumor-wide. We identified CD8+ T-cell clones specific for neoantigens derived from tumor-wide and conserved neojunctions in GNAS and RPL22, respectively. TCR-engineered CD8+ T-cells targeting these mutations conferred neoantigen-specific tumor cell eradication. Furthermore, we revealed that cancer-specific dysregulation in splicing factor expression leads to recurrent neojunction expression. Together, these data reveal that a subset of neojunctions are both intratumorally conserved and public, providing the molecular basis for novel T-cell-based immunotherapies that address intratumoral heterogeneity.\nAndrogen blockade primes NLRP3 in macrophages to induce tumor phagocytosis\nAuthors: Chaudagar, K. K.; Srikrishnan, R.; Mei, S.; Hirz, T.; Hu, Y.-M.; Argulian, A.; Labadie, B.; Desai, K.; Grimaldo, S.; Kahramangil, D.; Nair, R.; D Souza, S.; Zhou, D.; Li, M.; Doughan, F.; Chen, R.; Shafran, J.; Loyd, M.; Xia, Z.; Sykes, D. B.; Moran, A. E.; Patnaik, A.\nScore: 11.0, Published: 2023-10-21 DOI: 10.1101/2023.09.15.557996\nImmune-based therapies induce durable remissions in subsets of patients across multiple malignancies. However, there is limited efficacy of immunotherapy in metastatic castrate-resistant prostate cancer (mCRPC), manifested by an enrichment of immunosuppressive (M2) tumor- associated macrophages (TAM) in the tumor immune microenvironment (TME). Therefore, therapeutic strategies to overcome TAM-mediated immunosuppression are critically needed in mCRPC. Here we discovered that NLR family pyrin domain containing 3 (NLRP3), an innate immune sensing protein, is highly expressed in TAM from metastatic PC patients treated with standard-of-care androgen deprivation therapy (ADT). Importantly, ex vivo studies revealed that androgen receptor (AR) blockade in TAM upregulates NLRP3 expression, but not inflammasome activity, and concurrent AR blockade/NLRP3 agonist (NLRP3a) treatment promotes cancer cell phagocytosis by immunosuppressive M2 TAM. In contrast, NLRP3a monotherapy was sufficient to enhance phagocytosis of cancer cells in anti-tumor (M1) TAM, which exhibit high de novo NLRP3 expression. Critically, combinatorial treatment with ADT/NLRP3a in a murine model of advanced PC resulted in significant tumor control, with tumor clearance in 55% of mice via TAM phagocytosis. Collectively, our results demonstrate NLRP3 as an AR-regulated \"macrophage phagocytic checkpoint\", inducibly expressed in TAM by ADT and activated by NLRP3a treatment, the combination resulting in TAM-mediated phagocytosis and tumor control.\nMapping ovarian cancer spatial organization uncovers immune evasion drivers at the genetic, cellular, and tissue level\nAuthors: Yeh, C. Y.; Aguirre, K.; Laveroni, O.; Kim, S.; Wang, A.; Liang, B.; Zhang, X.; Han, L. M.; Valbuena, R.; Plevritis, S. K.; Bassik, M. C.; Snyder, M. P.; Howitt, B. E.; Jerby, L.\nScore: 78.4, Published: 2023-10-19 DOI: 10.1101/2023.10.16.562592\nImmune exclusion and evasion are central barriers to the success of immunotherapies and cell therapies in solid tumors. Here we applied single cell spatial and perturbational transcriptomics alongside clinical, histological, and genomic profiling to elucidate immune exclusion and evasion in high-grade serous tubo-ovarian cancer (HGSC). Using high-plex spatial transcriptomics we profiled more than 1.3 million cells from 95 tumors and 60 patients, revealing generalizable principles in HGSC tumor tissue organization. Our data demonstrates that effector T cells resist stroma-mediated trapping and sequestration. However, upon infiltration into the tumor, T cells, as well as Natural Killer (NK) cells, preferentially co-localize only with a subset of malignant cells that manifest a distinct transcriptional cell state. The latter consists of dozens of co-regulated genes and is repressed under various copy number alterations. Performing CRISPR Perturb-seq screens in ovarian cancer cells, we identified functionally diverse genetic perturbations - including knockout of the insulin sensing repressor PTPN1 and the epigenetic regulator ACTR8 - that de-repress the proposed immunogenic malignant cell state identified in patients and indeed sensitize ovarian cancer cells to T cell and NK cell cytotoxicity. Taken together, our study uncovered a profound connection between somatic genetic aberrations, malignant cell transcriptional dysregulation, and immune evasion at the cellular and tissue level, allowing us to identify targets that reprogram malignant cell states as an avenue to unleash anti-tumor immune responses.\nInteractions between ploidy and resource availability shape clonal interference at initiation and recurrence of glioblastoma\nAuthors: Nowicka, Z.; Rentzeperis, F.; Beck, R.; Tagal, V.; Forero Pinto, A. M.; Scanu, E.; Veith, T.; Cole, J.; Ilter, D.; Dominguez Viqueira, W.; Teer, J. K.; Maksin, K.; Pasetto, S.; Abdalah, M. A.; Fiandaca, G.; Prabhakaran, S.; Schultz, A.; Ojwang, M.; Barnholtz-Sloan, J. S.; Joaquim, F. M.; Gomes, A. P.; Katira, P.; Andor, N.\nScore: 6.8, Published: 2023-10-20 DOI: 10.1101/2023.10.17.562670\nGlioblastoma (GBM) is the most aggressive form of primary brain tumor. Complete surgical resection of GBM is almost impossible due to the infiltrative nature of the cancer. While no evidence for recent selection events have been found after diagnosis, the selective forces that govern gliomagenesis are strong, shaping the tumors cell composition during the initial progression to malignancy with late consequences for invasiveness and therapy response. We present a mathematical model that simulates the growth and invasion of a glioma, given its ploidy level and the nature of its brain tissue micro-environment (TME), and use it to make inferences about GBM initiation and response to standard-of-care treatment. We approximate the spatial distribution of resource access in the TME through integration of in-silico modelling, multi-omics data and image analysis of primary and recurrent GBM. In the pre-malignant setting, our in-silico results suggest that low ploidy cancer cells are more resistant to starvation-induced cell death. In the malignant setting, between first and second surgery, simulated tumors with different ploidy compositions progressed at different rates. Whether higher ploidy predicted fast recurrence, however, depended on the TME. Historical data supports this dependence on TME resources, as shown by a significant correlation between the median glucose uptake rates in human tissues and the median ploidy of cancer types that arise in the respective tissues (Spearman r = -0.70; P = 0.026). Taken together our findings suggest that availability of metabolic substrates in the TME drives different cell fate decisions for cancer cells with different ploidy and shapes GBM disease initiation and relapse characteristics.\nDosage amplification dictates oncogenic regulation by the NKX2-1 lineage factor in lung adenocarcinoma\nAuthors: Pulice, J. L.; Meyerson, M.\nScore: 6.2, Published: 2023-10-27 DOI: 10.1101/2023.10.26.563996\nAmplified oncogene expression is a critical and widespread driver event in cancer, yet our understanding of how amplification-mediated elevated dosage mediates oncogenic regulation is limited. Here, we find that the most significant focal amplification event in lung adenocarcinoma (LUAD) targets a lineage super-enhancer near the NKX2-1 lineage transcription factor. The NKX2-1 super-enhancer is targeted by focal and co-amplification with NKX2-1, and activation or repression controls NKX2-1 expression. We find that NKX2-1 is a widespread dependency in LUAD cell lines, where NKX2-1 pioneers enhancer accessibility to drive a lineage addicted state in LUAD, and NKX2-1 confers persistence to EGFR inhibitors. Notably, we find that oncogenic NKX2-1 regulation requires expression above a minimum dosage threshold: NKX2-1 dosage below this threshold is insufficient for cell viability, enhancer remodeling, and TKI persistence. Our data suggest that copy-number amplification can be a gain-of-function alteration, wherein amplification elevates oncogene expression above a critical dosage required for oncogenic regulation and cancer cell survival.\nMissense mutations in Myc Box I influence MYC cellular localization, mRNA partitioning and turnover to promote leukemogenesis\nAuthors: Arthur, N.; Christensen, K. A.; Mannino, K.; Ruzinova, M.; Kumar, A.; Gruszczynska, A.; Day, R. B.; Erdmann-Gilmore, P.; Mi, Y.; Sprung, R.; York, C. R.; Spencer, D. H.; Townsend, R.; Sykes, S. M.; Ferraro, F.\nScore: 6.1, Published: 2023-10-24 DOI: 10.1101/2023.10.22.563493\nSomatic missense mutations in the phosphodegron domain of the MYC gene (MYC Box I) are detected in the dominant clones of a subset of acute myeloid leukemia (AML) patients, but the mechanisms by which they contribute to AML are unknown. To unveil unique proprieties of MBI MYC mutant proteins, we systematically compared the cellular and molecular consequences of expressing similar oncogenic levels of wild type and MBI mutant MYC. We found that MBI MYC mutants can accelerate leukemia by driving unique transcriptional signatures in highly selected, myeloid progenitor subpopulations. Although these mutations increase MYC stability, they overall dampen MYC chromatin localization and lead to a cytoplasmic accumulation of the mutant proteins. This phenotype is coupled with increased translation of RNA binding proteins and nuclear export machinery, which results in altered RNA partitioning and accelerated decay of select transcripts encoding proapoptotic and proinflammatory genes. Heterozygous knockin mice harboring the germline MBI mutation Myc p.T73N exhibit cytoplasmic MYC localization, myeloid progenitors expansion with similar transcriptional signatures to the overexpression model, and eventually develop hematological malignancies. This study uncovers that MBI MYC mutations alter MYC localization and disrupt mRNA subcellular distribution and turnover of select transcripts to accelerate tumor initiation and growth.\nClinically relevant humanized mouse models of metastatic prostate cancer to evaluate cancer therapies\nAuthors: Kostlan, R. J.; Phoenix, J. T.; Budreika, A.; Ferrari, M. G.; Khurana, N.; Choi, J. E.; Juckette, K.; McCollum, B. L.; Moskal, R.; Qiao, Y.; Vander Griend, D. J.; Chinnaiyan, A. M.; Kregel, S.\nScore: 10.1, Published: 2023-10-18 DOI: 10.1101/2023.10.13.562280\nThere is tremendous need for improved prostate cancer (PCa) models. The mouse prostate does not spontaneously form tumors and is anatomically and developmentally different from the human prostate. Engineered mouse models lack the heterogeneity of human cancer and rarely establish metastatic growth. Human xenografts represent an alternative but rely on an immunocompromised host. Accordingly, we generated PCa murine xenograft models with an intact human immune system (huNOG and huNOG-EXL mice) to test whether humanizing tumor-immune interactions would improve modeling of metastatic PCa and the impact of hormonal and immunotherapies. These mice maintain multiple human cell lineages, including functional human T-cells and myeloid cells. In 22Rv1 xenografts, subcutaneous tumor size was not significantly altered across conditions; however, metastasis to secondary sites differed in castrate huNOG vs background-matched immunocompromised mice treated with enzalutamide (enza). VCaP xenograft tumors showed decreases in growth with enza and anti-Programed-Death-1 treatments in huNOG mice, and no effect was seen with treatment in NOG mice. Enza responses in huNOG and NOG mice were distinct and associated with increased T-cells within tumors of enza treated huNOG mice, and increased T-cell activation. In huNOG-EXL mice, which support human myeloid development, there was a strong population of immunosuppressive regulatory T-cells and Myeloid-Derived-Suppressor-Cells (MDSCs), and enza treatment showed no difference in metastasis. Results illustrate, to our knowledge, the first model of human PCa that metastasizes to clinically relevant locations, has an intact human immune system, responds appropriately to standard-of-care hormonal therapies, and can model both an immunosuppressive and checkpoint-inhibition responsive immune microenvironment.\n",
  "wordCount" : "2579",
  "inLanguage": "en",
  "datePublished": "2023-10-29T10:37:40Z",
  "dateModified": "2023-10-29T10:37:40Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on October 29, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562892">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562892" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562892">
        <p class="paperTitle">Ataxia-telangiectasia mutated (Atm) disruption sensitizes spatially-directed H3.3K27M/TP53 diffuse midline gliomas to radiation therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562892" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562892" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mangoli, A.; Wu, S.; Liu, H. Q.; Aksu, M.; Jain, V. J.; Foreman, B. E.; Regal, J. A.; Weidenhammer, L. B.; Stewart, C. E.; Guerra Garcia, M. E.; Hocke, E.; Abramson, K.; Williams, N. T.; Luo, L.; Deland, K.; Attardi, L.; Abe, K.; Hashizume, R.; Ashley, D. M.; Becher, O. J.; Kirsch, D. G.; Gregory, S. G.; Reitman, Z. J.</p>
        <p class="info">Score: 282.8, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562892' target='https://doi.org/10.1101/2023.10.18.562892'> 10.1101/2023.10.18.562892</a></p>
        <p class="abstract">Diffuse midline gliomas (DMGs) are lethal brain tumors characterized by p53-inactivating mutations and oncohistone H3.3K27M mutations that rewire the cellular response to genotoxic stress, which presents therapeutic opportunities. We used RCAS/tv-a retroviruses and Cre recombinase to inactivate p53 and induce K27M in the native H3f3a allele in a lineage- and spatially-directed manner, yielding primary mouse DMGs. Genetic or pharmacologic disruption of the DNA damage response kinase Ataxia-telangiectasia mutated (ATM) enhanced the efficacy of focal brain irradiation, extending mouse survival. This finding suggests that targeting ATM will enhance the efficacy of radiation therapy for p53-mutant DMG but not p53-wildtype DMG. We used spatial in situ transcriptomics and an allelic series of primary murine DMG models with different p53 mutations to identify transactivation-independent p53 activity as a key mediator of such radiosensitivity. These studies deeply profile a genetically faithful and versatile model of a lethal brain tumor to identify resistance mechanisms for a therapeutic strategy currently in clinical trials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563466">
        <p class="paperTitle">Programs, Origins, and Niches of Immunomodulatory Myeloid Cells in Gliomas</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Miller, T. E.; El Farran, C. A. K.; Couturier, C. P.; Chen, Z.; D&#39;Antonio, J. P.; Verga, J.; Villanueva, M. A.; Castro, L. N. G.; Tong, Y. E.; Al-Saadi, T.; Chiocca, A. N.; Fischer, D. S.; Heiland, D. H.; Guerriero, J. L.; Petrecca, K.; Suva, M. L.; Shalek, A. K.; Bernstein, B. E.</p>
        <p class="info">Score: 15.4, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563466' target='https://doi.org/10.1101/2023.10.24.563466'> 10.1101/2023.10.24.563466</a></p>
        <p class="abstract">Gliomas are incurable malignancies notable for an immunosuppressive microenvironment with abundant myeloid cells whose immunomodulatory properties remain poorly defined. Here, utilizing scRNA-seq data for 183,062 myeloid cells from 85 human tumors, we discover that nearly all glioma-associated myeloid cells express at least one of four immunomodulatory activity programs: Scavenger Immunosuppressive, C1Q Immunosuppressive, CXCR4 Inflammatory, and IL1B Inflammatory. All four programs are present in IDH1 mutant and wild-type gliomas and are expressed in macrophages, monocytes, and microglia whether of blood or resident myeloid cell origins. Integrating our scRNA-seq data with mitochondrial DNA-based lineage tracing, spatial transcriptomics, and organoid explant systems that model peripheral monocyte infiltration, we show that these programs are driven by microenvironmental cues and therapies rather than myeloid cell type, origin, or mutation status. The C1Q Immunosuppressive program is driven by routinely administered dexamethasone. The Scavenger Immunosuppressive program includes ligands with established roles in T-cell suppression, is induced in hypoxic regions, and is associated with immunotherapy resistance. Both immunosuppressive programs are less prevalent in lower-grade gliomas, which are instead enriched for the CXCR4 Inflammatory program. Our study provides a framework to understand immunomodulatory myeloid cells in glioma, and a foundation to develop more effective immunotherapies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.23.563687">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.23.563687" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.23.563687">
        <p class="paperTitle">Borrowing Transcriptional Kinases to Activate Apoptosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.23.563687" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.23.563687" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sarott, R. C.; Gourisankar, S. P.; Karim, B. A.; Nettles, S.; Yang, H.; Dwyer, B.; Simanauskaite, J. M.; Tse, J.; Abuzaid, H.; Krokhotin, A.; Zhang, T.; Hinshaw, S. M.; Green, M. R.; Crabtree, G. R.; Gray, N.</p>
        <p class="info">Score: 15.4, Published: 2023-10-25 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.23.563687' target='https://doi.org/10.1101/2023.10.23.563687'> 10.1101/2023.10.23.563687</a></p>
        <p class="abstract">Protein kinases are disease drivers whose therapeutic targeting traditionally centers on inhibition of enzymatic activity. Here chemically induced proximity is leveraged to convert kinase inhibitors into context-specific activators of therapeutic genes. Bivalent molecules that link ligands of the transcription factor B-cell lymphoma 6 (BCL6) to ATP-competitive inhibitors of cyclin-dependent kinases (CDKs) were developed to re-localize CDK to BCL6-bound loci on chromatin and direct phosphorylation of RNA Pol II. The resulting BCL6-target proapoptotic gene expression translated into killing of diffuse large B-cell lymphoma (DLBCL) cells at 72 h with EC50s of 0.9 - 10 nM and highly specific ablation of the BCL6-regulated germinal center response in mice. The molecules exhibited 10,000-fold lower cytotoxicity in normal lymphocytes and are well tolerated in mice. Genomic and proteomic evidence corroborated a gain-of-function mechanism where, instead of global enzyme inhibition, a fraction of total kinase activity is borrowed and re-localized to BCL6-bound loci. The strategy demonstrates how kinase inhibitors can be used to context-specifically activate transcription, accessing new therapeutic space.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.19.563178">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.19.563178" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.19.563178">
        <p class="paperTitle">Tumor-wide RNA splicing aberrations generate immunogenic public neoantigens</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.19.563178" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.19.563178" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kwok, D. W.; Stevers, N. O.; Nejo, T.; Chen, L. H.; Etxeberria, I.; Jung, J.; Okada, K.; Colton Cove, M.; Lakshmanachetty, S.; Gallus, M.; Barpanda, A.; Hong, C.; Chan, G. K. L.; Wu, S. H.; Ramos, E.; Yamamichi, A.; Liu, J. Y. Z.; Watchmaker, P.; Ogino, H.; Saijo, A.; Du, A.; Grishanina, N.; Woo, J.; Diaz, A.; Chang, S. M.; Phillips, J. J.; Wiita, A.; Klebanoff, C. A.; Costello, J. F.; Okada, H.</p>
        <p class="info">Score: 17.1, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.19.563178' target='https://doi.org/10.1101/2023.10.19.563178'> 10.1101/2023.10.19.563178</a></p>
        <p class="abstract">T-cell-mediated immunotherapies are limited by the extent to which cancer-specific antigens are homogenously expressed throughout a tumor. We reasoned that recurrent splicing aberrations in cancer represent a potential source of tumor-wide and public neoantigens, and to test this possibility, we developed a novel pipeline for identifying neojunctions expressed uniformly within a tumor across diverse cancer types. Our analyses revealed multiple neojunctions that recur across patients and either exhibited intratumor heterogeneity or, in some cases, were tumor-wide. We identified CD8&#43; T-cell clones specific for neoantigens derived from tumor-wide and conserved neojunctions in GNAS and RPL22, respectively. TCR-engineered CD8&#43; T-cells targeting these mutations conferred neoantigen-specific tumor cell eradication. Furthermore, we revealed that cancer-specific dysregulation in splicing factor expression leads to recurrent neojunction expression. Together, these data reveal that a subset of neojunctions are both intratumorally conserved and public, providing the molecular basis for novel T-cell-based immunotherapies that address intratumoral heterogeneity.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.15.557996">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.15.557996" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.15.557996">
        <p class="paperTitle">Androgen blockade primes NLRP3 in macrophages to induce tumor phagocytosis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.15.557996" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.15.557996" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Chaudagar, K. K.; Srikrishnan, R.; Mei, S.; Hirz, T.; Hu, Y.-M.; Argulian, A.; Labadie, B.; Desai, K.; Grimaldo, S.; Kahramangil, D.; Nair, R.; D Souza, S.; Zhou, D.; Li, M.; Doughan, F.; Chen, R.; Shafran, J.; Loyd, M.; Xia, Z.; Sykes, D. B.; Moran, A. E.; Patnaik, A.</p>
        <p class="info">Score: 11.0, Published: 2023-10-21 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.15.557996' target='https://doi.org/10.1101/2023.09.15.557996'> 10.1101/2023.09.15.557996</a></p>
        <p class="abstract">Immune-based therapies induce durable remissions in subsets of patients across multiple malignancies. However, there is limited efficacy of immunotherapy in metastatic castrate-resistant prostate cancer (mCRPC), manifested by an enrichment of immunosuppressive (M2) tumor- associated macrophages (TAM) in the tumor immune microenvironment (TME). Therefore, therapeutic strategies to overcome TAM-mediated immunosuppression are critically needed in mCRPC. Here we discovered that NLR family pyrin domain containing 3 (NLRP3), an innate immune sensing protein, is highly expressed in TAM from metastatic PC patients treated with standard-of-care androgen deprivation therapy (ADT). Importantly, ex vivo studies revealed that androgen receptor (AR) blockade in TAM upregulates NLRP3 expression, but not inflammasome activity, and concurrent AR blockade/NLRP3 agonist (NLRP3a) treatment promotes cancer cell phagocytosis by immunosuppressive M2 TAM. In contrast, NLRP3a monotherapy was sufficient to enhance phagocytosis of cancer cells in anti-tumor (M1) TAM, which exhibit high de novo NLRP3 expression. Critically, combinatorial treatment with ADT/NLRP3a in a murine model of advanced PC resulted in significant tumor control, with tumor clearance in 55% of mice via TAM phagocytosis. Collectively, our results demonstrate NLRP3 as an AR-regulated &#34;macrophage phagocytic checkpoint&#34;, inducibly expressed in TAM by ADT and activated by NLRP3a treatment, the combination resulting in TAM-mediated phagocytosis and tumor control.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.16.562592">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.16.562592" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.16.562592">
        <p class="paperTitle">Mapping ovarian cancer spatial organization uncovers immune evasion drivers at the genetic, cellular, and tissue level</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.16.562592" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.16.562592" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Yeh, C. Y.; Aguirre, K.; Laveroni, O.; Kim, S.; Wang, A.; Liang, B.; Zhang, X.; Han, L. M.; Valbuena, R.; Plevritis, S. K.; Bassik, M. C.; Snyder, M. P.; Howitt, B. E.; Jerby, L.</p>
        <p class="info">Score: 78.4, Published: 2023-10-19 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.16.562592' target='https://doi.org/10.1101/2023.10.16.562592'> 10.1101/2023.10.16.562592</a></p>
        <p class="abstract">Immune exclusion and evasion are central barriers to the success of immunotherapies and cell therapies in solid tumors. Here we applied single cell spatial and perturbational transcriptomics alongside clinical, histological, and genomic profiling to elucidate immune exclusion and evasion in high-grade serous tubo-ovarian cancer (HGSC). Using high-plex spatial transcriptomics we profiled more than 1.3 million cells from 95 tumors and 60 patients, revealing generalizable principles in HGSC tumor tissue organization. Our data demonstrates that effector T cells resist stroma-mediated trapping and sequestration. However, upon infiltration into the tumor, T cells, as well as Natural Killer (NK) cells, preferentially co-localize only with a subset of malignant cells that manifest a distinct transcriptional cell state. The latter consists of dozens of co-regulated genes and is repressed under various copy number alterations. Performing CRISPR Perturb-seq screens in ovarian cancer cells, we identified functionally diverse genetic perturbations - including knockout of the insulin sensing repressor PTPN1 and the epigenetic regulator ACTR8 - that de-repress the proposed immunogenic malignant cell state identified in patients and indeed sensitize ovarian cancer cells to T cell and NK cell cytotoxicity. Taken together, our study uncovered a profound connection between somatic genetic aberrations, malignant cell transcriptional dysregulation, and immune evasion at the cellular and tissue level, allowing us to identify targets that reprogram malignant cell states as an avenue to unleash anti-tumor immune responses.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.17.562670">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.17.562670" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.17.562670">
        <p class="paperTitle">Interactions between ploidy and resource availability shape clonal interference at initiation and recurrence of glioblastoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.17.562670" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.17.562670" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nowicka, Z.; Rentzeperis, F.; Beck, R.; Tagal, V.; Forero Pinto, A. M.; Scanu, E.; Veith, T.; Cole, J.; Ilter, D.; Dominguez Viqueira, W.; Teer, J. K.; Maksin, K.; Pasetto, S.; Abdalah, M. A.; Fiandaca, G.; Prabhakaran, S.; Schultz, A.; Ojwang, M.; Barnholtz-Sloan, J. S.; Joaquim, F. M.; Gomes, A. P.; Katira, P.; Andor, N.</p>
        <p class="info">Score: 6.8, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.17.562670' target='https://doi.org/10.1101/2023.10.17.562670'> 10.1101/2023.10.17.562670</a></p>
        <p class="abstract">Glioblastoma (GBM) is the most aggressive form of primary brain tumor. Complete surgical resection of GBM is almost impossible due to the infiltrative nature of the cancer. While no evidence for recent selection events have been found after diagnosis, the selective forces that govern gliomagenesis are strong, shaping the tumors cell composition during the initial progression to malignancy with late consequences for invasiveness and therapy response. We present a mathematical model that simulates the growth and invasion of a glioma, given its ploidy level and the nature of its brain tissue micro-environment (TME), and use it to make inferences about GBM initiation and response to standard-of-care treatment. We approximate the spatial distribution of resource access in the TME through integration of in-silico modelling, multi-omics data and image analysis of primary and recurrent GBM. In the pre-malignant setting, our in-silico results suggest that low ploidy cancer cells are more resistant to starvation-induced cell death. In the malignant setting, between first and second surgery, simulated tumors with different ploidy compositions progressed at different rates. Whether higher ploidy predicted fast recurrence, however, depended on the TME. Historical data supports this dependence on TME resources, as shown by a significant correlation between the median glucose uptake rates in human tissues and the median ploidy of cancer types that arise in the respective tissues (Spearman r = -0.70; P = 0.026). Taken together our findings suggest that availability of metabolic substrates in the TME drives different cell fate decisions for cancer cells with different ploidy and shapes GBM disease initiation and relapse characteristics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.26.563996">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.26.563996" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.26.563996">
        <p class="paperTitle">Dosage amplification dictates oncogenic regulation by the NKX2-1 lineage factor in lung adenocarcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.26.563996" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.26.563996" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Pulice, J. L.; Meyerson, M.</p>
        <p class="info">Score: 6.2, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.26.563996' target='https://doi.org/10.1101/2023.10.26.563996'> 10.1101/2023.10.26.563996</a></p>
        <p class="abstract">Amplified oncogene expression is a critical and widespread driver event in cancer, yet our understanding of how amplification-mediated elevated dosage mediates oncogenic regulation is limited. Here, we find that the most significant focal amplification event in lung adenocarcinoma (LUAD) targets a lineage super-enhancer near the NKX2-1 lineage transcription factor. The NKX2-1 super-enhancer is targeted by focal and co-amplification with NKX2-1, and activation or repression controls NKX2-1 expression. We find that NKX2-1 is a widespread dependency in LUAD cell lines, where NKX2-1 pioneers enhancer accessibility to drive a lineage addicted state in LUAD, and NKX2-1 confers persistence to EGFR inhibitors. Notably, we find that oncogenic NKX2-1 regulation requires expression above a minimum dosage threshold: NKX2-1 dosage below this threshold is insufficient for cell viability, enhancer remodeling, and TKI persistence. Our data suggest that copy-number amplification can be a gain-of-function alteration, wherein amplification elevates oncogene expression above a critical dosage required for oncogenic regulation and cancer cell survival.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.22.563493">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.22.563493" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.22.563493">
        <p class="paperTitle">Missense mutations in Myc Box I influence MYC cellular localization, mRNA partitioning and turnover to promote leukemogenesis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.22.563493" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.22.563493" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Arthur, N.; Christensen, K. A.; Mannino, K.; Ruzinova, M.; Kumar, A.; Gruszczynska, A.; Day, R. B.; Erdmann-Gilmore, P.; Mi, Y.; Sprung, R.; York, C. R.; Spencer, D. H.; Townsend, R.; Sykes, S. M.; Ferraro, F.</p>
        <p class="info">Score: 6.1, Published: 2023-10-24 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.22.563493' target='https://doi.org/10.1101/2023.10.22.563493'> 10.1101/2023.10.22.563493</a></p>
        <p class="abstract">Somatic missense mutations in the phosphodegron domain of the MYC gene (MYC Box I) are detected in the dominant clones of a subset of acute myeloid leukemia (AML) patients, but the mechanisms by which they contribute to AML are unknown. To unveil unique proprieties of MBI MYC mutant proteins, we systematically compared the cellular and molecular consequences of expressing similar oncogenic levels of wild type and MBI mutant MYC. We found that MBI MYC mutants can accelerate leukemia by driving unique transcriptional signatures in highly selected, myeloid progenitor subpopulations. Although these mutations increase MYC stability, they overall dampen MYC chromatin localization and lead to a cytoplasmic accumulation of the mutant proteins. This phenotype is coupled with increased translation of RNA binding proteins and nuclear export machinery, which results in altered RNA partitioning and accelerated decay of select transcripts encoding proapoptotic and proinflammatory genes. Heterozygous knockin mice harboring the germline MBI mutation Myc p.T73N exhibit cytoplasmic MYC localization, myeloid progenitors expansion with similar transcriptional signatures to the overexpression model, and eventually develop hematological malignancies. This study uncovers that MBI MYC mutations alter MYC localization and disrupt mRNA subcellular distribution and turnover of select transcripts to accelerate tumor initiation and growth.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562280">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562280" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562280">
        <p class="paperTitle">Clinically relevant humanized mouse models of metastatic prostate cancer to evaluate cancer therapies</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562280" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562280" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kostlan, R. J.; Phoenix, J. T.; Budreika, A.; Ferrari, M. G.; Khurana, N.; Choi, J. E.; Juckette, K.; McCollum, B. L.; Moskal, R.; Qiao, Y.; Vander Griend, D. J.; Chinnaiyan, A. M.; Kregel, S.</p>
        <p class="info">Score: 10.1, Published: 2023-10-18 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562280' target='https://doi.org/10.1101/2023.10.13.562280'> 10.1101/2023.10.13.562280</a></p>
        <p class="abstract">There is tremendous need for improved prostate cancer (PCa) models. The mouse prostate does not spontaneously form tumors and is anatomically and developmentally different from the human prostate. Engineered mouse models lack the heterogeneity of human cancer and rarely establish metastatic growth. Human xenografts represent an alternative but rely on an immunocompromised host. Accordingly, we generated PCa murine xenograft models with an intact human immune system (huNOG and huNOG-EXL mice) to test whether humanizing tumor-immune interactions would improve modeling of metastatic PCa and the impact of hormonal and immunotherapies. These mice maintain multiple human cell lineages, including functional human T-cells and myeloid cells. In 22Rv1 xenografts, subcutaneous tumor size was not significantly altered across conditions; however, metastasis to secondary sites differed in castrate huNOG vs background-matched immunocompromised mice treated with enzalutamide (enza). VCaP xenograft tumors showed decreases in growth with enza and anti-Programed-Death-1 treatments in huNOG mice, and no effect was seen with treatment in NOG mice. Enza responses in huNOG and NOG mice were distinct and associated with increased T-cells within tumors of enza treated huNOG mice, and increased T-cell activation. In huNOG-EXL mice, which support human myeloid development, there was a strong population of immunosuppressive regulatory T-cells and Myeloid-Derived-Suppressor-Cells (MDSCs), and enza treatment showed no difference in metastasis. Results illustrate, to our knowledge, the first model of human PCa that metastasizes to clinically relevant locations, has an intact human immune system, responds appropriately to standard-of-care hormonal therapies, and can model both an immunosuppressive and checkpoint-inhibition responsive immune microenvironment.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
