<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization
Authors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.
Score: 31.5, Published: 2023-10-27 DOI: 10.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization
Authors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.
Score: 31.5, Published: 2023-10-27 DOI: 10." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-11-12T10:37:43+00:00" />
<meta property="article:modified_time" content="2023-11-12T10:37:43+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization
Authors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.
Score: 31.5, Published: 2023-10-27 DOI: 10."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization\nAuthors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.\nScore: 31.5, Published: 2023-10-27 DOI: 10.",
  "keywords": [
    
  ],
  "articleBody": " Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization\nAuthors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.\nScore: 31.5, Published: 2023-10-27 DOI: 10.1101/2023.10.24.563749\nTumor-infiltrating macrophages support critical steps in tumor progression, and their accumulation in the tumor microenvironment (TME) is associated with adverse outcomes and therapeutic resistance across human cancers. In the TME, macrophages adopt diverse phenotypic alterations, giving rise to heterogeneous immune activation states and induction of cell cycle. While the transcriptional profiles of these activation states are well-annotated across human cancers, the underlying signals that regulate macrophage heterogeneity and accumulation remain incompletely understood. Here, we leveraged a novel ex vivo organotypic TME (oTME) model of breast cancer, in vivo murine models, and human samples to map the determinants of functional heterogeneity of TME macrophages. We identified a subset of F4/80highSca-1+ self-renewing macrophages maintained by type-I interferon (IFN) signaling and requiring physical contact with cancer-associated fibroblasts. We discovered that the contact-dependent self-renewal of TME macrophages is mediated via Notch4, and its inhibition abrogated tumor growth of breast and ovarian carcinomas in vivo, as well as lung dissemination in a PDX model of triple-negative breast cancer (TNBC). Through spatial multi-omic profiling of protein markers and transcriptomes, we found that the localization of macrophages further dictates functionally distinct but reversible phenotypes, regardless of their ontogeny. Whereas immune-stimulatory macrophages (CD11C+CD86+) populated the tumor epithelial nests, the stroma-associated macrophages (SAMs) were proliferative, immunosuppressive (Sca-1+CD206+PD-L1+), resistant to CSF-1R depletion, and associated with worse patient outcomes. Notably, following cessation of CSF-1R depletion, macrophages rebounded primarily to the SAM phenotype, which was associated with accelerated growth of mammary tumors. Our work reveals the spatial determinants of macrophage heterogeneity in breast cancer and highlights the disruption of macrophage self-renewal as a potential new therapeutic strategy.\nMethylation of histone H3 lysine 36 is a barrier for therapeutic interventions of head and neck squamous cell carcinoma.\nAuthors: Caeiro, L.; Nakata, Y.; Borges, R. L.; Garcia-Martinez, L.; Banuelos, C.; Stransky, S.; Chan, H. L.; Brabson, J.; Dominguez, D.; Zhang, Y.; Lewis, P. W.; Benitah, S. A.; Cimmino, L.; Bilbao, D.; Sidoli, S.; Verdun, R.; Morey, L.\nScore: 13.3, Published: 2023-11-06 DOI: 10.1101/2023.11.06.565847\nApproximately 20% of head and neck squamous cell carcinomas (HNSCC) exhibit reduced methylation on lysine 36 of histone H3 (H3K36me) due to mutations in histone methylase NSD1 or a lysine-to-methionine mutation in histone H3 (H3K36M). Whether such alterations of H3K36me can be exploited for therapeutic interventions is still unknown. Here, we show that HNSCC models expressing H3K36M can be divided into two groups: those that display aberrant accumulation of H3K27me3 and those that maintain steady levels of H3K27me3. The first group shows decreased proliferation, genome instability, and increased sensitivity to genotoxic agents, such as PARP1/2 inhibitors. In contrast, the H3K36M HNSCC models with steady H3K27me3 levels do not exhibit these characteristics unless H3K27me3 levels are elevated, either by DNA hypomethylating agents or by inhibiting the H3K27me3 demethylases KDM6A/B. Mechanistically, we found that H3K36M reduces H3K36me by directly impeding the activities of the histone methyltransferase NSD3 and the histone demethylase LSD2. Notably, we found that aberrant H3K27me3 levels induced by H3K36M expression is not a bona fide epigenetic mark in HNSCC since it requires continuous expression of H3K36M to be inherited. Moreover, increased sensitivity of H3K36M HNSCC models to PARP1/2 inhibitors solely depends on the increased H3K27me3 levels. Indeed, aberrantly high H3K27me3 levels decrease BRCA1 and FANCD2-dependent DNA repair, resulting in higher sensitivity to DNA breaks and replication stress. Finally, in support of our in vitro findings, a PARP1/2 inhibitor alone reduce tumor burden in a H3K36M HNSCC xenograft model with elevated H3K27me3, whereas in a H3K36M HNSCC xenograft model with consistent H3K27me3 levels, a combination of PARP1/2 inhibitors and agents that upregulate H3K27me3 proves to be successful. In conclusion, our findings underscore a delicate balance between H3K36 and H3K27 methylation, essential for maintaining genome stability. This equilibrium presents promising therapeutic opportunities for patients with H3K36me-deficient tumors.\nDissection and reconstruction of the colorectal cancer tumor microenvironment\nAuthors: Broguiere, N.; Lorenzo-Martin, L. F.; Ragusa, S.; Rivest, F.; Brandenberg, N.; Hohnel, S.; Dutta, D.; Gueye, M.; Alpern, D.; Deplancke, B.; Kandalaft, L.; Coukos, G.; Homicsko, K.; Lutolf, M. P.\nScore: 10.5, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565169\nPatient-derived organoids (PDOs) are the reference in vitro human disease models. However, the utility of colorectal cancer (CRC) PDOs is hindered by the lack of a tumor microenvironment (TME). To address this limitation, we built a living biobank of CRC PDOs with autologous stromal and immune TME. We characterized the original tumors and traditional monocultures using single-cell RNA-seq (scRNA-seq) and whole exome sequencing (WES) to obtain insights into cell type selection and phenotypic drift in culture. Subsequently, we developed culture conditions supporting all cell types to recapitulate the CRC-TME around PDOs. From the transcriptomes of \u003e180k cells obtained from 260 such co-cultures, we illuminated the mutual influence of cells within CRC tumors. Based on original tumor data, atlases of predicted interactions and transcriptional networks elucidated why monocultures were altered and suggested that TME reconstruction more accurately reflected original tumor behavior. We found that inflammatory signals were absent in vitro and recovered upon co-culture with tumor-infiltrating lymphocytes (TILs). We also functionally confirmed that stromal, not cancer cells, mediated immune evasion. Additionally, stroma induced an invasive phenotype in cancer cells. From this deep dive into CRC-TME interactions, we built the human CRC-TME atlas (https://crc-tme.com/), an online portal for interactive exploration of gene expression data, prediction of cell-cell interactions at the pathway and receptor/ligand levels, transcriptional networks, and more. We anticipate PDO cultures with reconstructed TMEs will be valuable for discovery efforts, preclinical studies, and personalized medicine, with the atlas as a framework and inspiration for future CRC-TME studies.\nA single-cell multi-omic and spatial atlas of nodal B-cell lymphomas reveals B-cell maturation drives intratumor heterogeneity\nAuthors: Fitzgerald, D.; Roider, T.; Baertsch, M. A.; Kibler, A.; Horlova, A.; Chung, E. K.; Voehringer, H.; Mathioudaki, A.; Budeus, B.; Passerini, V.; Knoll, M.; Mammen, J.; Li, L.; Caille, L.; Czernilofsky, F.; Bruch, P. M.; Liebers, N.; Meyer-Bender, M.; Weigert, O.; Zaugg, J. B.; Nolan, G. P.; Seifert, M.; Huber, W.; Dietrich, S.\nScore: 10.1, Published: 2023-11-07 DOI: 10.1101/2023.11.06.565756\nIntratumor heterogeneity is intrinsic to cancer pathogenesis and evolution, although little is known about how it relates to the differentiation trajectories of the tumors cell-of-origin. Nodal B-cell non-Hodgkin lymphomas are a diverse set of cancers thought to originate from distinct stages of B-cell maturation. Through a single-cell multi-omic and spatial atlas of diffuse large B-cell, mantle cell, follicular, and marginal zone lymphomas along with reactive lymph nodes (n=51), we found multiple B-cell maturation states coexist within the same tumors. Intratumor maturation states emerged from the same cell-of-origin, revealing that maturation remains plastic in malignancy. The maturation state composition varied across entities and tumors, which included mixtures of cell-of-origin subtypes. Intratumor maturation states inhabited unique spatial niches, which typically retained their maturation-associated cellular interactions and regulatory networks. Intratumor maturation states showed distinct expression patterns of genetic variants, suggesting that maturation and genetic aberrations are intertwined. Our findings put forward a transformative model for cancer pathogenesis, where differentiation continues in malignancy and is central to tumor heterogeneity and evolution.\nProteomics of Patient-derived Breast Tumours Identifies a Pro-migratory Osteomodulin-Cyclin Dependent Kinase 1 Axis which Drives Bone Metastasis\nAuthors: Parsons, J.; Harrison, H.; Kedward, T.; Fullwood, P.; Cabral, S.; Spence, K.; Barden, D.; Haworth, J.; Watson, J.; Tsafou, K.; Behan, C.; Ali, N.; Smith, M.; O'Brien, C.; Clarke, R.; Francavilla, C.\nScore: 8.5, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565489\nBreast cancer remains a leading cause of mortality, predominantly due to the development of metastases to vital organs. At present, predictive biomarkers of organ specific metastasis and therapies targeted to the metastatic niche are limited. Here, to identify the molecular determinants of breast cancer metastasis we analysed patient-derived breast tumours by combining quantitative proteomics, bioinformatics, and functional assays in vitro and in vivo. We identified elevated levels of the protein Osteomodulin (OMD) associated with breast cancer bone metastases in patient-derived samples. OMD overexpression in the breast cancer MDA-MB-231 cell model significantly increases cell migration in vitro and promotes the formation of bone metastases in vivo. Phosphoproteomics analysis of MDA-MB-231 cells expressing OMD identifies active Cyclin-dependent kinase 1 (CDK1) downstream of OMD. The importance of the OMD-CDK1 axis was validated using two independent phosphoproteomics datasets analysing patient-derived breast cancer samples. We also show that the OMD-CDK1 axis drives cell migration and cell viability in vitro and the formation of bone metastases in vivo. Finally, CDK1 inhibition reduces in vitro cell viability of an independent cohort of metastatic patient samples showing high CDK1 activity. Therefore, the OMD-CDK1 axis will determine which breast cancer patients develop bone metastases and is a therapeutic target to treat or prevent breast cancer bone metastases.\nMolecular subtypes of high grade serous ovarian cancer across racial groups and gene expression platforms\nAuthors: Davidson, N. R.; Barnard, M. E.; Hippen, A. A.; Campbell, A.; Johnson, C.; Way, G. P.; Dalley, B. K.; Berchuck, A.; Salas, L. A.; Peres, L. C.; Marks, J. R.; Schildkraut, J. M.; Greene, C. S.; Doherty, J. A.\nScore: 7.7, Published: 2023-11-04 DOI: 10.1101/2023.11.01.565179\nIntroductionBlack individuals with ovarian cancer experience poorer survival compared to non-Hispanic White individuals. High-grade serous carcinoma (HGSC) is the most common ovarian cancer histotype, with gene expression subtypes that are associated with differential survival. We characterized HGSC gene expression in Black individuals and considered whether gene expression differences by race may contribute to disparities in survival. MethodsWe performed gene expression analyses using RNA-Seq data from Black and White individuals with HGSC, as well as array-based genotyping data from four existing studies of HGSC. Our main analysis assigned subtypes by identifying dataset-specific clusters using K-means clustering for K=2-4. The cluster- and dataset-specific gene expression patterns were summarized by moderated t-scores that differentiate an individual cluster from all other clusters within each dataset. We compared the calculated gene expression patterns for each cluster across datasets by calculating the Pearson correlation between the two summarized vectors of moderated t-scores. Following K=4 subtype assignment and mapping to The Cancer Genome Atlas (TCGA)-derived HGSC subtypes, we used multivariable-adjusted Cox proportional hazards models to estimate subtype-specific survival separately for each dataset. ResultsWhen we compared HGSC gene expression between RNA-Seq data and array-based gene expression data, cluster-specific gene expression was similar across data types. Comparing the Black study population to White and Japanese study populations, the immunoreactive subtype was more common (39% versus 23%-28%) and the differentiated subtype less common (7% versus 22%-31%). Patterns of subtype-specific survival were similar between the Black and White populations with RNAseq data; when compared to mesenchymal HGSC, the risk of death was similar for proliferative and differentiated cases and lower, but not statistically significantly lower, for immunoreactive cases (HR=0.79 [0.55, 1.13] in the Black population and HR=0.86 [0.62, 1.19] in the White population). Patterns of subtype-specific survival were also similar in a Japanese population and in two of three predominantly White populations with array-based gene expression data. ConclusionsA single pipeline can be used to assign gene expression subtypes to HGSC with array-based or RNA-Seq gene expression data. We observed that while the prevalence of the immunoreactive subtype was higher, and the prevalence of the differentiated subtype was lower among Black individuals compared to White individuals, subtype-specific survival was similar.\nAtaxia-telangiectasia mutated (Atm) disruption sensitizes spatially-directed H3.3K27M/TP53 diffuse midline gliomas to radiation therapy\nAuthors: Mangoli, A.; Wu, S.; Liu, H. Q.; Aksu, M.; Jain, V. J.; Foreman, B. E.; Regal, J. A.; Weidenhammer, L. B.; Stewart, C. E.; Guerra Garcia, M. E.; Hocke, E.; Abramson, K.; Williams, N. T.; Luo, L.; Deland, K.; Attardi, L.; Abe, K.; Hashizume, R.; Ashley, D. M.; Becher, O. J.; Kirsch, D. G.; Gregory, S. G.; Reitman, Z. J.\nScore: 291.4, Published: 2023-10-20 DOI: 10.1101/2023.10.18.562892\nDiffuse midline gliomas (DMGs) are lethal brain tumors characterized by p53-inactivating mutations and oncohistone H3.3K27M mutations that rewire the cellular response to genotoxic stress, which presents therapeutic opportunities. We used RCAS/tv-a retroviruses and Cre recombinase to inactivate p53 and induce K27M in the native H3f3a allele in a lineage- and spatially-directed manner, yielding primary mouse DMGs. Genetic or pharmacologic disruption of the DNA damage response kinase Ataxia-telangiectasia mutated (ATM) enhanced the efficacy of focal brain irradiation, extending mouse survival. This finding suggests that targeting ATM will enhance the efficacy of radiation therapy for p53-mutant DMG but not p53-wildtype DMG. We used spatial in situ transcriptomics and an allelic series of primary murine DMG models with different p53 mutations to identify transactivation-independent p53 activity as a key mediator of such radiosensitivity. These studies deeply profile a genetically faithful and versatile model of a lethal brain tumor to identify resistance mechanisms for a therapeutic strategy currently in clinical trials.\nATF6 activation alters colonic lipid metabolism causing tumor-associated microbial adaptation\nAuthors: Coleman, O. I.; Sorbie, A.; Bierwirth, S.; Koevilein, J.; von Stern, M.; Koehler, N.; Wirbel, J.; Schmidt, C.; Kacprowski, T.; Dunkel, A.; Pauling, J. K.; Plagge, J.; Miedel-Cuadra, D.; Wagner, S.; Peng, T.; Metzler, T.; Schafmayer, C.; Hinz, S.; Roeder, C.; Roecken, C.; Stecher, B.; Rosenstiel, P.; Steiger, K.; Jesinghaus, M.; Liebisch, G.; Ecker, J.; Zeller, G.; Janssen, K.-P.; Haller, D.\nScore: 6.4, Published: 2023-11-05 DOI: 10.1101/2023.11.03.565267\nThe induction of endoplasmic reticulum unfolded protein responses (UPRER) contributes to cancer development and progression. We recently linked microbiota-related triggers to the tumor-promoting role of signal transducer activating transcription factor 6 (ATF6) in the colon. Here we substantiate the clinical relevance of ATF6 and related bacterial genera in colorectal cancer patient cohorts. Spatial and longitudinal bacterial profiling in ATF6 transgenic mice (nATF6IEC) identified tumor-initiating and tumor-progressing shifts in the mucosa-associated microbiota. Transcriptional analysis in intestinal epithelial cells (IEC) of germ-free and specific pathogen-free nATF6IEC mice defined bacteria-specific changes in cellular metabolism enriched for fatty acid biosynthesis. Untargeted metabolomics, isotope-labeling in intestinal organoids and FASN inhibition confirmed ATF6-mediated involvement of long-chain fatty acids in tumorigenesis. Multi-omics data integration identified a bacteria-lipid network characterized by fatty acid efflux, catabolism and detoxification. We postulate chronic ATF6 signaling to drive a clinically relevant pathologic response altering lipid metabolism to select for a tumor-promoting microbiota. Graphical Abstract O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=198 SRC=\"FIGDIR/small/565267v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (58K): org.highwire.dtl.DTLVardef@1e9334borg.highwire.dtl.DTLVardef@126bc60org.highwire.dtl.DTLVardef@196d46org.highwire.dtl.DTLVardef@15518f9_HPS_FORMAT_FIGEXP M_FIG C_FIG Chronic ATF6 signaling in the colonic epithelium alters lipid metabolism to select a tumor-promoting microbiota O_LInATF6 expression is induced in intestinal epithelial cells in transgenic mice (nATF6IEC) C_LIO_LIBiallelic nATF6IEC germ-free mice (grey circle) and floxed or monoallelic specific pathogen-free mice (black circle) remain in a state of homeostasis (yellow circle), while biallelic specific pathogen-free mice (black circle) develop spontaneous colon tumorigenesis (red circle) C_LIO_LIMechanistically, biallelic nATF6IEC mice in the presence of an intestinal microbiota alter colonic lipid metabolism, including the upregulation of Fasn C_LIO_LIThe microbial lipid-response to the altered lipid milieu results in dysbiosis and the subsequent formation of colon adenomas C_LIO_LIInhibition of FASN in biallelic nATF6IEC mice prevents colon tumor formation (yellow arrow) C_LI Created with BioRender.com nATF6: activated activating transcription factor 6; LCFA: long-chain fatty acids; SAFA: saturated fatty acids; Fasn: fatty acid synthase; C75 i.p.: intraperitoneal injection of the Fasn inhibitor C75.\n2D and 3D multiplexed subcellular profiling of nuclear instability in human cancer\nAuthors: Coy, S.; Cheng, B.; Lee, J. S.; Rashid, R.; Browning, L.; Xu, Y.; Chakrabarty, S. S.; Yapp, C.; Chan, S.; Tefft, J. B.; Spektor, A.; Ligon, K. L.; Baker, G. J.; Pellman, D.; Sorger, P. K.; Santagata, S.\nScore: 6.3, Published: 2023-11-11 DOI: 10.1101/2023.11.07.566063\nNuclear atypia, including altered nuclear size, contour, and chromatin organization, is ubiquitous in cancer cells. Atypical primary nuclei and micronuclei can rupture during interphase; however, the frequency, causes, and consequences of nuclear rupture are unknown in most cancers. We demonstrate that nuclear envelope rupture is surprisingly common in many human cancers, particularly glioblastoma. Using highly-multiplexed 2D and super-resolution 3D-imaging of glioblastoma tissues and patient-derived xenografts and cells, we link primary nuclear rupture with reduced lamin A/C and micronuclear rupture with reduced lamin B1. Moreover, ruptured glioblastoma cells activate cGAS-STING-signaling involved in innate immunity. We observe that local patterning of cell states influences tumor spatial organization and is linked to both lamin expression and rupture frequency, with neural-progenitor-cell-like states exhibiting the lowest lamin A/C levels and greatest susceptibility to primary nuclear rupture. Our study reveals that nuclear instability is a core feature of cancer, and links nuclear integrity, cell state, and immune signaling.\nComparison of Tug-of-War Models Assuming Moran versus Branching Process Population Dynamics\nAuthors: Dinh, K. N.; Kurpas, M. K.; Kimmel, M.\nScore: 8.9, Published: 2023-10-23 DOI: 10.1101/2023.10.20.563302\nMutations arising during cancer evolution are typically categorized as either drivers or passengers, depending on whether they increase the cell fitness. Recently, McFarland et al. introduced the tug-of-war model for the joint effect of rare advantageous drivers and frequent but deleterious passengers. We examine this model under two common but distinct frameworks, the Moran model and the branching process. We show that frequently used statistics are similar between a version of the Moran model and the branching process conditioned on the final cell count, under different selection scenarios. We infer the selection coefficients for three breast cancer samples, resulting in good fits of the shape of their Site Frequency Spectra. All fitted values for the selective disadvantage of passenger mutations are nonzero, supporting the view that they exert deleterious selection during tumorigenesis that driver mutations must compensate.\n",
  "wordCount" : "2914",
  "inLanguage": "en",
  "datePublished": "2023-11-12T10:37:43Z",
  "dateModified": "2023-11-12T10:37:43Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on November 12, 2023&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.24.563749">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.24.563749" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.24.563749">
        <p class="paperTitle">Breast Cancer Macrophage Heterogeneity and Self-renewal are Determined by Spatial Localization</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.24.563749" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.24.563749" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ben-Chetrit, N.; Niu, X.; Sotelo, J.; Swett, A. D.; Rajasekhar, V. K.; Jiao, M. S.; Stewart, C. M.; Bhardwaj, P.; Kottapalli, S.; Ganesan, S.; Loyher, P.-L.; Potenski, C.; Hannuna, A.; Brown, K. A.; Iyengar, N. M.; Giri, D. D.; Lowe, S. W.; Healey, J. H.; Geissmann, F.; Sagi, I.; Joyce, J. A.; Landau, D. A.</p>
        <p class="info">Score: 31.5, Published: 2023-10-27 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.24.563749' target='https://doi.org/10.1101/2023.10.24.563749'> 10.1101/2023.10.24.563749</a></p>
        <p class="abstract">Tumor-infiltrating macrophages support critical steps in tumor progression, and their accumulation in the tumor microenvironment (TME) is associated with adverse outcomes and therapeutic resistance across human cancers. In the TME, macrophages adopt diverse phenotypic alterations, giving rise to heterogeneous immune activation states and induction of cell cycle. While the transcriptional profiles of these activation states are well-annotated across human cancers, the underlying signals that regulate macrophage heterogeneity and accumulation remain incompletely understood. Here, we leveraged a novel ex vivo organotypic TME (oTME) model of breast cancer, in vivo murine models, and human samples to map the determinants of functional heterogeneity of TME macrophages. We identified a subset of F4/80highSca-1&#43; self-renewing macrophages maintained by type-I interferon (IFN) signaling and requiring physical contact with cancer-associated fibroblasts. We discovered that the contact-dependent self-renewal of TME macrophages is mediated via Notch4, and its inhibition abrogated tumor growth of breast and ovarian carcinomas in vivo, as well as lung dissemination in a PDX model of triple-negative breast cancer (TNBC). Through spatial multi-omic profiling of protein markers and transcriptomes, we found that the localization of macrophages further dictates functionally distinct but reversible phenotypes, regardless of their ontogeny. Whereas immune-stimulatory macrophages (CD11C&#43;CD86&#43;) populated the tumor epithelial nests, the stroma-associated macrophages (SAMs) were proliferative, immunosuppressive (Sca-1&#43;CD206&#43;PD-L1&#43;), resistant to CSF-1R depletion, and associated with worse patient outcomes. Notably, following cessation of CSF-1R depletion, macrophages rebounded primarily to the SAM phenotype, which was associated with accelerated growth of mammary tumors. Our work reveals the spatial determinants of macrophage heterogeneity in breast cancer and highlights the disruption of macrophage self-renewal as a potential new therapeutic strategy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565847">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565847" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565847">
        <p class="paperTitle">Methylation of histone H3 lysine 36 is a barrier for therapeutic interventions of head and neck squamous cell carcinoma.</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565847" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565847" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Caeiro, L.; Nakata, Y.; Borges, R. L.; Garcia-Martinez, L.; Banuelos, C.; Stransky, S.; Chan, H. L.; Brabson, J.; Dominguez, D.; Zhang, Y.; Lewis, P. W.; Benitah, S. A.; Cimmino, L.; Bilbao, D.; Sidoli, S.; Verdun, R.; Morey, L.</p>
        <p class="info">Score: 13.3, Published: 2023-11-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565847' target='https://doi.org/10.1101/2023.11.06.565847'> 10.1101/2023.11.06.565847</a></p>
        <p class="abstract">Approximately 20% of head and neck squamous cell carcinomas (HNSCC) exhibit reduced methylation on lysine 36 of histone H3 (H3K36me) due to mutations in histone methylase NSD1 or a lysine-to-methionine mutation in histone H3 (H3K36M). Whether such alterations of H3K36me can be exploited for therapeutic interventions is still unknown. Here, we show that HNSCC models expressing H3K36M can be divided into two groups: those that display aberrant accumulation of H3K27me3 and those that maintain steady levels of H3K27me3. The first group shows decreased proliferation, genome instability, and increased sensitivity to genotoxic agents, such as PARP1/2 inhibitors. In contrast, the H3K36M HNSCC models with steady H3K27me3 levels do not exhibit these characteristics unless H3K27me3 levels are elevated, either by DNA hypomethylating agents or by inhibiting the H3K27me3 demethylases KDM6A/B. Mechanistically, we found that H3K36M reduces H3K36me by directly impeding the activities of the histone methyltransferase NSD3 and the histone demethylase LSD2. Notably, we found that aberrant H3K27me3 levels induced by H3K36M expression is not a bona fide epigenetic mark in HNSCC since it requires continuous expression of H3K36M to be inherited. Moreover, increased sensitivity of H3K36M HNSCC models to PARP1/2 inhibitors solely depends on the increased H3K27me3 levels. Indeed, aberrantly high H3K27me3 levels decrease BRCA1 and FANCD2-dependent DNA repair, resulting in higher sensitivity to DNA breaks and replication stress. Finally, in support of our in vitro findings, a PARP1/2 inhibitor alone reduce tumor burden in a H3K36M HNSCC xenograft model with elevated H3K27me3, whereas in a H3K36M HNSCC xenograft model with consistent H3K27me3 levels, a combination of PARP1/2 inhibitors and agents that upregulate H3K27me3 proves to be successful. In conclusion, our findings underscore a delicate balance between H3K36 and H3K27 methylation, essential for maintaining genome stability. This equilibrium presents promising therapeutic opportunities for patients with H3K36me-deficient tumors.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565169">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565169" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565169">
        <p class="paperTitle">Dissection and reconstruction of the colorectal cancer tumor microenvironment</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565169" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565169" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Broguiere, N.; Lorenzo-Martin, L. F.; Ragusa, S.; Rivest, F.; Brandenberg, N.; Hohnel, S.; Dutta, D.; Gueye, M.; Alpern, D.; Deplancke, B.; Kandalaft, L.; Coukos, G.; Homicsko, K.; Lutolf, M. P.</p>
        <p class="info">Score: 10.5, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565169' target='https://doi.org/10.1101/2023.11.01.565169'> 10.1101/2023.11.01.565169</a></p>
        <p class="abstract">Patient-derived organoids (PDOs) are the reference in vitro human disease models. However, the utility of colorectal cancer (CRC) PDOs is hindered by the lack of a tumor microenvironment (TME). To address this limitation, we built a living biobank of CRC PDOs with autologous stromal and immune TME. We characterized the original tumors and traditional monocultures using single-cell RNA-seq (scRNA-seq) and whole exome sequencing (WES) to obtain insights into cell type selection and phenotypic drift in culture. Subsequently, we developed culture conditions supporting all cell types to recapitulate the CRC-TME around PDOs. From the transcriptomes of &gt;180k cells obtained from 260 such co-cultures, we illuminated the mutual influence of cells within CRC tumors. Based on original tumor data, atlases of predicted interactions and transcriptional networks elucidated why monocultures were altered and suggested that TME reconstruction more accurately reflected original tumor behavior. We found that inflammatory signals were absent in vitro and recovered upon co-culture with tumor-infiltrating lymphocytes (TILs). We also functionally confirmed that stromal, not cancer cells, mediated immune evasion. Additionally, stroma induced an invasive phenotype in cancer cells. From this deep dive into CRC-TME interactions, we built the human CRC-TME atlas (https://crc-tme.com/), an online portal for interactive exploration of gene expression data, prediction of cell-cell interactions at the pathway and receptor/ligand levels, transcriptional networks, and more. We anticipate PDO cultures with reconstructed TMEs will be valuable for discovery efforts, preclinical studies, and personalized medicine, with the atlas as a framework and inspiration for future CRC-TME studies.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.06.565756">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.06.565756" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.06.565756">
        <p class="paperTitle">A single-cell multi-omic and spatial atlas of nodal B-cell lymphomas reveals B-cell maturation drives intratumor heterogeneity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.06.565756" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.06.565756" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Fitzgerald, D.; Roider, T.; Baertsch, M. A.; Kibler, A.; Horlova, A.; Chung, E. K.; Voehringer, H.; Mathioudaki, A.; Budeus, B.; Passerini, V.; Knoll, M.; Mammen, J.; Li, L.; Caille, L.; Czernilofsky, F.; Bruch, P. M.; Liebers, N.; Meyer-Bender, M.; Weigert, O.; Zaugg, J. B.; Nolan, G. P.; Seifert, M.; Huber, W.; Dietrich, S.</p>
        <p class="info">Score: 10.1, Published: 2023-11-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.06.565756' target='https://doi.org/10.1101/2023.11.06.565756'> 10.1101/2023.11.06.565756</a></p>
        <p class="abstract">Intratumor heterogeneity is intrinsic to cancer pathogenesis and evolution, although little is known about how it relates to the differentiation trajectories of the tumors cell-of-origin. Nodal B-cell non-Hodgkin lymphomas are a diverse set of cancers thought to originate from distinct stages of B-cell maturation. Through a single-cell multi-omic and spatial atlas of diffuse large B-cell, mantle cell, follicular, and marginal zone lymphomas along with reactive lymph nodes (n=51), we found multiple B-cell maturation states coexist within the same tumors. Intratumor maturation states emerged from the same cell-of-origin, revealing that maturation remains plastic in malignancy. The maturation state composition varied across entities and tumors, which included mixtures of cell-of-origin subtypes. Intratumor maturation states inhabited unique spatial niches, which typically retained their maturation-associated cellular interactions and regulatory networks. Intratumor maturation states showed distinct expression patterns of genetic variants, suggesting that maturation and genetic aberrations are intertwined. Our findings put forward a transformative model for cancer pathogenesis, where differentiation continues in malignancy and is central to tumor heterogeneity and evolution.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565489">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565489" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565489">
        <p class="paperTitle">Proteomics of Patient-derived Breast Tumours Identifies a Pro-migratory Osteomodulin-Cyclin Dependent Kinase 1 Axis which Drives Bone Metastasis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565489" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565489" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Parsons, J.; Harrison, H.; Kedward, T.; Fullwood, P.; Cabral, S.; Spence, K.; Barden, D.; Haworth, J.; Watson, J.; Tsafou, K.; Behan, C.; Ali, N.; Smith, M.; O&#39;Brien, C.; Clarke, R.; Francavilla, C.</p>
        <p class="info">Score: 8.5, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565489' target='https://doi.org/10.1101/2023.11.03.565489'> 10.1101/2023.11.03.565489</a></p>
        <p class="abstract">Breast cancer remains a leading cause of mortality, predominantly due to the development of metastases to vital organs. At present, predictive biomarkers of organ specific metastasis and therapies targeted to the metastatic niche are limited. Here, to identify the molecular determinants of breast cancer metastasis we analysed patient-derived breast tumours by combining quantitative proteomics, bioinformatics, and functional assays in vitro and in vivo. We identified elevated levels of the protein Osteomodulin (OMD) associated with breast cancer bone metastases in patient-derived samples. OMD overexpression in the breast cancer MDA-MB-231 cell model significantly increases cell migration in vitro and promotes the formation of bone metastases in vivo. Phosphoproteomics analysis of MDA-MB-231 cells expressing OMD identifies active Cyclin-dependent kinase 1 (CDK1) downstream of OMD. The importance of the OMD-CDK1 axis was validated using two independent phosphoproteomics datasets analysing patient-derived breast cancer samples. We also show that the OMD-CDK1 axis drives cell migration and cell viability in vitro and the formation of bone metastases in vivo. Finally, CDK1 inhibition reduces in vitro cell viability of an independent cohort of metastatic patient samples showing high CDK1 activity. Therefore, the OMD-CDK1 axis will determine which breast cancer patients develop bone metastases and is a therapeutic target to treat or prevent breast cancer bone metastases.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.01.565179">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.01.565179" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.01.565179">
        <p class="paperTitle">Molecular subtypes of high grade serous ovarian cancer across racial groups and gene expression platforms</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.01.565179" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.01.565179" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Davidson, N. R.; Barnard, M. E.; Hippen, A. A.; Campbell, A.; Johnson, C.; Way, G. P.; Dalley, B. K.; Berchuck, A.; Salas, L. A.; Peres, L. C.; Marks, J. R.; Schildkraut, J. M.; Greene, C. S.; Doherty, J. A.</p>
        <p class="info">Score: 7.7, Published: 2023-11-04 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.01.565179' target='https://doi.org/10.1101/2023.11.01.565179'> 10.1101/2023.11.01.565179</a></p>
        <p class="abstract">IntroductionBlack individuals with ovarian cancer experience poorer survival compared to non-Hispanic White individuals. High-grade serous carcinoma (HGSC) is the most common ovarian cancer histotype, with gene expression subtypes that are associated with differential survival. We characterized HGSC gene expression in Black individuals and considered whether gene expression differences by race may contribute to disparities in survival.

MethodsWe performed gene expression analyses using RNA-Seq data from Black and White individuals with HGSC, as well as array-based genotyping data from four existing studies of HGSC. Our main analysis assigned subtypes by identifying dataset-specific clusters using K-means clustering for K=2-4. The cluster- and dataset-specific gene expression patterns were summarized by moderated t-scores that differentiate an individual cluster from all other clusters within each dataset. We compared the calculated gene expression patterns for each cluster across datasets by calculating the Pearson correlation between the two summarized vectors of moderated t-scores. Following K=4 subtype assignment and mapping to The Cancer Genome Atlas (TCGA)-derived HGSC subtypes, we used multivariable-adjusted Cox proportional hazards models to estimate subtype-specific survival separately for each dataset.

ResultsWhen we compared HGSC gene expression between RNA-Seq data and array-based gene expression data, cluster-specific gene expression was similar across data types. Comparing the Black study population to White and Japanese study populations, the immunoreactive subtype was more common (39% versus 23%-28%) and the differentiated subtype less common (7% versus 22%-31%). Patterns of subtype-specific survival were similar between the Black and White populations with RNAseq data; when compared to mesenchymal HGSC, the risk of death was similar for proliferative and differentiated cases and lower, but not statistically significantly lower, for immunoreactive cases (HR=0.79 [0.55, 1.13] in the Black population and HR=0.86 [0.62, 1.19] in the White population). Patterns of subtype-specific survival were also similar in a Japanese population and in two of three predominantly White populations with array-based gene expression data.

ConclusionsA single pipeline can be used to assign gene expression subtypes to HGSC with array-based or RNA-Seq gene expression data. We observed that while the prevalence of the immunoreactive subtype was higher, and the prevalence of the differentiated subtype was lower among Black individuals compared to White individuals, subtype-specific survival was similar.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.18.562892">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.18.562892" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.18.562892">
        <p class="paperTitle">Ataxia-telangiectasia mutated (Atm) disruption sensitizes spatially-directed H3.3K27M/TP53 diffuse midline gliomas to radiation therapy</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.18.562892" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.18.562892" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Mangoli, A.; Wu, S.; Liu, H. Q.; Aksu, M.; Jain, V. J.; Foreman, B. E.; Regal, J. A.; Weidenhammer, L. B.; Stewart, C. E.; Guerra Garcia, M. E.; Hocke, E.; Abramson, K.; Williams, N. T.; Luo, L.; Deland, K.; Attardi, L.; Abe, K.; Hashizume, R.; Ashley, D. M.; Becher, O. J.; Kirsch, D. G.; Gregory, S. G.; Reitman, Z. J.</p>
        <p class="info">Score: 291.4, Published: 2023-10-20 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.18.562892' target='https://doi.org/10.1101/2023.10.18.562892'> 10.1101/2023.10.18.562892</a></p>
        <p class="abstract">Diffuse midline gliomas (DMGs) are lethal brain tumors characterized by p53-inactivating mutations and oncohistone H3.3K27M mutations that rewire the cellular response to genotoxic stress, which presents therapeutic opportunities. We used RCAS/tv-a retroviruses and Cre recombinase to inactivate p53 and induce K27M in the native H3f3a allele in a lineage- and spatially-directed manner, yielding primary mouse DMGs. Genetic or pharmacologic disruption of the DNA damage response kinase Ataxia-telangiectasia mutated (ATM) enhanced the efficacy of focal brain irradiation, extending mouse survival. This finding suggests that targeting ATM will enhance the efficacy of radiation therapy for p53-mutant DMG but not p53-wildtype DMG. We used spatial in situ transcriptomics and an allelic series of primary murine DMG models with different p53 mutations to identify transactivation-independent p53 activity as a key mediator of such radiosensitivity. These studies deeply profile a genetically faithful and versatile model of a lethal brain tumor to identify resistance mechanisms for a therapeutic strategy currently in clinical trials.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.03.565267">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.03.565267" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.03.565267">
        <p class="paperTitle">ATF6 activation alters colonic lipid metabolism causing tumor-associated microbial adaptation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.03.565267" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.03.565267" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coleman, O. I.; Sorbie, A.; Bierwirth, S.; Koevilein, J.; von Stern, M.; Koehler, N.; Wirbel, J.; Schmidt, C.; Kacprowski, T.; Dunkel, A.; Pauling, J. K.; Plagge, J.; Miedel-Cuadra, D.; Wagner, S.; Peng, T.; Metzler, T.; Schafmayer, C.; Hinz, S.; Roeder, C.; Roecken, C.; Stecher, B.; Rosenstiel, P.; Steiger, K.; Jesinghaus, M.; Liebisch, G.; Ecker, J.; Zeller, G.; Janssen, K.-P.; Haller, D.</p>
        <p class="info">Score: 6.4, Published: 2023-11-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.03.565267' target='https://doi.org/10.1101/2023.11.03.565267'> 10.1101/2023.11.03.565267</a></p>
        <p class="abstract">The induction of endoplasmic reticulum unfolded protein responses (UPRER) contributes to cancer development and progression. We recently linked microbiota-related triggers to the tumor-promoting role of signal transducer activating transcription factor 6 (ATF6) in the colon. Here we substantiate the clinical relevance of ATF6 and related bacterial genera in colorectal cancer patient cohorts. Spatial and longitudinal bacterial profiling in ATF6 transgenic mice (nATF6IEC) identified tumor-initiating and tumor-progressing shifts in the mucosa-associated microbiota. Transcriptional analysis in intestinal epithelial cells (IEC) of germ-free and specific pathogen-free nATF6IEC mice defined bacteria-specific changes in cellular metabolism enriched for fatty acid biosynthesis. Untargeted metabolomics, isotope-labeling in intestinal organoids and FASN inhibition confirmed ATF6-mediated involvement of long-chain fatty acids in tumorigenesis. Multi-omics data integration identified a bacteria-lipid network characterized by fatty acid efflux, catabolism and detoxification. We postulate chronic ATF6 signaling to drive a clinically relevant pathologic response altering lipid metabolism to select for a tumor-promoting microbiota.

Graphical Abstract

O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=198 SRC=&#34;FIGDIR/small/565267v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (58K):
org.highwire.dtl.DTLVardef@1e9334borg.highwire.dtl.DTLVardef@126bc60org.highwire.dtl.DTLVardef@196d46org.highwire.dtl.DTLVardef@15518f9_HPS_FORMAT_FIGEXP  M_FIG C_FIG Chronic ATF6 signaling in the colonic epithelium alters lipid metabolism to select a tumor-promoting microbiota O_LInATF6 expression is induced in intestinal epithelial cells in transgenic mice (nATF6IEC)
C_LIO_LIBiallelic nATF6IEC germ-free mice (grey circle) and floxed or monoallelic specific pathogen-free mice (black circle) remain in a state of homeostasis (yellow circle), while biallelic specific pathogen-free mice (black circle) develop spontaneous colon tumorigenesis (red circle)
C_LIO_LIMechanistically, biallelic nATF6IEC mice in the presence of an intestinal microbiota alter colonic lipid metabolism, including the upregulation of Fasn
C_LIO_LIThe microbial lipid-response to the altered lipid milieu results in dysbiosis and the subsequent formation of colon adenomas
C_LIO_LIInhibition of FASN in biallelic nATF6IEC mice prevents colon tumor formation (yellow arrow)
C_LI

Created with BioRender.com

nATF6: activated activating transcription factor 6; LCFA: long-chain fatty acids; SAFA: saturated fatty acids; Fasn: fatty acid synthase; C75 i.p.: intraperitoneal injection of the Fasn inhibitor C75.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.11.07.566063">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.11.07.566063" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.11.07.566063">
        <p class="paperTitle">2D and 3D multiplexed subcellular profiling of nuclear instability in human cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.11.07.566063" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.11.07.566063" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Coy, S.; Cheng, B.; Lee, J. S.; Rashid, R.; Browning, L.; Xu, Y.; Chakrabarty, S. S.; Yapp, C.; Chan, S.; Tefft, J. B.; Spektor, A.; Ligon, K. L.; Baker, G. J.; Pellman, D.; Sorger, P. K.; Santagata, S.</p>
        <p class="info">Score: 6.3, Published: 2023-11-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.11.07.566063' target='https://doi.org/10.1101/2023.11.07.566063'> 10.1101/2023.11.07.566063</a></p>
        <p class="abstract">Nuclear atypia, including altered nuclear size, contour, and chromatin organization, is ubiquitous in cancer cells. Atypical primary nuclei and micronuclei can rupture during interphase; however, the frequency, causes, and consequences of nuclear rupture are unknown in most cancers. We demonstrate that nuclear envelope rupture is surprisingly common in many human cancers, particularly glioblastoma. Using highly-multiplexed 2D and super-resolution 3D-imaging of glioblastoma tissues and patient-derived xenografts and cells, we link primary nuclear rupture with reduced lamin A/C and micronuclear rupture with reduced lamin B1. Moreover, ruptured glioblastoma cells activate cGAS-STING-signaling involved in innate immunity. We observe that local patterning of cell states influences tumor spatial organization and is linked to both lamin expression and rupture frequency, with neural-progenitor-cell-like states exhibiting the lowest lamin A/C levels and greatest susceptibility to primary nuclear rupture. Our study reveals that nuclear instability is a core feature of cancer, and links nuclear integrity, cell state, and immune signaling.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.20.563302">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.20.563302" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.20.563302">
        <p class="paperTitle">Comparison of Tug-of-War Models Assuming Moran versus Branching Process Population Dynamics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.20.563302" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.20.563302" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Dinh, K. N.; Kurpas, M. K.; Kimmel, M.</p>
        <p class="info">Score: 8.9, Published: 2023-10-23 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.20.563302' target='https://doi.org/10.1101/2023.10.20.563302'> 10.1101/2023.10.20.563302</a></p>
        <p class="abstract">Mutations arising during cancer evolution are typically categorized as either  drivers or  passengers, depending on whether they increase the cell fitness. Recently, McFarland et al. introduced the tug-of-war model for the joint effect of rare advantageous drivers and frequent but deleterious passengers. We examine this model under two common but distinct frameworks, the Moran model and the branching process. We show that frequently used statistics are similar between a version of the Moran model and the branching process conditioned on the final cell count, under different selection scenarios. We infer the selection coefficients for three breast cancer samples, resulting in good fits of the shape of their Site Frequency Spectra. All fitted values for the selective disadvantage of passenger mutations are nonzero, supporting the view that they exert deleterious selection during tumorigenesis that driver mutations must compensate.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
