<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Mechanical confinement governs phenotypic plasticity in melanoma
Authors: Hunter, M. V.; Montal, E.; Ma, Y.; Moncada, R.; Yanai, I.; Koche, R. P.; White, R. M.
Score: 31.9, Published: 2024-02-02 DOI: 10.1101/2024.01.30.577120
Phenotype switching is a form of cellular plasticity in which cancer cells reversibly move between two opposite extremes - proliferative versus invasive states. While it has long been hypothesised that such switching is triggered by external cues, the identity of these cues has remained elusive.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Mechanical confinement governs phenotypic plasticity in melanoma
Authors: Hunter, M. V.; Montal, E.; Ma, Y.; Moncada, R.; Yanai, I.; Koche, R. P.; White, R. M.
Score: 31.9, Published: 2024-02-02 DOI: 10.1101/2024.01.30.577120
Phenotype switching is a form of cellular plasticity in which cancer cells reversibly move between two opposite extremes - proliferative versus invasive states. While it has long been hypothesised that such switching is triggered by external cues, the identity of these cues has remained elusive." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2024-02-11T10:38:55+00:00" />
<meta property="article:modified_time" content="2024-02-11T10:38:55+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Mechanical confinement governs phenotypic plasticity in melanoma
Authors: Hunter, M. V.; Montal, E.; Ma, Y.; Moncada, R.; Yanai, I.; Koche, R. P.; White, R. M.
Score: 31.9, Published: 2024-02-02 DOI: 10.1101/2024.01.30.577120
Phenotype switching is a form of cellular plasticity in which cancer cells reversibly move between two opposite extremes - proliferative versus invasive states. While it has long been hypothesised that such switching is triggered by external cues, the identity of these cues has remained elusive."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Mechanical confinement governs phenotypic plasticity in melanoma\nAuthors: Hunter, M. V.; Montal, E.; Ma, Y.; Moncada, R.; Yanai, I.; Koche, R. P.; White, R. M.\nScore: 31.9, Published: 2024-02-02 DOI: 10.1101/2024.01.30.577120\nPhenotype switching is a form of cellular plasticity in which cancer cells reversibly move between two opposite extremes - proliferative versus invasive states. While it has long been hypothesised that such switching is triggered by external cues, the identity of these cues has remained elusive.",
  "keywords": [
    
  ],
  "articleBody": " Mechanical confinement governs phenotypic plasticity in melanoma\nAuthors: Hunter, M. V.; Montal, E.; Ma, Y.; Moncada, R.; Yanai, I.; Koche, R. P.; White, R. M.\nScore: 31.9, Published: 2024-02-02 DOI: 10.1101/2024.01.30.577120\nPhenotype switching is a form of cellular plasticity in which cancer cells reversibly move between two opposite extremes - proliferative versus invasive states. While it has long been hypothesised that such switching is triggered by external cues, the identity of these cues has remained elusive. Here, we demonstrate that mechanical confinement mediates phenotype switching through chromatin remodelling. Using a zebrafish model of melanoma coupled with human samples, we profiled tumor cells at the interface between the tumor and surrounding microenvironment. Morphological analysis of these rare cells showed flattened, elliptical nuclei suggestive of mechanical confinement by adjacent tissue. Spatial and single-cell transcriptomics demonstrated that the interface cells adopted a gene program of neuronal invasion, including acquisition of an acetylated tubulin cage that protects the nucleus during migration. We identified the DNA-bending protein HMGB2 as a confinement-induced mediator of the neuronal state. HMGB2 is upregulated in confined cells, and quantitative modelling revealed that confinement prolongs contact time between HMGB2 and chromatin, leading to changes in chromatin configuration that favor the neuronal phenotype. Genetic disruption of HMGB2 showed that it regulates the trade-off between proliferative and invasive states, in which confined HMGB2high tumor cells are less proliferative but more drug resistant. Our results implicate the mechanical microenvironment as a mechanism driving phenotype switching in melanoma.\nαKG-mediated carnitine synthesis promotes homologous recombination via histone acetylation\nAuthors: Uboveja, A.; Huang, Z.; Buj, R.; Amalric, A.; Wang, H.; Tangudu, N. K.; Cole, A. R.; Megill, E.; Kantner, D.; Chatoff, A.; Ahmad, H.; Marcinkiewicz, M. M.; Disharoon, J. A.; Graff, S.; Dahl, E. S.; Hempel, N.; Stallaert, W.; Sidoli, S.; Bitler, B. G.; Long, D. T.; Snyder, N. W.; Aird, K. M.\nScore: 19.5, Published: 2024-02-07 DOI: 10.1101/2024.02.06.578742\nHomologous recombination (HR) deficiency enhances sensitivity to DNA damaging agents commonly used to treat cancer. In HR-proficient cancers, metabolic mechanisms driving response or resistance to DNA damaging agents remain unclear. Here we identified that depletion of alpha-ketoglutarate (KG) sensitizes HR-proficient cells to DNA damaging agents by metabolic regulation of histone acetylation. KG is required for the activity of KG-dependent dioxygenases (KGDDs), and prior work has shown that changes in KGDD affect demethylases. Using a targeted CRISPR knockout library consisting of 64 KGDDs, we discovered that Trimethyllysine Hydroxylase Epsilon (TMLHE), the first and rate-limiting enzyme in de novo carnitine synthesis, is necessary for proliferation of HR-proficient cells in the presence of DNA damaging agents. Unexpectedly, KG-mediated TMLHE-dependent carnitine synthesis was required for histone acetylation, while histone methylation was affected but dispensable. The increase in histone acetylation via KG-dependent carnitine synthesis promoted HR-mediated DNA repair through site- and substrate-specific histone acetylation. These data demonstrate for the first time that HR-proficiency is mediated through KG directly influencing histone acetylation via carnitine synthesis.\nBroad de-regulated U2AF1 splicing is prognostic and augments leukemic transformation via protein arginine methyltransferase activation\nAuthors: Venkatasubramanian, M.; Schwartz, L.; Ramachandra, N.; Bennett, J.; Subramanian, K. R.; Gordon-Mitchell, S.; Fromowitz, A.; Pradhan, K.; Shechter, D.; Sahu, S.; Heiser, D.; Scherle, P.; Chen, X.; Chetal, C.; Kulkarni, A.; Myers, K. C.; Weirauch, M. T.; Grimes, H. L.; Starczynowski, D. T.; Verma, A.; Salomonis, N. G.\nScore: 13.7, Published: 2024-02-08 DOI: 10.1101/2024.02.04.578798\nThe role of splicing dysregulation in cancer is underscored by splicing factor mutations; however, its impact in the absence of such rare mutations is poorly understood. To reveal complex patient subtypes and putative regulators of pathogenic splicing in Acute Myeloid Leukemia (AML), we developed a new approach called OncoSplice. Among diverse new subtypes, OncoSplice identified a biphasic poor prognosis signature that partially phenocopies U2AF1-mutant splicing, impacting thousands of genes in over 40% of adult and pediatric AML cases. U2AF1-like splicing co-opted a healthy circadian splicing program, was stable over time and induced a leukemia stem cell (LSC) program. Pharmacological inhibition of the implicated U2AF1-like splicing regulator, PRMT5, rescued leukemia mis-splicing and inhibited leukemic cell growth. Genetic deletion of IRAK4, a common target of U2AF1-like and PRMT5 treated cells, blocked leukemia development in xenograft models and induced differentiation. These analyses reveal a new prognostic alternative-splicing mechanism in malignancy, independent of splicing-factor mutations. Statement of significanceUsing a new in silico strategy we reveal counteracting determinants of patient survival in Acute Myeloid Leukemia that co-opt well-defined mutation-dependent splicing programs. Broad poor-prognosis splicing and leukemia stem cell survival could be rescued through pharmacological inhibition (PRMT5) or target deletion (IRAK4), opening the door for new precision therapies. Competing InterestsConflict-of-interest disclosure: DTS. serves on the scientific advisory board at Kurome Therapeutics; is a consultant for and/or received funding from Kurome Therapeutics, Captor Therapeutics, Treeline Biosciences, and Tolero Therapeutics; and has equity in Kurome Therapeutics. AV has received research funding from GlaxoSmithKline, BMS, Jannsen, Incyte, MedPacto, Celgene, Novartis, Curis, Prelude and Eli Lilly and Company, has received compensation as a scientific advisor to Novartis, Stelexis Therapeutics, Acceleron Pharma, and Celgene, and has equity ownership in Throws Exception and Stelexis Therapeutics.\nThree-dimensional assessments are necessary to determine the true, spatially-resolved composition of tissues\nAuthors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.\nScore: 52.9, Published: 2024-02-03 DOI: 10.1101/2023.12.04.569986\nMethods for spatially resolved cellular profiling using thinly cut sections have enabled in-depth quantitative tissue mapping to study inter-sample and intra-sample differences in normal human anatomy and disease onset and progression. These methods often profile extremely limited regions, which may impact the evaluation of heterogeneity due to tissue sub-sampling. Here, we applied CODA, a deep learning-based tissue mapping platform, to reconstruct the three-dimensional (3D) microanatomy of grossly normal and cancer-containing human pancreas biospecimens obtained from individuals who underwent pancreatic resection. To compare inter-and intra-sample heterogeneity, we assessed bulk and spatially resolved tissue composition in a cohort of two-dimensional (2D) whole slide images (WSIs) and a cohort of thick slabs of pancreas tissue that were digitally reconstructed in 3D from serial sections. To demonstrate the marked under sampling of 2D assessments, we simulated the number of WSIs and tissue microarrays (TMAs) necessary to represent the compositional heterogeneity of 3D data within 10% error to reveal that tens of WSIs and hundreds of TMA cores are sometimes needed. We show that spatial correlation of different pancreatic structures decay significantly within a span of microns, demonstrating that 2D histological sections may not be representative of their neighboring tissues. In sum, we demonstrate that 3D assessments are necessary to accurately assess tissue composition in normal and abnormal specimens and in order to accurately determine neoplastic content. These results emphasize the importance of intra-sample heterogeneity in tissue mapping efforts.\nIdentification of early-stage liver fibrosis by modifications in the interstitial space diffusive microenvironment using fluorescent single-walled carbon nanotubes\nAuthors: Lee, A.; Simon, A. A.; Boyreau, A.; Allain-Courtois, N.; Lambert, B.; PRADERE, J.-P.; SALTEL, F.; Cognet, L.\nScore: 6.6, Published: 2024-02-08 DOI: 10.1101/2024.02.06.579133\nDuring liver fibrosis, recurrent hepatic injuries lead to accumulation of collagen and other extracellular matrix components in the interstitial space, ultimately disrupting liver functions. Early stages of liver fibrosis may be reversible, but opportunities for diagnosis at these stages are currently limited. Here, we show that the alterations of the interstitial space associated with fibrosis can be probed by tracking individual fluorescent single-walled carbon nanotubes (SWCNTs) diffusing in that space. In a mouse model of early liver fibrosis, we find that nanotubes generally explore elongated areas, whose length decrease as the disease progresses even in regions where histopathological examination does not reveal fibrosis yet. Furthermore, this decrease in nanotube mobility is a purely geometrical effect, as the instantaneous nanotube diffusivity stays unmodified. This work establishes the promise of SWCNTs both for the diagnosis of liver fibrosis at an early stage, and for more in-depth studies of the biophysical effects of the disease.\nSYNERGISTIC ANTITUMOR EFFECT OF NAPROXEN AND SORAFENIB IN HEPATOCELLULAR CARCINOMA\nAuthors: Akar, E.; Unsal Beyge, S.; Kahraman, D. C.\nScore: 4.7, Published: 2024-02-05 DOI: 10.1101/2024.02.01.578341\nHepatocellular carcinoma (HCC) is one of the most prevalent and deadliest cancer types in the world. Due to the inevitable development of resistance to classical chemotherapy and current targeted therapies, combination therapy became an important strategy. Recently drug repurposing studies have emphasized the potential of non-steroidal anti-inflammatory drugs (NSAIDs) in exhibiting anti-tumor activities in different cancer types, including HCC. In this study we explored the synergistic bioactivity of naproxen in combination with sorafenib in HCC cell lines. Network reconstruction analysis, performed to identify key factors mediating the synergistic therapeutic effects of sorafenib and naproxen revealed apoptosis pathways enriched in both sorafenib and naproxen networks. CAPN2 emerged as the common gene shared between the two networks. In vitro studies on Huh7 and Mahlavu cells revealed that induction of apoptosis was associated mainly with the PI3K/AKT pathway inhibition. Furthermore, combined treatment of naproxen and sorafenib led to the downregulation of CAPN2 expression, suggesting a potential link to apoptosis and PI3K/AKT pathway regulation, which needs further investigation. In conclusion, combining sorafenib with naproxen presents a promising strategy with potential synergistic therapeutic benefits in HCC.\nRestraining Wnt activation and intestinal tumorigenesis by a Rab35 dependent GTPase relay\nAuthors: Redhai, S.; Eang, T.; Boonekamp, K. E.; Rueter, S.; Klemens, T. C.; Leible, S.; Bahuguna, S.; Port, F.; Pavlovic, B.; Holzem, M.; Doll, R.; Rindtorff, N.; Valentini, E.; Schmitt, B.; Richter, K.; Engel, U.; Huber, W.; Boutros, M.\nScore: 4.0, Published: 2024-02-06 DOI: 10.1101/2024.02.05.578891\nMaintenance of homeostatic processes ensure curtailment of intestinal tumorigenesis. Inactivating mutations to Adenomatous Polyposis Coli (Apc) result in aberrantly activated Wnt signalling and initiates colorectal cancer (CRC) in approx. 80% of cases, yet our understanding of the subcellular mechanisms that modulate dysregulated pathway activity is limited. Here, using a conditional in vivo genetic screen, we identify Rab35 GTPase as a novel tumour suppressor that modulates regional Wnt activity after loss of Apc in progenitor cells. Single cell analysis revealed that progenitor cells respond to Apc depletion by increasing the expression of a GTPase activating protein, which we named blackbelt, and triggering Rab35 disassociation from the plasma membrane. Mechanistically, we demonstrate that Rab35 controls the localisation and activation of the Rho GTPase, Cdc42, which functions as a relay to regulate JNK signalling. This in turn tunes the Wnt pathway upstream of {beta}-catenin to direct proliferation and differentiation of progenitor cells. Importantly, we show that maintaining active JNK signalling is important for the propagation of Apc mutant mouse colon organoids. Our findings highlight a novel GTPase cascade that sustains aberrant Wnt activity in specific segments of the intestine and provides impetus to therapeutically exploit this pathway to target CRC.\nNovel spirocyclic dimer, SpiD3, targets chronic lymphocytic leukemia survival pathways with potent preclinical effects\nAuthors: Eiken, A.; Smith, A. L.; Skupa, S. A.; Schmitz, E.; Rana, S.; Singh, S.; Kumar, S.; Mallareddy, J. R.; de Cubas, A. A.; Krishna, A.; Kalluchi, A.; Rowley, J.; D'Angelo, C. R.; Lunning, M. A.; Bociek, R. G.; Vose, J. M.; Natarajan, A.; ElGamal, D.\nScore: 4.0, Published: 2024-02-03 DOI: 10.1101/2024.01.31.578283\nChronic lymphocytic leukemia (CLL) cell survival and growth is fueled by the induction of B-cell receptor (BCR) signaling within the tumor microenvironment (TME) driving activation of NF- {kappa}B signaling and the unfolded protein response (UPR). Malignant cells have higher basal levels of UPR posing a unique therapeutic window to combat CLL cell growth using pharmacological agents that induce accumulation of misfolded proteins. Frontline CLL therapeutics that directly target BCR signaling such as Bruton-tyrosine kinase (BTK) inhibitors (e.g., ibrutinib) have enhanced patient survival. However, resistance mechanisms wherein tumor cells bypass BTK inhibition through acquired BTK mutations, and/or activation of alternative survival mechanisms have rendered ibrutinib ineffective, imposing the need for novel therapeutics. We evaluated SpiD3, a novel spirocyclic dimer, in CLL cell lines, patient-derived CLL samples, ibrutinib-resistant CLL cells, and in the E{micro}-TCL1 mouse model. Our integrated multi-omics and functional analyses revealed BCR signaling, NF-{kappa}B signaling, and endoplasmic reticulum stress among the top pathways modulated by SpiD3. This was accompanied by marked upregulation of the UPR and inhibition of global protein synthesis in CLL cell lines and patient-derived CLL cells. In ibrutinib-resistant CLL cells, SpiD3 retained its anti-leukemic effects, mirrored in reduced activation of key proliferative pathways (e.g., PRAS, ERK, MYC). Translationally, we observed reduced tumor burden in SpiD3-treated E{micro}-TCL1 mice. Our findings reveal that SpiD3 exploits critical vulnerabilities in CLL cells including NF-{kappa}B signaling and the UPR, culminating in profound anti-tumor properties independent of TME stimuli. STATEMENT OF SIGNIFICANCESpiD3 demonstrates cytotoxicity in CLL partially through inhibition of NF-{kappa}B signaling independent of tumor-supportive stimuli. By inducing the accumulation of unfolded proteins, SpiD3 activates the UPR and hinders protein synthesis in CLL cells. Overall, SpiD3 exploits critical CLL vulnerabilities (i.e., the NF-{kappa}B pathway and UPR) highlighting its use in drug-resistant CLL. VISUAL ABSTRACT O_FIG O_LINKSMALLFIG WIDTH=137 HEIGHT=200 SRC=\"FIGDIR/small/578283v1_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (27K): org.highwire.dtl.DTLVardef@9e416forg.highwire.dtl.DTLVardef@1da1725org.highwire.dtl.DTLVardef@125ab32org.highwire.dtl.DTLVardef@a77cc_HPS_FORMAT_FIGEXP M_FIG C_FIG\nExtrachromosomal DNA in cutaneous squamous cell carcinoma is associated with increased nodal disease\nAuthors: Bencomo, T.; Thind, A. S.; Ashford, B.; Ranson, M.; Lee, C. S.\nScore: 3.5, Published: 2024-02-08 DOI: 10.1101/2024.02.04.578845\nExtrachromosomal DNA (ecDNA) is a mechanism of oncogene amplification that occurs in many cancers and has been linked to worse patient outcomes. While ecDNA has previously been identified in cutaneous melanoma, it has yet to be described in cutaneous squamous cell carcinoma (cSCC). Using whole genome sequencing and transcriptomic data of primary and metastatic cSCC tumors, we establish the presence of ecDNA in cSCC and confirm elevated expression of ecDNA-associated genes. We also find that the presence of ecDNA correlates with a higher lymph node ratio, a measure of disease severity. Together, these findings are the first to report ecDNA in cSCC and suggest that ecDNA gene amplifications promote an aggressive cSCC phenotype.\nPharmacological control of CAR T cells through CRISPR-driven rapamycin resistance\nAuthors: Levesque, S.; Carleton, G.; Duque, V.; Goupil, C.; Fiset, J.-P.; Villeneuve, S.; Normandeau, E.; Morin, G.; Dumont, N.; Nelson, B. H.; Laganiere, J.; Boyle, B.; Lum, J. J.; Doyon, Y.\nScore: 22.8, Published: 2024-02-05 DOI: 10.1101/2023.09.14.557485\nChimeric antigen receptors (CARs) reprogram T cells to recognize and target cancer cells. Despite remarkable responses observed with CAR-T cell therapy in patients with hematological malignancies, CAR-T cell engineering still relies mostly on randomly integrating vectors, limiting the possibilities of fine-tuning T cell function. Here, we designed a CRISPR-based marker-free selection strategy to simultaneously target a therapeutic transgene and a gain-of-function mutation to the MTOR locus to enrich cells resistant to rapamycin, a clinically used immunosuppressant. We readily engineered rapamycin-resistant (RapaR) CAR-T cells by targeting CAR expression cassettes to the MTOR locus. Using in vitro cytotoxicity assays, and a humanized mouse model of acute lymphoblastic leukemia, we show that RapaR-CAR-T cells can efficiently target CD19+ leukemia cells in presence of immunosuppressing doses of rapamycin. Furthermore, our strategy allows multiplexed targeting of rapamycin-regulated immunoreceptors complexes (DARICs) to the MTOR and TRAC loci to pharmacologically control CAR-T cells activity. We foresee that our approach could both facilitate the enrichment of CRISPR-engineered CAR-T cells ex vivo and in vivo while improving tumor eradication.\n",
  "wordCount" : "2508",
  "inLanguage": "en",
  "datePublished": "2024-02-11T10:38:55Z",
  "dateModified": "2024-02-11T10:38:55Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta">&lt;span&gt;updated on February 11, 2024&lt;/span&gt;

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.30.577120">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.30.577120" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.30.577120">
        <p class="paperTitle">Mechanical confinement governs phenotypic plasticity in melanoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.30.577120" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.30.577120" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Hunter, M. V.; Montal, E.; Ma, Y.; Moncada, R.; Yanai, I.; Koche, R. P.; White, R. M.</p>
        <p class="info">Score: 31.9, Published: 2024-02-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.30.577120' target='https://doi.org/10.1101/2024.01.30.577120'> 10.1101/2024.01.30.577120</a></p>
        <p class="abstract">Phenotype switching is a form of cellular plasticity in which cancer cells reversibly move between two opposite extremes - proliferative versus invasive states. While it has long been hypothesised that such switching is triggered by external cues, the identity of these cues has remained elusive. Here, we demonstrate that mechanical confinement mediates phenotype switching through chromatin remodelling. Using a zebrafish model of melanoma coupled with human samples, we profiled tumor cells at the interface between the tumor and surrounding microenvironment. Morphological analysis of these rare cells showed flattened, elliptical nuclei suggestive of mechanical confinement by adjacent tissue. Spatial and single-cell transcriptomics demonstrated that the interface cells adopted a gene program of neuronal invasion, including acquisition of an acetylated tubulin cage that protects the nucleus during migration. We identified the DNA-bending protein HMGB2 as a confinement-induced mediator of the neuronal state. HMGB2 is upregulated in confined cells, and quantitative modelling revealed that confinement prolongs contact time between HMGB2 and chromatin, leading to changes in chromatin configuration that favor the neuronal phenotype. Genetic disruption of HMGB2 showed that it regulates the trade-off between proliferative and invasive states, in which confined HMGB2high tumor cells are less proliferative but more drug resistant. Our results implicate the mechanical microenvironment as a mechanism driving phenotype switching in melanoma.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.578742">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.578742" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.578742">
        <p class="paperTitle">αKG-mediated carnitine synthesis promotes homologous recombination via histone acetylation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.578742" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.578742" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Uboveja, A.; Huang, Z.; Buj, R.; Amalric, A.; Wang, H.; Tangudu, N. K.; Cole, A. R.; Megill, E.; Kantner, D.; Chatoff, A.; Ahmad, H.; Marcinkiewicz, M. M.; Disharoon, J. A.; Graff, S.; Dahl, E. S.; Hempel, N.; Stallaert, W.; Sidoli, S.; Bitler, B. G.; Long, D. T.; Snyder, N. W.; Aird, K. M.</p>
        <p class="info">Score: 19.5, Published: 2024-02-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.578742' target='https://doi.org/10.1101/2024.02.06.578742'> 10.1101/2024.02.06.578742</a></p>
        <p class="abstract">Homologous recombination (HR) deficiency enhances sensitivity to DNA damaging agents commonly used to treat cancer. In HR-proficient cancers, metabolic mechanisms driving response or resistance to DNA damaging agents remain unclear. Here we identified that depletion of alpha-ketoglutarate (KG) sensitizes HR-proficient cells to DNA damaging agents by metabolic regulation of histone acetylation. KG is required for the activity of KG-dependent dioxygenases (KGDDs), and prior work has shown that changes in KGDD affect demethylases. Using a targeted CRISPR knockout library consisting of 64 KGDDs, we discovered that Trimethyllysine Hydroxylase Epsilon (TMLHE), the first and rate-limiting enzyme in de novo carnitine synthesis, is necessary for proliferation of HR-proficient cells in the presence of DNA damaging agents. Unexpectedly, KG-mediated TMLHE-dependent carnitine synthesis was required for histone acetylation, while histone methylation was affected but dispensable. The increase in histone acetylation via KG-dependent carnitine synthesis promoted HR-mediated DNA repair through site- and substrate-specific histone acetylation. These data demonstrate for the first time that HR-proficiency is mediated through KG directly influencing histone acetylation via carnitine synthesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.04.578798">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.04.578798" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.04.578798">
        <p class="paperTitle">Broad de-regulated U2AF1 splicing is prognostic and augments leukemic transformation via protein arginine methyltransferase activation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.04.578798" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.04.578798" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Venkatasubramanian, M.; Schwartz, L.; Ramachandra, N.; Bennett, J.; Subramanian, K. R.; Gordon-Mitchell, S.; Fromowitz, A.; Pradhan, K.; Shechter, D.; Sahu, S.; Heiser, D.; Scherle, P.; Chen, X.; Chetal, C.; Kulkarni, A.; Myers, K. C.; Weirauch, M. T.; Grimes, H. L.; Starczynowski, D. T.; Verma, A.; Salomonis, N. G.</p>
        <p class="info">Score: 13.7, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.04.578798' target='https://doi.org/10.1101/2024.02.04.578798'> 10.1101/2024.02.04.578798</a></p>
        <p class="abstract">The role of splicing dysregulation in cancer is underscored by splicing factor mutations; however, its impact in the absence of such rare mutations is poorly understood. To reveal complex patient subtypes and putative regulators of pathogenic splicing in Acute Myeloid Leukemia (AML), we developed a new approach called OncoSplice. Among diverse new subtypes, OncoSplice identified a biphasic poor prognosis signature that partially phenocopies U2AF1-mutant splicing, impacting thousands of genes in over 40% of adult and pediatric AML cases. U2AF1-like splicing co-opted a healthy circadian splicing program, was stable over time and induced a leukemia stem cell (LSC) program. Pharmacological inhibition of the implicated U2AF1-like splicing regulator, PRMT5, rescued leukemia mis-splicing and inhibited leukemic cell growth. Genetic deletion of IRAK4, a common target of U2AF1-like and PRMT5 treated cells, blocked leukemia development in xenograft models and induced differentiation. These analyses reveal a new prognostic alternative-splicing mechanism in malignancy, independent of splicing-factor mutations.

Statement of significanceUsing a new in silico strategy we reveal counteracting determinants of patient survival in Acute Myeloid Leukemia that co-opt well-defined mutation-dependent splicing programs. Broad poor-prognosis splicing and leukemia stem cell survival could be rescued through pharmacological inhibition (PRMT5) or target deletion (IRAK4), opening the door for new precision therapies.

Competing InterestsConflict-of-interest disclosure: DTS. serves on the scientific advisory board at Kurome Therapeutics; is a consultant for and/or received funding from Kurome Therapeutics, Captor Therapeutics, Treeline Biosciences, and Tolero Therapeutics; and has equity in Kurome Therapeutics. AV has received research funding from GlaxoSmithKline, BMS, Jannsen, Incyte, MedPacto, Celgene, Novartis, Curis, Prelude and Eli Lilly and Company, has received compensation as a scientific advisor to Novartis, Stelexis Therapeutics, Acceleron Pharma, and Celgene, and has equity ownership in Throws Exception and Stelexis Therapeutics.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.12.04.569986">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.12.04.569986" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.12.04.569986">
        <p class="paperTitle">Three-dimensional assessments are necessary to determine the true, spatially-resolved composition of tissues</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.12.04.569986" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.12.04.569986" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Forjaz, A.; Vaz, E.; Romero, V. M.; Joshi, S.; Braxton, A. M.; Jiang, A. C.; Fujikura, K.; Cornish, T.; Hong, S.-M.; Hruban, R. H.; Wu, P.-H.; Wood, L. D.; Kiemen, A. L.; Wirtz, D.</p>
        <p class="info">Score: 52.9, Published: 2024-02-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.12.04.569986' target='https://doi.org/10.1101/2023.12.04.569986'> 10.1101/2023.12.04.569986</a></p>
        <p class="abstract">Methods for spatially resolved cellular profiling using thinly cut sections have enabled in-depth quantitative tissue mapping to study inter-sample and intra-sample differences in normal human anatomy and disease onset and progression. These methods often profile extremely limited regions, which may impact the evaluation of heterogeneity due to tissue sub-sampling. Here, we applied CODA, a deep learning-based tissue mapping platform, to reconstruct the three-dimensional (3D) microanatomy of grossly normal and cancer-containing human pancreas biospecimens obtained from individuals who underwent pancreatic resection. To compare inter-and intra-sample heterogeneity, we assessed bulk and spatially resolved tissue composition in a cohort of two-dimensional (2D) whole slide images (WSIs) and a cohort of thick slabs of pancreas tissue that were digitally reconstructed in 3D from serial sections. To demonstrate the marked under sampling of 2D assessments, we simulated the number of WSIs and tissue microarrays (TMAs) necessary to represent the compositional heterogeneity of 3D data within 10% error to reveal that tens of WSIs and hundreds of TMA cores are sometimes needed. We show that spatial correlation of different pancreatic structures decay significantly within a span of microns, demonstrating that 2D histological sections may not be representative of their neighboring tissues. In sum, we demonstrate that 3D assessments are necessary to accurately assess tissue composition in normal and abnormal specimens and in order to accurately determine neoplastic content. These results emphasize the importance of intra-sample heterogeneity in tissue mapping efforts.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.06.579133">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.06.579133" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.06.579133">
        <p class="paperTitle">Identification of early-stage liver fibrosis by modifications in the interstitial space diffusive microenvironment using fluorescent single-walled carbon nanotubes</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.06.579133" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.06.579133" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lee, A.; Simon, A. A.; Boyreau, A.; Allain-Courtois, N.; Lambert, B.; PRADERE, J.-P.; SALTEL, F.; Cognet, L.</p>
        <p class="info">Score: 6.6, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.06.579133' target='https://doi.org/10.1101/2024.02.06.579133'> 10.1101/2024.02.06.579133</a></p>
        <p class="abstract">During liver fibrosis, recurrent hepatic injuries lead to accumulation of collagen and other extracellular matrix components in the interstitial space, ultimately disrupting liver functions. Early stages of liver fibrosis may be reversible, but opportunities for diagnosis at these stages are currently limited. Here, we show that the alterations of the interstitial space associated with fibrosis can be probed by tracking individual fluorescent single-walled carbon nanotubes (SWCNTs) diffusing in that space. In a mouse model of early liver fibrosis, we find that nanotubes generally explore elongated areas, whose length decrease as the disease progresses even in regions where histopathological examination does not reveal fibrosis yet. Furthermore, this decrease in nanotube mobility is a purely geometrical effect, as the instantaneous nanotube diffusivity stays unmodified. This work establishes the promise of SWCNTs both for the diagnosis of liver fibrosis at an early stage, and for more in-depth studies of the biophysical effects of the disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.01.578341">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.01.578341" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.01.578341">
        <p class="paperTitle">SYNERGISTIC ANTITUMOR EFFECT OF NAPROXEN AND SORAFENIB IN HEPATOCELLULAR CARCINOMA</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.01.578341" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.01.578341" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Akar, E.; Unsal Beyge, S.; Kahraman, D. C.</p>
        <p class="info">Score: 4.7, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.01.578341' target='https://doi.org/10.1101/2024.02.01.578341'> 10.1101/2024.02.01.578341</a></p>
        <p class="abstract">Hepatocellular carcinoma (HCC) is one of the most prevalent and deadliest cancer types in the world. Due to the inevitable development of resistance to classical chemotherapy and current targeted therapies, combination therapy became an important strategy. Recently drug repurposing studies have emphasized the potential of non-steroidal anti-inflammatory drugs (NSAIDs) in exhibiting anti-tumor activities in different cancer types, including HCC. In this study we explored the synergistic bioactivity of naproxen in combination with sorafenib in HCC cell lines. Network reconstruction analysis, performed to identify key factors mediating the synergistic therapeutic effects of sorafenib and naproxen revealed apoptosis pathways enriched in both sorafenib and naproxen networks. CAPN2 emerged as the common gene shared between the two networks. In vitro studies on Huh7 and Mahlavu cells revealed that induction of apoptosis was associated mainly with the PI3K/AKT pathway inhibition. Furthermore, combined treatment of naproxen and sorafenib led to the downregulation of CAPN2 expression, suggesting a potential link to apoptosis and PI3K/AKT pathway regulation, which needs further investigation. In conclusion, combining sorafenib with naproxen presents a promising strategy with potential synergistic therapeutic benefits in HCC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.05.578891">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.05.578891" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.05.578891">
        <p class="paperTitle">Restraining Wnt activation and intestinal tumorigenesis by a Rab35 dependent GTPase relay</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.05.578891" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.05.578891" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Redhai, S.; Eang, T.; Boonekamp, K. E.; Rueter, S.; Klemens, T. C.; Leible, S.; Bahuguna, S.; Port, F.; Pavlovic, B.; Holzem, M.; Doll, R.; Rindtorff, N.; Valentini, E.; Schmitt, B.; Richter, K.; Engel, U.; Huber, W.; Boutros, M.</p>
        <p class="info">Score: 4.0, Published: 2024-02-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.05.578891' target='https://doi.org/10.1101/2024.02.05.578891'> 10.1101/2024.02.05.578891</a></p>
        <p class="abstract">Maintenance of homeostatic processes ensure curtailment of intestinal tumorigenesis. Inactivating mutations to Adenomatous Polyposis Coli (Apc) result in aberrantly activated Wnt signalling and initiates colorectal cancer (CRC) in approx. 80% of cases, yet our understanding of the subcellular mechanisms that modulate dysregulated pathway activity is limited. Here, using a conditional in vivo genetic screen, we identify Rab35 GTPase as a novel tumour suppressor that modulates regional Wnt activity after loss of Apc in progenitor cells. Single cell analysis revealed that progenitor cells respond to Apc depletion by increasing the expression of a GTPase activating protein, which we named blackbelt, and triggering Rab35 disassociation from the plasma membrane. Mechanistically, we demonstrate that Rab35 controls the localisation and activation of the Rho GTPase, Cdc42, which functions as a relay to regulate JNK signalling. This in turn tunes the Wnt pathway upstream of {beta}-catenin to direct proliferation and differentiation of progenitor cells. Importantly, we show that maintaining active JNK signalling is important for the propagation of Apc mutant mouse colon organoids. Our findings highlight a novel GTPase cascade that sustains aberrant Wnt activity in specific segments of the intestine and provides impetus to therapeutically exploit this pathway to target CRC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.01.31.578283">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.01.31.578283" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.01.31.578283">
        <p class="paperTitle">Novel spirocyclic dimer, SpiD3, targets chronic lymphocytic leukemia survival pathways with potent preclinical effects</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.01.31.578283" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.01.31.578283" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Eiken, A.; Smith, A. L.; Skupa, S. A.; Schmitz, E.; Rana, S.; Singh, S.; Kumar, S.; Mallareddy, J. R.; de Cubas, A. A.; Krishna, A.; Kalluchi, A.; Rowley, J.; D&#39;Angelo, C. R.; Lunning, M. A.; Bociek, R. G.; Vose, J. M.; Natarajan, A.; ElGamal, D.</p>
        <p class="info">Score: 4.0, Published: 2024-02-03 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.01.31.578283' target='https://doi.org/10.1101/2024.01.31.578283'> 10.1101/2024.01.31.578283</a></p>
        <p class="abstract">Chronic lymphocytic leukemia (CLL) cell survival and growth is fueled by the induction of B-cell receptor (BCR) signaling within the tumor microenvironment (TME) driving activation of NF- {kappa}B signaling and the unfolded protein response (UPR). Malignant cells have higher basal levels of UPR posing a unique therapeutic window to combat CLL cell growth using pharmacological agents that induce accumulation of misfolded proteins. Frontline CLL therapeutics that directly target BCR signaling such as Bruton-tyrosine kinase (BTK) inhibitors (e.g., ibrutinib) have enhanced patient survival. However, resistance mechanisms wherein tumor cells bypass BTK inhibition through acquired BTK mutations, and/or activation of alternative survival mechanisms have rendered ibrutinib ineffective, imposing the need for novel therapeutics. We evaluated SpiD3, a novel spirocyclic dimer, in CLL cell lines, patient-derived CLL samples, ibrutinib-resistant CLL cells, and in the E{micro}-TCL1 mouse model. Our integrated multi-omics and functional analyses revealed BCR signaling, NF-{kappa}B signaling, and endoplasmic reticulum stress among the top pathways modulated by SpiD3. This was accompanied by marked upregulation of the UPR and inhibition of global protein synthesis in CLL cell lines and patient-derived CLL cells. In ibrutinib-resistant CLL cells, SpiD3 retained its anti-leukemic effects, mirrored in reduced activation of key proliferative pathways (e.g., PRAS, ERK, MYC). Translationally, we observed reduced tumor burden in SpiD3-treated E{micro}-TCL1 mice. Our findings reveal that SpiD3 exploits critical vulnerabilities in CLL cells including NF-{kappa}B signaling and the UPR, culminating in profound anti-tumor properties independent of TME stimuli.

STATEMENT OF SIGNIFICANCESpiD3 demonstrates cytotoxicity in CLL partially through inhibition of NF-{kappa}B signaling independent of tumor-supportive stimuli. By inducing the accumulation of unfolded proteins, SpiD3 activates the UPR and hinders protein synthesis in CLL cells. Overall, SpiD3 exploits critical CLL vulnerabilities (i.e., the NF-{kappa}B pathway and UPR) highlighting its use in drug-resistant CLL.

VISUAL ABSTRACT

O_FIG O_LINKSMALLFIG WIDTH=137 HEIGHT=200 SRC=&#34;FIGDIR/small/578283v1_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (27K):
org.highwire.dtl.DTLVardef@9e416forg.highwire.dtl.DTLVardef@1da1725org.highwire.dtl.DTLVardef@125ab32org.highwire.dtl.DTLVardef@a77cc_HPS_FORMAT_FIGEXP  M_FIG C_FIG</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2024.02.04.578845">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2024.02.04.578845" aria-expanded="false" aria-controls="flush-collapse10.1101/2024.02.04.578845">
        <p class="paperTitle">Extrachromosomal DNA in cutaneous squamous cell carcinoma is associated with increased nodal disease</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2024.02.04.578845" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2024.02.04.578845" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bencomo, T.; Thind, A. S.; Ashford, B.; Ranson, M.; Lee, C. S.</p>
        <p class="info">Score: 3.5, Published: 2024-02-08 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2024.02.04.578845' target='https://doi.org/10.1101/2024.02.04.578845'> 10.1101/2024.02.04.578845</a></p>
        <p class="abstract">Extrachromosomal DNA (ecDNA) is a mechanism of oncogene amplification that occurs in many cancers and has been linked to worse patient outcomes. While ecDNA has previously been identified in cutaneous melanoma, it has yet to be described in cutaneous squamous cell carcinoma (cSCC). Using whole genome sequencing and transcriptomic data of primary and metastatic cSCC tumors, we establish the presence of ecDNA in cSCC and confirm elevated expression of ecDNA-associated genes. We also find that the presence of ecDNA correlates with a higher lymph node ratio, a measure of disease severity. Together, these findings are the first to report ecDNA in cSCC and suggest that ecDNA gene amplifications promote an aggressive cSCC phenotype.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.14.557485">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.14.557485" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.14.557485">
        <p class="paperTitle">Pharmacological control of CAR T cells through CRISPR-driven rapamycin resistance</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.14.557485" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.14.557485" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Levesque, S.; Carleton, G.; Duque, V.; Goupil, C.; Fiset, J.-P.; Villeneuve, S.; Normandeau, E.; Morin, G.; Dumont, N.; Nelson, B. H.; Laganiere, J.; Boyle, B.; Lum, J. J.; Doyon, Y.</p>
        <p class="info">Score: 22.8, Published: 2024-02-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.14.557485' target='https://doi.org/10.1101/2023.09.14.557485'> 10.1101/2023.09.14.557485</a></p>
        <p class="abstract">Chimeric antigen receptors (CARs) reprogram T cells to recognize and target cancer cells. Despite remarkable responses observed with CAR-T cell therapy in patients with hematological malignancies, CAR-T cell engineering still relies mostly on randomly integrating vectors, limiting the possibilities of fine-tuning T cell function. Here, we designed a CRISPR-based marker-free selection strategy to simultaneously target a therapeutic transgene and a gain-of-function mutation to the MTOR locus to enrich cells resistant to rapamycin, a clinically used immunosuppressant. We readily engineered rapamycin-resistant (RapaR) CAR-T cells by targeting CAR expression cassettes to the MTOR locus. Using in vitro cytotoxicity assays, and a humanized mouse model of acute lymphoblastic leukemia, we show that RapaR-CAR-T cells can efficiently target CD19&#43; leukemia cells in presence of immunosuppressing doses of rapamycin. Furthermore, our strategy allows multiplexed targeting of rapamycin-regulated immunoreceptors complexes (DARICs) to the MTOR and TRAC loci to pharmacologically control CAR-T cells activity. We foresee that our approach could both facilitate the enrichment of CRISPR-engineered CAR-T cells ex vivo and in vivo while improving tumor eradication.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2024 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
