<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="Major data analysis errors invalidate cancer microbiome findings
Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.
Score: 625.9, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993
We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="Major data analysis errors invalidate cancer microbiome findings
Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.
Score: 625.9, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993
We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-08-09T10:39:33+00:00" />
<meta property="article:modified_time" content="2023-08-09T10:39:33+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="Major data analysis errors invalidate cancer microbiome findings
Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.
Score: 625.9, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993
We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "Major data analysis errors invalidate cancer microbiome findings\nAuthors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.\nScore: 625.9, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993\nWe re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier.",
  "keywords": [
    
  ],
  "articleBody": " Major data analysis errors invalidate cancer microbiome findings\nAuthors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.\nScore: 625.9, Published: 2023-07-31 DOI: 10.1101/2023.07.28.550993\nWe re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier. Each of these problems invalidates the results, leading to the conclusion that the microbiome-based classifiers for identifying cancer presented in the study are entirely wrong. These flaws have subsequently affected more than a dozen additional published studies that used the same data and whose results are likely invalid as well.\nA pro-inflammatory stem cell niche drives myelofibrosis through a targetable galectin 1 axis\nAuthors: Li, R.; Colombo, M.; Wang, G.; Rodriguez-Romera, A.; O'Sullivan, J.; Clark, S.-A.; Perez Saez, J.; Meng, Y.; Khan, A.; Wen, S.; Dong, P.; Zhou, W.; Sousos, N.; Murphy, L.; Clarke, M.; Jooss, N.; Olijnik, A.-A.; Wong, Z.; Sirinukunwattana, K.; Ryou, H.; Norfo, R.; Cheng, Q.; Brierley, C.; Carrelha, J.; Ren, Z.; Thongjuea, S.; Rathinam, V.; Krishnan, A. K.; Royston, D.; Rabinovich, G.; Mead, A.; Psaila, B.\nScore: 24.6, Published: 2023-08-07 DOI: 10.1101/2023.08.05.550630\nMyeloproliferative neoplasms are stem cell-driven cancers associated with a large burden of morbidity and mortality. The majority of patients present with early-stage disease, but a substantial proportion progress to myelofibrosis and/or secondary leukemia, advanced cancers with a poor prognosis and high symptom burden. Currently, it remains difficult to predict progression, and we lack therapies that reliably prevent or reverse fibrosis development. A major bottleneck to the discovery of disease-modifying therapies has been an incomplete understanding of the interplay between perturbed cellular and molecular states. Several cell types have individually been implicated, but a comprehensive analysis of myelofibrotic bone marrow is lacking. We therefore mapped the crosstalk between bone marrow cell types in myelofibrotic bone marrow. We found that inflammation and fibrosis are orchestrated by a quartet of immune and stromal cell lineages-with basophils and mast cells creating a TNF signaling hub, communicating with megakaryocytes, mesenchymal stromal cells and pro-inflammatory fibroblasts. We identified the b-galactoside binding protein galectin 1 as a striking biomarker of progression to myelofibrosis and poor survival in multiple patient cohorts, and as a promising therapeutic target, with reduced myeloproliferation and fibrosis in vitro and in vivo and improved survival following galectin 1 inhibition. In human bone marrow organoids, TNF increased galectin 1 expression, suggesting a feedback loop wherein the pro-inflammatory MPN clone creates a self-reinforcing niche, fueling progression to advanced disease. This study provides a valuable resource for studying hematopoietic cell-niche interactions, with broad relevance for cancer-associated inflammation and disorders of tissue fibrosis.\nA genetic basis for cancer sex differences revealed in Xp11 translocation renal cell carcinoma\nAuthors: Achom, M.; Sadagopan, A.; Bao, C.; McBride, F.; Xu, Q.; Konda, P.; Tourdot, R. W.; Li, J.; Nakhoul, M.; Gallant, D.; Ahmed, U. A.; O'Toole, J.; Freeman, D.; Lee, G.-S. M.; Hecht, J.; Kauffman, E.; Einstein, D. J.; Choueiri, T. K.; Zhang, C.-Z.; Viswanathan, S. R.\nScore: 17.6, Published: 2023-08-06 DOI: 10.1101/2023.08.04.552029\nXp11 translocation renal cell carcinoma (tRCC) is a female-predominant kidney cancer driven by translocations between the TFE3 gene on chromosome Xp11.2 and partner genes located on either chrX or on autosomes. The rearrangement processes that underlie TFE3 fusions, and whether they are linked to the female sex bias of this cancer, are largely unexplored. Moreover, whether oncogenic TFE3 fusions arise from both the active and inactive X chromosomes in females remains unknown. Here we address these questions by haplotype-specific analyses of whole-genome sequences of 29 tRCC samples from 15 patients and by re-analysis of 145 published tRCC whole-exome sequences. We show that TFE3 fusions universally arise as reciprocal translocations with minimal DNA loss or insertion at paired break ends. Strikingly, we observe a near exact 2:1 female:male ratio in TFE3 fusions arising via X:autosomal translocation (but not via X inversion), which accounts for the female predominance of tRCC. This 2:1 ratio is at least partially attributable to oncogenic fusions involving the inactive X chromosome and is accompanied by partial re-activation of silenced chrX genes on the rearranged chromosome. Our results highlight how somatic alterations involving the X chromosome place unique constraints on tumor initiation and exemplify how genetic rearrangements of the sex chromosomes can underlie cancer sex differences.\nTargeting the dependence on PIK3C3-mTORC1 signaling in dormancy-prone breast cancer cells blunts metastasis initiation\nAuthors: Elkholi, I. E.; Robert, A.; Kuasne, H.; Drapela, S.; Macleod, G.; Hebert, S.; Pacis, A.; Calderon, V.; Kleinman, C. L.; Gomes, A. P.; Aguirre-Ghiso, J. A.; Park, M.; Angers, S.; Cote, J.-F.\nScore: 13.8, Published: 2023-08-05 DOI: 10.1101/2023.08.02.551681\nHalting breast cancer metastatic relapses following primary tumor removal and the clinical dormant phase, remains challenging, due to a lack of specific vulnerabilities to target during dormancy. To address this, we conducted genome-wide CRISPR screens on two breast cancer cell lines with distinct dormancy properties: 4T1 (short-term dormancy) and 4T07 (prolonged dormancy). We discovered that loss of class-III PI3K, Pik3c3, revealed a unique vulnerability in 4T07 cells. Surprisingly, dormancy-prone 4T07 cells exhibited higher mTORC1 activity than 4T1 cells, due to lysosome-dependent signaling occurring at the cell periphery. Pharmacological inhibition of Pik3c3 counteracted this phenotype in 4T07 cells, and selectively reduced metastasis burden only in the 4T07 dormancy prone model. This mechanism was also detected in xenografts from human breast cancer patients, supporting that it may be relevant in humans. Our findings suggest dormant cancer cell-initiated metastasis may be prevented in patients carrying tumor cells that display PIK3C3-peripheral lysosomal signaling to mTORC1. TeaserPIK3C3-mTORC1 signaling is a novel druggable pathway in breast cancer metastasis.\nLoss of Polycomb Repressive Complex 2 function causes asparaginase resistance in T-acute lymphoblastic leukemia through decreased WNT pathway activity\nAuthors: Lefeivre, T.; Tudose, C.; Grosu, T.-I.; Jones, L.; Leon, T.; Wynne, K.; Oliviero, G.; Smith, O.; Trinquand, A.; Mansour, M.; Ryan, C.; Bond, J.\nScore: 11.6, Published: 2023-08-06 DOI: 10.1101/2023.08.04.552014\nLoss-of-function mutations and deletions in core components of the epigenetic Polycomb Repressive Complex 2 (PRC2) are associated with poor prognosis and treatment resistance in T-acute lymphoblastic leukemia (T-ALL). We leveraged clinical mutational and transcriptional data to identify a functional link between PRC2 alterations and changes in WNT signaling pathway activity in leukemia cells. Computational integration of transcriptomic, proteomic and phosphoproteomic data from an isogenic T-ALL cellular model revealed reduced activity of the WNT-dependent stabilization of proteins (WNT/STOP) pathway in cells lacking core PRC2 factor EZH2. We discovered that PRC2 loss significantly reduced sensitivity to key T-ALL treatment asparaginase, and that this was mechanistically linked to increased cellular ubiquitination levels that bolstered leukemia cell asparagine reserves. We further found that asparaginase resistance in PRC2-depleted leukemic blasts could be mitigated by pharmaceutical proteasome inhibition, thereby providing a novel and clinically tractable means to tackle induction treatment failure in high-risk T-ALL.\nGlioma-neuronal circuit remodeling induces regional immunosuppression\nAuthors: Nejo, T.; Krishna, S.; Jimenez, C.; Yamamichi, A.; Young, J. S.; Lakshmanachetty, S.; Chen, T.; Phyu, S. S. S.; Ogino, H.; Watchmaker, P.; Diebold, D.; Choudhury, A.; Daniel, A. G. S.; Raleigh, D. R.; Hervey-Jumper, S. L.; Okada, H.\nScore: 9.9, Published: 2023-08-06 DOI: 10.1101/2023.08.04.548295\nNeuronal activity-driven mechanisms impact glioblastoma cell proliferation and invasion1-7, and glioblastoma remodels neuronal circuits8,9. Distinct intratumoral regions maintain functional connectivity via a subpopulation of malignant cells that mediate tumor-intrinsic neuronal connectivity and synaptogenesis through their transcriptional programs8. However, the effects of tumor-intrinsic neuronal activity on other cells, such as immune cells, remain unknown. Here we show that regions within glioblastomas with elevated connectivity are characterized by regional immunosuppression. This was accompanied by different cell compositions and inflammatory status of tumor-associated macrophages (TAMs) in the tumor microenvironment. In preclinical intracerebral syngeneic glioblastoma models, CRISPR/Cas9 gene knockout of Thrombospondin-1 (TSP-1/Thbs1), a synaptogenic factor critical for glioma-induced neuronal circuit remodeling, in glioblastoma cells suppressed synaptogenesis and glutamatergic neuronal hyperexcitability, while simultaneously restoring antigen-presentation and pro-inflammatory responses. Moreover, TSP-1 knockout prolonged survival of immunocompetent mice harboring intracerebral syngeneic glioblastoma, but not of immunocompromised mice, and promoted infiltrations of pro-inflammatory TAMs and CD8+ T-cells in the tumor microenvironment. Notably, pharmacological inhibition of glutamatergic excitatory signals redirected tumor-associated macrophages toward a less immunosuppressive phenotype, resulting in prolonged survival. Altogether, our results demonstrate previously unrecognized immunosuppression mechanisms resulting from glioma-neuronal circuit remodeling and suggest future strategies targeting glioma-neuron-immune crosstalk may open up new avenues for immunotherapy.\nThe relationship between diet, plasma glucose, and cancer prevalence across vertebrates\nAuthors: Kapsetaki, S. E.; Basile, A. J.; Compton, Z. T.; Rupp, S. M.; Duke, E. G.; Boddy, A. M.; Harrison, T. M.; Sweazea, K. L.; Maley, C. C.\nScore: 8.3, Published: 2023-08-02 DOI: 10.1101/2023.07.31.551378\nCould diet and mean plasma glucose concentration (MPGluC) explain the variation in cancer prevalence across species? We collected diet, MPGluC, and neoplasia data for 160 vertebrate species from existing databases. We found that MPGluC negatively correlates with cancer and neoplasia prevalence, mostly of gastrointestinal organs. Trophic level positively correlates with cancer and neoplasia prevalence even after controlling for species MPGluC. Most species with high MPGluC (50/78 species = 64.1%) were birds. Most species in high trophic levels (42/53 species = 79.2%) were reptiles and mammals. Our results may be explained by the evolution of insulin resistance in birds which selected for loss or downregulation of genes related to insulin-mediated glucose import in cells. This led to higher MPGluC, intracellular caloric restriction, production of fewer reactive oxygen species and inflammatory cytokines, and longer telomeres contributing to longer longevity and lower neoplasia prevalence in extant birds relative to other vertebrates.\nMEK inhibition enhances the antitumor effect of radiation therapy in NF1-deficient glioblastoma\nAuthors: Ioannou, M.; Lalwani, K.; Ayanlaja, A. A.; Chinnasamy, V.; Pratilas, C. A.; Schreck, K. C.\nScore: 7.4, Published: 2023-08-06 DOI: 10.1101/2023.08.04.552061\nIndividuals with neurofibromatosis type 1 (NF-1), an autosomal dominant neurogenetic and tumor predisposition syndrome, are susceptible to developing low-grade glioma (LGG) and, less commonly, high-grade glioma (HGG). These gliomas exhibit loss of the neurofibromin gene (NF1), and 10-15% of sporadic HGG have somatic NF1 alterations. Loss of NF1 leads to hyperactive RAS signaling, creating opportunity given the established efficacy of MEK inhibitors (MEKi) in plexiform neurofibromas and some individuals with LGG. We observed that NF1-deficient glioblastoma neurospheres were sensitive to the combination of a MEKi (mirdametinib) with irradiation, as evidenced by synergistic inhibition of cell growth, colony formation, and increased cell death. In contrast, NF1-intact neurospheres were not sensitive to the combination, despite complete ERK pathway inhibition. No neurosphere lines exhibited enhanced sensitivity to temozolomide combined with mirdametinib. Mirdametinib decreased transcription of homologous recombination genes and RAD51 foci, associated with DNA damage repair, in sensitive models. Heterotopic xenograft models displayed synergistic growthinhibition to mirdametinib combined with irradiation in NF1-deficient glioma xenografts, but not those with intact NF1. In sensitive models, benefits were observed at least three weeks beyond the completion of treatment, including sustained phospho-ERK inhibition on immunoblot and decreased Ki-67 expression. These observations demonstrate synergistic activity between mirdametinib and irradiation in NF1-deficient glioma models and may have clinical implications for patients with gliomas that harbor germline or somatic NF1 alterations.\nCharacterizing heterogeneity along EMT and metabolic axes in colorectal cancer reveals underlying consensus molecular subtype-specific trends\nAuthors: Sehgal, M.; R, S.; Vaz, J. M.; Raja G, Y.; Muralidharan, S.; Venkatraghavan, S.; Jolly, M. K.\nScore: 4.6, Published: 2023-08-06 DOI: 10.1101/2023.08.06.551165\nColorectal cancer (CRC) is highly heterogenous with variable survival outcomes and therapeutic vulnerabilities. A commonly used classification system in CRC is the Consensus Molecular Subtypes (CMS) based on gene expression patterns. However, how these CMS categories connect to axes of phenotypic plasticity and heterogeneity remains unclear. Here, we analyze 101 bulk transcriptomic datasets, along with patient tumor samples from TCGA and single-cell RNA sequencing data, to evaluate the extent of variation among CMS subtypes across metabolic plasticity and EMT axes. Our results show that CMS2 and CMS3 samples were relatively more epithelial as compared to CMS1 and CMS4. Single-cell RNA-seq analysis of CMS1 revealed two subpopulations: one close to CMS4 (more mesenchymal) and the other closer to CMS2 or CMS3 (more epithelial), indicating a partial EMT-like behavior. Further, in our meta-analysis and in TCGA data, epithelial phenotype score was positively correlated with scores of glycolysis, OXPHOS and FAO pathways, while mesenchymal scores showed CMS subtype-specific associations with metabolic axes. PD-L1 activity scores, however, consistently correlated positively with mesenchymal signature ones and negatively with epithelial signature ones, across the four CMS categories. Together, our results quantify the patterns of two interconnected axes of phenotypic heterogeneity - EMT and metabolic reprogramming - at a CMS subtype level in CRC.\nIdentification of a Hypoxia-induced Inflammatory Cell Death Pathway\nAuthors: Bhardwaj, A.; Antonelli, M.; Ueberheide, B.; Neel, B. G.\nScore: 4.0, Published: 2023-08-06 DOI: 10.1101/2023.08.05.552118\nHypoxic cancer cells resist many anti-neoplastic therapies and can seed recurrence. We found previously that PTP1B deficiency promotes HER2+ breast cancer cell death in hypoxia by activating RNF213, an ~600kDa protein containing AAA-ATPase domains and two ubiquitin ligase domains (RING and RZ), which is implicated in Moyamoya disease (MMD). Here we report that PTP1B and ABL1/2 reciprocally control RNF213 phosphorylation on tyrosine-1275. This phosphorylation promotes RNF213 oligomerization and RZ domain activation. The RZ domain ubiquitylates CYLD/SPATA2, and together with the LUBAC complex, induces their degradation. Decreased CYLD/SPATA2 levels cause NF-kB activation, which together with hypoxia-induced ER-stress triggers pyroptosis. Mutagenesis experiments show that the RING domain negatively regulates the RZ domain, and RING mutants, including MMD alleles, catalyze LUBAC-independent CYLD/SPATA2 degradation. Our results identify an inflammatory cell death pathway that kills hypoxic tumors, reveal new insights into RNF213 regulation, and have important implications for MMD, atherosclerosis, and inflammatory disorder pathogenesis.\n",
  "wordCount" : "2350",
  "inLanguage": "en",
  "datePublished": "2023-08-09T10:39:33Z",
  "dateModified": "2023-08-09T10:39:33Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on August 9, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.28.550993">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.28.550993" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.28.550993">
        <p class="paperTitle">Major data analysis errors invalidate cancer microbiome findings</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.28.550993" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.28.550993" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Gihawi, A.; Ge, Y.; Lu, J.; Puiu, D.; Xu, A.; Cooper, C. S.; Brewer, D. S.; Pertea, M.; Salzberg, S.</p>
        <p class="info">Score: 625.9, Published: 2023-07-31 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.28.550993' target='https://doi.org/10.1101/2023.07.28.550993'> 10.1101/2023.07.28.550993</a></p>
        <p class="abstract">We re-analyzed the data from a recent large-scale study that reported strong correlations between microbial organisms and 33 different cancer types, and that created machine learning predictors with near-perfect accuracy at distinguishing among cancers. We found at least two fundamental flaws in the reported data and in the methods: (1) errors in the genome database and the associated computational methods led to millions of false positive findings of bacterial reads across all samples, largely because most of the sequences identified as bacteria were instead human; and (2) errors in transformation of the raw data created an artificial signature, even for microbes with no reads detected, tagging each tumor type with a distinct signal that the machine learning programs then used to create an apparently accurate classifier. Each of these problems invalidates the results, leading to the conclusion that the microbiome-based classifiers for identifying cancer presented in the study are entirely wrong. These flaws have subsequently affected more than a dozen additional published studies that used the same data and whose results are likely invalid as well.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.550630">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.550630" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.550630">
        <p class="paperTitle">A pro-inflammatory stem cell niche drives myelofibrosis through a targetable galectin 1 axis</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.550630" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.550630" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Li, R.; Colombo, M.; Wang, G.; Rodriguez-Romera, A.; O&#39;Sullivan, J.; Clark, S.-A.; Perez Saez, J.; Meng, Y.; Khan, A.; Wen, S.; Dong, P.; Zhou, W.; Sousos, N.; Murphy, L.; Clarke, M.; Jooss, N.; Olijnik, A.-A.; Wong, Z.; Sirinukunwattana, K.; Ryou, H.; Norfo, R.; Cheng, Q.; Brierley, C.; Carrelha, J.; Ren, Z.; Thongjuea, S.; Rathinam, V.; Krishnan, A. K.; Royston, D.; Rabinovich, G.; Mead, A.; Psaila, B.</p>
        <p class="info">Score: 24.6, Published: 2023-08-07 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.550630' target='https://doi.org/10.1101/2023.08.05.550630'> 10.1101/2023.08.05.550630</a></p>
        <p class="abstract">Myeloproliferative neoplasms are stem cell-driven cancers associated with a large burden of morbidity and mortality. The majority of patients present with early-stage disease, but a substantial proportion progress to myelofibrosis and/or secondary leukemia, advanced cancers with a poor prognosis and high symptom burden. Currently, it remains difficult to predict progression, and we lack therapies that reliably prevent or reverse fibrosis development. A major bottleneck to the discovery of disease-modifying therapies has been an incomplete understanding of the interplay between perturbed cellular and molecular states. Several cell types have individually been implicated, but a comprehensive analysis of myelofibrotic bone marrow is lacking. We therefore mapped the crosstalk between bone marrow cell types in myelofibrotic bone marrow. We found that inflammation and fibrosis are orchestrated by a quartet of immune and stromal cell lineages-with basophils and mast cells creating a TNF signaling hub, communicating with megakaryocytes, mesenchymal stromal cells and pro-inflammatory fibroblasts. We identified the b-galactoside binding protein galectin 1 as a striking biomarker of progression to myelofibrosis and poor survival in multiple patient cohorts, and as a promising therapeutic target, with reduced myeloproliferation and fibrosis in vitro and in vivo and improved survival following galectin 1 inhibition. In human bone marrow organoids, TNF increased galectin 1 expression, suggesting a feedback loop wherein the pro-inflammatory MPN clone creates a self-reinforcing niche, fueling progression to advanced disease. This study provides a valuable resource for studying hematopoietic cell-niche interactions, with broad relevance for cancer-associated inflammation and disorders of tissue fibrosis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552029">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552029" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552029">
        <p class="paperTitle">A genetic basis for cancer sex differences revealed in Xp11 translocation renal cell carcinoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552029" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552029" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Achom, M.; Sadagopan, A.; Bao, C.; McBride, F.; Xu, Q.; Konda, P.; Tourdot, R. W.; Li, J.; Nakhoul, M.; Gallant, D.; Ahmed, U. A.; O&#39;Toole, J.; Freeman, D.; Lee, G.-S. M.; Hecht, J.; Kauffman, E.; Einstein, D. J.; Choueiri, T. K.; Zhang, C.-Z.; Viswanathan, S. R.</p>
        <p class="info">Score: 17.6, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552029' target='https://doi.org/10.1101/2023.08.04.552029'> 10.1101/2023.08.04.552029</a></p>
        <p class="abstract">Xp11 translocation renal cell carcinoma (tRCC) is a female-predominant kidney cancer driven by translocations between the TFE3 gene on chromosome Xp11.2 and partner genes located on either chrX or on autosomes. The rearrangement processes that underlie TFE3 fusions, and whether they are linked to the female sex bias of this cancer, are largely unexplored. Moreover, whether oncogenic TFE3 fusions arise from both the active and inactive X chromosomes in females remains unknown. Here we address these questions by haplotype-specific analyses of whole-genome sequences of 29 tRCC samples from 15 patients and by re-analysis of 145 published tRCC whole-exome sequences. We show that TFE3 fusions universally arise as reciprocal translocations with minimal DNA loss or insertion at paired break ends. Strikingly, we observe a near exact 2:1 female:male ratio in TFE3 fusions arising via X:autosomal translocation (but not via X inversion), which accounts for the female predominance of tRCC. This 2:1 ratio is at least partially attributable to oncogenic fusions involving the inactive X chromosome and is accompanied by partial re-activation of silenced chrX genes on the rearranged chromosome. Our results highlight how somatic alterations involving the X chromosome place unique constraints on tumor initiation and exemplify how genetic rearrangements of the sex chromosomes can underlie cancer sex differences.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.02.551681">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.02.551681" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.02.551681">
        <p class="paperTitle">Targeting the dependence on PIK3C3-mTORC1 signaling in dormancy-prone breast cancer cells blunts metastasis initiation</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.02.551681" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.02.551681" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Elkholi, I. E.; Robert, A.; Kuasne, H.; Drapela, S.; Macleod, G.; Hebert, S.; Pacis, A.; Calderon, V.; Kleinman, C. L.; Gomes, A. P.; Aguirre-Ghiso, J. A.; Park, M.; Angers, S.; Cote, J.-F.</p>
        <p class="info">Score: 13.8, Published: 2023-08-05 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.02.551681' target='https://doi.org/10.1101/2023.08.02.551681'> 10.1101/2023.08.02.551681</a></p>
        <p class="abstract">Halting breast cancer metastatic relapses following primary tumor removal and the clinical dormant phase, remains challenging, due to a lack of specific vulnerabilities to target during dormancy. To address this, we conducted genome-wide CRISPR screens on two breast cancer cell lines with distinct dormancy properties: 4T1 (short-term dormancy) and 4T07 (prolonged dormancy). We discovered that loss of class-III PI3K, Pik3c3, revealed a unique vulnerability in 4T07 cells. Surprisingly, dormancy-prone 4T07 cells exhibited higher mTORC1 activity than 4T1 cells, due to lysosome-dependent signaling occurring at the cell periphery. Pharmacological inhibition of Pik3c3 counteracted this phenotype in 4T07 cells, and selectively reduced metastasis burden only in the 4T07 dormancy prone model. This mechanism was also detected in xenografts from human breast cancer patients, supporting that it may be relevant in humans. Our findings suggest dormant cancer cell-initiated metastasis may be prevented in patients carrying tumor cells that display PIK3C3-peripheral lysosomal signaling to mTORC1.

TeaserPIK3C3-mTORC1 signaling is a novel druggable pathway in breast cancer metastasis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552014">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552014" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552014">
        <p class="paperTitle">Loss of Polycomb Repressive Complex 2 function causes asparaginase resistance in T-acute lymphoblastic leukemia through decreased WNT pathway activity</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552014" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552014" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Lefeivre, T.; Tudose, C.; Grosu, T.-I.; Jones, L.; Leon, T.; Wynne, K.; Oliviero, G.; Smith, O.; Trinquand, A.; Mansour, M.; Ryan, C.; Bond, J.</p>
        <p class="info">Score: 11.6, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552014' target='https://doi.org/10.1101/2023.08.04.552014'> 10.1101/2023.08.04.552014</a></p>
        <p class="abstract">Loss-of-function mutations and deletions in core components of the epigenetic Polycomb Repressive Complex 2 (PRC2) are associated with poor prognosis and treatment resistance in T-acute lymphoblastic leukemia (T-ALL). We leveraged clinical mutational and transcriptional data to identify a functional link between PRC2 alterations and changes in WNT signaling pathway activity in leukemia cells. Computational integration of transcriptomic, proteomic and phosphoproteomic data from an isogenic T-ALL cellular model revealed reduced activity of the WNT-dependent stabilization of proteins (WNT/STOP) pathway in cells lacking core PRC2 factor EZH2. We discovered that PRC2 loss significantly reduced sensitivity to key T-ALL treatment asparaginase, and that this was mechanistically linked to increased cellular ubiquitination levels that bolstered leukemia cell asparagine reserves. We further found that asparaginase resistance in PRC2-depleted leukemic blasts could be mitigated by pharmaceutical proteasome inhibition, thereby providing a novel and clinically tractable means to tackle induction treatment failure in high-risk T-ALL.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.548295">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.548295" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.548295">
        <p class="paperTitle">Glioma-neuronal circuit remodeling induces regional immunosuppression</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.548295" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.548295" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Nejo, T.; Krishna, S.; Jimenez, C.; Yamamichi, A.; Young, J. S.; Lakshmanachetty, S.; Chen, T.; Phyu, S. S. S.; Ogino, H.; Watchmaker, P.; Diebold, D.; Choudhury, A.; Daniel, A. G. S.; Raleigh, D. R.; Hervey-Jumper, S. L.; Okada, H.</p>
        <p class="info">Score: 9.9, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.548295' target='https://doi.org/10.1101/2023.08.04.548295'> 10.1101/2023.08.04.548295</a></p>
        <p class="abstract">Neuronal activity-driven mechanisms impact glioblastoma cell proliferation and invasion1-7, and glioblastoma remodels neuronal circuits8,9. Distinct intratumoral regions maintain functional connectivity via a subpopulation of malignant cells that mediate tumor-intrinsic neuronal connectivity and synaptogenesis through their transcriptional programs8. However, the effects of tumor-intrinsic neuronal activity on other cells, such as immune cells, remain unknown. Here we show that regions within glioblastomas with elevated connectivity are characterized by regional immunosuppression. This was accompanied by different cell compositions and inflammatory status of tumor-associated macrophages (TAMs) in the tumor microenvironment. In preclinical intracerebral syngeneic glioblastoma models, CRISPR/Cas9 gene knockout of Thrombospondin-1 (TSP-1/Thbs1), a synaptogenic factor critical for glioma-induced neuronal circuit remodeling, in glioblastoma cells suppressed synaptogenesis and glutamatergic neuronal hyperexcitability, while simultaneously restoring antigen-presentation and pro-inflammatory responses. Moreover, TSP-1 knockout prolonged survival of immunocompetent mice harboring intracerebral syngeneic glioblastoma, but not of immunocompromised mice, and promoted infiltrations of pro-inflammatory TAMs and CD8&#43; T-cells in the tumor microenvironment. Notably, pharmacological inhibition of glutamatergic excitatory signals redirected tumor-associated macrophages toward a less immunosuppressive phenotype, resulting in prolonged survival. Altogether, our results demonstrate previously unrecognized immunosuppression mechanisms resulting from glioma-neuronal circuit remodeling and suggest future strategies targeting glioma-neuron-immune crosstalk may open up new avenues for immunotherapy.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.07.31.551378">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.07.31.551378" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.07.31.551378">
        <p class="paperTitle">The relationship between diet, plasma glucose, and cancer prevalence across vertebrates</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.07.31.551378" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.07.31.551378" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Kapsetaki, S. E.; Basile, A. J.; Compton, Z. T.; Rupp, S. M.; Duke, E. G.; Boddy, A. M.; Harrison, T. M.; Sweazea, K. L.; Maley, C. C.</p>
        <p class="info">Score: 8.3, Published: 2023-08-02 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.07.31.551378' target='https://doi.org/10.1101/2023.07.31.551378'> 10.1101/2023.07.31.551378</a></p>
        <p class="abstract">Could diet and mean plasma glucose concentration (MPGluC) explain the variation in cancer prevalence across species? We collected diet, MPGluC, and neoplasia data for 160 vertebrate species from existing databases. We found that MPGluC negatively correlates with cancer and neoplasia prevalence, mostly of gastrointestinal organs. Trophic level positively correlates with cancer and neoplasia prevalence even after controlling for species MPGluC. Most species with high MPGluC (50/78 species = 64.1%) were birds. Most species in high trophic levels (42/53 species = 79.2%) were reptiles and mammals. Our results may be explained by the evolution of insulin resistance in birds which selected for loss or downregulation of genes related to insulin-mediated glucose import in cells. This led to higher MPGluC, intracellular caloric restriction, production of fewer reactive oxygen species and inflammatory cytokines, and longer telomeres contributing to longer longevity and lower neoplasia prevalence in extant birds relative to other vertebrates.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.04.552061">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.04.552061" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.04.552061">
        <p class="paperTitle">MEK inhibition enhances the antitumor effect of radiation therapy in NF1-deficient glioblastoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.04.552061" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.04.552061" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Ioannou, M.; Lalwani, K.; Ayanlaja, A. A.; Chinnasamy, V.; Pratilas, C. A.; Schreck, K. C.</p>
        <p class="info">Score: 7.4, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.04.552061' target='https://doi.org/10.1101/2023.08.04.552061'> 10.1101/2023.08.04.552061</a></p>
        <p class="abstract">Individuals with neurofibromatosis type 1 (NF-1), an autosomal dominant neurogenetic and tumor predisposition syndrome, are susceptible to developing low-grade glioma (LGG) and, less commonly, high-grade glioma (HGG). These gliomas exhibit loss of the neurofibromin gene (NF1), and 10-15% of sporadic HGG have somatic NF1 alterations. Loss of NF1 leads to hyperactive RAS signaling, creating opportunity given the established efficacy of MEK inhibitors (MEKi) in plexiform neurofibromas and some individuals with LGG. We observed that NF1-deficient glioblastoma neurospheres were sensitive to the combination of a MEKi (mirdametinib) with irradiation, as evidenced by synergistic inhibition of cell growth, colony formation, and increased cell death. In contrast, NF1-intact neurospheres were not sensitive to the combination, despite complete ERK pathway inhibition. No neurosphere lines exhibited enhanced sensitivity to temozolomide combined with mirdametinib. Mirdametinib decreased transcription of homologous recombination genes and RAD51 foci, associated with DNA damage repair, in sensitive models. Heterotopic xenograft models displayed synergistic growthinhibition to mirdametinib combined with irradiation in NF1-deficient glioma xenografts, but not those with intact NF1. In sensitive models, benefits were observed at least three weeks beyond the completion of treatment, including sustained phospho-ERK inhibition on immunoblot and decreased Ki-67 expression. These observations demonstrate synergistic activity between mirdametinib and irradiation in NF1-deficient glioma models and may have clinical implications for patients with gliomas that harbor germline or somatic NF1 alterations.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.06.551165">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.06.551165" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.06.551165">
        <p class="paperTitle">Characterizing heterogeneity along EMT and metabolic axes in colorectal cancer reveals underlying consensus molecular subtype-specific trends</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.06.551165" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.06.551165" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sehgal, M.; R, S.; Vaz, J. M.; Raja G, Y.; Muralidharan, S.; Venkatraghavan, S.; Jolly, M. K.</p>
        <p class="info">Score: 4.6, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.06.551165' target='https://doi.org/10.1101/2023.08.06.551165'> 10.1101/2023.08.06.551165</a></p>
        <p class="abstract">Colorectal cancer (CRC) is highly heterogenous with variable survival outcomes and therapeutic vulnerabilities. A commonly used classification system in CRC is the Consensus Molecular Subtypes (CMS) based on gene expression patterns. However, how these CMS categories connect to axes of phenotypic plasticity and heterogeneity remains unclear. Here, we analyze 101 bulk transcriptomic datasets, along with patient tumor samples from TCGA and single-cell RNA sequencing data, to evaluate the extent of variation among CMS subtypes across metabolic plasticity and EMT axes. Our results show that CMS2 and CMS3 samples were relatively more epithelial as compared to CMS1 and CMS4. Single-cell RNA-seq analysis of CMS1 revealed two subpopulations: one close to CMS4 (more mesenchymal) and the other closer to CMS2 or CMS3 (more epithelial), indicating a partial EMT-like behavior. Further, in our meta-analysis and in TCGA data, epithelial phenotype score was positively correlated with scores of glycolysis, OXPHOS and FAO pathways, while mesenchymal scores showed CMS subtype-specific associations with metabolic axes. PD-L1 activity scores, however, consistently correlated positively with mesenchymal signature ones and negatively with epithelial signature ones, across the four CMS categories. Together, our results quantify the patterns of two interconnected axes of phenotypic heterogeneity - EMT and metabolic reprogramming - at a CMS subtype level in CRC.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.08.05.552118">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.08.05.552118" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.08.05.552118">
        <p class="paperTitle">Identification of a Hypoxia-induced Inflammatory Cell Death Pathway</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.08.05.552118" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.08.05.552118" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Bhardwaj, A.; Antonelli, M.; Ueberheide, B.; Neel, B. G.</p>
        <p class="info">Score: 4.0, Published: 2023-08-06 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.08.05.552118' target='https://doi.org/10.1101/2023.08.05.552118'> 10.1101/2023.08.05.552118</a></p>
        <p class="abstract">Hypoxic cancer cells resist many anti-neoplastic therapies and can seed recurrence. We found previously that PTP1B deficiency promotes HER2&#43; breast cancer cell death in hypoxia by activating RNF213, an ~600kDa protein containing AAA-ATPase domains and two ubiquitin ligase domains (RING and RZ), which is implicated in Moyamoya disease (MMD). Here we report that PTP1B and ABL1/2 reciprocally control RNF213 phosphorylation on tyrosine-1275. This phosphorylation promotes RNF213 oligomerization and RZ domain activation. The RZ domain ubiquitylates CYLD/SPATA2, and together with the LUBAC complex, induces their degradation. Decreased CYLD/SPATA2 levels cause NF-kB activation, which together with hypoxia-induced ER-stress triggers pyroptosis. Mutagenesis experiments show that the RING domain negatively regulates the RZ domain, and RING mutants, including MMD alleles, catalyze LUBAC-independent CYLD/SPATA2 degradation. Our results identify an inflammatory cell death pathway that kills hypoxic tumors, reveal new insights into RNF213 regulation, and have important implications for MMD, atherosclerosis, and inflammatory disorder pathogenesis.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
