<!DOCTYPE html>
<html lang="en" dir="auto">

<head>
<head>
    <title>cancer biology</title>
    <meta charset="utf-8">
    <meta name="description"
        content="Website meta description for google search results go here" />
    <meta name="dc.relation" content="https://trxiv.yorks0n.com" />
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="theme-color" content="#1A94D2" />

    

    
    
    
    <link rel="stylesheet" href="/css/main.min.css" media="screen">

</head>


<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>cancer biology | TRxiv2</title>
<meta name="keywords" content="">
<meta name="description" content="ASCT2 is the primary serine transporter in cancer cells
Authors: Conger, K. O.; Chidley, C.; Ozgurses, M. E.; Zhao, H.; Kim, Y.; Semina, S. E.; Burns, P. A.; Rawat, V.; Sheldon, R.; Ben-Sahra, I.; Frasor, J.; Sorger, P. K.; DeNicola, G. M.; Coloff, J. L.
Score: 24.0, Published: 2023-10-11 DOI: 10.1101/2023.10.09.561530
The non-essential amino acid serine is a critical nutrient for cancer cells due to its diverse biosynthetic functions. While some tumors can synthesize serine de novo, others are auxotrophic for serine and therefore reliant on the uptake of exogenous serine.">
<meta name="author" content="">
<link rel="canonical" href="https://trxiv.yorks0n.com/posts/cancer-biology/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.904bd1e751cdd2a584fa6bed3fa1166dfd8ec9949ebfd0c4d69c5add5e17c23d.css" integrity="sha256-kEvR51HN0qWE&#43;mvtP6EWbf2OyZSev9DE1pxa3V4Xwj0=" rel="preload stylesheet" as="style">
<script defer crossorigin="anonymous" src="/assets/js/highlight.f413e19d0714851f6474e7ee9632408e58ac146fbdbe62747134bea2fa3415e0.js" integrity="sha256-9BPhnQcUhR9kdOfuljJAjlisFG&#43;9vmJ0cTS&#43;ovo0FeA="
    onload="hljs.initHighlightingOnLoad();"></script>
<link rel="icon" href="https://trxiv.yorks0n.com/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://trxiv.yorks0n.com/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://trxiv.yorks0n.com/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://trxiv.yorks0n.com/apple-touch-icon.png">
<link rel="mask-icon" href="https://trxiv.yorks0n.com/safari-pinned-tab.svg">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
    <style>
        @media (prefers-color-scheme: dark) {
            :root {
                --theme: rgb(29, 30, 32);
                --entry: rgb(46, 46, 51);
                --primary: rgb(218, 218, 219);
                --secondary: rgb(155, 156, 157);
                --tertiary: rgb(65, 66, 68);
                --content: rgb(196, 196, 197);
                --hljs-bg: rgb(46, 46, 51);
                --code-bg: rgb(55, 56, 62);
                --border: rgb(51, 51, 51);
            }

            .list {
                background: var(--theme);
            }

            .list:not(.dark)::-webkit-scrollbar-track {
                background: 0 0;
            }

            .list:not(.dark)::-webkit-scrollbar-thumb {
                border-color: var(--theme);
            }
        }

    </style>
</noscript><meta property="og:title" content="cancer biology" />
<meta property="og:description" content="ASCT2 is the primary serine transporter in cancer cells
Authors: Conger, K. O.; Chidley, C.; Ozgurses, M. E.; Zhao, H.; Kim, Y.; Semina, S. E.; Burns, P. A.; Rawat, V.; Sheldon, R.; Ben-Sahra, I.; Frasor, J.; Sorger, P. K.; DeNicola, G. M.; Coloff, J. L.
Score: 24.0, Published: 2023-10-11 DOI: 10.1101/2023.10.09.561530
The non-essential amino acid serine is a critical nutrient for cancer cells due to its diverse biosynthetic functions. While some tumors can synthesize serine de novo, others are auxotrophic for serine and therefore reliant on the uptake of exogenous serine." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://trxiv.yorks0n.com/posts/cancer-biology/" /><meta property="article:section" content="posts" />
<meta property="article:published_time" content="2023-10-18T10:38:41+00:00" />
<meta property="article:modified_time" content="2023-10-18T10:38:41+00:00" />

<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="cancer biology"/>
<meta name="twitter:description" content="ASCT2 is the primary serine transporter in cancer cells
Authors: Conger, K. O.; Chidley, C.; Ozgurses, M. E.; Zhao, H.; Kim, Y.; Semina, S. E.; Burns, P. A.; Rawat, V.; Sheldon, R.; Ben-Sahra, I.; Frasor, J.; Sorger, P. K.; DeNicola, G. M.; Coloff, J. L.
Score: 24.0, Published: 2023-10-11 DOI: 10.1101/2023.10.09.561530
The non-essential amino acid serine is a critical nutrient for cancer cells due to its diverse biosynthetic functions. While some tumors can synthesize serine de novo, others are auxotrophic for serine and therefore reliant on the uptake of exogenous serine."/>


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Posts",
      "item": "https://trxiv.yorks0n.com/posts/"
    }, 
    {
      "@type": "ListItem",
      "position":  3 ,
      "name": "cancer biology",
      "item": "https://trxiv.yorks0n.com/posts/cancer-biology/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "cancer biology",
  "name": "cancer biology",
  "description": "ASCT2 is the primary serine transporter in cancer cells\nAuthors: Conger, K. O.; Chidley, C.; Ozgurses, M. E.; Zhao, H.; Kim, Y.; Semina, S. E.; Burns, P. A.; Rawat, V.; Sheldon, R.; Ben-Sahra, I.; Frasor, J.; Sorger, P. K.; DeNicola, G. M.; Coloff, J. L.\nScore: 24.0, Published: 2023-10-11 DOI: 10.1101/2023.10.09.561530\nThe non-essential amino acid serine is a critical nutrient for cancer cells due to its diverse biosynthetic functions. While some tumors can synthesize serine de novo, others are auxotrophic for serine and therefore reliant on the uptake of exogenous serine.",
  "keywords": [
    
  ],
  "articleBody": " ASCT2 is the primary serine transporter in cancer cells\nAuthors: Conger, K. O.; Chidley, C.; Ozgurses, M. E.; Zhao, H.; Kim, Y.; Semina, S. E.; Burns, P. A.; Rawat, V.; Sheldon, R.; Ben-Sahra, I.; Frasor, J.; Sorger, P. K.; DeNicola, G. M.; Coloff, J. L.\nScore: 24.0, Published: 2023-10-11 DOI: 10.1101/2023.10.09.561530\nThe non-essential amino acid serine is a critical nutrient for cancer cells due to its diverse biosynthetic functions. While some tumors can synthesize serine de novo, others are auxotrophic for serine and therefore reliant on the uptake of exogenous serine. Importantly, however, the transporter(s) that mediate serine uptake in cancer cells are not known. Here, we characterize the amino acid transporter ASCT2 (coded for by the gene SLC1A5) as the primary serine transporter in cancer cells. ASCT2 is well-known as a glutamine transporter in cancer, and our work demonstrates that serine and glutamine compete for uptake through ASCT2. We further show that ASCT2-mediated serine uptake is essential for purine nucleotide biosynthesis and that ER promotes serine uptake by directly activating SLC1A5 transcription. Together, our work defines an additional important role for ASCT2 as a serine transporter in cancer and evaluates ASCT2 as a potential therapeutic target in serine metabolism.\nChromosome 8 gain drives cancer progression by hijacking the translation factor 4E-BP1 sensitizing for targeted CDK4/6 inhibition\nAuthors: Funk, C. M.; Ehlers, A.; Orth, M. F.; Aljakouch, K.; Li, J.; Hoelting, T. L. B.; Will, R.; Willis, F.; Vinca, E.; Ohmura, S.; Imle, R.; Siebenlist, J.; Knott, M. M. L.; Zahnow, F.; Sastre, A.; Alonso, J.; Sahm, F.; Loboda, A.; Peterziel, H.; Schneider, M.; Banito, A.; Leprivier, G.; Hartmann, W.; Dirksen, U.; Witt, O.; Oehme, I.; Pfister, S. M.; Romero-Perez, L.; Krijgsveld, J.; Cidre-Aranaz, F.; Gruenewald, T. G. P.; Musa, J.\nScore: 26.9, Published: 2023-10-13 DOI: 10.1101/2022.12.11.519935\nChromosome 8 (chr8) gains are common in cancer. However, their potential contribution to tumor heterogeneity is largely unexplored. Ewing sarcoma (EwS) is characterized by pathognomonic FET::ETS fusions, but show a general paucity of other recurrent somatic mutations that could explain the observed clinical diversity. In EwS, chr8 gains are the second most common genetic alteration rendering EwS an ideal model to investigate the relevance of chr8 gains in an otherwise silent genomic context. Here, we report that chr8 gain-driven gene expression patterns correlate with poor overall survival of EwS patients. This effect is predominantly mediated by increased expression of the translation initiation factor binding protein 4E-BP1 encoded by EIF4EBP1 on chr8. High EIF4EBP1 expression showed the strongest association with poor overall patient survival among all chr8-encoded genes and correlated with chr8 gains in EwS tumors. Similar findings were made in numerous entities of The Cancer Genome Atlas (TCGA). In the EwS model, silencing of 4E-BP1 reduced cell proliferation, clonogenicity, spheroidal growth in vitro, and tumorigenesis in vivo. Integrated multi- omics profiling uncovered that 4E-BP1 orchestrates a multifunctional proteomic network including hubs affecting RNA processing, translational regulation, and chromatin modification. Drug screens and functional assays revealed that high 4E-BP1 expression sensitizes for pharmacological CDK4/6 inhibition in preclinical models. Collectively, we establish chr8 gains and high 4E-BP1 expression as prognostic biomarkers in EwS and demonstrate that their association with poor patient outcome is primarily mediated by 4E-BP1 orchestrating a multifunctional proteomic network sensitizing EwS for CDK4/6 inhibitors. Our data suggest that testing for chr8 gain may improve risk-stratification and therapeutic management in EwS and other cancers.\nTelomeres control regulation of the human Telomerase (hTERT) gene through non-telomeric TRF2 and independent of Telomere looping\nAuthors: Sengupta, A.; Vinayagamurthy, S.; Soni, D.; Deb, R.; Mukherjee, A. K.; Dutta, S.; Jaiswal, J.; Yadav, M.; Sharma, S.; Bagri, S.; Roy, S. S.; Singh, A.; Khanna, D.; Bhatt, A. K.; Sharma, A.; Saurav, S.; Motiani, R. K.; Chowdhury, S.\nScore: 12.7, Published: 2023-10-10 DOI: 10.1101/2023.10.09.561466\nActivation of human telomerase (hTERT) and resulting maintenance of telomeres is widely understood. Whether telomeres, on the other hand, influence hTERT regulation is relatively less studied. We found hTERT was transcriptionally up/down regulated depending on telomere length (TL), but independent of telomere looping. Cells from different tissues, engineered for either telomere elongation/shortening gave increase/decrease in hTERT, respectively. Temporal increase of TL, followed by decrease over days, resulted in induction and subsequent repression of hTERT, supporting the causal role of TL. Further confirmed by TL-dependent promoter activity from exogenously inserted hTERT reporter. Mechanistically, we show non-telomeric TRF2 binding at the hTERT promoter to be TL-dependent. Promoter bound TRF2 recruited the canonical PRC2-complex inducing repressor histone H3K27-trimethylation. Together, these reveal a heretofore unaddressed model of telomere homeostasis in telomerase-positive cells, where TL-dependent hTERT levels in turn maintain long/short telomeres. This might help better understand telomere-related physiologies like cancer, ageing and pluripotency.\nONECUT2 Activates Diverse Resistance Drivers of Androgen Receptor-Independent Heterogeneity in Prostate Cancer\nAuthors: Qian, C.; Yang, Q.; Rotinen, M.; Huang, R.; Kim, h.; Gallent, B.; Yan, Y.; Cadaneanu, R. M.; Zhang, B.; Kaochar, S.; Freedland, S. J.; Posadas, E. M.; Ellis, L.; Di Vizio, D.; Morrissey, C.; Nelson, P. S.; Brady, L.; Murali, R.; Campbell, M. J.; Yang, W.; Knudsen, B. S.; Mostaghel, E. A.; Ye, H.; Garraway, I. P.; You, S.; Freeman, M. R.\nScore: 9.8, Published: 2023-10-12 DOI: 10.1101/2023.09.28.560025\nSignificance StatementONECUT2 (OC2) is a master transcription factor that alters lineage identity by activating gene networks associated with both neuroendocrine prostate cancer and prostate adenocarcinoma. A small molecule inhibitor of OC2 represses the lineage plasticity program activated by enzalutamide, suggesting OC2 inhibition as a novel therapeutic strategy to prevent emergence of treatment-resistant variants. O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=135 SRC=\"FIGDIR/small/560025v2_ufig1.gif\" ALT=\"Figure 1\"\u003e View larger version (41K): org.highwire.dtl.DTLVardef@6005b6org.highwire.dtl.DTLVardef@172128org.highwire.dtl.DTLVardef@81e9f7org.highwire.dtl.DTLVardef@1276efb_HPS_FORMAT_FIGEXP M_FIG O_FLOATNOGraphic AbstractC_FLOATNO C_FIG Androgen receptor-(AR-) indifference is a mechanism of resistance to hormonal therapy in prostate cancer (PC). Here we demonstrate that the HOX/CUT transcription factor ONECUT2 (OC2) activates resistance through multiple drivers associated with adenocarcinoma, stem-like and neuroendocrine (NE) variants. Direct OC2 targets include the glucocorticoid receptor and the NE splicing factor SRRM4, among others. OC2 regulates gene expression by promoter binding, enhancement of chromatin accessibility, and formation of novel super-enhancers. OC2 also activates glucuronidation genes that irreversibly disable androgen, thereby evoking phenotypic heterogeneity indirectly by hormone depletion. Pharmacologic inhibition of OC2 suppresses lineage plasticity reprogramming induced by the AR signaling inhibitor enzalutamide. These results demonstrate that OC2 activation promotes a range of drug resistance mechanisms associated with treatment-emergent lineage variation in PC. Our findings support enhanced efforts to therapeutically target this protein as a means of suppressing treatment-resistant disease.\nCellular landscape of adrenocortical carcinoma at single-nuclei resolution\nAuthors: Tourigny, D. S.; Altieri, B.; Secener, K. A.; Sbiera, S.; Schauer, M. P.; Arampatzi, P.; Herterich, S.; Sauer, S. S.; Fassnacht, M.; Ronchi, C. L.\nScore: 8.6, Published: 2023-10-11 DOI: 10.1101/2023.10.09.561514\nAdrenocortical carcinoma (ACC) is a rare yet devastating tumour of the adrenal gland with a molecular pathology that remain incompletely understood. To gain novel insights into the cellular landscape of ACC, we generated single-nuclei RNA sequencing (snRNA-seq) data sets from twelve ACC tumour samples and analysed these alongside a previously published snRNA-seq data set from normal adrenal glands (NAGs). We find the ACC tumour microenvironment to be relatively devoid of immune cells compared to NAG tissues, consistent with known high tumour purity values for ACC as an immunologically \"cold\" tumour. Our analysis identifies three separate groups of ACC samples that are characterised by different relative compositions of adrenocortical cell types, including two populations (ACC 1 and ACC 2) that are specifically enriched in the most aggressive tumours and display hallmarks of the epithelial to mesenchymal transition (EMT) and dysregulated steroidogenesis, respectively. In addition to cell types associated with hypoxic and metabolic signatures (ACC 3 and ACC 4) prevalent among less-aggressive tumours, we also identified and validated a population of mitotically active adrenocortical cells (ACC M) strongly overexpressing genes POLQ and DIAPH3 that possibly supports the expansion of malignant cell lineages. The smallest identified ACC specific cell type, ACC 5, displays characteristics of increased proliferation and growth factor signalling, and is therefore a potential progenitor-like or cell-of-origin candidate for the different lineages involved in adrenocortical carcinogenesis. Intriguingly, linage tracing suggests the fate adopted by malignant adrenocortical cells upon differentiation appears to be at least partly associated with the copy number or allelic balance state of the imprinted DLK1/MEG3 genomic locus, which we verified by assessing DNA methylation status among samples from the three groups of tumours defined by their different cell type compositions. Our results therefore provide new insights into the cellular heterogeneity of ACC, indicating that genetic perturbations to a hierarchical cellular differentiation mechanism underlying healthy adrenocortical renewal and zonation may explain the molecular basis for disease pathogenesis.\nTesting Adaptive Therapy Protocols using Gemcitabine and Capecitabine on a Mouse Model of Endocrine-Resistant Breast Cancer\nAuthors: SEYEDI, S.; Teo, R.; Foster, L.; Saha, D.; Mina, L.; Northfelt, D.; Anderson, K. S.; Shibata, D.; Gatenby, R.; Cisneros, L.; Troan, B.; Anderson, A. R. A.; Maley, C. C.\nScore: 11.0, Published: 2023-09-22 DOI: 10.1101/2023.09.18.558136\nHighly effective cancer therapies often face limitations due to acquired resistance and toxicity. Adaptive therapy, an ecologically inspired approach, seeks to control therapeutic resistance and minimize toxicity by leveraging competitive interactions between drug-sensitive and drug-resistant subclones, prioritizing patient survival and quality of life over maximum cell kill. In preparation for a clinical trial in breast cancer, we used large populations of MCF7 cells to rapidly generate endocrine-resistance breast cancer cell line. We then mimicked second line therapy in ER+ breast cancers by treating the endocrine-resistant MCF7 cells in a mouse xenograft model to test adaptive therapy with capecitabine, gemcitabine, or the combination of those two drugs. Dose-modulation adaptive therapy with capecitabine alone increased survival time relative to MTD, but not statistically significant (HR: 0.22, 95% CI 0.043-1.1 P = 0.065). However, when we alternated the drugs in both dose modulation (HR = 0.11, 95% CI: 0.024 - 0.55, P = 0.007) and intermittent adaptive therapies significantly increased survival time compared to high dose combination therapy (HR = 0.07, 95% CI: 0.013 - 0.42; P = 0.003). Overall, survival time increased with reduced dose for both single drugs (P \u003c 0.01) and combined drugs (P \u003c 0.001). Adaptive therapy protocols resulted in tumors with lower proportions of proliferating cells (P = 0.0026) and more apoptotic cells (P = 0.045). The results show that Adaptive therapy outperforms high-dose therapy in controlling endocrine-resistant breast cancer, favoring slower-growing tumors, and showing promise in two-drug alternating regimens.\nState-transition Modeling of Blood Transcriptome Predicts Disease Evolution and Treatment Response in Chronic Myeloid Leukemia (CML)\nAuthors: Frankhouser, D. E.; Rockne, R. C.; Uechi, L.; Zhao, D.; Branciamore, S.; O'Meally, D.; Izarriy, J.; Ghoda, L.; Ali, H.; Trent, J.; Forman, S.; Fu, Y.-H.; Kuo, Y.-H.; Zhang, B.; Marcucci, G.\nScore: 4.8, Published: 2023-10-12 DOI: 10.1101/2023.10.11.561908\nChronic myeloid leukemia (CML) is initiated and initially maintained solely by the fusion gene BCR-ABL, encoding a multifaceted chimeric kinase targeted in the clinic with tyrosine kinase inhibitors (TKIs) TKIs are effective in inducing long-term remission, but are also frequently not curative. Thus, CML is an ideal system to test our hypothesis that transcriptome-based state-transition models accurately predict cancer evolution and treatment response. To test our hypothesis, we collected time-sequential blood samples from tetracycline-off (Tet-Off) BCR-ABL-inducible transgenic mice and wild-type controls. Using the time-series bulk RNA-seq analysis to capture a system-wide view of distinct disease states, we identified a single principal component that constructed a CML state-space with a three-well BCR-ABL leukemogenic potential landscape. The potential stable critical points defined observable disease states. Early states were characterized by anti-CML genes opposing leukemic transformation; late states were characterized by pro-CML genes. Genes with expression patterns shaped similarly to the potential landscape were identified as disease transition drivers. Re-introduction of tetracyclines to silence the BCR/ABL gene returned diseased mice transcriptomes to a stable state near to health, without reaching it, suggesting partly irreversible transformation changes. TKI treatment only reverted the diseased mice transcriptomes to an earlier disease state, without approaching health; disease relapse occurred soon after treatment completion. Using only the earliest time-point as initial conditions, our parametrized state-transition models accurately predicted both disease progression and treatment response, supporting this as a potentially valuable approach to time clinical intervention even before phenotypic changes become detectable.\nSingle cell transcriptomic profiling identifies tumor-acquired and therapy-resistant cell states in pediatric rhabdomyosarcoma\nAuthors: Danielli, S. G.; Wei, Y.; Dyer, M. A.; Stewart, E.; Wachtel, M.; Schaefer, B. W.; Patel, A. G.; Langenau, D. M.\nScore: 4.3, Published: 2023-10-17 DOI: 10.1101/2023.10.13.562224\nRhabdomyosarcoma (RMS) is a pediatric tumor that resembles undifferentiated muscle cells; yet the extent to which cell state heterogeneity and molecular features are shared with human development have not been fully ascribed. Here, we report a single-cell/nucleus RNA sequencing atlas derived from 72 datasets that includes patient tumors, patient-derived xenografts, primary in vitro cultures, and established cell lines. We report four dominant muscle-lineage cell states in RMS: progenitors, proliferative, differentiated, and ground cells. We stratify these RMS cells along the continuum of human muscle development and show that RMS cells share expression patterns with fetal/embryonal myogenic precursors rather than postnatal satellite cells. Indeed, fusion-negative RMS (FN-RMS) have a discrete stem cell hierarchy that faithfully recapitulates fetal muscle development. We also identify therapy-resistant FN-RMS progenitor cells that share transcriptomic similarity with bipotent skeletal mesenchymal cells, while a subset of fusion-positive (FP) RMS have tumor-acquired cells states, including a neuronal cell state, that are not found in development. Chemotherapy induced upregulation of progenitor signatures in FN-RMS while the neuronal gene programs were retained after therapy in FP-RMS. Taken together, this work identifies new cell state heterogeneity including unique treatment-resistant and tumor-acquired cell states that differ across RMS subtypes.\nMapping chromatin state and transcriptional response in CIC-DUX4 undifferentiated round cell sarcoma\nAuthors: Thomas, N.; Luck, C.; Shlimon, N.; Kriska Ponce, R.; Kosibaty, Z.; Okimoto, R.\nScore: 3.2, Published: 2023-10-12 DOI: 10.1101/2023.10.11.561932\nCIC-DUX4 is a rare and understudied transcription factor fusion oncoprotein. CIC-DUX4 co-opts native gene targets to drive a lethal form of human sarcoma. The molecular underpinnings that lead to oncogenic reprograming and CIC-DUX4 sarcomagenesis remain largely undefined. Through an integrative ChIP and RNA-Seq analysis using patient-derived CIC-DUX4 cells, we define CIC-DUX4 mediated chromatin states and function. We show that CIC-DUX4 primarily localizes to proximal and distal cis-regulatory elements where it associates with active histone marks. Our findings nominate key signaling pathways and molecular targets that enable CIC-DUX4 to mediate tumor cell survival. Collectively, our data demonstrate how the CIC-DUX4 fusion oncoprotein impacts chromatin state and transcriptional responses to drive an oncogenic program in undifferentiated sarcoma. SignificanceCIC-DUX4 sarcoma is a rare and lethal sarcoma that affects children, adolescent young adults, and adults. CIC-DUX4 sarcoma is associated with rapid metastatic dissemination and relative insensitivity to chemotherapy. There are no current standard-of-care therapies for CIC-DUX4 sarcoma leading to universally poor outcomes for patients. Through a deep mechanistic understanding of how the CIC-DUX4 fusion oncoprotein reprograms chromatin state and function, we aim to improve outcomes for CIC-DUX4 patients.\nRevealing Molecular Mechanisms of Early-Onset Tongue Cancer by Spatial Transcriptomics\nAuthors: Patysheva, M.; Kolegova, E.; Khozyainova, A.; Prostakishina, E.; Menyailo, M.; Larionova, I.; Kovalev, O.; Zavyalova, M.; Fedorova, I.; Kulbakin, D.; Polyakov, A.; Yakovleva, L.; Kropotov, M.; Sukortseva, N.; Arora, R.; Choinzonov, E.; Bose, P.; Denisov, E.\nScore: 2.9, Published: 2023-10-16 DOI: 10.1101/2023.10.12.562054\nTongue cancer at a young age demonstrates an increase in incidence, aggressiveness, and poor response to therapy. Classic etiological factors for head and neck tumors such as tobacco, alcohol, and human papillomavirus are not related to early-onset tongue cancer. Mechanisms of development and progression of this cancer remain unclear. In this study, we performed spatial whole-transcriptome profiling of tongue cancer in young adults compared with elderly patients. Oxidative stress, vascular mimicry, and MAPK and JAK-STAT pathways were enriched in early-onset tongue cancer. Tumor microenvironment demonstrated increased gene signatures corresponding to myeloid-derived suppressor cells, tumor-associated macrophages, and plasma cells. The invasive front was accompanied by vascular mimicry with arrangement of tumor-associated macrophages and aggregations of plasma cells and lymphocytes organized into tertiary lymphoid structures. Taken together, these results indicate that early-onset tongue cancer has distinct spatial transcriptomic features and molecular mechanisms compared to older patients.\n",
  "wordCount" : "2663",
  "inLanguage": "en",
  "datePublished": "2023-10-18T10:38:41Z",
  "dateModified": "2023-10-18T10:38:41Z",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://trxiv.yorks0n.com/posts/cancer-biology/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "TRxiv2",
    "logo": {
      "@type": "ImageObject",
      "url": "https://trxiv.yorks0n.com/favicon.ico"
    }
  }
}
</script>
</head>

<body class="" id="top">
<script>
    if (localStorage.getItem("pref-theme") === "dark") {
        document.body.classList.add('dark');
    } else if (localStorage.getItem("pref-theme") === "light") {
        document.body.classList.remove('dark')
    } else if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
        document.body.classList.add('dark');
    }

</script>

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://trxiv.yorks0n.com" accesskey="h" title="TRxiv2 (Alt + H)">TRxiv2</a>
            <div class="logo-switches">
                <button id="theme-toggle" accesskey="t" title="(Alt + T)">
                    <svg id="moon" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
                    </svg>
                    <svg id="sun" xmlns="http://www.w3.org/2000/svg" width="24" height="18" viewBox="0 0 24 24"
                        fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
                        stroke-linejoin="round">
                        <circle cx="12" cy="12" r="5"></circle>
                        <line x1="12" y1="1" x2="12" y2="3"></line>
                        <line x1="12" y1="21" x2="12" y2="23"></line>
                        <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
                        <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
                        <line x1="1" y1="12" x2="3" y2="12"></line>
                        <line x1="21" y1="12" x2="23" y2="12"></line>
                        <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
                        <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
                    </svg>
                </button>
            </div>
        </div>
        <ul id="menu">
        </ul>
    </nav>
</header>
<main class="main">
<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title">
      cancer biology
    </h1>
    <div class="post-meta"><span>updated on October 18, 2023</span>

</div>
  </header> 
  <div class="post-content"><div class="accordion accordion-flush" id="accordionFlushExample"><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561530">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561530" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561530">
        <p class="paperTitle">ASCT2 is the primary serine transporter in cancer cells</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561530" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561530" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Conger, K. O.; Chidley, C.; Ozgurses, M. E.; Zhao, H.; Kim, Y.; Semina, S. E.; Burns, P. A.; Rawat, V.; Sheldon, R.; Ben-Sahra, I.; Frasor, J.; Sorger, P. K.; DeNicola, G. M.; Coloff, J. L.</p>
        <p class="info">Score: 24.0, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561530' target='https://doi.org/10.1101/2023.10.09.561530'> 10.1101/2023.10.09.561530</a></p>
        <p class="abstract">The non-essential amino acid serine is a critical nutrient for cancer cells due to its diverse biosynthetic functions. While some tumors can synthesize serine de novo, others are auxotrophic for serine and therefore reliant on the uptake of exogenous serine. Importantly, however, the transporter(s) that mediate serine uptake in cancer cells are not known. Here, we characterize the amino acid transporter ASCT2 (coded for by the gene SLC1A5) as the primary serine transporter in cancer cells. ASCT2 is well-known as a glutamine transporter in cancer, and our work demonstrates that serine and glutamine compete for uptake through ASCT2. We further show that ASCT2-mediated serine uptake is essential for purine nucleotide biosynthesis and that ER promotes serine uptake by directly activating SLC1A5 transcription. Together, our work defines an additional important role for ASCT2 as a serine transporter in cancer and evaluates ASCT2 as a potential therapeutic target in serine metabolism.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2022.12.11.519935">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2022.12.11.519935" aria-expanded="false" aria-controls="flush-collapse10.1101/2022.12.11.519935">
        <p class="paperTitle">Chromosome 8 gain drives cancer progression by hijacking the translation factor 4E-BP1 sensitizing for targeted CDK4/6 inhibition</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2022.12.11.519935" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2022.12.11.519935" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Funk, C. M.; Ehlers, A.; Orth, M. F.; Aljakouch, K.; Li, J.; Hoelting, T. L. B.; Will, R.; Willis, F.; Vinca, E.; Ohmura, S.; Imle, R.; Siebenlist, J.; Knott, M. M. L.; Zahnow, F.; Sastre, A.; Alonso, J.; Sahm, F.; Loboda, A.; Peterziel, H.; Schneider, M.; Banito, A.; Leprivier, G.; Hartmann, W.; Dirksen, U.; Witt, O.; Oehme, I.; Pfister, S. M.; Romero-Perez, L.; Krijgsveld, J.; Cidre-Aranaz, F.; Gruenewald, T. G. P.; Musa, J.</p>
        <p class="info">Score: 26.9, Published: 2023-10-13 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2022.12.11.519935' target='https://doi.org/10.1101/2022.12.11.519935'> 10.1101/2022.12.11.519935</a></p>
        <p class="abstract">Chromosome 8 (chr8) gains are common in cancer. However, their potential contribution to tumor heterogeneity is largely unexplored. Ewing sarcoma (EwS) is characterized by pathognomonic FET::ETS fusions, but show a general paucity of other recurrent somatic mutations that could explain the observed clinical diversity. In EwS, chr8 gains are the second most common genetic alteration rendering EwS an ideal model to investigate the relevance of chr8 gains in an otherwise silent genomic context.

Here, we report that chr8 gain-driven gene expression patterns correlate with poor overall survival of EwS patients. This effect is predominantly mediated by increased expression of the translation initiation factor binding protein 4E-BP1 encoded by EIF4EBP1 on chr8. High EIF4EBP1 expression showed the strongest association with poor overall patient survival among all chr8-encoded genes and correlated with chr8 gains in EwS tumors. Similar findings were made in numerous entities of The Cancer Genome Atlas (TCGA). In the EwS model, silencing of 4E-BP1 reduced cell proliferation, clonogenicity, spheroidal growth in vitro, and tumorigenesis in vivo. Integrated multi- omics profiling uncovered that 4E-BP1 orchestrates a multifunctional proteomic network including hubs affecting RNA processing, translational regulation, and chromatin modification. Drug screens and functional assays revealed that high 4E-BP1 expression sensitizes for pharmacological CDK4/6 inhibition in preclinical models.

Collectively, we establish chr8 gains and high 4E-BP1 expression as prognostic biomarkers in EwS and demonstrate that their association with poor patient outcome is primarily mediated by 4E-BP1 orchestrating a multifunctional proteomic network sensitizing EwS for CDK4/6 inhibitors. Our data suggest that testing for chr8 gain may improve risk-stratification and therapeutic management in EwS and other cancers.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561466">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561466" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561466">
        <p class="paperTitle">Telomeres control regulation of the human Telomerase (hTERT) gene through non-telomeric TRF2 and independent of Telomere looping</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561466" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561466" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Sengupta, A.; Vinayagamurthy, S.; Soni, D.; Deb, R.; Mukherjee, A. K.; Dutta, S.; Jaiswal, J.; Yadav, M.; Sharma, S.; Bagri, S.; Roy, S. S.; Singh, A.; Khanna, D.; Bhatt, A. K.; Sharma, A.; Saurav, S.; Motiani, R. K.; Chowdhury, S.</p>
        <p class="info">Score: 12.7, Published: 2023-10-10 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561466' target='https://doi.org/10.1101/2023.10.09.561466'> 10.1101/2023.10.09.561466</a></p>
        <p class="abstract">Activation of human telomerase (hTERT) and resulting maintenance of telomeres is widely understood. Whether telomeres, on the other hand, influence hTERT regulation is relatively less studied. We found hTERT was transcriptionally up/down regulated depending on telomere length (TL), but independent of telomere looping. Cells from different tissues, engineered for either telomere elongation/shortening gave increase/decrease in hTERT, respectively. Temporal increase of TL, followed by decrease over days, resulted in induction and subsequent repression of hTERT, supporting the causal role of TL. Further confirmed by TL-dependent promoter activity from exogenously inserted hTERT reporter. Mechanistically, we show non-telomeric TRF2 binding at the hTERT promoter to be TL-dependent. Promoter bound TRF2 recruited the canonical PRC2-complex inducing repressor histone H3K27-trimethylation. Together, these reveal a heretofore unaddressed model of telomere homeostasis in telomerase-positive cells, where TL-dependent hTERT levels in turn maintain long/short telomeres. This might help better understand telomere-related physiologies like cancer, ageing and pluripotency.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.28.560025">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.28.560025" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.28.560025">
        <p class="paperTitle">ONECUT2 Activates Diverse Resistance Drivers of Androgen Receptor-Independent Heterogeneity in Prostate Cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.28.560025" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.28.560025" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Qian, C.; Yang, Q.; Rotinen, M.; Huang, R.; Kim, h.; Gallent, B.; Yan, Y.; Cadaneanu, R. M.; Zhang, B.; Kaochar, S.; Freedland, S. J.; Posadas, E. M.; Ellis, L.; Di Vizio, D.; Morrissey, C.; Nelson, P. S.; Brady, L.; Murali, R.; Campbell, M. J.; Yang, W.; Knudsen, B. S.; Mostaghel, E. A.; Ye, H.; Garraway, I. P.; You, S.; Freeman, M. R.</p>
        <p class="info">Score: 9.8, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.28.560025' target='https://doi.org/10.1101/2023.09.28.560025'> 10.1101/2023.09.28.560025</a></p>
        <p class="abstract">Significance StatementONECUT2 (OC2) is a master transcription factor that alters lineage identity by activating gene networks associated with both neuroendocrine prostate cancer and prostate adenocarcinoma. A small molecule inhibitor of OC2 represses the lineage plasticity program activated by enzalutamide, suggesting OC2 inhibition as a novel therapeutic strategy to prevent emergence of treatment-resistant variants.



O_FIG O_LINKSMALLFIG WIDTH=200 HEIGHT=135 SRC=&#34;FIGDIR/small/560025v2_ufig1.gif&#34; ALT=&#34;Figure 1&#34;&gt;
View larger version (41K):
org.highwire.dtl.DTLVardef@6005b6org.highwire.dtl.DTLVardef@172128org.highwire.dtl.DTLVardef@81e9f7org.highwire.dtl.DTLVardef@1276efb_HPS_FORMAT_FIGEXP  M_FIG O_FLOATNOGraphic AbstractC_FLOATNO C_FIG Androgen receptor-(AR-) indifference is a mechanism of resistance to hormonal therapy in prostate cancer (PC). Here we demonstrate that the HOX/CUT transcription factor ONECUT2 (OC2) activates resistance through multiple drivers associated with adenocarcinoma, stem-like and neuroendocrine (NE) variants. Direct OC2 targets include the glucocorticoid receptor and the NE splicing factor SRRM4, among others. OC2 regulates gene expression by promoter binding, enhancement of chromatin accessibility, and formation of novel super-enhancers. OC2 also activates glucuronidation genes that irreversibly disable androgen, thereby evoking phenotypic heterogeneity indirectly by hormone depletion. Pharmacologic inhibition of OC2 suppresses lineage plasticity reprogramming induced by the AR signaling inhibitor enzalutamide. These results demonstrate that OC2 activation promotes a range of drug resistance mechanisms associated with treatment-emergent lineage variation in PC. Our findings support enhanced efforts to therapeutically target this protein as a means of suppressing treatment-resistant disease.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.09.561514">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.09.561514" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.09.561514">
        <p class="paperTitle">Cellular landscape of adrenocortical carcinoma at single-nuclei resolution</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.09.561514" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.09.561514" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Tourigny, D. S.; Altieri, B.; Secener, K. A.; Sbiera, S.; Schauer, M. P.; Arampatzi, P.; Herterich, S.; Sauer, S. S.; Fassnacht, M.; Ronchi, C. L.</p>
        <p class="info">Score: 8.6, Published: 2023-10-11 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.09.561514' target='https://doi.org/10.1101/2023.10.09.561514'> 10.1101/2023.10.09.561514</a></p>
        <p class="abstract">Adrenocortical carcinoma (ACC) is a rare yet devastating tumour of the adrenal gland with a molecular pathology that remain incompletely understood. To gain novel insights into the cellular landscape of ACC, we generated single-nuclei RNA sequencing (snRNA-seq) data sets from twelve ACC tumour samples and analysed these alongside a previously published snRNA-seq data set from normal adrenal glands (NAGs). We find the ACC tumour microenvironment to be relatively devoid of immune cells compared to NAG tissues, consistent with known high tumour purity values for ACC as an immunologically &#34;cold&#34; tumour. Our analysis identifies three separate groups of ACC samples that are characterised by different relative compositions of adrenocortical cell types, including two populations (ACC 1 and ACC 2) that are specifically enriched in the most aggressive tumours and display hallmarks of the epithelial to mesenchymal transition (EMT) and dysregulated steroidogenesis, respectively. In addition to cell types associated with hypoxic and metabolic signatures (ACC 3 and ACC 4) prevalent among less-aggressive tumours, we also identified and validated a population of mitotically active adrenocortical cells (ACC M) strongly overexpressing genes POLQ and DIAPH3 that possibly supports the expansion of malignant cell lineages. The smallest identified ACC specific cell type, ACC 5, displays characteristics of increased proliferation and growth factor signalling, and is therefore a potential progenitor-like or cell-of-origin candidate for the different lineages involved in adrenocortical carcinogenesis. Intriguingly, linage tracing suggests the fate adopted by malignant adrenocortical cells upon differentiation appears to be at least partly associated with the copy number or allelic balance state of the imprinted DLK1/MEG3 genomic locus, which we verified by assessing DNA methylation status among samples from the three groups of tumours defined by their different cell type compositions. Our results therefore provide new insights into the cellular heterogeneity of ACC, indicating that genetic perturbations to a hierarchical cellular differentiation mechanism underlying healthy adrenocortical renewal and zonation may explain the molecular basis for disease pathogenesis.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.09.18.558136">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.09.18.558136" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.09.18.558136">
        <p class="paperTitle">Testing Adaptive Therapy Protocols using Gemcitabine and Capecitabine on a Mouse Model of Endocrine-Resistant Breast Cancer</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.09.18.558136" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.09.18.558136" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: SEYEDI, S.; Teo, R.; Foster, L.; Saha, D.; Mina, L.; Northfelt, D.; Anderson, K. S.; Shibata, D.; Gatenby, R.; Cisneros, L.; Troan, B.; Anderson, A. R. A.; Maley, C. C.</p>
        <p class="info">Score: 11.0, Published: 2023-09-22 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.09.18.558136' target='https://doi.org/10.1101/2023.09.18.558136'> 10.1101/2023.09.18.558136</a></p>
        <p class="abstract">Highly effective cancer therapies often face limitations due to acquired resistance and toxicity. Adaptive therapy, an ecologically inspired approach, seeks to control therapeutic resistance and minimize toxicity by leveraging competitive interactions between drug-sensitive and drug-resistant subclones, prioritizing patient survival and quality of life over maximum cell kill. In preparation for a clinical trial in breast cancer, we used large populations of MCF7 cells to rapidly generate endocrine-resistance breast cancer cell line. We then mimicked second line therapy in ER&#43; breast cancers by treating the endocrine-resistant MCF7 cells in a mouse xenograft model to test adaptive therapy with capecitabine, gemcitabine, or the combination of those two drugs. Dose-modulation adaptive therapy with capecitabine alone increased survival time relative to MTD, but not statistically significant (HR: 0.22, 95% CI 0.043-1.1 P = 0.065). However, when we alternated the drugs in both dose modulation (HR = 0.11, 95% CI: 0.024 - 0.55, P = 0.007) and intermittent adaptive therapies significantly increased survival time compared to high dose combination therapy (HR = 0.07, 95% CI: 0.013 - 0.42; P = 0.003). Overall, survival time increased with reduced dose for both single drugs (P &lt; 0.01) and combined drugs (P &lt; 0.001). Adaptive therapy protocols resulted in tumors with lower proportions of proliferating cells (P = 0.0026) and more apoptotic cells (P = 0.045). The results show that Adaptive therapy outperforms high-dose therapy in controlling endocrine-resistant breast cancer, favoring slower-growing tumors, and showing promise in two-drug alternating regimens.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561908">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561908" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561908">
        <p class="paperTitle">State-transition Modeling of Blood Transcriptome Predicts Disease Evolution and Treatment Response in Chronic Myeloid Leukemia (CML)</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561908" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561908" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Frankhouser, D. E.; Rockne, R. C.; Uechi, L.; Zhao, D.; Branciamore, S.; O&#39;Meally, D.; Izarriy, J.; Ghoda, L.; Ali, H.; Trent, J.; Forman, S.; Fu, Y.-H.; Kuo, Y.-H.; Zhang, B.; Marcucci, G.</p>
        <p class="info">Score: 4.8, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561908' target='https://doi.org/10.1101/2023.10.11.561908'> 10.1101/2023.10.11.561908</a></p>
        <p class="abstract">Chronic myeloid leukemia (CML) is initiated and initially maintained solely by the fusion gene BCR-ABL, encoding a multifaceted chimeric kinase targeted in the clinic with tyrosine kinase inhibitors (TKIs) TKIs are effective in inducing long-term remission, but are also frequently not curative. Thus, CML is an ideal system to test our hypothesis that transcriptome-based state-transition models accurately predict cancer evolution and treatment response. To test our hypothesis, we collected time-sequential blood samples from tetracycline-off (Tet-Off) BCR-ABL-inducible transgenic mice and wild-type controls. Using the time-series bulk RNA-seq analysis to capture a system-wide view of distinct disease states, we identified a single principal component that constructed a CML state-space with a three-well BCR-ABL leukemogenic potential landscape. The potential stable critical points defined observable disease states. Early states were characterized by anti-CML genes opposing leukemic transformation; late states were characterized by pro-CML genes. Genes with expression patterns shaped similarly to the potential landscape were identified as disease transition drivers. Re-introduction of tetracyclines to silence the BCR/ABL gene returned diseased mice transcriptomes to a stable state near to health, without reaching it, suggesting partly irreversible transformation changes. TKI treatment only reverted the diseased mice transcriptomes to an earlier disease state, without approaching health; disease relapse occurred soon after treatment completion. Using only the earliest time-point as initial conditions, our parametrized state-transition models accurately predicted both disease progression and treatment response, supporting this as a potentially valuable approach to time clinical intervention even before phenotypic changes become detectable.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.13.562224">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.13.562224" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.13.562224">
        <p class="paperTitle">Single cell transcriptomic profiling identifies tumor-acquired and therapy-resistant cell states in pediatric rhabdomyosarcoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.13.562224" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.13.562224" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Danielli, S. G.; Wei, Y.; Dyer, M. A.; Stewart, E.; Wachtel, M.; Schaefer, B. W.; Patel, A. G.; Langenau, D. M.</p>
        <p class="info">Score: 4.3, Published: 2023-10-17 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.13.562224' target='https://doi.org/10.1101/2023.10.13.562224'> 10.1101/2023.10.13.562224</a></p>
        <p class="abstract">Rhabdomyosarcoma (RMS) is a pediatric tumor that resembles undifferentiated muscle cells; yet the extent to which cell state heterogeneity and molecular features are shared with human development have not been fully ascribed. Here, we report a single-cell/nucleus RNA sequencing atlas derived from 72 datasets that includes patient tumors, patient-derived xenografts, primary in vitro cultures, and established cell lines. We report four dominant muscle-lineage cell states in RMS: progenitors, proliferative, differentiated, and ground cells. We stratify these RMS cells along the continuum of human muscle development and show that RMS cells share expression patterns with fetal/embryonal myogenic precursors rather than postnatal satellite cells. Indeed, fusion-negative RMS (FN-RMS) have a discrete stem cell hierarchy that faithfully recapitulates fetal muscle development. We also identify therapy-resistant FN-RMS progenitor cells that share transcriptomic similarity with bipotent skeletal mesenchymal cells, while a subset of fusion-positive (FP) RMS have tumor-acquired cells states, including a neuronal cell state, that are not found in development. Chemotherapy induced upregulation of progenitor signatures in FN-RMS while the neuronal gene programs were retained after therapy in FP-RMS. Taken together, this work identifies new cell state heterogeneity including unique treatment-resistant and tumor-acquired cell states that differ across RMS subtypes.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.11.561932">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.11.561932" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.11.561932">
        <p class="paperTitle">Mapping chromatin state and transcriptional response in CIC-DUX4 undifferentiated round cell sarcoma</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.11.561932" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.11.561932" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Thomas, N.; Luck, C.; Shlimon, N.; Kriska Ponce, R.; Kosibaty, Z.; Okimoto, R.</p>
        <p class="info">Score: 3.2, Published: 2023-10-12 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.11.561932' target='https://doi.org/10.1101/2023.10.11.561932'> 10.1101/2023.10.11.561932</a></p>
        <p class="abstract">CIC-DUX4 is a rare and understudied transcription factor fusion oncoprotein. CIC-DUX4 co-opts native gene targets to drive a lethal form of human sarcoma. The molecular underpinnings that lead to oncogenic reprograming and CIC-DUX4 sarcomagenesis remain largely undefined. Through an integrative ChIP and RNA-Seq analysis using patient-derived CIC-DUX4 cells, we define CIC-DUX4 mediated chromatin states and function. We show that CIC-DUX4 primarily localizes to proximal and distal cis-regulatory elements where it associates with active histone marks. Our findings nominate key signaling pathways and molecular targets that enable CIC-DUX4 to mediate tumor cell survival. Collectively, our data demonstrate how the CIC-DUX4 fusion oncoprotein impacts chromatin state and transcriptional responses to drive an oncogenic program in undifferentiated sarcoma.

SignificanceCIC-DUX4 sarcoma is a rare and lethal sarcoma that affects children, adolescent young adults, and adults. CIC-DUX4 sarcoma is associated with rapid metastatic dissemination and relative insensitivity to chemotherapy. There are no current standard-of-care therapies for CIC-DUX4 sarcoma leading to universally poor outcomes for patients. Through a deep mechanistic understanding of how the CIC-DUX4 fusion oncoprotein reprograms chromatin state and function, we aim to improve outcomes for CIC-DUX4 patients.</p>
      </div>
    </div>
  </div><div class="accordion-item">
    <h3 class="accordion-header" id="flush-heading10.1101/2023.10.12.562054">
      <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapse10.1101/2023.10.12.562054" aria-expanded="false" aria-controls="flush-collapse10.1101/2023.10.12.562054">
        <p class="paperTitle">Revealing Molecular Mechanisms of Early-Onset Tongue Cancer by Spatial Transcriptomics</p>
      </button>
    </h3>
    <div id="flush-collapse10.1101/2023.10.12.562054" class="accordion-collapse collapse" aria-labelledby="flush-heading10.1101/2023.10.12.562054" data-bs-parent="#accordionFlushExample">
      <div class="accordion-body">
        <p class="author">Authors: Patysheva, M.; Kolegova, E.; Khozyainova, A.; Prostakishina, E.; Menyailo, M.; Larionova, I.; Kovalev, O.; Zavyalova, M.; Fedorova, I.; Kulbakin, D.; Polyakov, A.; Yakovleva, L.; Kropotov, M.; Sukortseva, N.; Arora, R.; Choinzonov, E.; Bose, P.; Denisov, E.</p>
        <p class="info">Score: 2.9, Published: 2023-10-16 </p>
        <p class="info">DOI: <a href='https://doi.org/10.1101/2023.10.12.562054' target='https://doi.org/10.1101/2023.10.12.562054'> 10.1101/2023.10.12.562054</a></p>
        <p class="abstract">Tongue cancer at a young age demonstrates an increase in incidence, aggressiveness, and poor response to therapy. Classic etiological factors for head and neck tumors such as tobacco, alcohol, and human papillomavirus are not related to early-onset tongue cancer. Mechanisms of development and progression of this cancer remain unclear. In this study, we performed spatial whole-transcriptome profiling of tongue cancer in young adults compared with elderly patients. Oxidative stress, vascular mimicry, and MAPK and JAK-STAT pathways were enriched in early-onset tongue cancer. Tumor microenvironment demonstrated increased gene signatures corresponding to myeloid-derived suppressor cells, tumor-associated macrophages, and plasma cells. The invasive front was accompanied by vascular mimicry with arrangement of tumor-associated macrophages and aggregations of plasma cells and lymphocytes organized into tertiary lymphoid structures. Taken together, these results indicate that early-onset tongue cancer has distinct spatial transcriptomic features and molecular mechanisms compared to older patients.</p>
      </div>
    </div>
  </div>
</div>









<script src="/js/bundle.min.js" defer></script>



  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2023 <a href="https://trxiv.yorks0n.com">TRxiv2</a></span>
    <span>
        · Made by Yorkson
    </span>
</footer>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.getElementById("theme-toggle").addEventListener("click", () => {
        if (document.body.className.includes("dark")) {
            document.body.classList.remove('dark');
            localStorage.setItem("pref-theme", 'light');
        } else {
            document.body.classList.add('dark');
            localStorage.setItem("pref-theme", 'dark');
        }
    })

</script>
</body>

</html>
